{
 "cells": [
  {
   "cell_type": "code",
   "execution_count": 1,
   "metadata": {},
   "outputs": [],
   "source": [
    "###IMPORT MODULES AS APPROPRIATE###"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 2,
   "metadata": {},
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "C:\\Users\\brady\\Anaconda3\\lib\\site-packages\\sklearn\\feature_extraction\\image.py:167: DeprecationWarning: `np.int` is a deprecated alias for the builtin `int`. To silence this warning, use `int` by itself. Doing this will not modify any behavior and is safe. When replacing `np.int`, you may wish to use e.g. `np.int64` or `np.int32` to specify the precision. If you wish to review your current use, check the release note link for additional information.\n",
      "Deprecated in NumPy 1.20; for more details and guidance: https://numpy.org/devdocs/release/1.20.0-notes.html#deprecations\n",
      "  dtype=np.int):\n",
      "C:\\Users\\brady\\Anaconda3\\lib\\site-packages\\sklearn\\linear_model\\least_angle.py:35: DeprecationWarning: `np.float` is a deprecated alias for the builtin `float`. To silence this warning, use `float` by itself. Doing this will not modify any behavior and is safe. If you specifically wanted the numpy scalar type, use `np.float64` here.\n",
      "Deprecated in NumPy 1.20; for more details and guidance: https://numpy.org/devdocs/release/1.20.0-notes.html#deprecations\n",
      "  eps=np.finfo(np.float).eps,\n",
      "C:\\Users\\brady\\Anaconda3\\lib\\site-packages\\sklearn\\linear_model\\least_angle.py:597: DeprecationWarning: `np.float` is a deprecated alias for the builtin `float`. To silence this warning, use `float` by itself. Doing this will not modify any behavior and is safe. If you specifically wanted the numpy scalar type, use `np.float64` here.\n",
      "Deprecated in NumPy 1.20; for more details and guidance: https://numpy.org/devdocs/release/1.20.0-notes.html#deprecations\n",
      "  eps=np.finfo(np.float).eps, copy_X=True, fit_path=True,\n",
      "C:\\Users\\brady\\Anaconda3\\lib\\site-packages\\sklearn\\linear_model\\least_angle.py:836: DeprecationWarning: `np.float` is a deprecated alias for the builtin `float`. To silence this warning, use `float` by itself. Doing this will not modify any behavior and is safe. If you specifically wanted the numpy scalar type, use `np.float64` here.\n",
      "Deprecated in NumPy 1.20; for more details and guidance: https://numpy.org/devdocs/release/1.20.0-notes.html#deprecations\n",
      "  eps=np.finfo(np.float).eps, copy_X=True, fit_path=True,\n",
      "C:\\Users\\brady\\Anaconda3\\lib\\site-packages\\sklearn\\linear_model\\least_angle.py:862: DeprecationWarning: `np.float` is a deprecated alias for the builtin `float`. To silence this warning, use `float` by itself. Doing this will not modify any behavior and is safe. If you specifically wanted the numpy scalar type, use `np.float64` here.\n",
      "Deprecated in NumPy 1.20; for more details and guidance: https://numpy.org/devdocs/release/1.20.0-notes.html#deprecations\n",
      "  eps=np.finfo(np.float).eps, positive=False):\n",
      "C:\\Users\\brady\\Anaconda3\\lib\\site-packages\\sklearn\\linear_model\\least_angle.py:1097: DeprecationWarning: `np.float` is a deprecated alias for the builtin `float`. To silence this warning, use `float` by itself. Doing this will not modify any behavior and is safe. If you specifically wanted the numpy scalar type, use `np.float64` here.\n",
      "Deprecated in NumPy 1.20; for more details and guidance: https://numpy.org/devdocs/release/1.20.0-notes.html#deprecations\n",
      "  max_n_alphas=1000, n_jobs=None, eps=np.finfo(np.float).eps,\n",
      "C:\\Users\\brady\\Anaconda3\\lib\\site-packages\\sklearn\\linear_model\\least_angle.py:1344: DeprecationWarning: `np.float` is a deprecated alias for the builtin `float`. To silence this warning, use `float` by itself. Doing this will not modify any behavior and is safe. If you specifically wanted the numpy scalar type, use `np.float64` here.\n",
      "Deprecated in NumPy 1.20; for more details and guidance: https://numpy.org/devdocs/release/1.20.0-notes.html#deprecations\n",
      "  max_n_alphas=1000, n_jobs=None, eps=np.finfo(np.float).eps,\n",
      "C:\\Users\\brady\\Anaconda3\\lib\\site-packages\\sklearn\\linear_model\\least_angle.py:1480: DeprecationWarning: `np.float` is a deprecated alias for the builtin `float`. To silence this warning, use `float` by itself. Doing this will not modify any behavior and is safe. If you specifically wanted the numpy scalar type, use `np.float64` here.\n",
      "Deprecated in NumPy 1.20; for more details and guidance: https://numpy.org/devdocs/release/1.20.0-notes.html#deprecations\n",
      "  eps=np.finfo(np.float).eps, copy_X=True, positive=False):\n",
      "C:\\Users\\brady\\Anaconda3\\lib\\site-packages\\sklearn\\linear_model\\randomized_l1.py:152: DeprecationWarning: `np.float` is a deprecated alias for the builtin `float`. To silence this warning, use `float` by itself. Doing this will not modify any behavior and is safe. If you specifically wanted the numpy scalar type, use `np.float64` here.\n",
      "Deprecated in NumPy 1.20; for more details and guidance: https://numpy.org/devdocs/release/1.20.0-notes.html#deprecations\n",
      "  precompute=False, eps=np.finfo(np.float).eps,\n",
      "C:\\Users\\brady\\Anaconda3\\lib\\site-packages\\sklearn\\linear_model\\randomized_l1.py:320: DeprecationWarning: `np.float` is a deprecated alias for the builtin `float`. To silence this warning, use `float` by itself. Doing this will not modify any behavior and is safe. If you specifically wanted the numpy scalar type, use `np.float64` here.\n",
      "Deprecated in NumPy 1.20; for more details and guidance: https://numpy.org/devdocs/release/1.20.0-notes.html#deprecations\n",
      "  eps=np.finfo(np.float).eps, random_state=None,\n",
      "C:\\Users\\brady\\Anaconda3\\lib\\site-packages\\sklearn\\linear_model\\randomized_l1.py:580: DeprecationWarning: `np.float` is a deprecated alias for the builtin `float`. To silence this warning, use `float` by itself. Doing this will not modify any behavior and is safe. If you specifically wanted the numpy scalar type, use `np.float64` here.\n",
      "Deprecated in NumPy 1.20; for more details and guidance: https://numpy.org/devdocs/release/1.20.0-notes.html#deprecations\n",
      "  eps=4 * np.finfo(np.float).eps, n_jobs=None,\n",
      "[nltk_data] Downloading package stopwords to\n",
      "[nltk_data]     C:\\Users\\brady\\AppData\\Roaming\\nltk_data...\n",
      "[nltk_data]   Package stopwords is already up-to-date!\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "True"
      ]
     },
     "execution_count": 2,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "import nltk; nltk.download('stopwords')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 3,
   "metadata": {},
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "C:\\Users\\brady\\Anaconda3\\lib\\site-packages\\pandas\\compat\\_optional.py:138: UserWarning: Pandas requires version '2.7.0' or newer of 'numexpr' (version '2.6.9' currently installed).\n",
      "  warnings.warn(msg, UserWarning)\n",
      "C:\\Users\\brady\\Anaconda3\\lib\\site-packages\\gensim\\similarities\\__init__.py:15: UserWarning: The gensim.similarities.levenshtein submodule is disabled, because the optional Levenshtein package <https://pypi.org/project/python-Levenshtein/> is unavailable. Install Levenhstein (e.g. `pip install python-Levenshtein`) to suppress this warning.\n",
      "  warnings.warn(msg)\n",
      "C:\\Users\\brady\\Anaconda3\\lib\\site-packages\\sklearn\\decomposition\\online_lda.py:31: DeprecationWarning: `np.float` is a deprecated alias for the builtin `float`. To silence this warning, use `float` by itself. Doing this will not modify any behavior and is safe. If you specifically wanted the numpy scalar type, use `np.float64` here.\n",
      "Deprecated in NumPy 1.20; for more details and guidance: https://numpy.org/devdocs/release/1.20.0-notes.html#deprecations\n",
      "  EPS = np.finfo(np.float).eps\n"
     ]
    }
   ],
   "source": [
    "import re\n",
    "import numpy as np\n",
    "import pandas as pd\n",
    "from pprint import pprint\n",
    "\n",
    "# Gensim\n",
    "import gensim\n",
    "import gensim.corpora as corpora\n",
    "from gensim.utils import simple_preprocess\n",
    "from gensim.models import CoherenceModel\n",
    "\n",
    "# spacy for lemmatization\n",
    "import spacy\n",
    "\n",
    "# Plotting tools\n",
    "import pyLDAvis\n",
    "import pyLDAvis.gensim  # don't skip this\n",
    "import matplotlib.pyplot as plt\n",
    "%matplotlib inline\n",
    "\n",
    "# Enable logging for gensim - optional\n",
    "import logging\n",
    "logging.basicConfig(format='%(asctime)s : %(levelname)s : %(message)s', level=logging.ERROR)\n",
    "\n",
    "import warnings\n",
    "warnings.filterwarnings(\"ignore\",category=DeprecationWarning)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 4,
   "metadata": {},
   "outputs": [],
   "source": [
    "###INSERT STOP WORDS AS NEEDED###"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 5,
   "metadata": {},
   "outputs": [],
   "source": [
    "#NLTK Stop words\n",
    "from nltk.corpus import stopwords\n",
    "stop_words = stopwords.words('english')\n",
    "#stop_words.extend(['from', 'subject', 're', 'edu', 'use'])"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 6,
   "metadata": {},
   "outputs": [],
   "source": [
    "###IMPORT DATA - THIS IS CURRENTLY UGLY BUT WHATEVER###\n",
    "\n",
    "###try this later:\n",
    "#import csv\n",
    "#reviews = [row for row in csv.reader(open('reviews.csv'))]\n",
    "#print reviews"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 7,
   "metadata": {},
   "outputs": [],
   "source": [
    "df = [\n",
    "\"The American College of Medical Genetics and Genomics (ACMG) previously developed guidance for the interpretation of sequence variants. 1 In the past decade, sequencing technology has evolved -rapidly with the advent of high-throughput next-generation -sequencing. By adopting and leveraging next-generation sequencing, clinical laboratories are now performing an ever-increasing catalogue of genetic testing spanning genotyping, single genes, gene panels, exomes, genomes, transcriptomes, and epigenetic assays for genetic disorders. By virtue of increased complexity, this shift in genetic testing has been accompanied by new challenges in sequence interpretation. In this context the ACMG convened a workgroup in 2013 comprising representatives from the ACMG, the Association for Molecular Pathology (AMP), and the College of American Pathologists to revisit and revise the standards and guidelines for the interpretation of sequence variants. The group consisted of clinical laboratory directors and clinicians. This report represents expert opinion of the workgroup with input from ACMG, AMP, and College of American Pathologists stakeholders. These recommendations primarily apply to the breadth of genetic tests used in clinical laboratories, including genotyping, single genes, panels, exomes, and genomes. This report recommends the use of specific standard terminology-pathogenic, likely pathogenic, uncertain significance, likely benign, and benign-to describe variants identified in genes that cause Mendelian disorders. Moreover, this recommendation describes a process for classifying variants into these five categories based on criteria using typical types of variant evidence (e.g., population data, computational data, functional data, segregation data). Because of the increased complexity of analysis and interpretation of clinical genetic testing described in this report, the ACMG strongly recommends that clinical molecular genetic testing should be performed in a Clinical Laboratory Improvement Amendments-approved laboratory, with results interpreted by a board-certified clinical molecular geneticist or molecular genetic pathologist or the equivalent.\", \n",
    "\"There is substantial heterogeneity among primary prostate cancers, evident in the spectrum of molecular abnormalities and its variable clinical course. As part of The Cancer Genome Atlas (TCGA), we present a comprehensive molecular analysis of 333 primary prostate carcinomas. Our results revealed a molecular taxonomy in which 74% of these tumors fell into one of seven subtypes defined by specific gene fusions (ERG, ETV1/4, and FLI1) or mutations (SPOP, FOXA1, and IDH1). Epigenetic profiles showed substantial heterogeneity, including an IDH1 mutant subset with a methylator phenotype. Androgen receptor (AR) activity varied widely and in a subtype-specific manner, with SPOP and FOXA1 mutant tumors having the highest levels of AR-induced transcripts. 25% of the prostate cancers had a presumed actionable lesion in the PI3K or MAPK signaling pathways, and DNA repair genes were inactivated in 19%. Our analysis reveals molecular heterogeneity among primary prostate cancers, as well as potentially actionable molecular defects.\", \n",
    "\"PD-L1 and PD-1 (PD) pathway blockade is a highly promising therapy and has elicited durable antitumor responses and long-term remissions in a subset of patients with a broad spectrum of cancers. How to improve, widen, and predict the clinical response to anti-PD therapy is a central theme in the field of cancer immunology and immunotherapy. Oncologic, immunologic, genetic, and biological studies focused on the human cancer microenvironment have yielded substantial insight into this issue. Here, we focus on tumor microenvironment and evaluate several potential therapeutic response markers including the PD-L1 and PD-1 expression pattern, genetic mutations within cancer cells and neoantigens, cancer epigenetics and effector T cell landscape, and microbiota. We further clarify the mechanisms of action of these markers and their roles in shaping, being shaped, and/or predicting therapeutic responses. We also discuss a variety of combinations with PD pathway blockade and their scientific rationales for cancer treatment.\", \n",
    "\"Ensembl (ext-link-type=uri xlink:href=http://www.ensembl.org xlink:type=simple>http://www.ensembl.org) is a genomic interpretation system providing the most up-to-date annotations, querying tools and access methods for chordates and key model organisms. This year we released updated annotation (gene models, comparative genomics, regulatory regions and variation) on the new human assembly, GRCh38, although we continue to support researchers using the GRCh37.p13 assembly through a dedicated site (ext-link-type=uri xlink:href=http://grch37.ensembl.org xlink:type=simple>http://grch37.ensembl.org). Our Regulatory Build has been revamped to identify regulatory regions of interest and to efficiently highlight their activity across disparate epigenetic data sets. A number of new interfaces allow users to perform large-scale comparisons of their data against our annotations. The REST server (ext-link-type=uri xlink:href=http://rest.ensembl.org xlink:type=simple>http://rest.ensembl.org), which allows programs written in any language to query our databases, has moved to a full service alongside our upgraded website tools. Our online Variant Effect Predictor tool has been updated to process more variants and calculate summary statistics. Lastly, the WiggleTools package enables users to summarize large collections of data sets and view them as single tracks in Ensembl. The Ensembl code base itself is more accessible: it is now hosted on our GitHub organization page (ext-link-type=uri xlink:href=https://github.com/Ensembl xlink:type=simple>https://github.com/Ensembl) under an Apache 2.0 open source license.\", \n",
    "\"Over the past 20 years, breakthrough discoveries of chromatin-modifying enzymes and associated mechanisms that alter chromatin in response to physiological or pathological signals have transformed our knowledge of epigenetics from a collection of curious biological phenomena to a functionally dissected research field. Here, we provide a personal perspective on the development of epigenetics, from its historical origins to what we define as 'the modern era of epigenetic research'. We primarily highlight key molecular mechanisms of and conceptual advances in epigenetic control that have changed our understanding of normal and perturbed development.\", \n",
    "\"Single-molecule, real-time sequencing developed by Pacific BioSciences offers longer read lengths than the second-generation sequencing (SGS) technologies, making it well-suited for unsolved problems in genome, transcriptome, and epigenetics research. The highly-contiguous de novo assemblies using PacBio sequencing can close gaps in current reference assemblies and characterize structural variation (SV) in personal genomes. With longer reads, we can sequence through extended repetitive regions and detect mutations, many of which are associated with diseases. Moreover, PacBio transcriptome sequencing is advantageous for the identification of gene isoforms and facilitates reliable discoveries of novel genes and novel isoforms of annotated genes, due to its ability to sequence full-length transcripts or fragments with significant lengths. Additionally, PacBio's sequencing technique provides information that is useful for the direct detection of base modifications, such as methylation. In addition to using PacBio sequencing alone, many hybrid sequencing strategies have been developed to make use of more accurate short reads in conjunction with PacBio long reads. In general, hybrid sequencing strategies are more affordable and scalable especially for small-size laboratories than using PacBio Sequencing alone. The advent of PacBio sequencing has made available much information that could not be obtained via SGS alone.\", \n",
    "\"MicroRNAs (miRNAs) are endogenous, small non-coding RNAs that function in regulation of gene expression. Compelling evidences have demonstrated that miRNA expression is dysregulated in human cancer through various mechanisms, including amplification or deletion of miRNA genes, abnormal transcriptional control of miRNAs, dysregulated epigenetic changes and defects in the miRNA biogenesis machinery. MiRNAs may function as either oncogenes or tumor suppressors under certain conditions. The dysregulated miRNAs have been shown to affect the hallmarks of cancer, including sustaining proliferative signaling, evading growth suppressors, resisting cell death, activating invasion and metastasis, and inducing angiogenesis. An increasing number of studies have identified miRNAs as potential biomarkers for human cancer diagnosis, prognosis and therapeutic targets or tools, which needs further investigation and validation. In this review, we focus on how miRNAs regulate the development of human tumors by acting as tumor suppressors or oncogenes.\", \n",
    "\"'If G-quadruplexes form so readily in vitro, Nature will have found a way of using them in vivo' (Statement by Aaron Klug over 30 years ago). During the last decade, four-stranded helical structures called G-quadruplex (or G4) have emerged from being a structural curiosity observed in vitro, to being recognized as a possible nucleic acid based mechanism for regulating multiple biological processes in vivo. The sequencing of many genomes has revealed that they are rich in sequence motifs that have the potential to form G-quadruplexes and that their location is non-random, correlating with functionally important genomic regions. In this short review, we summarize recent evidence for the in vivo presence and function of DNA and RNA G-quadruplexes in various cellular pathways including DNA replication, gene expression and telomere maintenance. We also highlight remaining open questions that will have to be addressed in the future.\", \n",
    "\"The aberrant expression of transcription factor Basonuclin (BNC) had been reported in different kinds of tumors. Here, we investigated the expression and methylation status of two Basonuclin homologs, BNC1 and BNC2 in hepatocellular carcinoma (HCC). We found that the expression levels of both BNC1 and BNC2 were down-regulated in HCC cell lines and primary HCC tissues. The frequency and intensity of BNC1 promoter hypermethylation in tumor tissues was significantly higher than that in adjacent non-tumor tissues. 5-Aza-2'-Deoxycytidine treatment could significantly increase the BNC1 expression in the methylated HCC cell lines, which implicated that epigenetic modification contributed to the down-regulation of BNC1. In addition, BNC1 hypermethylation in tumor tissues was more likely to happen in female patients. No methylation of the BNC2 promoter was found in HCC tumor tissues. However, a frequent deletion of the BNC2 gene was observed, which indicated that the chromosomal loss of the BNC2 gene might be one important reason for its lower expression level in HCC. Our results suggested that BNC1 and BNC2 were down-regulated in HCC which may provide new insight into the tumorigenesis of HCC.\", \n",
    "\"An increasingly recognized resistance mechanism to androgen receptor (AR)-directed therapy in prostate cancer involves epithelial plasticity, in which tumor cells demonstrate low to absent AR expression and often have neuroendocrine features. The etiology and molecular basis for this 'alternative' treatment-resistant cell state remain incompletely understood. Here, by analyzing whole-exome sequencing data of metastatic biopsies from patients, we observed substantial genomic overlap between castration-resistant tumors that were histologically characterized as prostate adenocarcinomas (CRPC-Adeno) and neuroendocrine prostate cancer (CRPC-NE); analysis of biopsy samples from the same individuals over time points to a model most consistent with divergent clonal evolution. Genome-wide DNA methylation analysis revealed marked epigenetic differences between CRPC-NE tumors and CRPC-Adeno, and also designated samples of CRPC-Adeno with clinical features of AR independence as CRPC-NE, suggesting that epigenetic modifiers may play a role in the induction and/or maintenance of this treatment-resistant state. This study supports the emergence of an alternative, 'AR-indifferent' cell state through divergent clonal evolution as a mechanism of treatment resistance in advanced prostate cancer.\", \n",
    "\"TP53 is the most frequently mutated gene in human cancer. Functionally, p53 is activated by a host of stress stimuli and, in turn, governs an exquisitely complex anti-proliferative transcriptional program that touches upon a bewildering array of biological responses. Despite the many unveiled facets of the p53 network, a clear appreciation of how and in what contexts p53 exerts its diverse effects remains unclear. How can we interpret p53's disparate activities and the consequences of its dysfunction to understand how cell type, mutation profile, and epigenetic cell state dictate outcomes, and how might we restore its tumor-suppressive activities in cancer?\", \n",
    "\"The field of machine learning, which aims to develop computer algorithms that improve with experience, holds promise to enable computers to assist humans in the analysis of large, complex data sets. Here, we provide an overview of machine learning applications for the analysis of genome sequencing data sets, including the annotation of sequence elements and epigenetic, proteomic or metabolomic data. We present considerations and recurrent challenges in the application of supervised, semi-supervised and unsupervised machine learning methods, as well as of generative and discriminative modelling approaches. We provide general guidelines to assist in the selection of these machine learning methods and their practical application for the analysis of genetic and genomic data sets.\", \n",
    "\"Mammalian DNA methylation is a critical epigenetic mechanism orchestrating gene expression networks in many biological processes. However, investigation of the functions of specific methylation events remains challenging. Here, we demonstrate that fusion of Tet1 or Dnmt3a with a catalytically inactive Cas9 (dCas9) enables targeted DNA methylation editing. Targeting of the dCas9-Tet1 or -Dnmt3a fusion protein to methylated or unmethylated promoter sequences caused activation or silencing, respectively, of an endogenous reporter. Targeted demethylation of the BDNF promoter IV or the MyoD distal enhancer by dCas9-Tet1 induced BDNF expression in post-mitotic neurons or activated MyoD facilitating reprogramming of fibroblasts into myoblasts, respectively. Targeted de novo methylation of a CTCF loop anchor site by dCas9-Dnmt3a blocked CTCF binding and interfered with DNA looping, causing altered gene expression in the neighboring loop. Finally, we show that these tools can edit DNA methylation in mice, demonstrating their wide utility for functional studies of epigenetic regulation.\", \n",
    "\"Diverse classes of RNA, ranging from small to long non-coding RNAs, have emerged as key regulators of gene expression, genome stability and defence against foreign genetic elements. Small RNAs modify chromatin structure and silence transcription by guiding Argonaute-containing complexes to complementary nascent RNA scaffolds and then mediating the recruitment of histone and DNA methyltransferases. In addition, recent advances suggest that chromatin-associated long non-coding RNA scaffolds also recruit chromatin-modifying complexes independently of small RNAs. These co-transcriptional silencing mechanisms form powerful RNA surveillance systems that detect and silence inappropriate transcription events, and provide a memory of these events via self-reinforcing epigenetic loops.\", \n",
    "\"Next-generation sequencing has revealed that more than 50% of human cancers harbour mutations in enzymes that are involved in chromatin organization. Tumour cells not only are activated by genetic and epigenetic alterations, but also routinely use epigenetic processes to ensure their escape from chemotherapy and host immune surveillance. Hence, a growing emphasis of recent drug discovery efforts has been on targeting the epigenome, including DNA methylation and histone modifications, with several new drugs being tested and some already approved by the US Food and Drug Administration (FDA). The future will see the increasing success of combining epigenetic drugs with other therapies. As epigenetic drugs target the epigenome as a whole, these true 'genomic medicines' lessen the need for precision approaches to individualized therapies.\", \n",
    "\"One-carbon (1C) metabolism, mediated by the folate cofactor, supports multiple physiological processes. These include biosynthesis (purines and thymidine), amino acid homeostasis (glycine, serine, and methionine), epigenetic maintenance, and redox defense. Both within eukaryotic cells and across organs, 1C metabolic reactions are compartmentalized. Here we review the fundamentals of mammalian 1C metabolism, including the pathways active in different compartments, cell types, and biological states. Emphasis is given to recent discoveries enabled by modern genetics, analytical chemistry, and isotope tracing. An emerging theme is the biological importance of mitochondrial 1C reactions, both for producing 1C units that are exported to the cytosol and for making additional products, including glycine and NADPH. Increased clarity regarding differential folate pathway usage in cancer, stem cells, development, and adult physiology is reviewed and highlights new opportunities for selective therapeutic intervention.\", \n",
    "\"Acetyl-coenzyme A (acetyl-CoA) is a central metabolic intermediate. The abundance of acetyl-CoA in distinct subcellular compartments reflects the general energetic state of the cell. Moreover, acetyl-CoA concentrations influence the activity or specificity of multiple enzymes, either in an allosteric manner or by altering substrate availability. Finally, by influencing the acetylation profile of several proteins, including histones, acetyl-CoA controls key cellular processes, including energy metabolism, mitosis, and autophagy, both directly and via the epigenetic regulation of gene expression. Thus, acetyl-CoA determines the balance between cellular catabolism and anabolism by simultaneously operating as a metabolic intermediate and as a second messenger.\", \n",
    "\"Cancer growth and progression are associated with immune suppression. Cancer cells have the ability to activate different immune checkpoint pathways that harbor immunosuppressive functions. Monoclonal antibodies that target immune checkpoints provided an immense breakthrough in cancer therapeutics. Among the immune checkpoint inhibitors, PD-1/PD-L1 and CTLA-4 inhibitors showed promising therapeutic outcomes, and some have been approved for certain cancer treatments, while others are under clinical trials. Recent reports have shown that patients with various malignancies benefit from immune checkpoint inhibitor treatment. However, mainstream initiation of immune checkpoint therapy to treat cancers is obstructed by the low response rate and immune-related adverse events in some cancer patients. This has given rise to the need for developing sets of biomarkers that predict the response to immune checkpoint blockade and immune-related adverse events. In this review, we discuss different predictive biomarkers for anti-PD-1/PD-L1 and anti-CTLA-4 inhibitors, including immune cells, PD-L1 overexpression, neoantigens, and genetic and epigenetic signatures. Potential approaches for further developing highly reliable predictive biomarkers should facilitate patient selection for and decision-making related to immune checkpoint inhibitor-based therapies.\", \n",
    "\"Nucleosome positioning varies between cell types. By deep sequencing cell-free DNA (cfDNA), isolated from circulating blood plasma, we generated maps of genome-wide in vivo nucleosome occupancy and found that short cfDNA fragments harbor footprints of transcription factors. The cfDNA nucleosome occupancies correlate well with the nuclear architecture, gene structure, and expression observed in cells, suggesting that they could inform the cell type of origin. Nucleosome spacing inferred from cfDNA in healthy individuals correlates most strongly with epigenetic features of lymphoid and myeloid cells, consistent with hematopoietic cell death as the normal source of cfDNA. We build on this observation to show how nucleosome footprints can be used to infer cell types contributing to cfDNA in pathological states such as cancer. Since this strategy does not rely on genetic differences to distinguish between contributing tissues, it may enable the noninvasive monitoring of a much broader set of clinical conditions than currently possible.\", \n",
    "\"The Cas9 protein (CRISPR-associated protein 9), derived from type II CRISPR (clustered regularly interspaced short palindromic repeats) bacterial immune systems, is emerging as a powerful tool for engineering the genome in diverse organisms. As an RNA-guided DNA endonuclease, Cas9 can be easily programmed to target new sites by altering its guide RNA sequence, and its development as a tool has made sequence-specific gene editing several magnitudes easier. The nuclease-deactivated form of Cas9 further provides a versatile RNA-guided DNA-targeting platform for regulating and imaging the genome, as well as for rewriting the epigenetic status, all in a sequence-specific manner. With all of these advances, we have just begun to explore the possible applications of Cas9 in biomedical research and therapeutics. In this review, we describe the current models of Cas9 function and the structural and biochemical studies that support it. We focus on the applications of Cas9 for genome editing, regulation, and imaging, discuss other possible applications and some technical considerations, and highlight the many advantages that CRISPR/Cas9 technology offers.\", \n",
    "\"Carcinogenesis cannot be explained only by genetic alterations, but also involves epigenetic processes. Modification of histones by acetylation plays a key role in epigenetic regulation of gene expression and is controlled by the balance between histone deacetylases (HDAC) and histone acetyltransferases (HAT). HDAC inhibitors induce cancer cell cycle arrest, differentiation and cell death, reduce angiogenesis and modulate immune response. Mechanisms of anticancer effects of HDAC inhibitors are not uniform; they may be different and depend on the cancer type, HDAC inhibitors, doses, etc. HDAC inhibitors seem to be promising anti-cancer drugs particularly in the combination with other anti-cancer drugs and/or radiotherapy. HDAC inhibitors vorinostat, romidepsin and belinostat have been approved for some T-cell lymphoma and panobinostat for multiple myeloma. Other HDAC inhibitors are in clinical trials for the treatment of hematological and solid malignancies. The results of such studies are promising but further larger studies are needed. Because of the reversibility of epigenetic changes during cancer development, the potency of epigenetic therapies seems to be of great importance. Here, we summarize the data on different classes of HDAC inhibitors, mechanisms of their actions and discuss novel results of preclinical and clinical studies, including the combination with other therapeutic modalities.\", \n",
    "\"The finding that small non-coding RNAs (ncRNAs) are able to control gene expression in a sequence specific manner has had a massive impact on biology. Recent improvements in high throughput sequencing and computational prediction methods have allowed the discovery and classification of several types of ncRNAs. Based on their precursor structures, biogenesis pathways and modes of action, ncRNAs are classified as small interfering RNAs (siRNAs), microRNAs (miRNAs), PIWI-interacting RNAs (piRNAs), endogenous small interfering RNAs (endo-siRNAs or esiRNAs), promoter associate RNAs (pRNAs), small nucleolar RNAs (snoRNAs) and sno-derived RNAs. Among these, miRNAs appear as important cytoplasmic regulators of gene expression. miRNAs act as post-transcriptional regulators of their messenger RNA (mRNA) targets via mRNA degradation and/or translational repression. However, it is becoming evident that miRNAs also have specific nuclear functions. Among these, the most studied and debated activity is the miRNA-guided transcriptional control of gene expression. Although available data detail quite precisely the effectors of this activity, the mechanisms by which miRNAs identify their gene targets to control transcription are still a matter of debate. Here, we focus on nuclear functions of miRNAs and on alternative mechanisms of target recognition, at the promoter lavel, by miRNAs in carrying out transcriptional gene silencing.\", \n",
    "\"Since decades it has been known that non-protein-coding RNAs have important cellular functions. Deep sequencing recently facilitated the discovery of thousands of novel transcripts, now classified as long noncoding RNAs (lncRNAs), in many vertebrate and invertebrate species. LncRNAs are involved in a wide range of cellular mechanisms, from almost all aspects of gene expression to protein translation and stability. Recent findings implicate lncRNAs as key players of cellular differentiation, cell lineage choice, organogenesis and tissue homeostasis. Moreover, lncRNAs are involved in pathological conditions such as cancer and cardiovascular disease, and therefore provide novel biomarkers and pharmaceutical targets. Here we discuss examples illustrating the versatility of lncRNAs in gene control, development and differentiation, as well as in human disease.\", \n",
    "\"Identifying and validating molecular targets of interventions that extend the human health span and lifespan has been difficult, as most clinical biomarkers are not sufficiently representative of the fundamental mechanisms of ageing to serve as their indicators. In a recent breakthrough, biomarkers of ageing based on DNA methylation data have enabled accurate age estimates for any tissue across the entire life course. These 'epigenetic clocks' link developmental and maintenance processes to biological ageing, giving rise to a unified theory of life course. Epigenetic biomarkers may help to address long-standing questions in many fields, including the central question: why do we age?\", \n",
    "\"Epithelial-mesenchymal transition (EMT) is a process in which epithelial cells acquire mesenchymal features. In cancer, EMT is associated with tumor initiation, invasion, metastasis, and resistance to therapy. Recently, it has been demonstrated that EMT is not a binary process, but occurs through distinct cellular states. Here, we review the recent studies that demonstrate the existence of these different EMT states in cancer and the mechanisms regulating their functions. We discuss the different functional characteristics, such as proliferation, propagation, plasticity, invasion, and metastasis associated with the distinct EMT states. We summarize the role of the transcriptional and epigenetic landscapes, gene regulatory network and their surrounding niche in controlling the transition through the different EMT states.\", \n",
    "\"Methylation of DNA and of histone 3 at Lys 9 (H3K9) are highly correlated with gene silencing in eukaryotes from fungi to humans. Both of these epigenetic marks need to be established at specific regions of the genome and then maintained at these sites through cell division. Protein structural domains that specifically recognize methylated DNA and methylated histones are key for targeting enzymes that catalyse these marks to appropriate genome sites. Genetic, genomic, structural and biochemical data reveal connections between these two epigenetic marks, and these domains mediate much of the crosstalk.\", \n",
    "\"This year is the tenth anniversary of the publication in this journal of a model suggesting the existence of 'tumour progenitor genes'. These genes are epigenetically disrupted at the earliest stages of malignancies, even before mutations, and thus cause altered differentiation throughout tumour evolution. The past decade of discovery in cancer epigenetics has revealed a number of similarities between cancer genes and stem cell reprogramming genes, widespread mutations in epigenetic regulators, and the part played by chromatin structure in cellular plasticity in both development and cancer. In the light of these discoveries, we suggest here a framework for cancer epigenetics involving three types of genes: 'epigenetic mediators', corresponding to the tumour progenitor genes suggested earlier; 'epigenetic modifiers' of the mediators, which are frequently mutated in cancer; and 'epigenetic modulators' upstream of the modifiers, which are responsive to changes in the cellular environment and often linked to the nuclear architecture. We suggest that this classification is helpful in framing new diagnostic and therapeutic approaches to cancer.\", \n",
    "\"Specification of primordial germ cells (PGCs) marks the beginning of the totipotent state. However, without a tractable experimental model, the mechanism of human PGC (hPGC) specification remains unclear. Here, we demonstrate specification of hPGC-like cells (hPGCLCs) from germline competent pluripotent stem cells. The characteristics of hPGCLCs are consistent with the embryonic hPGCs and a germline seminoma that share a CD38 cell-surface marker, which collectively defines likely progression of the early human germline. Remarkably, SOX17 is the key regulator of hPGC-like fate, whereas BLIMP1 represses endodermal and other somatic genes during specification of hPGCLCs. Notable mechanistic differences between mouse and human PGC specification could be attributed to their divergent embryonic development and pluripotent states, which might affect other early cell-fate decisions. We have established a foundation for future studies on resetting of the epigenome in hPGCLCs and hPGCs for totipotency and the transmission of genetic and epigenetic information.\", \n",
    "\"The discovery of long non-coding RNA (lncRNA) has dramatically altered our understanding of cancer. Here, we describe a comprehensive analysis of lncRNA alterations at transcriptional, genomic, and epigenetic levels in 5,037 human tumor specimens across 13 cancer types from The Cancer Genome Atlas. Our results suggest that the expression and dysregulation of lncRNAs are highly cancer type specific compared with protein- coding genes. Using the integrative data generated by this analysis, we present a clinically guided small interfering RNA screening strategy and a co-expression analysis approach to identify cancer driver lncRNAs and predict their functions. This provides a resource for investigating lncRNAs in cancer and lays the groundwork for the development of new diagnostics and treatments.\", \n",
    "\"Plant genonnes encode various small RNAs that function in distinct, yet overlapping, genetic and epigenetic silencing pathways. However, the abundance and diversity of small-RNA classes varies among plant species, suggesting coevolution between environmental adaptations and gene-silencing mechanisms. Biogenesis of small RNAs in plants is well understood, but we are just beginning to uncover their intricate regulation and activity. Here, we discuss the biogenesis of plant small RNAs, such as nnicroRNAs, secondary siRNAs and heterochromatic siRNAs, and their diverse cellular and developmental functions, including in reproductive transitions, genomic imprinting and paramutation. We also discuss the diversification of small-RNA-directed silencing pathways through the expansion of RNA-dependent RNA polymerases, DICER proteins and ARGONAUTE proteins.\", \n",
    "\"Epigenetic modifications, including DNA methylation, represent a potential mechanism for environmental impacts on human disease. Maternal smoking in pregnancy remains an important public health problem that impacts child health in a myriad of ways and has potential lifelong consequences. The mechanisms are largely unknown, but epigenetics most likely plays a role. We formed the Pregnancy And Childhood Epigenetics (PACE) consortium and meta-analyzed, across 13 cohorts (n = 6,685), the association between maternal smoking in pregnancy and newborn blood DNA methylation at over 450,000 CpG sites (CpGs) by using the Illumina 450K BeadChip. Over 6,000 CpGs were differentially methylated in relation to maternal smoking at genome-wide statistical significance (false discovery rate, 5%), including 2,965 CpGs corresponding to 2,017 genes not previously related to smoking and methylation in either newborns or adults. Several genes are relevant to diseases that can be caused by maternal smoking (e.g., orofacial clefts and asthma) or adult smoking (e.g., certain cancers). A number of differentially methylated CpGs were associated with gene expression. We observed enrichment in pathways and processes critical to development. In older children (5 cohorts, n = 3,187), 100% of CpGs gave at least nominal levels of significance, far more than expected by chance (p value < 2.2 x 10(-16)). Results were robust to different normalization methods used across studies and cell type adjustment. In this large scale meta-analysis of methylation data, we identified numerous loci involved in response to maternal smoking in pregnancy with persistence into later childhood and provide insights into mechanisms underlying effects of this important exposure.\", \n",
    "\"In 1964, Vincent Allfrey and colleagues reported the identification of histone acetylation and with deep insight proposed a regulatory role for this protein modification in transcription regulation. Subsequently, histone acetyltransferases (HATs), histone deacetylases (HDACs) and acetyl-Lys-binding proteins were identified as transcription regulators, thereby providing compelling evidence for his daring hypothesis. During the past 15 years, reversible protein acetylation and its modifying enzymes have been implicated in many cellular functions beyond transcription regulation. Here, we review the progress accomplished during the past 50 years and discuss the future of protein acetylation.\", \n",
    "\"N-6-methyladenosine (m(6)A) is the most prevalent and internal modification that occurs in the messenger RNAs (mRNA) of most eukaryotes, although its functional relevance remained a mystery for decades. This modification is installed by the m(6)A methylation writers and can be reversed by demethylases that serve as erasers. In this review, we mainly summarize recent progress in the study of the m(6)A mRNA methylation machineries across eukaryotes and discuss their newly uncovered biological functions. The broad roles of m(6)A in regulating cell fates and embryonic development highlight the existence of another layer of epigenetic regulation at the RNA level, where mRNA is subjected to chemical modifications that affect protein expression.\", \n",
    "\"The Bromo- and Extra-Terminal (BET) proteins BRD2, BRD3, and BRD4 play important roles in transcriptional regulation, epigenetics, and cancer and are the targets of pan-BET selective bromodomain inhibitor JQ1. However, the lack of intra-BET selectivity limits the scope of current inhibitors as probes for target validation and could lead to unwanted side effects or toxicity in a therapeutic setting. We designed Proteolysis Targeted Chimeras (PROTACs) that tether JQ1 to a ligand for the E3 ubiquitin ligase VHL, aimed at triggering the intracellular destruction of BET proteins. Compound MZ1 potently and rapidly induces reversible, long-lasting, and unexpectedly selective removal of BRD4 over BRD2 and BRD3. The activity of MZ1 is dependent on binding to VHL but is achieved at a sufficiently low concentration not to induce stabilization of HIF-1 alpha. Gene expression profiles of selected cancer-related genes responsive to JQ1 reveal distinct and more limited transcriptional responses induced by MZ1, consistent with selective suppression of BRD4. Our discovery opens up new opportunities to elucidate the cellular phenotypes and therapeutic implications associated with selective targeting of BRD4.\", \n",
    "\"We define the chromatin accessibility and transcriptional landscapes in 13 human primary blood cell types that span the hematopoietic hierarchy. Exploiting the finding that the enhancer landscape better reflects cell identity than mRNA levels, we enable 'enhancer cytometry' for enumeration of pure cell types from complex populations. We identify regulators governing hematopoietic differentiation and further show the lineage ontogeny of genetic elements linked to diverse human diseases. In acute myeloid leukemia (AML), chromatin accessibility uncovers unique regulatory evolution in cancer cells with a progressively increasing mutation burden. Single AML cells exhibit distinctive mixed regulome profiles corresponding to disparate developmental stages. A method to account for this regulatory heterogeneity identified cancer-specific deviations and implicated HOX factors as key regulators of preleukemic hematopoietic stem cell characteristics. Thus, regulome dynamics can provide diverse insights into hematopoietic development and disease.\", \n",
    "\"Treatment of cancer has been revolutionized by immune checkpoint blockade therapies. Despite the high rate of response in advanced melanoma, the majority of patients succumb to disease. To identify factors associated with success or failure of checkpoint therapy, we profiled transcriptomes of 16,291 individual immune cells from 48 tumor samples of melanoma patients treated with checkpoint inhibitors. Two distinct states of CD8(+) T cells were defined by clustering and associated with patient tumor regression or progression. A single transcription factor, TCF7, was visualized within CD8(+) T cells in fixed tumor samples and predicted positive clinical outcome in an independent cohort of checkpoint-treated patients. We delineated the epigenetic landscape and clonality of these T cell states and demonstrated enhanced antitumor immunity by targeting novel combinations of factors in exhausted cells. Our study of immune cell transcriptomes from tumors demonstrates a strategy for identifying predictors, mechanisms, and targets for enhancing checkpoint immunotherapy.\", \n",
    "\"The extent of tumor heterogeneity is an emerging theme that researchers are only beginning to understand. How genetic and epigenetic heterogeneity affects tumor evolution and clinical progression is unknown. The precise nature of the environmental factors that influence this heterogeneity is also yet to be characterized. Nature Medicine, Nature Biotechnology and the Volkswagen Foundation organized a meeting focused on identifying the obstacles that need to be overcome to advance translational research in and tumor heterogeneity. Once these key questions were established, the attendees devised potential solutions. Their ideas are presented here.\", \n",
    "\"Single-stranded guanine-rich DNA sequences can fold into four-stranded DNA structures called G-quadruplexes (G4s) that arise from the self-stacking of two or more guanine quartets. There has been considerable recent progress in the detection and mapping of G4 structures in the human genome and in biologically relevant contexts. These advancements, many of which align with predictions made previously in computational studies, provide important new insights into the functions of G4 structures in, for example, the regulation of transcription and genome stability, and uncover their potential relevance for cancer therapy.\", \n",
    "\"Humans show strong sex differences in immunity to infection and autoimmunity, suggesting sex hormones modulate immune responses. Indeed, receptors for estrogens (ERs) regulate cells and pathways in the innate and adaptive immune system, as well as immune cell development. ERs are ligand-dependent transcription factors that mediate long-range chromatin interactions and form complexes at gene regulatory elements, thus promoting epigenetic changes and transcription. ERs also participate in membrane-initiated steroid signaling to generate rapid responses. Estradiol and ER activity show profound dose- and context-dependent effects on innate immune signaling pathways and myeloid cell development. While estradiol most often promotes the production of type I interferon, innate pathways leading to pro-inflammatory cytokine production may be enhanced or dampened by ER activity. Regulation of innate immune cells and signaling by ERs may contribute to the reported sex differences in innate immune pathways. Here we review the recent literature and highlight several molecular mechanisms by which ERs regulate the development or functional responses of innate immune cells. (C) 2015 Elsevier Inc. All rights reserved.\", \n",
    "\"Epigenetic studies relied so far on correlations between epigenetic marks and gene expression pattern. Technologies developed for epigenome editing now enable direct study of functional relevance of precise epigenetic modifications and gene regulation. The reversible nature of epigenetic modifications, including DNA methylation, has been already exploited in cancer therapy for remodeling the aberrant epigenetic landscape. However, this was achieved non-selectively using epigenetic inhibitors. Epigenetic editing at specific loci represents a novel approach that might selectively and heritably alter gene expression. Here, we developed a CRISPR-Cas9-based tool for specific DNA methylation consisting of deactivated Cas9 (dCas9) nuclease and catalytic domain of the DNA methyltransferase DNMT3A targeted by co-expression of a guide RNA to any 20 bp DNA sequence followed by the NGG trinucleotide. We demonstrated targeted CpG methylation in a similar to 35 bp wide region by the fusion protein. We also showed that multiple guide RNAs could target the dCas9-DNMT3A construct to multiple adjacent sites, which enabled methylation of a larger part of the promoter. DNA methylation activity was specific for the targeted region and heritable across mitotic divisions. Finally, we demonstrated that directed DNA methylation of a wider promoter region of the target loci IL6ST and BACH2 decreased their expression.\", \n",
    "\"The DNA methyltransferase (DNMT) family comprises a conserved set of DNA-modifying enzymes that have a central role in epigenetic gene regulation. Recent studies have shown that the functions of the canonical DNMT enzymes-DNMT1, DNMT3A and DNMT3B-go beyond their traditional roles of establishing and maintaining DNA methylation patterns. This Review analyses how molecular interactions and changes in gene copy numbers modulate the activity of DNMTs in diverse gene regulatory functions, including transcriptional silencing, transcriptional activation and post-transcriptional regulation by DNMT2-dependent tRNA methylation. This mechanistic diversity enables the DNMT family to function as a versatile toolkit for epigenetic regulation.\", \n",
    "\"Estimates of biological age based on DNA methylation patterns, often referred to as epigenetic age, DNAm age, have been shown to be robust biomarkers of age in humans. We previously demonstrated that independent of chronological age, epigenetic age assessed in blood predicted all-cause mortality in four human cohorts. Here, we expanded our original observation to 13 different cohorts for a total sample size of 13,089 individuals, including three racial/ethnic groups. In addition, we examined whether incorporating information on blood cell composition into the epigenetic age metrics improves their predictive power for mortality. All considered measures of epigenetic age acceleration were predictive of mortality (p <= 8.2x10(-9)), independent of chronological age, even after adjusting for additional risk factors (p<5.4x10(-4)), and within the racial/ethnic groups that we examined (non-Hispanic whites, Hispanics, African Americans). Epigenetic age estimates that incorporated information on blood cell composition led to the smallest p-values for time to death (p=7.5x10(-43)). Overall, this study a) strengthens the evidence that epigenetic age predicts all-cause mortality above and beyond chronological age and traditional risk factors, and b) demonstrates that epigenetic age estimates that incorporate information on blood cell counts lead to highly significant associations with all-cause mortality.\", \n",
    "\"Childhood trauma confers higher risk of adulthood physical and mental illness; however, the biological mechanism mediating this association remains largely unknown. Recent research has suggested dysregulation of the immune system as a possible biological mediator. The present paper conducted a meta-analysis to establish whether early-life adversity contributes to potentially pathogenic pro-inflammatory phenotypes in adult individuals. A systematic search of Pubmed, PsycINFO, EMBASE, Scopus and Medline identified 25 articles for the meta-analysis, including 18 studies encompassing a sample of 16 870 individuals for C-reactive protein (CRP), 15 studies including 3751 individuals for interleukin-6 (IL-6) and 10 studies including 881 individuals for tumour necrosis factor-alpha (TNF-alpha). Random-effects meta-analysis showed that individuals exposed to childhood trauma had significantly elevated baseline peripheral levels of CRP (Fisher's z = 0.10, 95% confidence interval (CI) = 0.05-0.14), IL-6 (z = 0.08, 95% CI = 0.03-0.14) and TNF-alpha (z = 0.23, 95% CI = 0.14-0.32). Subgroup analyses for specific types of trauma (sexual, physical or emotional abuse) revealed that these impact differentially the single inflammatory markers. Moreover, meta-regression revealed greater effect sizes in clinical samples for the association between childhood trauma and CRP but not for IL-6 or TNF-alpha. Age, body mass index (BMI) and gender had no moderating effects. The analysis demonstrates that childhood trauma contributes to a pro-inflammatory state in adulthood, with specific inflammatory profiles depending on the specific type of trauma.\", \n",
    "\"The heart must continuously pump blood to supply the body with oxygen and nutrients. To maintain the high energy consumption required by this role, the heart is equipped with multiple complex biological systems that allow adaptation to changes of systemic demand. The processes of growth (hypertrophy), angiogenesis, and metabolic plasticity are critically involved in maintenance of cardiac homeostasis. Cardiac hypertrophy is classified as physiological when it is associated with normal cardiac function or as pathological when associated with cardiac dysfunction. Physiological hypertrophy of the heart occurs in response to normal growth of children or during pregnancy, as well as in athletes. In contrast, pathological hypertrophy is induced by factors such as prolonged and abnormal hemodynamic stress, due to hypertension, myocardial infarction etc. Pathological hypertrophy is associated with fibrosis, capillary rarefaction, increased production of pro-inflammatory cytokines, and cellular dysfunction (impairment of signaling, suppression of autophagy, and abnormal cardiomyocyte/non-cardiomyocyte interactions), as well as undesirable epigenetic changes, with these complex responses leading to maladaptive cardiac remodeling and heart failure. This review describes the key molecules and cellular responses involved in physiological/pathological cardiac hypertrophy. (C) 2016 Elsevier Ltd. All rights reserved.\", \n",
    "\"The process of aging results in a host of changes at the cellular and molecular levels, which include senescence, telomere shortening, and changes in gene expression. Epigenetic patterns also change over the lifespan, suggesting that epigenetic changes may constitute an important component of the aging process. The epigenetic mark that has been most highly studied is DNA methylation, the presence of methyl groups at CpG dinucleotides. These dinucleotides are often located near gene promoters and associate with gene expression levels. Early studies indicated that global levels of DNA methylation increase over the first few years of life and then decrease beginning in late adulthood. Recently, with the advent of microarray and next-generation sequencing technologies, increases in variability of DNA methylation with age have been observed, and a number of site-specific patterns have been identified. It has also been shown that certain CpG sites are highly associated with age, to the extent that prediction models using a small number of these sites can accurately predict the chronological age of the donor. Together, these observations point to the existence of two phenomena that both contribute to age-related DNA methylation changes: epigenetic drift and the epigenetic clock. In this review, we focus on healthy human aging throughout the lifetime and discuss the dynamics of DNA methylation as well as how interactions between the genome, environment, and the epigenome influence aging rates. We also discuss the impact of determining 'epigenetic age' for human health and outline some important caveats to existing and future studies.\", \n",
    "\"The ability to rapidly restore the integrity of a broken skin barrier is critical and is the ultimate goal of therapies for hard-to-heal-ulcers. Unfortunately effective treatments to enhance healing and reduce scarring are still lacking. A deeper understanding of the physiology of normal repair and of the pathology of delayed healing is a prerequisite for the development of more effective therapeutic interventions. Transition from the inflammatory to the proliferative phase is a key step during healing and accumulating evidence associates a compromised transition with wound healing disorders. Thus, targeting factors that impact this phase transition may offer a rationale for therapeutic development. This review summarizes mechanisms regulating the inflammation-proliferation transition at cellular and molecular levels. We propose that identification of such mechanisms will reveal promising targets for development of more effective therapies.\", \n",
    "\"Adult somatic stem cells in various organs maintain homeostatic tissue regeneration and enhance plasticity. Since its initial discovery five decades ago, investigations of adult neurogenesis and neural stem cells have led to an established and expanding field that has significantly influenced many facets of neuroscience, developmental biology, and regenerative medicine. Here we review recent progress and focus on questions related to adult mammalian neural stem cells that also apply to other somatic stem cells. We further discuss emerging topics that are guiding the field toward better understanding adult neural stem cells and ultimately applying these principles to improve human health.\", \n",
    "\"The dogma that adaptive immunity is the only arm of the immune response with memory capacity has been recently challenged by several studies demonstrating evidence for memory-like innate immune training. However, the underlying mechanisms and location for generating such innate memory responses in vivo remain unknown. Here, we show that access of Bacillus Calmette-Guerin (BCG) to the bone marrow (BM) changes the transcriptional landscape of hematopoietic stem cells (HSCs) and multipotent progenitors (MPPs), leading to local cell expansion and enhanced myelopoiesis at the expense of lymphopoiesis. Importantly, BCG-educated HSCs generate epigenetically modified macrophages that provide significantly better protection against virulent M. tuberculosis infection than naive macrophages. By using parabiotic and chimeric mice, as well as adoptive transfer approaches, we demonstrate that training of the monocyte/macrophage lineage via BCG-induced HSC reprogramming is sustainable in vivo. Our results indicate that targeting the HSC compartment provides a novel approach for vaccine development.\", \n",
    "\"Experimental modelling of human disorders enables the definition of the cellular and molecular mechanisms underlying diseases and the development of therapies for treating them. The availability of human pluripotent stem cells (PSCs), which are capable of self-renewal and have the potential to differentiate into virtually any cell type, can now help to overcome the limitations of animal models for certain disorders. The ability to model human diseases using cultured PSCs has revolutionized the ways in which we study monogenic, complex and epigenetic disorders, as well as early- and late-onset diseases. Several strategies are used to generate such disease models using either embryonic stem cells (ES cells) or patient-specific induced PSCs (iPSCs), creating new possibilities for the establishment of models and their use in drug screening.\", \n",
    "\"Resetting of the epigenome in human primordial germ cells (hPGCs) is critical for development. We show that the transcriptional program of hPGCs is distinct from that in mice, with co-expression of somatic specifiers and naive pluripotency genes TFCP2L1 and KLF4. This unique gene regulatory network, established by SOX17 and BLIMP1, drives comprehensive germline DNA demethylation by repressing DNA methylation pathways and activating TET-mediated hydroxymethylation. Base-resolution methylome analysis reveals progressive DNA demethylation to basal levels in week 5-7 in vivo hPGCs. Concurrently, hPGCs undergo chromatin reorganization, X reactivation, and imprint erasure. Despite global hypomethylation, evolutionarily young and potentially hazardous retroelements, like SVA, remain methylated. Remarkably, some loci associated with metabolic and neurological disorders are also resistant to DNA demethylation, revealing potential for transgenerational epigenetic inheritance that may have phenotypic consequences. We provide comprehensive insight on early human germline transcriptional network and epigenetic reprogramming that subsequently impacts human development and disease.\", \n",
    "\"The cBioPortal for Cancer Genomics (http://cbioportal.org) provides a Web resource for exploring, visualizing, and analyzing multidimensional cancer genomics data. The portal reduces molecular profiling data from cancer tissues and cell lines into readily understandable genetic, epigenetic, gene expression, and proteomic events. The query interface combined with customized data storage enables researchers to interactively explore genetic alterations across samples, genes, and pathways and, when available in the underlying data, to link these to clinical outcomes. The portal provides graphical summaries of gene-level data from multiple platforms, network visualization and analysis, survival analysis, patient-centric queries, and software programmatic access. The intuitive Web interface of the portal makes complex cancer genomics profiles accessible to researchers and clinicians without requiring bioinformatics expertise, thus facilitating biological discoveries. Here, we provide a practical guide to the analysis and visualization features of the cBioPortal for Cancer Genomics.\", \n",
    "\"Aging is characterized by a progressive loss of physiological integrity, leading to impaired function and increased vulnerability to death. This deterioration is the primary risk factor for major human pathologies, including cancer, diabetes, cardiovascular disorders, and neurodegenerative diseases. Aging research has experienced an unprecedented advance over recent years, particularly with the discovery that the rate of aging is controlled, at least to some extent, by genetic pathways and biochemical processes conserved in evolution. This Review enumerates nine tentative hallmarks that represent common denominators of aging in different organisms, with special emphasis on mammalian aging. These hallmarks are: genomic instability, telomere attrition, epigenetic alterations, loss of proteostasis, deregulated nutrient sensing, mitochondrial dysfunction, cellular senescence, stem cell exhaustion, and altered intercellular communication. A major challenge is to dissect the interconnectedness between the candidate hallmarks and their relative contributions to aging, with the final goal of identifying pharmaceutical targets to improve human health during aging, with minimal side effects.\", \n",
    "\"DNA methylation is frequently described as a 'silencing' epigenetic mark, and indeed this function of 5-methylcytosine was originally proposed in the 1970s. Now, thanks to improved genome-scale mapping of methylation, we can evaluate DNA methylation in different genomic contexts: transcriptional start sites with or without CpG islands, in gene bodies, at regulatory elements and at repeat sequences. The emerging picture is that the function of DNA methylation seems to vary with context, and the relationship between DNA methylation and transcription is more nuanced than we realized at first. Improving our understanding of the functions of DNA methylation is necessary for interpreting changes in this mark that are observed in diseases such as cancer.\", \n",
    "\"The Cancer Genome Atlas (TCGA) Research Network has profiled and analyzed large numbers of human tumors to discover molecular aberrations at the DNA, RNA, protein and epigenetic levels. The resulting rich data provide a major opportunity to develop an integrated picture of commonalities, differences and emergent themes across tumor lineages. The Pan-Cancer initiative compares the first 12 tumor types profiled by TCGA. Analysis of the molecular aberrations and their functional roles across tumor types will teach us how to extend therapies effective in one cancer type to others with a similar genomic profile.\", \n",
    "\"The central dogma of gene expression is that DNA is transcribed into messenger RNAs, which in turn serve as the template for protein synthesis. The discovery of extensive transcription of large RNA transcripts that do not code for proteins, termed long noncoding RNAs (lncRNAs), provides an important new perspective on the centrality of RNA in gene regulation. Here, we discuss genome-scale strategies to discover and characterize lncRNAs. An emerging theme from multiple model systems is that lncRNAs form extensive networks of ribonucleoprotein (RNP) complexes with numerous chromatin regulators and then target these enzymatic activities to appropriate locations in the genome. Consistent with this notion, lncRNAs can function as modular scaffolds to specify higher-order organization in RNP complexes and in chromatin states. The importance of these modes of regulation is underscored by the newly recognized roles of long RNAs for proper gene control across all kingdoms of life.\", \n",
    "\"Metastases represent the end products of a multistep cell-biological process termed the invasion-metastasis cascade, which involves dissemination of cancer cells to anatomically distant organ sites and their subsequent adaptation to foreign tissue microenvironments. Each of these events is driven by the acquisition of genetic and/or epigenetic alterations within tumor cells and the co-option of nonneoplastic stromal cells, which together endow incipient metastatic cells with traits needed to generate macroscopic metastases. Recent advances provide provocative insights into these cell-biological and molecular changes, which have implications regarding the steps of the invasion-metastasis cascade that appear amenable to therapeutic targeting.\", \n",
    "\"Background: It is not yet known whether DNA methylation levels can be used to accurately predict age across a broad spectrum of human tissues and cell types, nor whether the resulting age prediction is a biologically meaningful measure. Results: I developed a multi-tissue predictor of age that allows one to estimate the DNA methylation age of most tissues and cell types. The predictor, which is freely available, was developed using 8,000 samples from 82 Illumina DNA methylation array datasets, encompassing 51 healthy tissues and cell types. I found that DNA methylation age has the following properties: first, it is close to zero for embryonic and induced pluripotent stem cells; second, it correlates with cell passage number; third, it gives rise to a highly heritable measure of age acceleration; and, fourth, it is applicable to chimpanzee tissues. Analysis of 6,000 cancer samples from 32 datasets showed that all of the considered 20 cancer types exhibit significant age acceleration, with an average of 36 years. Low age-acceleration of cancer tissue is associated with a high number of somatic mutations and TP53 mutations, while mutations in steroid receptors greatly accelerate DNA methylation age in breast cancer. Finally, I characterize the 353 CpG sites that together form an aging clock in terms of chromatin states and tissue variance. Conclusions: I propose that DNA methylation age measures the cumulative effect of an epigenetic maintenance system. This novel epigenetic clock can be used to address a host of questions in developmental biology, cancer and aging research.\", \n",
    "\"Cytosine DNA methylation is a stable epigenetic mark that is crucial for diverse biological processes, including gene and transposon silencing, imprinting and X chromosome inactivation. Recent findings in plants and animals have greatly increased our understanding of the pathways used to accurately target, maintain and modify patterns of DNA methylation and have revealed unanticipated mechanistic similarities between these organisms. Key roles have emerged for small RNAs, proteins with domains that bind methylated DNA and DNA glycosylases in these processes. Drawing on insights from both plants and animals should deepen our understanding of the regulation and biological significance of DNA methylation.\", \n",
    "\"A flexible statistical framework is developed for the analysis of read counts from RNA-Seq gene expression studies. It provides the ability to analyse complex experiments involving multiple treatment conditions and blocking variables while still taking full account of biological variation. Biological variation between RNA samples is estimated separately from the technical variation associated with sequencing technologies. Novel empirical Bayes methods allow each gene to have its own specific variability, even when there are relatively few biological replicates from which to estimate such variability. The pipeline is implemented in the edgeR package of the Bioconductor project. A case study analysis of carcinoma data demonstrates the ability of generalized linear model methods (GLMs) to detect differential expression in a paired design, and even to detect tumour-specific expression changes. The case study demonstrates the need to allow for gene-specific variability, rather than assuming a common dispersion across genes or a fixed relationship between abundance and variability. Genewise dispersions de-prioritize genes with inconsistent results and allow the main analysis to focus on changes that are consistent between biological replicates. Parallel computational approaches are developed to make non-linear model fitting faster and more reliable, making the application of GLMs to genomic data more convenient and practical. Simulations demonstrate the ability of adjusted profile likelihood estimators to return accurate estimators of biological variability in complex situations. When variation is gene-specific, empirical Bayes estimators provide an advantageous compromise between the extremes of assuming common dispersion or separate genewise dispersion. The methods developed here can also be applied to count data arising from DNA-Seq applications, including ChIP-Seq for epigenetic marks and DNA methylation analyses.\", \n",
    "\"Cancer-associated IDH mutations are characterized by neomorphic enzyme activity and resultant 2-hydroxyglutarate (2HG) production. Mutational and epigenetic profiling of a large acute myeloid leukemia (AML) patient cohort revealed that IDH1/2-mutant AMLs display global DNA hypermethylation and a specific hypermethylation signature. Furthermore, expression of 2HG-producing IDH alleles in cells induced global DNA hypermethylation. In the AML cohort, IDH1/2 mutations were mutually exclusive with mutations in the alpha-ketoglutarate-dependent enzyme TET2, and TET2 loss-of-function mutations were associated with similar epigenetic defects as IDH1/2 mutants. Consistent with these genetic and epigenetic data, expression of IDH mutants impaired TET2 catalytic function in cells. Finally, either expression of mutant IDH1/2 or Tet2 depletion impaired hematopoietic differentiation and increased stem/progenitor cell marker expression, suggesting a shared proleukemogenic effect.\", \n",
    "\"The epigenetic regulation of DNA-templated processes has been intensely studied over the last 15 years. DNA methylation, histone modification, nucleosome remodeling, and RNA-mediated targeting regulate many biological processes that are fundamental to the genesis of cancer. Here, we present the basic principles behind these epigenetic pathways and highlight the evidence suggesting that their misregulation can culminate in cancer. This information, along with the promising clinical and preclinical results seen with epigenetic drugs against chromatin regulators, signifies that it is time to embrace the central role of epigenetics in cancer.\", \n",
    "\"DNA methylation is among the best studied epigenetic modifications and is essential to mammalian development. Although the methylation status of most CpG dinucteotides in the genome is stably propagated through mitosis, improvements to methods for measuring methylation have identified numerous regions in which it is dynamically regulated. In this Review, we discuss key concepts in the function of DNA methylation in mammals, stemming from more than two decades of research, including many recent studies that have elucidated when and where DNA methylation has a regulatory role in the genome. We include insights from early development, embryonic stem cells and adult Lineages, particularly haematopoiesis, to highlight the general features of this modification as it participates in both global and localized epigenetic regulation..\", \n",
    "\"Methylation of the N-6 position of adenosine (m(6)A) is a posttranscriptional modification of RNA with poorly understood prevalence and physiological relevance. The recent discovery that FTO, an obesity risk gene, encodes an m(6)A demethylase implicates m(6)A as an important regulator of physiological processes. Here, we present a method for transcriptome-wide m(6)A localization, which combines m(6)A-specific methylated RNA immunoprecipitation with next-generation sequencing (MeRIP-Seq). We use this method to identify mRNAs of 7,676 mammalian genes that contain m(6)A, indicating that m(6)A is a common base modification of mRNA. The m(6)A modification exhibits tissue-specific regulation and is markedly increased throughout brain development. We find that m(6)A sites are enriched near stop codons and in 3' UTRs, and we uncover an association between m 6 A residues and microRNA-binding sites within 3' UTRs. These findings provide a resource for identifying transcripts that are substrates for adenosine methylation and reveal insights into the epigenetic regulation of the mammalian transcriptome.\", \n",
    "\"Chromatin regulators have become attractive targets for cancer therapy, but it is unclear why inhibition of these ubiquitous regulators should have gene-specific effects in tumor cells. Here, we investigate how inhibition of the widely expressed transcriptional coactivator BRD4 leads to selective inhibition of the MYC oncogene in multiple myeloma (MM). BRD4 and Mediator were found to co-occupy thousands of enhancers associated with active genes. They also co-occupied a small set of exceptionally large super-enhancers associated with genes that feature prominently in MM biology, including the MYC oncogene. Treatment of MM tumor cells with the BET-bromodomain inhibitor JQ1 led to preferential loss of BRD4 at super-enhancers and consequent transcription elongation defects that preferentially impacted genes with super-enhancers, including MYC. Super-enhancers were found at key oncogenic drivers in many other tumor cells. These observations have implications for the discovery of cancer therapeutics directed at components of super-enhancers in diverse tumor types.\", \n",
    "\"The ability to measure human aging from molecular profiles has practical implications in many fields, including disease prevention and treatment, forensics, and extension of life. Although chronological age has been linked to changes in DNA methylation, the methylome has not yet been used to measure and compare human aging rates. Here, we build a quantitative model of aging using measurements at more than 450,000 CpG markers from the whole blood of 656 human individuals, aged 19 to 101. This model measures the rate at which an individual's methylome ages, which we show is impacted by gender and genetic variants. We also show that differences in aging rates help explain epigenetic drift and are reflected in the transcriptome. Moreover, we show how our aging model is upheld in other human tissues and reveals an advanced aging rate in tumor tissue. Our model highlights specific components of the aging process and provides a quantitative readout for studying the role of methylation in age-related disease.\", \n",
    "\"Organisms require an appropriate balance of stability and reversibility in gene expression programmes to maintain cell identity or to enable responses to stimuli; epigenetic regulation is integral to this dynamic control. Post-translational modification of histones by methylation is an important and widespread type of chromatin modification that is known to influence biological processes in the context of development and cellular responses. To evaluate how histone methylation contributes to stable or reversible control, we provide a broad overview of how histone methylation is regulated and leads to biological outcomes. The importance of appropriately maintaining or reprogramming histone methylation is illustrated by its links to disease and ageing and possibly to transmission of traits across generations.\", \n",
    "\"Chromosomes are the physical realization of genetic information and thus form the basis for its readout and propagation. Here we present a high-resolution chromosomal contact map derived from a modified genome-wide chromosome conformation capture approach applied to Drosophila embryonic nuclei. The data show that the entire genome is linearly partitioned into well-demarcated physical domains that overlap extensively with active and repressive epigenetic marks. Chromosomal contacts are hierarchically organized between domains. Global modeling of contact density and clustering of domains show that inactive domains are condensed and confined to their chromosomal territories, whereas active domains reach out of the territory to form remote intra-and interchromosomal contacts. Moreover, we systematically identify specific long-range intrachromosomal contacts between Polycomb-repressed domains. Together, these observations allow for quantitative prediction of the Drosophila chromosomal contact map, laying the foundation for detailed studies of chromosome structure and function in a genetically tractable system.\", \n",
    "\"Hybridization has many and varied impacts on the process of speciation. Hybridization may slow or reverse differentiation by allowing gene flow and recombination. It may accelerate speciation via adaptive introgression or cause near-instantaneous speciation by allopolyploidization. It may have multiple effects at different stages and in different spatial contexts within a single speciation event. We offer a perspective on the context and evolutionary significance of hybridization during speciation, highlighting issues of current interest and debate. In secondary contact zones, it is uncertain if barriers to gene flow will be strengthened or broken down due to recombination and gene flow. Theory and empirical evidence suggest the latter is more likely, except within and around strongly selected genomic regions. Hybridization may contribute to speciation through the formation of new hybrid taxa, whereas introgression of a few loci may promote adaptive divergence and so facilitate speciation. Gene regulatory networks, epigenetic effects and the evolution of selfish genetic material in the genome suggest that the DobzhanskyMuller model of hybrid incompatibilities requires a broader interpretation. Finally, although the incidence of reinforcement remains uncertain, this and other interactions in areas of sympatry may have knock-on effects on speciation both within and outside regions of hybridization.\", \n",
    "\"Glioblastoma (GBM) is a brain tumor that carries a dismal prognosis and displays considerable heterogeneity. We have recently identified recurrent H3F3A mutations affecting two critical amino acids (K27 and G34) of histone H3.3 in one-third of pediatric GBM. Here, we show that each H3F3A mutation defines an epigenetic subgroup of GBM with a distinct global methylation pattern, and that they are mutually exclusive with IDH1 mutations, which characterize a third mutation-defined subgroup. Three further epigenetic subgroups were enriched for hallmark genetic events of adult GBM and/or established transcriptomic signatures. We also demonstrate that the two H3F3A mutations give rise to GBMs in separate anatomic compartments, with differential regulation of transcription factors OLIG1, OLIG2, and FOXG1, possibly reflecting different cellular origins.\", \n",
    "\"Epigenetic phenomena in animals and plants are mediated by DNA methylation and stable chromatin modifications. There has been considerable interest in whether environmental factors modulate the establishment and maintenance of epigenetic modifications, and could thereby influence gene expression and phenotype. Chemical pollutants, dietary components, temperature changes and other external stresses can indeed have long-lasting effects on development, metabolism and health, sometimes even in subsequent generations. Although the underlying mechanisms remain largely unknown, particularly in humans, mechanistic insights are emerging from experimental model systems. These have implications for structuring future research and understanding disease and development.\", \n",
    "\"Methylation of cytosine bases in DNA provides a layer of epigenetic control in many eukaryotes that has important implications for normal biology and disease. Therefore, profiling DNA methylation across the genome is vital to understanding the influence of epigenetics. There has been a revolution in DNA methylation analysis technology over the past decade: analyses that previously were restricted to specific loci can now be performed on a genome-scale and entire methylomes can be characterized at single-base-pair resolution. However, there is such a diversity of DNA methylation profiling techniques that it can be challenging to select one. This Review discusses the different approaches and their relative merits and introduces considerations for data analysis.\", \n",
    "\"We have developed a new generation of genome-wide DNA methylation Bead Chip which allows high-throughput methylation profiling of the human genome. The new high density BeadChip can assay over 480K CpG sites and analyze twelve samples in parallel. The innovative content includes coverage of 99% of RefSeq genes with multiple probes per gene, 96% of CpG islands from the UCSC database, CpG island shores and additional content selected from whole-genome bisulfite sequencing data and input from DNA methylation experts. The well-characterized Infinium (R) Assay is used for analysis of CpG methylation using bisulfite-converted genomic DNA. We applied this technology to analyze DNA methylation in normal and tumor DNA samples and compared results with whole-genome bisulfite sequencing (WGBS) data obtained for the same samples. Highly comparable DNA methylation profiles were generated by the array and sequencing methods (average R-2 of 0.95). The ability to determine genome-wide methylation patterns will rapidly advance methylation research. (C) 2011 Elsevier Inc. All rights reserved.\", \n",
    "\"Expression of the INK4b/ARF/INK4a tumor suppressor locus in normal and cancerous cell growth is controlled by methylation of histone H3 at lysine 27 (H3K27me) as directed by the Polycomb group proteins. The antisense noncoding RNA ANRIL of the INK4b/ARF/INK4a locus is also important for expression of the protein-coding genes in cis, but its mechanism has remained elusive. Here we report that chromobox 7 (CBX7) within the polycomb repressive complex 1 binds to ANRIL, and both CBX7 and ANRIL are found at elevated levels in prostate cancer tissues. In concert with H3K27me recognition, binding to RNA contributes to CBX7 function, and disruption of either interaction impacts the ability of CBX7 to repress the INK4b/ARF/INK4a locus and control senescence. Structure-guided analysis reveals the molecular interplay between noncoding RNA and H3K27me as mediated by the conserved chromodomain. Our study suggests a mechanism by which noncoding RNA participates directly in epigenetic transcriptional repression.\", \n",
    "\"Determining how somatic copy number alterations (SCNAs) promote cancer is an important goal. We characterized SCNA patterns in 4,934 cancers from The Cancer Genome Atlas Pan-Cancer data set. Whole-genome doubling, observed in 37% of cancers, was associated with higher rates of every other type of SCNA, TP53 mutations, CCNE1 amplifications and alterations of the PPP2R complex. SCNAs that were internal to chromosomes tended to be shorter than telomere-bounded SCNAs, suggesting different mechanisms underlying their generation. Significantly recurrent focal SCNAs were observed in 140 regions, including 102 without known oncogene or tumor suppressor gene targets and 50 with significantly mutated genes. Amplified regions without known oncogenes were enriched for genes involved in epigenetic regulation. When levels of genomic disruption were accounted for, 7% of region pairs were anticorrelated, and these regions tended to encompass genes whose proteins physically interact, suggesting related functions. These results provide insights into mechanisms of generation and functional consequences of cancer-related SCNAs.\", \n",
    "\"High-throughput technologies are widely used, for example to assay genetic variants, gene and protein expression, and epigenetic modifications. One often overlooked complication with such studies is batch effects, which occur because measurements are affected by laboratory conditions, reagent lots and personnel differences. This becomes a major problem when batch effects are correlated with an outcome of interest and lead to incorrect conclusions. Using both published studies and our own analyses, we argue that batch effects (as well as other technical and biological artefacts) are widespread and critical to address. We review experimental and computational approaches for doing so.\", \n",
    "\"Cytosine methylation is the major covalent modification of mammalian genomic DNA and plays important roles in transcriptional regulation. The molecular mechanism underlying the enzymatic removal of this epigenetic mark, however, remains elusive. Here, we show that 5-methylcytosine (5mC) hydroxylase TET1, by converting 5mCs to 5-hydroxymethylcytosines (5hmCs), promotes DNA demethylation in mammalian cells through a process that requires the base excision repair pathway. Though expression of the 12 known human DNA glycosylases individually did not enhance removal of 5hmCs in mammalian cells, demethylation of both exogenously introduced and endogenous 5hmCs is promoted by the AID (activation-induced deaminase)/APOBEC (apolipoprotein B mRNA-editing enzyme complex) family of cytidine deaminases. Furthermore, Tet1 and Apobec1 are involved in neuronal activity-induced, region-specific, active DNA demethylation and subsequent gene expression in the dentate gyrus of the adult mouse brain in vivo. Our study suggests a TET1-induced oxidation-deamination mechanism for active DNA demethylation in mammals.\", \n",
    "\"Nuclear factor-erythroid 2 p45-related factor 2 (Nrf2, also called Nfe212) is a transcription factor that regulates the cellular redox status. Nrf2 is controlled through a complex transcriptional/epigenetic and post-translational network that ensures its activity increases during redox perturbation, inflammation, growth factor stimulation and nutrient/energy fluxes, thereby enabling the factor to orchestrate adaptive responses to diverse forms of stress. Besides mediating stress-stimulated induction of antioxidant and detoxification genes, Nr12 contributes to adaptation by upregulating the repair and degradation of damaged macromolecules, and by modulating intermediary metabolism. In the latter case, Nrf2 inhibits lipogenesis, supports 0-oxidation of fatty acids, facilitates flux through the pentose phosphate pathway, and increases NADPH regeneration and purine biosynthesis; these observations suggest Nrf2 directs metabolic reprogramming during stress.\", \n",
    "\"Genomes of complex organisms encode an abundance and diversity of long noncoding RNAs (IncRNAs) that are expressed throughout the cell and fulfill a wide variety of regulatory roles at almost every stage of gene expression. These roles, which encompass sensory, guiding, scaffolding and allosteric capacities, derive from folded modular domains in IncRNAs. In this diverse functional repertoire, we focus on the well-characterized ability for IncRNAs to function as epigenetic modulators. Many IncRNAs bind to chromatin-modifying proteins and recruit their catalytic activity to specific sites in the genome, thereby modulating chromatin states and impacting gene expression. Considering this regulatory potential in combination with the abundance of IncRNAs suggests that IncRNAs may be part of a broad epigenetic regulatory network.\", \n",
    "\"Bromodomains (BRDs) are protein interaction modules that specifically recognize epsilon-N-lysine acetylation motifs, a key event in the reading process of epigenetic marks. The 61 BRDs in the human genome cluster into eight families based on structure/sequence similarity. Here, we present 29 high-resolution crystal structures, covering all BRD families. Comprehensive crossfamily structural analysis identifies conserved and family-specific structural features that are necessary for specific acetylation-dependent substrate recognition. Screening of more than 30 representative BRDs against systematic histone-peptide arrays identifies new BRD substrates and reveals a strong influence of flanking posttranslational modifications, such as acetylation and phosphorylation, suggesting that BRDs recognize combinations of marks rather than singly acetylated sequences. We further uncovered a structural mechanism for the simultaneous binding and recognition of diverse diacetyl-containing peptides by BRD4. These data provide a foundation for structure-based drug design of specific inhibitors for this emerging target family.\", \n",
    "\"Epigenetics is one of the most promising and expanding fields in the current biomedical research landscape. Since the inception of epigenetics in the 1940s, the discoveries regarding its implications in normal and disease biology have not stopped, compiling a vast amount of knowledge in the past decade. The field has moved from just one recognized marker, DNA methylation, to a variety of others, including a wide spectrum of histone modifications. From the methodological standpoint, the successful initial single gene candidate approaches have been complemented by the current comprehensive epigenomic approaches that allow the interrogation of genomes to search for translational applications in an unbiased manner. Most important, the discovery of mutations in the epigenetic machinery and the approval of the first epigenetic drugs for the treatment of subtypes of leukemias and lymphomas has been an eye-opener for many biomedical scientists and clinicians. Herein, we will summarize the progress in the field of cancer epigenetics research that has reached mainstream oncology in the development of new biomarkers of the disease and new pharmacological strategies.\", \n",
    "\"Abnormalities of chromosome 7q are common in myeloid malignancies, but no specific target genes have yet been identified. Here, we describe the finding of homozygous EZH2 mutations in 9 of 12 individuals with 7q acquired uniparental disomy. Screening of a total of 614 individuals with myeloid disorders revealed 49 monoallelic or biallelic EZH2 mutations in 42 individuals; the mutations were found most commonly in those with myelodysplastic/myeloproliferative neoplasms (27 out of 219 individuals, or 12%) and in those with myelofibrosis (4 out of 30 individuals, or 13%). EZH2 encodes the catalytic subunit of the polycomb repressive complex 2 (PRC2), a highly conserved histone H3 lysine 27 (H3K27) methyltransferase that influences stem cell renewal by epigenetic repression of genes involved in cell fate decisions. EZH2 has oncogenic activity, and its overexpression has previously been causally linked to differentiation blocks in epithelial tumors. Notably, the mutations we identified resulted in premature chain termination or direct abrogation of histone methyltransferase activity, suggesting that EZH2 acts as a tumor suppressor for myeloid malignancies.\", \n",
    "\"With rare exceptions, spontaneous tumors originate from a single cell. Yet, at the time of clinical diagnosis, the majority of human tumors display startling heterogeneity in many morphological and physiological features, such as expression of cell surface receptors, proliferative and angiogenic potential. To a substantial extent, this heterogeneity might be attributed to morphological and epigenetic plasticity, but there is also strong evidence for the co-existence of genetically divergent tumor cell clones within tumors. In this perspective, we summarize the sources of intra-tumor phenotypic heterogeneity with emphasis on genetic heterogeneity. We review experimental evidence for the existence of both intra-tumor clonal heterogeneity as well as frequent evolutionary divergence between primary tumors and metastatic outgrowths. Furthermore, we discuss potential biological and clinical implications of intra-tumor clonal heterogeneity. (C) 2009 Elsevier B.V. All rights reserved.\", \n",
    "\"Since the human genome was sequenced, the term ``epigenetics'' is increasingly being associated with the hope that we are more than just the sum of our genes. Might what we eat, the air we breathe, or even the emotions we feel influence not only our genes but those of descendants? The environment can certainly influence gene expression and can lead to disease, but transgenerational consequences are another matter. Although the inheritance of epigenetic characters can certainly occur-particularly in plants-how much is due to the environment and the extent to which it happens in humans remain unclear.\", \n",
    "\"The incorporation of histone H3 variants has been implicated in the epigenetic memory of cellular state. Using genome editing with zinc-finger nucleases to tag endogenous H3.3, we report genome-wide profiles of H3 variants in mammalian embryonic stem cells and neuronal precursor cells. Genome-wide patterns of H3.3 are dependent on amino acid sequence and change with cellular differentiation at developmentally regulated loci. The H3.3 chaperone Hira is required for H3.3 enrichment at active and repressed genes. Strikingly, Hira is not essential for localization of H3.3 at telomeres and many transcription factor binding sites. Immunoaffinity purification and mass spectrometry reveal that the proteins Atrx and Daxx associate with H3.3 in a Hira-independent manner. Atrx is required for Hira-independent localization of H3.3 at telomeres and for the repression of telomeric RNA. Our data demonstrate that multiple and distinct factors are responsible for H3.3 localization at specific genomic locations in mammalian cells.\", \n",
    "\"Cyclin-dependent kinases (Cdks) are serine/threonine kinases and their catalytic activities are modulated by interactions with cyclins and Cdk inhibitors (CKIs). Close cooperation between this trio is necessary for ensuring orderly progression through the cell cycle. In addition to their well-established function in cell cycle control, it is becoming increasingly apparent that mammalian Cdks, cyclins and CKIs play indispensable roles in processes such as transcription, epigenetic regulation, metabolism, stem cell self-renewal, neuronal functions and spermatogenesis. Even more remarkably, they can accomplish some of these tasks individually, without the need for Cdk/cyclin complex formation or kinase activity. In this Review, we discuss the latest revelations about Cdks, cyclins and CKIs with the goal of showcasing their functional diversity beyond cell cycle regulation and their impact on development and disease in mammals.\", \n",
    "\"During the course of malignant cancer progression, neoplastic cells undergo dynamic and reversible transitions between multiple phenotypic states, the extremes of which are defined by the expression of epithelial and mesenchymal phenotypes. This plasticity is enabled by underlying shifts in epigenetic regulation. A small cohort of pleiotropically acting transcription factors is widely recognized to effect these shifts by controlling the expression of a constituency of key target genes. These master regulators depend on complex epigenetic regulatory mechanisms, notably the induction of changes in the modifications of chromatin-associated histones, in order to achieve the widespread changes in gene expression observed during epithelial-mesenchymal transitions (EMTs). These associations indicate that an understanding of the functional interactions between such EMT-inducing transcription factors and the modulators of chromatin configuration will provide crucial insights into the fundamental mechanisms underlying cancer progression and may, in the longer term, generate new diagnostic and therapeutic modalities for treating high-grade malignancies.\", \n",
    "\"Despite the success of genome-wide association studies (GWASs) in identifying loci associated with common diseases, a substantial proportion of the causality remains unexplained. Recent advances in genomic technologies have placed us in a position to initiate large-scale studies of human disease-associated epigenetic variation, specifically variation in DNA methylation. Such epigenome-wide association studies (EWASs) present novel opportunities but also create new challenges that are not encountered in GWASs. We discuss EWAS design, cohort and sample selections, statistical significance and power, confounding factors and follow-up studies. We also discuss how integration of EWASs with GWASs can help to dissect complex GWAS haplotypes for functional analysis.\", \n",
    "\"Discoveries over the past decade portend a paradigm shift in molecular biology. Evidence suggests that RNA is not only functional as a messenger between DNA and protein but also involved in the regulation of genome organization and gene expression, which is increasingly elaborate in complex organisms. Regulatory RNA seems to operate at many levels; in particular, it plays an important part in the epigenetic processes that control differentiation and development. These discoveries suggest a central role for RNA in human evolution and ontogeny. Here, we review the emergence of the previously unsuspected world of regulatory RNA from a historical perspective.\", \n",
    "\"DNA methylation, an important type of epigenetic modification in humans, participates in crucial cellular processes, such as embryonic development, X-inactivation, genomic imprinting and chromosome stability. Several platforms have been developed to study genome-wide DNA methylation. Many investigators in the field have chosen the Illumina Infinium HumanMethylation microarray for its ability to reliably assess DNA methylation following sodium bisulfite conversion. Here, we analyzed methylation profiles of 489 adult males and 357 adult females generated by the Infinium HumanMethylation450 microarray. Among the autosomal CpG sites that displayed significant methylation differences between the two sexes, we observed a significant enrichment of cross-reactive probes co-hybridizing to the sex chromosomes with more than 94% sequence identity. This could lead investigators to mistakenly infer the existence of significant autosomal sex-associated methylation. Using sequence identity cutoffs derived from the sex methylation analysis, we concluded that 6% of the array probes can potentially generate spurious signals because of co-hybridization to alternate genomic sequences highly homologous to the intended targets. Additionally, we discovered probes targeting polymorphic CpGs that overlapped SNPs. The methylation levels detected by these probes are simply the reflection of underlying genetic polymorphisms but could be misinterpreted as true signals. The existence of probes that are cross-reactive or of target polymorphic CpGs in the Illumina HumanMethylation microarrays can confound data obtained from such microarrays. Therefore, investigators should exercise caution when significant biological associations are found using these array platforms. A list of all cross-reactive probes and polymorphic CpGs identified by us are annotated in this paper.\", \n",
    "\"Cancer therapy is challenged by the diversity of molecular implementations of oncogenic processes and by the resulting variation in therapeutic responses. Projects such as The Cancer Genome Atlas (TCGA) provide molecular tumor maps in unprecedented detail. The interpretation of these maps remains a major challenge. Here we distilled thousands of genetic and epigenetic features altered in cancers to similar to 500 selected functional events (SFEs). Using this simplified description, we derived a hierarchical classification of 3,299 TCGA tumors from 12 cancer types. The top classes are dominated by either mutations (M class) or copy number changes (C class). This distinction is clearest at the extremes of genomic instability, indicating the presence of different oncogenic processes. The full hierarchy shows functional event patterns characteristic of multiple cross-tissue groups of tumors, termed oncogenic signature classes. Targetable functional events in a tumor class are suggestive of class-specific combination therapy. These results may assist in the definition of clinical trials to match actionable oncogenic signatures with personalized therapies.\", \n",
    "\"DNA methylation is one of the best-characterized epigenetic modifications and has been implicated in numerous biological processes, including transposable element silencing, genomic imprinting and X chromosome inactivation. Compared with other epigenetic modifications, DNA methylation is thought to be relatively stable. Despite its role in long-term silencing, DNA methylation is more dynamic than originally thought as active DNA demethylation has been observed during specific stages of development. In the past decade, many enzymes have been proposed to carry out active DNA demethylation and growing evidence suggests that, depending on the context, this process may be achieved by multiple mechanisms. Insight into how DNA methylation is dynamically regulated will broaden our understanding of epigenetic regulation and have great implications in somatic cell reprogramming and regenerative medicine.\", \n",
    "\"DNA methylation is one of the most intensely studied epigenetic modifications in mammals. In normal cells, it assures the proper regulation of gene expression and stable gene silencing. DNA methylation is associated with histone modifications and the interplay of these epigenetic modifications is crucial to regulate the functioning of the genome by changing chromatin architecture. The covalent addition of a methyl group occurs generally in cytosine within CpG dinucleotides which are concentrated in large clusters called CpG islands. DNA methyltransferases are responsible for establishing and maintenance of methylation pattern. It is commonly known that inactivation of certain tumor-suppressor genes occurs as a consequence of hypermethylation within the promoter regions and a numerous studies have demonstrated a broad range of genes silenced by DNA methylation in different cancer types. On the other hand, global hypomethylation, inducing genomic instability, also contributes to cell transformation. Apart from DNA methylation alterations in promoter regions and repetitive DNA sequences, this phenomenon is associated also with regulation of expression of noncoding RNAs such as microRNAs that may play role in tumor suppression. DNA methylation seems to be promising in putative translational use in patients and hypermethylated promoters may serve as biomarkers. Moreover, unlike genetic alterations, DNA methylation is reversible what makes it extremely interesting for therapy approaches. The importance of DNA methylation alterations in tumorigenesis encourages us to decode the human epigenome. Different DNA methylome mapping techniques are indispensable to realize this project in the future. (C) 2010, Elsevier Inc.\", \n",
    "\"Epigenetic information can be inherited through the mammalian germline and represents a plausible transgenerational carrier of environmental information. To test whether transgenerational inheritance of environmental information occurs in mammals, we carried out an expression profiling screen for genes in mice that responded to paternal diet. Offspring of males fed a low-protein diet exhibited elevated hepatic expression of many genes involved in lipid and cholesterol biosynthesis and decreased levels of cholesterol esters, relative to the offspring of males fed a control diet. Epigenomic profiling of offspring livers revealed numerous modest (similar to 20%) changes in cytosine methylation depending on paternal diet, including reproducible changes in methylation over a likely enhancer for the key lipid regulator Ppara. These results, in conjunction with recent human epidemiological data, indicate that parental diet can affect cholesterol and lipid metabolism in offspring and define a model system to study environmental reprogramming of the heritable epigenome.\", \n",
    "\"Induction of cell death and inhibition of cell survival are the main principles of cancer therapy. Resistance to chemotherapeutic agents is a major problem in oncology, which limits the effectiveness of anticancer drugs. A variety of factors contribute to drug resistance, including host factors, specific genetic or epigenetic alterations in the cancer cells and so on. Although various mechanisms by which cancer cells become resistant to anticancer drugs in the microenvironment have been well elucidated, how to circumvent this resistance to improve anticancer efficacy remains to be defined. Autophagy, an important homeostatic cellular recycling mechanism, is now emerging as a crucial player in response to metabolic and therapeutic stresses, which attempts to maintain/restore metabolic homeostasis through the catabolic lysis of excessive or unnecessary proteins and injured or aged organelles. Recently, several studies have shown that autophagy constitutes a potential target for cancer therapy and the induction of autophagy in response to therapeutics can be viewed as having a prodeath or a prosurvival role, which contributes to the anticancer efficacy of these drugs as well as drug resistance. Thus, understanding the novel function of autophagy may allow us to develop a promising therapeutic strategy to enhance the effects of chemotherapy and improve clinical outcomes in the treatment of cancer patients.\", \n",
    "\"DNA methylation is a critical epigenetic regulator in mammalian development. Here, we present a whole-genome comparative view of DNA methylation using bisulfite sequencing of three cultured cell types representing progressive stages of differentiation: human embryonic stem cells (hESCs), a fibroblastic differentiated derivative of the hESCs, and neonatal fibroblasts. As a reference, we compared our maps with a methylome map of a fully differentiated adult cell type, mature peripheral blood mononuclear cells (monocytes). We observed many notable common and cell-type-specific features among all cell types. Promoter hypomethylation (both CG and CA) and higher levels of gene body methylation were positively correlated with transcription in all cell types. Exons were more highly methylated than introns, and sharp transitions of methylation occurred at exon-intron boundaries, suggesting a role for differential methylation in transcript splicing. Developmental stage was reflected in both the level of global methylation and extent of non-CpG methylation, with hESC highest, fibroblasts intermediate, and monocytes lowest. Differentiation-associated differential methylation profiles were observed for developmentally regulated genes, including the HOX clusters, other homeobox transcription factors, and pluripotence-associated genes such as POU5F1, TCF3, and KLF4. Our results highlight the value of high-resolution methylation maps, in conjunction with other systems-level analyses, for investigation of previously undetectable developmental regulatory mechanisms.\", \n",
    "\"RNA-directed DNA methylation (RdDM) is the major small RNA-mediated epigenetic pathway in plants. RdDM requires a specialized transcriptional machinery that comprises two plant-specific RNA polymerases - Pot IV and Pot V - and a growing number of accessory proteins, the functions of which in the RdDM mechanism are only partially understood. Recent work has revealed variations in the canonical RdDM pathway and identified factors that recruit Pot IV and Pol V to specific target sequences. RdDM, which transcriptionally represses a subset of transposons and genes, is implicated in pathogen defence, stress responses and reproduction, as well as in interallelic and intercellular communication.\", \n",
    "\"Tumor heterogeneity is a major barrier to effective cancer diagnosis and treatment. We recently identified cancer-specific differentially DNA-methylated regions (cDMRs) in colon cancer, which also distinguish normal tissue types from each other, suggesting that these cDMRs might be generalized across cancer types. Here we show stochastic methylation variation of the same cDMRs, distinguishing cancer from normal tissue, in colon, lung, breast, thyroid and Wilms' tumors, with intermediate variation in adenomas. Whole-genome bisulfite sequencing shows these variable cDMRs are related to loss of sharply delimited methylation boundaries at CpG islands. Furthermore, we find hypomethylation of discrete blocks encompassing half the genome, with extreme gene expression variability. Genes associated with the cDMRs and large blocks are involved in mitosis and matrix remodeling, respectively. We suggest a model for cancer involving loss of epigenetic stability of well-defined genomic domains that underlies increased methylation variability in cancer that may contribute to tumor heterogeneity.\", \n",
    "\"Emergence of form and function during embryogenesis arises in large part through cell-type- and cell-state-specific variation in gene expression patterns, mediated by specialized cis-regulatory elements called enhancers. Recent large-scale epigenomic mapping revealed unexpected complexity and dynamics of enhancer utilization patterns, with 400,000 putative human enhancers annotated by the ENCODE project alone. These large-scale efforts were largely enabled through the understanding that enhancers share certain stereotypical chromatin features. However, an important question still lingers: what is the functional significance of enhancer chromatin modification? Here we give an overview of enhancer-associated modifications of histones and DNA and discuss enzymatic activities involved in their dynamic deposition and removal. We describe potential downstream effectors of these marks and propose models for exploring functions of chromatin modification in regulating enhancer activity during development.\", \n",
    "\"The generation of properly functioning gametes in vitro requires reconstitution of the multistepped pathway of germ cell development. We demonstrate here the generation of primordial germ cell-like cells (PGCLCs) in mice with robust capacity for spermatogenesis. PGCLCs were generated from embryonic stem cells (ESCs) and induced pluripotent stem cells (iPSCs) through epiblast-like cells (EpiLCs), a cellular state highly similar to pregastrulating epiblasts but distinct from epiblast stem cells (EpiSCs). Reflecting epiblast development, EpiLC induction from ESCs/iPSCs is a progressive process, and EpiLCs highly competent for the PGC fate are a transient entity. The global transcription profiles, epigenetic reprogramming, and cellular dynamics during PGCLC induction from EpiLCs meticulously capture those associated with PGC specification from the epiblasts. Furthermore, we identify Integrin-beta 3 and SSEA1 as markers that allow the isolation of PGCLCs with spermatogenic capacity from tumorigenic undifferentiated cells. Our findings provide a paradigm for the first step of in vitro gametogenesis.\", \n",
    "\"Observational epidemiological studies are prone to confounding, reverse causation and various biases and have generated findings that have proved to be unreliable indicators of the causal effects of modifiable exposures on disease outcomes. Mendelian randomization (MR) is a method that utilizes genetic variants that are robustly associated with such modifiable exposures to generate more reliable evidence regarding which interventions should produce health benefits. The approach is being widely applied, and various ways to strengthen inference given the known potential limitations of MR are now available. Developments of MR, including two-sample MR, bidirectional MR, network MR, two-step MR, factorial MR and multiphenotype MR, are outlined in this review. The integration of genetic information into population-based epidemiological studies presents translational opportunities, which capitalize on the investment in genomic discovery research.\", \n",
    "\"Successful reprogramming of differentiated human somatic cells into a pluripotent state would allow creation of patient- and disease-specific stem cells. We previously reported generation of induced pluripotent stem (iPS) cells, capable of germline transmission, from mouse somatic cells by transduction of four defined transcription factors. Here, we demonstrate the generation of iPS cells from adult human dermal fibroblasts with the same four factors: Oct3/4, Sox2, Klf4, and c-Myc. Human iPS cells were similar to human embryonic stem (ES) cells in morphology, proliferation, surface antigens, gene expression, epigenetic status of pluripotent cell-specific genes, and telomerase activity. Furthermore, these cells could differentiate into cell types of the three germ layers in vitro and in teratomas. These findings demonstrate that iPS cells can be generated from adult human fibroblasts.\", \n",
    "\"The most highly conserved noncoding elements (HCNEs) in mammalian genomes cluster within regions enriched for genes encoding developmentally important transcription factors (TFs). This suggests that HCNE-rich regions may contain key regulatory controls involved in development. We explored this by examining histone methylation in mouse embryonic stem (ES) cells across 56 large HCNE-rich loci. We identified a specific modification pattern, termed bivalent domains, consisting of large regions of H3 lysine 27 methylation harboring smaller regions of H3 lysine 4 methylation. Bivalent domains tend to coincide with TF genes expressed at low levels. We propose that bivalent domains silence developmental genes in ES cells while keeping them poised for activation. We also found striking correspondences between genome sequence and histone methylation in ES cells, which become notably weaker in differentiated cells. These results highlight the importance of DNA sequence in defining the initial epigenetic landscape and suggest a novel chromatin-based mechanism for maintaining pluripotency.\", \n",
    "\"RNA is not only a messenger operating between DNA and protein. Transcription of essentially the entire eukaryotic genome generates a myriad of non-protein-coding RNA species that show complex overlapping patterns of expression and regulation. Although long noncoding RNAs (lncRNAs) are among the least well-understood of these transcript species, they cannot all be dismissed as merely transcriptional noise.'' Here, we review the evolution of lncRNAs and their roles in transcriptional regulation, epigenetic gene regulation, and disease.\", \n",
    "\"Noncoding RNAs (ncRNA) participate in epigenetic regulation but are poorly understood. Here we characterize the transcriptional landscape of the four human HOX loci at five base pair resolution in 11 anatomic sites and identify 231 HOX ncRNAs that extend known transcribed regions by more than 30 kilobases. HOX ncRNAs are spatially expressed along developmental axes and possess unique sequence motifs, and their expression demarcates broad chromosomal domains of differential histone methylation and RNA polymerase accessibility. We identified a 2.2 kilobase ncRNA residing in the HOXC locus, termed HOTAIR, which represses transcription in trans across 40 kilobases of the HOXD locus. HOTAIR interacts with Polycomb Repressive Complex 2 (PRC2) and is required for PRC2 occupancy and histone H3 lysine-27 trimethylation of HOXD locus. Thus, transcription of ncRNA may demarcate chromosomal domains of gene silencing at a distance; these results have broad implications for gene regulation in development and disease states.\", \n",
    "\"Over the past several years it has become clear that alterations in the expression of microRNA (miRNA) genes contribute to the pathogenesis of most - if not all - human malignancies. These alterations can be caused by various mechanisms, including deletions, amplifications or mutations involving miRNA loci, epigenetic silencing or the dysregulation of transcription factors that target specific miRNAs. Because malignant cells show dependence on the dysregulated expression of miRNA genes, which in turn control or are controlled by the dysregulation of multiple protein-coding oncogenes or tumour suppressor genes, these small RNAs provide important opportunities for the development of future miRNA-based therapies.\", \n",
    "\"The genomes of many animals, plants and fungi are tagged by methylation of DNA cytosine. To understand the biological significance of this epigenetic mark it is essential to know where in the genome it is located. New techniques are making it easier to map DNA methylation patterns on a large scale and the results have already provided surprises. In particular, the conventional view that DNA methylation functions predominantly to irreversibly silence transcription is being challenged. Not only is promoter methylation often highly dynamic during development, but many organisms also seem to target DNA methylation specifically to the bodies of active genes.\", \n",
    "\"DNA methylation is a crucial epigenetic modification of the genome that is involved in regulating many cellular processes. These include embryonic development, transcription, chromatin structure, X chromosome inactivation, genomic imprinting and chromosome stability. Consistent with these important roles, a growing number of human diseases have been found to be associated with aberrant DNA methylation. The study of these diseases has provided new and fundamental insights into the roles that DNA methylation and other epigenetic modifications have in development and normal cellular homeostasis.\", \n",
    "\"Methylation of DNA at position five of the cytosine ring occurs at most CpG dinucleotides in the mammalian genome and is essential for embryonic viability. With several of the key proteins now known, it has become possible to approach the biological significance of this epigenetic system through both biochemistry and genetics. As a result, advances have been made in our understanding of the mechanisms by which DNA methylation is targeted to specific regions of the genome and interpreted by methyl-CpG-binding proteins. Recent studies have illuminated the role of DNA methylation in controlling gene expression and have strengthened its links with histone modification and chromatin remodelling.\", \n",
    "\"Epidemiological evidence increasingly suggests that environmental exposures early in development have a role in susceptibility to disease in later life. In addition, some of these environmental effects seem to be passed on through subsequent generations. Epigenetic modifications provide a plausible link between the environment and alterations in gene expression that might lead to disease phenotypes. An increasing body of evidence from animal studies supports the role of environmental epigenetics in disease susceptibility. Furthermore, recent studies have demonstrated for the first time that heritable environmentally induced epigenetic modifications underlie reversible transgenerational alterations in phenotype. Methods are now becoming available to investigate the relevance of these phenomena to human disease.\", \n",
    "\"Deciphering the multiple layers of epigenetic regulation that control transcription is critical to understanding how plants develop and respond to their environment. Using sequencing-by-synthesis technology we directly sequenced the cytosine methylome (methylC-seq), transcriptome (mRNA-seq), and small RNA transcriptome (smRNA-seq) to generate highly integrated epigenome maps for wild-type Arabidopsis thaliana and mutants defective in DNA methyltransferase or demethylase activity. At singlebase resolution we discovered extensive, previously undetected DNA methylation, identified the context and level of methylation at each site, and observed local sequence effects upon methylation state. Deep sequencing of smRNAs revealed a direct relationship between the location of smRNAs and DNA methylation, perturbation of smRNA biogenesis upon loss of CpG DNA methylation, and a tendency for smRNAs to direct strand-specific DNA methylation in regions of RNA-DNA homology. Finally, strandspecific mRNA-seq revealed altered transcript abundance of hundreds of genes, transposons, and unannotated intergenic transcripts upon modification of the DNA methylation state.\", \n",
    "\"Plants and animals use small RNAs (microRNAs [miRNAs] and siRNAs) as guides for posttranscriptional and epigenetic regulation. In plants, miRNAs and trans-acting (ta) siRNAs form through distinct biogenesis pathways, although they both interact with target transcripts and guide cleavage. An integrated approach to identify targets of Arabidopsis thaliana miRNAs and ta-siRNAs revealed several new classes of small RNA-regulated genes, including conventional genes such as Argonaute2 and an E2-ubiquitin conjugating enzyme. Surprisingly, five tasiRNA-generating transcripts were identified as targets of miR173 or miR390. Rather than functioning as negative regulators, miR173- and miR390-guided cleavage was shown to set the 21-nucleotide phase for ta-siRNA precursor processing. These data support a model in which miRNA-guided formation of a 5' or 3' terminus within pre-ta-siRNA transcripts, followed by RDR6-dependent formation of dsRNA and Dicer-like processing, yields phased ta-siRNAs that negatively regulate other genes.\", \n",
    "\"For the past 25 years, it has been known that alterations in DNA methylation (DNAm) occur in cancer, including hypomethylation of oncogenes and hypermethylation of tumor suppressor genes. However, most studies of cancer methylation have assumed that functionally important DNAm will occur in promoters, and that most DNAm changes in cancer occur in CpG islands. Here we show that most methylation alterations in colon cancer occur not in promoters, and also not in CpG islands, but in sequences up to 2 kb distant, which we term 'CpG island shores'. CpG island shore methylation was strongly related to gene expression, and it was highly conserved in mouse, discriminating tissue types regardless of species of origin. There was a notable overlap (45-65%) of the locations of colon cancer-related methylation changes with those that distinguished normal tissues, with hypermethylation enriched closer to the associated CpG islands, and hypomethylation enriched further from the associated CpG island and resembling that of noncolon normal tissues. Thus, methylation changes in cancer are at sites that vary normally in tissue differentiation, consistent with the epigenetic progenitor model of cancer, which proposes that epigenetic alterations affecting tissue-specific differentiation are the predominant mechanism by which epigenetic changes cause cancer.\", \n",
    "\"An altered pattern of epigenetic modifications is central to many common human diseases, including cancer. Many studies have explored the mosaic patterns of DNA methylation and histone modification in cancer cells on a gene-by-gene basis; among their results has been the seminal finding of transcriptional silencing of tumour-suppressor genes by CpG-island-promoter hypermethylation. However, recent technological advances are now allowing cancer epigenetics to be studied genome-wide-an approach that has already begun to provide both biological insight and new avenues for translational research. It is time to 'upgrade' cancer epigenetics research and put together an ambitious plan to tackle the many unanswered questions in this field using epigenomics approaches.\", \n",
    "\"The term non-coding RNA (ncRNA) is commonly employed for RNA that does not encode a protein, but this does not mean that such RNAs do not contain information nor have function. Although it has been generally assumed that most genetic information is transacted by proteins, recent evidence suggests that the majority of the genomes of mammals and other complex organisms is in fact transcribed into ncRNAs, many of which are alternatively spliced and/or processed into smaller products. These ncRNAs include microRNAs and snoRNAs (many if not most of which remain to be identified), as well as likely other classes of yet-to-be-discovered small regulatory RNAs, and tens of thousands of longer transcripts (including complex patterns of interlacing and overlapping sense and antisense transcripts), most of whose functions are unknown. These RNAs (including those derived from introns) appear to comprise a hidden layer of internal signals that control various levels of gene expression in physiology and development, including chromatin architecture/epigenetic memory, transcription, RNA splicing, editing, translation and turnover. RNA regulatory networks may determine most of our complex characteristics, play a significant role in disease and constitute an unexplored world of genetic variation both within and between species.\", \n",
    "\" \", \n",
    "\"X-chromosome inactivation is the mammalian dosage compensation mechanism by which transcription of X-linked genes is equalized between females and males. In an N-ethyl-N-nitrosourea (ENU) mutagenesis screen on mice for modifiers of epigenetic reprogramming, we identified the MommeD1 (modifier of murine metastable epialleles) mutation as a semidominant suppressor of variegation. MommeD1 shows homozygous female-specific mid-gestation lethality and hypomethylation of the X-linked gene Hprt1, suggestive of a defect in X inactivation(1). Here we report that the causative point mutation lies in a previously uncharacterized gene, Smchd1 (structural maintenance of chromosomes hinge domain containing 1). We find that SmcHD1 is not required for correct Xist expression, but localizes to the inactive X and has a role in the maintenance of X inactivation and the hypermethylation of CpG islands associated with the inactive X. This finding links a group of proteins normally associated with structural aspects of chromosome biology with epigenetic gene silencing.\", \n",
    "\" \", \n",
    "\"The packaging of chromosomal DNA by nucleosomes condenses and organizes the genome, but occludes many regulatory DNA elements. However, this constraint also allows nucleosomes and other chromatin components to actively participate in the regulation of transcription, chromosome segregation, DNA replication, and DNA repair. To enable dynamic access to packaged DNA and to tailor nucleosome composition in chromosomal regions, cells have evolved a set of specialized chromatin remodeling complexes (remodelers). Remodelers use the energy of ATP hydrolysis to move, destabilize, eject, or restructure nucleosomes. Here, we address many aspects of remodeler biology: their targeting, mechanism, regulation, shared and unique properties, and specialization for particular biological processes. We also address roles for remodelers in development, cancer, and human syndromes.\", \n",
    "\"Ectopic expression of the four transcription factors Oct4, Sox2, c-Myc, and Klf4 is sufficient to confer a pluripotent state upon the fibroblast genome, generating induced pluripotent stem (PS) cells. It remains unknown if nuclear reprogramming induced by these four factors globally resets epigenetic differences between differentiated and pluripotent cells. Here, using novel selection approaches, we have generated PS cells from fibroblasts to characterize their epigenetic state. Female PS cells showed reactivation of a somatically silenced X chromosome and underwent random X inactivation upon differentiation. Genome-wide analysis of two key histone modifications indicated that PS cells are highly similar to ES cells. Consistent with these observations, PS cells gave rise to viable high-degree chimeras with contribution to the germline. These data show that transcription factor-induced reprogramming leads to the global reversion of the somatic epigenome into an ES-like state. Our results provide a paradigm for studying the epigenetic modifications that accompany nuclear reprogramming and suggest that abnormal epigenetic reprogramming does not pose a problem for the potential therapeutic applications of PS cells.\", \n",
    "\"Cancer is widely perceived as a heterogeneous group of disorders with markedly different biological properties, which are caused by a series of clonally selected genetic changes in key tumour-suppressor genes and oncogenes. However, recent data suggest that cancer has a fundamentally common basis that is grounded in a polyclonal epigenetic disruption of stem/progenitor cells, mediated by 'tumour-progenitor genes'. Furthermore, tumour cell heterogeneity is due in part to epigenetic variation in progenitor cells, and epigenetic plasticity together with genetic lesions drives tumour progression. This crucial early role for epigenetic alterations in cancer is in addition to epigenetic alterations that can substitute for genetic variation later in tumour progression. Therefore, non-neoplastic but epigenetically disrupted stem/progenitor cells might be a crucial target for cancer risk assessment and chemoprevention.\", \n",
    "\"Polyploids - organisms that have multiple sets of chromosomes - are common in certain plant and animal taxa, and can be surprisingly stable. The evidence that has emerged from genome analyses also indicates that many other eukaryotic genomes have a polyploid ancestry, suggesting that both humans and most other eukaryotes have either benefited from or endured polyploidy. Studies of polyploids soon after their formation have revealed genetic and epigenetic interactions between redundant genes. These interactions can be related to the phenotypes and evolutionary fates of polyploids. Here, I consider the advantages and challenges of polyploidy, and its evolutionary potential.\", \n",
    "\"CpG island hypermethylation and global genomic hypomethylation are common epigenetic features of cancer cells. Less attention has been focused on histone modifications in cancer cells. We characterized post-translational modifications to histone H4 in a comprehensive panel of normal tissues, cancer cell lines and primary tumors. Using immunodetection, high-performance capillary electrophoresis and mass spectrometry, we found that cancer cells had a loss of monoacetylated and trimethylated forms of histone H4. These changes appeared early and accumulated during the tumorigenic process, as we showed in a mouse model of multistage skin carcinogenesis. The losses occurred predominantly at the acetylated Lys16 and trimethylated Lys20 residues of histone H4 and were associated with the hypomethylation of DNA repetitive sequences, a well-known characteristic of cancer cells. Our data suggest that the global loss of monoacetylation and trimethylation of histone H4 is a common hallmark of human tumor cells.\", \n",
    "\"Cytosine methylation is required for mammalian development and is often perturbed in human cancer. To determine how this epigenetic modification is distributed in the genomes of primary and transformed cells, we used an immunocapturing approach followed by DNA microarray analysis to generate methylation profiles of all human chromosomes at 80-kb resolution and for a large set of CpG islands. In primary cells we identified broad genomic regions of differential methylation with higher levels in gene-rich neighborhoods. Female and male cells had indistinguishable profiles for autosomes but differences on the X chromosome. The inactive X chromosome ( Xi) was hypermethylated at only a subset of gene-rich regions and, unexpectedly, overall hypomethylated relative to its active counterpart. The chromosomal methylation profile of transformed cells was similar to that of primary cells. Nevertheless, we detected large genomic segments with hypomethylation in the transformed cell residing in gene-poor areas. Furthermore, analysis of 6,000 CpG islands showed that only a small set of promoters was methylated differentially, suggesting that aberrant methylation of CpG island promoters in malignancy might be less frequent than previously hypothesized.\", \n",
    "\"Inheritance requires genome duplication, reproduction of chromatin and its epigenetic information, mechanisms to ensure genome integrity, and faithful transmission of the information to progeny. Proliferating cell nuclear antigen (PCNA)-a cofactor of DNA polymerases that encircles DNA-orchestrates several of these functions by recruiting crucial players to the replication fork. Remarkably, many factors that are involved in replication-linked processes interact with a particular face of PCNA and through the same interaction domain, indicating that these interactions do not occur simultaneously during replication. Switching of PCNA partners may be triggered by affinity-driven competition, phosphorylation, proteolysis, and modification of PCNA by ubiquitin and SUMO.\", \n",
    "\"Cytosine methylation is important for transposon silencing and epigenetic regulation of endogenous genes, although the extent to which this DNA modification functions to regulate the genome is still unknown. Here we report the first comprehensive DNA methylation map of an entire genome, at 35 base pair resolution, using the flowering plant Arabidopsis thaliana as a model. We find that pericentromeric heterochromatin, repetitive sequences, and regions producing small interfering RNAs are heavily methylated. Unexpectedly, over one-third of expressed genes contain methylation within transcribed regions, whereas only similar to 5% of genes show methylation within promoter regions. Interestingly, genes methylated in transcribed regions are highly expressed and constitutively active, whereas promoter-methylated genes show a greater degree of tissue-specific expression. Whole-genome tiling-array transcriptional profiling of DNA methyltransferase null mutants identified hundreds of genes and intergenic noncoding RNAs with altered expression levels, many of which may be epigenetically controlled by DNA methylation.\", \n",
    "\"Overlapping epigenetic mechanisms have evolved in eukaryotic cells to silence the expression and mobility of transposable elements (TEs). Owing to their ability to recruit the silencing machinery, TEs have served as building blocks for epigenetic phenomena, both at the level of single genes and across larger chromosomal regions. Important progress has been made recently in understanding these silencing mechanisms. In addition, new insights have been gained into how this silencing has been co-opted to serve essential functions in 'host' cells, highlighting the importance of TEs in the epigenetic regulation of the genome.\", \n",
    "\"Cellular senescence, a state of irreversible growth arrest, can be triggered by multiple mechanisms including telomere shortening, the epigenetic derepression of the INK4a/ARF locus, and DNA damage. Together these mechanisms limit excessive or aberrant cellular proliferation, and so the state of senescence protects against the development of cancer. Recent evidence suggests that cellular senescence also may be involved in aging.\", \n",
    "\"Histones comprise the major protein component of chromatin, the scaffold in which the eukaryotic genome is packaged, and are subject to many types of post-translational modifications (PTMs), especially on their flexible tails. These modifications may constitute a 'histone code' and could be used to manage epigenetic information that helps extend the genetic message beyond DNA sequences. This proposed code, read in part by histone PTM-binding 'effector' modules and their associated complexes, is predicted to define unique functional states of chromatin and/or regulate various chromatin-templated processes. A wealth of structural and functional data show how chromatin effector modules target their cognate covalent histone modifications. Here we summarize key features in molecular recognition of histone PTMs by a diverse family of 'reader pockets', highlighting specific readout mechanisms for individual marks, common themes and insights into the downstream functional consequences of the interactions. Changes in these interactions may have far-reaching implications for human biology and disease, notably cancer.\", \n",
    "\"Chromatin immunoprecipitation followed by sequencing (ChIP-seq) is a technique for genome-wide profiling of DNA-binding proteins, histone modifications or nucleosomes. Owing to the tremendous progress in next-generation sequencing technology, ChIP-seq offers higher resolution, less noise and greater coverage than its array-based predecessor ChIP-chip. With the decreasing cost of sequencing, ChIP-seq has become an indispensable tool for studying gene regulation and epigenetic mechanisms. In this Review, I describe the benefits and challenges in harnessing this technique with an emphasis on issues related to experimental design and data analysis. ChIP-seq experiments generate large quantities of data, and effective computational analysis will be crucial for uncovering biological mechanisms.\", \n",
    "\"Inflammation is linked clinically and epidemiologically to cancer, and NF-kappa B appears to play a causative role, but the mechanisms are poorly understood. We show that transient activation of Src oncoprotein can mediate an epigenetic switch from immortalized breast cells to a stably transformed line that forms self-renewing mammospheres that contain cancer stem cells. Src activation triggers an inflammatory response mediated by NF-kappa B that directly activates Lin28 transcription and rapidly reduces let-7 microRNA levels. Let-7 directly inhibits IL6 expression, resulting in higher levels of IL6 than achieved by NF-kappa B activation. IL6-mediated activation of the STAT3 transcription factor is necessary for transformation, and IL6 activates NF-kappa B, thereby completing a positive feedback loop. This regulatory circuit operates in other cancer cells lines, and its transcriptional signature is found in human cancer tissues. Thus, inflammation activates a positive feedback loop that maintains the epigenetic transformed state for many generations in the absence of the inducing signal.\", \n",
    "\"Expression profiling of T24 cells revealed that 17 out of 313 human miRNAs were upregulated more than 3-fold by simultaneous treatment with the chromatin-modifying drugs 5-aza-2'-deoxycytidine and 4-phenylbutyric acid. One of these, miR-127, is embedded in a CpG island and is highly induced from its own promoter after treatment. miR-127 is usually expressed as part of a miRNA cluster in normal cells but not in cancer cells, suggesting that it is subject to epigenetic silencing. In addition, the proto-oncogene BCL6, a potential target of miR-127, was translationally downregulated after treatment. These results suggest that DNA demethylation and histone deacetylase inhibition can activate expression of miRNAs that may act as tumor suppressors.\", \n",
    "\"Polycomb proteins form chromatin-modifying complexes that implement transcriptional silencing in higher eukaryotes. Hundreds of genes are silenced by Polycomb proteins, including dozens of genes that encode crucial developmental regulators in organisms ranging from plants to humans. Two main families of complexes, called Polycomb repressive complex 1 (PRC1) and PRC2, are targeted to repressed regions. Recent studies have advanced our understanding of these complexes, including their potential mechanisms of gene silencing, the roles of chromatin modifications, their means of delivery to target genes and the functional distinctions among variant complexes. Emerging concepts include the existence of a Polycomb barrier to transcription elongation and the involvement of non-coding RNAs in the targeting of Polycomb complexes. These findings have an impact on the epigenetic programming of gene expression in many biological systems.\", \n",
    "\"A recent meeting (December 2008) regarding chromatin-based epigenetics was hosted by the Banbury Conference Center and Cold Spring Harbor Laboratory. The intent was to discuss aspects of epigenetic control of genomic function, and to arrive at a consensus definition of epigenetics'' to be considered by the broader community. It was evident that multiple mechanistic steps lead to the stable heritance of the epigenetic phenotype. Below we provide our view and interpretation of the proceedings at the meeting.\", \n",
    "\"Epigenetic genome modifications are thought to be important for specifying the lineage and developmental stage of cells within a multicellular organism. Here, we show that the epigenetic profile of pluripotent embryonic stem cells (ES) is distinct from that of embryonic carcinoma cells, haematopoietic stem cells (HSC) and their differentiated progeny. Silent, lineage-specific genes replicated earlier in pluripotent cells than in tissue-specific stem cells or differentiated cells and had unexpectedly high levels of acetylated H3K9 and methylated H3K4. Unusually, in ES cells these markers of open chromatin were also combined with H3K27 trimethylation at some non-expressed genes. Thus, pluripotency of ES cells is characterized by a specific epigenetic profile where lineage-specific genes may be accessible but, if so, carry repressive H3K27 trimethylation modifications. H3K27 methylation is functionally important for preventing expression of these genes in ES cells as premature expression occurs in embryonic ectoderm development (Eed)-deficient ES cells. Our data suggest that lineage-specific genes are primed for expression in ES cells but are held in check by opposing chromatin modifications.\", \n",
    "\"DNA methylation is the most stable type of epigenetic modification modulating the transcriptional plasticity of mammalian genomes. Using bisulfite DNA sequencing, we report high-resolution methylation profiles of human chromosomes 6, 20 and 22, providing a resource of about 1.9 million CpG methylation values derived from 12 different tissues. Analysis of six annotation categories showed that evolutionarily conserved regions are the predominant sites for differential DNA methylation and that a core region surrounding the transcriptional start site is an informative surrogate for promoter methylation. We find that 17% of the 873 analyzed genes are differentially methylated in their 5 ' UTRs and that about one-third of the differentially methylated 5 ' UTRs are inversely correlated with transcription. Despite the fact that our study controlled for factors reported to affect DNA methylation such as sex and age, we did not find any significant attributable effects. Our data suggest DNA methylation to be ontogenetically more stable than previously thought.\", \n",
    "\"The Polycomb group (PcG) proteins form chromatin-modifying complexes that are essential for embryonic development and stem cell renewal and are commonly deregulated in cancer. Here, we identify their target genes using genome-wide location analysis in human embryonic fibroblasts. We find that Polycomb-Repressive Complex I (PRC1), PRC2, and tri-methylated histone H3K27 co-occupy > 1000 silenced genes with a strong functional bias for embryonic development and cell fate decisions. We functionally identify 40 genes derepressed in human embryonic fibroblasts depleted of the PRC2 components (EZH2, EED, SUZ12) and the PRC1 component, BMI-1. Interestingly, several markers of osteogenesis, adipogenesis, and chrondrogenesis are among these genes, consistent with the mesenchymal origin of fibroblasts. Using a neuronal model of differentiation, we delineate two different mechanisms for regulating PcG target genes. For genes activated during differentiation, PcGs are displaced. However, for genes repressed during differentiation, we paradoxically find that they are already bound by the PcGs in nondifferentiated cells despite being actively transcribed. Our results are consistent with the hypothesis that PcGs are part of a preprogrammed memory system established during embryogenesis marking certain key genes for repressive signals during subsequent developmental and differentiation processes.\", \n",
    "\"Epigenetic marking systems confer stability of gene expression during mammalian development. Genome-wide epigenetic reprogramming occurs at stages when developmental potency of cells changes. At fertilization, the paternal genome exchanges protamines for histones, undergoes DNA demethylation, and acquires histone modifications, whereas the maternal genome appears epigenetically more static. During preimplantation development, there is passive DNA demethylation and further reorganization of histone modifications. In blastocysts, embryonic and extraembryonic lineages first show different epigenetic marks. This epigenetic reprogramming is likely to be needed for totipotency, correct initiation of embryonic gene expression, and early lineage development in the embryo. Comparative work demonstrates reprogramming in all mammalian species analysed, but the extent and timing varies, consistent with notable differences between species during preimplantation development. Parental imprinting marks originate in sperm and oocytes and are generally protected from this genome-wide reprogramming. Early primordial germ cells possess imprinting marks similar to those of somatic cells. However, rapid DNA demethylation after midgestation erases these parental imprints, in preparation for sex-specific de novo methylation during gametogenesis. Aberrant reprogramming of somatic epigenetic marks after somatic cell nuclear transfer leads to epigenetic defects in cloned embryos and stem cells. Links between epigenetic marking systems appear to be developmentally regulated contributing to plasticity. A number of activities that confer epigenetic marks are firmly established, while for those that remove marks, particularly methylation, some interesting candidates have emerged recently which need thorough testing in vivo. A mechanistic understanding of reprogramming will be crucial for medical applications of stem cell technology.\", \n",
    "\"Background: In animal models, variations in early maternal care are associated with differences in hypothalamic-pituitary-adrenal (HPA) stress response in the offspring, mediated via changes in the epigenetic regulation of glucocorticoid receptor (GR) gene (Nr3c1) expression. Objective: To study this in humans, relationships between prenatal exposure to maternal mood and the methylation status of a CpG-rich region in the promoter and exon 1F of the human GR gene (NR3C1) in newborns and HPA stress reactivity at age three months were examined. Results: Prenatal exposure to increased third trimester maternal depressed/anxious mood was associated with increased methylation of NR3C1 at a predicted NGFI-A binding site. Increased NR3C1 methylation at this site was also associated with increased salivary cortisol stress responses at 3 months, controlling for prenatal SRI exposure, postnatal age and pre and postnatal maternal mood. Methods: The methylation status of a CpG-rich region of the NR3C1 gene, including exon 1F, in genomic DNA from cord blood mononuclear cells was quantified by bisulfite pyrosequencing in infants of depressed mothers treated with a serotonin reuptake inhibitor antidepressant (SRI) (n = 33), infants of depressed non treated mothers (n = 13) and infants of non depressed/non treated mothers (n = 36). To study the functional implications of the newborn methylation status of NR3C1 in newborns, HPA function was assessed at three months using salivary cortisol obtained before and following a non noxious stressor and at a late afternoon basal time. Conclusions: Methylation status of the human NR3C1 gene in newborns is sensitive to prenatal maternal mood and may offer a potential epigenetic process that links antenatal maternal mood and altered HPA stress reactivity during infancy.\", \n",
    "\"The formation of heterochromatin, which requires methylation of histone H3 at lysine 9 and the subsequent recruitment of chromodomain proteins such as heterochromatin protein HP1, serves as a model for the role of histone modifications and chromatin assembly in epigenetic control of the genome. Recent studies in Schizosaccharomyces pombe indicate that heterochromatin serves as a dynamic platform to recruit and spread a myriad of regulatory proteins across extended domains to control various chromosomal processes, including transcription, chromosome segregation and long-range chromatin interactions.\", \n",
    "\"Compelling evidence suggests that the transcription factor Foxp3 acts as a master switch governing the development and function of CD4(+) regulatory T cells (Tregs). However, whether transcriptional control of Foxp3 expression itself contributes to the development of a stable Treg lineage has thus far not been investigated. We here identified an evolutionarily conserved region within the foxp3 locus upstream of exon-1 possessing transcriptional activity. Bisulphite sequencing and chromatin immunoprecipitation revealed complete demethylation of CpG motifs as well as histone modifications within the conserved region in ex vivo isolated Foxp3(+)CD25(+)CD4(+) Tregs, but not in naive CD25(-) CD4(+) T cells. Partial DNA demethylation is already found within developing Foxp3(+) thymocytes; however, Tregs induced by TGF-beta in vitro display only incomplete demethylation despite high Foxp3 expression. In contrast to natural Tregs, these TGF-beta-induced Foxp3(+) Tregs lose both Foxp3 expression and suppressive activity upon restimulation in the absence of TGF-beta. Our data suggest that expression of Foxp3 must be stabilized by epigenetic modification to allow the development of a permanent suppressor cell lineage, a finding of significant importance for therapeutic applications involving induction or transfer of Tregs and for the understanding of long-term cell lineage decisions.\", \n",
    "\"Many genes associated with CpG islands undergo de novo methylation in cancer. Studies have suggested that the pattern of this modification may be partially determined by an instructive mechanism that recognizes specifically marked regions of the genome(1). Using chromatin immunoprecipitation analysis, here we show that genes methylated in cancer cells are specifically packaged with nucleosomes containing histone H3 trimethylated on Lys27. This chromatin mark is established on these unmethylated CpG island genes early in development and then maintained in differentiated cell types by the presence of an EZH2- containing Polycomb complex(2,3). In cancer cells(4), as opposed to normal cells, the presence of this complex brings about the recruitment of DNA methyl transferases, leading to de novo methylation. These results suggest that tumor-specific targeting of de novo methylation is pre-programmed by an established epigenetic system that normally has a role in marking embryonic genes for repression(2).\", \n",
    "\"Ageing results from complex genetically and epigenetically programmed processes that are elicited in part by noxious or stressful events that cause programmed cell death. Here, we report that administration of spermidine, a natural polyamine whose intracellular concentration declines during human ageing, markedly extended the lifespan of yeast, flies and worms, and human immune cells. In addition, spermidine administration potently inhibited oxidative stress in ageing mice. In ageing yeast, spermidine treatment triggered epigenetic deacetylation of histone H3 through inhibition of histone acetyltransferases (HAT), suppressing oxidative stress and necrosis. Conversely, depletion of endogenous polyamines led to hyperacetylation, generation of reactive oxygen species, early necrotic death and decreased lifespan. The altered acetylation status of the chromatin led to significant upregulation of various autophagy-related transcripts, triggering autophagy in yeast, flies, worms and human cells. Finally, we found that enhanced autophagy is crucial for polyamine-induced suppression of necrosis and enhanced longevity.\", \n",
    "\"Induced pluripotent stem (iPS) cells are derived by epigenetic reprogramming, but their DNA methylation patterns have not yet been analyzed on a genome-wide scale. Here, we find substantial hypermethylation and hypomethylation of cytosine-phosphate-guanine (CpG) island shores in nine human iPS cell lines as compared to their parental fibroblasts. The differentially methylated regions (DMRs) in the reprogrammed cells (denoted R-DMRs) were significantly enriched in tissue-specific (T-DMRs; 2.6-fold, P < 10(-4)) and cancer-specific DMRs (C-DMRs; 3.6-fold, P < 10(-4)). Notably, even though the iPS cells are derived from fibroblasts, their R-DMRs can distinguish between normal brain, liver and spleen cells and between colon cancer and normal colon cells. Thus, many DMRs are broadly involved in tissue differentiation, epigenetic reprogramming and cancer. We observed colocalization of hypomethylated R-DMRs with hypermethylated C-DMRs and bivalent chromatin marks, and colocalization of hypermethylated R-DMRs with hypomethylated C-DMRs and the absence of bivalent marks, suggesting two mechanisms for epigenetic reprogramming in iPS cells and cancer.\", \n",
    "\"Histone methylation has important roles in regulating gene expression and forms part of the epigenetic memory system that regulates cell fate and identity. Enzymes that directly remove methyl marks from histones have recently been identified, revealing a new level of plasticity within this epigenetic modification system. Here we analyse the evolutionary relationship between Jumonji C ( JmjC)-domain-containing proteins and discuss their cellular functions in relation to their potential enzymatic activities.\", \n",
    "\"Protein lysine deacetylases have a pivotal role in numerous biological processes and can be divided into the Rpd3/Hda1 and sirtuin families, each having members in diverse organisms including prokaryotes. In vertebrates, the Rpd3/Hda1 family contains 11 members, traditionally referred to as histone deacetylases (HDAC)1-11, which are further grouped into classes I, II and IV. Whereas most class I HDACs are subunits of multiprotein nuclear complexes that are crucial for transcriptional repression and epigenetic landscaping, class II members regulate cytoplasmic processes or function as signal transducers that shuttle between the cytoplasm and the nucleus. Little is known about class IV HDAC11, although its evolutionary conservation implies a fundamental role in various organisms.\", \n",
    "\"Recent investigations have implicated long antisense noncoding RNAs in the epigenetic regulation of chromosomal domains. Here we show that Kcnq1ot1 is an RNA polymerase II-encoded, 91 kb-long, moderately stable nuclear transcript and that its stability is important for bidirectional silencing of genes in the Kcnq1 domain. Kcnq1ot1 interacts with chromatin and with the H3K9- and H3K27-specific histone methyltransferases G9a and the PRC2 complex in a lineage-specific manner. This interaction correlates with the presence of extended regions of chromatin enriched with H3K9me3 and H3K27me3 in the Kcnq1 domain in placenta, whereas fetal liver lacks both chromatin interactions and heterochromatin structures. In addition, the Kcnq1 domain is more often found in contact with the nucleolar compartment in placenta than in liver. Taken together, our data describe a mechanism whereby Kcnq1ot1 establishes lineage-specific transcriptional silencing patterns through recruitment of chromatin remodeling complexes and maintenance of these patterns through subsequent cell divisions occurs via targeting the associated regions to the perinucleolar compartment.\", \n",
    "\"It is more evident now than ever that nucleosomes can transmit epigenetic information from one cell generation to the next. It has been demonstrated during the past decade that the post-translational modifications of histone proteins within the chromosome impact chromatin structure, gene transcription, and epigenetic information. Multiple modifications decorate each histone tail within the nucleosome, including some amino acids that can be modified in several different ways. Covalent modifications of histone tails known thus far include acetylation, phosphorylation, surnoylation, ubiquitination, and methylation. A large body of experimental evidence compiled during the past several years has demonstrated the impact of histone acetylation on transcriptional control. Although histone modification by methylation and ubiquitination was discovered long ago, it was only recently that functional roles for these modifications in transcriptional regulation began to surface. Highlighted in this review are the recent biochemical, molecular, cellular, and physiological functions of histone methylation and ubiquitination involved in the regulation of gene expression as determined by a combination of enzymological, structural, and genetic methodologies.\", \n",
    "\"Cells employ elaborate mechanisms to introduce structural and chemical variation into chromatin. Here, we focus on one such element of variation: methylation of lysine 4 in histone H3 (H3K4). We assess a growing body of literature, including treatment of how the mark is established, the patterns of methylation, and the functional consequences of this epigenetic signature. We discuss structural aspects of the H3K4 methyl recognition by the downstream effectors and propose a distinction between sequence-specific recruitment mechanisms and stabilization on chromatin through methyllysine recognition. Finally, we hypothesize how the unique properties of the polyvalent chromatin fiber and associated effectors may amplify small differences in methyl-lysine recognition, simultaneously allowing for a dynamic chromatin architecture.\", \n",
    "\"Embryonic stem cells rely on Polycomb group proteins to reversibly repress genes required for differentiation. We report that stem cell Polycomb group targets are up to 12-fold more likely to have cancer- specific promoter DNA hypermethylation than non-targets, supporting a stem cell origin of cancer in which reversible gene repression is replaced by permanent silencing, locking the cell into a perpetual state of self-renewal and thereby predisposing to subsequent malignant transformation.\", \n",
    "\"Adult cancers may derive from stem or early progenitor cells(1,2). Epigenetic modulation of gene expression is essential for normal function of these early cells but is highly abnormal in cancers, which often show aberrant promoter CpG island hypermethylation and transcriptional silencing of tumor suppressor genes and pro-differentiation factors(3-5). We find that for such genes, both normal and malignant embryonic cells generally lack the hypermethylation of DNA found in adult cancers. In embryonic stem cells, these genes are held in a 'transcription- ready' state mediated by a 'bivalent' promoter chromatin pattern consisting of the repressive mark, histone H3 methylated at Lys27 ( H3K27) by Polycomb group proteins, plus the active mark, methylated H3K4. However, embryonic carcinoma cells add two key repressive marks, dimethylated H3K9 and trimethylated H3K9, both associated with DNA hypermethylation in adult cancers(6-8). We hypothesize that cell chromatin patterns and transient silencing of these important regulatory genes in stem or progenitor cells may leave these genes vulnerable to aberrant DNA hypermethylation and heritable gene silencing during tumor initiation and progression.\", \n",
    "\" \", \n",
    "\"Patterns of DNA methylation and chromatin structure are profoundly altered in neoplasia and include genome-wide losses of, and regional gains in, DNA methylation. The recent explosion in our knowledge of how chromatin organization modulates gene transcription has further highlighted the importance of epigenetic mechanisms in the initiation and progression of human cancer. These epigenetic changes - in particular, aberrant promoter hypermethylation that is associated with inappropriate gene silencing - affect virtually every step in tumour progression. In this review, we discuss these epigenetic events and the molecular alterations that might cause them and/or underlie altered gene expression in cancer.\", \n",
    "\"Cells of a multicellular organism are genetically homogeneous but structurally and functionally heterogeneous owing to the differential expression of genes. Many of these differences in gene expression arise during development and are subsequently retained through mitosis. Stable alterations of this kind are said to be 'epigenetic', because they are heritable in the short term but do not involve mutations of the DNA itself. Research over the past few years has focused on two molecular mechanisms that mediate epigenetic phenomena: DNA methylation and histone modifications. Here, we review advances in the understanding of the mechanism and role of DNA methylation in biological processes. Epigenetic effects by means of DNA methylation have an important role in development but can also arise stochastically as animals age. Identification of proteins that mediate these effects has provided insight into this complex process and diseases that occur when it is perturbed. External influences on epigenetic processes are seen in the effects of diet on long-term diseases such as cancer. Thus, epigenetic mechanisms seem to allow an organism to respond to the environment through changes in gene expression. The extent to which environmental effects can provoke epigenetic responses represents an exciting area of future research.\", \n",
    "\"Epithelial and hematopoietic cells have a high turnover and their progenitor cells divide continuously, making them prime targets for genetic and epigenetic changes that lead to cell transformation and tumorigenesis. The consequent changes in cell behavior and responsiveness result not only from genetic alterations such as activation of oncogenes or inactivation of tumor suppressor genes, but also from altered production of, or responsiveness to, stimulatory or inhibitory growth and differentiation factors. Among these, transforming growth factor beta (TGF-beta) and its signaling effectors act as key determinants of carcinoma cell behavior. The autocrine and paracrine effects of TGF-beta on tumor cells and the tumor micro-environment exert both positive and negative influences on cancer development. Accordingly, the TGF-beta signaling pathway has been considered as both a tumor suppressor pathway and a promoter of tumor progression and invasion. Here we evaluate the role of TGF-beta in tumor development and attempt to reconcile the positive and negative effects of TGF-beta in carcinogenesis.\", \n",
    "\"Genomic imprinting affects several dozen mammalian genes and results in the expression of those genes from only one of the two parental chromosomes. This is brought about by epigenetic instructions - imprints - that are laid down in the parental germ cells. Imprinting is a particularly important genetic mechanism in mammals, and is thought to influence the transfer of nutrients to the fetus and the newborn from the mother. Consistent with this view is the fact that imprinted genes tend to affect growth in the womb and behaviour after birth. Aberrant imprinting disturbs development and is the cause of various disease syndromes. The study of imprinting also provides new insights into epigenetic gene modification during development.\", \n",
    "\"The developmental programme of embryogenesis is controlled by both genetic and epigenetic mechanisms. An emerging theme from recent studies is that the regulation of higher-order chromatin structures by DNA methylation and histone modification is crucial for genome reprogramming during early embryogenesis and gametogenesis, and for tissue-specific gene expression and global gene silencing. Disruptions to chromatin modification can lead to the dysregulation of developmental processes, such as X-chromosome inactivation and genomic imprinting, and to various diseases. Understanding the process of epigenetic reprogramming in development is important for studies of cloning and the clinical application of stem-cell therapy.\", \n",
    "\"Early nutrition affects adult metabolism in humans and other mammals, potentially via persistent alterations in DNA methylation. With viable yellow agouti (A(vy)) mice, which harbor a transposable element in the agouti gene, we tested the hypothesis that the metastable methylation status of specific transposable element insertion sites renders them epigenetically labile to early methyl donor nutrition. Our results show that dietary methyl supplementation of ala dams with extra folic acid, vitamin B-12, choline, and betaine alter the phenotype of their A(vy)/a offspring via increased CpG methylation at the A(vy) locus and that the epigenetic metastability which confers this lability is due to the A(vy) transposable element. These findings suggest that dietary supplementation, long presumed to be purely beneficial, may have unintended deleterious influences on the establishment of epigenetic gene regulation in humans.\", \n",
    "\"Recently, the concept that epigenetic, as well as genetic, events might be central to the evolution of human cancer is re-emerging. Cancers often exhibit an aberrant methylation of gene promoter regions that is associated with loss of gene function. This DNA change constitutes a heritable state, not mediated by altered nucleotide sequence, that appears to be tightly linked to the formation of transcriptionally repressive chromatin. This epigenetic process acts as an alternative to mutations to disrupt tumor-suppressor gene function and can predispose to genetic alterations through inactivating DNA-repair genes. Dissecting the molecular processes that mediate these methylation changes will enhance our understanding of chromatin modeling and gene regulation and might present novel possibilities for cancer therapy. Methylation changes constitute potentially sensitive molecular markers to define risk states, monitor prevention strategies, achieve early diagnosis, and track the prognosis of cancer.\", \n",
    "\"Polyploidy is a prominent process in plants and has been significant in the evolutionary history of vertebrates and other eukaryotes. In plants, interdisciplinary approaches combining phylogenetic and molecular genetic perspectives have enhanced our awareness of the myriad genetic interactions made possible by polyploidy. Here, processes and mechanisms of gene and genome evolution in polyploids are reviewed. Genes duplicated by polyploidy may retain their original or similar function, undergo diversification in protein function or regulation, or one copy may become silenced through mutational or epigenetic means. Duplicated genes also may interact through inter-locus recombination, gene conversion, or concerted evolution. Recent experiments have illuminated important processes in polyploids that operate above the organizational level of duplicated genes. These include inter-genomic chromosomal exchanges, saltational, non-Mendelian genomic evolution in nascent polyploids, inter-genomic invasion, and cytonuclear stabilization. Notwithstanding many recent insights, much remains to be learned about many aspects of polyploid evolution, including: the role of transposable elements in structural and regulatory gene evolution; processes and significance of epigenetic silencing; underlying controls of chromosome pairing; mechanisms and functional significance of rapid genome changes; cytonuclear accommodation; and coordination of regulatory factors contributed by two, sometimes divergent progenitor genomes. Continued application of molecular genetic approaches to questions of polyploid genome evolution holds promise for producing lasting insight into processes by which novel genotypes are generated and ultimately into how polyploidy facilitates evolution and adaptation.\", \n",
    "\"Histone H3 lysine 9 methylation has been proposed to provide a major switch for the functional organization of chromosomal subdomains. Here, we show that the murine Suv39h histone methyltransferases (HMTases) govern H3-K9 methylation at pericentric heterochromatin and induce a specialized histone methylation pattern that differs from the broad H3-K9 methylation present at other chromosomal regions. Suv39h-deficient mice display severely impaired viability and chromosomal instabilities that are associated with an increased tumor risk and perturbed chromosome interactions during male meiosis. These in vivo data assign a crucial role for pericentric H3-K9 methylation in protecting genome stability, and define the: Suv39h HMTases as important epigenetic regulators for mammalian development.\", \n",
    "\"Transcriptional activation of a gene involves an orchestrated recruitment of components of the basal transcription machinery and intermediate factors, concomitant with an alteration in local chromatin structure generated by posttranslational modifications of histone tails and nucleosome remodeling. We provide here a comprehensive picture of events resulting in transcriptional activation of a gene, through evaluating the estrogen receptor-alpha (NR3A1) target pS2 gene promoter in MCF-7 cells. This description integrates chromatin remodeling with a kinetic evaluation of cyclical networks of association of 46 transcription factors with the promoter, as determined by chromatin immunoprecipitation assays. We define the concept of a transcriptional clock that directs and achieves the sequential and combinatorial assembly of a transcriptionally productive complex on a promoter. Furthermore, the unanticipated findings of key roles for histone deacetylases and nucleosome-remodeling complexes in limiting transcription implies that transcriptional activation is a cyclical process that requires both activating and repressive epigenetic processes.\", \n",
    "\"Hepatocarcinogenesis is a slow process during which genomic changes progressively alter the hepatocellular phenotype to produce cellular intermediates that evolve into hepatocellular carcinoma. During the long pre-neoplastic stage, in which the liver is often the site of chronic hepatitis, cirrhosis, or both, hepatocyte cycling is accelerated by upregulation of mitogenic pathways, in part through epigenetic mechanisms. This leads to the production of monoclonal populations of aberrant and dysplastic hepatocytes that have telomere erosion and telomerase re-expression, sometimes microsatellite instability, and occasionally structural aberrations in genes and chromosomes. Development of dysplastic hepatocytes in foci and nodules and emergence of hepatocellular carcinoma are associated with the accumulation of irreversible structural alterations in genes and chromosomes, but the genomic basis of the malignant phenotype is heterogeneous. The malignant hepatocyte phenotype may be produced by the disruption of a number of genes that function in different regulatory pathways, producing several molecular variants of hepatocellular carcinoma. New strategies should enable these variants to be characterized.\", \n",
    "\"Enhancer of Zeste is a Polycomb Group protein essential for the establishment and maintenance of repression of homeotic and other genes. In the early embryo it is found in a complex that includes ESC and is recruited to Polycomb Response Elements. We show that this complex contains a methyltransferase activity that methylates lysine 9 and lysine 27 of histone H3, but the activity is lost when the E(Z) SET domain is mutated. The lysine 9 position is trimethylated and this mark is closely associated with Polycomb binding sites on polytene chromosomes but is also found in centric heterochromatin, chromosome 4, and telomeric sites. Histone H3 methylated in vitro by the E(Z)/ ESC complex binds specifically to Polycomb protein.\", \n",
    "\"Cytosine-5 DNA methylation occurs in the context of CpG dinucleotides in vertebrates. Aberrant methylation of CpG islands in human tumors has been shown to cause transcriptional silencing of tumor-suppressor genes. Most methods used to analyze cytosine-5 methylation patterns require cumbersome manual techniques that employ gel electrophoresis, restriction enzyme digestion, radiolabeled dNTPs or hybridization probes. The development of high-throughput technology for the analysis of DNA methylation would significantly expand our ability to derive molecular information from clinical specimens. This study describes a high-throughput quantitative methylation assay that utilizes fluorescence-based real-time PCR (TaqMan (R)) technology that requires no further manipulations after the PCR step. Methy-Light is a highly sensitive assay, capable of detecting methylated alleles in the presence of a 10 000-fold excess of unmethylated alleles. The assay is also highly quantitative and can very accurately determine the relative prevalence of a particular pattern of DNA methylation. We show that MethyLight can distinguish between mono-allelic and bi-allelic methylation of the MLH1 mismatch repair gene in human colorectal tumor specimens. The development of this technique should considerably enhance our ability to rapidly and accurately generate epigenetic profiles of tumor samples.\", \n",
    "\"Cancer-associated DNA hypomethylation is as prevalent as cancer-linked hypermethylation, but these two types of epigenetic abnormalities usually seem to affect different DNA sequences. Much more of the genome is generally subject to undermethylation rather than overmethylation. Genomic hypermethylation in cancer has been observed most often in CpG islands in gene regions. In contrast, very frequent hypomethylation is seen in both highly and moderately repeated DNA sequences in cancer, including heterochromatic DNA repeats, dispersed retrotransposons, and endogenous retroviral elements. Also, unique sequences, including transcription control sequences, are often subject to cancer-associated undermethylation. The high frequency of cancer-linked DNA hypomethylation, the nature of the affected sequences, and the absence of associations with DNA hypermethylation are consistent with an independent role for DNA undermethylation in cancer formation or tumor progression. Increased karyotypic instability and activation of tumor-promoting genes by cis or trans effects, that might include altered heterochromatin-euchromatin interactions, may be important consequences of DNA hypomethylation which favor oncogenesis. The relationship of DNA hypomethylation to tumorigenesis is important to be considered in the light of cancer therapies involving decreasing DNA methylation. Inducing DNA hypomethylation may have short-term anticancer effects, but might also help speed tumor progression from cancer cells surviving the DNA demethylation chemotherapy.\", \n",
    "\"Sirtuins (Sir2-like proteins) are present in prokaryotes and eukaryotes, Here, two new human sirtuins (SIRT6 and SIRT7) are found to be similar to a particular subset of insect, nematode, plant, and protozoan sirtuins. Molecular phylogenetic analysis of 60 sirtuin conserved core domain sequences from a diverse array of organisms (including archaeans, bacteria, yeasts, plants, protozoans, and metazoans) shows that eukaryotic Sir2-like proteins group into four main branches designated here as classes I-IV. Prokaryotic sirtuins include members of classes II and III. A fifth class of sirtuin is present in gram positive bacteria and Thermotoga maritima. Saccharomyces cerevisiae has five class I sirtuins. Caenorhabditis elegans and Drosophila melanogaster have sirtuin genes from classes I, II, and TV. The seven human sirtuin genes include all four classes: SIRT1, SIRT2, and SIRT3 are class I, SIRT4 is class II, SIRT5 is class III, and SIRT6 and SIRT7 are class IV. (C) 2000 Academic Press.\", \n",
    "\"Chromatin is the physiologically relevant substrate for all genetic processes inside the nuclei of eukaryotic cells. Dynamic changes in the local and global organization of chromatin are emerging as key regulators of genomic function. Indeed, a multitude of signals from outside and inside the cell converges on this gigantic signaling platform. Numerous post-translational modifications of histones, the main protein components of chromatin, have been documented and analyzed in detail. These 'marks' appear to crucially mediate the functional activity of the genome in response to upstream signaling pathways. Different layers of cross-talk between several components of this complex regulatory system are emerging, and these epigenetic circuits are the focus of this review.\", \n",
    "\"Allelic loss at the short arm of chromosome 3 is one of the most common and earliest events in the pathogenesis of lung cancer, and is observed in more than 90% of small-cell lung cancers (SCLCs) and in 50-80% of non-small-cell lung cancers(1,2) (NSCLCs). Frequent and early loss of heterozygosity and the presence of homozygous deletions suggested a critical role of the region 3p21.3 in tumorigenesis(2-4) and a region of common homozygous deletion in 3p21.3 was narrowed to 120 kb (ref. 5). Several putative tumour-suppressor genes located at 3p21 have been characterized, but none of these genes appear to be altered in lung cancer. Here we describe the cloning and characterization of a human RAS effector homologue (RASSF1) located in the 120-kb region of minimal homozygous deletion. We identified three transcripts, A, B and C, derived from alternative splicing and promoter usage. The major transcripts A and C were expressed in all normal tissues. Transcript A was missing in all SCLC cell lines analysed and in several other cancer cell lines. loss of expression was correlated with methylation of the CpG-island promoter sequence of RASSF1A. The promoter was highly methylated in 24 of 60 (40%) primary lung tumours, and 4 of 41 tumours analysed carried missense mutations. Re-expression of transcript A in lung carcinoma cells reduced colony formation, suppressed anchorage-independent growth and inhibited tumour formation in nude mice. These characteristics indicate a potential role for RASSF1A asa lung tumour suppresor gene.\", \n",
    "\"Deposition of the major histone H3 (H3.1) is coupled to DNA synthesis during DNA replication and possibly DNA repair, whereas histone variant H3.3 serves as the replacement variant for the DNA-synthesis-independent deposition pathway. To address how histories H3.1 and H3.3 are deposited into chromatin through distinct pathways, we have purified deposition machineries for these histories. The H3.1 and H3.3 complexes contain distinct histone chaperones, CAF-1 and HIRA, that we show are necessary to mediate DNA-synthesis-dependent and -independent nucleosome assembly, respectively. Notably, these complexes possess one molecule each of H3.1/H3.3 and H4, suggesting that histories H3 and H4 exist as dimeric units that are important intermediates in nucleosome formation. This finding provides new insights into possible mechanisms for maintenance of epigenetic information after chromatin duplication.\", \n",
    "\"Dynamic epigenetic modification of the genome occurs during early development of the mouse. Active demethylation of the paternal genome occurs in the zygote, followed by passive demethylation during cleavage stages, and de novo methylation, which is thought to happen after implantation. We have investigated these processes by using indirect immunofluoresence with an antibody to 5-methyl cytosine. In contrast to previous work, we show that demethylation of the male pronucleus is completed within 4 h of fertilisation. This activity is intricately linked with and not separable from pronucleus formation. In conditions permissive for polyspermy, up to five male pronuclei underwent demethylation in the same oocyte. Paternal demethylation in fertilised oocytes deficient for MBD2, the only candidate demethylase, occurred normally. Passive loss of methylation occurred in a stepwise fashion up to the morulae stage without any evidence of spatial compartmentalisation. De novo methylation was observed specifically in the inner cell mass (ICM) but not in the trophectoderm of the blastocyst and hence may have an important role in early lineage specification. This is the first complete and detailed analysis of the epigenetic reprogramming cycle during preimplantation development. The three phases of methylation reprogramming may have roles in imprinting, the control of gene expression, and the establishment of nuclear totipotency. (C) 2001 Elsevier Science.\", \n",
    "\"5-Azacytidine was first synthesized almost 40 years ago. It was demonstrated to have a wide range of anti-metabolic activities when tested against cultured cancer cells and to be an effective chemotherapeutic agent for acute myelogenous leukemia. However, because of 5-azacytidine's general toxicity, other nucleoside analogs were favored as therapeutics. The finding that 5-azacytidine was incorporated into DNA and that, when present in DNA, it inhibited DNA methylation, led to widespread use of 5-azacytidine and 5-aza-2'-deoxycytidine (Decitabine) to demonstrate the correlation between loss of methylation in specific gene regions and activation of the associated genes. There is now a revived interest in the use of Decitabine as a therapeutic agent for cancers in which epigenetic silencing of critical regulatory genes has occurred. Here, the current status of our understanding of the mechanism(s) by which 5-azacytosine residues in DNA inhibit DNA methylation is reviewed with an emphasis on the interactions of these residues with bacterial and mammalian DNA (cytosine-C5) methyltransferases. The implications of these mechanistic studies for development of less toxic inhibitors of DNA methylation are discussed.\", \n",
    "\"The enzyme-catalyzed acetylation of the N-terminal tail domains of core histones provides a rich potential source of epigenetic information. This may be used both to mediate transient changes in transcription, through modification of promoter-proximal nucleosomes, and for the longer-term maintenance and modulation of patterns of gene expression. The latter may be achieved by setting specific patterns of histone acetylation, perhaps involving acetylation of particular lysine residues, across relatively large chromatin domains. The histone acetylating and deacetylating enzymes (HATs and HDACs, respectively) can be targeted to specific regions of the genome and show varying degrees of substrate specificity, properties that are consistent with a role in maintaining a dynamic, acetylation-based epigenetic code. The code may be read (ie. exert a functional effect) either through non-histone proteins that bind in an acetylation-dependent manner, or through direct effects on chromatin structure. Recent evidence raises the interesting possibility that an acetylation-based code may operate through both mitosis and meiosis, providing a possible mechanism for germ-line transmission of epigenetic changes. BioEssays 22:836-845, 2000. (C) 2000 John Wiley & Sons, Inc.\", \n",
    "\"We have come a long way since the first reports of the existence of aberrant DNA methylation in human cancer. Hypermethylation of CpG islands located in the promoter regions of tumor suppressor genes is now firmly established as an important mechanism for gene inactivation. CpG island hypermethylation has been described in almost every tumor type. Many cellular pathways are inactivated by this type of epigenetic lesion: DNA repair (hMLH1, MGMT), cell cycle (p16(INK4a), p15(INK4b), p14(ARF)), apoptosis (DAPK), cell adherence (CDH1, CDH13), detoxification (GSTP1), etc... However, we still know little of the mechanisms of aberrant methylation and why certain genes are selected over others. Hypermethylation is not an isolated layer of epigenetic control, but is linked to the other pieces of the puzzle such as methyl-binding proteins, DNA methyltransferases and histone deacetylase, but our understanding of the degree of specificity of these epigenetic layers in the silencing of specific tumor suppressor genes remains incomplete. The explosion of user-friendly technologies has given rise to a rapidly increasing list of hypermethylated genes. Careful functional and genetic studies are necessary to determine which hypermethylation events are truly relevant for human tumorigenesis. The development of CpG island hypermethylation profiles for every form of human tumors has yielded valuable pilot clinical data in monitoring and treating cancer patients based in our knowledge of DNA methylation. Basic and translational will both be needed in the near future to fully understand the mechanisms, roles and uses of CpG island hypermethylation in human cancer. The expectations are high.\", \n",
    "\"Genome-wide epigenetic reprogramming in mammalian germ cells, zygote and early embryos, plays a crucial role in regulating genome functions at critical stages of development. We show here that mouse primordial germ cells (PGCs) exhibit dynamic changes in epigenetic modifications between days 10.5 and 12.5 post coitum (dpc). First, contrary to previous suggestions, we show that PGCs do indeed acquire genome-wide de novo methylation during early development and migration into the genital ridge. However, following their entry into the genital ridge, there is rapid erasure of DNA methylation of regions within imprinted and non-imprinted loci. For most genes, the erasure commences simultaneously in PGCs in both male and female embryos, which is completed within I day of development. Based on the kinetics of this process, we suggest that this is an active demethylation process initiated upon the entry of PGCs into the gonadal anlagen. The timing of reprogramming in PGCs is crucial since it ensures that germ cells of both sexes acquire an equivalent epigenetic state prior to the differentiation of the definitive male and female germ cells in which new parental imprints are established subsequently. Some repetitive elements, however, show incomplete erasure, which may be essential for chromosome stability and for preventing activation of transposons to reduce the risk of germline mutations. Aberrant epigenetic reprogramming in the germ line would cause the inheritance of epimutations that may have consequences for human diseases as suggested by studies on mouse models. (C) 2002 Elsevier Science Ireland Ltd. All rights reserved.\", \n",
    "\"Aberrant WNT pathway signaling is an early progression event in 90% of colorectal cancers(1). It occurs through mutations mainly of APC and less often of CTNNB1 (encoding beta-catenin)(1-3) or AXIN2 (encoding axin-2, also known as conductin)(4). These mutations allow ligand-independent WNT signaling that culminates in abnormal accumulation of free beta-catenin in the nucleus(1-3). We previously identified frequent promoter hypermethylation and gene silencing of the genes encoding secreted frizzled-related proteins (SFRPs) in colorectal cancer(5). SFRPs possess a domain similar to one in the WNT-receptor frizzled proteins and can inhibit WNT receptor binding to downregulate pathway signaling during development(6-10). Here we show that restoration of SFRP function in colorectal cancer cells attenuates WNT signaling even in the presence of downstream mutations. We also show that the epigenetic loss of SFRP function occurs early in colorectal cancer progression and may thus provide constitutive WNT signaling that is required to complement downstream mutations in the evolution of colorectal cancer.\", \n",
    "\"Methylation of position-specific lysine residues in histone N termini is a central modification for regulating epigenetic transitions in chromatin. Each methylatable lysine residue can exist in a mono-, di-, or trimethylated state, thereby extending the indexing potential of this particular modification. Here, we examine all possible methylation states for histone H3 lysine 9 (H3-K9) and lysine 27 (H3-K27) in mammalian chromatin. Using highly specific antibodies together with quantitative mass spectrometry, we demonstrate that pericentric heterochromatin is selectively enriched for H3-K27 monomethylation and H3-K9 trimethylation. This heterochromatic methylation profile is dependent on the Suv39h histone methyltransferases (HMTases) but independent of the euchromatic G9a HMTase. In Suv39h double null cells, pericentric heterochromatin is converted to alternative methylation imprints and accumulates H3-K27 trimethylation and H3-K9 monomethylation. Our data underscore the selective presence of distinct histone lysine methylation states in partitioning chromosomal subdomains but also reveal a surprising plasticity in propagating methylation patterns in eukaryotic chromatin.\", \n",
    "\"During the development of multicellular organisms, cells become different from one another by changing their genetic program in response to transient stimuli. Long after the stimulus is gone, cellular memory mechanisms enable cells to remember their chosen fate over many cell divisions. The Polycomb and Trithorax groups of proteins, respectively, work to maintain repressed or active transcription states of developmentally important genes through many rounds of cell division. Here we review current ideas on the protein and DNA components of this transcriptional memory system and how they interact dynamically with each other to orchestrate cellular memory for several hundred genes.\", \n",
    "\"DNA methylation has recently moved to centre stage in the aetiology of human neurodevelopmental syndromes such as the fragile X, ICF and Rett syndromes. These diseases result from the misregulation of genes that occurs with the toss of appropriate epigenetic controls during neuronal development. Recent advances have connected DNA methylation to chromatin-remodeling enzymes, and understanding this link will be central to the design of new therapeutic tools.\", \n",
    "\"The centromere is a chromosomal locus that ensures delivery of one copy of each chromosome to each daughter at cell division. Efforts to understand the nature and specification of the centromere have demonstrated that this central element for ensuring inheritance is itself epigenetically determined. The kinetochore, the protein complex assembled at each centromere, serves as the attachment site for spindle microtubules and the site at which motors generate forces to power chromosome movement. Unattached kinetochores are also the signal generators for the mitotic checkpoint, which arrests mitosis until all kinetochores have correctly attached to spindle microtubules, thereby representing the major cell cycle control mechanism protecting against loss of a chromosome (aneuploidy).\", \n",
    "\"On the histone H3 tail, Lys 9 and Lys 27 are both methylation sites associated with epigenetic repression, and reside within a highly related sequence motif ARKS. Here we show that the chromodomain proteins Polycomb (Pc) and HP1 (heterochromatin protein 1) are highly discriminatory for binding to these sites in vivo and in vitro. In Drosophila S2 cells, and on polytene chromosomes, methyl-Lys 27 and Pc are both excluded from areas that are enriched in methyl-Lys 9 and HP1. Swapping of the chromodomain regions of Pc and HP1 is sufficient for switching the nuclear localization patterns of these factors, indicating a role for their chromodomains in both target site binding and discrimination. To better understand the molecular basis for the selection of methyl-lysine binding sites, we solved the 1.8 Angstrom structure of the Pc chromodomain in complex with a H3 peptide bearing trimethyl-Lys 27, and compared it with our previously determined structure of the HP1 chromodomain in complex with a H3 peptide bearing trimethyl-Lys 9. The Pc chromodomain distinguishes its methylation target on the H3 tail via an extended recognition groove that binds five additional residues preceding the ARKS motif.\", \n",
    "\"DNA methylation is essential for the control of a number of biological mechanisms in mammals [1]. Mammalian development is accompanied by two major waves of genome-wide demethylation and remethylation: one during germ-cell development and the other after fertilisation [2-7]. Most previous studies have suggested that the genome-wide demethylation observed after fertilisation occurs passively, that is, by the lack of maintenance methylation following DNA replication and cell division [6,7], although one other study has reported that replication-independent demethylation may also occur during early embryogenesis [8]. Here, we report that genes that are highly methylated in sperm are rapidly demethylated in the zygote only hours after fertilisation, before the first round of DNA replication commences. By contrast, the oocyte-derived maternal alleles are unaffected by this reprogramming. They either remain methylated after fertilisation or become further methylated de novo. These results provide the first direct evidence for active demethylation of single-copy genes in the mammalian zygote and, moreover, reveal a striking asymmetry in epigenetic methylation reprogramming. Whereas paternally (sperm)-derived sequences are exposed to putative active demethylases in the oocyte cytoplasm, maternally (oocyte)-derived sequences are protected from this reaction. These results, whose generality is supported by findings of Mayer et al. [9], have important implications for the establishment of biparental genetic totipotency after fertilisation, the establishment and maintenance of genomic imprinting, and the reprogramming of somatic cells during cloning.\", \n",
    "\"Aberrant hypermethylation of gene promoters is a major mechanism associated with inactivation of tumor-suppressor genes in cancer. We previously showed this transcriptional silencing to be mediated by both methylation and histone deacetylase activity, with methylation being dominant. Here, we have used cDNA microarray analysis to screen for genes that are epigenetically silenced in human colorectal cancer. By screening over 10,000 genes, we show that our approach can identify a substantial number of genes with promoter hypermethylation in a given cancer; these are distinct from genes with unmethylated promoters, for which increased expression is produced by histone deacetylase inhibition alone. Many of the hypermethylated genes we identified have high potential for roles in tumorigenesis by virtue of their predicted function and chromosome position. We also identified a group of genes that are preferentially hypermethylated in colorectal cancer and gastric cancer. One of these genes, SFRP1, belongs to a gene family; we show that hypermethylation of four genes in this family occurs very frequently in colorectal cancer, providing for (i) a unique potential mechanism for loss of tumor-suppressor gene function and (ii) construction of a molecular marker panel that could detect virtually all colorectal cancer.\", \n",
    "\"Gene function in cancer can be disrupted either through genetic alterations, which directly mutate or delete genes, or epigenetic alterations, which alter the heritable state of gene expression. The latter events are mediated by formation of transcriptionally repressive chromatin states around gene transcription start sites and an associated gain of methylation in normally unmethylated CpG islands in these regions. The genes affected include over half of the tumor suppressor genes that cause familial cancers when mutated in the germline; the selective advantage for genetic and epigenetic dysfunction in these genes is very similar. The aberrant methylation can begin very early in tumor progression and mediate most of the important pathway abnormalities in cancer including loss of cell cycle control, altered function of transcription factors, altered receptor function, disruption of normal cell-cell and cell-substratum interaction, inactivation of signal transduction pathways, loss of apoptotic signals and genetic instability. The active role of the aberrant methylation in transcriptional silencing of genes is becoming increasingly understood and involves a synergy between the methylation and histone deacetylase (HDAC) activity, This synergy can be mediated directly by HDAC interaction with DNA methylating enzymes and by recruitment through complexes involving methyl-cytosine binding proteins. In the translational arena, the promoter hypermethylation changes hold great promise as DNA tumor markers and their potentially reversible state creates a target for cancer therapeutic strategies involving gene reactivation.\", \n",
    "\"DNA methylation plays an important role in mammalian development and correlates with chromatin-associated gene silencing. The recruitment of MeCP2 to methylated CpG dinucleotides represents a major mechanism by which DNA methylation can repress transcription. MeCP2 silences gene expression partly by recruiting histone deacetylase (HDAC) activity, resulting in chromatin remodeling. Here, we show that MeCP2 associates with histone methyltransferase activity in vivo and that this activity is directed against Lys(9) of histone H3. Two characterized repression domains of MeCP2 are involved in tethering the histone methyltransferase to MeCP2. We asked if MeCP2 can deliver Lys(9)H3 methylation to the H19 gene, whose activity it represses. We show that the presence of MeCP2 on nucleosomes within the repressor region of the H19 gene (the differentially methylated domain) coincides with an increase in H3 Lys(9) methylation. Our data provide evidence that MeCP2 reinforces a repressive chromatin state by acting as a bridge between two global epigenetic modifications, DNA methylation and histone methylation.\", \n",
    "\"A large set of endogenous small RNAs of predominantly 21 to 24 nucleotides was identified in Arabidopsis. These small RNAs resembled micro-RNAs from animals and were similar in size to small interfering RNAs; that accumulated during RNA silencing triggered by multiple types of inducers. Among the 125 sequences identified, the vast majority (90%) arose from intergenic regions, although small RNAs corresponding to predicted protein-coding genes, transposon-like sequences, and a structural RNA gene also were identified. Evidence consistent with the derivation of small RNAs of both polarities, and from highly base-paired precursors, was obtained through the identification and analysis of clusters of small RNA loci. The accumulation of specific small RNAs was regulated developmentally. We propose that Arabidopsis small RNAs; participate in a wide range of post-transcriptional and epigenetic events.\", \n",
    "\"Recent results from diverse organisms point to a self-reinforcing network of interactions among the three best-characterized covalent modifications that mark heterochromatin: histone hypoacetylation, histone H3-Lys9 methylation, and cytosine methylation. These modification systems suggest a mechanistic basis for spreading of heterochromatin over large domains and for stable epigenetic inheritance of the silent state. All three modifications used in packaging heterochromatin are also used in stable silencing of euchromatic genes.\", \n",
    "\"Recent data in humans and animals suggest that assisted reproductive technology (ART) might affect the epigenetics of early embryogenesis and might cause birth defects. We report the first evidence, to our knowledge, that ART is associated with a human overgrowth syndrome-namely, Beckwith-Wiedemann syndrome (BWS). In a prospective study, the prevalence of ART was 4.6% (3 of 65), versus the background rate of 0.8% in the United States. A total of seven children with BWS were born after ART-five of whom were conceived after intracytoplasmic sperm injection. Molecular studies of six of the children indicate that five of the six have specific epigenetic alterations associated with BWS-four at LIT1 and one at both LIT1 and H19. We discuss the implications of our finding that ART is associated with human overgrowth, similar to the large offspring syndrome reported in ruminants.\", \n",
    "\"Cadmium is a heavy metal of considerable environmental and occupational concern. Cadmium compounds are classified as human carcinogens by several regulatory agencies. The most convincing data that cadmium is carcinogenic in humans comes from studies indicating occupational cadmium exposure is associated with lung cancer. Cadmium exposure has also been linked to human prostate and renal cancer, although this linkage is weaker than for lung cancer. Other target sites of cadmium carcinogenesis in humans, such as liver, pancreas and stomach, are considered equivocal. In animals, cadmium effectively induces cancers at multiple sites and by various routes. Cadmium inhalation in rats induces pulmonary adenocarcinomas, in accord with its role in human lung cancer. Cadmium can induce tumors and/or preneoplastic lesions within the rat prostate after ingestion or injection. At relatively high doses, cadmium induces benign testicular tumors in rats, but these appear to be due to early toxic lesions and loss of testicular function, rather than from a specific carcinogenic effect of cadmium. Like many other metals, cadmium salts will induce mesenchymal tumors at the site of subcutaneous (s.c.) or intramuscular (i.m.) injections, but the human relevance of these is dubious. Other targets of cadmium in rodents include the liver, adrenal, pancreas, pituitary, and hematopoietic system. With the exception of testicular tumors in rodents, the mechanisms of cadmium carcinogenesis are poorly defined. Cadmium can cause any number of molecular lesions that would be relevant to oncogenesis in various cellular model systems. Most studies indicate cadmium is poorly mutagenic and probably acts through indirect or epigenetic mechanisms, potentially including aberrant activation of oncogenes and suppression of apoptosis. (C) 2003 Elsevier B.V. All rights reserved.\", \n",
    "\"Diverse post-translational modifications of histone amino termini represent an important epigenetic mechanism for the organisation of chromatin structure and the regulation of gene activity. Within the past two years, great progress has been made in understanding the functional implications of histone methylation; in particular through the characterisation of histone methyltransferases that direct the site-specific methylation of, for example, lysine 9 and lysine 4 positions in the histone H3 amino terminus. All known histone methyltransferases of this type contain the evolutionarily conserved SET domain and appear to be able to stimulate either gene repression or gene activation. Methylation of H3 Lys9 and Lys4 has been visualised in native chromatin, indicating opposite roles in structuring repressive or accessible chromatin domains. For example, at the mating-type loci in Schizosaccharomyces pombe, at pericentric heterochromatin and at the inactive X chromosome in mammals, striking differences between these distinct marks have been observed. H3 Lys9 methylation is also important to direct additional epigenetic signals such as DNA methylation - for example, in Neurospora crassa and in Arabidopsis thaliana. Together, the available data strongly establish histone lysine methylation as a central modification for the epigenetic organisation of eukaryotic genomes.\", \n",
    "\"Polyploidy has long been recognized as a prominent force shaping the evolution of eukaryotes, especially flowering plants. New phenotypes often arise with polyploid formation and can contribute to the success of polyploids in nature or their selection for use in agriculture. Although the causes of novel variation in polyploids are not well understood, they could involve changes in gene expression through increased variation in dosage-regulated gene expression, altered regulatory interactions, and rapid genetic and epigenetic changes. New research approaches are being used to study these mechanisms and the results should provide a more complete understanding of polyploidy.\", \n",
    "\"Adult germline stem cells are capable of self-renewal, tissue regeneration and production of large numbers of differentiated progeny. We show here that the classical mouse mutant luxoid affects adult germline stem cell self-renewal. Young homozygous luxoid mutant mice produce limited numbers of normal spermatozoa and then progressively lose their germ line after birth. Transplantation studies showed that germ cells from mutant mice did not colonize recipient testes, suggesting that the defect is intrinsic to the stem cells. We determined that the luxoid mutant contains a nonsense mutation in the gene encoding Plzf, a transcriptional repressor that regulates the epigenetic state of undifferentiated cells, and showed that Plzf is coexpressed with Oct4 in undifferentiated spermatogonia. This is the first gene shown to be required in germ cells for stem cell self-renewal in mammals.\", \n",
    "\"The covalent modification of nucleosomal histones has emerged as a major determinant of chromatin structure and gene activity. To understand the interplay between various histone modifications, including acetylation and methylation, we performed a genome-wide chromatin structure analysis in a higher eukaryote. We found a binary pattern of histone modifications among euchromatic genes, with active genes being hyperacetylated for H3 and H4 and hypermethylated at Lys 4 and Lys 79 of 113, and inactive genes being hypomethylated and deacetylated at the same residues. Furthermore, the degree of modification correlates with the level of transcription, and modifications are largely restricted to transcribed regions, suggesting that their regulation is tightly linked to polymerase activity.\", \n",
    "\"Manipulation or non-physiological embryo culture environments can lead to defective fetal programming in livestock. Our demonstration of reduced fetal methylation and expression of ovine IGF2R suggests pre-implantation embryo procedures may be vulnerable to epigenetic alterations in imprinted genes. This highlights the potential benefits of epigenetic diagnostic screening in developing embryo procedures.\", \n",
    "\"Proteins from the Polycomb group (PcG) are epigenetic chromatin modifiers involved in cancer development and also in the maintenance of embryonic and adult stem cells, The therapeutic potential of stem cells and the growing conviction that tumors contain stem cells highlights the importance of understanding the extrinsic and intrinsic circuitry controlling stem cell fate and their connections to cancer.\", \n",
    "\"Genomes are organized into active regions known as euchromatin and inactive regions known as heterochromatin, or silenced chromatin. This review describes contemporary knowledge and models for how silenced chromatin in Saccharomyces cerevisiae forms, functions, and is inherited. In S. cerevisiae, Sir proteins are the key structural components of silenced chromatin. Sir proteins interact first with silencers, which dictate which regions are silenced, and then with histone tails in nucleosomes as the Sir proteins spread from silencers along chromosomes. Importantly, the spreading of silenced chromatin requires the histone deacetylase activity of Sir2p. This requirement leads to a general model for the spreading and inheritance of silenced chromatin or other special chromatin states Such chromatin domains are marked by modifications of the nucleosomes or DNA, and this mark is able to recruit an enzyme that makes further marks. Thus, among different organisms, multiple forms of repressive chromatin can be formed using similar strategies but completely different proteins. We also describe emerging evidence that mutations that cause global changes in the modification of histones can alter the balance between euchromatin and silenced chromatin within a cell.\", \n",
    "\"T lymphocytes differentiate in discrete stages within the thymus. Immature thymocytes lacking CD4 and CD8 coreceptors differentiate into double-positive cells (CD4(+)CD8(+)), which are selected to become either CD4(+)CD8(-)helper cells or CD4(-)CD8(+) cytotoxic cells. A stage-specific transcriptional silencer regulates expression of CD4 in both immature and CD4(-)CD8(+) thymocytes. We show here that binding sites for Runt domain transcription factors are essential for CD4 silencer function at both stages, and that different Runx family members are required to fulfill unique functions at each stage. Runxi is required for active repression in CD4(-)CD8(-) thymocytes whereas Runx3 is required for establishing epigenetic silencing in cytotoxic lineage thymocytes. Runx3-deficient cytotoxic T cells, but not helper cells, have defective responses to antigen, suggesting that Runx proteins have critical functions in lineage specification and homeostasis of CD8lineage T lymphocytes.\", \n",
    "\"The Cyclin D-Cdk4,6/lNK4/Rb/E2F pathway plays a key role in controlling cell growth by integrating multiple mitogenic and antimitogenic stimuli. The components of this pathway are gene families with a high level of structural and functional redundancy and are expressed in an overlapping fashion in most tissues and cell types. Using classical transgenic technology as well as gene-targeting in ES cells, a series of mouse models have been developed to study the in vivo function of individual components of this pathway in both normal homeostasis and tumor development. These models have proven to be useful to define specific as well as redundant roles among members of these cell cycle regulatory gene families. This pathway is deregulated in the vast majority of human tumors by genetic and epigenetic alterations that target at least some of its key members such as Cyclin D1, Cdk4, INK4a and INK4b, pRb etc. As a consequence, some of these molecules are currently being considered as targets for cancer therapy, and several novel molecules, such as Cdk inhibitors, are under development as potential anti-cancer drugs. (C) 2002 Elsevier Science B.V. All rights reserved.\", \n",
    "\"Twin studies have been vital for establishing an important genetic contribution to the etiology of schizophrenia. The five newest studies since 1995 from Europe and Japan have confirmed earlier findings. They yielded probandwise concordance rates of 41-65% in monozygotic (MZ) pairs and 0-28% in dizygotic (DZ) pairs, and heritability estimates of approximately 80-85%. Twin studies are also Valuable for investigating the etiological relationships between schizophrenia and other disorders, and the genetic basis of clinical heterogeneity within schizophrenia. Studies of discordant MZ pairs provide further insights into noninherited factors that contribute to the multifactorial etiology of this disorder. More recently, twin studies have begun to be used to directly investigate molecular genetic and epigenetic processes underlying schizophrenia. (C) 2000 Wiley-Liss, Inc.\", \n",
    "\"Interspecific or intergeneric hybridization, followed by chromosome doubling, can lead to the formation of new allopolyploid species. Recent studies indicate that allopolyploid formation is associated with genetic and epigenetic changes, although little is known about the type of changes that occur, how rapidly they occur, and the type of sequences involved. To address these matters, we have surveyed F1 hybrids between diploid species from the wheat (Aegilops and Triticum) group and their derived allotetraploids by screening a large number of loci using amplified fragment length polymorphism and DNA gel blot analysis and by assaying the extent of cytosine methylation. We found that sequence elimination is one of the major and immediate responses of the wheat genome to wide hybridization or allopolyploidy, that it affects a large fraction of the genome, and that it is reproducible. In one cross between Ae. sharonensis x Ae. umbellulata, 14% of the loci from Ae. sharonensis were eliminated compared with only 0.5% from Ae. umbellulata, with most changes occurring in the F1 hybrid. In contrast, crosses between Ae. longissima x T. urartu showed that sequence elimination was more frequent after chromosome doubling. Alterations in cytosine methylation occurred in similar to 13% of the loci, either in the F1 hybrid or in the allopolyploid. For eight of nine bands that were isolated, the sequences that underwent elimination corresponded to low-copy DNA, whereas alterations in methylation patterns affected both repetitive DNA sequences, such as retrotransposons, and low-copy DNA in approximately equal proportions.\", \n",
    "\"Cytosine methylation in CpG dinucleotides is believed to be important in gene regulation, and is generally associated with reduced levels of transcription. Methylation-mediated gene silencing involves a series of DNA-protein and protein-protein interactions that begins with the binding of methyl-CpG binding proteins (MBPs) followed by the recruitment of histone-modifying enzymes that together promote chromatin condensation and inactivation. It is widely known that alterations in methylation patterns, and associated gene activities, are often found in human tumors. However, the mechanisms by which methylation patterns are altered are not currently understood. In this paper, we investigate the impact of oxidative damage to a methyl-CpG site on MBP binding by the selective placement of 8-oxoguanine (8-oxoG) and 5-hydroxymethylcytosine (HmC) in a MBP recognition sequence. Duplexes containing these specific modifications were assayed for binding to the methyl-CpG binding domain (MBD) of one member of the MBP family, methyl-CpG binding protein 2 (MeCP2). Our results reveal that oxidation of either a single guanine to 8-oxoG or of a single 5mC to HmC, significantly inhibits binding of the MBD to the oligonucleotide duplex, reducing the binding affinity by at least an order of magnitude. Oxidative damage to DNA could therefore result in heritable, epigenetic changes in chromatin organization.\", \n",
    "\"Rett syndrome(1) (RTT, MIM 312750) is a progressive neurodevelopmental disorder and one of the most common causes of mental retardation in females, with an incidence of 1 in 10,000-15,000 (ref. 2). Patients with classic RTT appear to develop normally until 6-18 months of age, then gradually lose speech and purposeful hand use, and develop microcephaly, seizures, autism, ataxia, intermittent hyperventilation and stereotypic hand movements(3). After initial regression, the condition stabilizes and patients usually survive into adulthood. As RTT occurs almost exclusively in females, it has been proposed that RTT is caused by an X-linked dominant mutation with lethality in hemizygous males(3-8). Previous exclusion mapping studies using RTT families mapped the locus to Xq28 (refs 6,7,9-11). Using a systematic gene screening approach, we have identified mutations in the gene (MECP2) encoding X-linked methyl-CpG-binding protein 2 (MeCP2) as the cause of some Eases of RTT. MeCP2 selectively binds CpG dinucleotides in the mammalian genome and mediates transcriptional repression through interaction with histone deacetylase and the corepressor SIN3A (refs 12,13). In 5 of 21 sporadic patients, we found 3 de novo missense mutations in the region encoding the highly conserved methyl-binding domain (MBD) as well as a de novo frameshift and a de novo nonsense mutation, both of which disrupt the transcription repression domain (TRD). In two affected half-sisters of a RTT family, we found segregation of an additional missense mutation not detected in their obligate carrier mother. This suggests that the mother is a germline mosaic for this mutation. Our study reports the first disease-causing mutations in RTT and points to abnormal epigenetic regulation as the mechanism underlying the pathogenesis of RTT.\", \n",
    "\"The discovery of numerous hypermethylated promoters of tumour-suppressor genes, along with a better understanding of gene-silencing mechanisms, has moved DNA methylation from obscurity to recognition as an alternative mechanism of tumour-suppressor inactivation in cancer. Epigenetic events can also facilitate genetic damage, as illustrated by the increased mutagenicity of 5-methylcytosine and the silencing of the MLH1 mismatch repair gene by DNA methylation in colorectal tumours. We review here current mechanistic understanding of the role of DNA methylation in malignant transformation, and suggest Knudson's two-hit hypothesis should now be expanded to include epigenetic mechanisms of gene inactivation.\", \n",
    "\"Densely methylated DNA associates with transcriptionally repressive chromatin characterized by the presence of underacetylated histones(1,2). Recently, these two epigenetic processes have been dynamically linked. The methyl-CpG-binding protein MeCP2 appears to reside in a complex with histone deacetylase activity(3,4). MeCP2 can mediate formation of transcriptionally repressive chromatin on methylated promoter templates in vitro, and this process can be reversed by trichostatin A (TSA), a specific inhibitor of histone deacetylase(3-5). Little is known, however, about the relative roles of methylation and histone deacetylase activity in the stable inhibition of transcription on densely methylated endogenous promoters, such as those for silenced alleles of imprinted genes(6), genes on the female inactive X chromosome(7) and tumour-suppressor genes inactivated in cancer cells(8,9). We show here that the hypermethylated genes MLH1, TIMP3 (TIMP3), CDKN2B (INK4B, p15) and CDKN2A (INK4, p16) cannot be transcriptionally reactivated with TSA alone in tumour cells in which we have shown that TSA alone can upregulate the expression of non-methylated genes. Following minimal demethylation and slight gene reactivation in the presence of low dose 5-aza-2'deoxycytidine (5Aza-dC), however, TSA treatment results in robust re-expression of each gene. TSA does not contribute to demethylation of the genes, and none of the treatments alter the chromatin structure associated with the hypermethylated promoters. Thus, although DNA methylation and histone deacetylation appear to act as synergistic layers for the silencing of genes in cancer, dense CpG island methylation is dominant for the stable maintenance of a silent state at these loci.\", \n",
    "\" \", \n",
    "\"'Viable yellow' (A(vy)/alpha) mice are larger, obese, hyperinsulinemic, more susceptible to cancer, and, on average, shorter lived than their non-yellow siblings. They are epigenetic mosaics ranging from a yellow phenotype with maximum ectopic agouti overexpression, through a continuum of mottled agouti/yellow phenotypes with partial agouti overexpression, to a pseudoagouti phenotype with minimal ectopic expression. Pseudoagouti A(vy)/a mice are lean, healthy, and longer lived than their yellow siblings. Here we report that feeding pregnant black a/a dams methyl-supplemented diets alters epigenetic regulation of agouti expression in their offspring, as indicated by increased agouti/black mottling in the direction of the pseudoagouti phenotype, We also present confirmatory evidence that epigenetic phenotypes are maternally heritable. Thus A(vy) expression, already known to be modulated by imprinting, strain-specific modification, and maternal epigenetic inheritance, is also modulated by maternal diet. These observations suggest, at least in this special case, that maternal dietary supplementation may positively affect health and longevity of the offspring, Therefore, this experimental system should be useful for identifying maternal factors that modulate epigenetic mechanisms, especially DNA methylation, in developing embryos.\", \n",
    "\"The yeast Sir2 protein regulates epigenetic gene silencing and as a possible antiaging effect it suppresses recombination of rDNA. Studies involving cobB, a bacterial SIR2-like gene, have suggested it could encode a pyridine nucleotide transferase. Here five human sirtuin cDNAs are characterized. The SIRT1 sequence has the closest homology to the S. cerevisiae Sir2p. The SIRT4 and SIRT5 sirtuins more closely resemble prokaryotic sirtuin sequences. The five human sirtuins are widely expressed in fetal and adult tissues. Recombinant E. coli cobT and cobB proteins each showed a weak NAD-dependent mono-ADP-ribosyltransferase activity using 5,6-dimethylbenzimidazole as a substrate. Recombinant E. coli cobB and human SIRT2 sirtuin proteins were able to cause radioactivity to be transfered from [P-32]NAD to bovine serum albumin (BSA). When a conserved histidine within the human SIRT2 sirtuin was converted to a tyrosine, the mutant recombinant protein was unable to transfer radioactivity from [P-32]NAD to BSA. These results suggest that the sirtuins may function via mono-ADP-ribosylation Of proteins. (C) 1999 Academic Press.\", \n",
    "\"Gene activation in eukaryotes requires chromatin remodeling complexes like Swi/Snf and histone acetylases like SAGA. How these factors are recruited to promoters is not yet understood. Using CHIP, we measured recruitment of Swi/Snf, SAGA, the repressor Ash1p, and transcription factors Swi5p and SBF to the HO endonuclease promoter as cells progress through the yeast cell cycle. Swi5p's entry into nuclei at the end of anaphase recruits Swi/Snf, which then recruits SAGA. These two factors then facilitate SBF's binding. Ash1p, which only accumulates in daughter cell nuclei, binds to HO soon after Swi5p and aborts recruitment of Swi/Snf, SAGA, and SBF. Swi5p remains at HO for only 5 min. Swi/Snf's and SAGA's subsequent persistence at HO is self sustaining and constitutes an epigenetic memory of HO's transient interaction with Swi5p.\", \n",
    "\"Altering cytosine methylation by genetic means leads to a variety of developmental defects in mice(1), plants(2-5) and fungi(6,7) Deregulation of cytosine methylation also has a role in human carcinogenesis(8). In some cases, these defects have been tied to the inheritance of epigenetic alterations (such as chromatin imprints and DNA methylation patterns) that do not involve changes in DNA sequence(3,8-10). Using a forward genetic screen, we identified a gene (DDM1, decrease in DNA methylation) from the flowering plant Arabidopsis thaliana required to maintain normal cytosine methylation patterns(11). Additional ddm1 alleles (som4, 5, 6, 7, 8) were isolated in a selection for mutations that relieved transgene silencing(12) (E.J.R., unpublished data). Loss of DDM1 function causes a 70% reduction of genomic cytosine methylation, with most of the immediate hypomethylation occurring in repeated sequences(11). In contrast, many low-copy sequences initially retain their methylation in ddm1 homozygotes, but lose methylation over time as the mutants are propagated through multiple generations by self-pollination(3,13). The progressive effect of ddm1 mutations on low-copy sequence methylation suggests that ddm1 mutations compromise the efficiency of methylation of newly incorporated cytosines after DNA replication. In parallel with the slow decay of methylation during inbreeding, ddm1 mutants accumulate heritable alterations (mutations or stable epialleles) at dispersed sites in the genome that lead to morphological abnormalities(3,5,14). Here we report that DDM1 encodes a SWI2/SNF2-like protein, implicating chromatin remodelling as an important process for maintenance of DNA methylation and genome integrity.\", \n",
    "\"Chromatin assembly factor 1 (CAF-1) is required for inheritance of epigenetically determined chromosomal states in vivo and promotes assembly of chromatin during DNA replication in vitro. Herein, we demonstrate that after DNA replication, replicated, but not unreplicated, DNA is also competent for CAF-1-dependent chromatin assembly. The proliferating cell nuclear antigen (PCNA), a DNA polymerase clamp, is a component of the replication-dependent marking of DNA for chromatin assembly. The clamp loader, replication factor C (RFC), can reverse this mark by unloading PCNA from the replicated DNA. PCNA binds directly to p150, the largest subunit of CAF-1, and the two proteins colocalize at sites of DNA replication in cells. We suggest that PCNA and CAF-1 connect DNA replication to chromatin assembly and the inheritance of epigenetic chromosome states.\", \n",
    "\"A handful of autosomal genes in the mammalian genome are inherited in a silent state from one of the two parents, and in a fully active form from the other, thereby rendering the organism functionally hemizygous for imprinted genes. To date 19 imprinted genes have been identified; 5 are expressed from the maternal chromosome while the rest are expressed from the paternal chromosome. Allele-specific methylation of CpG residues, established in one of the germlines and maintained throughout embryogenesis, has been clearly implicated in the maintenance of imprinting in somatic cells. Although the function of imprinting remains a subject of some debate, the process is thought to have an important role in regulating the rate of fetal growth.\", \n",
    "\"Members of the Plasmodium falciparum var gene family encode clonally variant adhesins, which play an important role in the pathogenicity of tropical malaria. Here we employ a selective panning protocol to generate isogenic P falciparum populations with defined adhesive phenotypes for CD36, ICAM-1 and CSA, expressing single and distinct var gene variants. This technique has established the framework for examining var gene expression, its regulation and switching. It was found that var gene switching occurs irt situ. Ubiquitous transcription of all var gene variants appears to occur in early ring stages. However, var gene expression is tightly regulated in trophozoites and is exerted through a silencing mechanism. Transcriptional control is mutually exclusive in parasites that express defined adhesive phenotypes. lit situ var gene switching is apparently mediated at the level of transcriptional initiation, as demonstrated by nuclear run-on analyses. Our results suggest that an epigenetic mechanism(s) is involved in var gene regulation.\", \n",
    "\"Nuclear receptors (NRs) are ligand-inducible transcription factors that mediate complex effects on development, differentiation and homeostasis. They regulate the transcription of their target genes through binding to cognate DNA sequences as homodimers or heterodimers, The molecular mechanisms underlying transcriptional activation by NRs are still poorly understood, although intermediary factors (mediators) appear to be involved in mediating the transactivation functions of NRs, TIF1 has been identified previously as a protein that interacts specifically with the ligand binding domain of several nuclear receptors, both in yeast and in vitro. The characteristics of these interactions have led us to suggest that TIF1 might be a mediator of the NR ligand-inducible activation function AF-2, Using a two-hybrid screening in yeast, we have now identified two TLFP-binding proteins, mHP1 alpha and mMOD1, that are mouse homologues of the Drosophila heterochromatinic protein 1. Using mHP1 alpha as a bait in a second two-hybrid screening, we have isolated cDNAs encoding proteins that are also very likely to be involved in chromatin structure and function, as well as a protein structurally and functionally related to TIF1 (renamed TLF1 alpha), which was named TIF1 beta. Here we discuss how the function of members of the TIF1 family in the control of transcription could be exerted at the level of the structure of the chromatin template.\", \n",
    "\"The past few years have seen a wider acceptance of a role for DNA methylation in cancer. This can be attributed to three developments. First, the documentation of the over-representation of mutations at CpG dinucleotides has convincingly implicated DNA methylation in the generation of oncogenic point mutations. The second important advance has been the demonstration of epigenetic silencing of tumor suppressor genes by DNA methylation. The third development has been the utilization of experimental methods to manipulate DNA methylation levels. These studies demonstrate that DNA methylation changes in cancer cells are not mere by-products of malignant transformation, but can play an instrumental role in the cancer process. It seems clear that DNA methylation plays a variety of roles in different cancer types and probably at different stages of oncogenesis. DNA methylation is intricately involved in a wide diversity of cellular processes. Likewise, it appears to exert its influence on the cancer process through a diverse array of mechanisms. It is our task not only to identify these mechanisms, but to determine their relative importance for each stage and type of cancer. Our hope then will be to translate that knowledge into clinical applications.\", \n",
    "\"The inactive X chromosome differs from the active X in a number of ways; some of these, such as allocyclic replication and altered histone acetylation, are associated with all types of epigenetic silencing, whereas others, such as DNA methylation, are of more restricted use. These features are acquired progressively by the inactive X after onset of initiation. Initiation of X-inactivation is controlled by the X-inactivation center (Xic) and influenced by the X chromosome controlling element (Xce), which causes primary nonrandom X-inactivation. Other examples of nonrandom X-inactivation are also presented in this review. The definition of a major role for Xist, a noncoding RNA, in X-inactivation has enabled investigation of the mechanism leading to establishment of the heterochromatinized X-chromosome and also of the interactions between X-inactivation and imprinting as well as between X-inactivation and developmental processes in the early embryo.\", \n",
    "\"The centromere is required to ensure the equal distribution of replicated chromosomes to daughter nuclei. Centromeres are frequently associated with heterochromatin, an enigmatic nuclear component that causes the epigenetic transcriptional repression of nearby marker genes (position-effect variegation of silencing). The process of chromosome segregation by movement along microtubules to spindle poles is highly conserved, yet the putative cis-acting centromeric DNA sequences bear little or no similarity across species. Recently, studies in several systems have revealed that the centromere itself might be epigenetically regulated and that the higher-order structure of the underlying heterochromatin contributes to centromere function and kinetochore assembly.\", \n",
    "\"The chrome and SET domains are conserved sequence moths present in chromosomal proteins that function in epigenetic control of gene expression, presumably by modulating higher order chromatin, Based on sequence information from the SET domain, we have isolated human (SUV39H1) and mouse (Suv39h1) homologues of the dominant Drosophila modifier of position-effect-variegation (PEV) Su(var)3-9. Mammalian homologues contain, in addition to the SET domain, the characteristic chrome domain, a combination that is also preserved in the Schizosaccharyomyces pombe silencing factor clr4. Chromatin-dependent gene regulation is demonstrated by the potential of human SUV39H1 to increase repression of the pericentromeric white marker gene in transgenic flies. Immunodetection of endogenous Suv39h1/SUV39H1 proteins in a variety of mammalian cell lines reveals enriched distribution at heterochromatic foci during interphase and centromere-specific localization during metaphase, In addition, Suv39h1/SUV39H1 proteins associate with M31, currently the only other characterized mammalian SU(VAR) homologue, These data indicate the existence of a mammalian SU(VAR) complex and define Suv39h1/SUV39H1 as novel components of mammalian higher order chromatin.\", \n",
    "\"post-transcriptional gene silencing (PTGS) is a homology-dependent process that reduces cytoplasmic RNA levels. In several experimental systems, there is also an association of PTGS with methylation of DNA. To investigate this association, we used plants carrying a transgene encoding the green fluorescent protein (GFP), Gene silencing was induced using potato virus X RNA vectors carrying parts of the coding sequence or the promoter of the GFP transgene. In each instance, homology-based, RNA-directed methylation was associated with silencing. When the GFP-transcribed region was targeted, PTGS affected both transgene and viral RNA revels. When methylation was targeted to a promoter region, transgene RNA levels were reduced; however, viral RNA revels were unaffected. For comparison, we induced PTGS of the gene encoding the endogenous ribulose-1,5-bisphosphate carboxylase oxygenase (Rubisco) small subunit (rbcS) by inoculation with potato virus X-rbcS, In this example, no methylation of the rbcS DNA was associated with the reduction in rbcS transcript levels, and viral RNA levels were unaffected. Finally, we investigated DNA methylation by using GFP-transformed plants in which PTGS was induced by localized introduction of a T-DNA carrying GFP sequences. In these plants, there was methylation of a GFP transgene associated with systemic spread of a gene-silencing signal from the infiltrated part of the plant. This transgene methylation was not affected when systemic PTGS was blocked by suppressors of silencing encoded by potato virus Y and cucumber mosaic virus. Combined, these data support an epigenetic model of PTGS in which transgene methylation is associated with an RNA-DNA interaction that ensures that PTGS is maintained.\", \n",
    "\"Although the link between the BRCA1 tumour-suppressor gene and hereditary breast and ovarian cancer is established(1-5), the role, if any, of BRCA1 in non-familial cancers is unclear. BRCA1 mutations are rare in sporadic cancers(6-8), but loss of BRCA1 resulting from reduced expression or incorrect subcellular localization(9,10) is postulated to be important in non-familial breast and ovarian cancers. Epigenetic loss, however, has not received general acceptance due to controversy regarding the subcellular localization of BRCA1 proteins, reports of which have ranged from exclusively nuclear(11-15), to conditionally nuclear(10), to the ER/golgi(16), to cytoplasmic invaginations into the nucleus(17). In an attempt to resolve this issue, we have comprehensively characterized 19 anti-BRCA1. antibodies. These reagents detect a 220-kD protein localized in discrete nuclear foci in all epithelial cell lines, including those derived from breast malignancies. Immunohistochemical staining of human breast specimens also revealed BRCA1 nuclear foci in benign breast, invasive lobular cancers and low-grade ductal carcinomas. Conversely, BRCA1 expression was reduced or undetectable in the majority of high-grade, ductal carcinomas, suggesting that absence of BRCA1 may contribute to the pathogenesis of a significant percentage of sporadic breast cancers.\", \n",
    "\"CpG island hypermethylation is known to be associated with gene silencing in cancer. This epigenetic event is generally accepted as a stochastic process in tumor cells resulting from aberrant DNA methyltransferase (DNA-MTase) activities. Specific patterns of CPG island methylation could result from clonal selection of cells having growth advantages due to silencing of associated tumor suppressor genes. Alternatively, methylation patterns may be determined by other, as yet unidentified factors, To explore further the underlying mechanisms, we developed a novel array-based method, called differential methylation hybridization (DMH), which allows a genome-wide screening of hypermethylated CpG islands in tumor cells. DMH was used to determine the methylation status of >276 CpG island loci in a group of breast cancer cell lines. Between 5 and 14% of these loci were hypermethylated extensively in these cells relative to a normal control. Pattern analysis of 30 positive loci by Southern hybridization indicated that CpG islands might differ in their susceptibility to hypermethylation, Loci exhibiting preexisting methylation in normal controls were more susceptible to de novo methylation in these cancer cells than loci without this condition, In addition, these cell lines exhibited different intrinsic abilities to methylate CpG islands not directly associated with methyltransferase activities. Our study provides evidence that, aside from random DNA-MTase action, additional cellular factors exist that govern aberrant methylation in breast cancer cells.\", \n",
    "\"Genomic reprogramming of primordial germ cells (PGCs), which includes genome-wide demethylation, prevents aberrant epigenetic modifications from being transmitted to subsequent generations, This process also ensures that homologous chromosomes first acquire an identical epigenetic status before an appropriate switch in the imprintable loci in the female and male germ lines. Embryonic germ (EG) cells have a similar epigenotype to PGCs from which they are derived, We used EG cells to investigate the mechanism of epigenetic modifications in the germ line by analysing the effects on a somatic nucleus in the EG-thymic lymphocyte hybrid cells. There were striking changes in methylation of the somatic nucleus, resulting in demethylation of several imprinted and non-imprinted genes, These epigenetic modifications were heritable and affected gene expression as judged by re-activation of the silent maternal allele of Peg1/Mest imprinted gene in the somatic nucleus. This remarkable change in the epigenotype of the somatic nucleus is consistent with the observed pluripotency of the EG-somatic hybrid cells as they differentiated into a variety of tissues in chimeric embryos. The epigenetic modifications observed in EG-somatic cell hybrids in vitro are comparable to the reprogramming events that occur during germ cell development.\", \n",
    "\"To determine which CD3 components are required for early T cell development, we generated mice with a targeted mutation of the CD3-epsilon gene and characterized their T cell populations relative to those found in CD3-zeta/eta- and recombinase activating gene (RAG)-deficient mice, In the absence of intact CD3-epsilon subunit, thymocytes do not progress beyond the CD44(-/low)CD25(+) triple-negative stage and appear to be arrested at the very same developmental control point as RAG-deficient thymocytes, In contrast, the disruption of the CD3-zeta/eta gene does not totally abrogate the progression through this control point, CD3-epsilon-deficient thymocytes do rearrange their T cell receptor (TCR) beta gene segments and produce low levels of full-length TCR beta transcripts. Taken together, these results establish an essential role for the CD3-epsilon gene products during T cell development and further suggest that the CD3-epsilon polypeptides start to exert their function as part of a pre-TCR through which CD44(-/low)CD25(+) triple-negative cells monitor the occurrence of productive TCR beta gene rearrangements. Finally, the absence of intact CD3-epsilon polypeptides had no discernible effect on the completion of TCR gamma and TCR delta gene rearrangements, emphasizing that they are probably not subjected to the same epigenetic controls as those operating on the expression of TCR alpha and beta genes.\", \n",
    "\"The constitutive DNase I hypersensitive site at the 5' end of the chicken beta-globin locus marks the boundary of the active chromatin domain in erythroid cells. The DNA sequence containing this site has the properties of an insulator, as shown by its ability in stable transformation experiments to block enhancer-promoter interaction when it lies between the two, but not when it lies outside, and to protect against position effects in Drosophila. We now show that the chicken insulator can protect a stably integrated gene, which is otherwise subject to great variability of expression, from chromatin-mediated repression in cell culture. When the integrated reporter gene is surrounded by insulator elements, stably transformed cell lines display consistent enhancer-dependent expression levels, in accord with the strength of the enhancer. In the absence of insulators, long-term nonselective propagation of cells carrying the integrated reporter gene results in gradual extinction of the reporter's expression, with expression patterns from tandemly repeated inserted genes suggesting that the extinction of adjacent genes is coupled. We show that the uninsulated reporter genes, in addition to becoming transcriptionally inactive, lose several epigenetic hallmarks of active chromatin, including nuclease accessibility, DNA hypomethylation, and histone hyperacetylation during time in culture. Treatment with inhibitors of histone deacetylase or DNA methylation reverses the extinction of the uninsulated genes. Extinction is completely prevented by flanking the reporter construct with insulators. Furthermore, in contrast to the uninsulated reporter genes, chromatin over the insulated genes retains nuclease accessibility and histone hyperacetylation. However, there is no clear correlation between the presence of the insulators and the level of DNA methylation. This leads us to propose a model for the insulator's ability to protect against extinction in the transformed cell lines and to function as a chromatin boundary for the chicken beta-globin locus in normal erythroid cells.\", \n",
    "\"Kruppel-associated box (KRAB) domains are present in approximately one-third of all human zinc finger proteins (ZFPs) and are potent transcriptional repression modules. We have previously cloned a corepressor for the KRAB domain, KAP-1, which is required for KRAB-mediated repression in vivo. To characterize the repression mechanism utilized by KAP-1, we have analyzed the ability of KAP-1 to interact with murine (M31 and M32) and human (HP1 alpha and HP1 gamma) homologues of the HPI protein family, a class of nonhistone heterochromatin-associated proteins with a well-established epigenetic gene silencing function in Drosophila, In vitro studies confirmed that KAP-1 is capable of directly interacting with M31 and hHP1 alpha, which are normally found in centromeric heterochromatin, as well as M32 and hHP1 gamma, both of which are found in euchromatin, Mapping of the region in KAP-1 required for HPI interaction showed that amino acid substitutions which abolish HP1 binding in vitro reduce KAP-1 mediated repression in vivo. We observed colocalization of KAP-1 with M31 and M32 in interphase nuclei, lending support to the biochemical evidence that M31 and M32 directly interact with KAP-1, The colocalization of KAP-1 with M31 is sometimes found in subnuclear territories of potential pericentromeric heterochromatin, whereas colocalization of KAP-1 and M32 occurs in punctate euchromatic domains throughout the nucleus. This work suggests a mechanism for the recruitment of HP1-like gene products by the KRAB-ZFP-KAP-1 complex to specific loci within the genome through formation of heterochromatin-like complexes that silence gene activity. We speculate that gene-specific repression may be a consequence of the formation of such complexes, ultimately leading to silenced genes in newly formed heterochromatic chromosomal environments.\", \n",
    "\"Flower pigmentation patterns were scored in 185 sense Chalcone synthase (Chs) transgenotes and 85 antisense Chs transgenotes; upon first flowering, 139 (75%) of sense transgenotes were found to be phenotypically altered, as were 70 (82%) of the antisense transgenotes. The observed patterns document the range of phenotypic variations that occur, as well as confirm and extend the finding that sense Chs constructs produce several types of morphology-based flower pigmentation patterns that antisense Chs constructs do not. Long-term monitoring for epigenetic variations in one population of 44 sense Chs transgenotes showed that 43 (98%) were capable of producing a cosuppression phenotype. The primary determinant of sense-specific patterns of cosuppression of Chs was found to be the repetitiveness and organization pattern of the transgene, not 'position effects' by, or 'readthrough' from, flanking plant DNA sequences. The degree of cosuppression observed in progeny of transgenotes carrying multiple, dispersed copies as compared to that observed with a single copy of the transgene suggests that sense cosuppression of Chs is subject to a transgene dosage effect.\", \n",
    "\"A transgenic gpt(+) Chinese hamster cell line (G12) was found to be susceptible to carcinogenic nickel-induced inactivation of gpt expression without mutagenesis or deletion of the transgene. Many nickel-induced 6-thioguanine-resistant variants spontaneously reverted to actively express gpt, as indicated by both reversion assays and direct enzyme measurements. Since reversion was enhanced in many of the nickel-induced variant cell lines following 24-h treatment with the demethylating agent 5-azacytidine, the involvement of DNA methylation in silencing gpt expression was suspected. This was confirmed by demonstrations of increased DNA methylation, as well as by evidence indicating condensed chromatin and heterochromatinization of the gpt integration site in 6 thioguanine-resistant cells. Upon reversion to active gpt expression, DNA methylation and condensation are lost. We propose that DNA condensation and methylation result in heterochromatinization of the gpt sequence with subsequent inheritance of the now silenced gene. This mechanism is supported by direct evidence showing that acute nickel treatment of cultured cells, and of isolated nuclei in vitro, can indeed facilitate gpt sequence-specific chromatin condensation. Epigenetic mechanisms have been implicated in the actions of some nonmutagenic carcinogens, and DNA methylation changes are now known to be important in carcinogenesis. This paper further supports the emerging theory that nickel is a human carcinogen that can alter gene expression by enhanced DNA methylation and compaction, rather than by mutagenic mechanisms.\", \n",
    "\"In vitro manipulation of preimplantation mammalian embryos can influence differentiation and growth at later stages of development, In the mouse, culture of embryonic stem (ES) cells affects their totipotency and may give rise to fetal abnormalities, To investigate whether this is associated with epigenetic alterations in imprinted genes, we analysed two maternally expressed genes (Igf2r, H19) and two paternally expressed genes (Igf2, U2af1-rs1) in ES cells and in completely ES cell-derived fetuses, Altered allelic methylation patterns were detected in all four genes, and these were consistently associated with allelic changes in gene expression, All the methylation changes that had arisen in the ES cells persisted on in vivo differentiation to fetal stages, Alterations included loss of methylation with biallelic expression of U2af1-rs1, maternal methylation and predominantly maternal expression of Igf2, and biallelic methylation and expression of Igf2r In many of the ES fetuses, the levels of H19 expression were strongly reduced, and this biallelic repression was associated with biallellic methylation of the H19 upstream region, Surprisingly, biallelic H19 repression was not associated with equal levels of Igf2 expression from both parental chromosomes, but rather with a strong activation of the maternal Igf2 allele, ES fetuses derived from two of the four ES lines appeared developmentally compromised, with polyhydramnios, poor mandible development and interstitial bleeding and, in chimeric fetuses, the degree of chimerism correlated with increased fetal mass, Our study establishes a model for how early embryonic epigenetic alterations in imprinted genes persist to later developmental stages, and are associated with aberrant phenotypes.\", \n",
    "\"Polycomb group (PcG) and trithorax group (trxG) gene products are responsible for the maintenance of repressed and active expression patterns of many developmentally important regulatory genes including the homeotic genes. In Drosophila embryos, Polycomb protein and the trxG protein GAGA factor colocalize at the Fab-7 DNA element of the bithorax complex. In transgenic lines, the Fab-7 element induces extensive silencing on a flanking GAL4-driven lacZ reporter and mini-white genes. However, a short single pulse of GAL4 during embryogenesis is sufficient to release PcG-dependent silencing from the transgene. Such an activated state of Fab-7 is mitotically inheritable through development and can be transmitted in a GAL4-independent manner to the subsequent generations through female meiosis. Thus, Fab-7 is a switchable chromosomal element, which can convey memory of epigenetically determined active and repressed chromatin states.\", \n",
    "\"Methylation of cytosine residues in DNA provides a mechanism of gene control. There are two classes of methyltransferase in Arabidopsis; one has a carboxy-terminal methyltransferase domain fused to an amino-terminal regulatory domain and is similar to mammalian methyltransferases. The Second class apparently lacks an amino-terminal domain and is less well conserved. Methylcytosine can occur at any cytosine residue, but it is likely that clonal transmission of methylation patterns only occurs for cytosines in strand-symmetrical sequences CpG and CpNpG. In plants, as in mammals, DNA methylation has dual roles in defense against invading DNA and transposable elements and in gene regulation. Although originally reported as having no phenotypic consequence, reduced DNA methylation disrupts normal plant development.\", \n",
    "\"The trilaminar kinetochore directs the segregation of chromosomes in mitosis and meiosis. Despite its importance, the molecular architecture of this structure remains poorly understood [1]. The best known component of the kinetochore plates is CENP-C, a protein that is required for kinetochore assembly [2], but whose molecular role in kinetochore structure and function is unknown. Here we have raised for the first time monospecific antisera to CENP-A [3], a 17 kD centromere-specific histone variant that is 62% identical to the carboxy-terminal domain of histone H3 [4,5] and that resembles the yeast centromeric component CSE4 [6]. We have found by simultaneous immunofluorescence with centromere antigens of known ultrastructural location that CENP-A is concentrated in the region of the inner kinetochore plate at active centromeres. Because CENP A was previously shown to co-purify with nucleosomes [7], our data suggest a specific nucleosomal substructure for the kinetochore. In human cells, these kinetochore-specific nucleosomes are enriched in alpha-satellite DNA [8]. However, the association of CENP-A with neocentromeres lacking detectable alpha-satellite DNA, and the lack of CENP-A association with alpha-satellite-rich inactive centromeres of dicentric chromosomes together suggest that CENP-A association with kinetochores is unlikely to be determined solely by DNA sequence recognition. We speculate that CENP-A binding could be a consequence of epigenetic tagging of mammalian centromeres.\", \n",
    "\"Nucleolar dominance is an epigenetic phenomenon that describes nucleolus formation around rRNA genes inherited from only one progenitor of an interspecific hy;brid or allopolyploid. The phenomenon is widespread, occurring in plants, insects, amphibians, and mammals, yet its molecular basis remains unclear. We have demonstrated nucleolar dominance in three allotetraploids of the plant genus Brassica. In Brassica napus, accurately initiated pre-rRNA transcripts from one progenitor, Brassica rapa are detected readily, whereas transcripts from the similar to 3000 rRNA genes inherited from the other progenitor, Brassica oleracea, are undetectable. Nuclear run-on confirmed that dominance is controlled at the level of transcription. Growth of B. napus seedlings on 5-aza-2'-deoxycytidine to inhibit cytosine methylation caused the normally silent, under-dominant B. oleracea rRNA genes to become expressed to high levels. The histone deacetylase inhibitors sodium butyrate and trichostatin A also de-epressed silent rRNA genes. These results reveal an enforcement mechanism for nucleolar dominance in which DNA methylation and histone modifications combine to regulate rRNA gene loci spanning tens of megabase pairs of DNA.\", \n",
    "\"Genomic imprinting is an epigenetic chromosomal modification in the gamete or zygote causing preferential expression of a specific parental allele in somatic cells of the offspring. We and others have identified three imprinted human genes on 11p15.5, IGF2 (refs 1-4), H19 (refs 1,5), and p57(KIP2) (ref. 6), although the latter gene is separated by 700 kb from the other two, and it is unclear whether there are other imprinted genes within this large interval. We previously mapped an embryonal tumour suppressor gene to this region(7), as well as five balanced germline chromosomal rearrangement breakpoints from patients with Beckwith-Wiedemann syndrome (BWS)(8), a condition characterized by prenatal overgrowth and cancer. We isolated the upstream exons of the previously identified gene KVLQT1, which causes the familial cardiac defect long-QT (LQT) syndrome. We found that KVLQT1 spans much of the interval between p57(KIP2) and IGF2, and that it is also imprinted. We demonstrated that the gene is disrupted by chromosomal chromosomal translocation in an embryonal rhabdoid tumour. Furthermore, the lack of parent-of-origin effect in LQT syndrome appears to be due to relative lack of imprinting in the affected tissue, cardiac muscle, representing a novel mechanism for variable penetrance of a human disease gene.\", \n",
    "\"A multisubunit cohesin complex holds sister chromatids together after DNA replication. Using chromatin immunoprecipitation, we detected cohesin association with centromeres and with discrete sites along chromosome arms from S phase until metaphase in S. cerevisiae, Short DNA sequences (130-280 bp) are sufficient to confer cohesin association. Cohesin association with a centromere depends on Mif2p, the centromere binding factor CBF3, and a centromere-specific histone variant, Cse4p. Because only active centromeres confer cohesin association with centromeric DNA, we suggest that cohesin is recruited by the same chromatin structure that confers the attachment of microtubules. Propagation of this structure might be partly epigenetic. Finally, cohesion associated with minimal centromeres is insufficient to resist the splitting force exerted by microtubules and appears to be reinforced by cohesion provided by their flanking DNA sequences.\", \n",
    "\"Loss of imprinting (LOI) is an epigenetic alteration of some cancers involving loss of parental origin-specific expression of imprinted genes. We observed LOI of the insulin-like growth factor-ii gene in twelve of twenty-seven informative colorectal cancer patients (44%), as well as in the matched normal colonic mucosa of the patients with LOI in their cancers, and in peripheral blood samples of four patients. Ten of eleven cancers (91%) with microsatellite instability showed LOI, compared with only two of sixteen tumors (12%) without microsatellite instability (P< 0.001). Control patients without cancer showed LOI in colonic mucosa of only two of sixteen cases (12%, P < 0.001) and two of fifteen blood samples (13%, P < 0.001). These data suggest that LOI in tumor and normal tissue identifies most colorectal cancer patients with microsatellite instability in their tumors, and that LOI may identify an important subset of the population with cancer or at risk of developing cancer.\", \n",
    "\"Silencing is an epigenetic form of transcriptional regulation whereby genes are heritably, but not necessarily permanently, inactivated. We have identified the Saccharomyces cerevisiae genes SAS2 and SAS3 through a screen for enhancers of sir1 epigenetic silencing defects. SAS2, SAS3 and a Schizosaccharomyces pombe homologue are closely related to several human genes, including one associated with acute myeloid leukaemia arising from the recurrent translocation t(8;16)(p11;p13) and one implicated in HIV-1 Tat interactions. All of these genes encode proteins with an atypical zinc finger and well-conserved similarities to acetyltransferases. Sequence similarities and yeast mutant phenotypes suggest that SAS-like genes function in transcriptional regulation and cell-cycle exit and reveal novel connections between transcriptional silencing and human disease.\", \n",
    "\"The multisubunit MSL dosage compensation complex binds to hundreds of sites along the Drosophila single male X chromosome, mediating its hypertranscription, The male X chromosome is also coated with noncoding roX RNAs. When either msl3, mle, or mof is mutant, a partial MSL complex is bound at only similar to 35 unusual sites distributed along the X, We show that two of these sites are the roX1 and roX2 genes and postulate that one of their functions is to provide entry sites for the MSL complex to recognize the X chromosome. The roX1 gene provides a nucleation site for extensive spreading of the MSL complex into flanking chromatin even when moved to an autosome. The spreading can occur in cis or in trans between paired homologs. We present a model for how the dosage compensation complex recognizes X chromatin.\", \n",
    "\"Defective DNA mismatch repair in human turners leads to genome-wide instability of microsatellite repeats and a molecular phenotype referred to as microsatellite instability (MSI), MSI has been reported in a variety of cancers and is a consistent feature of tumors from patients with hereditary non-polyposis colorectal cancer. Approximately 20% of cancers of the uterine endometrium, the fifth most common cancer of women world-wide, exhibit MSI. Although the frequency of MSI is higher in endometrial cancers than in any other common malignancy, the genetic basis of MSI in these tumors has remained elusive, We investigated the role that methylation of the MLH1 DNA mismatch repair gene plays in the genesis of MSI in a large series of sporadic endometrial cancers. The MLH1 promoter was methylated in 41 of 53 (77%) MSI-positive cancers investigated. In MSI-negative tumors on the other hand, there was evidence for limited methylation in only one of 11 tumors studied. Immunohistochemical investigation of a subset of the tumors revealed that methylation of the MLH1 promoter in MSI-positive tumors was associated with loss of MLH1 expression. Immunohistochemistry proved that two MSI-positive tumors lacking MLH1 methylation failed to express the MSH2 mismatch repair gene. Both of these cancers came from women who had family and medical histories suggestive of inherited cancer susceptibility. These observations suggest that epigenetic changes in the MLH1 locus account for MSI in most cases of sporadic endometrial cancers and provide additional evidence that the MSH2 gene may contribute substantially to inherited forms of endometrial cancer.\", \n",
    "\"Several lines of evidence strongly suggest that DNA methylation is involved in embryo development, Perhaps the most direct evidence comes from experiments with methyltransferase deficient mice. Embryos that express low levels of the maintenance methyltransferase do not develop to term and die at the 5 to 20 somite stage corresponding to the level of the enzyme. In the mouse, dramatic methylation changes have been observed during the early steps of embryo development. Most genes are subject to a process of active demethylation starting promptly after fertilization. Except for a small number of methylated CpG sites in imprinted genes the vast majority of the sites are unmethylated by the stage of cavitation (16 cell). Such genome-wide demethylation may signify an erasure of epigenetic information originating in the highly differentiated gametes. This erasure may be essential for the establishment of a pluripotent state, before specific cell lineages can be determined. The process of laying down a new developmental program involves, initially, global de novo methylation at the stage of pregastrulation followed by gene specific demethylations associated with the onset of activity of these genes. CpG islands characteristic of housekeeping genes, appear to be protected from the global de novo methylation. An exception to this rule is the hypermethylation of CpG islands in X-linked housekeeping genes on the inactive X chromosome and of specific differentially methylated CpG sites in imprinted genes. Primordial germ cells escape the global de novo methylation which takes place at the pregastrula stage and undergo a very similar de novo methylation process in the differentiated gonads (15.5-18.5 days post coitum), forming the methylation patterns which are specific to the gametes. Specific demethylations then form a terminal methylation pattern which is then clonaly inherited in the soma and erased after fertilization.\", \n",
    "\"During the development of a multicellular organism from a zygote, a large number of epigenetic interactions take place on every level of suborganismal organization. This raises the possibility that the system of epigenetic interactions may compensate or ''buffer'' some of the changes that occur as mutations on its lowest levels, and thus stabilize the phenotype with respect to mutations. This hypothetical phenomenon will be called ''epigenetic stability.'' Its potential importance stems from the fact that phenotypic variation with a genetic basis is an essential prerequisite for evolution. Thus, variation in epigenetic stability might profoundly affect attainable rates of evolution. While representing a systemic property of a developmental system, epigenetic stability might itself be genetically determined and thus be subject to evolutionary change, Whether or not this is the case should ideally be answered directly, that is, by experimentation. The time seals involved and our insufficient quantitative understanding of developmental pathways will probably preclude such an approach in the foreseeable future. Preliminary answers are sought here by using a biochemically motivated model of a small bur central part of a developmental pathway. Modeled are sets of transcriptional regulators that mutually regulate each other's expression and thereby form stable gene expression patterns. Such gene-expression patterns, crucially involved in determining developmental pattern formation events, are most likely subject to strong stabilizing natural selection, After long periods of stabilizing selection. the fraction of mutations causing changes in gene-expression pattern is substantially reduced in the model. Epigenetic stability has increased. This phenomenon is found for widely varying regulatory scenarios among transcription factor genes. Its discussed that only epistatic (nonlinear) gene interactions can cause such change in epigenetic stability. Evidence from paleontology, molecular evolution, development, and genetics, consistent with the existence of variation in epigenetic stability, is discussed, The relation of epigenetic stability to developmental canalization is outlined. Experimental scenarios at suggested thar may provide further evidence.\", \n",
    "\"X-chromosome inactivation is the process by which a cell recognizes the presence of two copies of an X chromosome early in the development of XX embryos and chooses one to be active and one to be inactive(1) Although it is commonly believed that the initiation of X inactivation is random, with an equal probability (50:50) that either X chromosome will be the inactive X in a given cell, significant variation in the proportion of cells with either X inactive is observed both in mice heterozygous for alleles at the Xce locus(2) and among normal human females in the population(3-5). Families in which multiple females demonstrate extremely skewed inactivation patterns that are otherwise quite rare in the general population are thought to reflect possible genetic influences on the X-inactivation process(5-7). Here we report a rare cytosine to guanine mutation in the XIST minimal promoter that underlies both epigenetic and functional differences between the two X chromosomes in nine females from two unrelated families. All females demonstrate preferential inactivation of the X chromosome carrying the mutation, suggesting that there is an association between alterations in the regulation of XIST expression and X-chromosome inactivation.\", \n",
    "\"Genomic imprinting is an epigenetic process that results in the preferential silencing of one of the two parental copies of a gene. Although the precise mechanisms by which genomic imprinting occurs are unknown, the tendency of imprinted genes to exist in chromosomal clusters suggests long-range regulation through shared regulatory elements. We characterize a 800-kb region on the distal end of mouse chromosome 7 that contains a cluster of four maternally expressed genes, H19, Mash2, Kvlqt1, and p57(Kip2), as well as two paternally expressed genes, Igf2 and Ins2, and assess the expression and imprinting of Mash2, Kvlqt1, and p57(Kip2) during development in embryonic and extraembryonic tissues. Unlike Igf2 and Ins2, which depend on H19 for their imprinting, Mash2, p57(Kip2), and Kvlqt1 are unaffected by a deletion of the H19 gene region, suggesting that these more telomeric genes are not regulated by the mechanism that controls H19, Igf2, and Ins2. Mutations in human p57(Kip2) have been implicated in Beckwith-Wiedemann syndrome, a disease that has also been associated with loss of imprinting of IGF2. We find, however, that a deletion of the gene has no effect on imprinting within the cluster. Surprisingly, the three maternally expressed genes are regulated very differently by DNA methylation; p57(Kip2) activated, Kvlqt1 is silenced, and Mash2 is unaffected in mice lacking DNA methyltransferase. We conclude that H19 is not a global regulator of imprinting on distal chromosome 7 and that the telomeric genes are imprinted by a separate mechanism(s).\", \n",
    "\"Skeletal myoblasts are derived from paraxial mesoderm, but how myoblasts acquire their identity is still a matter of speculation. The characterization of molecular markers and, in some cases, the analysis of mutations in the corresponding genes, has now made it possible to ask specific questions about this process. Specification of somite cell fate depends on epigenetic factors. Adjacent tissues, such as the neural tube, notochord, dorsal ectoderm and lateral mesoderm, act either positively or negatively on the different myogenic precursor populations in the somite. Candidate molecules for this complex signalling activity include sonic hedgehog and the Wnt proteins as positive signals, and BMP4 as a possible inhibitor. Although it is generally assumed that induction is required, some observations suggest that embryonic cells might have a tendency to undergo myogenesis as a 'default' pathway. By analogy with Drosophila, where the neurogenic genes affect myogenesis, the vertebrate homologues of notch and its ligands could be candidate molecules for a repression or derepression mechanism. Similar studies with cultured muscle cells also implicate other HLH factors as potential inhibitors of the MyoD family and, hence, of inappropriate myogenesis.\", \n",
    "\"Overexpression of chimeric transgenes in plants can trigger post-transcriptional gene silencing that is dependent on epigenetic information and physiological conditions. The current view is that unproductive RNA serves as a crucial signal for gene silencing, although direct evidence is lacking for this theory. A signalling cascade then leads to strongly enhanced turnover of all RNAs that share a critical degree of sequence similarity. The molecular details of the mechanism are, however, insufficiently understood to explain the phenomenon completely and to comprehend its biological significance.\", \n",
    "\" \", \n",
    "\"Photocarcinogenesis represents the sum of a complex of simultaneous and sequential biochemical events that ultimately lead to the occurrence of skin cancer. These events, initiated by UV radiation of appropriate wavelength, include the formation of DNA photoproducts: DNA repair; mutation of proto-oncogenes and tumor suppressor genes; UV-production of radical species with subsequent effects on mutation and extra-nuclear function; and other epigenetic events that influence the course of carcinogenesis. The epigenetic influences may include immunological responses, antioxidant defenses, and dietary factors, This review represents an effort to provide current research results in the aforementioned areas and an attempt to meld these events into a comprehensive overview of photocarcinogenesis. If effective prevention and intervention strategies for skin cancer are to be developed, a more thorough understanding of the disease process is imperative. (C) 1997 Elsevier Science S.A.\", \n",
    "\" \", \n",
    "\"Several proteins that contribute to epigenetic mechanisms of gene regulation contain a characteristic motif of unknown function called the SET (Suvar3-9, Enhancer-of-zeste, Trithorax) domain. We have demonstrated that SET domains mediate highly conserved interactions with a specific family of proteins that display similarity with dual-specificity phosphatases (dsPTPases). These include myotubularin, the gene of which is mutated in a subset of patients with X-linked myotubular myopathy, and Sbf1, a newly isolated homologue of myotubularin. In contrast with myotubularin, Sbf1 lacks a functional catalytic domain which dephosphorylates phospho-tyrosine and serine-containing peptides in vitro. Competitive interference of endogenous SET domain-dsPTPase interactions by forced expression of Sbf1 induced oncogenic transformation of NIH 3T3 fibroblasts and impaired the in vitro differentiation of C2 myoblast cells. We conclude that myotubularin-type phosphatases link SET-domain containing components of the epigenetic regulatory machinery with signalling pathways involved in growth and differentiation.\", \n",
    "\"O-6-Methylguanine DNA methyltransferase (MGMT) repairs the mutagenic and cytotoxic O-6-alkylguanine lesions produced bg environmental carcinogens and the chemotherapeutic nitrosoureas, As such, MGMT-mediated repair of O-6-alkylguanine lesions constitutes a major form of resistance to nitrosourea chemotherapy and makes control of MGMT expression of clinical interest, The variability of expression in cell lines and tissues, along with the ease with which the MGMT phenotype reverts under various conditions, suggests that MGMT is under epigenetic control, One such epigenetic mechanism, 5-methylation of cytosines, has been linked to MGMT expression, We have used an isogenic human multiple myeloma tumor cell line model composed of an MGMT-positive parent cell line, RPMI 8226/S, and its MGMT-negative variant, termed 8226/V, to study the control of MGMT expression. The loss of MGMT activity in 8226/V was found to he due to the loss of detectable MGMT gene expression, Bisulfite sequencing of the MGMT CpG island promoter revealed large increases in the levels of CpG methylation within discrete regions of the 8226/V MGMT CpG island compared to those in 8226/S, These changes in CpG methylation are associated with local heterochromatinization of the 8226/V MGMT transcription start site and provide a likely mechanism for the loss of MGMT transcription in 8226/V.\", \n",
    "\"Inheritance of the active and inactive states of gene expression by individual cells is crucial for development. In fission yeast, mating-type region consists of three loci called mat1, mat2, and mat3. Transcriptionally silent mat2 and mat3 loci are separated by a 15 kb interval, designated the K-region, and serve as donors of information for transcriptionally active mat1 interconversion. In a strain carrying replacement of 7.5 kb of the K-region with the ura4 gene, we discovered that ura4 silencing and efficiency of mating-type switching were covariegated and were regulated by an epigenetic mechanism. Genetic analyses demonstrated that epigenetic states were remarkably stable not only in mitosis but also in meiosis and were linked to the mating-type region. This study indicates that different epigenetic states are heritable forms of chromatin organization at the mat region.\", \n",
    "\"Microdeletions of a region termed the imprinting center (IC) in chromosome 15q11-q13 have been identified in several families with Prader-Willi syndrome (PWS) or Angelman syndrome who show epigenetic inheritance for this region that is consistent with a mutation in the imprinting process. The IC controls resetting of parental imprints in 15q11-q13 during gametogenesis. We have identified a larger series of cases of familial PWS, including one case with a deletion of only 7.5 kb, that narrows the PWS critical region to <4.3 kb spanning the SNRPN gene CpG island and exon 1. Identification of a strong DNase I hypersensitive site, specific for the paternal allele, and six evolutionarily conserved (human-mouse) sequences that are potential transcription-factor binding sites is consistent with this region defining the SNRPN gene promoter. These findings suggest that promoter elements at SNRPN play a key role in the initiation of imprint switching during spermatogenesis. We also identified three patients with sporadic PWS who have an imprinting mutation (IM) and no detectable mutation in the IC. An inherited 15q11-q13 mutation or a trans-factor gene mutation are unlikely; thus, the disease in these patients may arise from a developmental or stochastic failure to switch the maternal-to-paternal imprint during parental spermatogenesis. These studies allow a better understanding of a novel mechanism of human disease, since the epigenetic effect of an IM in the parental germ line determines the phenotypic effect in the patient.\", \n",
    "\"The Wassilewskija strain of Arabidopsis has four genes encoding the tryptophan enzyme phosphoribosylanthranilate isomerase (PAI) located at three unlinked sites, These four PAI genes are methylated over their regions of DNA homology, When PAI copy number is reduced by deletion of two tandemly arrayed genes (MePAI1-PAI4), a mutant with fluorescent, tryptophan-deficient phenotypes results, because the two remaining methylated PAI genes (MePAI2 and MePAI3) supply insufficient PAI activity, These two methylated genes can be inherited through meiosis, even when they are segregated away from each other in crosses to a strain with unmethylated PAI genes, However, the mutant phenotypes conferred by the methylated PAI genes are unstable, and mutant plants yield occasional revertant somatic sectors and progeny. Revertant lines display coordinately reduced methylation of both PAI2 and PAI3, implying that this hypomethylation acts in a concerted manner across the genome rather than at individual sites.\", \n",
    "\"We report that CpG island methylation, an epigenetic modification of DNA known to correlate closely with silencing of gene transcription, appears in the oestrogen receptor (ER) gene in a subpopulation of cells which increases as a direct function of age in human colonic mucosa. This same methylation change characterizes virtually all cells in art 45 colorectal tumours examined, including the earliest stages of tumour formation. ER gene expression is diminished or absent in colorectal tumours, and introduction of an exogenous ER gene in cultured colon carcinoma cells resulted in marked growth suppression. Our data suggest that methylation associated inactivation of the ER gene in ageing cotorectal mucosa could be one of the earliest events that predispose to sporadic cotorectal tumorigenesis.\", \n",
    "\"A very large oxidative damage rate to DNA occurs as part of normal metabolism. In each rat cell the steady-state level is estimated to be about 10(6) oxidative adducts and about 10(5) new adducts are formed daily. It is argued that this endogenous DNA damage is a major contributor to aging and the degenerative diseases of aging, such as cancer. The oxidative damage rate in mammalian species with a high metabolic rate, short life span, and high age-specific cancer rate is much higher than the rate in humans, a long-lived creature with a lower metabolic rate and a lower age-specific cancer rate. It is argued that deficiency of micronutrients, such as dietary antioxidants or folate, is a major contributor to human cancer and degenerative diseases. Understanding the role of mitogenesis in mutagenesis is critical for clarifying the mechanisms of carcinogenesis and interpreting high-dose animal cancer tests. High-dose animal cancer tests have been done mainly on synthetic industrial chemicals, yet almost all of the chemicals humans are exposed to are natural. About half of natural chemicals tested in high-dose animal cancer tests are rodent carcinogens, a finding that is consistent with the view that high-dose tests frequently increase mitogenesis rates. Animals have numerous defenses against toxins that make them very well buffered against low doses of almost all toxins, whether synthetic or natural.\", \n",
    "\"The expression of the H19 gene is governed by parental imprinting in mammals. H19, an unusual gene encoding an RNA with no known function, is exclusively expressed from the maternal chromosome. In mouse, it lies 90 kb downstream from the Igf2 gene, which encodes a fetal-specific growth factor, insulin-like growth factor II, and is expressed primarily from the paternally inherited chromosome. In this report we have utilized interspecific hybrid mice to identify male-specific DNA methylation of a 7- to 9-kb domain surrounding the H19 gene and its promoter. This allele-specific methylation could function as a mark to suppress transcription of the H19 paternal allele. Consistent with this proposal, the H19 promoter displayed an open chromatin conformation only on the relatively unmethylated active maternal allele. In contrast, a cell type-specific enhancer that lies outside the methylation domain is hypersensitive to restriction enzyme digestion in nuclei on both maternal and paternal chromosomes. That the allele-specific methylation domain, coupled to the two H19 enhancers, contains all the information necessary for its imprinting was tested by examining two transgenic lines containing an internally deleted H19 transgene. Both displayed paternal-specific methylation of the transgene and maternal-specific expression. Although neither line has been tested in an inbred genetic background, and therefore the action of complex modifiers cannot be formally excluded, the result suggests that the sequences necessary for the imprinting of H19 have been identified.\", \n",
    "\"The insulin-like growth factor-II (IGF2) and H19 genes are imprinted in mouse and human, with expression of the paternal IGF2 and maternal H19 alleles. IGF2 undergoes loss of imprinting (LOI) in most Wilms' tumours (WT). We now show that: (i) LOI of IGF2 is associated with a 80-fold down regulation of H19 expression; (ii) these changes are associated with alterations in parental-origin-specific, tissue-independent sites of DNA methylation in the H19 promoter; and (iii) loss of heterozygosity is also associated with loss of H19 expression. Thus, imprinting of a large domain of the maternal chromosome results in a reversal to a paternal epigenotype. These data also suggest an epigenetic mechanism for inactivation of H19 as a tumour suppressor gene.\", \n",
    "\"The recent discovery of widely distributed targeting determinants, which govern the polarized cell-surface distribution of plasma membrane proteins in epithelial cells, has significantly changed our view of how polarized cells generate functionally distinct membrane domains. Together with the surprising finding that the same determinants are recognized on both the biosynthetic and the endocytic pathways, it now appears likely that a common epigenetic code may exist that controls molecular sorting of membrane proteins in a wide variety of polarized, and perhaps even non-polarized, cell types.\", \n",
    "\"Inactivation of the retinoblastoma gene appears to have a fundamental role in the genesis of retinoblastoma, osteosarcoma, and other malignant tumors. The gene is generally inactivated because of loss-of-function mutations, although epigenetic phenomena, such as hypermethylation of the promoter region, could possibly have the same effect. We investigated the methylation pattern at the 5' end of the retinoblastoma gene, including its promoter region and exon 1, in DNA purified from 56 primary retinoblastomas. We found five tumors with evidence for hypermethylation, all from unilateral, simplex patients. No methylation abnormalities were detected in DNA purified from the leukocytes from these patients. It is interesting that in one of these tumors the hypermethylation was confined to one allele. There were no mutations in a 1,306-bp sequence including the hypermethylated region that might account for the allele-specific hypermethylation. We believe that the hypermethylation of the retinoblastoma gene that we found in these tumors corresponds to the allelic inactivation of the gene, and we speculate that erroneous hypermethylation without alteration of nucleotide sequence occasionally plays a role in the genesis of this cancer. If this is true, then retinoblastomas with hypermethylation might be treatable with chemotherapeutic agents that interfere with methylation of DNA.\", \n",
    "\"Becker muscular dystrophy (BMD) often results from in-frame mutations of the dystrophin gene that allow production of an altered but partially functional protein. To address potential structure-function relationships for the various domains of dystrophin, we examined both the dystrophin gene and protein in 68 patients with abnormal dystrophin. Eighty-six percent of BMD patients with dystrophin of altered size have deletions or duplications, and the observed sizes of dystrophin fit well with predictions based on DNA data. Deletions within the amino-terminal domain I tended to result in low levels of dystrophin and a more severe phenotype. The phenotypes of patients with deletions or duplications in the central rod domain were more variable. This region can be divided into three portions based on differences in clinical presentations of patients. Deletions around exons 4553 were most common and generally caused typical BMD; however, phenotypic variability among patients with similar mutations suggests that epigenetic and/or environmental factors play an important role in determining the clinical progression. In contrast, deletions or duplications in the proximal portion of this domain tended to cause severe cramps and myalgia. Finally, loss of the middle of this region probably causes a very mild phenotype, as only one such patient was found and his only symptom was elevated serum creatine phosphokinase levels.\", \n",
    "\"Massive cell migration, proliferation, phenotypic differentiation, and enhanced biosynthetic activities characterize the sites of wound healing and fibrosis. Regulation of cellular functions by extracellular matrix, which consists of a dynamic assemblage of a variety of interacting molecules capable of reorganization in response to endogenous and exogenous stimuli, represents a fundamental epigenetic mechanism regulating cellular behavior and phenotype. Interactions of the individual components of extracellular matrix with specific cell surface molecules, integrin receptors, and proteoglycans initiate a cascade of signal transduction leading to varied short-term or persistent cellular responses. Extracellular matrix also serves as an important reservoir of cytokines and growth factors, thus modulating the action of a host of potent biological response modifiers by their selective, local accumulation and release. Currently known mechanisms by which extracellular matrix modulates different facets of the process of tissue remodeling after injury, which culminate either in normal wound repair or fibrosis, are discussed.\", \n",
    "\"To test the potential role of H19 as a tumour suppressor gene we have examined its expression and DNA methylation in Wilms' tumours (WTs). In most WTs (18/25), H19 RNA was reduced at least 20-fold from fetal kidney levels. Of the expression-negative tumours ten retained 11p15.5 heterozygosity: in nine of these, H19 DNA was biallelically hypermethylated and in two cases hypermethylation locally restricted to H19 sequences was also present in the non-neoplastic kidney parenchyma. IGF2 mRNA was expressed in most but not all WTs and expression patterns were consistent with IGF2/H19 enhancer competition without obligate inverse coupling. These observations implicate genetic and epigenetic inactivation of H19 in Wilms' tumorigenesis.\", \n",
    "\"Segmentation genes provide the signals for the activation and regulation of homeotic genes in Drosophila but cannot maintain the resulting pattern of expression because their activity ceases halfway through embryogenesis. Maintenance of the pattern is due to the Polycomb group of genes (Pc-G) and the trithorax group of genes (trx-G), responsible for the persistence of the active or repressed state of homeotic genes. We have identified a regulatory element in the Ubx gene that responds to Pc-G and trx-G genes. Transposons carrying this element create new binding sites for Pc-G products in the polytene chromosomes. This Pc-G maintenance element (PRE), establishes a repressive complex that keeps enhancers repressed in cells in which they were originally repressed and maintains this state through many cell divisions. The trx-G products stimulate the expression of enhancers in cells in which they were originally active. This mechanism is responsible for the correct regulation of imaginal disc enhancers, which lack themselves antero-posterior positional information. The PRE also causes severe variegation of the mini-white gene present in the transposon, a phenomenon very similar to heterochromatic position-effect variegation. The significance of this mechanism for homeotic gene regulation is discussed.\", \n",
    "\"Chromatin structure at Schizosaccharomyces pombe centromeres is unusual. The insertion of the ura4 gene within these centromeres resulted in genetically identical cells mosaic for its expression. Placement of the ade6 gene within cen1 or cen3 resulted in red-white sectored colonies, demonstrating the instability of gene expression. The occurrence of pink colonies implied that intermediate levels of repression were established. Repression of both genes within centromeres was temperature sensitive. The chromatin structure of the ura4 gene at centromeres was altered, suggesting that the unusual chromatin encroaches into the gene and inhibits normal expression. These repressive effects at S. pombe centromeres resemble the classical phenomenon of position effect variegation imposed by Drosophila heterochromatin on nearby genes. However, since the epigenetic states can be set at intermediate levels of expression, a purely euchromatin-heterochromatin dichotomy does not apply. A model for the epigenetic regulation of genes placed within S. pombe centromeres is presented.\", \n",
    "\"The mouse insulin-like growth factor II (Igf2) gene, which is located on distal chromosome 7 (Chr7), has been shown previously to undergo tissue-specific parental imprinting. This imprinting results in expression of Igf2 from the paternally inherited chromosome and repression of the maternally inherited allele in most tissues of the developing embryo. We are using embryos with the maternal duplication and paternal deficiency of distal Chr7 to characterize the mechanism that underlies repression of the maternal allele. We show that the chromatin of the 5' region of the repressed Igf2 allele is potentially active for transcription rather than heterochromatic. In particular, a CpG island that comprises one of the two strong promoters is unmethylated at both parental alleles, and DNase I hypersensitive sites in and around the strong promoters are consistently present on both parental chromosomes. In agreement with the chromatin state, primary transcripts from the maternal Igf2 allele have been detected at low but significant levels. These findings differ from observations in other instances of imprinting, namely, X-chromosome inactivation and transgene imprinting in mice. Although no parent-specific differences were detected in either DNA methylation or sensitivity to nucleases at these promoters, we have observed parental methylation differences in a region several kilobases upstream of the first exon. The differential activity of the parental Igf2 alleles could be achieved through epigenetic modifications situated outside the promoters or by subtle and yet unidentified modifications at the promoters.\", \n",
    "\"The Insulin-like growth factor 2 (Igf2) and H19 genes are reciprocally imprinted and closely linked. Igf2 encodes a fetal growth-factor and is predominantly expressed from the paternal allele, while H19 is expressed from the maternal allele and encodes a transcript which may downregulate cellular proliferation. One of the epigenetic modifications thought to be involved in parental imprinting is DNA methylation. Here we analyse methylation in two regions of the Igf2 gene, one approx. 3 kb upstream of the gene and one in the 3' part of the gene. Both regions are more methylated on the expressed paternal chromosome. Genomic sequencing of individual chromosomes in the first region shows this parent-specific methylation to be highly mosaic; interestingly, individual sperm chromosomes carry different methylation patterns into the egg. In the more 3' region, which is fully methylated in sperm, the level of methylation on the paternal allele is highly tissue-specific and is correlated with expression of the gene in fetal tissues. Hence, the paternal allele is highly methylated in fetal liver (high expression) but is undermethylated in fetal brain (virtually no expression). Adult choroid plexus, a brain tissue in which Igf2 is expressed from both alleles and H19 is not expressed, represents an apparent loss of imprinting. Here, both Igf2 and H19 adopt a paternal type methylation pattern on both parental chromosomes. Analysis of early-passage androgenetic and parthenogenetic embryonic stem (ES) cells shows that the methylation patterns of Igf2 and H19 on maternal and paternal chromosomes are very similar. Androgenetic and parthenogenetic teratomas derived from these ES cells show the appropriate paternal and maternal patterns, respectively, of allelic methylation in both genes. Our results suggest that allelic methylation patterns in Igf2 and H19 arise early in embryogenesis and change progressively during development. Some of these developmental changes are apparently under tissue-specific control.\", \n",
    "\"In several plant systems expression of structurally intact genes may be silenced epigenetically when a transgenic construct increases the copy number of DNA sequences. Here we report epigenetic silencing in Arabidopsis lines containing transgenic inserts of defined genetic structure, all at the same genomic locus. These comprise an allelic series that includes a single copy of the primary insert, which carries repeated drug resistance transgenes, and a set of its derivatives, which as a result of recombination within the insert carry different numbers and alleles of resistance genes. Although the drug resistance genes remained intact, both the primary and some recombinant lines nevertheless segregated many progeny that were partly or fully drug-sensitive because of silencing. As in other systems silencing was reversible, and correlated with decreased steady-state mRNA and increased DNA methylation. Each different number and combination of genes, on the same or different (i.e., homologous) chromosomes, conditioned its own idiosyncratic segregation pattern. Strikingly, lines with a single gene segregated only a few slightly drug-sensitive progeny whereas multi-gene lines segregated many highly sensitive progeny, indicating dependence of silencing at this locus on repeated sequences. This argues strongly against explanations based on antisense RNA, but is consistent with explanations based on ectopic DNA pairing. One possibility is that silencing reflects the interaction of paired homologous DNA with flanking heterologous DNA, which induces condensation of chromatin into a non-transcribable state.\", \n",
    "\"Modifiers of position-effect-variegation in Drosophila encode proteins that are thought to modify chromatin, rendering it heritably changed in its expressibility. In an attempt to identify similar modifier genes in other species we have utilized a known sequence homology, termed chromo box, between a suppressor of position-effect-variegation, Heterochromatin protein 1 (HP1), and a repressor of homeotic genes, Polycomb (Pc). A PCR generated probe encompassing the HP1 chromo box was used to clone full-length murine cDNAs that contain conserved chromo box motifs. Sequence comparisons, in situ hybridization experiments, and RNA Northern blot analysis suggest that the murine and human sequences presented in this report are homologues of the Drosophila HP1 gene. Chromo box sequences can also be detected in other animal species, and in plants, predicting a strongly conserved structural role for the peptide encoded by this sequence. We propose that epigenetic (yet heritable) changes in gene expressibility, characteristic of chromosomal imprinting phenomena, can largely be explained by the action of such modifier genes. The evolutionary conservation of the chromo box motif now enables the isolation and study of putative modifier genes in those animal and plant species where chromosomal imprinting has been described.\", \n",
    "\" \", \n",
    "\" \", \n",
    "\"The molecular mechanisms introducing epigenetic modifications that lead to differential silencing of some autosomal alleles depending on their parental legacy are still largely unknown, but recent results from studies of endogenously imprinted genes and particular transgenes make DNA methylation a strong candidate. At the same time, these results have raised new questions about the details of the imprinting process.\", \n",
    "\"Previous work has shown that two homologous, unlinked transgene loci can interact in plant nuclei, leading to non-reciprocal trans-inactivation and methylation of genes at one locus. Here, we report the structure and methylation of different transgene loci that contain the same construct but are variably able to inactivate and methylate a partially homologous, unlinked target locus. Silencing loci comprised multiple, methylated copies of the transgene construct, whereas a non-silencing locus contained a single, unmethylated copy. The correspondence between strength of silencing activity and copy number/degree of methylation was further demonstrated by producing novel alleles of a strong silencing locus: reducing the transgene copy number and methylation within this silencing locus decreased its ability to inactivate the target locus. The strong silencing locus, which was located close to a telomere, trans-inactivated various structural variants of the original target construct, regardless of their location in the genome. This suggests that the silencing locus can scan the entire genome for homologous regions, a process possibly aided by its telomeric location. Our data support the idea that epistatic trans-inactivation of unlinked, homologous transgenes in plants results from a pre-existing epigenetic difference between transgene loci, which is subsequently equalized by ''epigene conversion'' involving DNA-DNA pairing.\", \n",
    "\" \", \n",
    "\"Major epigenetic modifications apparently occur during early development in the mouse. The factors that induce such modifications are complex and may involve the various components of a zygote. We have started to explore whether changes in the nucleocytoplasmic composition brought about by micromanipulation can induce phenotypic effects through epigenetic modifications. Nucleocytoplasmic hybrids were therefore prepared by transplanting a female pronucleus into a recipient egg from a different genotype. As a result, the maternal genome was of a different genetic background as compared with the egg cytoplasm. Specifically, experimental zygotes had cytoplasm from the inbred strain C57BL/6, a maternal genome from DBA/2, and a paternal genome from C57BL/6 (termed BDB hybrids). The mirror-image combination, termed DBD, was also made. The reconstituted zygotes were transferred to recipients and allowed to develop to term. Mice born from manipulated zygotes showed transcriptional repression and DNA methylation of major urinary protein genes in their liver, as well as growth deficiency resulting in reduced adult body weight. No altered phenotype was observed in controls in which the maternal pronucleus was simply transplanted back into another zygote of the same genetic background. These results clearly demonstrate phenotypic as well as molecular effects on DNA methylation and expression of at least one gene. Phenotype was therefore no longer predicted by genotype as a result of epigenetic modifications in experimental embryos. What precisely triggers the phenotypic and epigenetic changes is unknown, but presumably, nucleocytoplasmic interactions in hybrid zygotes may be partly responsible.\", \n",
    "\"The DNA of eukaryotic cells is organized in a complex with proteins, either as interphase chromatin or mitotic chromosomes. Nucleosomes, the structural subunits of chromatin, have long been considered as static structures, incompatible with processes occurring in chromatin. During the past few years it has become evident that the histone part of the nucleosome has important regulatory functions. Some of these functions are mediated by the N-terminal core histone domains which contain sites for posttranslational modifications, among them lysine residues for reversible acetylation. Recent results indicate that acetylation and deacetylation of N-terminal lysines of nucleosomal core histones represent a means of molecular communication between chromatin and the cellular signal transduction network, resulting in heritable epigenetic information. Data on enzymes involved in acetylation and the pattern of acetylated lysine sites on chromosomes, as well as genetic data on yeast transcriptional repression, suggest that acetylation may lead to structural transitions as well as specific signalling within distinct chromatin domains.\", \n",
    "\" \", \n",
    "\"Messenger RNA and methylation levels of four imprinted genes, H19, Igf2r, Igf-2 and Snrpn were examined by northern and Southern blotting in mouse parthenogenetic, androgenetic and normal or wild-type embryonic stem cell lines during their differentiation in vitro as embryoid bodies. Zn most instances, mRNA levels in parthenogenetic and androgenetic embryoid bodies differed from wild type as expected from previously determined patterns of monoallelic expression in midgestation embryos and at later stages of development. These findings implicate aberrant mRNA levels of these genes in the abnormal development of parthenogenetic and androgenetic embryos and chimeras. Whereas complete silence of one of the parental alleles has previously been observed in vivo, we detected some mRNA in the corresponding embryonic stem cell line. This 'leakage' phenomenon could be explained by partial erasure, bypass or override of imprints, or could represent the actual activity status at very early stages of development. The mRNA levels of H19, Igf2r and Igf-2 and the degree of methylation at specific associated sequences were correlated according to previous studies in embryos, and thereby are consistent with suggestions that the methylation might play a role in controlling transcription of these genes. Paternal-specific methylation of the H19 promoter region is absent in sperm, yet we observed its presence in undifferentiated androgenetic embryonic stem cells, or before the potential expression phase of this gene in embryoid bodies. As such methylation is likely to invoke a repressive effect, this finding raises the possibility that it is part of the imprinting mechanism of H19, taking the form of a secondary imprint or postfertilization epigenetic modification necessary for repression of the paternal allele.\", \n",
    "\"The SNRPN gene encodes a small nuclear ribonucleoprotein subunit, SmN, thought to be involved in splicing of pre-mRNA. A closely related protein, SmB/B', is constitutively expressed in all tissues except the brain, where SmN is predominantly expressed. The mouse homolog of the SNRPN gene has been shown to be functionally imprinted in mouse brain, being expressed only from the paternally derived chromosome. SNRPN has been mapped to human chromosome 15q11-q13 within the shortest region of deletion overlap for the Prader-Willi syndrome. We have now demonstrated functional imprinting of the human SNRPN gene using reverse transcription followed by the polymerase chain reaction (RT-PCR). No expression was observed in cultured skin fibroblasts of Prader-Willi patients, but was found in all Angelman patients and normal controls examined. We have also demonstrated a parent-specific DNA methylation imprint within intron 5 of the SNRPN gene, which suggests an epigenetic mechanism by which parent-specific expression of this gene might be inherited. Our findings indicate that SNRPN is expressed only from the paternally derived chromosome 15 in humans and therefore may fullfill one major criterion for being involved in the pathogenesis of the Prader-Willi syndrome.\", \n",
    "\"Development in mammals is influenced by genome imprinting which results in differences in the expression of some homologous maternal and paternal alleles. This process, initiated in the germline, can continue following fertilization with interactions between oocyte cytoplasmic factors and the parental genomes involving modifier genes. Further epigenetic modifications may follow to render the 'imprints' heritable through subsequent cell divisions during development. Imprinting of genes can be critical for their dosage affecting embryonic growth, cell proliferation and differentiation. The cumulative effects of all the imprinted genes are observed in androgenones (AG) and parthenogenones (PG), which reveal complementary phenotypes with respect to embryonic and extraembryonic tissues. The presence of PG cells in chimeras causes growth retardation, while that of AG cells enhanced growth. AG cells apparently have a higher cell proliferation rate and, unlike PG cells, are less prone to selective elimination. However, the PG germ cells are exempt from cell selection. In chimeras, PG cells are more likely to be found in ectodermal derivatives such as epidermis and brain in contrast to AG cells which make pronounced contributions to many mesodermal derivatives such as muscle, kidney, dermis and skeleton. The presence of androgenetic cells in chimeras also results in the disproportionate elongation of the anterior-posterior axis and sometimes in the abnormal development of skeletal elements along the axis. Genetic studies highlight the influence of subsets of imprinted genes, and identify those that are critical for development.\", \n",
    "\"Zea mays transformants produced by particle bombardment of embryogenic suspension culture cells of the genotype A188 x B73 and selected on kanamycin or bialaphos were characterized with respect to transgene integration, expression, and inheritance. Selection on bialaphos, mediated by the bar or par genes, was more efficient than selection on kanamycin, mediated by the nptII gene. Most transformants contained multicopy, single locus, transgene insertion events. A transgene expression cassette was more likely to be rearranged if expression of that gene was not selected for during callus growth. Not all plants regenerated from calli representing single transformation events expressed the transgenes, and a nonselectable gene (uidA) was expressed in fewer plants than was the selectable transgene. Mendelian inheritance of transgenes consistent with transgene insertion at a single locus was observed for approximately two thirds of the transformants assessed. Transgene expression was typically, but not always, predictable in progeny plants - transgene silencing, as well as poor transgene transmission to progeny, was observed in some plant lines in which the parent plants had expressed the transgene.\", \n",
    "\"Although Angelman (AS) and Prader-Willi (PWS) syndromes are human genetic disorders with distinctly different developmental and neurobehavioural phenotypes, they both have abnormalities in inheritance of chromosome 15q11-q13. Whether AS or PWS arises depends on the parental origin of a deletion or uniparental disomy (the inheritance of 2 copies of a genetic locus from only one parent) for 15q11-q13. Normal development requires a genetic contribution for this genetic region from both a male and a female parent. The dependence on parental origin implies that genes in human 15q11-q13 have distinct functions depending upon epigenetic, parent-of-origin differences, known as genomic imprinting. Here, I review the role of uniparental disomy and genomic imprinting in the pathogenesis of AS and PWS, and briefly discuss phenotype-genotype correlations using candidate genes and mouse models, in particular for hypopigmentation.\", \n",
    "\"Because development is epigenetic, diverse aspects of morphology are integrated during ontogeny. Using the method of thin-plate splines, and the decomposition of these splines by their principal warps, we examine the ontogeny of integrated features of skull growth of the cotton rat, Sigmodon fulviventer as observed in landmark locations in the ventral view. Postnatal growth of the skull in Sigmodon is not adequately described by the familiar contrast between relatively rapid facial elongation and slow, precocial growth of the cranial base. No developmental units corresponding to facial skull and cranial base emerge from analysis of geometric shape change. Rather, skull growth is both more integrated and more complex, exhibiting both skull-wide integration and locally individualized regions. Like skull shape, integration has an ontogeny; different regions of the skull can be partitioned into developmentally individualized parts in different ways at different ages. The effective count of individualized parts decreases substantially before weaning occurs, suggesting that the integration required by the functionally demanding activity of chewing gradually develops before the functional transition occurs. Our description of skull growth and integration does not depend upon arbitrary a priori choices about what to measure; rather, we base our decomposition of the whole into parts upon results of the data analysis. Our approach complicates the study of heterochrony, but, because it expresses the spatiotemporal organization of ontogeny, it enables the study of heterotopy.\", \n",
    "\"A novel epigenetic mechanism that can affect minichromosome centromere function in vivo has been identified in S. pombe. This epigenetic system can result in the conversion of a nonfunctional centromere to a functional one without changes in the content, structural arrangement, or chemical modification state of the minichromosomal DNA. The conversion from a centromere-inactive to an active state, which is evident with minichromosomes carrying abbreviated centromeric DNA constructions, occurs with a relatively high frequency during mitotic cell divisions and readily provides an in vivo essay for proper centromere formation. The centromere-targeted epigenetic system supports a model for centromere function that involves specific de novo folding of centromeric components into a higher order chromatin structure.\", \n",
    "\"During development, patterns of differential gene expression, defining determined states of cells, need to be maintained over many cell generations. In Drosophila, genetic and molecular analyses led to the discovery of a set of proteins which seem to exert such a memory function by using epigenetic mechanisms. Recent experiments demonstrate that, in particular, the heritable inactivation of regulatory genes relies on stable changes in the higher-order constitution of chromatin.\", \n",
    "\"The ability of chemicals with tumor-promoting or tumor-inhibiting activity to modulate gap junctional intercellular communication is reviewed. The two most extensively used types of assays for screening tests are (1) metabolic cooperation assays involving exchange between cells of precursors of nucleic acid synthesis and (2) dye-transfer assays that measure exchange of fluorescent dye from loaded cells to adjacent cells. About 300 substances of different biological activities have been studied using various assays. For tumor promoters/epigenetic carcinogens, metabolic cooperation assays have a sensitivity of 62% and dye-transfer assays 60%. Thirty percent of DNA-reactive carcinogens also possess the ability to uncouple cells. The complete estimation of the predictive power of these assays could not be made because the majority of the substances studied for intercellular communication effects in vitro have not yet been studied for promoting activity in vivo. Both metabolic cooperation assays and dye transfer assays respond well to the following classes of substances: phorbol esters, organochlorine pesticides, polybrominated biphenyls, promoters for urinary bladder, some biological toxins, peroxisome proliferators, and some complex mixtures. Results of in vitro assays for such tumor promoters/nongenotoxic carcinogens, such as some bile acids, some peroxides, alkanes, some hormones, mineral dusts, ascorbic acid, okadaic acid, and benz(e)pyrene, do not correlate with the data of in vivo two-stage or complete carcinogenesis. Enhancement of intercellular communication was found for 18 chemicals. Among these, cAMP, retinoids, and carotenoids have demonstrated inhibition of carcinogenesis. We examine a number of factors that are important for routine screening, including the requirement for biotransformation for some agents to exert effects on gap junctions. We also discuss the mechanisms of tumor promoter and tumor inhibitor effects on gap junctional permeability, including influences of protein kinase activation, changes in proton and Ca2+ intracellular concentrations, and effects of oxy radical production.\", \n",
    "\"Ascobolus immersus artificial gene repeats were shown previously to be subject premeiotically to both cytosine methylation and inactivation. We studied sexual progenies of strains harbouring two wild type copies of the endogenous met2 gene lying either in tandem array or at ectopic unlinked positions, by (i) investigating the methylation status, (ii) searching for mutations and (iii) analysing the inheritance of inactivation both in mitotic and sexual offspring. 100% of the 'tandem' progeny and 64% of the 'ectopic' progeny had methylated repeats and displayed gene inactivation. Similar methylation patterns involving all or most of the cytosine residues within the repeats were observed in both arrangements. The inactivated met2 copies were totally devoid of mutation, as deduced from: (i) extensive restriction site analysis and DNA sequencing; (ii) the finding that all the Met- derivatives tested reverted to prototrophy in selective conditions; and (iii) the finding that an inactivated copy of met2 stripped of its methylation through amplification in Escherichia coli regained activity when reintroduced in A. immersus. In the absence of selection, gene silencing and methylation were faithfully maintained through mitotic divisions and through five successive sexual cycles. Altogether, these data show the epimutational nature of this methylation induced premeiotically (MIP) process.\", \n",
    "\"Transposable elements from the Robertson's Mutator family are highly active insertional mutagens in maize. However, mutations caused by the insertion of responder (non-autonomous) elements frequently depend on the presence of active regulator (autonomous) elements for their phenotypic effects. The hcf106::Mu1 mutation has been previously shown to depend on Mu activity in this way. The dominant Lesion-mimic 28 mutation also requires Mu activity for its phenotypic effects. We have used double mutants to show that the loss of Mu activity results in the coordinate suppression of both mutant phenotypes. This loss can occur somatically resulting in large clones of cells that have a wild-type phenotype. Autonomous and non-autonomous Mutator elements within these clones are insensitive to digestion with methylation-sensitive enzymes, suggesting extensive methylation of CG and non-CC cytosine residues. Our data are consistent with the sectors being caused by the cycling of MuDR regulatory elements between active and inactive phases. The pattern of sectors suggests that they are clonal and that they are derived from the apical cells of the vegetative shoot meristem. We propose that these cells are more likely to undergo epigenetic loss of Mu activity because of their longer cell division cycle during shoot growth. Coordinate suppression of unlinked mutations can be used to perform mosaic analysis in maize.\", \n",
    "\"The neural crest is a fascinating structure of the vertebrate embryo; its ontogeny includes a transient period during which its component cells undergo an epithelio-mesenchymal transition and become migratory. This phase was shown recently to be controlled by the 'Slug' gene which belongs to the 'Snail' family of Drosophila transcription factors. After homing to specific sites in the embryo, the crest-derived cells produce a large variety of phenotypes. Recent advances have shown that during migration most crest cells exhibit various degrees of pluripotentiality, some being already committed to a single and definite fate. Moreover, several lines of evidence point to the existence of totipotent stem cells in the neural crest, the progeny of which become progressively diversified through a combination of intrinsic and extrinsic influences. The latter have been documented by the disruption of several neurotrophin genes, which results in severe deficiencies of selected subsets of neural crest derivatives. The neural crest has also been shown to play an important role in the development of the vertebrate head and hypobranchial region. The genetic control of this process depends on the. activity of developmental genes, among which the vertebrate Hox genes are essential, particularly at the rhombencephalic level.\", \n",
    "\"The ability of simian virus 40-encoded large T antigen to disrupt the growth control of a variety of cell types is related to its ability to interfere with certain cellular proteins, such as p53 and the retinoblastoma susceptibility gene product (pRB). We have used wild-type and mutant forms of T antigen in transgenic mice to dissect the roles of pRB, p53, and other cellular proteins in tumorigenesis of different cell types. In this study, using a cell-specific promoter to target expression specifically to brain epithelium (the choroid plexus) and to B and T lymphoid cells, we characterize the tumorigenic capacity of a T-antigen fragment that comprises only the amino-terminal 121 residues. This fragment (dl1137) retains the ability to interact with pRB and p107 but lacks the p53-binding domain. While loss of the p53-binding region results in loss of the capacity to induce lymphoid abnormalities, dl1137 retains the ability to induce choroid plexus tumors that are histologically indistinguishable from those induced by wild-type T antigen. Tumors induced by dl1137 develop much more slowly, however, reaching an end point at around 8 months of age rather than at 1 to 2 months. Analysis of tumor progression indicates that tumor induction by dl1137 does not require secondary genetic or epigenetic events. Rather, the tumor growth rate is significantly slowed, indicating that the T-antigen C-terminal region contributes to tumor progression in this cell type. In contrast, the pRB-binding region appears essential for tumorigenesis as mutation of residue 107, known to disrupt pRB and p107 binding to wild-type T antigen, abolishes the ability of the dl1137 protein to induce growth abnormalities in the brain.\", \n",
    "\"In this study, we used the ADE2 gene in a colony color assay to monitor transcription from the normally silent HMR mating-type locus in Saccharomyces cerevisiae. This sensitive assay reveals that some previously identified cis- and trans-acting mutations destabilize silencing, causing genetically identical cells to switch between repressed and derepressed transcriptional states. Deletion of the autonomously replicating sequence (ARS) consensus element at the HMR-E silencer or mutation of the silencer binding protein RAP1 (rap1s) results in the presence of large sectors within individual colonies of both repressed (Ade-, pink) and derepressed (Ade+, white) cells. These results suggest that both the ARS consensus element and the RA-P1 protein play a role in the establishment of repression at HMR. In diploid cells, the two copies of HMR appear to behave identically, suggesting that the switching event, though apparently stochastic, reflects some property of the cell rather than a specific event at each HMR locus. In the ADE2 assay system, silencing depends completely upon the function of the SIR genes, known trans-acting regulators of the silent loci, and is sensitive to the gene dosage of two SIR genes, SIR1 and SIR4. Using the ADE2 colony color assay in a genetic screen for suppressors of rap1s, silencer ARS element deletion double mutants, we have identified a large number of genes that may affect the establishment of repression at the HMR silent mating-type locus.\", \n",
    "\"We analysed de novo DNA methylation occurring in plants obtained from the transgenic petunia line R101-17. This line contains one copy of the maize Al gene that leads to the production of brick-red pelargonidin pigment in the flowers. Due to its integration into an unmethylated genomic region the A1 transgene is hypomethylated and transcriptionally active. Several epigenetic variants of line 17 were selected that exhibit characteristic and somatically stable pigmentation patterns, displaying fully coloured, marbled or colourless flowers. Analysis of the DNA methylation patterns revealed that the decrease in pigmentation among the epigenetic variants was correlated with an increase in methylation, specifically of the transgene DNA. No change in methylation of the hypomethylated integration region could be detected. A similar increase in methylation, specifically in the transgene region, was also observed among progeny of R101-17del, a deletion derivative of R101-17 that no longer produces pelargonidin pigments due to a deletion in the Al coding region. Again de novo methylation is specifically directed to the transgene, while the hypomethylated character of neighbouring regions is not affected. Possible mechanisms for transgene-specific methylation and its consequences for long-term use of transgenic material are discussed.\", \n",
    "\"Plant cells growing in an artificial culture environment make numerous genetic mistakes. These alterations are manifested as increased frequencies of single-gene mutations, chromosome breakages, transposable element activations, quantitative trait variations, and modifications of normal DNA methylation patterns. Evidence is presented that indicates a high frequency of DNA hypomethylation as the result of the tissue culture process. Fifteen percent of the methylation changes appear to have been homozygous in the original regenerated plants. A hypothesis is advanced that relates DNA methylation to the variety of genetic alterations found among maize tissue culture regenerants and their progenies. The epigenetic nature of DNA methylation raises questions concerning the stability of tissue culture-induced changes in self-pollinations and crosses.\", \n",
    "\"In this essay review I describe the relationship between plant organization, variation, and evolution. The hypothetical optimal design of an organism living in a given environment must be specified to test whether evolution is adaptive. In a variable environment this design should include a system for plasticity. Plasticity in plants is made possible by modular growth. In modular growth new parts can constantly be added to a body, resulting in a population-like structure of the organism. The developmental basis for modular growth is an open-ended ontogeny in which the formation of organs is not controlled by patterns of cell divisions and migrations and where there is no separation into germ-line and soma. Epigenetic pattern formation, late differentiation, and loose integration of the individual arising from a zygote (genet) are characteristic for plants. Because the phenotype is the result of an interaction between genotype and environment, the contributions of the two cannot be separated. Nevertheless, with appropriate experimental procedures it is possible to separate the variation among phenotypes into variation among genotypes, variation among environments, and covariation of genotypes and environments. Several genotypes must be grown in several environments. If the environments can be put on a scale, as in gradient experiments, reaction norms of genotypes may be quantified. Local specialization of genotypes in a heterogeneous environment may be recognized by transplant experiments. New results from both types of experiments indicate that the 'general-purpose genotype' is more common than originally thought and suggest that phenotypic plasticity in plants has relatively low costs. However, the critical experiments to measure these costs have not yet been published. The genetic analysis of phenotypic variation and the use of phenotypic characters for reconstructing phylogenies is hampered by the particular plant features mentioned above. Environment-induced phenotypic effects can be persistent and can be transmitted to clonal or even sexual progeny. Gene flow among taxa via sexual and possibly other mechanisms together with the loose integration of the individual make it difficult to recognize phylogenies for systematic groups in plants. Some of these problems are illustrated with species from the genera Solidago and Carex.\", \n",
    "\"Mating-type genes resident in the silent cassette HML at the left arm of chromosome III are repressed by the action of four SIR gene products, mediated independently through two cis-acting sites, termed the E and I silencers. We have found that in the absence of the I silencer, deletion of any one of three distinct elements within E yields partial derepression of the mating-type genes resident at HML, whereas deletion of any two yields full derepression. These elements correspond to a binding site for the abundant DNA-binding protein RAP1, an autonomous replicating sequence (ARS), and an as yet undistinguished region. From detailed deletion analysis of the E site we conclude that the ARS element contributes to silencer function in a capacity distinct from its role as an initiator of DNA replication. In addition, we find that strains deleted for any one of these elements comprise two genetically identical but phenotypically distinct types of cells: Those with HML apparently fully derepressed, and those with HML apparently completely repressed. These results reinforce the notion that epigenetic inheritance is an intrinsic characteristic of silencer action.\", \n",
    "\"Two instances of genetic transmission of spontaneous epimutation of the maize P-rr gene were identified. Transmission gave rise to two similar, moderately stable alleles, designated P-pr-1 and P-pr-2, that exhibited Mendelian behavior. Both isolates of P-pr conditioned a variable and variegated phenotype, unlike the uniform pigmentation conditioned by P-rr. Extensive genomic analysis failed to reveal insertions, deletions or restriction site polymorphisms between the new allele and its progenitor. However, methylation of the P gene was increased in P-pr relative to P-rr, and was greatly reduced (though not lost) in a revertant to uniform pigmentation. Variability in pigmentation conditioned by P-pr correlated with variability in transcript levels of the P gene, and both correlated inversely with variability in its methylation. Part of the variability in methylation could be accounted for by a developmental decrease in methylation in all tissues of plants carrying P-pr. We hypothesize that the variegated phenotype results from a general epigenetic pathway which causes a progressive decrease in methylation and increase in expression potential of the P gene as a function of cell divisions in each meristem of the plant. This renders all tissues chimeric for a functional gene; chimerism is visualized as variegation only in pericarp due to the tissue specificity of P gene expression. Therefore, this allele that originates from epimutation may exemplify an epigenetic mechanism for variegation in maize.\", \n",
    "\"Previous work has shown that two unlinked, partially homologous transgene loci can interact in plant nuclei, leading to reversible methylation and inactivation of one transgene locus in the presence of the second. To study whether the chromosomal location of a transgene influences its susceptibility to trans-inactivation, we retransformed four transgenic lines, which contained the same construct (H) integrated in different chromosomal locations, with a second, partially homologous construct (K). At least 50 double transformants (DTs) were regenerated from each single transformant (ST) and screened for inactivation of markers [chloramphenicol acetyltransferase (CAT); hygromycin resistance (HYG(R))] at the resident H locus. For two STs, H locus markers were inactivated in less than 1% of the DTs, suggesting that, at these integration sites, H was relatively resistant to trans-inactivation. In contrast, the other two STs appeared to be more sensitive to trans-inactivation: 4-10% of the DTs were CAT(-) and/or Hyg(s). Inactivation of H locus markers could be attributed to two distinct phenomena: 1. Regeneration from cells containing different epigenetic states of H, in which either both, one or none of the H alleles was active. This instability in the expression of the H locus, which was independent of K, was more pronounced in the homozygous state, and was associated with cellular mosaicism of expression and methylation. 2. The presence of an unlinked K locus could weaken the Hyg(R) phenotype by transcriptional inactivation and increased methylation of the hph gene at the H locus. These results indicated that a susceptible transgene locus is inherently unstable and partially methylated, and that these characteristics are exacerbated when the locus is homozygous for the transgene and/or when an unlinked homologous transgene is present.\", \n",
    "\"An illustrative list is presented of human and animal studies which each point to the existence of a third source, in addition to genetic and environmental factors, underlying phenotypic differences in development. It is argued that this third source may consist of nonlinear epigenetic processes that can create variability at all phenotypical-somatic and behavioral-levels. In a quantitative genetic analysis with human subjects, these processes are confounded with within-family environmental influences. A preliminary model to quantify these influences is introduced.\", \n",
    "\"Analysis of the developmental capacities of androgenetic and gynogenetic mouse embryos (bearing two paternal or two maternal pronuclei, respectively) revealed a defect in blastocyst formation of androgenetic, but not gynogenetic, embryos that was a function of the maternal genotype. Androgenetic embryos constructed using fertilized eggs from C57BL/6 or (B6D2)F1 mice developed to the blastocyst stage at frequencies similar to those previously reported, whereas androgenetic embryos constructed with fertilized eggs from DBA/2 mice developed poorly, the majority failing to progress beyond the 16-cell stage and unable to form a blastocoel-like cavity, regardless of whether the male pronuclei were of C57BL6 or DBA/2 origin. This impaired development was observed even in androgenetic embryos constructed by transplanting two male pronuclei from fertilized DBA/2 eggs to enucleated C57BL/6 eggs, indicating that the defect cannot be explained as the lack of some essential component in the DBA/2 cytoplasm that might otherwise compensate for androgeny. Rather, the DBA/2 egg cytoplasm apparently modifies the incoming male pronuclei differently than does C57BL/6 egg cytoplasm. Several specific alterations in the protein synthesis pattern of DBA/2 androgenones were observed that reflect a defect in the regulatory mechanisms that normally modulate the synthesis of these proteins between the 8-cell and blastocyst stages. These results are consistent with a model in which cytoplasmic factors present in the egg direct a strain-dependent modification of paternal genome function in response to epigenetic modifications (genomic imprinting) established during gametogenesis and indicate that preimplantation development can be affected by these modifications at both the morphological and biochemical levels.\", \n",
    "\"Functional consequences of either suppressing or intensifying spontaneous neuronal firing have been studied in developing rat cerebral cortex cultures using, respectively, tetrodotoxin (TTX) and picrotoxin (PTX) added chronically to the growth medium. Simple measures derived from the interspike interval histogram were able to powerfully discriminate between age and treatment groups. After return to control medium, most TTX-treated neurons spontaneously displayed stereotyped clustering of action potentials ('phasic' firing) which closely resembled the characteristic firing patterns seen acutely in the presence of PTX. The 'TTX-syndrome' thus suggests that GABAergic synaptic inhibition is ineffective in cortical networks grown under conditions which prevent the expression of bioelectric activity. In contrast, after return to control medium, neurons which had been partially disinhibited throughout development (by continuous exposure to PTX) had even less phasic firing than was measured in age-matched controls. Based upon these and previous findings, a two (main) factor model is put forth which can economically account for the major effects. The working hypothesis embodied in this model is that phasic neuronal discharges not only accelerate the maturation of excitatory connections within the neocortex but, even more important, are crucial for the development of adequate inhibitory synaptic transmission.\", \n",
    "\"Differences in the activity of maternally and paternally derived genomes in maize endosperm have been observed at three levels of genetic manipulation. When the balance of entire chromosome sets departs from the standard ratio of two of maternal origin to one of paternal origin, development is impaired, often leading to seed failure. At the level of individual chromosomes, absence of a paternal representative for 8 of the 19 chromosome arms tested causes a marked reduction in kernel size. Replacement of the missing arms by ones of maternal origin does not complement this defect. At the gene level, some alleles of R confer solid coloration on the aleurone layer when transmitted maternally but patchy coloration (mottled) when transmitted via pollen. In contrast with the endosperm, no effect of parentage on R phenotype has been detected in embryonic and seedling tissues. Furthermore, gynogenetic and androgenetic haploid plants are viable in maize and are similar in appearance. The detection of parental effects in the endosperm, but not the embryo, points to the few cell divisions of the gametophytes as a critical stage in imprinting. Chromosomally based epigenetic variation originating at this stage would be reflected as imprinting effects. A separate fertilization establishes a line of genetic descent in the embryo that appears to be relatively free of imprinted genes.\", \n",
    "\" \", \n",
    "\"Herein we describe a general multivariate quantitative genetic model that incorporates two potentially important developmental phenomena, maternal effects and epigenetic effects. Maternal and epigenetic effects are defined as partial regression coefficients and phenotypic variances are derived in terms of age-specific genetic and environmental variances. As a starting point, the traditional quantitative genetic model of additive gene effects and random environmental effects is cast in a developmental time framework. From this framework, we first extend a maternal effects model to include multiple developmental ages for the occurrence of maternal effects. An example of maternal effects occurring at multiple developmental ages is prenatal and postnatal maternal effects in mammals. Subsequently, a model of intrinsic and epigenetic effects in the absence of maternal effects is described. It is shown that genetic correlations can arise through epigenetic effects, and in the absence of other developmental effects, epigenetic effects are in general confounded with age-specific intrinsic genetic effects. Finally, the two effects are incorporated into the basic quantitative genetic model. For this more biologically realistic model combining maternal and epigenetic effects, it is shown that the phenotypic regressions of offspring on mother and offspring on father can be used in some cases to estimate simultaneously maternal effects and epigenetic effects.\", \n",
    "\"Butylated hydroxyanisole (BHA) is a synthetic food antioxidant used to prevent oils, fats and shortenings from oxidative deterioration and rancidity. This review depicts the current knowledge on BHA. The physical and chemical characteristics of BHA are summarized and its function as a food antioxidant is made clear. The toxicological characteristics of BHA and its metabolic fate in man and animal are briefly reviewed. Special emphasis is laid on the carcinogenicity of BHA in the forestomach of rodents and to related events in the forestomach and other tissues in experimental animals. At present there is sufficient evidence for carcinogenicity of BHA, but there is hardly any indication that BHA is genotoxic. Therefore risk assessment for this epigenetic carcinogen is based on non-stochastic principles. However, the mechanism underlying the tumorigenicity of BHA is not known. In the last part of this review an attempt is made to unravel the unknown mechanism of carcinogenicity. It is hypothesized that BHA gives rise to tumor formation in rodent forestomach by inducing heritable changes in DNA. Evidence is being provided that reactive oxygen species, in particular hydroxylradicals, may play a crucial role. The key question with respect to risk assessment for BHA is whether or not the underlying mechanism is thresholded, which is important for the choice of the appropriate model to assess the risk, if any, for man and to manage any potential risk.\", \n",
    "]\n",
    "\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 8,
   "metadata": {},
   "outputs": [],
   "source": [
    "###import pandas as pd\n",
    "\n",
    "###df = pd.read_excel (r'C:/Users/brady/Desktop/TM/Evo-Bio All Time/Test1.xlsx')\n",
    "###print (df)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 9,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "['The American College of Medical Genetics and Genomics (ACMG) previously developed guidance for the interpretation of sequence variants. 1 In the past decade, sequencing technology has evolved -rapidly with the advent of high-throughput next-generation -sequencing. By adopting and leveraging next-generation sequencing, clinical laboratories are now performing an ever-increasing catalogue of genetic testing spanning genotyping, single genes, gene panels, exomes, genomes, transcriptomes, and epigenetic assays for genetic disorders. By virtue of increased complexity, this shift in genetic testing has been accompanied by new challenges in sequence interpretation. In this context the ACMG convened a workgroup in 2013 comprising representatives from the ACMG, the Association for Molecular Pathology (AMP), and the College of American Pathologists to revisit and revise the standards and guidelines for the interpretation of sequence variants. The group consisted of clinical laboratory directors and clinicians. This report represents expert opinion of the workgroup with input from ACMG, AMP, and College of American Pathologists stakeholders. These recommendations primarily apply to the breadth of genetic tests used in clinical laboratories, including genotyping, single genes, panels, exomes, and genomes. This report recommends the use of specific standard terminology-pathogenic, likely pathogenic, uncertain significance, likely benign, and benign-to describe variants identified in genes that cause Mendelian disorders. Moreover, this recommendation describes a process for classifying variants into these five categories based on criteria using typical types of variant evidence (e.g., population data, computational data, functional data, segregation data). Because of the increased complexity of analysis and interpretation of clinical genetic testing described in this report, the ACMG strongly recommends that clinical molecular genetic testing should be performed in a Clinical Laboratory Improvement Amendments-approved laboratory, with results interpreted by a board-certified clinical molecular geneticist or molecular genetic pathologist or the equivalent.', 'There is substantial heterogeneity among primary prostate cancers, evident in the spectrum of molecular abnormalities and its variable clinical course. As part of The Cancer Genome Atlas (TCGA), we present a comprehensive molecular analysis of 333 primary prostate carcinomas. Our results revealed a molecular taxonomy in which 74% of these tumors fell into one of seven subtypes defined by specific gene fusions (ERG, ETV1/4, and FLI1) or mutations (SPOP, FOXA1, and IDH1). Epigenetic profiles showed substantial heterogeneity, including an IDH1 mutant subset with a methylator phenotype. Androgen receptor (AR) activity varied widely and in a subtype-specific manner, with SPOP and FOXA1 mutant tumors having the highest levels of AR-induced transcripts. 25% of the prostate cancers had a presumed actionable lesion in the PI3K or MAPK signaling pathways, and DNA repair genes were inactivated in 19%. Our analysis reveals molecular heterogeneity among primary prostate cancers, as well as potentially actionable molecular defects.', 'PD-L1 and PD-1 (PD) pathway blockade is a highly promising therapy and has elicited durable antitumor responses and long-term remissions in a subset of patients with a broad spectrum of cancers. How to improve, widen, and predict the clinical response to anti-PD therapy is a central theme in the field of cancer immunology and immunotherapy. Oncologic, immunologic, genetic, and biological studies focused on the human cancer microenvironment have yielded substantial insight into this issue. Here, we focus on tumor microenvironment and evaluate several potential therapeutic response markers including the PD-L1 and PD-1 expression pattern, genetic mutations within cancer cells and neoantigens, cancer epigenetics and effector T cell landscape, and microbiota. We further clarify the mechanisms of action of these markers and their roles in shaping, being shaped, and/or predicting therapeutic responses. We also discuss a variety of combinations with PD pathway blockade and their scientific rationales for cancer treatment.', 'Ensembl (ext-link-type=uri xlink:href=http://www.ensembl.org xlink:type=simple>http://www.ensembl.org) is a genomic interpretation system providing the most up-to-date annotations, querying tools and access methods for chordates and key model organisms. This year we released updated annotation (gene models, comparative genomics, regulatory regions and variation) on the new human assembly, GRCh38, although we continue to support researchers using the GRCh37.p13 assembly through a dedicated site (ext-link-type=uri xlink:href=http://grch37.ensembl.org xlink:type=simple>http://grch37.ensembl.org). Our Regulatory Build has been revamped to identify regulatory regions of interest and to efficiently highlight their activity across disparate epigenetic data sets. A number of new interfaces allow users to perform large-scale comparisons of their data against our annotations. The REST server (ext-link-type=uri xlink:href=http://rest.ensembl.org xlink:type=simple>http://rest.ensembl.org), which allows programs written in any language to query our databases, has moved to a full service alongside our upgraded website tools. Our online Variant Effect Predictor tool has been updated to process more variants and calculate summary statistics. Lastly, the WiggleTools package enables users to summarize large collections of data sets and view them as single tracks in Ensembl. The Ensembl code base itself is more accessible: it is now hosted on our GitHub organization page (ext-link-type=uri xlink:href=https://github.com/Ensembl xlink:type=simple>https://github.com/Ensembl) under an Apache 2.0 open source license.', \"Over the past 20 years, breakthrough discoveries of chromatin-modifying enzymes and associated mechanisms that alter chromatin in response to physiological or pathological signals have transformed our knowledge of epigenetics from a collection of curious biological phenomena to a functionally dissected research field. Here, we provide a personal perspective on the development of epigenetics, from its historical origins to what we define as 'the modern era of epigenetic research'. We primarily highlight key molecular mechanisms of and conceptual advances in epigenetic control that have changed our understanding of normal and perturbed development.\", \"Single-molecule, real-time sequencing developed by Pacific BioSciences offers longer read lengths than the second-generation sequencing (SGS) technologies, making it well-suited for unsolved problems in genome, transcriptome, and epigenetics research. The highly-contiguous de novo assemblies using PacBio sequencing can close gaps in current reference assemblies and characterize structural variation (SV) in personal genomes. With longer reads, we can sequence through extended repetitive regions and detect mutations, many of which are associated with diseases. Moreover, PacBio transcriptome sequencing is advantageous for the identification of gene isoforms and facilitates reliable discoveries of novel genes and novel isoforms of annotated genes, due to its ability to sequence full-length transcripts or fragments with significant lengths. Additionally, PacBio's sequencing technique provides information that is useful for the direct detection of base modifications, such as methylation. In addition to using PacBio sequencing alone, many hybrid sequencing strategies have been developed to make use of more accurate short reads in conjunction with PacBio long reads. In general, hybrid sequencing strategies are more affordable and scalable especially for small-size laboratories than using PacBio Sequencing alone. The advent of PacBio sequencing has made available much information that could not be obtained via SGS alone.\", 'MicroRNAs (miRNAs) are endogenous, small non-coding RNAs that function in regulation of gene expression. Compelling evidences have demonstrated that miRNA expression is dysregulated in human cancer through various mechanisms, including amplification or deletion of miRNA genes, abnormal transcriptional control of miRNAs, dysregulated epigenetic changes and defects in the miRNA biogenesis machinery. MiRNAs may function as either oncogenes or tumor suppressors under certain conditions. The dysregulated miRNAs have been shown to affect the hallmarks of cancer, including sustaining proliferative signaling, evading growth suppressors, resisting cell death, activating invasion and metastasis, and inducing angiogenesis. An increasing number of studies have identified miRNAs as potential biomarkers for human cancer diagnosis, prognosis and therapeutic targets or tools, which needs further investigation and validation. In this review, we focus on how miRNAs regulate the development of human tumors by acting as tumor suppressors or oncogenes.', \"'If G-quadruplexes form so readily in vitro, Nature will have found a way of using them in vivo' (Statement by Aaron Klug over 30 years ago). During the last decade, four-stranded helical structures called G-quadruplex (or G4) have emerged from being a structural curiosity observed in vitro, to being recognized as a possible nucleic acid based mechanism for regulating multiple biological processes in vivo. The sequencing of many genomes has revealed that they are rich in sequence motifs that have the potential to form G-quadruplexes and that their location is non-random, correlating with functionally important genomic regions. In this short review, we summarize recent evidence for the in vivo presence and function of DNA and RNA G-quadruplexes in various cellular pathways including DNA replication, gene expression and telomere maintenance. We also highlight remaining open questions that will have to be addressed in the future.\", \"The aberrant expression of transcription factor Basonuclin (BNC) had been reported in different kinds of tumors. Here, we investigated the expression and methylation status of two Basonuclin homologs, BNC1 and BNC2 in hepatocellular carcinoma (HCC). We found that the expression levels of both BNC1 and BNC2 were down-regulated in HCC cell lines and primary HCC tissues. The frequency and intensity of BNC1 promoter hypermethylation in tumor tissues was significantly higher than that in adjacent non-tumor tissues. 5-Aza-2'-Deoxycytidine treatment could significantly increase the BNC1 expression in the methylated HCC cell lines, which implicated that epigenetic modification contributed to the down-regulation of BNC1. In addition, BNC1 hypermethylation in tumor tissues was more likely to happen in female patients. No methylation of the BNC2 promoter was found in HCC tumor tissues. However, a frequent deletion of the BNC2 gene was observed, which indicated that the chromosomal loss of the BNC2 gene might be one important reason for its lower expression level in HCC. Our results suggested that BNC1 and BNC2 were down-regulated in HCC which may provide new insight into the tumorigenesis of HCC.\", \"An increasingly recognized resistance mechanism to androgen receptor (AR)-directed therapy in prostate cancer involves epithelial plasticity, in which tumor cells demonstrate low to absent AR expression and often have neuroendocrine features. The etiology and molecular basis for this 'alternative' treatment-resistant cell state remain incompletely understood. Here, by analyzing whole-exome sequencing data of metastatic biopsies from patients, we observed substantial genomic overlap between castration-resistant tumors that were histologically characterized as prostate adenocarcinomas (CRPC-Adeno) and neuroendocrine prostate cancer (CRPC-NE); analysis of biopsy samples from the same individuals over time points to a model most consistent with divergent clonal evolution. Genome-wide DNA methylation analysis revealed marked epigenetic differences between CRPC-NE tumors and CRPC-Adeno, and also designated samples of CRPC-Adeno with clinical features of AR independence as CRPC-NE, suggesting that epigenetic modifiers may play a role in the induction and/or maintenance of this treatment-resistant state. This study supports the emergence of an alternative, 'AR-indifferent' cell state through divergent clonal evolution as a mechanism of treatment resistance in advanced prostate cancer.\", \"TP53 is the most frequently mutated gene in human cancer. Functionally, p53 is activated by a host of stress stimuli and, in turn, governs an exquisitely complex anti-proliferative transcriptional program that touches upon a bewildering array of biological responses. Despite the many unveiled facets of the p53 network, a clear appreciation of how and in what contexts p53 exerts its diverse effects remains unclear. How can we interpret p53's disparate activities and the consequences of its dysfunction to understand how cell type, mutation profile, and epigenetic cell state dictate outcomes, and how might we restore its tumor-suppressive activities in cancer?\", 'The field of machine learning, which aims to develop computer algorithms that improve with experience, holds promise to enable computers to assist humans in the analysis of large, complex data sets. Here, we provide an overview of machine learning applications for the analysis of genome sequencing data sets, including the annotation of sequence elements and epigenetic, proteomic or metabolomic data. We present considerations and recurrent challenges in the application of supervised, semi-supervised and unsupervised machine learning methods, as well as of generative and discriminative modelling approaches. We provide general guidelines to assist in the selection of these machine learning methods and their practical application for the analysis of genetic and genomic data sets.', 'Mammalian DNA methylation is a critical epigenetic mechanism orchestrating gene expression networks in many biological processes. However, investigation of the functions of specific methylation events remains challenging. Here, we demonstrate that fusion of Tet1 or Dnmt3a with a catalytically inactive Cas9 (dCas9) enables targeted DNA methylation editing. Targeting of the dCas9-Tet1 or -Dnmt3a fusion protein to methylated or unmethylated promoter sequences caused activation or silencing, respectively, of an endogenous reporter. Targeted demethylation of the BDNF promoter IV or the MyoD distal enhancer by dCas9-Tet1 induced BDNF expression in post-mitotic neurons or activated MyoD facilitating reprogramming of fibroblasts into myoblasts, respectively. Targeted de novo methylation of a CTCF loop anchor site by dCas9-Dnmt3a blocked CTCF binding and interfered with DNA looping, causing altered gene expression in the neighboring loop. Finally, we show that these tools can edit DNA methylation in mice, demonstrating their wide utility for functional studies of epigenetic regulation.', 'Diverse classes of RNA, ranging from small to long non-coding RNAs, have emerged as key regulators of gene expression, genome stability and defence against foreign genetic elements. Small RNAs modify chromatin structure and silence transcription by guiding Argonaute-containing complexes to complementary nascent RNA scaffolds and then mediating the recruitment of histone and DNA methyltransferases. In addition, recent advances suggest that chromatin-associated long non-coding RNA scaffolds also recruit chromatin-modifying complexes independently of small RNAs. These co-transcriptional silencing mechanisms form powerful RNA surveillance systems that detect and silence inappropriate transcription events, and provide a memory of these events via self-reinforcing epigenetic loops.', \"Next-generation sequencing has revealed that more than 50% of human cancers harbour mutations in enzymes that are involved in chromatin organization. Tumour cells not only are activated by genetic and epigenetic alterations, but also routinely use epigenetic processes to ensure their escape from chemotherapy and host immune surveillance. Hence, a growing emphasis of recent drug discovery efforts has been on targeting the epigenome, including DNA methylation and histone modifications, with several new drugs being tested and some already approved by the US Food and Drug Administration (FDA). The future will see the increasing success of combining epigenetic drugs with other therapies. As epigenetic drugs target the epigenome as a whole, these true 'genomic medicines' lessen the need for precision approaches to individualized therapies.\", 'One-carbon (1C) metabolism, mediated by the folate cofactor, supports multiple physiological processes. These include biosynthesis (purines and thymidine), amino acid homeostasis (glycine, serine, and methionine), epigenetic maintenance, and redox defense. Both within eukaryotic cells and across organs, 1C metabolic reactions are compartmentalized. Here we review the fundamentals of mammalian 1C metabolism, including the pathways active in different compartments, cell types, and biological states. Emphasis is given to recent discoveries enabled by modern genetics, analytical chemistry, and isotope tracing. An emerging theme is the biological importance of mitochondrial 1C reactions, both for producing 1C units that are exported to the cytosol and for making additional products, including glycine and NADPH. Increased clarity regarding differential folate pathway usage in cancer, stem cells, development, and adult physiology is reviewed and highlights new opportunities for selective therapeutic intervention.', 'Acetyl-coenzyme A (acetyl-CoA) is a central metabolic intermediate. The abundance of acetyl-CoA in distinct subcellular compartments reflects the general energetic state of the cell. Moreover, acetyl-CoA concentrations influence the activity or specificity of multiple enzymes, either in an allosteric manner or by altering substrate availability. Finally, by influencing the acetylation profile of several proteins, including histones, acetyl-CoA controls key cellular processes, including energy metabolism, mitosis, and autophagy, both directly and via the epigenetic regulation of gene expression. Thus, acetyl-CoA determines the balance between cellular catabolism and anabolism by simultaneously operating as a metabolic intermediate and as a second messenger.', 'Cancer growth and progression are associated with immune suppression. Cancer cells have the ability to activate different immune checkpoint pathways that harbor immunosuppressive functions. Monoclonal antibodies that target immune checkpoints provided an immense breakthrough in cancer therapeutics. Among the immune checkpoint inhibitors, PD-1/PD-L1 and CTLA-4 inhibitors showed promising therapeutic outcomes, and some have been approved for certain cancer treatments, while others are under clinical trials. Recent reports have shown that patients with various malignancies benefit from immune checkpoint inhibitor treatment. However, mainstream initiation of immune checkpoint therapy to treat cancers is obstructed by the low response rate and immune-related adverse events in some cancer patients. This has given rise to the need for developing sets of biomarkers that predict the response to immune checkpoint blockade and immune-related adverse events. In this review, we discuss different predictive biomarkers for anti-PD-1/PD-L1 and anti-CTLA-4 inhibitors, including immune cells, PD-L1 overexpression, neoantigens, and genetic and epigenetic signatures. Potential approaches for further developing highly reliable predictive biomarkers should facilitate patient selection for and decision-making related to immune checkpoint inhibitor-based therapies.', 'Nucleosome positioning varies between cell types. By deep sequencing cell-free DNA (cfDNA), isolated from circulating blood plasma, we generated maps of genome-wide in vivo nucleosome occupancy and found that short cfDNA fragments harbor footprints of transcription factors. The cfDNA nucleosome occupancies correlate well with the nuclear architecture, gene structure, and expression observed in cells, suggesting that they could inform the cell type of origin. Nucleosome spacing inferred from cfDNA in healthy individuals correlates most strongly with epigenetic features of lymphoid and myeloid cells, consistent with hematopoietic cell death as the normal source of cfDNA. We build on this observation to show how nucleosome footprints can be used to infer cell types contributing to cfDNA in pathological states such as cancer. Since this strategy does not rely on genetic differences to distinguish between contributing tissues, it may enable the noninvasive monitoring of a much broader set of clinical conditions than currently possible.', 'The Cas9 protein (CRISPR-associated protein 9), derived from type II CRISPR (clustered regularly interspaced short palindromic repeats) bacterial immune systems, is emerging as a powerful tool for engineering the genome in diverse organisms. As an RNA-guided DNA endonuclease, Cas9 can be easily programmed to target new sites by altering its guide RNA sequence, and its development as a tool has made sequence-specific gene editing several magnitudes easier. The nuclease-deactivated form of Cas9 further provides a versatile RNA-guided DNA-targeting platform for regulating and imaging the genome, as well as for rewriting the epigenetic status, all in a sequence-specific manner. With all of these advances, we have just begun to explore the possible applications of Cas9 in biomedical research and therapeutics. In this review, we describe the current models of Cas9 function and the structural and biochemical studies that support it. We focus on the applications of Cas9 for genome editing, regulation, and imaging, discuss other possible applications and some technical considerations, and highlight the many advantages that CRISPR/Cas9 technology offers.', 'Carcinogenesis cannot be explained only by genetic alterations, but also involves epigenetic processes. Modification of histones by acetylation plays a key role in epigenetic regulation of gene expression and is controlled by the balance between histone deacetylases (HDAC) and histone acetyltransferases (HAT). HDAC inhibitors induce cancer cell cycle arrest, differentiation and cell death, reduce angiogenesis and modulate immune response. Mechanisms of anticancer effects of HDAC inhibitors are not uniform; they may be different and depend on the cancer type, HDAC inhibitors, doses, etc. HDAC inhibitors seem to be promising anti-cancer drugs particularly in the combination with other anti-cancer drugs and/or radiotherapy. HDAC inhibitors vorinostat, romidepsin and belinostat have been approved for some T-cell lymphoma and panobinostat for multiple myeloma. Other HDAC inhibitors are in clinical trials for the treatment of hematological and solid malignancies. The results of such studies are promising but further larger studies are needed. Because of the reversibility of epigenetic changes during cancer development, the potency of epigenetic therapies seems to be of great importance. Here, we summarize the data on different classes of HDAC inhibitors, mechanisms of their actions and discuss novel results of preclinical and clinical studies, including the combination with other therapeutic modalities.', 'The finding that small non-coding RNAs (ncRNAs) are able to control gene expression in a sequence specific manner has had a massive impact on biology. Recent improvements in high throughput sequencing and computational prediction methods have allowed the discovery and classification of several types of ncRNAs. Based on their precursor structures, biogenesis pathways and modes of action, ncRNAs are classified as small interfering RNAs (siRNAs), microRNAs (miRNAs), PIWI-interacting RNAs (piRNAs), endogenous small interfering RNAs (endo-siRNAs or esiRNAs), promoter associate RNAs (pRNAs), small nucleolar RNAs (snoRNAs) and sno-derived RNAs. Among these, miRNAs appear as important cytoplasmic regulators of gene expression. miRNAs act as post-transcriptional regulators of their messenger RNA (mRNA) targets via mRNA degradation and/or translational repression. However, it is becoming evident that miRNAs also have specific nuclear functions. Among these, the most studied and debated activity is the miRNA-guided transcriptional control of gene expression. Although available data detail quite precisely the effectors of this activity, the mechanisms by which miRNAs identify their gene targets to control transcription are still a matter of debate. Here, we focus on nuclear functions of miRNAs and on alternative mechanisms of target recognition, at the promoter lavel, by miRNAs in carrying out transcriptional gene silencing.', 'Since decades it has been known that non-protein-coding RNAs have important cellular functions. Deep sequencing recently facilitated the discovery of thousands of novel transcripts, now classified as long noncoding RNAs (lncRNAs), in many vertebrate and invertebrate species. LncRNAs are involved in a wide range of cellular mechanisms, from almost all aspects of gene expression to protein translation and stability. Recent findings implicate lncRNAs as key players of cellular differentiation, cell lineage choice, organogenesis and tissue homeostasis. Moreover, lncRNAs are involved in pathological conditions such as cancer and cardiovascular disease, and therefore provide novel biomarkers and pharmaceutical targets. Here we discuss examples illustrating the versatility of lncRNAs in gene control, development and differentiation, as well as in human disease.', \"Identifying and validating molecular targets of interventions that extend the human health span and lifespan has been difficult, as most clinical biomarkers are not sufficiently representative of the fundamental mechanisms of ageing to serve as their indicators. In a recent breakthrough, biomarkers of ageing based on DNA methylation data have enabled accurate age estimates for any tissue across the entire life course. These 'epigenetic clocks' link developmental and maintenance processes to biological ageing, giving rise to a unified theory of life course. Epigenetic biomarkers may help to address long-standing questions in many fields, including the central question: why do we age?\", 'Epithelial-mesenchymal transition (EMT) is a process in which epithelial cells acquire mesenchymal features. In cancer, EMT is associated with tumor initiation, invasion, metastasis, and resistance to therapy. Recently, it has been demonstrated that EMT is not a binary process, but occurs through distinct cellular states. Here, we review the recent studies that demonstrate the existence of these different EMT states in cancer and the mechanisms regulating their functions. We discuss the different functional characteristics, such as proliferation, propagation, plasticity, invasion, and metastasis associated with the distinct EMT states. We summarize the role of the transcriptional and epigenetic landscapes, gene regulatory network and their surrounding niche in controlling the transition through the different EMT states.', 'Methylation of DNA and of histone 3 at Lys 9 (H3K9) are highly correlated with gene silencing in eukaryotes from fungi to humans. Both of these epigenetic marks need to be established at specific regions of the genome and then maintained at these sites through cell division. Protein structural domains that specifically recognize methylated DNA and methylated histones are key for targeting enzymes that catalyse these marks to appropriate genome sites. Genetic, genomic, structural and biochemical data reveal connections between these two epigenetic marks, and these domains mediate much of the crosstalk.', \"This year is the tenth anniversary of the publication in this journal of a model suggesting the existence of 'tumour progenitor genes'. These genes are epigenetically disrupted at the earliest stages of malignancies, even before mutations, and thus cause altered differentiation throughout tumour evolution. The past decade of discovery in cancer epigenetics has revealed a number of similarities between cancer genes and stem cell reprogramming genes, widespread mutations in epigenetic regulators, and the part played by chromatin structure in cellular plasticity in both development and cancer. In the light of these discoveries, we suggest here a framework for cancer epigenetics involving three types of genes: 'epigenetic mediators', corresponding to the tumour progenitor genes suggested earlier; 'epigenetic modifiers' of the mediators, which are frequently mutated in cancer; and 'epigenetic modulators' upstream of the modifiers, which are responsive to changes in the cellular environment and often linked to the nuclear architecture. We suggest that this classification is helpful in framing new diagnostic and therapeutic approaches to cancer.\", 'Specification of primordial germ cells (PGCs) marks the beginning of the totipotent state. However, without a tractable experimental model, the mechanism of human PGC (hPGC) specification remains unclear. Here, we demonstrate specification of hPGC-like cells (hPGCLCs) from germline competent pluripotent stem cells. The characteristics of hPGCLCs are consistent with the embryonic hPGCs and a germline seminoma that share a CD38 cell-surface marker, which collectively defines likely progression of the early human germline. Remarkably, SOX17 is the key regulator of hPGC-like fate, whereas BLIMP1 represses endodermal and other somatic genes during specification of hPGCLCs. Notable mechanistic differences between mouse and human PGC specification could be attributed to their divergent embryonic development and pluripotent states, which might affect other early cell-fate decisions. We have established a foundation for future studies on resetting of the epigenome in hPGCLCs and hPGCs for totipotency and the transmission of genetic and epigenetic information.', 'The discovery of long non-coding RNA (lncRNA) has dramatically altered our understanding of cancer. Here, we describe a comprehensive analysis of lncRNA alterations at transcriptional, genomic, and epigenetic levels in 5,037 human tumor specimens across 13 cancer types from The Cancer Genome Atlas. Our results suggest that the expression and dysregulation of lncRNAs are highly cancer type specific compared with protein- coding genes. Using the integrative data generated by this analysis, we present a clinically guided small interfering RNA screening strategy and a co-expression analysis approach to identify cancer driver lncRNAs and predict their functions. This provides a resource for investigating lncRNAs in cancer and lays the groundwork for the development of new diagnostics and treatments.', 'Plant genonnes encode various small RNAs that function in distinct, yet overlapping, genetic and epigenetic silencing pathways. However, the abundance and diversity of small-RNA classes varies among plant species, suggesting coevolution between environmental adaptations and gene-silencing mechanisms. Biogenesis of small RNAs in plants is well understood, but we are just beginning to uncover their intricate regulation and activity. Here, we discuss the biogenesis of plant small RNAs, such as nnicroRNAs, secondary siRNAs and heterochromatic siRNAs, and their diverse cellular and developmental functions, including in reproductive transitions, genomic imprinting and paramutation. We also discuss the diversification of small-RNA-directed silencing pathways through the expansion of RNA-dependent RNA polymerases, DICER proteins and ARGONAUTE proteins.', 'Epigenetic modifications, including DNA methylation, represent a potential mechanism for environmental impacts on human disease. Maternal smoking in pregnancy remains an important public health problem that impacts child health in a myriad of ways and has potential lifelong consequences. The mechanisms are largely unknown, but epigenetics most likely plays a role. We formed the Pregnancy And Childhood Epigenetics (PACE) consortium and meta-analyzed, across 13 cohorts (n = 6,685), the association between maternal smoking in pregnancy and newborn blood DNA methylation at over 450,000 CpG sites (CpGs) by using the Illumina 450K BeadChip. Over 6,000 CpGs were differentially methylated in relation to maternal smoking at genome-wide statistical significance (false discovery rate, 5%), including 2,965 CpGs corresponding to 2,017 genes not previously related to smoking and methylation in either newborns or adults. Several genes are relevant to diseases that can be caused by maternal smoking (e.g., orofacial clefts and asthma) or adult smoking (e.g., certain cancers). A number of differentially methylated CpGs were associated with gene expression. We observed enrichment in pathways and processes critical to development. In older children (5 cohorts, n = 3,187), 100% of CpGs gave at least nominal levels of significance, far more than expected by chance (p value < 2.2 x 10(-16)). Results were robust to different normalization methods used across studies and cell type adjustment. In this large scale meta-analysis of methylation data, we identified numerous loci involved in response to maternal smoking in pregnancy with persistence into later childhood and provide insights into mechanisms underlying effects of this important exposure.', 'In 1964, Vincent Allfrey and colleagues reported the identification of histone acetylation and with deep insight proposed a regulatory role for this protein modification in transcription regulation. Subsequently, histone acetyltransferases (HATs), histone deacetylases (HDACs) and acetyl-Lys-binding proteins were identified as transcription regulators, thereby providing compelling evidence for his daring hypothesis. During the past 15 years, reversible protein acetylation and its modifying enzymes have been implicated in many cellular functions beyond transcription regulation. Here, we review the progress accomplished during the past 50 years and discuss the future of protein acetylation.', 'N-6-methyladenosine (m(6)A) is the most prevalent and internal modification that occurs in the messenger RNAs (mRNA) of most eukaryotes, although its functional relevance remained a mystery for decades. This modification is installed by the m(6)A methylation writers and can be reversed by demethylases that serve as erasers. In this review, we mainly summarize recent progress in the study of the m(6)A mRNA methylation machineries across eukaryotes and discuss their newly uncovered biological functions. The broad roles of m(6)A in regulating cell fates and embryonic development highlight the existence of another layer of epigenetic regulation at the RNA level, where mRNA is subjected to chemical modifications that affect protein expression.', 'The Bromo- and Extra-Terminal (BET) proteins BRD2, BRD3, and BRD4 play important roles in transcriptional regulation, epigenetics, and cancer and are the targets of pan-BET selective bromodomain inhibitor JQ1. However, the lack of intra-BET selectivity limits the scope of current inhibitors as probes for target validation and could lead to unwanted side effects or toxicity in a therapeutic setting. We designed Proteolysis Targeted Chimeras (PROTACs) that tether JQ1 to a ligand for the E3 ubiquitin ligase VHL, aimed at triggering the intracellular destruction of BET proteins. Compound MZ1 potently and rapidly induces reversible, long-lasting, and unexpectedly selective removal of BRD4 over BRD2 and BRD3. The activity of MZ1 is dependent on binding to VHL but is achieved at a sufficiently low concentration not to induce stabilization of HIF-1 alpha. Gene expression profiles of selected cancer-related genes responsive to JQ1 reveal distinct and more limited transcriptional responses induced by MZ1, consistent with selective suppression of BRD4. Our discovery opens up new opportunities to elucidate the cellular phenotypes and therapeutic implications associated with selective targeting of BRD4.', \"We define the chromatin accessibility and transcriptional landscapes in 13 human primary blood cell types that span the hematopoietic hierarchy. Exploiting the finding that the enhancer landscape better reflects cell identity than mRNA levels, we enable 'enhancer cytometry' for enumeration of pure cell types from complex populations. We identify regulators governing hematopoietic differentiation and further show the lineage ontogeny of genetic elements linked to diverse human diseases. In acute myeloid leukemia (AML), chromatin accessibility uncovers unique regulatory evolution in cancer cells with a progressively increasing mutation burden. Single AML cells exhibit distinctive mixed regulome profiles corresponding to disparate developmental stages. A method to account for this regulatory heterogeneity identified cancer-specific deviations and implicated HOX factors as key regulators of preleukemic hematopoietic stem cell characteristics. Thus, regulome dynamics can provide diverse insights into hematopoietic development and disease.\", 'Treatment of cancer has been revolutionized by immune checkpoint blockade therapies. Despite the high rate of response in advanced melanoma, the majority of patients succumb to disease. To identify factors associated with success or failure of checkpoint therapy, we profiled transcriptomes of 16,291 individual immune cells from 48 tumor samples of melanoma patients treated with checkpoint inhibitors. Two distinct states of CD8(+) T cells were defined by clustering and associated with patient tumor regression or progression. A single transcription factor, TCF7, was visualized within CD8(+) T cells in fixed tumor samples and predicted positive clinical outcome in an independent cohort of checkpoint-treated patients. We delineated the epigenetic landscape and clonality of these T cell states and demonstrated enhanced antitumor immunity by targeting novel combinations of factors in exhausted cells. Our study of immune cell transcriptomes from tumors demonstrates a strategy for identifying predictors, mechanisms, and targets for enhancing checkpoint immunotherapy.', 'The extent of tumor heterogeneity is an emerging theme that researchers are only beginning to understand. How genetic and epigenetic heterogeneity affects tumor evolution and clinical progression is unknown. The precise nature of the environmental factors that influence this heterogeneity is also yet to be characterized. Nature Medicine, Nature Biotechnology and the Volkswagen Foundation organized a meeting focused on identifying the obstacles that need to be overcome to advance translational research in and tumor heterogeneity. Once these key questions were established, the attendees devised potential solutions. Their ideas are presented here.', 'Single-stranded guanine-rich DNA sequences can fold into four-stranded DNA structures called G-quadruplexes (G4s) that arise from the self-stacking of two or more guanine quartets. There has been considerable recent progress in the detection and mapping of G4 structures in the human genome and in biologically relevant contexts. These advancements, many of which align with predictions made previously in computational studies, provide important new insights into the functions of G4 structures in, for example, the regulation of transcription and genome stability, and uncover their potential relevance for cancer therapy.', 'Humans show strong sex differences in immunity to infection and autoimmunity, suggesting sex hormones modulate immune responses. Indeed, receptors for estrogens (ERs) regulate cells and pathways in the innate and adaptive immune system, as well as immune cell development. ERs are ligand-dependent transcription factors that mediate long-range chromatin interactions and form complexes at gene regulatory elements, thus promoting epigenetic changes and transcription. ERs also participate in membrane-initiated steroid signaling to generate rapid responses. Estradiol and ER activity show profound dose- and context-dependent effects on innate immune signaling pathways and myeloid cell development. While estradiol most often promotes the production of type I interferon, innate pathways leading to pro-inflammatory cytokine production may be enhanced or dampened by ER activity. Regulation of innate immune cells and signaling by ERs may contribute to the reported sex differences in innate immune pathways. Here we review the recent literature and highlight several molecular mechanisms by which ERs regulate the development or functional responses of innate immune cells. (C) 2015 Elsevier Inc. All rights reserved.', 'Epigenetic studies relied so far on correlations between epigenetic marks and gene expression pattern. Technologies developed for epigenome editing now enable direct study of functional relevance of precise epigenetic modifications and gene regulation. The reversible nature of epigenetic modifications, including DNA methylation, has been already exploited in cancer therapy for remodeling the aberrant epigenetic landscape. However, this was achieved non-selectively using epigenetic inhibitors. Epigenetic editing at specific loci represents a novel approach that might selectively and heritably alter gene expression. Here, we developed a CRISPR-Cas9-based tool for specific DNA methylation consisting of deactivated Cas9 (dCas9) nuclease and catalytic domain of the DNA methyltransferase DNMT3A targeted by co-expression of a guide RNA to any 20 bp DNA sequence followed by the NGG trinucleotide. We demonstrated targeted CpG methylation in a similar to 35 bp wide region by the fusion protein. We also showed that multiple guide RNAs could target the dCas9-DNMT3A construct to multiple adjacent sites, which enabled methylation of a larger part of the promoter. DNA methylation activity was specific for the targeted region and heritable across mitotic divisions. Finally, we demonstrated that directed DNA methylation of a wider promoter region of the target loci IL6ST and BACH2 decreased their expression.', 'The DNA methyltransferase (DNMT) family comprises a conserved set of DNA-modifying enzymes that have a central role in epigenetic gene regulation. Recent studies have shown that the functions of the canonical DNMT enzymes-DNMT1, DNMT3A and DNMT3B-go beyond their traditional roles of establishing and maintaining DNA methylation patterns. This Review analyses how molecular interactions and changes in gene copy numbers modulate the activity of DNMTs in diverse gene regulatory functions, including transcriptional silencing, transcriptional activation and post-transcriptional regulation by DNMT2-dependent tRNA methylation. This mechanistic diversity enables the DNMT family to function as a versatile toolkit for epigenetic regulation.', 'Estimates of biological age based on DNA methylation patterns, often referred to as epigenetic age, DNAm age, have been shown to be robust biomarkers of age in humans. We previously demonstrated that independent of chronological age, epigenetic age assessed in blood predicted all-cause mortality in four human cohorts. Here, we expanded our original observation to 13 different cohorts for a total sample size of 13,089 individuals, including three racial/ethnic groups. In addition, we examined whether incorporating information on blood cell composition into the epigenetic age metrics improves their predictive power for mortality. All considered measures of epigenetic age acceleration were predictive of mortality (p <= 8.2x10(-9)), independent of chronological age, even after adjusting for additional risk factors (p<5.4x10(-4)), and within the racial/ethnic groups that we examined (non-Hispanic whites, Hispanics, African Americans). Epigenetic age estimates that incorporated information on blood cell composition led to the smallest p-values for time to death (p=7.5x10(-43)). Overall, this study a) strengthens the evidence that epigenetic age predicts all-cause mortality above and beyond chronological age and traditional risk factors, and b) demonstrates that epigenetic age estimates that incorporate information on blood cell counts lead to highly significant associations with all-cause mortality.', \"Childhood trauma confers higher risk of adulthood physical and mental illness; however, the biological mechanism mediating this association remains largely unknown. Recent research has suggested dysregulation of the immune system as a possible biological mediator. The present paper conducted a meta-analysis to establish whether early-life adversity contributes to potentially pathogenic pro-inflammatory phenotypes in adult individuals. A systematic search of Pubmed, PsycINFO, EMBASE, Scopus and Medline identified 25 articles for the meta-analysis, including 18 studies encompassing a sample of 16 870 individuals for C-reactive protein (CRP), 15 studies including 3751 individuals for interleukin-6 (IL-6) and 10 studies including 881 individuals for tumour necrosis factor-alpha (TNF-alpha). Random-effects meta-analysis showed that individuals exposed to childhood trauma had significantly elevated baseline peripheral levels of CRP (Fisher's z = 0.10, 95% confidence interval (CI) = 0.05-0.14), IL-6 (z = 0.08, 95% CI = 0.03-0.14) and TNF-alpha (z = 0.23, 95% CI = 0.14-0.32). Subgroup analyses for specific types of trauma (sexual, physical or emotional abuse) revealed that these impact differentially the single inflammatory markers. Moreover, meta-regression revealed greater effect sizes in clinical samples for the association between childhood trauma and CRP but not for IL-6 or TNF-alpha. Age, body mass index (BMI) and gender had no moderating effects. The analysis demonstrates that childhood trauma contributes to a pro-inflammatory state in adulthood, with specific inflammatory profiles depending on the specific type of trauma.\", 'The heart must continuously pump blood to supply the body with oxygen and nutrients. To maintain the high energy consumption required by this role, the heart is equipped with multiple complex biological systems that allow adaptation to changes of systemic demand. The processes of growth (hypertrophy), angiogenesis, and metabolic plasticity are critically involved in maintenance of cardiac homeostasis. Cardiac hypertrophy is classified as physiological when it is associated with normal cardiac function or as pathological when associated with cardiac dysfunction. Physiological hypertrophy of the heart occurs in response to normal growth of children or during pregnancy, as well as in athletes. In contrast, pathological hypertrophy is induced by factors such as prolonged and abnormal hemodynamic stress, due to hypertension, myocardial infarction etc. Pathological hypertrophy is associated with fibrosis, capillary rarefaction, increased production of pro-inflammatory cytokines, and cellular dysfunction (impairment of signaling, suppression of autophagy, and abnormal cardiomyocyte/non-cardiomyocyte interactions), as well as undesirable epigenetic changes, with these complex responses leading to maladaptive cardiac remodeling and heart failure. This review describes the key molecules and cellular responses involved in physiological/pathological cardiac hypertrophy. (C) 2016 Elsevier Ltd. All rights reserved.', \"The process of aging results in a host of changes at the cellular and molecular levels, which include senescence, telomere shortening, and changes in gene expression. Epigenetic patterns also change over the lifespan, suggesting that epigenetic changes may constitute an important component of the aging process. The epigenetic mark that has been most highly studied is DNA methylation, the presence of methyl groups at CpG dinucleotides. These dinucleotides are often located near gene promoters and associate with gene expression levels. Early studies indicated that global levels of DNA methylation increase over the first few years of life and then decrease beginning in late adulthood. Recently, with the advent of microarray and next-generation sequencing technologies, increases in variability of DNA methylation with age have been observed, and a number of site-specific patterns have been identified. It has also been shown that certain CpG sites are highly associated with age, to the extent that prediction models using a small number of these sites can accurately predict the chronological age of the donor. Together, these observations point to the existence of two phenomena that both contribute to age-related DNA methylation changes: epigenetic drift and the epigenetic clock. In this review, we focus on healthy human aging throughout the lifetime and discuss the dynamics of DNA methylation as well as how interactions between the genome, environment, and the epigenome influence aging rates. We also discuss the impact of determining 'epigenetic age' for human health and outline some important caveats to existing and future studies.\", 'The ability to rapidly restore the integrity of a broken skin barrier is critical and is the ultimate goal of therapies for hard-to-heal-ulcers. Unfortunately effective treatments to enhance healing and reduce scarring are still lacking. A deeper understanding of the physiology of normal repair and of the pathology of delayed healing is a prerequisite for the development of more effective therapeutic interventions. Transition from the inflammatory to the proliferative phase is a key step during healing and accumulating evidence associates a compromised transition with wound healing disorders. Thus, targeting factors that impact this phase transition may offer a rationale for therapeutic development. This review summarizes mechanisms regulating the inflammation-proliferation transition at cellular and molecular levels. We propose that identification of such mechanisms will reveal promising targets for development of more effective therapies.', 'Adult somatic stem cells in various organs maintain homeostatic tissue regeneration and enhance plasticity. Since its initial discovery five decades ago, investigations of adult neurogenesis and neural stem cells have led to an established and expanding field that has significantly influenced many facets of neuroscience, developmental biology, and regenerative medicine. Here we review recent progress and focus on questions related to adult mammalian neural stem cells that also apply to other somatic stem cells. We further discuss emerging topics that are guiding the field toward better understanding adult neural stem cells and ultimately applying these principles to improve human health.', 'The dogma that adaptive immunity is the only arm of the immune response with memory capacity has been recently challenged by several studies demonstrating evidence for memory-like innate immune training. However, the underlying mechanisms and location for generating such innate memory responses in vivo remain unknown. Here, we show that access of Bacillus Calmette-Guerin (BCG) to the bone marrow (BM) changes the transcriptional landscape of hematopoietic stem cells (HSCs) and multipotent progenitors (MPPs), leading to local cell expansion and enhanced myelopoiesis at the expense of lymphopoiesis. Importantly, BCG-educated HSCs generate epigenetically modified macrophages that provide significantly better protection against virulent M. tuberculosis infection than naive macrophages. By using parabiotic and chimeric mice, as well as adoptive transfer approaches, we demonstrate that training of the monocyte/macrophage lineage via BCG-induced HSC reprogramming is sustainable in vivo. Our results indicate that targeting the HSC compartment provides a novel approach for vaccine development.', 'Experimental modelling of human disorders enables the definition of the cellular and molecular mechanisms underlying diseases and the development of therapies for treating them. The availability of human pluripotent stem cells (PSCs), which are capable of self-renewal and have the potential to differentiate into virtually any cell type, can now help to overcome the limitations of animal models for certain disorders. The ability to model human diseases using cultured PSCs has revolutionized the ways in which we study monogenic, complex and epigenetic disorders, as well as early- and late-onset diseases. Several strategies are used to generate such disease models using either embryonic stem cells (ES cells) or patient-specific induced PSCs (iPSCs), creating new possibilities for the establishment of models and their use in drug screening.', 'Resetting of the epigenome in human primordial germ cells (hPGCs) is critical for development. We show that the transcriptional program of hPGCs is distinct from that in mice, with co-expression of somatic specifiers and naive pluripotency genes TFCP2L1 and KLF4. This unique gene regulatory network, established by SOX17 and BLIMP1, drives comprehensive germline DNA demethylation by repressing DNA methylation pathways and activating TET-mediated hydroxymethylation. Base-resolution methylome analysis reveals progressive DNA demethylation to basal levels in week 5-7 in vivo hPGCs. Concurrently, hPGCs undergo chromatin reorganization, X reactivation, and imprint erasure. Despite global hypomethylation, evolutionarily young and potentially hazardous retroelements, like SVA, remain methylated. Remarkably, some loci associated with metabolic and neurological disorders are also resistant to DNA demethylation, revealing potential for transgenerational epigenetic inheritance that may have phenotypic consequences. We provide comprehensive insight on early human germline transcriptional network and epigenetic reprogramming that subsequently impacts human development and disease.', 'The cBioPortal for Cancer Genomics (http://cbioportal.org) provides a Web resource for exploring, visualizing, and analyzing multidimensional cancer genomics data. The portal reduces molecular profiling data from cancer tissues and cell lines into readily understandable genetic, epigenetic, gene expression, and proteomic events. The query interface combined with customized data storage enables researchers to interactively explore genetic alterations across samples, genes, and pathways and, when available in the underlying data, to link these to clinical outcomes. The portal provides graphical summaries of gene-level data from multiple platforms, network visualization and analysis, survival analysis, patient-centric queries, and software programmatic access. The intuitive Web interface of the portal makes complex cancer genomics profiles accessible to researchers and clinicians without requiring bioinformatics expertise, thus facilitating biological discoveries. Here, we provide a practical guide to the analysis and visualization features of the cBioPortal for Cancer Genomics.', 'Aging is characterized by a progressive loss of physiological integrity, leading to impaired function and increased vulnerability to death. This deterioration is the primary risk factor for major human pathologies, including cancer, diabetes, cardiovascular disorders, and neurodegenerative diseases. Aging research has experienced an unprecedented advance over recent years, particularly with the discovery that the rate of aging is controlled, at least to some extent, by genetic pathways and biochemical processes conserved in evolution. This Review enumerates nine tentative hallmarks that represent common denominators of aging in different organisms, with special emphasis on mammalian aging. These hallmarks are: genomic instability, telomere attrition, epigenetic alterations, loss of proteostasis, deregulated nutrient sensing, mitochondrial dysfunction, cellular senescence, stem cell exhaustion, and altered intercellular communication. A major challenge is to dissect the interconnectedness between the candidate hallmarks and their relative contributions to aging, with the final goal of identifying pharmaceutical targets to improve human health during aging, with minimal side effects.', \"DNA methylation is frequently described as a 'silencing' epigenetic mark, and indeed this function of 5-methylcytosine was originally proposed in the 1970s. Now, thanks to improved genome-scale mapping of methylation, we can evaluate DNA methylation in different genomic contexts: transcriptional start sites with or without CpG islands, in gene bodies, at regulatory elements and at repeat sequences. The emerging picture is that the function of DNA methylation seems to vary with context, and the relationship between DNA methylation and transcription is more nuanced than we realized at first. Improving our understanding of the functions of DNA methylation is necessary for interpreting changes in this mark that are observed in diseases such as cancer.\", 'The Cancer Genome Atlas (TCGA) Research Network has profiled and analyzed large numbers of human tumors to discover molecular aberrations at the DNA, RNA, protein and epigenetic levels. The resulting rich data provide a major opportunity to develop an integrated picture of commonalities, differences and emergent themes across tumor lineages. The Pan-Cancer initiative compares the first 12 tumor types profiled by TCGA. Analysis of the molecular aberrations and their functional roles across tumor types will teach us how to extend therapies effective in one cancer type to others with a similar genomic profile.', 'The central dogma of gene expression is that DNA is transcribed into messenger RNAs, which in turn serve as the template for protein synthesis. The discovery of extensive transcription of large RNA transcripts that do not code for proteins, termed long noncoding RNAs (lncRNAs), provides an important new perspective on the centrality of RNA in gene regulation. Here, we discuss genome-scale strategies to discover and characterize lncRNAs. An emerging theme from multiple model systems is that lncRNAs form extensive networks of ribonucleoprotein (RNP) complexes with numerous chromatin regulators and then target these enzymatic activities to appropriate locations in the genome. Consistent with this notion, lncRNAs can function as modular scaffolds to specify higher-order organization in RNP complexes and in chromatin states. The importance of these modes of regulation is underscored by the newly recognized roles of long RNAs for proper gene control across all kingdoms of life.', 'Metastases represent the end products of a multistep cell-biological process termed the invasion-metastasis cascade, which involves dissemination of cancer cells to anatomically distant organ sites and their subsequent adaptation to foreign tissue microenvironments. Each of these events is driven by the acquisition of genetic and/or epigenetic alterations within tumor cells and the co-option of nonneoplastic stromal cells, which together endow incipient metastatic cells with traits needed to generate macroscopic metastases. Recent advances provide provocative insights into these cell-biological and molecular changes, which have implications regarding the steps of the invasion-metastasis cascade that appear amenable to therapeutic targeting.', 'Background: It is not yet known whether DNA methylation levels can be used to accurately predict age across a broad spectrum of human tissues and cell types, nor whether the resulting age prediction is a biologically meaningful measure. Results: I developed a multi-tissue predictor of age that allows one to estimate the DNA methylation age of most tissues and cell types. The predictor, which is freely available, was developed using 8,000 samples from 82 Illumina DNA methylation array datasets, encompassing 51 healthy tissues and cell types. I found that DNA methylation age has the following properties: first, it is close to zero for embryonic and induced pluripotent stem cells; second, it correlates with cell passage number; third, it gives rise to a highly heritable measure of age acceleration; and, fourth, it is applicable to chimpanzee tissues. Analysis of 6,000 cancer samples from 32 datasets showed that all of the considered 20 cancer types exhibit significant age acceleration, with an average of 36 years. Low age-acceleration of cancer tissue is associated with a high number of somatic mutations and TP53 mutations, while mutations in steroid receptors greatly accelerate DNA methylation age in breast cancer. Finally, I characterize the 353 CpG sites that together form an aging clock in terms of chromatin states and tissue variance. Conclusions: I propose that DNA methylation age measures the cumulative effect of an epigenetic maintenance system. This novel epigenetic clock can be used to address a host of questions in developmental biology, cancer and aging research.', 'Cytosine DNA methylation is a stable epigenetic mark that is crucial for diverse biological processes, including gene and transposon silencing, imprinting and X chromosome inactivation. Recent findings in plants and animals have greatly increased our understanding of the pathways used to accurately target, maintain and modify patterns of DNA methylation and have revealed unanticipated mechanistic similarities between these organisms. Key roles have emerged for small RNAs, proteins with domains that bind methylated DNA and DNA glycosylases in these processes. Drawing on insights from both plants and animals should deepen our understanding of the regulation and biological significance of DNA methylation.', 'A flexible statistical framework is developed for the analysis of read counts from RNA-Seq gene expression studies. It provides the ability to analyse complex experiments involving multiple treatment conditions and blocking variables while still taking full account of biological variation. Biological variation between RNA samples is estimated separately from the technical variation associated with sequencing technologies. Novel empirical Bayes methods allow each gene to have its own specific variability, even when there are relatively few biological replicates from which to estimate such variability. The pipeline is implemented in the edgeR package of the Bioconductor project. A case study analysis of carcinoma data demonstrates the ability of generalized linear model methods (GLMs) to detect differential expression in a paired design, and even to detect tumour-specific expression changes. The case study demonstrates the need to allow for gene-specific variability, rather than assuming a common dispersion across genes or a fixed relationship between abundance and variability. Genewise dispersions de-prioritize genes with inconsistent results and allow the main analysis to focus on changes that are consistent between biological replicates. Parallel computational approaches are developed to make non-linear model fitting faster and more reliable, making the application of GLMs to genomic data more convenient and practical. Simulations demonstrate the ability of adjusted profile likelihood estimators to return accurate estimators of biological variability in complex situations. When variation is gene-specific, empirical Bayes estimators provide an advantageous compromise between the extremes of assuming common dispersion or separate genewise dispersion. The methods developed here can also be applied to count data arising from DNA-Seq applications, including ChIP-Seq for epigenetic marks and DNA methylation analyses.', 'Cancer-associated IDH mutations are characterized by neomorphic enzyme activity and resultant 2-hydroxyglutarate (2HG) production. Mutational and epigenetic profiling of a large acute myeloid leukemia (AML) patient cohort revealed that IDH1/2-mutant AMLs display global DNA hypermethylation and a specific hypermethylation signature. Furthermore, expression of 2HG-producing IDH alleles in cells induced global DNA hypermethylation. In the AML cohort, IDH1/2 mutations were mutually exclusive with mutations in the alpha-ketoglutarate-dependent enzyme TET2, and TET2 loss-of-function mutations were associated with similar epigenetic defects as IDH1/2 mutants. Consistent with these genetic and epigenetic data, expression of IDH mutants impaired TET2 catalytic function in cells. Finally, either expression of mutant IDH1/2 or Tet2 depletion impaired hematopoietic differentiation and increased stem/progenitor cell marker expression, suggesting a shared proleukemogenic effect.', 'The epigenetic regulation of DNA-templated processes has been intensely studied over the last 15 years. DNA methylation, histone modification, nucleosome remodeling, and RNA-mediated targeting regulate many biological processes that are fundamental to the genesis of cancer. Here, we present the basic principles behind these epigenetic pathways and highlight the evidence suggesting that their misregulation can culminate in cancer. This information, along with the promising clinical and preclinical results seen with epigenetic drugs against chromatin regulators, signifies that it is time to embrace the central role of epigenetics in cancer.', 'DNA methylation is among the best studied epigenetic modifications and is essential to mammalian development. Although the methylation status of most CpG dinucteotides in the genome is stably propagated through mitosis, improvements to methods for measuring methylation have identified numerous regions in which it is dynamically regulated. In this Review, we discuss key concepts in the function of DNA methylation in mammals, stemming from more than two decades of research, including many recent studies that have elucidated when and where DNA methylation has a regulatory role in the genome. We include insights from early development, embryonic stem cells and adult Lineages, particularly haematopoiesis, to highlight the general features of this modification as it participates in both global and localized epigenetic regulation..', \"Methylation of the N-6 position of adenosine (m(6)A) is a posttranscriptional modification of RNA with poorly understood prevalence and physiological relevance. The recent discovery that FTO, an obesity risk gene, encodes an m(6)A demethylase implicates m(6)A as an important regulator of physiological processes. Here, we present a method for transcriptome-wide m(6)A localization, which combines m(6)A-specific methylated RNA immunoprecipitation with next-generation sequencing (MeRIP-Seq). We use this method to identify mRNAs of 7,676 mammalian genes that contain m(6)A, indicating that m(6)A is a common base modification of mRNA. The m(6)A modification exhibits tissue-specific regulation and is markedly increased throughout brain development. We find that m(6)A sites are enriched near stop codons and in 3' UTRs, and we uncover an association between m 6 A residues and microRNA-binding sites within 3' UTRs. These findings provide a resource for identifying transcripts that are substrates for adenosine methylation and reveal insights into the epigenetic regulation of the mammalian transcriptome.\", 'Chromatin regulators have become attractive targets for cancer therapy, but it is unclear why inhibition of these ubiquitous regulators should have gene-specific effects in tumor cells. Here, we investigate how inhibition of the widely expressed transcriptional coactivator BRD4 leads to selective inhibition of the MYC oncogene in multiple myeloma (MM). BRD4 and Mediator were found to co-occupy thousands of enhancers associated with active genes. They also co-occupied a small set of exceptionally large super-enhancers associated with genes that feature prominently in MM biology, including the MYC oncogene. Treatment of MM tumor cells with the BET-bromodomain inhibitor JQ1 led to preferential loss of BRD4 at super-enhancers and consequent transcription elongation defects that preferentially impacted genes with super-enhancers, including MYC. Super-enhancers were found at key oncogenic drivers in many other tumor cells. These observations have implications for the discovery of cancer therapeutics directed at components of super-enhancers in diverse tumor types.', \"The ability to measure human aging from molecular profiles has practical implications in many fields, including disease prevention and treatment, forensics, and extension of life. Although chronological age has been linked to changes in DNA methylation, the methylome has not yet been used to measure and compare human aging rates. Here, we build a quantitative model of aging using measurements at more than 450,000 CpG markers from the whole blood of 656 human individuals, aged 19 to 101. This model measures the rate at which an individual's methylome ages, which we show is impacted by gender and genetic variants. We also show that differences in aging rates help explain epigenetic drift and are reflected in the transcriptome. Moreover, we show how our aging model is upheld in other human tissues and reveals an advanced aging rate in tumor tissue. Our model highlights specific components of the aging process and provides a quantitative readout for studying the role of methylation in age-related disease.\", 'Organisms require an appropriate balance of stability and reversibility in gene expression programmes to maintain cell identity or to enable responses to stimuli; epigenetic regulation is integral to this dynamic control. Post-translational modification of histones by methylation is an important and widespread type of chromatin modification that is known to influence biological processes in the context of development and cellular responses. To evaluate how histone methylation contributes to stable or reversible control, we provide a broad overview of how histone methylation is regulated and leads to biological outcomes. The importance of appropriately maintaining or reprogramming histone methylation is illustrated by its links to disease and ageing and possibly to transmission of traits across generations.', 'Chromosomes are the physical realization of genetic information and thus form the basis for its readout and propagation. Here we present a high-resolution chromosomal contact map derived from a modified genome-wide chromosome conformation capture approach applied to Drosophila embryonic nuclei. The data show that the entire genome is linearly partitioned into well-demarcated physical domains that overlap extensively with active and repressive epigenetic marks. Chromosomal contacts are hierarchically organized between domains. Global modeling of contact density and clustering of domains show that inactive domains are condensed and confined to their chromosomal territories, whereas active domains reach out of the territory to form remote intra-and interchromosomal contacts. Moreover, we systematically identify specific long-range intrachromosomal contacts between Polycomb-repressed domains. Together, these observations allow for quantitative prediction of the Drosophila chromosomal contact map, laying the foundation for detailed studies of chromosome structure and function in a genetically tractable system.', 'Hybridization has many and varied impacts on the process of speciation. Hybridization may slow or reverse differentiation by allowing gene flow and recombination. It may accelerate speciation via adaptive introgression or cause near-instantaneous speciation by allopolyploidization. It may have multiple effects at different stages and in different spatial contexts within a single speciation event. We offer a perspective on the context and evolutionary significance of hybridization during speciation, highlighting issues of current interest and debate. In secondary contact zones, it is uncertain if barriers to gene flow will be strengthened or broken down due to recombination and gene flow. Theory and empirical evidence suggest the latter is more likely, except within and around strongly selected genomic regions. Hybridization may contribute to speciation through the formation of new hybrid taxa, whereas introgression of a few loci may promote adaptive divergence and so facilitate speciation. Gene regulatory networks, epigenetic effects and the evolution of selfish genetic material in the genome suggest that the DobzhanskyMuller model of hybrid incompatibilities requires a broader interpretation. Finally, although the incidence of reinforcement remains uncertain, this and other interactions in areas of sympatry may have knock-on effects on speciation both within and outside regions of hybridization.', 'Glioblastoma (GBM) is a brain tumor that carries a dismal prognosis and displays considerable heterogeneity. We have recently identified recurrent H3F3A mutations affecting two critical amino acids (K27 and G34) of histone H3.3 in one-third of pediatric GBM. Here, we show that each H3F3A mutation defines an epigenetic subgroup of GBM with a distinct global methylation pattern, and that they are mutually exclusive with IDH1 mutations, which characterize a third mutation-defined subgroup. Three further epigenetic subgroups were enriched for hallmark genetic events of adult GBM and/or established transcriptomic signatures. We also demonstrate that the two H3F3A mutations give rise to GBMs in separate anatomic compartments, with differential regulation of transcription factors OLIG1, OLIG2, and FOXG1, possibly reflecting different cellular origins.', 'Epigenetic phenomena in animals and plants are mediated by DNA methylation and stable chromatin modifications. There has been considerable interest in whether environmental factors modulate the establishment and maintenance of epigenetic modifications, and could thereby influence gene expression and phenotype. Chemical pollutants, dietary components, temperature changes and other external stresses can indeed have long-lasting effects on development, metabolism and health, sometimes even in subsequent generations. Although the underlying mechanisms remain largely unknown, particularly in humans, mechanistic insights are emerging from experimental model systems. These have implications for structuring future research and understanding disease and development.', 'Methylation of cytosine bases in DNA provides a layer of epigenetic control in many eukaryotes that has important implications for normal biology and disease. Therefore, profiling DNA methylation across the genome is vital to understanding the influence of epigenetics. There has been a revolution in DNA methylation analysis technology over the past decade: analyses that previously were restricted to specific loci can now be performed on a genome-scale and entire methylomes can be characterized at single-base-pair resolution. However, there is such a diversity of DNA methylation profiling techniques that it can be challenging to select one. This Review discusses the different approaches and their relative merits and introduces considerations for data analysis.', 'We have developed a new generation of genome-wide DNA methylation Bead Chip which allows high-throughput methylation profiling of the human genome. The new high density BeadChip can assay over 480K CpG sites and analyze twelve samples in parallel. The innovative content includes coverage of 99% of RefSeq genes with multiple probes per gene, 96% of CpG islands from the UCSC database, CpG island shores and additional content selected from whole-genome bisulfite sequencing data and input from DNA methylation experts. The well-characterized Infinium (R) Assay is used for analysis of CpG methylation using bisulfite-converted genomic DNA. We applied this technology to analyze DNA methylation in normal and tumor DNA samples and compared results with whole-genome bisulfite sequencing (WGBS) data obtained for the same samples. Highly comparable DNA methylation profiles were generated by the array and sequencing methods (average R-2 of 0.95). The ability to determine genome-wide methylation patterns will rapidly advance methylation research. (C) 2011 Elsevier Inc. All rights reserved.', 'Expression of the INK4b/ARF/INK4a tumor suppressor locus in normal and cancerous cell growth is controlled by methylation of histone H3 at lysine 27 (H3K27me) as directed by the Polycomb group proteins. The antisense noncoding RNA ANRIL of the INK4b/ARF/INK4a locus is also important for expression of the protein-coding genes in cis, but its mechanism has remained elusive. Here we report that chromobox 7 (CBX7) within the polycomb repressive complex 1 binds to ANRIL, and both CBX7 and ANRIL are found at elevated levels in prostate cancer tissues. In concert with H3K27me recognition, binding to RNA contributes to CBX7 function, and disruption of either interaction impacts the ability of CBX7 to repress the INK4b/ARF/INK4a locus and control senescence. Structure-guided analysis reveals the molecular interplay between noncoding RNA and H3K27me as mediated by the conserved chromodomain. Our study suggests a mechanism by which noncoding RNA participates directly in epigenetic transcriptional repression.', 'Determining how somatic copy number alterations (SCNAs) promote cancer is an important goal. We characterized SCNA patterns in 4,934 cancers from The Cancer Genome Atlas Pan-Cancer data set. Whole-genome doubling, observed in 37% of cancers, was associated with higher rates of every other type of SCNA, TP53 mutations, CCNE1 amplifications and alterations of the PPP2R complex. SCNAs that were internal to chromosomes tended to be shorter than telomere-bounded SCNAs, suggesting different mechanisms underlying their generation. Significantly recurrent focal SCNAs were observed in 140 regions, including 102 without known oncogene or tumor suppressor gene targets and 50 with significantly mutated genes. Amplified regions without known oncogenes were enriched for genes involved in epigenetic regulation. When levels of genomic disruption were accounted for, 7% of region pairs were anticorrelated, and these regions tended to encompass genes whose proteins physically interact, suggesting related functions. These results provide insights into mechanisms of generation and functional consequences of cancer-related SCNAs.', 'High-throughput technologies are widely used, for example to assay genetic variants, gene and protein expression, and epigenetic modifications. One often overlooked complication with such studies is batch effects, which occur because measurements are affected by laboratory conditions, reagent lots and personnel differences. This becomes a major problem when batch effects are correlated with an outcome of interest and lead to incorrect conclusions. Using both published studies and our own analyses, we argue that batch effects (as well as other technical and biological artefacts) are widespread and critical to address. We review experimental and computational approaches for doing so.', 'Cytosine methylation is the major covalent modification of mammalian genomic DNA and plays important roles in transcriptional regulation. The molecular mechanism underlying the enzymatic removal of this epigenetic mark, however, remains elusive. Here, we show that 5-methylcytosine (5mC) hydroxylase TET1, by converting 5mCs to 5-hydroxymethylcytosines (5hmCs), promotes DNA demethylation in mammalian cells through a process that requires the base excision repair pathway. Though expression of the 12 known human DNA glycosylases individually did not enhance removal of 5hmCs in mammalian cells, demethylation of both exogenously introduced and endogenous 5hmCs is promoted by the AID (activation-induced deaminase)/APOBEC (apolipoprotein B mRNA-editing enzyme complex) family of cytidine deaminases. Furthermore, Tet1 and Apobec1 are involved in neuronal activity-induced, region-specific, active DNA demethylation and subsequent gene expression in the dentate gyrus of the adult mouse brain in vivo. Our study suggests a TET1-induced oxidation-deamination mechanism for active DNA demethylation in mammals.', 'Nuclear factor-erythroid 2 p45-related factor 2 (Nrf2, also called Nfe212) is a transcription factor that regulates the cellular redox status. Nrf2 is controlled through a complex transcriptional/epigenetic and post-translational network that ensures its activity increases during redox perturbation, inflammation, growth factor stimulation and nutrient/energy fluxes, thereby enabling the factor to orchestrate adaptive responses to diverse forms of stress. Besides mediating stress-stimulated induction of antioxidant and detoxification genes, Nr12 contributes to adaptation by upregulating the repair and degradation of damaged macromolecules, and by modulating intermediary metabolism. In the latter case, Nrf2 inhibits lipogenesis, supports 0-oxidation of fatty acids, facilitates flux through the pentose phosphate pathway, and increases NADPH regeneration and purine biosynthesis; these observations suggest Nrf2 directs metabolic reprogramming during stress.', 'Genomes of complex organisms encode an abundance and diversity of long noncoding RNAs (IncRNAs) that are expressed throughout the cell and fulfill a wide variety of regulatory roles at almost every stage of gene expression. These roles, which encompass sensory, guiding, scaffolding and allosteric capacities, derive from folded modular domains in IncRNAs. In this diverse functional repertoire, we focus on the well-characterized ability for IncRNAs to function as epigenetic modulators. Many IncRNAs bind to chromatin-modifying proteins and recruit their catalytic activity to specific sites in the genome, thereby modulating chromatin states and impacting gene expression. Considering this regulatory potential in combination with the abundance of IncRNAs suggests that IncRNAs may be part of a broad epigenetic regulatory network.', 'Bromodomains (BRDs) are protein interaction modules that specifically recognize epsilon-N-lysine acetylation motifs, a key event in the reading process of epigenetic marks. The 61 BRDs in the human genome cluster into eight families based on structure/sequence similarity. Here, we present 29 high-resolution crystal structures, covering all BRD families. Comprehensive crossfamily structural analysis identifies conserved and family-specific structural features that are necessary for specific acetylation-dependent substrate recognition. Screening of more than 30 representative BRDs against systematic histone-peptide arrays identifies new BRD substrates and reveals a strong influence of flanking posttranslational modifications, such as acetylation and phosphorylation, suggesting that BRDs recognize combinations of marks rather than singly acetylated sequences. We further uncovered a structural mechanism for the simultaneous binding and recognition of diverse diacetyl-containing peptides by BRD4. These data provide a foundation for structure-based drug design of specific inhibitors for this emerging target family.', 'Epigenetics is one of the most promising and expanding fields in the current biomedical research landscape. Since the inception of epigenetics in the 1940s, the discoveries regarding its implications in normal and disease biology have not stopped, compiling a vast amount of knowledge in the past decade. The field has moved from just one recognized marker, DNA methylation, to a variety of others, including a wide spectrum of histone modifications. From the methodological standpoint, the successful initial single gene candidate approaches have been complemented by the current comprehensive epigenomic approaches that allow the interrogation of genomes to search for translational applications in an unbiased manner. Most important, the discovery of mutations in the epigenetic machinery and the approval of the first epigenetic drugs for the treatment of subtypes of leukemias and lymphomas has been an eye-opener for many biomedical scientists and clinicians. Herein, we will summarize the progress in the field of cancer epigenetics research that has reached mainstream oncology in the development of new biomarkers of the disease and new pharmacological strategies.', 'Abnormalities of chromosome 7q are common in myeloid malignancies, but no specific target genes have yet been identified. Here, we describe the finding of homozygous EZH2 mutations in 9 of 12 individuals with 7q acquired uniparental disomy. Screening of a total of 614 individuals with myeloid disorders revealed 49 monoallelic or biallelic EZH2 mutations in 42 individuals; the mutations were found most commonly in those with myelodysplastic/myeloproliferative neoplasms (27 out of 219 individuals, or 12%) and in those with myelofibrosis (4 out of 30 individuals, or 13%). EZH2 encodes the catalytic subunit of the polycomb repressive complex 2 (PRC2), a highly conserved histone H3 lysine 27 (H3K27) methyltransferase that influences stem cell renewal by epigenetic repression of genes involved in cell fate decisions. EZH2 has oncogenic activity, and its overexpression has previously been causally linked to differentiation blocks in epithelial tumors. Notably, the mutations we identified resulted in premature chain termination or direct abrogation of histone methyltransferase activity, suggesting that EZH2 acts as a tumor suppressor for myeloid malignancies.', 'With rare exceptions, spontaneous tumors originate from a single cell. Yet, at the time of clinical diagnosis, the majority of human tumors display startling heterogeneity in many morphological and physiological features, such as expression of cell surface receptors, proliferative and angiogenic potential. To a substantial extent, this heterogeneity might be attributed to morphological and epigenetic plasticity, but there is also strong evidence for the co-existence of genetically divergent tumor cell clones within tumors. In this perspective, we summarize the sources of intra-tumor phenotypic heterogeneity with emphasis on genetic heterogeneity. We review experimental evidence for the existence of both intra-tumor clonal heterogeneity as well as frequent evolutionary divergence between primary tumors and metastatic outgrowths. Furthermore, we discuss potential biological and clinical implications of intra-tumor clonal heterogeneity. (C) 2009 Elsevier B.V. All rights reserved.', \"Since the human genome was sequenced, the term ``epigenetics'' is increasingly being associated with the hope that we are more than just the sum of our genes. Might what we eat, the air we breathe, or even the emotions we feel influence not only our genes but those of descendants? The environment can certainly influence gene expression and can lead to disease, but transgenerational consequences are another matter. Although the inheritance of epigenetic characters can certainly occur-particularly in plants-how much is due to the environment and the extent to which it happens in humans remain unclear.\", 'The incorporation of histone H3 variants has been implicated in the epigenetic memory of cellular state. Using genome editing with zinc-finger nucleases to tag endogenous H3.3, we report genome-wide profiles of H3 variants in mammalian embryonic stem cells and neuronal precursor cells. Genome-wide patterns of H3.3 are dependent on amino acid sequence and change with cellular differentiation at developmentally regulated loci. The H3.3 chaperone Hira is required for H3.3 enrichment at active and repressed genes. Strikingly, Hira is not essential for localization of H3.3 at telomeres and many transcription factor binding sites. Immunoaffinity purification and mass spectrometry reveal that the proteins Atrx and Daxx associate with H3.3 in a Hira-independent manner. Atrx is required for Hira-independent localization of H3.3 at telomeres and for the repression of telomeric RNA. Our data demonstrate that multiple and distinct factors are responsible for H3.3 localization at specific genomic locations in mammalian cells.', 'Cyclin-dependent kinases (Cdks) are serine/threonine kinases and their catalytic activities are modulated by interactions with cyclins and Cdk inhibitors (CKIs). Close cooperation between this trio is necessary for ensuring orderly progression through the cell cycle. In addition to their well-established function in cell cycle control, it is becoming increasingly apparent that mammalian Cdks, cyclins and CKIs play indispensable roles in processes such as transcription, epigenetic regulation, metabolism, stem cell self-renewal, neuronal functions and spermatogenesis. Even more remarkably, they can accomplish some of these tasks individually, without the need for Cdk/cyclin complex formation or kinase activity. In this Review, we discuss the latest revelations about Cdks, cyclins and CKIs with the goal of showcasing their functional diversity beyond cell cycle regulation and their impact on development and disease in mammals.', 'During the course of malignant cancer progression, neoplastic cells undergo dynamic and reversible transitions between multiple phenotypic states, the extremes of which are defined by the expression of epithelial and mesenchymal phenotypes. This plasticity is enabled by underlying shifts in epigenetic regulation. A small cohort of pleiotropically acting transcription factors is widely recognized to effect these shifts by controlling the expression of a constituency of key target genes. These master regulators depend on complex epigenetic regulatory mechanisms, notably the induction of changes in the modifications of chromatin-associated histones, in order to achieve the widespread changes in gene expression observed during epithelial-mesenchymal transitions (EMTs). These associations indicate that an understanding of the functional interactions between such EMT-inducing transcription factors and the modulators of chromatin configuration will provide crucial insights into the fundamental mechanisms underlying cancer progression and may, in the longer term, generate new diagnostic and therapeutic modalities for treating high-grade malignancies.', 'Despite the success of genome-wide association studies (GWASs) in identifying loci associated with common diseases, a substantial proportion of the causality remains unexplained. Recent advances in genomic technologies have placed us in a position to initiate large-scale studies of human disease-associated epigenetic variation, specifically variation in DNA methylation. Such epigenome-wide association studies (EWASs) present novel opportunities but also create new challenges that are not encountered in GWASs. We discuss EWAS design, cohort and sample selections, statistical significance and power, confounding factors and follow-up studies. We also discuss how integration of EWASs with GWASs can help to dissect complex GWAS haplotypes for functional analysis.', 'Discoveries over the past decade portend a paradigm shift in molecular biology. Evidence suggests that RNA is not only functional as a messenger between DNA and protein but also involved in the regulation of genome organization and gene expression, which is increasingly elaborate in complex organisms. Regulatory RNA seems to operate at many levels; in particular, it plays an important part in the epigenetic processes that control differentiation and development. These discoveries suggest a central role for RNA in human evolution and ontogeny. Here, we review the emergence of the previously unsuspected world of regulatory RNA from a historical perspective.', 'DNA methylation, an important type of epigenetic modification in humans, participates in crucial cellular processes, such as embryonic development, X-inactivation, genomic imprinting and chromosome stability. Several platforms have been developed to study genome-wide DNA methylation. Many investigators in the field have chosen the Illumina Infinium HumanMethylation microarray for its ability to reliably assess DNA methylation following sodium bisulfite conversion. Here, we analyzed methylation profiles of 489 adult males and 357 adult females generated by the Infinium HumanMethylation450 microarray. Among the autosomal CpG sites that displayed significant methylation differences between the two sexes, we observed a significant enrichment of cross-reactive probes co-hybridizing to the sex chromosomes with more than 94% sequence identity. This could lead investigators to mistakenly infer the existence of significant autosomal sex-associated methylation. Using sequence identity cutoffs derived from the sex methylation analysis, we concluded that 6% of the array probes can potentially generate spurious signals because of co-hybridization to alternate genomic sequences highly homologous to the intended targets. Additionally, we discovered probes targeting polymorphic CpGs that overlapped SNPs. The methylation levels detected by these probes are simply the reflection of underlying genetic polymorphisms but could be misinterpreted as true signals. The existence of probes that are cross-reactive or of target polymorphic CpGs in the Illumina HumanMethylation microarrays can confound data obtained from such microarrays. Therefore, investigators should exercise caution when significant biological associations are found using these array platforms. A list of all cross-reactive probes and polymorphic CpGs identified by us are annotated in this paper.', 'Cancer therapy is challenged by the diversity of molecular implementations of oncogenic processes and by the resulting variation in therapeutic responses. Projects such as The Cancer Genome Atlas (TCGA) provide molecular tumor maps in unprecedented detail. The interpretation of these maps remains a major challenge. Here we distilled thousands of genetic and epigenetic features altered in cancers to similar to 500 selected functional events (SFEs). Using this simplified description, we derived a hierarchical classification of 3,299 TCGA tumors from 12 cancer types. The top classes are dominated by either mutations (M class) or copy number changes (C class). This distinction is clearest at the extremes of genomic instability, indicating the presence of different oncogenic processes. The full hierarchy shows functional event patterns characteristic of multiple cross-tissue groups of tumors, termed oncogenic signature classes. Targetable functional events in a tumor class are suggestive of class-specific combination therapy. These results may assist in the definition of clinical trials to match actionable oncogenic signatures with personalized therapies.', 'DNA methylation is one of the best-characterized epigenetic modifications and has been implicated in numerous biological processes, including transposable element silencing, genomic imprinting and X chromosome inactivation. Compared with other epigenetic modifications, DNA methylation is thought to be relatively stable. Despite its role in long-term silencing, DNA methylation is more dynamic than originally thought as active DNA demethylation has been observed during specific stages of development. In the past decade, many enzymes have been proposed to carry out active DNA demethylation and growing evidence suggests that, depending on the context, this process may be achieved by multiple mechanisms. Insight into how DNA methylation is dynamically regulated will broaden our understanding of epigenetic regulation and have great implications in somatic cell reprogramming and regenerative medicine.', 'DNA methylation is one of the most intensely studied epigenetic modifications in mammals. In normal cells, it assures the proper regulation of gene expression and stable gene silencing. DNA methylation is associated with histone modifications and the interplay of these epigenetic modifications is crucial to regulate the functioning of the genome by changing chromatin architecture. The covalent addition of a methyl group occurs generally in cytosine within CpG dinucleotides which are concentrated in large clusters called CpG islands. DNA methyltransferases are responsible for establishing and maintenance of methylation pattern. It is commonly known that inactivation of certain tumor-suppressor genes occurs as a consequence of hypermethylation within the promoter regions and a numerous studies have demonstrated a broad range of genes silenced by DNA methylation in different cancer types. On the other hand, global hypomethylation, inducing genomic instability, also contributes to cell transformation. Apart from DNA methylation alterations in promoter regions and repetitive DNA sequences, this phenomenon is associated also with regulation of expression of noncoding RNAs such as microRNAs that may play role in tumor suppression. DNA methylation seems to be promising in putative translational use in patients and hypermethylated promoters may serve as biomarkers. Moreover, unlike genetic alterations, DNA methylation is reversible what makes it extremely interesting for therapy approaches. The importance of DNA methylation alterations in tumorigenesis encourages us to decode the human epigenome. Different DNA methylome mapping techniques are indispensable to realize this project in the future. (C) 2010, Elsevier Inc.', 'Epigenetic information can be inherited through the mammalian germline and represents a plausible transgenerational carrier of environmental information. To test whether transgenerational inheritance of environmental information occurs in mammals, we carried out an expression profiling screen for genes in mice that responded to paternal diet. Offspring of males fed a low-protein diet exhibited elevated hepatic expression of many genes involved in lipid and cholesterol biosynthesis and decreased levels of cholesterol esters, relative to the offspring of males fed a control diet. Epigenomic profiling of offspring livers revealed numerous modest (similar to 20%) changes in cytosine methylation depending on paternal diet, including reproducible changes in methylation over a likely enhancer for the key lipid regulator Ppara. These results, in conjunction with recent human epidemiological data, indicate that parental diet can affect cholesterol and lipid metabolism in offspring and define a model system to study environmental reprogramming of the heritable epigenome.', 'Induction of cell death and inhibition of cell survival are the main principles of cancer therapy. Resistance to chemotherapeutic agents is a major problem in oncology, which limits the effectiveness of anticancer drugs. A variety of factors contribute to drug resistance, including host factors, specific genetic or epigenetic alterations in the cancer cells and so on. Although various mechanisms by which cancer cells become resistant to anticancer drugs in the microenvironment have been well elucidated, how to circumvent this resistance to improve anticancer efficacy remains to be defined. Autophagy, an important homeostatic cellular recycling mechanism, is now emerging as a crucial player in response to metabolic and therapeutic stresses, which attempts to maintain/restore metabolic homeostasis through the catabolic lysis of excessive or unnecessary proteins and injured or aged organelles. Recently, several studies have shown that autophagy constitutes a potential target for cancer therapy and the induction of autophagy in response to therapeutics can be viewed as having a prodeath or a prosurvival role, which contributes to the anticancer efficacy of these drugs as well as drug resistance. Thus, understanding the novel function of autophagy may allow us to develop a promising therapeutic strategy to enhance the effects of chemotherapy and improve clinical outcomes in the treatment of cancer patients.', 'DNA methylation is a critical epigenetic regulator in mammalian development. Here, we present a whole-genome comparative view of DNA methylation using bisulfite sequencing of three cultured cell types representing progressive stages of differentiation: human embryonic stem cells (hESCs), a fibroblastic differentiated derivative of the hESCs, and neonatal fibroblasts. As a reference, we compared our maps with a methylome map of a fully differentiated adult cell type, mature peripheral blood mononuclear cells (monocytes). We observed many notable common and cell-type-specific features among all cell types. Promoter hypomethylation (both CG and CA) and higher levels of gene body methylation were positively correlated with transcription in all cell types. Exons were more highly methylated than introns, and sharp transitions of methylation occurred at exon-intron boundaries, suggesting a role for differential methylation in transcript splicing. Developmental stage was reflected in both the level of global methylation and extent of non-CpG methylation, with hESC highest, fibroblasts intermediate, and monocytes lowest. Differentiation-associated differential methylation profiles were observed for developmentally regulated genes, including the HOX clusters, other homeobox transcription factors, and pluripotence-associated genes such as POU5F1, TCF3, and KLF4. Our results highlight the value of high-resolution methylation maps, in conjunction with other systems-level analyses, for investigation of previously undetectable developmental regulatory mechanisms.', 'RNA-directed DNA methylation (RdDM) is the major small RNA-mediated epigenetic pathway in plants. RdDM requires a specialized transcriptional machinery that comprises two plant-specific RNA polymerases - Pot IV and Pot V - and a growing number of accessory proteins, the functions of which in the RdDM mechanism are only partially understood. Recent work has revealed variations in the canonical RdDM pathway and identified factors that recruit Pot IV and Pol V to specific target sequences. RdDM, which transcriptionally represses a subset of transposons and genes, is implicated in pathogen defence, stress responses and reproduction, as well as in interallelic and intercellular communication.', \"Tumor heterogeneity is a major barrier to effective cancer diagnosis and treatment. We recently identified cancer-specific differentially DNA-methylated regions (cDMRs) in colon cancer, which also distinguish normal tissue types from each other, suggesting that these cDMRs might be generalized across cancer types. Here we show stochastic methylation variation of the same cDMRs, distinguishing cancer from normal tissue, in colon, lung, breast, thyroid and Wilms' tumors, with intermediate variation in adenomas. Whole-genome bisulfite sequencing shows these variable cDMRs are related to loss of sharply delimited methylation boundaries at CpG islands. Furthermore, we find hypomethylation of discrete blocks encompassing half the genome, with extreme gene expression variability. Genes associated with the cDMRs and large blocks are involved in mitosis and matrix remodeling, respectively. We suggest a model for cancer involving loss of epigenetic stability of well-defined genomic domains that underlies increased methylation variability in cancer that may contribute to tumor heterogeneity.\", 'Emergence of form and function during embryogenesis arises in large part through cell-type- and cell-state-specific variation in gene expression patterns, mediated by specialized cis-regulatory elements called enhancers. Recent large-scale epigenomic mapping revealed unexpected complexity and dynamics of enhancer utilization patterns, with 400,000 putative human enhancers annotated by the ENCODE project alone. These large-scale efforts were largely enabled through the understanding that enhancers share certain stereotypical chromatin features. However, an important question still lingers: what is the functional significance of enhancer chromatin modification? Here we give an overview of enhancer-associated modifications of histones and DNA and discuss enzymatic activities involved in their dynamic deposition and removal. We describe potential downstream effectors of these marks and propose models for exploring functions of chromatin modification in regulating enhancer activity during development.', 'The generation of properly functioning gametes in vitro requires reconstitution of the multistepped pathway of germ cell development. We demonstrate here the generation of primordial germ cell-like cells (PGCLCs) in mice with robust capacity for spermatogenesis. PGCLCs were generated from embryonic stem cells (ESCs) and induced pluripotent stem cells (iPSCs) through epiblast-like cells (EpiLCs), a cellular state highly similar to pregastrulating epiblasts but distinct from epiblast stem cells (EpiSCs). Reflecting epiblast development, EpiLC induction from ESCs/iPSCs is a progressive process, and EpiLCs highly competent for the PGC fate are a transient entity. The global transcription profiles, epigenetic reprogramming, and cellular dynamics during PGCLC induction from EpiLCs meticulously capture those associated with PGC specification from the epiblasts. Furthermore, we identify Integrin-beta 3 and SSEA1 as markers that allow the isolation of PGCLCs with spermatogenic capacity from tumorigenic undifferentiated cells. Our findings provide a paradigm for the first step of in vitro gametogenesis.', 'Observational epidemiological studies are prone to confounding, reverse causation and various biases and have generated findings that have proved to be unreliable indicators of the causal effects of modifiable exposures on disease outcomes. Mendelian randomization (MR) is a method that utilizes genetic variants that are robustly associated with such modifiable exposures to generate more reliable evidence regarding which interventions should produce health benefits. The approach is being widely applied, and various ways to strengthen inference given the known potential limitations of MR are now available. Developments of MR, including two-sample MR, bidirectional MR, network MR, two-step MR, factorial MR and multiphenotype MR, are outlined in this review. The integration of genetic information into population-based epidemiological studies presents translational opportunities, which capitalize on the investment in genomic discovery research.', 'Successful reprogramming of differentiated human somatic cells into a pluripotent state would allow creation of patient- and disease-specific stem cells. We previously reported generation of induced pluripotent stem (iPS) cells, capable of germline transmission, from mouse somatic cells by transduction of four defined transcription factors. Here, we demonstrate the generation of iPS cells from adult human dermal fibroblasts with the same four factors: Oct3/4, Sox2, Klf4, and c-Myc. Human iPS cells were similar to human embryonic stem (ES) cells in morphology, proliferation, surface antigens, gene expression, epigenetic status of pluripotent cell-specific genes, and telomerase activity. Furthermore, these cells could differentiate into cell types of the three germ layers in vitro and in teratomas. These findings demonstrate that iPS cells can be generated from adult human fibroblasts.', 'The most highly conserved noncoding elements (HCNEs) in mammalian genomes cluster within regions enriched for genes encoding developmentally important transcription factors (TFs). This suggests that HCNE-rich regions may contain key regulatory controls involved in development. We explored this by examining histone methylation in mouse embryonic stem (ES) cells across 56 large HCNE-rich loci. We identified a specific modification pattern, termed bivalent domains, consisting of large regions of H3 lysine 27 methylation harboring smaller regions of H3 lysine 4 methylation. Bivalent domains tend to coincide with TF genes expressed at low levels. We propose that bivalent domains silence developmental genes in ES cells while keeping them poised for activation. We also found striking correspondences between genome sequence and histone methylation in ES cells, which become notably weaker in differentiated cells. These results highlight the importance of DNA sequence in defining the initial epigenetic landscape and suggest a novel chromatin-based mechanism for maintaining pluripotency.', \"RNA is not only a messenger operating between DNA and protein. Transcription of essentially the entire eukaryotic genome generates a myriad of non-protein-coding RNA species that show complex overlapping patterns of expression and regulation. Although long noncoding RNAs (lncRNAs) are among the least well-understood of these transcript species, they cannot all be dismissed as merely transcriptional noise.'' Here, we review the evolution of lncRNAs and their roles in transcriptional regulation, epigenetic gene regulation, and disease.\", 'Noncoding RNAs (ncRNA) participate in epigenetic regulation but are poorly understood. Here we characterize the transcriptional landscape of the four human HOX loci at five base pair resolution in 11 anatomic sites and identify 231 HOX ncRNAs that extend known transcribed regions by more than 30 kilobases. HOX ncRNAs are spatially expressed along developmental axes and possess unique sequence motifs, and their expression demarcates broad chromosomal domains of differential histone methylation and RNA polymerase accessibility. We identified a 2.2 kilobase ncRNA residing in the HOXC locus, termed HOTAIR, which represses transcription in trans across 40 kilobases of the HOXD locus. HOTAIR interacts with Polycomb Repressive Complex 2 (PRC2) and is required for PRC2 occupancy and histone H3 lysine-27 trimethylation of HOXD locus. Thus, transcription of ncRNA may demarcate chromosomal domains of gene silencing at a distance; these results have broad implications for gene regulation in development and disease states.', 'Over the past several years it has become clear that alterations in the expression of microRNA (miRNA) genes contribute to the pathogenesis of most - if not all - human malignancies. These alterations can be caused by various mechanisms, including deletions, amplifications or mutations involving miRNA loci, epigenetic silencing or the dysregulation of transcription factors that target specific miRNAs. Because malignant cells show dependence on the dysregulated expression of miRNA genes, which in turn control or are controlled by the dysregulation of multiple protein-coding oncogenes or tumour suppressor genes, these small RNAs provide important opportunities for the development of future miRNA-based therapies.', 'The genomes of many animals, plants and fungi are tagged by methylation of DNA cytosine. To understand the biological significance of this epigenetic mark it is essential to know where in the genome it is located. New techniques are making it easier to map DNA methylation patterns on a large scale and the results have already provided surprises. In particular, the conventional view that DNA methylation functions predominantly to irreversibly silence transcription is being challenged. Not only is promoter methylation often highly dynamic during development, but many organisms also seem to target DNA methylation specifically to the bodies of active genes.', 'DNA methylation is a crucial epigenetic modification of the genome that is involved in regulating many cellular processes. These include embryonic development, transcription, chromatin structure, X chromosome inactivation, genomic imprinting and chromosome stability. Consistent with these important roles, a growing number of human diseases have been found to be associated with aberrant DNA methylation. The study of these diseases has provided new and fundamental insights into the roles that DNA methylation and other epigenetic modifications have in development and normal cellular homeostasis.', 'Methylation of DNA at position five of the cytosine ring occurs at most CpG dinucleotides in the mammalian genome and is essential for embryonic viability. With several of the key proteins now known, it has become possible to approach the biological significance of this epigenetic system through both biochemistry and genetics. As a result, advances have been made in our understanding of the mechanisms by which DNA methylation is targeted to specific regions of the genome and interpreted by methyl-CpG-binding proteins. Recent studies have illuminated the role of DNA methylation in controlling gene expression and have strengthened its links with histone modification and chromatin remodelling.', 'Epidemiological evidence increasingly suggests that environmental exposures early in development have a role in susceptibility to disease in later life. In addition, some of these environmental effects seem to be passed on through subsequent generations. Epigenetic modifications provide a plausible link between the environment and alterations in gene expression that might lead to disease phenotypes. An increasing body of evidence from animal studies supports the role of environmental epigenetics in disease susceptibility. Furthermore, recent studies have demonstrated for the first time that heritable environmentally induced epigenetic modifications underlie reversible transgenerational alterations in phenotype. Methods are now becoming available to investigate the relevance of these phenomena to human disease.', 'Deciphering the multiple layers of epigenetic regulation that control transcription is critical to understanding how plants develop and respond to their environment. Using sequencing-by-synthesis technology we directly sequenced the cytosine methylome (methylC-seq), transcriptome (mRNA-seq), and small RNA transcriptome (smRNA-seq) to generate highly integrated epigenome maps for wild-type Arabidopsis thaliana and mutants defective in DNA methyltransferase or demethylase activity. At singlebase resolution we discovered extensive, previously undetected DNA methylation, identified the context and level of methylation at each site, and observed local sequence effects upon methylation state. Deep sequencing of smRNAs revealed a direct relationship between the location of smRNAs and DNA methylation, perturbation of smRNA biogenesis upon loss of CpG DNA methylation, and a tendency for smRNAs to direct strand-specific DNA methylation in regions of RNA-DNA homology. Finally, strandspecific mRNA-seq revealed altered transcript abundance of hundreds of genes, transposons, and unannotated intergenic transcripts upon modification of the DNA methylation state.', \"Plants and animals use small RNAs (microRNAs [miRNAs] and siRNAs) as guides for posttranscriptional and epigenetic regulation. In plants, miRNAs and trans-acting (ta) siRNAs form through distinct biogenesis pathways, although they both interact with target transcripts and guide cleavage. An integrated approach to identify targets of Arabidopsis thaliana miRNAs and ta-siRNAs revealed several new classes of small RNA-regulated genes, including conventional genes such as Argonaute2 and an E2-ubiquitin conjugating enzyme. Surprisingly, five tasiRNA-generating transcripts were identified as targets of miR173 or miR390. Rather than functioning as negative regulators, miR173- and miR390-guided cleavage was shown to set the 21-nucleotide phase for ta-siRNA precursor processing. These data support a model in which miRNA-guided formation of a 5' or 3' terminus within pre-ta-siRNA transcripts, followed by RDR6-dependent formation of dsRNA and Dicer-like processing, yields phased ta-siRNAs that negatively regulate other genes.\", \"For the past 25 years, it has been known that alterations in DNA methylation (DNAm) occur in cancer, including hypomethylation of oncogenes and hypermethylation of tumor suppressor genes. However, most studies of cancer methylation have assumed that functionally important DNAm will occur in promoters, and that most DNAm changes in cancer occur in CpG islands. Here we show that most methylation alterations in colon cancer occur not in promoters, and also not in CpG islands, but in sequences up to 2 kb distant, which we term 'CpG island shores'. CpG island shore methylation was strongly related to gene expression, and it was highly conserved in mouse, discriminating tissue types regardless of species of origin. There was a notable overlap (45-65%) of the locations of colon cancer-related methylation changes with those that distinguished normal tissues, with hypermethylation enriched closer to the associated CpG islands, and hypomethylation enriched further from the associated CpG island and resembling that of noncolon normal tissues. Thus, methylation changes in cancer are at sites that vary normally in tissue differentiation, consistent with the epigenetic progenitor model of cancer, which proposes that epigenetic alterations affecting tissue-specific differentiation are the predominant mechanism by which epigenetic changes cause cancer.\", \"An altered pattern of epigenetic modifications is central to many common human diseases, including cancer. Many studies have explored the mosaic patterns of DNA methylation and histone modification in cancer cells on a gene-by-gene basis; among their results has been the seminal finding of transcriptional silencing of tumour-suppressor genes by CpG-island-promoter hypermethylation. However, recent technological advances are now allowing cancer epigenetics to be studied genome-wide-an approach that has already begun to provide both biological insight and new avenues for translational research. It is time to 'upgrade' cancer epigenetics research and put together an ambitious plan to tackle the many unanswered questions in this field using epigenomics approaches.\", 'The term non-coding RNA (ncRNA) is commonly employed for RNA that does not encode a protein, but this does not mean that such RNAs do not contain information nor have function. Although it has been generally assumed that most genetic information is transacted by proteins, recent evidence suggests that the majority of the genomes of mammals and other complex organisms is in fact transcribed into ncRNAs, many of which are alternatively spliced and/or processed into smaller products. These ncRNAs include microRNAs and snoRNAs (many if not most of which remain to be identified), as well as likely other classes of yet-to-be-discovered small regulatory RNAs, and tens of thousands of longer transcripts (including complex patterns of interlacing and overlapping sense and antisense transcripts), most of whose functions are unknown. These RNAs (including those derived from introns) appear to comprise a hidden layer of internal signals that control various levels of gene expression in physiology and development, including chromatin architecture/epigenetic memory, transcription, RNA splicing, editing, translation and turnover. RNA regulatory networks may determine most of our complex characteristics, play a significant role in disease and constitute an unexplored world of genetic variation both within and between species.', ' ', 'X-chromosome inactivation is the mammalian dosage compensation mechanism by which transcription of X-linked genes is equalized between females and males. In an N-ethyl-N-nitrosourea (ENU) mutagenesis screen on mice for modifiers of epigenetic reprogramming, we identified the MommeD1 (modifier of murine metastable epialleles) mutation as a semidominant suppressor of variegation. MommeD1 shows homozygous female-specific mid-gestation lethality and hypomethylation of the X-linked gene Hprt1, suggestive of a defect in X inactivation(1). Here we report that the causative point mutation lies in a previously uncharacterized gene, Smchd1 (structural maintenance of chromosomes hinge domain containing 1). We find that SmcHD1 is not required for correct Xist expression, but localizes to the inactive X and has a role in the maintenance of X inactivation and the hypermethylation of CpG islands associated with the inactive X. This finding links a group of proteins normally associated with structural aspects of chromosome biology with epigenetic gene silencing.', ' ', 'The packaging of chromosomal DNA by nucleosomes condenses and organizes the genome, but occludes many regulatory DNA elements. However, this constraint also allows nucleosomes and other chromatin components to actively participate in the regulation of transcription, chromosome segregation, DNA replication, and DNA repair. To enable dynamic access to packaged DNA and to tailor nucleosome composition in chromosomal regions, cells have evolved a set of specialized chromatin remodeling complexes (remodelers). Remodelers use the energy of ATP hydrolysis to move, destabilize, eject, or restructure nucleosomes. Here, we address many aspects of remodeler biology: their targeting, mechanism, regulation, shared and unique properties, and specialization for particular biological processes. We also address roles for remodelers in development, cancer, and human syndromes.', 'Ectopic expression of the four transcription factors Oct4, Sox2, c-Myc, and Klf4 is sufficient to confer a pluripotent state upon the fibroblast genome, generating induced pluripotent stem (PS) cells. It remains unknown if nuclear reprogramming induced by these four factors globally resets epigenetic differences between differentiated and pluripotent cells. Here, using novel selection approaches, we have generated PS cells from fibroblasts to characterize their epigenetic state. Female PS cells showed reactivation of a somatically silenced X chromosome and underwent random X inactivation upon differentiation. Genome-wide analysis of two key histone modifications indicated that PS cells are highly similar to ES cells. Consistent with these observations, PS cells gave rise to viable high-degree chimeras with contribution to the germline. These data show that transcription factor-induced reprogramming leads to the global reversion of the somatic epigenome into an ES-like state. Our results provide a paradigm for studying the epigenetic modifications that accompany nuclear reprogramming and suggest that abnormal epigenetic reprogramming does not pose a problem for the potential therapeutic applications of PS cells.', \"Cancer is widely perceived as a heterogeneous group of disorders with markedly different biological properties, which are caused by a series of clonally selected genetic changes in key tumour-suppressor genes and oncogenes. However, recent data suggest that cancer has a fundamentally common basis that is grounded in a polyclonal epigenetic disruption of stem/progenitor cells, mediated by 'tumour-progenitor genes'. Furthermore, tumour cell heterogeneity is due in part to epigenetic variation in progenitor cells, and epigenetic plasticity together with genetic lesions drives tumour progression. This crucial early role for epigenetic alterations in cancer is in addition to epigenetic alterations that can substitute for genetic variation later in tumour progression. Therefore, non-neoplastic but epigenetically disrupted stem/progenitor cells might be a crucial target for cancer risk assessment and chemoprevention.\", 'Polyploids - organisms that have multiple sets of chromosomes - are common in certain plant and animal taxa, and can be surprisingly stable. The evidence that has emerged from genome analyses also indicates that many other eukaryotic genomes have a polyploid ancestry, suggesting that both humans and most other eukaryotes have either benefited from or endured polyploidy. Studies of polyploids soon after their formation have revealed genetic and epigenetic interactions between redundant genes. These interactions can be related to the phenotypes and evolutionary fates of polyploids. Here, I consider the advantages and challenges of polyploidy, and its evolutionary potential.', 'CpG island hypermethylation and global genomic hypomethylation are common epigenetic features of cancer cells. Less attention has been focused on histone modifications in cancer cells. We characterized post-translational modifications to histone H4 in a comprehensive panel of normal tissues, cancer cell lines and primary tumors. Using immunodetection, high-performance capillary electrophoresis and mass spectrometry, we found that cancer cells had a loss of monoacetylated and trimethylated forms of histone H4. These changes appeared early and accumulated during the tumorigenic process, as we showed in a mouse model of multistage skin carcinogenesis. The losses occurred predominantly at the acetylated Lys16 and trimethylated Lys20 residues of histone H4 and were associated with the hypomethylation of DNA repetitive sequences, a well-known characteristic of cancer cells. Our data suggest that the global loss of monoacetylation and trimethylation of histone H4 is a common hallmark of human tumor cells.', 'Cytosine methylation is required for mammalian development and is often perturbed in human cancer. To determine how this epigenetic modification is distributed in the genomes of primary and transformed cells, we used an immunocapturing approach followed by DNA microarray analysis to generate methylation profiles of all human chromosomes at 80-kb resolution and for a large set of CpG islands. In primary cells we identified broad genomic regions of differential methylation with higher levels in gene-rich neighborhoods. Female and male cells had indistinguishable profiles for autosomes but differences on the X chromosome. The inactive X chromosome ( Xi) was hypermethylated at only a subset of gene-rich regions and, unexpectedly, overall hypomethylated relative to its active counterpart. The chromosomal methylation profile of transformed cells was similar to that of primary cells. Nevertheless, we detected large genomic segments with hypomethylation in the transformed cell residing in gene-poor areas. Furthermore, analysis of 6,000 CpG islands showed that only a small set of promoters was methylated differentially, suggesting that aberrant methylation of CpG island promoters in malignancy might be less frequent than previously hypothesized.', 'Inheritance requires genome duplication, reproduction of chromatin and its epigenetic information, mechanisms to ensure genome integrity, and faithful transmission of the information to progeny. Proliferating cell nuclear antigen (PCNA)-a cofactor of DNA polymerases that encircles DNA-orchestrates several of these functions by recruiting crucial players to the replication fork. Remarkably, many factors that are involved in replication-linked processes interact with a particular face of PCNA and through the same interaction domain, indicating that these interactions do not occur simultaneously during replication. Switching of PCNA partners may be triggered by affinity-driven competition, phosphorylation, proteolysis, and modification of PCNA by ubiquitin and SUMO.', 'Cytosine methylation is important for transposon silencing and epigenetic regulation of endogenous genes, although the extent to which this DNA modification functions to regulate the genome is still unknown. Here we report the first comprehensive DNA methylation map of an entire genome, at 35 base pair resolution, using the flowering plant Arabidopsis thaliana as a model. We find that pericentromeric heterochromatin, repetitive sequences, and regions producing small interfering RNAs are heavily methylated. Unexpectedly, over one-third of expressed genes contain methylation within transcribed regions, whereas only similar to 5% of genes show methylation within promoter regions. Interestingly, genes methylated in transcribed regions are highly expressed and constitutively active, whereas promoter-methylated genes show a greater degree of tissue-specific expression. Whole-genome tiling-array transcriptional profiling of DNA methyltransferase null mutants identified hundreds of genes and intergenic noncoding RNAs with altered expression levels, many of which may be epigenetically controlled by DNA methylation.', \"Overlapping epigenetic mechanisms have evolved in eukaryotic cells to silence the expression and mobility of transposable elements (TEs). Owing to their ability to recruit the silencing machinery, TEs have served as building blocks for epigenetic phenomena, both at the level of single genes and across larger chromosomal regions. Important progress has been made recently in understanding these silencing mechanisms. In addition, new insights have been gained into how this silencing has been co-opted to serve essential functions in 'host' cells, highlighting the importance of TEs in the epigenetic regulation of the genome.\", 'Cellular senescence, a state of irreversible growth arrest, can be triggered by multiple mechanisms including telomere shortening, the epigenetic derepression of the INK4a/ARF locus, and DNA damage. Together these mechanisms limit excessive or aberrant cellular proliferation, and so the state of senescence protects against the development of cancer. Recent evidence suggests that cellular senescence also may be involved in aging.', \"Histones comprise the major protein component of chromatin, the scaffold in which the eukaryotic genome is packaged, and are subject to many types of post-translational modifications (PTMs), especially on their flexible tails. These modifications may constitute a 'histone code' and could be used to manage epigenetic information that helps extend the genetic message beyond DNA sequences. This proposed code, read in part by histone PTM-binding 'effector' modules and their associated complexes, is predicted to define unique functional states of chromatin and/or regulate various chromatin-templated processes. A wealth of structural and functional data show how chromatin effector modules target their cognate covalent histone modifications. Here we summarize key features in molecular recognition of histone PTMs by a diverse family of 'reader pockets', highlighting specific readout mechanisms for individual marks, common themes and insights into the downstream functional consequences of the interactions. Changes in these interactions may have far-reaching implications for human biology and disease, notably cancer.\", 'Chromatin immunoprecipitation followed by sequencing (ChIP-seq) is a technique for genome-wide profiling of DNA-binding proteins, histone modifications or nucleosomes. Owing to the tremendous progress in next-generation sequencing technology, ChIP-seq offers higher resolution, less noise and greater coverage than its array-based predecessor ChIP-chip. With the decreasing cost of sequencing, ChIP-seq has become an indispensable tool for studying gene regulation and epigenetic mechanisms. In this Review, I describe the benefits and challenges in harnessing this technique with an emphasis on issues related to experimental design and data analysis. ChIP-seq experiments generate large quantities of data, and effective computational analysis will be crucial for uncovering biological mechanisms.', 'Inflammation is linked clinically and epidemiologically to cancer, and NF-kappa B appears to play a causative role, but the mechanisms are poorly understood. We show that transient activation of Src oncoprotein can mediate an epigenetic switch from immortalized breast cells to a stably transformed line that forms self-renewing mammospheres that contain cancer stem cells. Src activation triggers an inflammatory response mediated by NF-kappa B that directly activates Lin28 transcription and rapidly reduces let-7 microRNA levels. Let-7 directly inhibits IL6 expression, resulting in higher levels of IL6 than achieved by NF-kappa B activation. IL6-mediated activation of the STAT3 transcription factor is necessary for transformation, and IL6 activates NF-kappa B, thereby completing a positive feedback loop. This regulatory circuit operates in other cancer cells lines, and its transcriptional signature is found in human cancer tissues. Thus, inflammation activates a positive feedback loop that maintains the epigenetic transformed state for many generations in the absence of the inducing signal.', \"Expression profiling of T24 cells revealed that 17 out of 313 human miRNAs were upregulated more than 3-fold by simultaneous treatment with the chromatin-modifying drugs 5-aza-2'-deoxycytidine and 4-phenylbutyric acid. One of these, miR-127, is embedded in a CpG island and is highly induced from its own promoter after treatment. miR-127 is usually expressed as part of a miRNA cluster in normal cells but not in cancer cells, suggesting that it is subject to epigenetic silencing. In addition, the proto-oncogene BCL6, a potential target of miR-127, was translationally downregulated after treatment. These results suggest that DNA demethylation and histone deacetylase inhibition can activate expression of miRNAs that may act as tumor suppressors.\", 'Polycomb proteins form chromatin-modifying complexes that implement transcriptional silencing in higher eukaryotes. Hundreds of genes are silenced by Polycomb proteins, including dozens of genes that encode crucial developmental regulators in organisms ranging from plants to humans. Two main families of complexes, called Polycomb repressive complex 1 (PRC1) and PRC2, are targeted to repressed regions. Recent studies have advanced our understanding of these complexes, including their potential mechanisms of gene silencing, the roles of chromatin modifications, their means of delivery to target genes and the functional distinctions among variant complexes. Emerging concepts include the existence of a Polycomb barrier to transcription elongation and the involvement of non-coding RNAs in the targeting of Polycomb complexes. These findings have an impact on the epigenetic programming of gene expression in many biological systems.', \"A recent meeting (December 2008) regarding chromatin-based epigenetics was hosted by the Banbury Conference Center and Cold Spring Harbor Laboratory. The intent was to discuss aspects of epigenetic control of genomic function, and to arrive at a consensus definition of epigenetics'' to be considered by the broader community. It was evident that multiple mechanistic steps lead to the stable heritance of the epigenetic phenotype. Below we provide our view and interpretation of the proceedings at the meeting.\", 'Epigenetic genome modifications are thought to be important for specifying the lineage and developmental stage of cells within a multicellular organism. Here, we show that the epigenetic profile of pluripotent embryonic stem cells (ES) is distinct from that of embryonic carcinoma cells, haematopoietic stem cells (HSC) and their differentiated progeny. Silent, lineage-specific genes replicated earlier in pluripotent cells than in tissue-specific stem cells or differentiated cells and had unexpectedly high levels of acetylated H3K9 and methylated H3K4. Unusually, in ES cells these markers of open chromatin were also combined with H3K27 trimethylation at some non-expressed genes. Thus, pluripotency of ES cells is characterized by a specific epigenetic profile where lineage-specific genes may be accessible but, if so, carry repressive H3K27 trimethylation modifications. H3K27 methylation is functionally important for preventing expression of these genes in ES cells as premature expression occurs in embryonic ectoderm development (Eed)-deficient ES cells. Our data suggest that lineage-specific genes are primed for expression in ES cells but are held in check by opposing chromatin modifications.', \"DNA methylation is the most stable type of epigenetic modification modulating the transcriptional plasticity of mammalian genomes. Using bisulfite DNA sequencing, we report high-resolution methylation profiles of human chromosomes 6, 20 and 22, providing a resource of about 1.9 million CpG methylation values derived from 12 different tissues. Analysis of six annotation categories showed that evolutionarily conserved regions are the predominant sites for differential DNA methylation and that a core region surrounding the transcriptional start site is an informative surrogate for promoter methylation. We find that 17% of the 873 analyzed genes are differentially methylated in their 5 ' UTRs and that about one-third of the differentially methylated 5 ' UTRs are inversely correlated with transcription. Despite the fact that our study controlled for factors reported to affect DNA methylation such as sex and age, we did not find any significant attributable effects. Our data suggest DNA methylation to be ontogenetically more stable than previously thought.\", 'The Polycomb group (PcG) proteins form chromatin-modifying complexes that are essential for embryonic development and stem cell renewal and are commonly deregulated in cancer. Here, we identify their target genes using genome-wide location analysis in human embryonic fibroblasts. We find that Polycomb-Repressive Complex I (PRC1), PRC2, and tri-methylated histone H3K27 co-occupy > 1000 silenced genes with a strong functional bias for embryonic development and cell fate decisions. We functionally identify 40 genes derepressed in human embryonic fibroblasts depleted of the PRC2 components (EZH2, EED, SUZ12) and the PRC1 component, BMI-1. Interestingly, several markers of osteogenesis, adipogenesis, and chrondrogenesis are among these genes, consistent with the mesenchymal origin of fibroblasts. Using a neuronal model of differentiation, we delineate two different mechanisms for regulating PcG target genes. For genes activated during differentiation, PcGs are displaced. However, for genes repressed during differentiation, we paradoxically find that they are already bound by the PcGs in nondifferentiated cells despite being actively transcribed. Our results are consistent with the hypothesis that PcGs are part of a preprogrammed memory system established during embryogenesis marking certain key genes for repressive signals during subsequent developmental and differentiation processes.', 'Epigenetic marking systems confer stability of gene expression during mammalian development. Genome-wide epigenetic reprogramming occurs at stages when developmental potency of cells changes. At fertilization, the paternal genome exchanges protamines for histones, undergoes DNA demethylation, and acquires histone modifications, whereas the maternal genome appears epigenetically more static. During preimplantation development, there is passive DNA demethylation and further reorganization of histone modifications. In blastocysts, embryonic and extraembryonic lineages first show different epigenetic marks. This epigenetic reprogramming is likely to be needed for totipotency, correct initiation of embryonic gene expression, and early lineage development in the embryo. Comparative work demonstrates reprogramming in all mammalian species analysed, but the extent and timing varies, consistent with notable differences between species during preimplantation development. Parental imprinting marks originate in sperm and oocytes and are generally protected from this genome-wide reprogramming. Early primordial germ cells possess imprinting marks similar to those of somatic cells. However, rapid DNA demethylation after midgestation erases these parental imprints, in preparation for sex-specific de novo methylation during gametogenesis. Aberrant reprogramming of somatic epigenetic marks after somatic cell nuclear transfer leads to epigenetic defects in cloned embryos and stem cells. Links between epigenetic marking systems appear to be developmentally regulated contributing to plasticity. A number of activities that confer epigenetic marks are firmly established, while for those that remove marks, particularly methylation, some interesting candidates have emerged recently which need thorough testing in vivo. A mechanistic understanding of reprogramming will be crucial for medical applications of stem cell technology.', 'Background: In animal models, variations in early maternal care are associated with differences in hypothalamic-pituitary-adrenal (HPA) stress response in the offspring, mediated via changes in the epigenetic regulation of glucocorticoid receptor (GR) gene (Nr3c1) expression. Objective: To study this in humans, relationships between prenatal exposure to maternal mood and the methylation status of a CpG-rich region in the promoter and exon 1F of the human GR gene (NR3C1) in newborns and HPA stress reactivity at age three months were examined. Results: Prenatal exposure to increased third trimester maternal depressed/anxious mood was associated with increased methylation of NR3C1 at a predicted NGFI-A binding site. Increased NR3C1 methylation at this site was also associated with increased salivary cortisol stress responses at 3 months, controlling for prenatal SRI exposure, postnatal age and pre and postnatal maternal mood. Methods: The methylation status of a CpG-rich region of the NR3C1 gene, including exon 1F, in genomic DNA from cord blood mononuclear cells was quantified by bisulfite pyrosequencing in infants of depressed mothers treated with a serotonin reuptake inhibitor antidepressant (SRI) (n = 33), infants of depressed non treated mothers (n = 13) and infants of non depressed/non treated mothers (n = 36). To study the functional implications of the newborn methylation status of NR3C1 in newborns, HPA function was assessed at three months using salivary cortisol obtained before and following a non noxious stressor and at a late afternoon basal time. Conclusions: Methylation status of the human NR3C1 gene in newborns is sensitive to prenatal maternal mood and may offer a potential epigenetic process that links antenatal maternal mood and altered HPA stress reactivity during infancy.', 'The formation of heterochromatin, which requires methylation of histone H3 at lysine 9 and the subsequent recruitment of chromodomain proteins such as heterochromatin protein HP1, serves as a model for the role of histone modifications and chromatin assembly in epigenetic control of the genome. Recent studies in Schizosaccharomyces pombe indicate that heterochromatin serves as a dynamic platform to recruit and spread a myriad of regulatory proteins across extended domains to control various chromosomal processes, including transcription, chromosome segregation and long-range chromatin interactions.', 'Compelling evidence suggests that the transcription factor Foxp3 acts as a master switch governing the development and function of CD4(+) regulatory T cells (Tregs). However, whether transcriptional control of Foxp3 expression itself contributes to the development of a stable Treg lineage has thus far not been investigated. We here identified an evolutionarily conserved region within the foxp3 locus upstream of exon-1 possessing transcriptional activity. Bisulphite sequencing and chromatin immunoprecipitation revealed complete demethylation of CpG motifs as well as histone modifications within the conserved region in ex vivo isolated Foxp3(+)CD25(+)CD4(+) Tregs, but not in naive CD25(-) CD4(+) T cells. Partial DNA demethylation is already found within developing Foxp3(+) thymocytes; however, Tregs induced by TGF-beta in vitro display only incomplete demethylation despite high Foxp3 expression. In contrast to natural Tregs, these TGF-beta-induced Foxp3(+) Tregs lose both Foxp3 expression and suppressive activity upon restimulation in the absence of TGF-beta. Our data suggest that expression of Foxp3 must be stabilized by epigenetic modification to allow the development of a permanent suppressor cell lineage, a finding of significant importance for therapeutic applications involving induction or transfer of Tregs and for the understanding of long-term cell lineage decisions.', 'Many genes associated with CpG islands undergo de novo methylation in cancer. Studies have suggested that the pattern of this modification may be partially determined by an instructive mechanism that recognizes specifically marked regions of the genome(1). Using chromatin immunoprecipitation analysis, here we show that genes methylated in cancer cells are specifically packaged with nucleosomes containing histone H3 trimethylated on Lys27. This chromatin mark is established on these unmethylated CpG island genes early in development and then maintained in differentiated cell types by the presence of an EZH2- containing Polycomb complex(2,3). In cancer cells(4), as opposed to normal cells, the presence of this complex brings about the recruitment of DNA methyl transferases, leading to de novo methylation. These results suggest that tumor-specific targeting of de novo methylation is pre-programmed by an established epigenetic system that normally has a role in marking embryonic genes for repression(2).', 'Ageing results from complex genetically and epigenetically programmed processes that are elicited in part by noxious or stressful events that cause programmed cell death. Here, we report that administration of spermidine, a natural polyamine whose intracellular concentration declines during human ageing, markedly extended the lifespan of yeast, flies and worms, and human immune cells. In addition, spermidine administration potently inhibited oxidative stress in ageing mice. In ageing yeast, spermidine treatment triggered epigenetic deacetylation of histone H3 through inhibition of histone acetyltransferases (HAT), suppressing oxidative stress and necrosis. Conversely, depletion of endogenous polyamines led to hyperacetylation, generation of reactive oxygen species, early necrotic death and decreased lifespan. The altered acetylation status of the chromatin led to significant upregulation of various autophagy-related transcripts, triggering autophagy in yeast, flies, worms and human cells. Finally, we found that enhanced autophagy is crucial for polyamine-induced suppression of necrosis and enhanced longevity.', 'Induced pluripotent stem (iPS) cells are derived by epigenetic reprogramming, but their DNA methylation patterns have not yet been analyzed on a genome-wide scale. Here, we find substantial hypermethylation and hypomethylation of cytosine-phosphate-guanine (CpG) island shores in nine human iPS cell lines as compared to their parental fibroblasts. The differentially methylated regions (DMRs) in the reprogrammed cells (denoted R-DMRs) were significantly enriched in tissue-specific (T-DMRs; 2.6-fold, P < 10(-4)) and cancer-specific DMRs (C-DMRs; 3.6-fold, P < 10(-4)). Notably, even though the iPS cells are derived from fibroblasts, their R-DMRs can distinguish between normal brain, liver and spleen cells and between colon cancer and normal colon cells. Thus, many DMRs are broadly involved in tissue differentiation, epigenetic reprogramming and cancer. We observed colocalization of hypomethylated R-DMRs with hypermethylated C-DMRs and bivalent chromatin marks, and colocalization of hypermethylated R-DMRs with hypomethylated C-DMRs and the absence of bivalent marks, suggesting two mechanisms for epigenetic reprogramming in iPS cells and cancer.', 'Histone methylation has important roles in regulating gene expression and forms part of the epigenetic memory system that regulates cell fate and identity. Enzymes that directly remove methyl marks from histones have recently been identified, revealing a new level of plasticity within this epigenetic modification system. Here we analyse the evolutionary relationship between Jumonji C ( JmjC)-domain-containing proteins and discuss their cellular functions in relation to their potential enzymatic activities.', 'Protein lysine deacetylases have a pivotal role in numerous biological processes and can be divided into the Rpd3/Hda1 and sirtuin families, each having members in diverse organisms including prokaryotes. In vertebrates, the Rpd3/Hda1 family contains 11 members, traditionally referred to as histone deacetylases (HDAC)1-11, which are further grouped into classes I, II and IV. Whereas most class I HDACs are subunits of multiprotein nuclear complexes that are crucial for transcriptional repression and epigenetic landscaping, class II members regulate cytoplasmic processes or function as signal transducers that shuttle between the cytoplasm and the nucleus. Little is known about class IV HDAC11, although its evolutionary conservation implies a fundamental role in various organisms.', 'Recent investigations have implicated long antisense noncoding RNAs in the epigenetic regulation of chromosomal domains. Here we show that Kcnq1ot1 is an RNA polymerase II-encoded, 91 kb-long, moderately stable nuclear transcript and that its stability is important for bidirectional silencing of genes in the Kcnq1 domain. Kcnq1ot1 interacts with chromatin and with the H3K9- and H3K27-specific histone methyltransferases G9a and the PRC2 complex in a lineage-specific manner. This interaction correlates with the presence of extended regions of chromatin enriched with H3K9me3 and H3K27me3 in the Kcnq1 domain in placenta, whereas fetal liver lacks both chromatin interactions and heterochromatin structures. In addition, the Kcnq1 domain is more often found in contact with the nucleolar compartment in placenta than in liver. Taken together, our data describe a mechanism whereby Kcnq1ot1 establishes lineage-specific transcriptional silencing patterns through recruitment of chromatin remodeling complexes and maintenance of these patterns through subsequent cell divisions occurs via targeting the associated regions to the perinucleolar compartment.', 'It is more evident now than ever that nucleosomes can transmit epigenetic information from one cell generation to the next. It has been demonstrated during the past decade that the post-translational modifications of histone proteins within the chromosome impact chromatin structure, gene transcription, and epigenetic information. Multiple modifications decorate each histone tail within the nucleosome, including some amino acids that can be modified in several different ways. Covalent modifications of histone tails known thus far include acetylation, phosphorylation, surnoylation, ubiquitination, and methylation. A large body of experimental evidence compiled during the past several years has demonstrated the impact of histone acetylation on transcriptional control. Although histone modification by methylation and ubiquitination was discovered long ago, it was only recently that functional roles for these modifications in transcriptional regulation began to surface. Highlighted in this review are the recent biochemical, molecular, cellular, and physiological functions of histone methylation and ubiquitination involved in the regulation of gene expression as determined by a combination of enzymological, structural, and genetic methodologies.', 'Cells employ elaborate mechanisms to introduce structural and chemical variation into chromatin. Here, we focus on one such element of variation: methylation of lysine 4 in histone H3 (H3K4). We assess a growing body of literature, including treatment of how the mark is established, the patterns of methylation, and the functional consequences of this epigenetic signature. We discuss structural aspects of the H3K4 methyl recognition by the downstream effectors and propose a distinction between sequence-specific recruitment mechanisms and stabilization on chromatin through methyllysine recognition. Finally, we hypothesize how the unique properties of the polyvalent chromatin fiber and associated effectors may amplify small differences in methyl-lysine recognition, simultaneously allowing for a dynamic chromatin architecture.', 'Embryonic stem cells rely on Polycomb group proteins to reversibly repress genes required for differentiation. We report that stem cell Polycomb group targets are up to 12-fold more likely to have cancer- specific promoter DNA hypermethylation than non-targets, supporting a stem cell origin of cancer in which reversible gene repression is replaced by permanent silencing, locking the cell into a perpetual state of self-renewal and thereby predisposing to subsequent malignant transformation.', \"Adult cancers may derive from stem or early progenitor cells(1,2). Epigenetic modulation of gene expression is essential for normal function of these early cells but is highly abnormal in cancers, which often show aberrant promoter CpG island hypermethylation and transcriptional silencing of tumor suppressor genes and pro-differentiation factors(3-5). We find that for such genes, both normal and malignant embryonic cells generally lack the hypermethylation of DNA found in adult cancers. In embryonic stem cells, these genes are held in a 'transcription- ready' state mediated by a 'bivalent' promoter chromatin pattern consisting of the repressive mark, histone H3 methylated at Lys27 ( H3K27) by Polycomb group proteins, plus the active mark, methylated H3K4. However, embryonic carcinoma cells add two key repressive marks, dimethylated H3K9 and trimethylated H3K9, both associated with DNA hypermethylation in adult cancers(6-8). We hypothesize that cell chromatin patterns and transient silencing of these important regulatory genes in stem or progenitor cells may leave these genes vulnerable to aberrant DNA hypermethylation and heritable gene silencing during tumor initiation and progression.\", ' ', 'Patterns of DNA methylation and chromatin structure are profoundly altered in neoplasia and include genome-wide losses of, and regional gains in, DNA methylation. The recent explosion in our knowledge of how chromatin organization modulates gene transcription has further highlighted the importance of epigenetic mechanisms in the initiation and progression of human cancer. These epigenetic changes - in particular, aberrant promoter hypermethylation that is associated with inappropriate gene silencing - affect virtually every step in tumour progression. In this review, we discuss these epigenetic events and the molecular alterations that might cause them and/or underlie altered gene expression in cancer.', \"Cells of a multicellular organism are genetically homogeneous but structurally and functionally heterogeneous owing to the differential expression of genes. Many of these differences in gene expression arise during development and are subsequently retained through mitosis. Stable alterations of this kind are said to be 'epigenetic', because they are heritable in the short term but do not involve mutations of the DNA itself. Research over the past few years has focused on two molecular mechanisms that mediate epigenetic phenomena: DNA methylation and histone modifications. Here, we review advances in the understanding of the mechanism and role of DNA methylation in biological processes. Epigenetic effects by means of DNA methylation have an important role in development but can also arise stochastically as animals age. Identification of proteins that mediate these effects has provided insight into this complex process and diseases that occur when it is perturbed. External influences on epigenetic processes are seen in the effects of diet on long-term diseases such as cancer. Thus, epigenetic mechanisms seem to allow an organism to respond to the environment through changes in gene expression. The extent to which environmental effects can provoke epigenetic responses represents an exciting area of future research.\", 'Epithelial and hematopoietic cells have a high turnover and their progenitor cells divide continuously, making them prime targets for genetic and epigenetic changes that lead to cell transformation and tumorigenesis. The consequent changes in cell behavior and responsiveness result not only from genetic alterations such as activation of oncogenes or inactivation of tumor suppressor genes, but also from altered production of, or responsiveness to, stimulatory or inhibitory growth and differentiation factors. Among these, transforming growth factor beta (TGF-beta) and its signaling effectors act as key determinants of carcinoma cell behavior. The autocrine and paracrine effects of TGF-beta on tumor cells and the tumor micro-environment exert both positive and negative influences on cancer development. Accordingly, the TGF-beta signaling pathway has been considered as both a tumor suppressor pathway and a promoter of tumor progression and invasion. Here we evaluate the role of TGF-beta in tumor development and attempt to reconcile the positive and negative effects of TGF-beta in carcinogenesis.', 'Genomic imprinting affects several dozen mammalian genes and results in the expression of those genes from only one of the two parental chromosomes. This is brought about by epigenetic instructions - imprints - that are laid down in the parental germ cells. Imprinting is a particularly important genetic mechanism in mammals, and is thought to influence the transfer of nutrients to the fetus and the newborn from the mother. Consistent with this view is the fact that imprinted genes tend to affect growth in the womb and behaviour after birth. Aberrant imprinting disturbs development and is the cause of various disease syndromes. The study of imprinting also provides new insights into epigenetic gene modification during development.', 'The developmental programme of embryogenesis is controlled by both genetic and epigenetic mechanisms. An emerging theme from recent studies is that the regulation of higher-order chromatin structures by DNA methylation and histone modification is crucial for genome reprogramming during early embryogenesis and gametogenesis, and for tissue-specific gene expression and global gene silencing. Disruptions to chromatin modification can lead to the dysregulation of developmental processes, such as X-chromosome inactivation and genomic imprinting, and to various diseases. Understanding the process of epigenetic reprogramming in development is important for studies of cloning and the clinical application of stem-cell therapy.', 'Early nutrition affects adult metabolism in humans and other mammals, potentially via persistent alterations in DNA methylation. With viable yellow agouti (A(vy)) mice, which harbor a transposable element in the agouti gene, we tested the hypothesis that the metastable methylation status of specific transposable element insertion sites renders them epigenetically labile to early methyl donor nutrition. Our results show that dietary methyl supplementation of ala dams with extra folic acid, vitamin B-12, choline, and betaine alter the phenotype of their A(vy)/a offspring via increased CpG methylation at the A(vy) locus and that the epigenetic metastability which confers this lability is due to the A(vy) transposable element. These findings suggest that dietary supplementation, long presumed to be purely beneficial, may have unintended deleterious influences on the establishment of epigenetic gene regulation in humans.', 'Recently, the concept that epigenetic, as well as genetic, events might be central to the evolution of human cancer is re-emerging. Cancers often exhibit an aberrant methylation of gene promoter regions that is associated with loss of gene function. This DNA change constitutes a heritable state, not mediated by altered nucleotide sequence, that appears to be tightly linked to the formation of transcriptionally repressive chromatin. This epigenetic process acts as an alternative to mutations to disrupt tumor-suppressor gene function and can predispose to genetic alterations through inactivating DNA-repair genes. Dissecting the molecular processes that mediate these methylation changes will enhance our understanding of chromatin modeling and gene regulation and might present novel possibilities for cancer therapy. Methylation changes constitute potentially sensitive molecular markers to define risk states, monitor prevention strategies, achieve early diagnosis, and track the prognosis of cancer.', 'Polyploidy is a prominent process in plants and has been significant in the evolutionary history of vertebrates and other eukaryotes. In plants, interdisciplinary approaches combining phylogenetic and molecular genetic perspectives have enhanced our awareness of the myriad genetic interactions made possible by polyploidy. Here, processes and mechanisms of gene and genome evolution in polyploids are reviewed. Genes duplicated by polyploidy may retain their original or similar function, undergo diversification in protein function or regulation, or one copy may become silenced through mutational or epigenetic means. Duplicated genes also may interact through inter-locus recombination, gene conversion, or concerted evolution. Recent experiments have illuminated important processes in polyploids that operate above the organizational level of duplicated genes. These include inter-genomic chromosomal exchanges, saltational, non-Mendelian genomic evolution in nascent polyploids, inter-genomic invasion, and cytonuclear stabilization. Notwithstanding many recent insights, much remains to be learned about many aspects of polyploid evolution, including: the role of transposable elements in structural and regulatory gene evolution; processes and significance of epigenetic silencing; underlying controls of chromosome pairing; mechanisms and functional significance of rapid genome changes; cytonuclear accommodation; and coordination of regulatory factors contributed by two, sometimes divergent progenitor genomes. Continued application of molecular genetic approaches to questions of polyploid genome evolution holds promise for producing lasting insight into processes by which novel genotypes are generated and ultimately into how polyploidy facilitates evolution and adaptation.', 'Histone H3 lysine 9 methylation has been proposed to provide a major switch for the functional organization of chromosomal subdomains. Here, we show that the murine Suv39h histone methyltransferases (HMTases) govern H3-K9 methylation at pericentric heterochromatin and induce a specialized histone methylation pattern that differs from the broad H3-K9 methylation present at other chromosomal regions. Suv39h-deficient mice display severely impaired viability and chromosomal instabilities that are associated with an increased tumor risk and perturbed chromosome interactions during male meiosis. These in vivo data assign a crucial role for pericentric H3-K9 methylation in protecting genome stability, and define the: Suv39h HMTases as important epigenetic regulators for mammalian development.', 'Transcriptional activation of a gene involves an orchestrated recruitment of components of the basal transcription machinery and intermediate factors, concomitant with an alteration in local chromatin structure generated by posttranslational modifications of histone tails and nucleosome remodeling. We provide here a comprehensive picture of events resulting in transcriptional activation of a gene, through evaluating the estrogen receptor-alpha (NR3A1) target pS2 gene promoter in MCF-7 cells. This description integrates chromatin remodeling with a kinetic evaluation of cyclical networks of association of 46 transcription factors with the promoter, as determined by chromatin immunoprecipitation assays. We define the concept of a transcriptional clock that directs and achieves the sequential and combinatorial assembly of a transcriptionally productive complex on a promoter. Furthermore, the unanticipated findings of key roles for histone deacetylases and nucleosome-remodeling complexes in limiting transcription implies that transcriptional activation is a cyclical process that requires both activating and repressive epigenetic processes.', 'Hepatocarcinogenesis is a slow process during which genomic changes progressively alter the hepatocellular phenotype to produce cellular intermediates that evolve into hepatocellular carcinoma. During the long pre-neoplastic stage, in which the liver is often the site of chronic hepatitis, cirrhosis, or both, hepatocyte cycling is accelerated by upregulation of mitogenic pathways, in part through epigenetic mechanisms. This leads to the production of monoclonal populations of aberrant and dysplastic hepatocytes that have telomere erosion and telomerase re-expression, sometimes microsatellite instability, and occasionally structural aberrations in genes and chromosomes. Development of dysplastic hepatocytes in foci and nodules and emergence of hepatocellular carcinoma are associated with the accumulation of irreversible structural alterations in genes and chromosomes, but the genomic basis of the malignant phenotype is heterogeneous. The malignant hepatocyte phenotype may be produced by the disruption of a number of genes that function in different regulatory pathways, producing several molecular variants of hepatocellular carcinoma. New strategies should enable these variants to be characterized.', 'Enhancer of Zeste is a Polycomb Group protein essential for the establishment and maintenance of repression of homeotic and other genes. In the early embryo it is found in a complex that includes ESC and is recruited to Polycomb Response Elements. We show that this complex contains a methyltransferase activity that methylates lysine 9 and lysine 27 of histone H3, but the activity is lost when the E(Z) SET domain is mutated. The lysine 9 position is trimethylated and this mark is closely associated with Polycomb binding sites on polytene chromosomes but is also found in centric heterochromatin, chromosome 4, and telomeric sites. Histone H3 methylated in vitro by the E(Z)/ ESC complex binds specifically to Polycomb protein.', 'Cytosine-5 DNA methylation occurs in the context of CpG dinucleotides in vertebrates. Aberrant methylation of CpG islands in human tumors has been shown to cause transcriptional silencing of tumor-suppressor genes. Most methods used to analyze cytosine-5 methylation patterns require cumbersome manual techniques that employ gel electrophoresis, restriction enzyme digestion, radiolabeled dNTPs or hybridization probes. The development of high-throughput technology for the analysis of DNA methylation would significantly expand our ability to derive molecular information from clinical specimens. This study describes a high-throughput quantitative methylation assay that utilizes fluorescence-based real-time PCR (TaqMan (R)) technology that requires no further manipulations after the PCR step. Methy-Light is a highly sensitive assay, capable of detecting methylated alleles in the presence of a 10 000-fold excess of unmethylated alleles. The assay is also highly quantitative and can very accurately determine the relative prevalence of a particular pattern of DNA methylation. We show that MethyLight can distinguish between mono-allelic and bi-allelic methylation of the MLH1 mismatch repair gene in human colorectal tumor specimens. The development of this technique should considerably enhance our ability to rapidly and accurately generate epigenetic profiles of tumor samples.', 'Cancer-associated DNA hypomethylation is as prevalent as cancer-linked hypermethylation, but these two types of epigenetic abnormalities usually seem to affect different DNA sequences. Much more of the genome is generally subject to undermethylation rather than overmethylation. Genomic hypermethylation in cancer has been observed most often in CpG islands in gene regions. In contrast, very frequent hypomethylation is seen in both highly and moderately repeated DNA sequences in cancer, including heterochromatic DNA repeats, dispersed retrotransposons, and endogenous retroviral elements. Also, unique sequences, including transcription control sequences, are often subject to cancer-associated undermethylation. The high frequency of cancer-linked DNA hypomethylation, the nature of the affected sequences, and the absence of associations with DNA hypermethylation are consistent with an independent role for DNA undermethylation in cancer formation or tumor progression. Increased karyotypic instability and activation of tumor-promoting genes by cis or trans effects, that might include altered heterochromatin-euchromatin interactions, may be important consequences of DNA hypomethylation which favor oncogenesis. The relationship of DNA hypomethylation to tumorigenesis is important to be considered in the light of cancer therapies involving decreasing DNA methylation. Inducing DNA hypomethylation may have short-term anticancer effects, but might also help speed tumor progression from cancer cells surviving the DNA demethylation chemotherapy.', 'Sirtuins (Sir2-like proteins) are present in prokaryotes and eukaryotes, Here, two new human sirtuins (SIRT6 and SIRT7) are found to be similar to a particular subset of insect, nematode, plant, and protozoan sirtuins. Molecular phylogenetic analysis of 60 sirtuin conserved core domain sequences from a diverse array of organisms (including archaeans, bacteria, yeasts, plants, protozoans, and metazoans) shows that eukaryotic Sir2-like proteins group into four main branches designated here as classes I-IV. Prokaryotic sirtuins include members of classes II and III. A fifth class of sirtuin is present in gram positive bacteria and Thermotoga maritima. Saccharomyces cerevisiae has five class I sirtuins. Caenorhabditis elegans and Drosophila melanogaster have sirtuin genes from classes I, II, and TV. The seven human sirtuin genes include all four classes: SIRT1, SIRT2, and SIRT3 are class I, SIRT4 is class II, SIRT5 is class III, and SIRT6 and SIRT7 are class IV. (C) 2000 Academic Press.', \"Chromatin is the physiologically relevant substrate for all genetic processes inside the nuclei of eukaryotic cells. Dynamic changes in the local and global organization of chromatin are emerging as key regulators of genomic function. Indeed, a multitude of signals from outside and inside the cell converges on this gigantic signaling platform. Numerous post-translational modifications of histones, the main protein components of chromatin, have been documented and analyzed in detail. These 'marks' appear to crucially mediate the functional activity of the genome in response to upstream signaling pathways. Different layers of cross-talk between several components of this complex regulatory system are emerging, and these epigenetic circuits are the focus of this review.\", 'Allelic loss at the short arm of chromosome 3 is one of the most common and earliest events in the pathogenesis of lung cancer, and is observed in more than 90% of small-cell lung cancers (SCLCs) and in 50-80% of non-small-cell lung cancers(1,2) (NSCLCs). Frequent and early loss of heterozygosity and the presence of homozygous deletions suggested a critical role of the region 3p21.3 in tumorigenesis(2-4) and a region of common homozygous deletion in 3p21.3 was narrowed to 120 kb (ref. 5). Several putative tumour-suppressor genes located at 3p21 have been characterized, but none of these genes appear to be altered in lung cancer. Here we describe the cloning and characterization of a human RAS effector homologue (RASSF1) located in the 120-kb region of minimal homozygous deletion. We identified three transcripts, A, B and C, derived from alternative splicing and promoter usage. The major transcripts A and C were expressed in all normal tissues. Transcript A was missing in all SCLC cell lines analysed and in several other cancer cell lines. loss of expression was correlated with methylation of the CpG-island promoter sequence of RASSF1A. The promoter was highly methylated in 24 of 60 (40%) primary lung tumours, and 4 of 41 tumours analysed carried missense mutations. Re-expression of transcript A in lung carcinoma cells reduced colony formation, suppressed anchorage-independent growth and inhibited tumour formation in nude mice. These characteristics indicate a potential role for RASSF1A asa lung tumour suppresor gene.', 'Deposition of the major histone H3 (H3.1) is coupled to DNA synthesis during DNA replication and possibly DNA repair, whereas histone variant H3.3 serves as the replacement variant for the DNA-synthesis-independent deposition pathway. To address how histories H3.1 and H3.3 are deposited into chromatin through distinct pathways, we have purified deposition machineries for these histories. The H3.1 and H3.3 complexes contain distinct histone chaperones, CAF-1 and HIRA, that we show are necessary to mediate DNA-synthesis-dependent and -independent nucleosome assembly, respectively. Notably, these complexes possess one molecule each of H3.1/H3.3 and H4, suggesting that histories H3 and H4 exist as dimeric units that are important intermediates in nucleosome formation. This finding provides new insights into possible mechanisms for maintenance of epigenetic information after chromatin duplication.', 'Dynamic epigenetic modification of the genome occurs during early development of the mouse. Active demethylation of the paternal genome occurs in the zygote, followed by passive demethylation during cleavage stages, and de novo methylation, which is thought to happen after implantation. We have investigated these processes by using indirect immunofluoresence with an antibody to 5-methyl cytosine. In contrast to previous work, we show that demethylation of the male pronucleus is completed within 4 h of fertilisation. This activity is intricately linked with and not separable from pronucleus formation. In conditions permissive for polyspermy, up to five male pronuclei underwent demethylation in the same oocyte. Paternal demethylation in fertilised oocytes deficient for MBD2, the only candidate demethylase, occurred normally. Passive loss of methylation occurred in a stepwise fashion up to the morulae stage without any evidence of spatial compartmentalisation. De novo methylation was observed specifically in the inner cell mass (ICM) but not in the trophectoderm of the blastocyst and hence may have an important role in early lineage specification. This is the first complete and detailed analysis of the epigenetic reprogramming cycle during preimplantation development. The three phases of methylation reprogramming may have roles in imprinting, the control of gene expression, and the establishment of nuclear totipotency. (C) 2001 Elsevier Science.', \"5-Azacytidine was first synthesized almost 40 years ago. It was demonstrated to have a wide range of anti-metabolic activities when tested against cultured cancer cells and to be an effective chemotherapeutic agent for acute myelogenous leukemia. However, because of 5-azacytidine's general toxicity, other nucleoside analogs were favored as therapeutics. The finding that 5-azacytidine was incorporated into DNA and that, when present in DNA, it inhibited DNA methylation, led to widespread use of 5-azacytidine and 5-aza-2'-deoxycytidine (Decitabine) to demonstrate the correlation between loss of methylation in specific gene regions and activation of the associated genes. There is now a revived interest in the use of Decitabine as a therapeutic agent for cancers in which epigenetic silencing of critical regulatory genes has occurred. Here, the current status of our understanding of the mechanism(s) by which 5-azacytosine residues in DNA inhibit DNA methylation is reviewed with an emphasis on the interactions of these residues with bacterial and mammalian DNA (cytosine-C5) methyltransferases. The implications of these mechanistic studies for development of less toxic inhibitors of DNA methylation are discussed.\", 'The enzyme-catalyzed acetylation of the N-terminal tail domains of core histones provides a rich potential source of epigenetic information. This may be used both to mediate transient changes in transcription, through modification of promoter-proximal nucleosomes, and for the longer-term maintenance and modulation of patterns of gene expression. The latter may be achieved by setting specific patterns of histone acetylation, perhaps involving acetylation of particular lysine residues, across relatively large chromatin domains. The histone acetylating and deacetylating enzymes (HATs and HDACs, respectively) can be targeted to specific regions of the genome and show varying degrees of substrate specificity, properties that are consistent with a role in maintaining a dynamic, acetylation-based epigenetic code. The code may be read (ie. exert a functional effect) either through non-histone proteins that bind in an acetylation-dependent manner, or through direct effects on chromatin structure. Recent evidence raises the interesting possibility that an acetylation-based code may operate through both mitosis and meiosis, providing a possible mechanism for germ-line transmission of epigenetic changes. BioEssays 22:836-845, 2000. (C) 2000 John Wiley & Sons, Inc.', 'We have come a long way since the first reports of the existence of aberrant DNA methylation in human cancer. Hypermethylation of CpG islands located in the promoter regions of tumor suppressor genes is now firmly established as an important mechanism for gene inactivation. CpG island hypermethylation has been described in almost every tumor type. Many cellular pathways are inactivated by this type of epigenetic lesion: DNA repair (hMLH1, MGMT), cell cycle (p16(INK4a), p15(INK4b), p14(ARF)), apoptosis (DAPK), cell adherence (CDH1, CDH13), detoxification (GSTP1), etc... However, we still know little of the mechanisms of aberrant methylation and why certain genes are selected over others. Hypermethylation is not an isolated layer of epigenetic control, but is linked to the other pieces of the puzzle such as methyl-binding proteins, DNA methyltransferases and histone deacetylase, but our understanding of the degree of specificity of these epigenetic layers in the silencing of specific tumor suppressor genes remains incomplete. The explosion of user-friendly technologies has given rise to a rapidly increasing list of hypermethylated genes. Careful functional and genetic studies are necessary to determine which hypermethylation events are truly relevant for human tumorigenesis. The development of CpG island hypermethylation profiles for every form of human tumors has yielded valuable pilot clinical data in monitoring and treating cancer patients based in our knowledge of DNA methylation. Basic and translational will both be needed in the near future to fully understand the mechanisms, roles and uses of CpG island hypermethylation in human cancer. The expectations are high.', 'Genome-wide epigenetic reprogramming in mammalian germ cells, zygote and early embryos, plays a crucial role in regulating genome functions at critical stages of development. We show here that mouse primordial germ cells (PGCs) exhibit dynamic changes in epigenetic modifications between days 10.5 and 12.5 post coitum (dpc). First, contrary to previous suggestions, we show that PGCs do indeed acquire genome-wide de novo methylation during early development and migration into the genital ridge. However, following their entry into the genital ridge, there is rapid erasure of DNA methylation of regions within imprinted and non-imprinted loci. For most genes, the erasure commences simultaneously in PGCs in both male and female embryos, which is completed within I day of development. Based on the kinetics of this process, we suggest that this is an active demethylation process initiated upon the entry of PGCs into the gonadal anlagen. The timing of reprogramming in PGCs is crucial since it ensures that germ cells of both sexes acquire an equivalent epigenetic state prior to the differentiation of the definitive male and female germ cells in which new parental imprints are established subsequently. Some repetitive elements, however, show incomplete erasure, which may be essential for chromosome stability and for preventing activation of transposons to reduce the risk of germline mutations. Aberrant epigenetic reprogramming in the germ line would cause the inheritance of epimutations that may have consequences for human diseases as suggested by studies on mouse models. (C) 2002 Elsevier Science Ireland Ltd. All rights reserved.', 'Aberrant WNT pathway signaling is an early progression event in 90% of colorectal cancers(1). It occurs through mutations mainly of APC and less often of CTNNB1 (encoding beta-catenin)(1-3) or AXIN2 (encoding axin-2, also known as conductin)(4). These mutations allow ligand-independent WNT signaling that culminates in abnormal accumulation of free beta-catenin in the nucleus(1-3). We previously identified frequent promoter hypermethylation and gene silencing of the genes encoding secreted frizzled-related proteins (SFRPs) in colorectal cancer(5). SFRPs possess a domain similar to one in the WNT-receptor frizzled proteins and can inhibit WNT receptor binding to downregulate pathway signaling during development(6-10). Here we show that restoration of SFRP function in colorectal cancer cells attenuates WNT signaling even in the presence of downstream mutations. We also show that the epigenetic loss of SFRP function occurs early in colorectal cancer progression and may thus provide constitutive WNT signaling that is required to complement downstream mutations in the evolution of colorectal cancer.', 'Methylation of position-specific lysine residues in histone N termini is a central modification for regulating epigenetic transitions in chromatin. Each methylatable lysine residue can exist in a mono-, di-, or trimethylated state, thereby extending the indexing potential of this particular modification. Here, we examine all possible methylation states for histone H3 lysine 9 (H3-K9) and lysine 27 (H3-K27) in mammalian chromatin. Using highly specific antibodies together with quantitative mass spectrometry, we demonstrate that pericentric heterochromatin is selectively enriched for H3-K27 monomethylation and H3-K9 trimethylation. This heterochromatic methylation profile is dependent on the Suv39h histone methyltransferases (HMTases) but independent of the euchromatic G9a HMTase. In Suv39h double null cells, pericentric heterochromatin is converted to alternative methylation imprints and accumulates H3-K27 trimethylation and H3-K9 monomethylation. Our data underscore the selective presence of distinct histone lysine methylation states in partitioning chromosomal subdomains but also reveal a surprising plasticity in propagating methylation patterns in eukaryotic chromatin.', 'During the development of multicellular organisms, cells become different from one another by changing their genetic program in response to transient stimuli. Long after the stimulus is gone, cellular memory mechanisms enable cells to remember their chosen fate over many cell divisions. The Polycomb and Trithorax groups of proteins, respectively, work to maintain repressed or active transcription states of developmentally important genes through many rounds of cell division. Here we review current ideas on the protein and DNA components of this transcriptional memory system and how they interact dynamically with each other to orchestrate cellular memory for several hundred genes.', 'DNA methylation has recently moved to centre stage in the aetiology of human neurodevelopmental syndromes such as the fragile X, ICF and Rett syndromes. These diseases result from the misregulation of genes that occurs with the toss of appropriate epigenetic controls during neuronal development. Recent advances have connected DNA methylation to chromatin-remodeling enzymes, and understanding this link will be central to the design of new therapeutic tools.', 'The centromere is a chromosomal locus that ensures delivery of one copy of each chromosome to each daughter at cell division. Efforts to understand the nature and specification of the centromere have demonstrated that this central element for ensuring inheritance is itself epigenetically determined. The kinetochore, the protein complex assembled at each centromere, serves as the attachment site for spindle microtubules and the site at which motors generate forces to power chromosome movement. Unattached kinetochores are also the signal generators for the mitotic checkpoint, which arrests mitosis until all kinetochores have correctly attached to spindle microtubules, thereby representing the major cell cycle control mechanism protecting against loss of a chromosome (aneuploidy).', 'On the histone H3 tail, Lys 9 and Lys 27 are both methylation sites associated with epigenetic repression, and reside within a highly related sequence motif ARKS. Here we show that the chromodomain proteins Polycomb (Pc) and HP1 (heterochromatin protein 1) are highly discriminatory for binding to these sites in vivo and in vitro. In Drosophila S2 cells, and on polytene chromosomes, methyl-Lys 27 and Pc are both excluded from areas that are enriched in methyl-Lys 9 and HP1. Swapping of the chromodomain regions of Pc and HP1 is sufficient for switching the nuclear localization patterns of these factors, indicating a role for their chromodomains in both target site binding and discrimination. To better understand the molecular basis for the selection of methyl-lysine binding sites, we solved the 1.8 Angstrom structure of the Pc chromodomain in complex with a H3 peptide bearing trimethyl-Lys 27, and compared it with our previously determined structure of the HP1 chromodomain in complex with a H3 peptide bearing trimethyl-Lys 9. The Pc chromodomain distinguishes its methylation target on the H3 tail via an extended recognition groove that binds five additional residues preceding the ARKS motif.', 'DNA methylation is essential for the control of a number of biological mechanisms in mammals [1]. Mammalian development is accompanied by two major waves of genome-wide demethylation and remethylation: one during germ-cell development and the other after fertilisation [2-7]. Most previous studies have suggested that the genome-wide demethylation observed after fertilisation occurs passively, that is, by the lack of maintenance methylation following DNA replication and cell division [6,7], although one other study has reported that replication-independent demethylation may also occur during early embryogenesis [8]. Here, we report that genes that are highly methylated in sperm are rapidly demethylated in the zygote only hours after fertilisation, before the first round of DNA replication commences. By contrast, the oocyte-derived maternal alleles are unaffected by this reprogramming. They either remain methylated after fertilisation or become further methylated de novo. These results provide the first direct evidence for active demethylation of single-copy genes in the mammalian zygote and, moreover, reveal a striking asymmetry in epigenetic methylation reprogramming. Whereas paternally (sperm)-derived sequences are exposed to putative active demethylases in the oocyte cytoplasm, maternally (oocyte)-derived sequences are protected from this reaction. These results, whose generality is supported by findings of Mayer et al. [9], have important implications for the establishment of biparental genetic totipotency after fertilisation, the establishment and maintenance of genomic imprinting, and the reprogramming of somatic cells during cloning.', 'Aberrant hypermethylation of gene promoters is a major mechanism associated with inactivation of tumor-suppressor genes in cancer. We previously showed this transcriptional silencing to be mediated by both methylation and histone deacetylase activity, with methylation being dominant. Here, we have used cDNA microarray analysis to screen for genes that are epigenetically silenced in human colorectal cancer. By screening over 10,000 genes, we show that our approach can identify a substantial number of genes with promoter hypermethylation in a given cancer; these are distinct from genes with unmethylated promoters, for which increased expression is produced by histone deacetylase inhibition alone. Many of the hypermethylated genes we identified have high potential for roles in tumorigenesis by virtue of their predicted function and chromosome position. We also identified a group of genes that are preferentially hypermethylated in colorectal cancer and gastric cancer. One of these genes, SFRP1, belongs to a gene family; we show that hypermethylation of four genes in this family occurs very frequently in colorectal cancer, providing for (i) a unique potential mechanism for loss of tumor-suppressor gene function and (ii) construction of a molecular marker panel that could detect virtually all colorectal cancer.', 'Gene function in cancer can be disrupted either through genetic alterations, which directly mutate or delete genes, or epigenetic alterations, which alter the heritable state of gene expression. The latter events are mediated by formation of transcriptionally repressive chromatin states around gene transcription start sites and an associated gain of methylation in normally unmethylated CpG islands in these regions. The genes affected include over half of the tumor suppressor genes that cause familial cancers when mutated in the germline; the selective advantage for genetic and epigenetic dysfunction in these genes is very similar. The aberrant methylation can begin very early in tumor progression and mediate most of the important pathway abnormalities in cancer including loss of cell cycle control, altered function of transcription factors, altered receptor function, disruption of normal cell-cell and cell-substratum interaction, inactivation of signal transduction pathways, loss of apoptotic signals and genetic instability. The active role of the aberrant methylation in transcriptional silencing of genes is becoming increasingly understood and involves a synergy between the methylation and histone deacetylase (HDAC) activity, This synergy can be mediated directly by HDAC interaction with DNA methylating enzymes and by recruitment through complexes involving methyl-cytosine binding proteins. In the translational arena, the promoter hypermethylation changes hold great promise as DNA tumor markers and their potentially reversible state creates a target for cancer therapeutic strategies involving gene reactivation.', 'DNA methylation plays an important role in mammalian development and correlates with chromatin-associated gene silencing. The recruitment of MeCP2 to methylated CpG dinucleotides represents a major mechanism by which DNA methylation can repress transcription. MeCP2 silences gene expression partly by recruiting histone deacetylase (HDAC) activity, resulting in chromatin remodeling. Here, we show that MeCP2 associates with histone methyltransferase activity in vivo and that this activity is directed against Lys(9) of histone H3. Two characterized repression domains of MeCP2 are involved in tethering the histone methyltransferase to MeCP2. We asked if MeCP2 can deliver Lys(9)H3 methylation to the H19 gene, whose activity it represses. We show that the presence of MeCP2 on nucleosomes within the repressor region of the H19 gene (the differentially methylated domain) coincides with an increase in H3 Lys(9) methylation. Our data provide evidence that MeCP2 reinforces a repressive chromatin state by acting as a bridge between two global epigenetic modifications, DNA methylation and histone methylation.', 'A large set of endogenous small RNAs of predominantly 21 to 24 nucleotides was identified in Arabidopsis. These small RNAs resembled micro-RNAs from animals and were similar in size to small interfering RNAs; that accumulated during RNA silencing triggered by multiple types of inducers. Among the 125 sequences identified, the vast majority (90%) arose from intergenic regions, although small RNAs corresponding to predicted protein-coding genes, transposon-like sequences, and a structural RNA gene also were identified. Evidence consistent with the derivation of small RNAs of both polarities, and from highly base-paired precursors, was obtained through the identification and analysis of clusters of small RNA loci. The accumulation of specific small RNAs was regulated developmentally. We propose that Arabidopsis small RNAs; participate in a wide range of post-transcriptional and epigenetic events.', 'Recent results from diverse organisms point to a self-reinforcing network of interactions among the three best-characterized covalent modifications that mark heterochromatin: histone hypoacetylation, histone H3-Lys9 methylation, and cytosine methylation. These modification systems suggest a mechanistic basis for spreading of heterochromatin over large domains and for stable epigenetic inheritance of the silent state. All three modifications used in packaging heterochromatin are also used in stable silencing of euchromatic genes.', 'Recent data in humans and animals suggest that assisted reproductive technology (ART) might affect the epigenetics of early embryogenesis and might cause birth defects. We report the first evidence, to our knowledge, that ART is associated with a human overgrowth syndrome-namely, Beckwith-Wiedemann syndrome (BWS). In a prospective study, the prevalence of ART was 4.6% (3 of 65), versus the background rate of 0.8% in the United States. A total of seven children with BWS were born after ART-five of whom were conceived after intracytoplasmic sperm injection. Molecular studies of six of the children indicate that five of the six have specific epigenetic alterations associated with BWS-four at LIT1 and one at both LIT1 and H19. We discuss the implications of our finding that ART is associated with human overgrowth, similar to the large offspring syndrome reported in ruminants.', 'Cadmium is a heavy metal of considerable environmental and occupational concern. Cadmium compounds are classified as human carcinogens by several regulatory agencies. The most convincing data that cadmium is carcinogenic in humans comes from studies indicating occupational cadmium exposure is associated with lung cancer. Cadmium exposure has also been linked to human prostate and renal cancer, although this linkage is weaker than for lung cancer. Other target sites of cadmium carcinogenesis in humans, such as liver, pancreas and stomach, are considered equivocal. In animals, cadmium effectively induces cancers at multiple sites and by various routes. Cadmium inhalation in rats induces pulmonary adenocarcinomas, in accord with its role in human lung cancer. Cadmium can induce tumors and/or preneoplastic lesions within the rat prostate after ingestion or injection. At relatively high doses, cadmium induces benign testicular tumors in rats, but these appear to be due to early toxic lesions and loss of testicular function, rather than from a specific carcinogenic effect of cadmium. Like many other metals, cadmium salts will induce mesenchymal tumors at the site of subcutaneous (s.c.) or intramuscular (i.m.) injections, but the human relevance of these is dubious. Other targets of cadmium in rodents include the liver, adrenal, pancreas, pituitary, and hematopoietic system. With the exception of testicular tumors in rodents, the mechanisms of cadmium carcinogenesis are poorly defined. Cadmium can cause any number of molecular lesions that would be relevant to oncogenesis in various cellular model systems. Most studies indicate cadmium is poorly mutagenic and probably acts through indirect or epigenetic mechanisms, potentially including aberrant activation of oncogenes and suppression of apoptosis. (C) 2003 Elsevier B.V. All rights reserved.', 'Diverse post-translational modifications of histone amino termini represent an important epigenetic mechanism for the organisation of chromatin structure and the regulation of gene activity. Within the past two years, great progress has been made in understanding the functional implications of histone methylation; in particular through the characterisation of histone methyltransferases that direct the site-specific methylation of, for example, lysine 9 and lysine 4 positions in the histone H3 amino terminus. All known histone methyltransferases of this type contain the evolutionarily conserved SET domain and appear to be able to stimulate either gene repression or gene activation. Methylation of H3 Lys9 and Lys4 has been visualised in native chromatin, indicating opposite roles in structuring repressive or accessible chromatin domains. For example, at the mating-type loci in Schizosaccharomyces pombe, at pericentric heterochromatin and at the inactive X chromosome in mammals, striking differences between these distinct marks have been observed. H3 Lys9 methylation is also important to direct additional epigenetic signals such as DNA methylation - for example, in Neurospora crassa and in Arabidopsis thaliana. Together, the available data strongly establish histone lysine methylation as a central modification for the epigenetic organisation of eukaryotic genomes.', 'Polyploidy has long been recognized as a prominent force shaping the evolution of eukaryotes, especially flowering plants. New phenotypes often arise with polyploid formation and can contribute to the success of polyploids in nature or their selection for use in agriculture. Although the causes of novel variation in polyploids are not well understood, they could involve changes in gene expression through increased variation in dosage-regulated gene expression, altered regulatory interactions, and rapid genetic and epigenetic changes. New research approaches are being used to study these mechanisms and the results should provide a more complete understanding of polyploidy.', 'Adult germline stem cells are capable of self-renewal, tissue regeneration and production of large numbers of differentiated progeny. We show here that the classical mouse mutant luxoid affects adult germline stem cell self-renewal. Young homozygous luxoid mutant mice produce limited numbers of normal spermatozoa and then progressively lose their germ line after birth. Transplantation studies showed that germ cells from mutant mice did not colonize recipient testes, suggesting that the defect is intrinsic to the stem cells. We determined that the luxoid mutant contains a nonsense mutation in the gene encoding Plzf, a transcriptional repressor that regulates the epigenetic state of undifferentiated cells, and showed that Plzf is coexpressed with Oct4 in undifferentiated spermatogonia. This is the first gene shown to be required in germ cells for stem cell self-renewal in mammals.', 'The covalent modification of nucleosomal histones has emerged as a major determinant of chromatin structure and gene activity. To understand the interplay between various histone modifications, including acetylation and methylation, we performed a genome-wide chromatin structure analysis in a higher eukaryote. We found a binary pattern of histone modifications among euchromatic genes, with active genes being hyperacetylated for H3 and H4 and hypermethylated at Lys 4 and Lys 79 of 113, and inactive genes being hypomethylated and deacetylated at the same residues. Furthermore, the degree of modification correlates with the level of transcription, and modifications are largely restricted to transcribed regions, suggesting that their regulation is tightly linked to polymerase activity.', 'Manipulation or non-physiological embryo culture environments can lead to defective fetal programming in livestock. Our demonstration of reduced fetal methylation and expression of ovine IGF2R suggests pre-implantation embryo procedures may be vulnerable to epigenetic alterations in imprinted genes. This highlights the potential benefits of epigenetic diagnostic screening in developing embryo procedures.', 'Proteins from the Polycomb group (PcG) are epigenetic chromatin modifiers involved in cancer development and also in the maintenance of embryonic and adult stem cells, The therapeutic potential of stem cells and the growing conviction that tumors contain stem cells highlights the importance of understanding the extrinsic and intrinsic circuitry controlling stem cell fate and their connections to cancer.', 'Genomes are organized into active regions known as euchromatin and inactive regions known as heterochromatin, or silenced chromatin. This review describes contemporary knowledge and models for how silenced chromatin in Saccharomyces cerevisiae forms, functions, and is inherited. In S. cerevisiae, Sir proteins are the key structural components of silenced chromatin. Sir proteins interact first with silencers, which dictate which regions are silenced, and then with histone tails in nucleosomes as the Sir proteins spread from silencers along chromosomes. Importantly, the spreading of silenced chromatin requires the histone deacetylase activity of Sir2p. This requirement leads to a general model for the spreading and inheritance of silenced chromatin or other special chromatin states Such chromatin domains are marked by modifications of the nucleosomes or DNA, and this mark is able to recruit an enzyme that makes further marks. Thus, among different organisms, multiple forms of repressive chromatin can be formed using similar strategies but completely different proteins. We also describe emerging evidence that mutations that cause global changes in the modification of histones can alter the balance between euchromatin and silenced chromatin within a cell.', 'T lymphocytes differentiate in discrete stages within the thymus. Immature thymocytes lacking CD4 and CD8 coreceptors differentiate into double-positive cells (CD4(+)CD8(+)), which are selected to become either CD4(+)CD8(-)helper cells or CD4(-)CD8(+) cytotoxic cells. A stage-specific transcriptional silencer regulates expression of CD4 in both immature and CD4(-)CD8(+) thymocytes. We show here that binding sites for Runt domain transcription factors are essential for CD4 silencer function at both stages, and that different Runx family members are required to fulfill unique functions at each stage. Runxi is required for active repression in CD4(-)CD8(-) thymocytes whereas Runx3 is required for establishing epigenetic silencing in cytotoxic lineage thymocytes. Runx3-deficient cytotoxic T cells, but not helper cells, have defective responses to antigen, suggesting that Runx proteins have critical functions in lineage specification and homeostasis of CD8lineage T lymphocytes.', 'The Cyclin D-Cdk4,6/lNK4/Rb/E2F pathway plays a key role in controlling cell growth by integrating multiple mitogenic and antimitogenic stimuli. The components of this pathway are gene families with a high level of structural and functional redundancy and are expressed in an overlapping fashion in most tissues and cell types. Using classical transgenic technology as well as gene-targeting in ES cells, a series of mouse models have been developed to study the in vivo function of individual components of this pathway in both normal homeostasis and tumor development. These models have proven to be useful to define specific as well as redundant roles among members of these cell cycle regulatory gene families. This pathway is deregulated in the vast majority of human tumors by genetic and epigenetic alterations that target at least some of its key members such as Cyclin D1, Cdk4, INK4a and INK4b, pRb etc. As a consequence, some of these molecules are currently being considered as targets for cancer therapy, and several novel molecules, such as Cdk inhibitors, are under development as potential anti-cancer drugs. (C) 2002 Elsevier Science B.V. All rights reserved.', 'Twin studies have been vital for establishing an important genetic contribution to the etiology of schizophrenia. The five newest studies since 1995 from Europe and Japan have confirmed earlier findings. They yielded probandwise concordance rates of 41-65% in monozygotic (MZ) pairs and 0-28% in dizygotic (DZ) pairs, and heritability estimates of approximately 80-85%. Twin studies are also Valuable for investigating the etiological relationships between schizophrenia and other disorders, and the genetic basis of clinical heterogeneity within schizophrenia. Studies of discordant MZ pairs provide further insights into noninherited factors that contribute to the multifactorial etiology of this disorder. More recently, twin studies have begun to be used to directly investigate molecular genetic and epigenetic processes underlying schizophrenia. (C) 2000 Wiley-Liss, Inc.', 'Interspecific or intergeneric hybridization, followed by chromosome doubling, can lead to the formation of new allopolyploid species. Recent studies indicate that allopolyploid formation is associated with genetic and epigenetic changes, although little is known about the type of changes that occur, how rapidly they occur, and the type of sequences involved. To address these matters, we have surveyed F1 hybrids between diploid species from the wheat (Aegilops and Triticum) group and their derived allotetraploids by screening a large number of loci using amplified fragment length polymorphism and DNA gel blot analysis and by assaying the extent of cytosine methylation. We found that sequence elimination is one of the major and immediate responses of the wheat genome to wide hybridization or allopolyploidy, that it affects a large fraction of the genome, and that it is reproducible. In one cross between Ae. sharonensis x Ae. umbellulata, 14% of the loci from Ae. sharonensis were eliminated compared with only 0.5% from Ae. umbellulata, with most changes occurring in the F1 hybrid. In contrast, crosses between Ae. longissima x T. urartu showed that sequence elimination was more frequent after chromosome doubling. Alterations in cytosine methylation occurred in similar to 13% of the loci, either in the F1 hybrid or in the allopolyploid. For eight of nine bands that were isolated, the sequences that underwent elimination corresponded to low-copy DNA, whereas alterations in methylation patterns affected both repetitive DNA sequences, such as retrotransposons, and low-copy DNA in approximately equal proportions.', 'Cytosine methylation in CpG dinucleotides is believed to be important in gene regulation, and is generally associated with reduced levels of transcription. Methylation-mediated gene silencing involves a series of DNA-protein and protein-protein interactions that begins with the binding of methyl-CpG binding proteins (MBPs) followed by the recruitment of histone-modifying enzymes that together promote chromatin condensation and inactivation. It is widely known that alterations in methylation patterns, and associated gene activities, are often found in human tumors. However, the mechanisms by which methylation patterns are altered are not currently understood. In this paper, we investigate the impact of oxidative damage to a methyl-CpG site on MBP binding by the selective placement of 8-oxoguanine (8-oxoG) and 5-hydroxymethylcytosine (HmC) in a MBP recognition sequence. Duplexes containing these specific modifications were assayed for binding to the methyl-CpG binding domain (MBD) of one member of the MBP family, methyl-CpG binding protein 2 (MeCP2). Our results reveal that oxidation of either a single guanine to 8-oxoG or of a single 5mC to HmC, significantly inhibits binding of the MBD to the oligonucleotide duplex, reducing the binding affinity by at least an order of magnitude. Oxidative damage to DNA could therefore result in heritable, epigenetic changes in chromatin organization.', 'Rett syndrome(1) (RTT, MIM 312750) is a progressive neurodevelopmental disorder and one of the most common causes of mental retardation in females, with an incidence of 1 in 10,000-15,000 (ref. 2). Patients with classic RTT appear to develop normally until 6-18 months of age, then gradually lose speech and purposeful hand use, and develop microcephaly, seizures, autism, ataxia, intermittent hyperventilation and stereotypic hand movements(3). After initial regression, the condition stabilizes and patients usually survive into adulthood. As RTT occurs almost exclusively in females, it has been proposed that RTT is caused by an X-linked dominant mutation with lethality in hemizygous males(3-8). Previous exclusion mapping studies using RTT families mapped the locus to Xq28 (refs 6,7,9-11). Using a systematic gene screening approach, we have identified mutations in the gene (MECP2) encoding X-linked methyl-CpG-binding protein 2 (MeCP2) as the cause of some Eases of RTT. MeCP2 selectively binds CpG dinucleotides in the mammalian genome and mediates transcriptional repression through interaction with histone deacetylase and the corepressor SIN3A (refs 12,13). In 5 of 21 sporadic patients, we found 3 de novo missense mutations in the region encoding the highly conserved methyl-binding domain (MBD) as well as a de novo frameshift and a de novo nonsense mutation, both of which disrupt the transcription repression domain (TRD). In two affected half-sisters of a RTT family, we found segregation of an additional missense mutation not detected in their obligate carrier mother. This suggests that the mother is a germline mosaic for this mutation. Our study reports the first disease-causing mutations in RTT and points to abnormal epigenetic regulation as the mechanism underlying the pathogenesis of RTT.', \"The discovery of numerous hypermethylated promoters of tumour-suppressor genes, along with a better understanding of gene-silencing mechanisms, has moved DNA methylation from obscurity to recognition as an alternative mechanism of tumour-suppressor inactivation in cancer. Epigenetic events can also facilitate genetic damage, as illustrated by the increased mutagenicity of 5-methylcytosine and the silencing of the MLH1 mismatch repair gene by DNA methylation in colorectal tumours. We review here current mechanistic understanding of the role of DNA methylation in malignant transformation, and suggest Knudson's two-hit hypothesis should now be expanded to include epigenetic mechanisms of gene inactivation.\", \"Densely methylated DNA associates with transcriptionally repressive chromatin characterized by the presence of underacetylated histones(1,2). Recently, these two epigenetic processes have been dynamically linked. The methyl-CpG-binding protein MeCP2 appears to reside in a complex with histone deacetylase activity(3,4). MeCP2 can mediate formation of transcriptionally repressive chromatin on methylated promoter templates in vitro, and this process can be reversed by trichostatin A (TSA), a specific inhibitor of histone deacetylase(3-5). Little is known, however, about the relative roles of methylation and histone deacetylase activity in the stable inhibition of transcription on densely methylated endogenous promoters, such as those for silenced alleles of imprinted genes(6), genes on the female inactive X chromosome(7) and tumour-suppressor genes inactivated in cancer cells(8,9). We show here that the hypermethylated genes MLH1, TIMP3 (TIMP3), CDKN2B (INK4B, p15) and CDKN2A (INK4, p16) cannot be transcriptionally reactivated with TSA alone in tumour cells in which we have shown that TSA alone can upregulate the expression of non-methylated genes. Following minimal demethylation and slight gene reactivation in the presence of low dose 5-aza-2'deoxycytidine (5Aza-dC), however, TSA treatment results in robust re-expression of each gene. TSA does not contribute to demethylation of the genes, and none of the treatments alter the chromatin structure associated with the hypermethylated promoters. Thus, although DNA methylation and histone deacetylation appear to act as synergistic layers for the silencing of genes in cancer, dense CpG island methylation is dominant for the stable maintenance of a silent state at these loci.\", ' ', \"'Viable yellow' (A(vy)/alpha) mice are larger, obese, hyperinsulinemic, more susceptible to cancer, and, on average, shorter lived than their non-yellow siblings. They are epigenetic mosaics ranging from a yellow phenotype with maximum ectopic agouti overexpression, through a continuum of mottled agouti/yellow phenotypes with partial agouti overexpression, to a pseudoagouti phenotype with minimal ectopic expression. Pseudoagouti A(vy)/a mice are lean, healthy, and longer lived than their yellow siblings. Here we report that feeding pregnant black a/a dams methyl-supplemented diets alters epigenetic regulation of agouti expression in their offspring, as indicated by increased agouti/black mottling in the direction of the pseudoagouti phenotype, We also present confirmatory evidence that epigenetic phenotypes are maternally heritable. Thus A(vy) expression, already known to be modulated by imprinting, strain-specific modification, and maternal epigenetic inheritance, is also modulated by maternal diet. These observations suggest, at least in this special case, that maternal dietary supplementation may positively affect health and longevity of the offspring, Therefore, this experimental system should be useful for identifying maternal factors that modulate epigenetic mechanisms, especially DNA methylation, in developing embryos.\", 'The yeast Sir2 protein regulates epigenetic gene silencing and as a possible antiaging effect it suppresses recombination of rDNA. Studies involving cobB, a bacterial SIR2-like gene, have suggested it could encode a pyridine nucleotide transferase. Here five human sirtuin cDNAs are characterized. The SIRT1 sequence has the closest homology to the S. cerevisiae Sir2p. The SIRT4 and SIRT5 sirtuins more closely resemble prokaryotic sirtuin sequences. The five human sirtuins are widely expressed in fetal and adult tissues. Recombinant E. coli cobT and cobB proteins each showed a weak NAD-dependent mono-ADP-ribosyltransferase activity using 5,6-dimethylbenzimidazole as a substrate. Recombinant E. coli cobB and human SIRT2 sirtuin proteins were able to cause radioactivity to be transfered from [P-32]NAD to bovine serum albumin (BSA). When a conserved histidine within the human SIRT2 sirtuin was converted to a tyrosine, the mutant recombinant protein was unable to transfer radioactivity from [P-32]NAD to BSA. These results suggest that the sirtuins may function via mono-ADP-ribosylation Of proteins. (C) 1999 Academic Press.', \"Gene activation in eukaryotes requires chromatin remodeling complexes like Swi/Snf and histone acetylases like SAGA. How these factors are recruited to promoters is not yet understood. Using CHIP, we measured recruitment of Swi/Snf, SAGA, the repressor Ash1p, and transcription factors Swi5p and SBF to the HO endonuclease promoter as cells progress through the yeast cell cycle. Swi5p's entry into nuclei at the end of anaphase recruits Swi/Snf, which then recruits SAGA. These two factors then facilitate SBF's binding. Ash1p, which only accumulates in daughter cell nuclei, binds to HO soon after Swi5p and aborts recruitment of Swi/Snf, SAGA, and SBF. Swi5p remains at HO for only 5 min. Swi/Snf's and SAGA's subsequent persistence at HO is self sustaining and constitutes an epigenetic memory of HO's transient interaction with Swi5p.\", 'Altering cytosine methylation by genetic means leads to a variety of developmental defects in mice(1), plants(2-5) and fungi(6,7) Deregulation of cytosine methylation also has a role in human carcinogenesis(8). In some cases, these defects have been tied to the inheritance of epigenetic alterations (such as chromatin imprints and DNA methylation patterns) that do not involve changes in DNA sequence(3,8-10). Using a forward genetic screen, we identified a gene (DDM1, decrease in DNA methylation) from the flowering plant Arabidopsis thaliana required to maintain normal cytosine methylation patterns(11). Additional ddm1 alleles (som4, 5, 6, 7, 8) were isolated in a selection for mutations that relieved transgene silencing(12) (E.J.R., unpublished data). Loss of DDM1 function causes a 70% reduction of genomic cytosine methylation, with most of the immediate hypomethylation occurring in repeated sequences(11). In contrast, many low-copy sequences initially retain their methylation in ddm1 homozygotes, but lose methylation over time as the mutants are propagated through multiple generations by self-pollination(3,13). The progressive effect of ddm1 mutations on low-copy sequence methylation suggests that ddm1 mutations compromise the efficiency of methylation of newly incorporated cytosines after DNA replication. In parallel with the slow decay of methylation during inbreeding, ddm1 mutants accumulate heritable alterations (mutations or stable epialleles) at dispersed sites in the genome that lead to morphological abnormalities(3,5,14). Here we report that DDM1 encodes a SWI2/SNF2-like protein, implicating chromatin remodelling as an important process for maintenance of DNA methylation and genome integrity.', 'Chromatin assembly factor 1 (CAF-1) is required for inheritance of epigenetically determined chromosomal states in vivo and promotes assembly of chromatin during DNA replication in vitro. Herein, we demonstrate that after DNA replication, replicated, but not unreplicated, DNA is also competent for CAF-1-dependent chromatin assembly. The proliferating cell nuclear antigen (PCNA), a DNA polymerase clamp, is a component of the replication-dependent marking of DNA for chromatin assembly. The clamp loader, replication factor C (RFC), can reverse this mark by unloading PCNA from the replicated DNA. PCNA binds directly to p150, the largest subunit of CAF-1, and the two proteins colocalize at sites of DNA replication in cells. We suggest that PCNA and CAF-1 connect DNA replication to chromatin assembly and the inheritance of epigenetic chromosome states.', 'A handful of autosomal genes in the mammalian genome are inherited in a silent state from one of the two parents, and in a fully active form from the other, thereby rendering the organism functionally hemizygous for imprinted genes. To date 19 imprinted genes have been identified; 5 are expressed from the maternal chromosome while the rest are expressed from the paternal chromosome. Allele-specific methylation of CpG residues, established in one of the germlines and maintained throughout embryogenesis, has been clearly implicated in the maintenance of imprinting in somatic cells. Although the function of imprinting remains a subject of some debate, the process is thought to have an important role in regulating the rate of fetal growth.', 'Members of the Plasmodium falciparum var gene family encode clonally variant adhesins, which play an important role in the pathogenicity of tropical malaria. Here we employ a selective panning protocol to generate isogenic P falciparum populations with defined adhesive phenotypes for CD36, ICAM-1 and CSA, expressing single and distinct var gene variants. This technique has established the framework for examining var gene expression, its regulation and switching. It was found that var gene switching occurs irt situ. Ubiquitous transcription of all var gene variants appears to occur in early ring stages. However, var gene expression is tightly regulated in trophozoites and is exerted through a silencing mechanism. Transcriptional control is mutually exclusive in parasites that express defined adhesive phenotypes. lit situ var gene switching is apparently mediated at the level of transcriptional initiation, as demonstrated by nuclear run-on analyses. Our results suggest that an epigenetic mechanism(s) is involved in var gene regulation.', 'Nuclear receptors (NRs) are ligand-inducible transcription factors that mediate complex effects on development, differentiation and homeostasis. They regulate the transcription of their target genes through binding to cognate DNA sequences as homodimers or heterodimers, The molecular mechanisms underlying transcriptional activation by NRs are still poorly understood, although intermediary factors (mediators) appear to be involved in mediating the transactivation functions of NRs, TIF1 has been identified previously as a protein that interacts specifically with the ligand binding domain of several nuclear receptors, both in yeast and in vitro. The characteristics of these interactions have led us to suggest that TIF1 might be a mediator of the NR ligand-inducible activation function AF-2, Using a two-hybrid screening in yeast, we have now identified two TLFP-binding proteins, mHP1 alpha and mMOD1, that are mouse homologues of the Drosophila heterochromatinic protein 1. Using mHP1 alpha as a bait in a second two-hybrid screening, we have isolated cDNAs encoding proteins that are also very likely to be involved in chromatin structure and function, as well as a protein structurally and functionally related to TIF1 (renamed TLF1 alpha), which was named TIF1 beta. Here we discuss how the function of members of the TIF1 family in the control of transcription could be exerted at the level of the structure of the chromatin template.', 'The past few years have seen a wider acceptance of a role for DNA methylation in cancer. This can be attributed to three developments. First, the documentation of the over-representation of mutations at CpG dinucleotides has convincingly implicated DNA methylation in the generation of oncogenic point mutations. The second important advance has been the demonstration of epigenetic silencing of tumor suppressor genes by DNA methylation. The third development has been the utilization of experimental methods to manipulate DNA methylation levels. These studies demonstrate that DNA methylation changes in cancer cells are not mere by-products of malignant transformation, but can play an instrumental role in the cancer process. It seems clear that DNA methylation plays a variety of roles in different cancer types and probably at different stages of oncogenesis. DNA methylation is intricately involved in a wide diversity of cellular processes. Likewise, it appears to exert its influence on the cancer process through a diverse array of mechanisms. It is our task not only to identify these mechanisms, but to determine their relative importance for each stage and type of cancer. Our hope then will be to translate that knowledge into clinical applications.', 'The inactive X chromosome differs from the active X in a number of ways; some of these, such as allocyclic replication and altered histone acetylation, are associated with all types of epigenetic silencing, whereas others, such as DNA methylation, are of more restricted use. These features are acquired progressively by the inactive X after onset of initiation. Initiation of X-inactivation is controlled by the X-inactivation center (Xic) and influenced by the X chromosome controlling element (Xce), which causes primary nonrandom X-inactivation. Other examples of nonrandom X-inactivation are also presented in this review. The definition of a major role for Xist, a noncoding RNA, in X-inactivation has enabled investigation of the mechanism leading to establishment of the heterochromatinized X-chromosome and also of the interactions between X-inactivation and imprinting as well as between X-inactivation and developmental processes in the early embryo.', 'The centromere is required to ensure the equal distribution of replicated chromosomes to daughter nuclei. Centromeres are frequently associated with heterochromatin, an enigmatic nuclear component that causes the epigenetic transcriptional repression of nearby marker genes (position-effect variegation of silencing). The process of chromosome segregation by movement along microtubules to spindle poles is highly conserved, yet the putative cis-acting centromeric DNA sequences bear little or no similarity across species. Recently, studies in several systems have revealed that the centromere itself might be epigenetically regulated and that the higher-order structure of the underlying heterochromatin contributes to centromere function and kinetochore assembly.', 'The chrome and SET domains are conserved sequence moths present in chromosomal proteins that function in epigenetic control of gene expression, presumably by modulating higher order chromatin, Based on sequence information from the SET domain, we have isolated human (SUV39H1) and mouse (Suv39h1) homologues of the dominant Drosophila modifier of position-effect-variegation (PEV) Su(var)3-9. Mammalian homologues contain, in addition to the SET domain, the characteristic chrome domain, a combination that is also preserved in the Schizosaccharyomyces pombe silencing factor clr4. Chromatin-dependent gene regulation is demonstrated by the potential of human SUV39H1 to increase repression of the pericentromeric white marker gene in transgenic flies. Immunodetection of endogenous Suv39h1/SUV39H1 proteins in a variety of mammalian cell lines reveals enriched distribution at heterochromatic foci during interphase and centromere-specific localization during metaphase, In addition, Suv39h1/SUV39H1 proteins associate with M31, currently the only other characterized mammalian SU(VAR) homologue, These data indicate the existence of a mammalian SU(VAR) complex and define Suv39h1/SUV39H1 as novel components of mammalian higher order chromatin.', 'post-transcriptional gene silencing (PTGS) is a homology-dependent process that reduces cytoplasmic RNA levels. In several experimental systems, there is also an association of PTGS with methylation of DNA. To investigate this association, we used plants carrying a transgene encoding the green fluorescent protein (GFP), Gene silencing was induced using potato virus X RNA vectors carrying parts of the coding sequence or the promoter of the GFP transgene. In each instance, homology-based, RNA-directed methylation was associated with silencing. When the GFP-transcribed region was targeted, PTGS affected both transgene and viral RNA revels. When methylation was targeted to a promoter region, transgene RNA levels were reduced; however, viral RNA revels were unaffected. For comparison, we induced PTGS of the gene encoding the endogenous ribulose-1,5-bisphosphate carboxylase oxygenase (Rubisco) small subunit (rbcS) by inoculation with potato virus X-rbcS, In this example, no methylation of the rbcS DNA was associated with the reduction in rbcS transcript levels, and viral RNA levels were unaffected. Finally, we investigated DNA methylation by using GFP-transformed plants in which PTGS was induced by localized introduction of a T-DNA carrying GFP sequences. In these plants, there was methylation of a GFP transgene associated with systemic spread of a gene-silencing signal from the infiltrated part of the plant. This transgene methylation was not affected when systemic PTGS was blocked by suppressors of silencing encoded by potato virus Y and cucumber mosaic virus. Combined, these data support an epigenetic model of PTGS in which transgene methylation is associated with an RNA-DNA interaction that ensures that PTGS is maintained.', 'Although the link between the BRCA1 tumour-suppressor gene and hereditary breast and ovarian cancer is established(1-5), the role, if any, of BRCA1 in non-familial cancers is unclear. BRCA1 mutations are rare in sporadic cancers(6-8), but loss of BRCA1 resulting from reduced expression or incorrect subcellular localization(9,10) is postulated to be important in non-familial breast and ovarian cancers. Epigenetic loss, however, has not received general acceptance due to controversy regarding the subcellular localization of BRCA1 proteins, reports of which have ranged from exclusively nuclear(11-15), to conditionally nuclear(10), to the ER/golgi(16), to cytoplasmic invaginations into the nucleus(17). In an attempt to resolve this issue, we have comprehensively characterized 19 anti-BRCA1. antibodies. These reagents detect a 220-kD protein localized in discrete nuclear foci in all epithelial cell lines, including those derived from breast malignancies. Immunohistochemical staining of human breast specimens also revealed BRCA1 nuclear foci in benign breast, invasive lobular cancers and low-grade ductal carcinomas. Conversely, BRCA1 expression was reduced or undetectable in the majority of high-grade, ductal carcinomas, suggesting that absence of BRCA1 may contribute to the pathogenesis of a significant percentage of sporadic breast cancers.', 'CpG island hypermethylation is known to be associated with gene silencing in cancer. This epigenetic event is generally accepted as a stochastic process in tumor cells resulting from aberrant DNA methyltransferase (DNA-MTase) activities. Specific patterns of CPG island methylation could result from clonal selection of cells having growth advantages due to silencing of associated tumor suppressor genes. Alternatively, methylation patterns may be determined by other, as yet unidentified factors, To explore further the underlying mechanisms, we developed a novel array-based method, called differential methylation hybridization (DMH), which allows a genome-wide screening of hypermethylated CpG islands in tumor cells. DMH was used to determine the methylation status of >276 CpG island loci in a group of breast cancer cell lines. Between 5 and 14% of these loci were hypermethylated extensively in these cells relative to a normal control. Pattern analysis of 30 positive loci by Southern hybridization indicated that CpG islands might differ in their susceptibility to hypermethylation, Loci exhibiting preexisting methylation in normal controls were more susceptible to de novo methylation in these cancer cells than loci without this condition, In addition, these cell lines exhibited different intrinsic abilities to methylate CpG islands not directly associated with methyltransferase activities. Our study provides evidence that, aside from random DNA-MTase action, additional cellular factors exist that govern aberrant methylation in breast cancer cells.', 'Genomic reprogramming of primordial germ cells (PGCs), which includes genome-wide demethylation, prevents aberrant epigenetic modifications from being transmitted to subsequent generations, This process also ensures that homologous chromosomes first acquire an identical epigenetic status before an appropriate switch in the imprintable loci in the female and male germ lines. Embryonic germ (EG) cells have a similar epigenotype to PGCs from which they are derived, We used EG cells to investigate the mechanism of epigenetic modifications in the germ line by analysing the effects on a somatic nucleus in the EG-thymic lymphocyte hybrid cells. There were striking changes in methylation of the somatic nucleus, resulting in demethylation of several imprinted and non-imprinted genes, These epigenetic modifications were heritable and affected gene expression as judged by re-activation of the silent maternal allele of Peg1/Mest imprinted gene in the somatic nucleus. This remarkable change in the epigenotype of the somatic nucleus is consistent with the observed pluripotency of the EG-somatic hybrid cells as they differentiated into a variety of tissues in chimeric embryos. The epigenetic modifications observed in EG-somatic cell hybrids in vitro are comparable to the reprogramming events that occur during germ cell development.', 'To determine which CD3 components are required for early T cell development, we generated mice with a targeted mutation of the CD3-epsilon gene and characterized their T cell populations relative to those found in CD3-zeta/eta- and recombinase activating gene (RAG)-deficient mice, In the absence of intact CD3-epsilon subunit, thymocytes do not progress beyond the CD44(-/low)CD25(+) triple-negative stage and appear to be arrested at the very same developmental control point as RAG-deficient thymocytes, In contrast, the disruption of the CD3-zeta/eta gene does not totally abrogate the progression through this control point, CD3-epsilon-deficient thymocytes do rearrange their T cell receptor (TCR) beta gene segments and produce low levels of full-length TCR beta transcripts. Taken together, these results establish an essential role for the CD3-epsilon gene products during T cell development and further suggest that the CD3-epsilon polypeptides start to exert their function as part of a pre-TCR through which CD44(-/low)CD25(+) triple-negative cells monitor the occurrence of productive TCR beta gene rearrangements. Finally, the absence of intact CD3-epsilon polypeptides had no discernible effect on the completion of TCR gamma and TCR delta gene rearrangements, emphasizing that they are probably not subjected to the same epigenetic controls as those operating on the expression of TCR alpha and beta genes.', \"The constitutive DNase I hypersensitive site at the 5' end of the chicken beta-globin locus marks the boundary of the active chromatin domain in erythroid cells. The DNA sequence containing this site has the properties of an insulator, as shown by its ability in stable transformation experiments to block enhancer-promoter interaction when it lies between the two, but not when it lies outside, and to protect against position effects in Drosophila. We now show that the chicken insulator can protect a stably integrated gene, which is otherwise subject to great variability of expression, from chromatin-mediated repression in cell culture. When the integrated reporter gene is surrounded by insulator elements, stably transformed cell lines display consistent enhancer-dependent expression levels, in accord with the strength of the enhancer. In the absence of insulators, long-term nonselective propagation of cells carrying the integrated reporter gene results in gradual extinction of the reporter's expression, with expression patterns from tandemly repeated inserted genes suggesting that the extinction of adjacent genes is coupled. We show that the uninsulated reporter genes, in addition to becoming transcriptionally inactive, lose several epigenetic hallmarks of active chromatin, including nuclease accessibility, DNA hypomethylation, and histone hyperacetylation during time in culture. Treatment with inhibitors of histone deacetylase or DNA methylation reverses the extinction of the uninsulated genes. Extinction is completely prevented by flanking the reporter construct with insulators. Furthermore, in contrast to the uninsulated reporter genes, chromatin over the insulated genes retains nuclease accessibility and histone hyperacetylation. However, there is no clear correlation between the presence of the insulators and the level of DNA methylation. This leads us to propose a model for the insulator's ability to protect against extinction in the transformed cell lines and to function as a chromatin boundary for the chicken beta-globin locus in normal erythroid cells.\", 'Kruppel-associated box (KRAB) domains are present in approximately one-third of all human zinc finger proteins (ZFPs) and are potent transcriptional repression modules. We have previously cloned a corepressor for the KRAB domain, KAP-1, which is required for KRAB-mediated repression in vivo. To characterize the repression mechanism utilized by KAP-1, we have analyzed the ability of KAP-1 to interact with murine (M31 and M32) and human (HP1 alpha and HP1 gamma) homologues of the HPI protein family, a class of nonhistone heterochromatin-associated proteins with a well-established epigenetic gene silencing function in Drosophila, In vitro studies confirmed that KAP-1 is capable of directly interacting with M31 and hHP1 alpha, which are normally found in centromeric heterochromatin, as well as M32 and hHP1 gamma, both of which are found in euchromatin, Mapping of the region in KAP-1 required for HPI interaction showed that amino acid substitutions which abolish HP1 binding in vitro reduce KAP-1 mediated repression in vivo. We observed colocalization of KAP-1 with M31 and M32 in interphase nuclei, lending support to the biochemical evidence that M31 and M32 directly interact with KAP-1, The colocalization of KAP-1 with M31 is sometimes found in subnuclear territories of potential pericentromeric heterochromatin, whereas colocalization of KAP-1 and M32 occurs in punctate euchromatic domains throughout the nucleus. This work suggests a mechanism for the recruitment of HP1-like gene products by the KRAB-ZFP-KAP-1 complex to specific loci within the genome through formation of heterochromatin-like complexes that silence gene activity. We speculate that gene-specific repression may be a consequence of the formation of such complexes, ultimately leading to silenced genes in newly formed heterochromatic chromosomal environments.', \"Flower pigmentation patterns were scored in 185 sense Chalcone synthase (Chs) transgenotes and 85 antisense Chs transgenotes; upon first flowering, 139 (75%) of sense transgenotes were found to be phenotypically altered, as were 70 (82%) of the antisense transgenotes. The observed patterns document the range of phenotypic variations that occur, as well as confirm and extend the finding that sense Chs constructs produce several types of morphology-based flower pigmentation patterns that antisense Chs constructs do not. Long-term monitoring for epigenetic variations in one population of 44 sense Chs transgenotes showed that 43 (98%) were capable of producing a cosuppression phenotype. The primary determinant of sense-specific patterns of cosuppression of Chs was found to be the repetitiveness and organization pattern of the transgene, not 'position effects' by, or 'readthrough' from, flanking plant DNA sequences. The degree of cosuppression observed in progeny of transgenotes carrying multiple, dispersed copies as compared to that observed with a single copy of the transgene suggests that sense cosuppression of Chs is subject to a transgene dosage effect.\", 'A transgenic gpt(+) Chinese hamster cell line (G12) was found to be susceptible to carcinogenic nickel-induced inactivation of gpt expression without mutagenesis or deletion of the transgene. Many nickel-induced 6-thioguanine-resistant variants spontaneously reverted to actively express gpt, as indicated by both reversion assays and direct enzyme measurements. Since reversion was enhanced in many of the nickel-induced variant cell lines following 24-h treatment with the demethylating agent 5-azacytidine, the involvement of DNA methylation in silencing gpt expression was suspected. This was confirmed by demonstrations of increased DNA methylation, as well as by evidence indicating condensed chromatin and heterochromatinization of the gpt integration site in 6 thioguanine-resistant cells. Upon reversion to active gpt expression, DNA methylation and condensation are lost. We propose that DNA condensation and methylation result in heterochromatinization of the gpt sequence with subsequent inheritance of the now silenced gene. This mechanism is supported by direct evidence showing that acute nickel treatment of cultured cells, and of isolated nuclei in vitro, can indeed facilitate gpt sequence-specific chromatin condensation. Epigenetic mechanisms have been implicated in the actions of some nonmutagenic carcinogens, and DNA methylation changes are now known to be important in carcinogenesis. This paper further supports the emerging theory that nickel is a human carcinogen that can alter gene expression by enhanced DNA methylation and compaction, rather than by mutagenic mechanisms.', 'In vitro manipulation of preimplantation mammalian embryos can influence differentiation and growth at later stages of development, In the mouse, culture of embryonic stem (ES) cells affects their totipotency and may give rise to fetal abnormalities, To investigate whether this is associated with epigenetic alterations in imprinted genes, we analysed two maternally expressed genes (Igf2r, H19) and two paternally expressed genes (Igf2, U2af1-rs1) in ES cells and in completely ES cell-derived fetuses, Altered allelic methylation patterns were detected in all four genes, and these were consistently associated with allelic changes in gene expression, All the methylation changes that had arisen in the ES cells persisted on in vivo differentiation to fetal stages, Alterations included loss of methylation with biallelic expression of U2af1-rs1, maternal methylation and predominantly maternal expression of Igf2, and biallelic methylation and expression of Igf2r In many of the ES fetuses, the levels of H19 expression were strongly reduced, and this biallelic repression was associated with biallellic methylation of the H19 upstream region, Surprisingly, biallelic H19 repression was not associated with equal levels of Igf2 expression from both parental chromosomes, but rather with a strong activation of the maternal Igf2 allele, ES fetuses derived from two of the four ES lines appeared developmentally compromised, with polyhydramnios, poor mandible development and interstitial bleeding and, in chimeric fetuses, the degree of chimerism correlated with increased fetal mass, Our study establishes a model for how early embryonic epigenetic alterations in imprinted genes persist to later developmental stages, and are associated with aberrant phenotypes.', 'Polycomb group (PcG) and trithorax group (trxG) gene products are responsible for the maintenance of repressed and active expression patterns of many developmentally important regulatory genes including the homeotic genes. In Drosophila embryos, Polycomb protein and the trxG protein GAGA factor colocalize at the Fab-7 DNA element of the bithorax complex. In transgenic lines, the Fab-7 element induces extensive silencing on a flanking GAL4-driven lacZ reporter and mini-white genes. However, a short single pulse of GAL4 during embryogenesis is sufficient to release PcG-dependent silencing from the transgene. Such an activated state of Fab-7 is mitotically inheritable through development and can be transmitted in a GAL4-independent manner to the subsequent generations through female meiosis. Thus, Fab-7 is a switchable chromosomal element, which can convey memory of epigenetically determined active and repressed chromatin states.', 'Methylation of cytosine residues in DNA provides a mechanism of gene control. There are two classes of methyltransferase in Arabidopsis; one has a carboxy-terminal methyltransferase domain fused to an amino-terminal regulatory domain and is similar to mammalian methyltransferases. The Second class apparently lacks an amino-terminal domain and is less well conserved. Methylcytosine can occur at any cytosine residue, but it is likely that clonal transmission of methylation patterns only occurs for cytosines in strand-symmetrical sequences CpG and CpNpG. In plants, as in mammals, DNA methylation has dual roles in defense against invading DNA and transposable elements and in gene regulation. Although originally reported as having no phenotypic consequence, reduced DNA methylation disrupts normal plant development.', 'The trilaminar kinetochore directs the segregation of chromosomes in mitosis and meiosis. Despite its importance, the molecular architecture of this structure remains poorly understood [1]. The best known component of the kinetochore plates is CENP-C, a protein that is required for kinetochore assembly [2], but whose molecular role in kinetochore structure and function is unknown. Here we have raised for the first time monospecific antisera to CENP-A [3], a 17 kD centromere-specific histone variant that is 62% identical to the carboxy-terminal domain of histone H3 [4,5] and that resembles the yeast centromeric component CSE4 [6]. We have found by simultaneous immunofluorescence with centromere antigens of known ultrastructural location that CENP-A is concentrated in the region of the inner kinetochore plate at active centromeres. Because CENP A was previously shown to co-purify with nucleosomes [7], our data suggest a specific nucleosomal substructure for the kinetochore. In human cells, these kinetochore-specific nucleosomes are enriched in alpha-satellite DNA [8]. However, the association of CENP-A with neocentromeres lacking detectable alpha-satellite DNA, and the lack of CENP-A association with alpha-satellite-rich inactive centromeres of dicentric chromosomes together suggest that CENP-A association with kinetochores is unlikely to be determined solely by DNA sequence recognition. We speculate that CENP-A binding could be a consequence of epigenetic tagging of mammalian centromeres.', \"Nucleolar dominance is an epigenetic phenomenon that describes nucleolus formation around rRNA genes inherited from only one progenitor of an interspecific hy;brid or allopolyploid. The phenomenon is widespread, occurring in plants, insects, amphibians, and mammals, yet its molecular basis remains unclear. We have demonstrated nucleolar dominance in three allotetraploids of the plant genus Brassica. In Brassica napus, accurately initiated pre-rRNA transcripts from one progenitor, Brassica rapa are detected readily, whereas transcripts from the similar to 3000 rRNA genes inherited from the other progenitor, Brassica oleracea, are undetectable. Nuclear run-on confirmed that dominance is controlled at the level of transcription. Growth of B. napus seedlings on 5-aza-2'-deoxycytidine to inhibit cytosine methylation caused the normally silent, under-dominant B. oleracea rRNA genes to become expressed to high levels. The histone deacetylase inhibitors sodium butyrate and trichostatin A also de-epressed silent rRNA genes. These results reveal an enforcement mechanism for nucleolar dominance in which DNA methylation and histone modifications combine to regulate rRNA gene loci spanning tens of megabase pairs of DNA.\", 'Genomic imprinting is an epigenetic chromosomal modification in the gamete or zygote causing preferential expression of a specific parental allele in somatic cells of the offspring. We and others have identified three imprinted human genes on 11p15.5, IGF2 (refs 1-4), H19 (refs 1,5), and p57(KIP2) (ref. 6), although the latter gene is separated by 700 kb from the other two, and it is unclear whether there are other imprinted genes within this large interval. We previously mapped an embryonal tumour suppressor gene to this region(7), as well as five balanced germline chromosomal rearrangement breakpoints from patients with Beckwith-Wiedemann syndrome (BWS)(8), a condition characterized by prenatal overgrowth and cancer. We isolated the upstream exons of the previously identified gene KVLQT1, which causes the familial cardiac defect long-QT (LQT) syndrome. We found that KVLQT1 spans much of the interval between p57(KIP2) and IGF2, and that it is also imprinted. We demonstrated that the gene is disrupted by chromosomal chromosomal translocation in an embryonal rhabdoid tumour. Furthermore, the lack of parent-of-origin effect in LQT syndrome appears to be due to relative lack of imprinting in the affected tissue, cardiac muscle, representing a novel mechanism for variable penetrance of a human disease gene.', 'A multisubunit cohesin complex holds sister chromatids together after DNA replication. Using chromatin immunoprecipitation, we detected cohesin association with centromeres and with discrete sites along chromosome arms from S phase until metaphase in S. cerevisiae, Short DNA sequences (130-280 bp) are sufficient to confer cohesin association. Cohesin association with a centromere depends on Mif2p, the centromere binding factor CBF3, and a centromere-specific histone variant, Cse4p. Because only active centromeres confer cohesin association with centromeric DNA, we suggest that cohesin is recruited by the same chromatin structure that confers the attachment of microtubules. Propagation of this structure might be partly epigenetic. Finally, cohesion associated with minimal centromeres is insufficient to resist the splitting force exerted by microtubules and appears to be reinforced by cohesion provided by their flanking DNA sequences.', 'Loss of imprinting (LOI) is an epigenetic alteration of some cancers involving loss of parental origin-specific expression of imprinted genes. We observed LOI of the insulin-like growth factor-ii gene in twelve of twenty-seven informative colorectal cancer patients (44%), as well as in the matched normal colonic mucosa of the patients with LOI in their cancers, and in peripheral blood samples of four patients. Ten of eleven cancers (91%) with microsatellite instability showed LOI, compared with only two of sixteen tumors (12%) without microsatellite instability (P< 0.001). Control patients without cancer showed LOI in colonic mucosa of only two of sixteen cases (12%, P < 0.001) and two of fifteen blood samples (13%, P < 0.001). These data suggest that LOI in tumor and normal tissue identifies most colorectal cancer patients with microsatellite instability in their tumors, and that LOI may identify an important subset of the population with cancer or at risk of developing cancer.', 'Silencing is an epigenetic form of transcriptional regulation whereby genes are heritably, but not necessarily permanently, inactivated. We have identified the Saccharomyces cerevisiae genes SAS2 and SAS3 through a screen for enhancers of sir1 epigenetic silencing defects. SAS2, SAS3 and a Schizosaccharomyces pombe homologue are closely related to several human genes, including one associated with acute myeloid leukaemia arising from the recurrent translocation t(8;16)(p11;p13) and one implicated in HIV-1 Tat interactions. All of these genes encode proteins with an atypical zinc finger and well-conserved similarities to acetyltransferases. Sequence similarities and yeast mutant phenotypes suggest that SAS-like genes function in transcriptional regulation and cell-cycle exit and reveal novel connections between transcriptional silencing and human disease.', 'The multisubunit MSL dosage compensation complex binds to hundreds of sites along the Drosophila single male X chromosome, mediating its hypertranscription, The male X chromosome is also coated with noncoding roX RNAs. When either msl3, mle, or mof is mutant, a partial MSL complex is bound at only similar to 35 unusual sites distributed along the X, We show that two of these sites are the roX1 and roX2 genes and postulate that one of their functions is to provide entry sites for the MSL complex to recognize the X chromosome. The roX1 gene provides a nucleation site for extensive spreading of the MSL complex into flanking chromatin even when moved to an autosome. The spreading can occur in cis or in trans between paired homologs. We present a model for how the dosage compensation complex recognizes X chromatin.', 'Defective DNA mismatch repair in human turners leads to genome-wide instability of microsatellite repeats and a molecular phenotype referred to as microsatellite instability (MSI), MSI has been reported in a variety of cancers and is a consistent feature of tumors from patients with hereditary non-polyposis colorectal cancer. Approximately 20% of cancers of the uterine endometrium, the fifth most common cancer of women world-wide, exhibit MSI. Although the frequency of MSI is higher in endometrial cancers than in any other common malignancy, the genetic basis of MSI in these tumors has remained elusive, We investigated the role that methylation of the MLH1 DNA mismatch repair gene plays in the genesis of MSI in a large series of sporadic endometrial cancers. The MLH1 promoter was methylated in 41 of 53 (77%) MSI-positive cancers investigated. In MSI-negative tumors on the other hand, there was evidence for limited methylation in only one of 11 tumors studied. Immunohistochemical investigation of a subset of the tumors revealed that methylation of the MLH1 promoter in MSI-positive tumors was associated with loss of MLH1 expression. Immunohistochemistry proved that two MSI-positive tumors lacking MLH1 methylation failed to express the MSH2 mismatch repair gene. Both of these cancers came from women who had family and medical histories suggestive of inherited cancer susceptibility. These observations suggest that epigenetic changes in the MLH1 locus account for MSI in most cases of sporadic endometrial cancers and provide additional evidence that the MSH2 gene may contribute substantially to inherited forms of endometrial cancer.', 'Several lines of evidence strongly suggest that DNA methylation is involved in embryo development, Perhaps the most direct evidence comes from experiments with methyltransferase deficient mice. Embryos that express low levels of the maintenance methyltransferase do not develop to term and die at the 5 to 20 somite stage corresponding to the level of the enzyme. In the mouse, dramatic methylation changes have been observed during the early steps of embryo development. Most genes are subject to a process of active demethylation starting promptly after fertilization. Except for a small number of methylated CpG sites in imprinted genes the vast majority of the sites are unmethylated by the stage of cavitation (16 cell). Such genome-wide demethylation may signify an erasure of epigenetic information originating in the highly differentiated gametes. This erasure may be essential for the establishment of a pluripotent state, before specific cell lineages can be determined. The process of laying down a new developmental program involves, initially, global de novo methylation at the stage of pregastrulation followed by gene specific demethylations associated with the onset of activity of these genes. CpG islands characteristic of housekeeping genes, appear to be protected from the global de novo methylation. An exception to this rule is the hypermethylation of CpG islands in X-linked housekeeping genes on the inactive X chromosome and of specific differentially methylated CpG sites in imprinted genes. Primordial germ cells escape the global de novo methylation which takes place at the pregastrula stage and undergo a very similar de novo methylation process in the differentiated gonads (15.5-18.5 days post coitum), forming the methylation patterns which are specific to the gametes. Specific demethylations then form a terminal methylation pattern which is then clonaly inherited in the soma and erased after fertilization.', \"During the development of a multicellular organism from a zygote, a large number of epigenetic interactions take place on every level of suborganismal organization. This raises the possibility that the system of epigenetic interactions may compensate or ''buffer'' some of the changes that occur as mutations on its lowest levels, and thus stabilize the phenotype with respect to mutations. This hypothetical phenomenon will be called ''epigenetic stability.'' Its potential importance stems from the fact that phenotypic variation with a genetic basis is an essential prerequisite for evolution. Thus, variation in epigenetic stability might profoundly affect attainable rates of evolution. While representing a systemic property of a developmental system, epigenetic stability might itself be genetically determined and thus be subject to evolutionary change, Whether or not this is the case should ideally be answered directly, that is, by experimentation. The time seals involved and our insufficient quantitative understanding of developmental pathways will probably preclude such an approach in the foreseeable future. Preliminary answers are sought here by using a biochemically motivated model of a small bur central part of a developmental pathway. Modeled are sets of transcriptional regulators that mutually regulate each other's expression and thereby form stable gene expression patterns. Such gene-expression patterns, crucially involved in determining developmental pattern formation events, are most likely subject to strong stabilizing natural selection, After long periods of stabilizing selection. the fraction of mutations causing changes in gene-expression pattern is substantially reduced in the model. Epigenetic stability has increased. This phenomenon is found for widely varying regulatory scenarios among transcription factor genes. Its discussed that only epistatic (nonlinear) gene interactions can cause such change in epigenetic stability. Evidence from paleontology, molecular evolution, development, and genetics, consistent with the existence of variation in epigenetic stability, is discussed, The relation of epigenetic stability to developmental canalization is outlined. Experimental scenarios at suggested thar may provide further evidence.\", 'X-chromosome inactivation is the process by which a cell recognizes the presence of two copies of an X chromosome early in the development of XX embryos and chooses one to be active and one to be inactive(1) Although it is commonly believed that the initiation of X inactivation is random, with an equal probability (50:50) that either X chromosome will be the inactive X in a given cell, significant variation in the proportion of cells with either X inactive is observed both in mice heterozygous for alleles at the Xce locus(2) and among normal human females in the population(3-5). Families in which multiple females demonstrate extremely skewed inactivation patterns that are otherwise quite rare in the general population are thought to reflect possible genetic influences on the X-inactivation process(5-7). Here we report a rare cytosine to guanine mutation in the XIST minimal promoter that underlies both epigenetic and functional differences between the two X chromosomes in nine females from two unrelated families. All females demonstrate preferential inactivation of the X chromosome carrying the mutation, suggesting that there is an association between alterations in the regulation of XIST expression and X-chromosome inactivation.', 'Genomic imprinting is an epigenetic process that results in the preferential silencing of one of the two parental copies of a gene. Although the precise mechanisms by which genomic imprinting occurs are unknown, the tendency of imprinted genes to exist in chromosomal clusters suggests long-range regulation through shared regulatory elements. We characterize a 800-kb region on the distal end of mouse chromosome 7 that contains a cluster of four maternally expressed genes, H19, Mash2, Kvlqt1, and p57(Kip2), as well as two paternally expressed genes, Igf2 and Ins2, and assess the expression and imprinting of Mash2, Kvlqt1, and p57(Kip2) during development in embryonic and extraembryonic tissues. Unlike Igf2 and Ins2, which depend on H19 for their imprinting, Mash2, p57(Kip2), and Kvlqt1 are unaffected by a deletion of the H19 gene region, suggesting that these more telomeric genes are not regulated by the mechanism that controls H19, Igf2, and Ins2. Mutations in human p57(Kip2) have been implicated in Beckwith-Wiedemann syndrome, a disease that has also been associated with loss of imprinting of IGF2. We find, however, that a deletion of the gene has no effect on imprinting within the cluster. Surprisingly, the three maternally expressed genes are regulated very differently by DNA methylation; p57(Kip2) activated, Kvlqt1 is silenced, and Mash2 is unaffected in mice lacking DNA methyltransferase. We conclude that H19 is not a global regulator of imprinting on distal chromosome 7 and that the telomeric genes are imprinted by a separate mechanism(s).', \"Skeletal myoblasts are derived from paraxial mesoderm, but how myoblasts acquire their identity is still a matter of speculation. The characterization of molecular markers and, in some cases, the analysis of mutations in the corresponding genes, has now made it possible to ask specific questions about this process. Specification of somite cell fate depends on epigenetic factors. Adjacent tissues, such as the neural tube, notochord, dorsal ectoderm and lateral mesoderm, act either positively or negatively on the different myogenic precursor populations in the somite. Candidate molecules for this complex signalling activity include sonic hedgehog and the Wnt proteins as positive signals, and BMP4 as a possible inhibitor. Although it is generally assumed that induction is required, some observations suggest that embryonic cells might have a tendency to undergo myogenesis as a 'default' pathway. By analogy with Drosophila, where the neurogenic genes affect myogenesis, the vertebrate homologues of notch and its ligands could be candidate molecules for a repression or derepression mechanism. Similar studies with cultured muscle cells also implicate other HLH factors as potential inhibitors of the MyoD family and, hence, of inappropriate myogenesis.\", 'Overexpression of chimeric transgenes in plants can trigger post-transcriptional gene silencing that is dependent on epigenetic information and physiological conditions. The current view is that unproductive RNA serves as a crucial signal for gene silencing, although direct evidence is lacking for this theory. A signalling cascade then leads to strongly enhanced turnover of all RNAs that share a critical degree of sequence similarity. The molecular details of the mechanism are, however, insufficiently understood to explain the phenomenon completely and to comprehend its biological significance.', ' ', 'Photocarcinogenesis represents the sum of a complex of simultaneous and sequential biochemical events that ultimately lead to the occurrence of skin cancer. These events, initiated by UV radiation of appropriate wavelength, include the formation of DNA photoproducts: DNA repair; mutation of proto-oncogenes and tumor suppressor genes; UV-production of radical species with subsequent effects on mutation and extra-nuclear function; and other epigenetic events that influence the course of carcinogenesis. The epigenetic influences may include immunological responses, antioxidant defenses, and dietary factors, This review represents an effort to provide current research results in the aforementioned areas and an attempt to meld these events into a comprehensive overview of photocarcinogenesis. If effective prevention and intervention strategies for skin cancer are to be developed, a more thorough understanding of the disease process is imperative. (C) 1997 Elsevier Science S.A.', ' ', 'Several proteins that contribute to epigenetic mechanisms of gene regulation contain a characteristic motif of unknown function called the SET (Suvar3-9, Enhancer-of-zeste, Trithorax) domain. We have demonstrated that SET domains mediate highly conserved interactions with a specific family of proteins that display similarity with dual-specificity phosphatases (dsPTPases). These include myotubularin, the gene of which is mutated in a subset of patients with X-linked myotubular myopathy, and Sbf1, a newly isolated homologue of myotubularin. In contrast with myotubularin, Sbf1 lacks a functional catalytic domain which dephosphorylates phospho-tyrosine and serine-containing peptides in vitro. Competitive interference of endogenous SET domain-dsPTPase interactions by forced expression of Sbf1 induced oncogenic transformation of NIH 3T3 fibroblasts and impaired the in vitro differentiation of C2 myoblast cells. We conclude that myotubularin-type phosphatases link SET-domain containing components of the epigenetic regulatory machinery with signalling pathways involved in growth and differentiation.', 'O-6-Methylguanine DNA methyltransferase (MGMT) repairs the mutagenic and cytotoxic O-6-alkylguanine lesions produced bg environmental carcinogens and the chemotherapeutic nitrosoureas, As such, MGMT-mediated repair of O-6-alkylguanine lesions constitutes a major form of resistance to nitrosourea chemotherapy and makes control of MGMT expression of clinical interest, The variability of expression in cell lines and tissues, along with the ease with which the MGMT phenotype reverts under various conditions, suggests that MGMT is under epigenetic control, One such epigenetic mechanism, 5-methylation of cytosines, has been linked to MGMT expression, We have used an isogenic human multiple myeloma tumor cell line model composed of an MGMT-positive parent cell line, RPMI 8226/S, and its MGMT-negative variant, termed 8226/V, to study the control of MGMT expression. The loss of MGMT activity in 8226/V was found to he due to the loss of detectable MGMT gene expression, Bisulfite sequencing of the MGMT CpG island promoter revealed large increases in the levels of CpG methylation within discrete regions of the 8226/V MGMT CpG island compared to those in 8226/S, These changes in CpG methylation are associated with local heterochromatinization of the 8226/V MGMT transcription start site and provide a likely mechanism for the loss of MGMT transcription in 8226/V.', 'Inheritance of the active and inactive states of gene expression by individual cells is crucial for development. In fission yeast, mating-type region consists of three loci called mat1, mat2, and mat3. Transcriptionally silent mat2 and mat3 loci are separated by a 15 kb interval, designated the K-region, and serve as donors of information for transcriptionally active mat1 interconversion. In a strain carrying replacement of 7.5 kb of the K-region with the ura4 gene, we discovered that ura4 silencing and efficiency of mating-type switching were covariegated and were regulated by an epigenetic mechanism. Genetic analyses demonstrated that epigenetic states were remarkably stable not only in mitosis but also in meiosis and were linked to the mating-type region. This study indicates that different epigenetic states are heritable forms of chromatin organization at the mat region.', 'Microdeletions of a region termed the imprinting center (IC) in chromosome 15q11-q13 have been identified in several families with Prader-Willi syndrome (PWS) or Angelman syndrome who show epigenetic inheritance for this region that is consistent with a mutation in the imprinting process. The IC controls resetting of parental imprints in 15q11-q13 during gametogenesis. We have identified a larger series of cases of familial PWS, including one case with a deletion of only 7.5 kb, that narrows the PWS critical region to <4.3 kb spanning the SNRPN gene CpG island and exon 1. Identification of a strong DNase I hypersensitive site, specific for the paternal allele, and six evolutionarily conserved (human-mouse) sequences that are potential transcription-factor binding sites is consistent with this region defining the SNRPN gene promoter. These findings suggest that promoter elements at SNRPN play a key role in the initiation of imprint switching during spermatogenesis. We also identified three patients with sporadic PWS who have an imprinting mutation (IM) and no detectable mutation in the IC. An inherited 15q11-q13 mutation or a trans-factor gene mutation are unlikely; thus, the disease in these patients may arise from a developmental or stochastic failure to switch the maternal-to-paternal imprint during parental spermatogenesis. These studies allow a better understanding of a novel mechanism of human disease, since the epigenetic effect of an IM in the parental germ line determines the phenotypic effect in the patient.', 'The Wassilewskija strain of Arabidopsis has four genes encoding the tryptophan enzyme phosphoribosylanthranilate isomerase (PAI) located at three unlinked sites, These four PAI genes are methylated over their regions of DNA homology, When PAI copy number is reduced by deletion of two tandemly arrayed genes (MePAI1-PAI4), a mutant with fluorescent, tryptophan-deficient phenotypes results, because the two remaining methylated PAI genes (MePAI2 and MePAI3) supply insufficient PAI activity, These two methylated genes can be inherited through meiosis, even when they are segregated away from each other in crosses to a strain with unmethylated PAI genes, However, the mutant phenotypes conferred by the methylated PAI genes are unstable, and mutant plants yield occasional revertant somatic sectors and progeny. Revertant lines display coordinately reduced methylation of both PAI2 and PAI3, implying that this hypomethylation acts in a concerted manner across the genome rather than at individual sites.', 'We report that CpG island methylation, an epigenetic modification of DNA known to correlate closely with silencing of gene transcription, appears in the oestrogen receptor (ER) gene in a subpopulation of cells which increases as a direct function of age in human colonic mucosa. This same methylation change characterizes virtually all cells in art 45 colorectal tumours examined, including the earliest stages of tumour formation. ER gene expression is diminished or absent in colorectal tumours, and introduction of an exogenous ER gene in cultured colon carcinoma cells resulted in marked growth suppression. Our data suggest that methylation associated inactivation of the ER gene in ageing cotorectal mucosa could be one of the earliest events that predispose to sporadic cotorectal tumorigenesis.', 'A very large oxidative damage rate to DNA occurs as part of normal metabolism. In each rat cell the steady-state level is estimated to be about 10(6) oxidative adducts and about 10(5) new adducts are formed daily. It is argued that this endogenous DNA damage is a major contributor to aging and the degenerative diseases of aging, such as cancer. The oxidative damage rate in mammalian species with a high metabolic rate, short life span, and high age-specific cancer rate is much higher than the rate in humans, a long-lived creature with a lower metabolic rate and a lower age-specific cancer rate. It is argued that deficiency of micronutrients, such as dietary antioxidants or folate, is a major contributor to human cancer and degenerative diseases. Understanding the role of mitogenesis in mutagenesis is critical for clarifying the mechanisms of carcinogenesis and interpreting high-dose animal cancer tests. High-dose animal cancer tests have been done mainly on synthetic industrial chemicals, yet almost all of the chemicals humans are exposed to are natural. About half of natural chemicals tested in high-dose animal cancer tests are rodent carcinogens, a finding that is consistent with the view that high-dose tests frequently increase mitogenesis rates. Animals have numerous defenses against toxins that make them very well buffered against low doses of almost all toxins, whether synthetic or natural.', 'The expression of the H19 gene is governed by parental imprinting in mammals. H19, an unusual gene encoding an RNA with no known function, is exclusively expressed from the maternal chromosome. In mouse, it lies 90 kb downstream from the Igf2 gene, which encodes a fetal-specific growth factor, insulin-like growth factor II, and is expressed primarily from the paternally inherited chromosome. In this report we have utilized interspecific hybrid mice to identify male-specific DNA methylation of a 7- to 9-kb domain surrounding the H19 gene and its promoter. This allele-specific methylation could function as a mark to suppress transcription of the H19 paternal allele. Consistent with this proposal, the H19 promoter displayed an open chromatin conformation only on the relatively unmethylated active maternal allele. In contrast, a cell type-specific enhancer that lies outside the methylation domain is hypersensitive to restriction enzyme digestion in nuclei on both maternal and paternal chromosomes. That the allele-specific methylation domain, coupled to the two H19 enhancers, contains all the information necessary for its imprinting was tested by examining two transgenic lines containing an internally deleted H19 transgene. Both displayed paternal-specific methylation of the transgene and maternal-specific expression. Although neither line has been tested in an inbred genetic background, and therefore the action of complex modifiers cannot be formally excluded, the result suggests that the sequences necessary for the imprinting of H19 have been identified.', \"The insulin-like growth factor-II (IGF2) and H19 genes are imprinted in mouse and human, with expression of the paternal IGF2 and maternal H19 alleles. IGF2 undergoes loss of imprinting (LOI) in most Wilms' tumours (WT). We now show that: (i) LOI of IGF2 is associated with a 80-fold down regulation of H19 expression; (ii) these changes are associated with alterations in parental-origin-specific, tissue-independent sites of DNA methylation in the H19 promoter; and (iii) loss of heterozygosity is also associated with loss of H19 expression. Thus, imprinting of a large domain of the maternal chromosome results in a reversal to a paternal epigenotype. These data also suggest an epigenetic mechanism for inactivation of H19 as a tumour suppressor gene.\", 'The recent discovery of widely distributed targeting determinants, which govern the polarized cell-surface distribution of plasma membrane proteins in epithelial cells, has significantly changed our view of how polarized cells generate functionally distinct membrane domains. Together with the surprising finding that the same determinants are recognized on both the biosynthetic and the endocytic pathways, it now appears likely that a common epigenetic code may exist that controls molecular sorting of membrane proteins in a wide variety of polarized, and perhaps even non-polarized, cell types.', \"Inactivation of the retinoblastoma gene appears to have a fundamental role in the genesis of retinoblastoma, osteosarcoma, and other malignant tumors. The gene is generally inactivated because of loss-of-function mutations, although epigenetic phenomena, such as hypermethylation of the promoter region, could possibly have the same effect. We investigated the methylation pattern at the 5' end of the retinoblastoma gene, including its promoter region and exon 1, in DNA purified from 56 primary retinoblastomas. We found five tumors with evidence for hypermethylation, all from unilateral, simplex patients. No methylation abnormalities were detected in DNA purified from the leukocytes from these patients. It is interesting that in one of these tumors the hypermethylation was confined to one allele. There were no mutations in a 1,306-bp sequence including the hypermethylated region that might account for the allele-specific hypermethylation. We believe that the hypermethylation of the retinoblastoma gene that we found in these tumors corresponds to the allelic inactivation of the gene, and we speculate that erroneous hypermethylation without alteration of nucleotide sequence occasionally plays a role in the genesis of this cancer. If this is true, then retinoblastomas with hypermethylation might be treatable with chemotherapeutic agents that interfere with methylation of DNA.\", 'Becker muscular dystrophy (BMD) often results from in-frame mutations of the dystrophin gene that allow production of an altered but partially functional protein. To address potential structure-function relationships for the various domains of dystrophin, we examined both the dystrophin gene and protein in 68 patients with abnormal dystrophin. Eighty-six percent of BMD patients with dystrophin of altered size have deletions or duplications, and the observed sizes of dystrophin fit well with predictions based on DNA data. Deletions within the amino-terminal domain I tended to result in low levels of dystrophin and a more severe phenotype. The phenotypes of patients with deletions or duplications in the central rod domain were more variable. This region can be divided into three portions based on differences in clinical presentations of patients. Deletions around exons 4553 were most common and generally caused typical BMD; however, phenotypic variability among patients with similar mutations suggests that epigenetic and/or environmental factors play an important role in determining the clinical progression. In contrast, deletions or duplications in the proximal portion of this domain tended to cause severe cramps and myalgia. Finally, loss of the middle of this region probably causes a very mild phenotype, as only one such patient was found and his only symptom was elevated serum creatine phosphokinase levels.', 'Massive cell migration, proliferation, phenotypic differentiation, and enhanced biosynthetic activities characterize the sites of wound healing and fibrosis. Regulation of cellular functions by extracellular matrix, which consists of a dynamic assemblage of a variety of interacting molecules capable of reorganization in response to endogenous and exogenous stimuli, represents a fundamental epigenetic mechanism regulating cellular behavior and phenotype. Interactions of the individual components of extracellular matrix with specific cell surface molecules, integrin receptors, and proteoglycans initiate a cascade of signal transduction leading to varied short-term or persistent cellular responses. Extracellular matrix also serves as an important reservoir of cytokines and growth factors, thus modulating the action of a host of potent biological response modifiers by their selective, local accumulation and release. Currently known mechanisms by which extracellular matrix modulates different facets of the process of tissue remodeling after injury, which culminate either in normal wound repair or fibrosis, are discussed.', \"To test the potential role of H19 as a tumour suppressor gene we have examined its expression and DNA methylation in Wilms' tumours (WTs). In most WTs (18/25), H19 RNA was reduced at least 20-fold from fetal kidney levels. Of the expression-negative tumours ten retained 11p15.5 heterozygosity: in nine of these, H19 DNA was biallelically hypermethylated and in two cases hypermethylation locally restricted to H19 sequences was also present in the non-neoplastic kidney parenchyma. IGF2 mRNA was expressed in most but not all WTs and expression patterns were consistent with IGF2/H19 enhancer competition without obligate inverse coupling. These observations implicate genetic and epigenetic inactivation of H19 in Wilms' tumorigenesis.\", 'Segmentation genes provide the signals for the activation and regulation of homeotic genes in Drosophila but cannot maintain the resulting pattern of expression because their activity ceases halfway through embryogenesis. Maintenance of the pattern is due to the Polycomb group of genes (Pc-G) and the trithorax group of genes (trx-G), responsible for the persistence of the active or repressed state of homeotic genes. We have identified a regulatory element in the Ubx gene that responds to Pc-G and trx-G genes. Transposons carrying this element create new binding sites for Pc-G products in the polytene chromosomes. This Pc-G maintenance element (PRE), establishes a repressive complex that keeps enhancers repressed in cells in which they were originally repressed and maintains this state through many cell divisions. The trx-G products stimulate the expression of enhancers in cells in which they were originally active. This mechanism is responsible for the correct regulation of imaginal disc enhancers, which lack themselves antero-posterior positional information. The PRE also causes severe variegation of the mini-white gene present in the transposon, a phenomenon very similar to heterochromatic position-effect variegation. The significance of this mechanism for homeotic gene regulation is discussed.', 'Chromatin structure at Schizosaccharomyces pombe centromeres is unusual. The insertion of the ura4 gene within these centromeres resulted in genetically identical cells mosaic for its expression. Placement of the ade6 gene within cen1 or cen3 resulted in red-white sectored colonies, demonstrating the instability of gene expression. The occurrence of pink colonies implied that intermediate levels of repression were established. Repression of both genes within centromeres was temperature sensitive. The chromatin structure of the ura4 gene at centromeres was altered, suggesting that the unusual chromatin encroaches into the gene and inhibits normal expression. These repressive effects at S. pombe centromeres resemble the classical phenomenon of position effect variegation imposed by Drosophila heterochromatin on nearby genes. However, since the epigenetic states can be set at intermediate levels of expression, a purely euchromatin-heterochromatin dichotomy does not apply. A model for the epigenetic regulation of genes placed within S. pombe centromeres is presented.', \"The mouse insulin-like growth factor II (Igf2) gene, which is located on distal chromosome 7 (Chr7), has been shown previously to undergo tissue-specific parental imprinting. This imprinting results in expression of Igf2 from the paternally inherited chromosome and repression of the maternally inherited allele in most tissues of the developing embryo. We are using embryos with the maternal duplication and paternal deficiency of distal Chr7 to characterize the mechanism that underlies repression of the maternal allele. We show that the chromatin of the 5' region of the repressed Igf2 allele is potentially active for transcription rather than heterochromatic. In particular, a CpG island that comprises one of the two strong promoters is unmethylated at both parental alleles, and DNase I hypersensitive sites in and around the strong promoters are consistently present on both parental chromosomes. In agreement with the chromatin state, primary transcripts from the maternal Igf2 allele have been detected at low but significant levels. These findings differ from observations in other instances of imprinting, namely, X-chromosome inactivation and transgene imprinting in mice. Although no parent-specific differences were detected in either DNA methylation or sensitivity to nucleases at these promoters, we have observed parental methylation differences in a region several kilobases upstream of the first exon. The differential activity of the parental Igf2 alleles could be achieved through epigenetic modifications situated outside the promoters or by subtle and yet unidentified modifications at the promoters.\", \"The Insulin-like growth factor 2 (Igf2) and H19 genes are reciprocally imprinted and closely linked. Igf2 encodes a fetal growth-factor and is predominantly expressed from the paternal allele, while H19 is expressed from the maternal allele and encodes a transcript which may downregulate cellular proliferation. One of the epigenetic modifications thought to be involved in parental imprinting is DNA methylation. Here we analyse methylation in two regions of the Igf2 gene, one approx. 3 kb upstream of the gene and one in the 3' part of the gene. Both regions are more methylated on the expressed paternal chromosome. Genomic sequencing of individual chromosomes in the first region shows this parent-specific methylation to be highly mosaic; interestingly, individual sperm chromosomes carry different methylation patterns into the egg. In the more 3' region, which is fully methylated in sperm, the level of methylation on the paternal allele is highly tissue-specific and is correlated with expression of the gene in fetal tissues. Hence, the paternal allele is highly methylated in fetal liver (high expression) but is undermethylated in fetal brain (virtually no expression). Adult choroid plexus, a brain tissue in which Igf2 is expressed from both alleles and H19 is not expressed, represents an apparent loss of imprinting. Here, both Igf2 and H19 adopt a paternal type methylation pattern on both parental chromosomes. Analysis of early-passage androgenetic and parthenogenetic embryonic stem (ES) cells shows that the methylation patterns of Igf2 and H19 on maternal and paternal chromosomes are very similar. Androgenetic and parthenogenetic teratomas derived from these ES cells show the appropriate paternal and maternal patterns, respectively, of allelic methylation in both genes. Our results suggest that allelic methylation patterns in Igf2 and H19 arise early in embryogenesis and change progressively during development. Some of these developmental changes are apparently under tissue-specific control.\", 'In several plant systems expression of structurally intact genes may be silenced epigenetically when a transgenic construct increases the copy number of DNA sequences. Here we report epigenetic silencing in Arabidopsis lines containing transgenic inserts of defined genetic structure, all at the same genomic locus. These comprise an allelic series that includes a single copy of the primary insert, which carries repeated drug resistance transgenes, and a set of its derivatives, which as a result of recombination within the insert carry different numbers and alleles of resistance genes. Although the drug resistance genes remained intact, both the primary and some recombinant lines nevertheless segregated many progeny that were partly or fully drug-sensitive because of silencing. As in other systems silencing was reversible, and correlated with decreased steady-state mRNA and increased DNA methylation. Each different number and combination of genes, on the same or different (i.e., homologous) chromosomes, conditioned its own idiosyncratic segregation pattern. Strikingly, lines with a single gene segregated only a few slightly drug-sensitive progeny whereas multi-gene lines segregated many highly sensitive progeny, indicating dependence of silencing at this locus on repeated sequences. This argues strongly against explanations based on antisense RNA, but is consistent with explanations based on ectopic DNA pairing. One possibility is that silencing reflects the interaction of paired homologous DNA with flanking heterologous DNA, which induces condensation of chromatin into a non-transcribable state.', 'Modifiers of position-effect-variegation in Drosophila encode proteins that are thought to modify chromatin, rendering it heritably changed in its expressibility. In an attempt to identify similar modifier genes in other species we have utilized a known sequence homology, termed chromo box, between a suppressor of position-effect-variegation, Heterochromatin protein 1 (HP1), and a repressor of homeotic genes, Polycomb (Pc). A PCR generated probe encompassing the HP1 chromo box was used to clone full-length murine cDNAs that contain conserved chromo box motifs. Sequence comparisons, in situ hybridization experiments, and RNA Northern blot analysis suggest that the murine and human sequences presented in this report are homologues of the Drosophila HP1 gene. Chromo box sequences can also be detected in other animal species, and in plants, predicting a strongly conserved structural role for the peptide encoded by this sequence. We propose that epigenetic (yet heritable) changes in gene expressibility, characteristic of chromosomal imprinting phenomena, can largely be explained by the action of such modifier genes. The evolutionary conservation of the chromo box motif now enables the isolation and study of putative modifier genes in those animal and plant species where chromosomal imprinting has been described.', ' ', ' ', 'The molecular mechanisms introducing epigenetic modifications that lead to differential silencing of some autosomal alleles depending on their parental legacy are still largely unknown, but recent results from studies of endogenously imprinted genes and particular transgenes make DNA methylation a strong candidate. At the same time, these results have raised new questions about the details of the imprinting process.', \"Previous work has shown that two homologous, unlinked transgene loci can interact in plant nuclei, leading to non-reciprocal trans-inactivation and methylation of genes at one locus. Here, we report the structure and methylation of different transgene loci that contain the same construct but are variably able to inactivate and methylate a partially homologous, unlinked target locus. Silencing loci comprised multiple, methylated copies of the transgene construct, whereas a non-silencing locus contained a single, unmethylated copy. The correspondence between strength of silencing activity and copy number/degree of methylation was further demonstrated by producing novel alleles of a strong silencing locus: reducing the transgene copy number and methylation within this silencing locus decreased its ability to inactivate the target locus. The strong silencing locus, which was located close to a telomere, trans-inactivated various structural variants of the original target construct, regardless of their location in the genome. This suggests that the silencing locus can scan the entire genome for homologous regions, a process possibly aided by its telomeric location. Our data support the idea that epistatic trans-inactivation of unlinked, homologous transgenes in plants results from a pre-existing epigenetic difference between transgene loci, which is subsequently equalized by ''epigene conversion'' involving DNA-DNA pairing.\", ' ', 'Major epigenetic modifications apparently occur during early development in the mouse. The factors that induce such modifications are complex and may involve the various components of a zygote. We have started to explore whether changes in the nucleocytoplasmic composition brought about by micromanipulation can induce phenotypic effects through epigenetic modifications. Nucleocytoplasmic hybrids were therefore prepared by transplanting a female pronucleus into a recipient egg from a different genotype. As a result, the maternal genome was of a different genetic background as compared with the egg cytoplasm. Specifically, experimental zygotes had cytoplasm from the inbred strain C57BL/6, a maternal genome from DBA/2, and a paternal genome from C57BL/6 (termed BDB hybrids). The mirror-image combination, termed DBD, was also made. The reconstituted zygotes were transferred to recipients and allowed to develop to term. Mice born from manipulated zygotes showed transcriptional repression and DNA methylation of major urinary protein genes in their liver, as well as growth deficiency resulting in reduced adult body weight. No altered phenotype was observed in controls in which the maternal pronucleus was simply transplanted back into another zygote of the same genetic background. These results clearly demonstrate phenotypic as well as molecular effects on DNA methylation and expression of at least one gene. Phenotype was therefore no longer predicted by genotype as a result of epigenetic modifications in experimental embryos. What precisely triggers the phenotypic and epigenetic changes is unknown, but presumably, nucleocytoplasmic interactions in hybrid zygotes may be partly responsible.', 'The DNA of eukaryotic cells is organized in a complex with proteins, either as interphase chromatin or mitotic chromosomes. Nucleosomes, the structural subunits of chromatin, have long been considered as static structures, incompatible with processes occurring in chromatin. During the past few years it has become evident that the histone part of the nucleosome has important regulatory functions. Some of these functions are mediated by the N-terminal core histone domains which contain sites for posttranslational modifications, among them lysine residues for reversible acetylation. Recent results indicate that acetylation and deacetylation of N-terminal lysines of nucleosomal core histones represent a means of molecular communication between chromatin and the cellular signal transduction network, resulting in heritable epigenetic information. Data on enzymes involved in acetylation and the pattern of acetylated lysine sites on chromosomes, as well as genetic data on yeast transcriptional repression, suggest that acetylation may lead to structural transitions as well as specific signalling within distinct chromatin domains.', ' ', \"Messenger RNA and methylation levels of four imprinted genes, H19, Igf2r, Igf-2 and Snrpn were examined by northern and Southern blotting in mouse parthenogenetic, androgenetic and normal or wild-type embryonic stem cell lines during their differentiation in vitro as embryoid bodies. Zn most instances, mRNA levels in parthenogenetic and androgenetic embryoid bodies differed from wild type as expected from previously determined patterns of monoallelic expression in midgestation embryos and at later stages of development. These findings implicate aberrant mRNA levels of these genes in the abnormal development of parthenogenetic and androgenetic embryos and chimeras. Whereas complete silence of one of the parental alleles has previously been observed in vivo, we detected some mRNA in the corresponding embryonic stem cell line. This 'leakage' phenomenon could be explained by partial erasure, bypass or override of imprints, or could represent the actual activity status at very early stages of development. The mRNA levels of H19, Igf2r and Igf-2 and the degree of methylation at specific associated sequences were correlated according to previous studies in embryos, and thereby are consistent with suggestions that the methylation might play a role in controlling transcription of these genes. Paternal-specific methylation of the H19 promoter region is absent in sperm, yet we observed its presence in undifferentiated androgenetic embryonic stem cells, or before the potential expression phase of this gene in embryoid bodies. As such methylation is likely to invoke a repressive effect, this finding raises the possibility that it is part of the imprinting mechanism of H19, taking the form of a secondary imprint or postfertilization epigenetic modification necessary for repression of the paternal allele.\", \"The SNRPN gene encodes a small nuclear ribonucleoprotein subunit, SmN, thought to be involved in splicing of pre-mRNA. A closely related protein, SmB/B', is constitutively expressed in all tissues except the brain, where SmN is predominantly expressed. The mouse homolog of the SNRPN gene has been shown to be functionally imprinted in mouse brain, being expressed only from the paternally derived chromosome. SNRPN has been mapped to human chromosome 15q11-q13 within the shortest region of deletion overlap for the Prader-Willi syndrome. We have now demonstrated functional imprinting of the human SNRPN gene using reverse transcription followed by the polymerase chain reaction (RT-PCR). No expression was observed in cultured skin fibroblasts of Prader-Willi patients, but was found in all Angelman patients and normal controls examined. We have also demonstrated a parent-specific DNA methylation imprint within intron 5 of the SNRPN gene, which suggests an epigenetic mechanism by which parent-specific expression of this gene might be inherited. Our findings indicate that SNRPN is expressed only from the paternally derived chromosome 15 in humans and therefore may fullfill one major criterion for being involved in the pathogenesis of the Prader-Willi syndrome.\", \"Development in mammals is influenced by genome imprinting which results in differences in the expression of some homologous maternal and paternal alleles. This process, initiated in the germline, can continue following fertilization with interactions between oocyte cytoplasmic factors and the parental genomes involving modifier genes. Further epigenetic modifications may follow to render the 'imprints' heritable through subsequent cell divisions during development. Imprinting of genes can be critical for their dosage affecting embryonic growth, cell proliferation and differentiation. The cumulative effects of all the imprinted genes are observed in androgenones (AG) and parthenogenones (PG), which reveal complementary phenotypes with respect to embryonic and extraembryonic tissues. The presence of PG cells in chimeras causes growth retardation, while that of AG cells enhanced growth. AG cells apparently have a higher cell proliferation rate and, unlike PG cells, are less prone to selective elimination. However, the PG germ cells are exempt from cell selection. In chimeras, PG cells are more likely to be found in ectodermal derivatives such as epidermis and brain in contrast to AG cells which make pronounced contributions to many mesodermal derivatives such as muscle, kidney, dermis and skeleton. The presence of androgenetic cells in chimeras also results in the disproportionate elongation of the anterior-posterior axis and sometimes in the abnormal development of skeletal elements along the axis. Genetic studies highlight the influence of subsets of imprinted genes, and identify those that are critical for development.\", 'Zea mays transformants produced by particle bombardment of embryogenic suspension culture cells of the genotype A188 x B73 and selected on kanamycin or bialaphos were characterized with respect to transgene integration, expression, and inheritance. Selection on bialaphos, mediated by the bar or par genes, was more efficient than selection on kanamycin, mediated by the nptII gene. Most transformants contained multicopy, single locus, transgene insertion events. A transgene expression cassette was more likely to be rearranged if expression of that gene was not selected for during callus growth. Not all plants regenerated from calli representing single transformation events expressed the transgenes, and a nonselectable gene (uidA) was expressed in fewer plants than was the selectable transgene. Mendelian inheritance of transgenes consistent with transgene insertion at a single locus was observed for approximately two thirds of the transformants assessed. Transgene expression was typically, but not always, predictable in progeny plants - transgene silencing, as well as poor transgene transmission to progeny, was observed in some plant lines in which the parent plants had expressed the transgene.', 'Although Angelman (AS) and Prader-Willi (PWS) syndromes are human genetic disorders with distinctly different developmental and neurobehavioural phenotypes, they both have abnormalities in inheritance of chromosome 15q11-q13. Whether AS or PWS arises depends on the parental origin of a deletion or uniparental disomy (the inheritance of 2 copies of a genetic locus from only one parent) for 15q11-q13. Normal development requires a genetic contribution for this genetic region from both a male and a female parent. The dependence on parental origin implies that genes in human 15q11-q13 have distinct functions depending upon epigenetic, parent-of-origin differences, known as genomic imprinting. Here, I review the role of uniparental disomy and genomic imprinting in the pathogenesis of AS and PWS, and briefly discuss phenotype-genotype correlations using candidate genes and mouse models, in particular for hypopigmentation.', 'Because development is epigenetic, diverse aspects of morphology are integrated during ontogeny. Using the method of thin-plate splines, and the decomposition of these splines by their principal warps, we examine the ontogeny of integrated features of skull growth of the cotton rat, Sigmodon fulviventer as observed in landmark locations in the ventral view. Postnatal growth of the skull in Sigmodon is not adequately described by the familiar contrast between relatively rapid facial elongation and slow, precocial growth of the cranial base. No developmental units corresponding to facial skull and cranial base emerge from analysis of geometric shape change. Rather, skull growth is both more integrated and more complex, exhibiting both skull-wide integration and locally individualized regions. Like skull shape, integration has an ontogeny; different regions of the skull can be partitioned into developmentally individualized parts in different ways at different ages. The effective count of individualized parts decreases substantially before weaning occurs, suggesting that the integration required by the functionally demanding activity of chewing gradually develops before the functional transition occurs. Our description of skull growth and integration does not depend upon arbitrary a priori choices about what to measure; rather, we base our decomposition of the whole into parts upon results of the data analysis. Our approach complicates the study of heterochrony, but, because it expresses the spatiotemporal organization of ontogeny, it enables the study of heterotopy.', 'A novel epigenetic mechanism that can affect minichromosome centromere function in vivo has been identified in S. pombe. This epigenetic system can result in the conversion of a nonfunctional centromere to a functional one without changes in the content, structural arrangement, or chemical modification state of the minichromosomal DNA. The conversion from a centromere-inactive to an active state, which is evident with minichromosomes carrying abbreviated centromeric DNA constructions, occurs with a relatively high frequency during mitotic cell divisions and readily provides an in vivo essay for proper centromere formation. The centromere-targeted epigenetic system supports a model for centromere function that involves specific de novo folding of centromeric components into a higher order chromatin structure.', 'During development, patterns of differential gene expression, defining determined states of cells, need to be maintained over many cell generations. In Drosophila, genetic and molecular analyses led to the discovery of a set of proteins which seem to exert such a memory function by using epigenetic mechanisms. Recent experiments demonstrate that, in particular, the heritable inactivation of regulatory genes relies on stable changes in the higher-order constitution of chromatin.', 'The ability of chemicals with tumor-promoting or tumor-inhibiting activity to modulate gap junctional intercellular communication is reviewed. The two most extensively used types of assays for screening tests are (1) metabolic cooperation assays involving exchange between cells of precursors of nucleic acid synthesis and (2) dye-transfer assays that measure exchange of fluorescent dye from loaded cells to adjacent cells. About 300 substances of different biological activities have been studied using various assays. For tumor promoters/epigenetic carcinogens, metabolic cooperation assays have a sensitivity of 62% and dye-transfer assays 60%. Thirty percent of DNA-reactive carcinogens also possess the ability to uncouple cells. The complete estimation of the predictive power of these assays could not be made because the majority of the substances studied for intercellular communication effects in vitro have not yet been studied for promoting activity in vivo. Both metabolic cooperation assays and dye transfer assays respond well to the following classes of substances: phorbol esters, organochlorine pesticides, polybrominated biphenyls, promoters for urinary bladder, some biological toxins, peroxisome proliferators, and some complex mixtures. Results of in vitro assays for such tumor promoters/nongenotoxic carcinogens, such as some bile acids, some peroxides, alkanes, some hormones, mineral dusts, ascorbic acid, okadaic acid, and benz(e)pyrene, do not correlate with the data of in vivo two-stage or complete carcinogenesis. Enhancement of intercellular communication was found for 18 chemicals. Among these, cAMP, retinoids, and carotenoids have demonstrated inhibition of carcinogenesis. We examine a number of factors that are important for routine screening, including the requirement for biotransformation for some agents to exert effects on gap junctions. We also discuss the mechanisms of tumor promoter and tumor inhibitor effects on gap junctional permeability, including influences of protein kinase activation, changes in proton and Ca2+ intracellular concentrations, and effects of oxy radical production.', \"Ascobolus immersus artificial gene repeats were shown previously to be subject premeiotically to both cytosine methylation and inactivation. We studied sexual progenies of strains harbouring two wild type copies of the endogenous met2 gene lying either in tandem array or at ectopic unlinked positions, by (i) investigating the methylation status, (ii) searching for mutations and (iii) analysing the inheritance of inactivation both in mitotic and sexual offspring. 100% of the 'tandem' progeny and 64% of the 'ectopic' progeny had methylated repeats and displayed gene inactivation. Similar methylation patterns involving all or most of the cytosine residues within the repeats were observed in both arrangements. The inactivated met2 copies were totally devoid of mutation, as deduced from: (i) extensive restriction site analysis and DNA sequencing; (ii) the finding that all the Met- derivatives tested reverted to prototrophy in selective conditions; and (iii) the finding that an inactivated copy of met2 stripped of its methylation through amplification in Escherichia coli regained activity when reintroduced in A. immersus. In the absence of selection, gene silencing and methylation were faithfully maintained through mitotic divisions and through five successive sexual cycles. Altogether, these data show the epimutational nature of this methylation induced premeiotically (MIP) process.\", \"Transposable elements from the Robertson's Mutator family are highly active insertional mutagens in maize. However, mutations caused by the insertion of responder (non-autonomous) elements frequently depend on the presence of active regulator (autonomous) elements for their phenotypic effects. The hcf106::Mu1 mutation has been previously shown to depend on Mu activity in this way. The dominant Lesion-mimic 28 mutation also requires Mu activity for its phenotypic effects. We have used double mutants to show that the loss of Mu activity results in the coordinate suppression of both mutant phenotypes. This loss can occur somatically resulting in large clones of cells that have a wild-type phenotype. Autonomous and non-autonomous Mutator elements within these clones are insensitive to digestion with methylation-sensitive enzymes, suggesting extensive methylation of CG and non-CC cytosine residues. Our data are consistent with the sectors being caused by the cycling of MuDR regulatory elements between active and inactive phases. The pattern of sectors suggests that they are clonal and that they are derived from the apical cells of the vegetative shoot meristem. We propose that these cells are more likely to undergo epigenetic loss of Mu activity because of their longer cell division cycle during shoot growth. Coordinate suppression of unlinked mutations can be used to perform mosaic analysis in maize.\", \"The neural crest is a fascinating structure of the vertebrate embryo; its ontogeny includes a transient period during which its component cells undergo an epithelio-mesenchymal transition and become migratory. This phase was shown recently to be controlled by the 'Slug' gene which belongs to the 'Snail' family of Drosophila transcription factors. After homing to specific sites in the embryo, the crest-derived cells produce a large variety of phenotypes. Recent advances have shown that during migration most crest cells exhibit various degrees of pluripotentiality, some being already committed to a single and definite fate. Moreover, several lines of evidence point to the existence of totipotent stem cells in the neural crest, the progeny of which become progressively diversified through a combination of intrinsic and extrinsic influences. The latter have been documented by the disruption of several neurotrophin genes, which results in severe deficiencies of selected subsets of neural crest derivatives. The neural crest has also been shown to play an important role in the development of the vertebrate head and hypobranchial region. The genetic control of this process depends on the. activity of developmental genes, among which the vertebrate Hox genes are essential, particularly at the rhombencephalic level.\", 'The ability of simian virus 40-encoded large T antigen to disrupt the growth control of a variety of cell types is related to its ability to interfere with certain cellular proteins, such as p53 and the retinoblastoma susceptibility gene product (pRB). We have used wild-type and mutant forms of T antigen in transgenic mice to dissect the roles of pRB, p53, and other cellular proteins in tumorigenesis of different cell types. In this study, using a cell-specific promoter to target expression specifically to brain epithelium (the choroid plexus) and to B and T lymphoid cells, we characterize the tumorigenic capacity of a T-antigen fragment that comprises only the amino-terminal 121 residues. This fragment (dl1137) retains the ability to interact with pRB and p107 but lacks the p53-binding domain. While loss of the p53-binding region results in loss of the capacity to induce lymphoid abnormalities, dl1137 retains the ability to induce choroid plexus tumors that are histologically indistinguishable from those induced by wild-type T antigen. Tumors induced by dl1137 develop much more slowly, however, reaching an end point at around 8 months of age rather than at 1 to 2 months. Analysis of tumor progression indicates that tumor induction by dl1137 does not require secondary genetic or epigenetic events. Rather, the tumor growth rate is significantly slowed, indicating that the T-antigen C-terminal region contributes to tumor progression in this cell type. In contrast, the pRB-binding region appears essential for tumorigenesis as mutation of residue 107, known to disrupt pRB and p107 binding to wild-type T antigen, abolishes the ability of the dl1137 protein to induce growth abnormalities in the brain.', 'In this study, we used the ADE2 gene in a colony color assay to monitor transcription from the normally silent HMR mating-type locus in Saccharomyces cerevisiae. This sensitive assay reveals that some previously identified cis- and trans-acting mutations destabilize silencing, causing genetically identical cells to switch between repressed and derepressed transcriptional states. Deletion of the autonomously replicating sequence (ARS) consensus element at the HMR-E silencer or mutation of the silencer binding protein RAP1 (rap1s) results in the presence of large sectors within individual colonies of both repressed (Ade-, pink) and derepressed (Ade+, white) cells. These results suggest that both the ARS consensus element and the RA-P1 protein play a role in the establishment of repression at HMR. In diploid cells, the two copies of HMR appear to behave identically, suggesting that the switching event, though apparently stochastic, reflects some property of the cell rather than a specific event at each HMR locus. In the ADE2 assay system, silencing depends completely upon the function of the SIR genes, known trans-acting regulators of the silent loci, and is sensitive to the gene dosage of two SIR genes, SIR1 and SIR4. Using the ADE2 colony color assay in a genetic screen for suppressors of rap1s, silencer ARS element deletion double mutants, we have identified a large number of genes that may affect the establishment of repression at the HMR silent mating-type locus.', 'We analysed de novo DNA methylation occurring in plants obtained from the transgenic petunia line R101-17. This line contains one copy of the maize Al gene that leads to the production of brick-red pelargonidin pigment in the flowers. Due to its integration into an unmethylated genomic region the A1 transgene is hypomethylated and transcriptionally active. Several epigenetic variants of line 17 were selected that exhibit characteristic and somatically stable pigmentation patterns, displaying fully coloured, marbled or colourless flowers. Analysis of the DNA methylation patterns revealed that the decrease in pigmentation among the epigenetic variants was correlated with an increase in methylation, specifically of the transgene DNA. No change in methylation of the hypomethylated integration region could be detected. A similar increase in methylation, specifically in the transgene region, was also observed among progeny of R101-17del, a deletion derivative of R101-17 that no longer produces pelargonidin pigments due to a deletion in the Al coding region. Again de novo methylation is specifically directed to the transgene, while the hypomethylated character of neighbouring regions is not affected. Possible mechanisms for transgene-specific methylation and its consequences for long-term use of transgenic material are discussed.', 'Plant cells growing in an artificial culture environment make numerous genetic mistakes. These alterations are manifested as increased frequencies of single-gene mutations, chromosome breakages, transposable element activations, quantitative trait variations, and modifications of normal DNA methylation patterns. Evidence is presented that indicates a high frequency of DNA hypomethylation as the result of the tissue culture process. Fifteen percent of the methylation changes appear to have been homozygous in the original regenerated plants. A hypothesis is advanced that relates DNA methylation to the variety of genetic alterations found among maize tissue culture regenerants and their progenies. The epigenetic nature of DNA methylation raises questions concerning the stability of tissue culture-induced changes in self-pollinations and crosses.', \"In this essay review I describe the relationship between plant organization, variation, and evolution. The hypothetical optimal design of an organism living in a given environment must be specified to test whether evolution is adaptive. In a variable environment this design should include a system for plasticity. Plasticity in plants is made possible by modular growth. In modular growth new parts can constantly be added to a body, resulting in a population-like structure of the organism. The developmental basis for modular growth is an open-ended ontogeny in which the formation of organs is not controlled by patterns of cell divisions and migrations and where there is no separation into germ-line and soma. Epigenetic pattern formation, late differentiation, and loose integration of the individual arising from a zygote (genet) are characteristic for plants. Because the phenotype is the result of an interaction between genotype and environment, the contributions of the two cannot be separated. Nevertheless, with appropriate experimental procedures it is possible to separate the variation among phenotypes into variation among genotypes, variation among environments, and covariation of genotypes and environments. Several genotypes must be grown in several environments. If the environments can be put on a scale, as in gradient experiments, reaction norms of genotypes may be quantified. Local specialization of genotypes in a heterogeneous environment may be recognized by transplant experiments. New results from both types of experiments indicate that the 'general-purpose genotype' is more common than originally thought and suggest that phenotypic plasticity in plants has relatively low costs. However, the critical experiments to measure these costs have not yet been published. The genetic analysis of phenotypic variation and the use of phenotypic characters for reconstructing phylogenies is hampered by the particular plant features mentioned above. Environment-induced phenotypic effects can be persistent and can be transmitted to clonal or even sexual progeny. Gene flow among taxa via sexual and possibly other mechanisms together with the loose integration of the individual make it difficult to recognize phylogenies for systematic groups in plants. Some of these problems are illustrated with species from the genera Solidago and Carex.\", 'Mating-type genes resident in the silent cassette HML at the left arm of chromosome III are repressed by the action of four SIR gene products, mediated independently through two cis-acting sites, termed the E and I silencers. We have found that in the absence of the I silencer, deletion of any one of three distinct elements within E yields partial derepression of the mating-type genes resident at HML, whereas deletion of any two yields full derepression. These elements correspond to a binding site for the abundant DNA-binding protein RAP1, an autonomous replicating sequence (ARS), and an as yet undistinguished region. From detailed deletion analysis of the E site we conclude that the ARS element contributes to silencer function in a capacity distinct from its role as an initiator of DNA replication. In addition, we find that strains deleted for any one of these elements comprise two genetically identical but phenotypically distinct types of cells: Those with HML apparently fully derepressed, and those with HML apparently completely repressed. These results reinforce the notion that epigenetic inheritance is an intrinsic characteristic of silencer action.', 'Two instances of genetic transmission of spontaneous epimutation of the maize P-rr gene were identified. Transmission gave rise to two similar, moderately stable alleles, designated P-pr-1 and P-pr-2, that exhibited Mendelian behavior. Both isolates of P-pr conditioned a variable and variegated phenotype, unlike the uniform pigmentation conditioned by P-rr. Extensive genomic analysis failed to reveal insertions, deletions or restriction site polymorphisms between the new allele and its progenitor. However, methylation of the P gene was increased in P-pr relative to P-rr, and was greatly reduced (though not lost) in a revertant to uniform pigmentation. Variability in pigmentation conditioned by P-pr correlated with variability in transcript levels of the P gene, and both correlated inversely with variability in its methylation. Part of the variability in methylation could be accounted for by a developmental decrease in methylation in all tissues of plants carrying P-pr. We hypothesize that the variegated phenotype results from a general epigenetic pathway which causes a progressive decrease in methylation and increase in expression potential of the P gene as a function of cell divisions in each meristem of the plant. This renders all tissues chimeric for a functional gene; chimerism is visualized as variegation only in pericarp due to the tissue specificity of P gene expression. Therefore, this allele that originates from epimutation may exemplify an epigenetic mechanism for variegation in maize.', 'Previous work has shown that two unlinked, partially homologous transgene loci can interact in plant nuclei, leading to reversible methylation and inactivation of one transgene locus in the presence of the second. To study whether the chromosomal location of a transgene influences its susceptibility to trans-inactivation, we retransformed four transgenic lines, which contained the same construct (H) integrated in different chromosomal locations, with a second, partially homologous construct (K). At least 50 double transformants (DTs) were regenerated from each single transformant (ST) and screened for inactivation of markers [chloramphenicol acetyltransferase (CAT); hygromycin resistance (HYG(R))] at the resident H locus. For two STs, H locus markers were inactivated in less than 1% of the DTs, suggesting that, at these integration sites, H was relatively resistant to trans-inactivation. In contrast, the other two STs appeared to be more sensitive to trans-inactivation: 4-10% of the DTs were CAT(-) and/or Hyg(s). Inactivation of H locus markers could be attributed to two distinct phenomena: 1. Regeneration from cells containing different epigenetic states of H, in which either both, one or none of the H alleles was active. This instability in the expression of the H locus, which was independent of K, was more pronounced in the homozygous state, and was associated with cellular mosaicism of expression and methylation. 2. The presence of an unlinked K locus could weaken the Hyg(R) phenotype by transcriptional inactivation and increased methylation of the hph gene at the H locus. These results indicated that a susceptible transgene locus is inherently unstable and partially methylated, and that these characteristics are exacerbated when the locus is homozygous for the transgene and/or when an unlinked homologous transgene is present.', 'An illustrative list is presented of human and animal studies which each point to the existence of a third source, in addition to genetic and environmental factors, underlying phenotypic differences in development. It is argued that this third source may consist of nonlinear epigenetic processes that can create variability at all phenotypical-somatic and behavioral-levels. In a quantitative genetic analysis with human subjects, these processes are confounded with within-family environmental influences. A preliminary model to quantify these influences is introduced.', 'Analysis of the developmental capacities of androgenetic and gynogenetic mouse embryos (bearing two paternal or two maternal pronuclei, respectively) revealed a defect in blastocyst formation of androgenetic, but not gynogenetic, embryos that was a function of the maternal genotype. Androgenetic embryos constructed using fertilized eggs from C57BL/6 or (B6D2)F1 mice developed to the blastocyst stage at frequencies similar to those previously reported, whereas androgenetic embryos constructed with fertilized eggs from DBA/2 mice developed poorly, the majority failing to progress beyond the 16-cell stage and unable to form a blastocoel-like cavity, regardless of whether the male pronuclei were of C57BL6 or DBA/2 origin. This impaired development was observed even in androgenetic embryos constructed by transplanting two male pronuclei from fertilized DBA/2 eggs to enucleated C57BL/6 eggs, indicating that the defect cannot be explained as the lack of some essential component in the DBA/2 cytoplasm that might otherwise compensate for androgeny. Rather, the DBA/2 egg cytoplasm apparently modifies the incoming male pronuclei differently than does C57BL/6 egg cytoplasm. Several specific alterations in the protein synthesis pattern of DBA/2 androgenones were observed that reflect a defect in the regulatory mechanisms that normally modulate the synthesis of these proteins between the 8-cell and blastocyst stages. These results are consistent with a model in which cytoplasmic factors present in the egg direct a strain-dependent modification of paternal genome function in response to epigenetic modifications (genomic imprinting) established during gametogenesis and indicate that preimplantation development can be affected by these modifications at both the morphological and biochemical levels.', \"Functional consequences of either suppressing or intensifying spontaneous neuronal firing have been studied in developing rat cerebral cortex cultures using, respectively, tetrodotoxin (TTX) and picrotoxin (PTX) added chronically to the growth medium. Simple measures derived from the interspike interval histogram were able to powerfully discriminate between age and treatment groups. After return to control medium, most TTX-treated neurons spontaneously displayed stereotyped clustering of action potentials ('phasic' firing) which closely resembled the characteristic firing patterns seen acutely in the presence of PTX. The 'TTX-syndrome' thus suggests that GABAergic synaptic inhibition is ineffective in cortical networks grown under conditions which prevent the expression of bioelectric activity. In contrast, after return to control medium, neurons which had been partially disinhibited throughout development (by continuous exposure to PTX) had even less phasic firing than was measured in age-matched controls. Based upon these and previous findings, a two (main) factor model is put forth which can economically account for the major effects. The working hypothesis embodied in this model is that phasic neuronal discharges not only accelerate the maturation of excitatory connections within the neocortex but, even more important, are crucial for the development of adequate inhibitory synaptic transmission.\", 'Differences in the activity of maternally and paternally derived genomes in maize endosperm have been observed at three levels of genetic manipulation. When the balance of entire chromosome sets departs from the standard ratio of two of maternal origin to one of paternal origin, development is impaired, often leading to seed failure. At the level of individual chromosomes, absence of a paternal representative for 8 of the 19 chromosome arms tested causes a marked reduction in kernel size. Replacement of the missing arms by ones of maternal origin does not complement this defect. At the gene level, some alleles of R confer solid coloration on the aleurone layer when transmitted maternally but patchy coloration (mottled) when transmitted via pollen. In contrast with the endosperm, no effect of parentage on R phenotype has been detected in embryonic and seedling tissues. Furthermore, gynogenetic and androgenetic haploid plants are viable in maize and are similar in appearance. The detection of parental effects in the endosperm, but not the embryo, points to the few cell divisions of the gametophytes as a critical stage in imprinting. Chromosomally based epigenetic variation originating at this stage would be reflected as imprinting effects. A separate fertilization establishes a line of genetic descent in the embryo that appears to be relatively free of imprinted genes.', ' ', 'Herein we describe a general multivariate quantitative genetic model that incorporates two potentially important developmental phenomena, maternal effects and epigenetic effects. Maternal and epigenetic effects are defined as partial regression coefficients and phenotypic variances are derived in terms of age-specific genetic and environmental variances. As a starting point, the traditional quantitative genetic model of additive gene effects and random environmental effects is cast in a developmental time framework. From this framework, we first extend a maternal effects model to include multiple developmental ages for the occurrence of maternal effects. An example of maternal effects occurring at multiple developmental ages is prenatal and postnatal maternal effects in mammals. Subsequently, a model of intrinsic and epigenetic effects in the absence of maternal effects is described. It is shown that genetic correlations can arise through epigenetic effects, and in the absence of other developmental effects, epigenetic effects are in general confounded with age-specific intrinsic genetic effects. Finally, the two effects are incorporated into the basic quantitative genetic model. For this more biologically realistic model combining maternal and epigenetic effects, it is shown that the phenotypic regressions of offspring on mother and offspring on father can be used in some cases to estimate simultaneously maternal effects and epigenetic effects.', 'Butylated hydroxyanisole (BHA) is a synthetic food antioxidant used to prevent oils, fats and shortenings from oxidative deterioration and rancidity. This review depicts the current knowledge on BHA. The physical and chemical characteristics of BHA are summarized and its function as a food antioxidant is made clear. The toxicological characteristics of BHA and its metabolic fate in man and animal are briefly reviewed. Special emphasis is laid on the carcinogenicity of BHA in the forestomach of rodents and to related events in the forestomach and other tissues in experimental animals. At present there is sufficient evidence for carcinogenicity of BHA, but there is hardly any indication that BHA is genotoxic. Therefore risk assessment for this epigenetic carcinogen is based on non-stochastic principles. However, the mechanism underlying the tumorigenicity of BHA is not known. In the last part of this review an attempt is made to unravel the unknown mechanism of carcinogenicity. It is hypothesized that BHA gives rise to tumor formation in rodent forestomach by inducing heritable changes in DNA. Evidence is being provided that reactive oxygen species, in particular hydroxylradicals, may play a crucial role. The key question with respect to risk assessment for BHA is whether or not the underlying mechanism is thresholded, which is important for the choice of the appropriate model to assess the risk, if any, for man and to manage any potential risk.']\n"
     ]
    }
   ],
   "source": [
    "print(df)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 10,
   "metadata": {},
   "outputs": [],
   "source": [
    "###CLEAN DATA###"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 11,
   "metadata": {},
   "outputs": [],
   "source": [
    "# Convert to list\n",
    "data = [simple_preprocess(doc) for doc in df]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 12,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "[['the', 'american', 'college', 'of', 'medical', 'genetics', 'and', 'genomics', 'acmg', 'previously', 'developed', 'guidance', 'for', 'the', 'interpretation', 'of', 'sequence', 'variants', 'in', 'the', 'past', 'decade', 'sequencing', 'technology', 'has', 'evolved', 'rapidly', 'with', 'the', 'advent', 'of', 'high', 'throughput', 'next', 'generation', 'sequencing', 'by', 'adopting', 'and', 'leveraging', 'next', 'generation', 'sequencing', 'clinical', 'laboratories', 'are', 'now', 'performing', 'an', 'ever', 'increasing', 'catalogue', 'of', 'genetic', 'testing', 'spanning', 'genotyping', 'single', 'genes', 'gene', 'panels', 'exomes', 'genomes', 'transcriptomes', 'and', 'epigenetic', 'assays', 'for', 'genetic', 'disorders', 'by', 'virtue', 'of', 'increased', 'complexity', 'this', 'shift', 'in', 'genetic', 'testing', 'has', 'been', 'accompanied', 'by', 'new', 'challenges', 'in', 'sequence', 'interpretation', 'in', 'this', 'context', 'the', 'acmg', 'convened', 'workgroup', 'in', 'comprising', 'representatives', 'from', 'the', 'acmg', 'the', 'association', 'for', 'molecular', 'pathology', 'amp', 'and', 'the', 'college', 'of', 'american', 'pathologists', 'to', 'revisit', 'and', 'revise', 'the', 'standards', 'and', 'guidelines', 'for', 'the', 'interpretation', 'of', 'sequence', 'variants', 'the', 'group', 'consisted', 'of', 'clinical', 'laboratory', 'directors', 'and', 'clinicians', 'this', 'report', 'represents', 'expert', 'opinion', 'of', 'the', 'workgroup', 'with', 'input', 'from', 'acmg', 'amp', 'and', 'college', 'of', 'american', 'pathologists', 'stakeholders', 'these', 'recommendations', 'primarily', 'apply', 'to', 'the', 'breadth', 'of', 'genetic', 'tests', 'used', 'in', 'clinical', 'laboratories', 'including', 'genotyping', 'single', 'genes', 'panels', 'exomes', 'and', 'genomes', 'this', 'report', 'recommends', 'the', 'use', 'of', 'specific', 'standard', 'terminology', 'pathogenic', 'likely', 'pathogenic', 'uncertain', 'significance', 'likely', 'benign', 'and', 'benign', 'to', 'describe', 'variants', 'identified', 'in', 'genes', 'that', 'cause', 'mendelian', 'disorders', 'moreover', 'this', 'recommendation', 'describes', 'process', 'for', 'classifying', 'variants', 'into', 'these', 'five', 'categories', 'based', 'on', 'criteria', 'using', 'typical', 'types', 'of', 'variant', 'evidence', 'population', 'data', 'computational', 'data', 'functional', 'data', 'segregation', 'data', 'because', 'of', 'the', 'increased', 'complexity', 'of', 'analysis', 'and', 'interpretation', 'of', 'clinical', 'genetic', 'testing', 'described', 'in', 'this', 'report', 'the', 'acmg', 'strongly', 'recommends', 'that', 'clinical', 'molecular', 'genetic', 'testing', 'should', 'be', 'performed', 'in', 'clinical', 'laboratory', 'improvement', 'amendments', 'approved', 'laboratory', 'with', 'results', 'interpreted', 'by', 'board', 'certified', 'clinical', 'molecular', 'geneticist', 'or', 'molecular', 'genetic', 'pathologist', 'or', 'the', 'equivalent'], ['there', 'is', 'substantial', 'heterogeneity', 'among', 'primary', 'prostate', 'cancers', 'evident', 'in', 'the', 'spectrum', 'of', 'molecular', 'abnormalities', 'and', 'its', 'variable', 'clinical', 'course', 'as', 'part', 'of', 'the', 'cancer', 'genome', 'atlas', 'tcga', 'we', 'present', 'comprehensive', 'molecular', 'analysis', 'of', 'primary', 'prostate', 'carcinomas', 'our', 'results', 'revealed', 'molecular', 'taxonomy', 'in', 'which', 'of', 'these', 'tumors', 'fell', 'into', 'one', 'of', 'seven', 'subtypes', 'defined', 'by', 'specific', 'gene', 'fusions', 'erg', 'etv', 'and', 'fli', 'or', 'mutations', 'spop', 'foxa', 'and', 'idh', 'epigenetic', 'profiles', 'showed', 'substantial', 'heterogeneity', 'including', 'an', 'idh', 'mutant', 'subset', 'with', 'methylator', 'phenotype', 'androgen', 'receptor', 'ar', 'activity', 'varied', 'widely', 'and', 'in', 'subtype', 'specific', 'manner', 'with', 'spop', 'and', 'foxa', 'mutant', 'tumors', 'having', 'the', 'highest', 'levels', 'of', 'ar', 'induced', 'transcripts', 'of', 'the', 'prostate', 'cancers', 'had', 'presumed', 'actionable', 'lesion', 'in', 'the', 'pi', 'or', 'mapk', 'signaling', 'pathways', 'and', 'dna', 'repair', 'genes', 'were', 'inactivated', 'in', 'our', 'analysis', 'reveals', 'molecular', 'heterogeneity', 'among', 'primary', 'prostate', 'cancers', 'as', 'well', 'as', 'potentially', 'actionable', 'molecular', 'defects'], ['pd', 'and', 'pd', 'pd', 'pathway', 'blockade', 'is', 'highly', 'promising', 'therapy', 'and', 'has', 'elicited', 'durable', 'antitumor', 'responses', 'and', 'long', 'term', 'remissions', 'in', 'subset', 'of', 'patients', 'with', 'broad', 'spectrum', 'of', 'cancers', 'how', 'to', 'improve', 'widen', 'and', 'predict', 'the', 'clinical', 'response', 'to', 'anti', 'pd', 'therapy', 'is', 'central', 'theme', 'in', 'the', 'field', 'of', 'cancer', 'immunology', 'and', 'immunotherapy', 'oncologic', 'immunologic', 'genetic', 'and', 'biological', 'studies', 'focused', 'on', 'the', 'human', 'cancer', 'have', 'yielded', 'substantial', 'insight', 'into', 'this', 'issue', 'here', 'we', 'focus', 'on', 'tumor', 'and', 'evaluate', 'several', 'potential', 'therapeutic', 'response', 'markers', 'including', 'the', 'pd', 'and', 'pd', 'expression', 'pattern', 'genetic', 'mutations', 'within', 'cancer', 'cells', 'and', 'neoantigens', 'cancer', 'epigenetics', 'and', 'effector', 'cell', 'landscape', 'and', 'microbiota', 'we', 'further', 'clarify', 'the', 'mechanisms', 'of', 'action', 'of', 'these', 'markers', 'and', 'their', 'roles', 'in', 'shaping', 'being', 'shaped', 'and', 'or', 'predicting', 'therapeutic', 'responses', 'we', 'also', 'discuss', 'variety', 'of', 'combinations', 'with', 'pd', 'pathway', 'blockade', 'and', 'their', 'scientific', 'rationales', 'for', 'cancer', 'treatment'], ['ensembl', 'ext', 'link', 'type', 'uri', 'xlink', 'href', 'http', 'www', 'ensembl', 'org', 'xlink', 'type', 'simple', 'http', 'www', 'ensembl', 'org', 'is', 'genomic', 'interpretation', 'system', 'providing', 'the', 'most', 'up', 'to', 'date', 'annotations', 'querying', 'tools', 'and', 'access', 'methods', 'for', 'chordates', 'and', 'key', 'model', 'organisms', 'this', 'year', 'we', 'released', 'updated', 'annotation', 'gene', 'models', 'comparative', 'genomics', 'regulatory', 'regions', 'and', 'variation', 'on', 'the', 'new', 'human', 'assembly', 'grch', 'although', 'we', 'continue', 'to', 'support', 'researchers', 'using', 'the', 'grch', 'assembly', 'through', 'dedicated', 'site', 'ext', 'link', 'type', 'uri', 'xlink', 'href', 'http', 'grch', 'ensembl', 'org', 'xlink', 'type', 'simple', 'http', 'grch', 'ensembl', 'org', 'our', 'regulatory', 'build', 'has', 'been', 'revamped', 'to', 'identify', 'regulatory', 'regions', 'of', 'interest', 'and', 'to', 'efficiently', 'highlight', 'their', 'activity', 'across', 'disparate', 'epigenetic', 'data', 'sets', 'number', 'of', 'new', 'interfaces', 'allow', 'users', 'to', 'perform', 'large', 'scale', 'comparisons', 'of', 'their', 'data', 'against', 'our', 'annotations', 'the', 'rest', 'server', 'ext', 'link', 'type', 'uri', 'xlink', 'href', 'http', 'rest', 'ensembl', 'org', 'xlink', 'type', 'simple', 'http', 'rest', 'ensembl', 'org', 'which', 'allows', 'programs', 'written', 'in', 'any', 'language', 'to', 'query', 'our', 'databases', 'has', 'moved', 'to', 'full', 'service', 'alongside', 'our', 'upgraded', 'website', 'tools', 'our', 'online', 'variant', 'effect', 'predictor', 'tool', 'has', 'been', 'updated', 'to', 'process', 'more', 'variants', 'and', 'calculate', 'summary', 'statistics', 'lastly', 'the', 'wiggletools', 'package', 'enables', 'users', 'to', 'summarize', 'large', 'collections', 'of', 'data', 'sets', 'and', 'view', 'them', 'as', 'single', 'tracks', 'in', 'ensembl', 'the', 'ensembl', 'code', 'base', 'itself', 'is', 'more', 'accessible', 'it', 'is', 'now', 'hosted', 'on', 'our', 'github', 'organization', 'page', 'ext', 'link', 'type', 'uri', 'xlink', 'href', 'https', 'github', 'com', 'ensembl', 'xlink', 'type', 'simple', 'https', 'github', 'com', 'ensembl', 'under', 'an', 'apache', 'open', 'source', 'license'], ['over', 'the', 'past', 'years', 'breakthrough', 'discoveries', 'of', 'chromatin', 'modifying', 'enzymes', 'and', 'associated', 'mechanisms', 'that', 'alter', 'chromatin', 'in', 'response', 'to', 'physiological', 'or', 'pathological', 'signals', 'have', 'transformed', 'our', 'knowledge', 'of', 'epigenetics', 'from', 'collection', 'of', 'curious', 'biological', 'phenomena', 'to', 'functionally', 'dissected', 'research', 'field', 'here', 'we', 'provide', 'personal', 'perspective', 'on', 'the', 'development', 'of', 'epigenetics', 'from', 'its', 'historical', 'origins', 'to', 'what', 'we', 'define', 'as', 'the', 'modern', 'era', 'of', 'epigenetic', 'research', 'we', 'primarily', 'highlight', 'key', 'molecular', 'mechanisms', 'of', 'and', 'conceptual', 'advances', 'in', 'epigenetic', 'control', 'that', 'have', 'changed', 'our', 'understanding', 'of', 'normal', 'and', 'perturbed', 'development'], ['single', 'molecule', 'real', 'time', 'sequencing', 'developed', 'by', 'pacific', 'biosciences', 'offers', 'longer', 'read', 'lengths', 'than', 'the', 'second', 'generation', 'sequencing', 'sgs', 'technologies', 'making', 'it', 'well', 'suited', 'for', 'unsolved', 'problems', 'in', 'genome', 'transcriptome', 'and', 'epigenetics', 'research', 'the', 'highly', 'contiguous', 'de', 'novo', 'assemblies', 'using', 'pacbio', 'sequencing', 'can', 'close', 'gaps', 'in', 'current', 'reference', 'assemblies', 'and', 'characterize', 'structural', 'variation', 'sv', 'in', 'personal', 'genomes', 'with', 'longer', 'reads', 'we', 'can', 'sequence', 'through', 'extended', 'repetitive', 'regions', 'and', 'detect', 'mutations', 'many', 'of', 'which', 'are', 'associated', 'with', 'diseases', 'moreover', 'pacbio', 'transcriptome', 'sequencing', 'is', 'advantageous', 'for', 'the', 'identification', 'of', 'gene', 'isoforms', 'and', 'facilitates', 'reliable', 'discoveries', 'of', 'novel', 'genes', 'and', 'novel', 'isoforms', 'of', 'annotated', 'genes', 'due', 'to', 'its', 'ability', 'to', 'sequence', 'full', 'length', 'transcripts', 'or', 'fragments', 'with', 'significant', 'lengths', 'additionally', 'pacbio', 'sequencing', 'technique', 'provides', 'information', 'that', 'is', 'useful', 'for', 'the', 'direct', 'detection', 'of', 'base', 'modifications', 'such', 'as', 'methylation', 'in', 'addition', 'to', 'using', 'pacbio', 'sequencing', 'alone', 'many', 'hybrid', 'sequencing', 'strategies', 'have', 'been', 'developed', 'to', 'make', 'use', 'of', 'more', 'accurate', 'short', 'reads', 'in', 'conjunction', 'with', 'pacbio', 'long', 'reads', 'in', 'general', 'hybrid', 'sequencing', 'strategies', 'are', 'more', 'affordable', 'and', 'scalable', 'especially', 'for', 'small', 'size', 'laboratories', 'than', 'using', 'pacbio', 'sequencing', 'alone', 'the', 'advent', 'of', 'pacbio', 'sequencing', 'has', 'made', 'available', 'much', 'information', 'that', 'could', 'not', 'be', 'obtained', 'via', 'sgs', 'alone'], ['micrornas', 'mirnas', 'are', 'endogenous', 'small', 'non', 'coding', 'rnas', 'that', 'function', 'in', 'regulation', 'of', 'gene', 'expression', 'compelling', 'evidences', 'have', 'demonstrated', 'that', 'mirna', 'expression', 'is', 'dysregulated', 'in', 'human', 'cancer', 'through', 'various', 'mechanisms', 'including', 'amplification', 'or', 'deletion', 'of', 'mirna', 'genes', 'abnormal', 'transcriptional', 'control', 'of', 'mirnas', 'dysregulated', 'epigenetic', 'changes', 'and', 'defects', 'in', 'the', 'mirna', 'biogenesis', 'machinery', 'mirnas', 'may', 'function', 'as', 'either', 'oncogenes', 'or', 'tumor', 'suppressors', 'under', 'certain', 'conditions', 'the', 'dysregulated', 'mirnas', 'have', 'been', 'shown', 'to', 'affect', 'the', 'hallmarks', 'of', 'cancer', 'including', 'sustaining', 'proliferative', 'signaling', 'evading', 'growth', 'suppressors', 'resisting', 'cell', 'death', 'activating', 'invasion', 'and', 'metastasis', 'and', 'inducing', 'angiogenesis', 'an', 'increasing', 'number', 'of', 'studies', 'have', 'identified', 'mirnas', 'as', 'potential', 'biomarkers', 'for', 'human', 'cancer', 'diagnosis', 'prognosis', 'and', 'therapeutic', 'targets', 'or', 'tools', 'which', 'needs', 'further', 'investigation', 'and', 'validation', 'in', 'this', 'review', 'we', 'focus', 'on', 'how', 'mirnas', 'regulate', 'the', 'development', 'of', 'human', 'tumors', 'by', 'acting', 'as', 'tumor', 'suppressors', 'or', 'oncogenes'], ['if', 'quadruplexes', 'form', 'so', 'readily', 'in', 'vitro', 'nature', 'will', 'have', 'found', 'way', 'of', 'using', 'them', 'in', 'vivo', 'statement', 'by', 'aaron', 'klug', 'over', 'years', 'ago', 'during', 'the', 'last', 'decade', 'four', 'stranded', 'helical', 'structures', 'called', 'quadruplex', 'or', 'have', 'emerged', 'from', 'being', 'structural', 'curiosity', 'observed', 'in', 'vitro', 'to', 'being', 'recognized', 'as', 'possible', 'nucleic', 'acid', 'based', 'mechanism', 'for', 'regulating', 'multiple', 'biological', 'processes', 'in', 'vivo', 'the', 'sequencing', 'of', 'many', 'genomes', 'has', 'revealed', 'that', 'they', 'are', 'rich', 'in', 'sequence', 'motifs', 'that', 'have', 'the', 'potential', 'to', 'form', 'quadruplexes', 'and', 'that', 'their', 'location', 'is', 'non', 'random', 'correlating', 'with', 'functionally', 'important', 'genomic', 'regions', 'in', 'this', 'short', 'review', 'we', 'summarize', 'recent', 'evidence', 'for', 'the', 'in', 'vivo', 'presence', 'and', 'function', 'of', 'dna', 'and', 'rna', 'quadruplexes', 'in', 'various', 'cellular', 'pathways', 'including', 'dna', 'replication', 'gene', 'expression', 'and', 'telomere', 'maintenance', 'we', 'also', 'highlight', 'remaining', 'open', 'questions', 'that', 'will', 'have', 'to', 'be', 'addressed', 'in', 'the', 'future'], ['the', 'aberrant', 'expression', 'of', 'transcription', 'factor', 'basonuclin', 'bnc', 'had', 'been', 'reported', 'in', 'different', 'kinds', 'of', 'tumors', 'here', 'we', 'investigated', 'the', 'expression', 'and', 'methylation', 'status', 'of', 'two', 'basonuclin', 'homologs', 'bnc', 'and', 'bnc', 'in', 'hepatocellular', 'carcinoma', 'hcc', 'we', 'found', 'that', 'the', 'expression', 'levels', 'of', 'both', 'bnc', 'and', 'bnc', 'were', 'down', 'regulated', 'in', 'hcc', 'cell', 'lines', 'and', 'primary', 'hcc', 'tissues', 'the', 'frequency', 'and', 'intensity', 'of', 'bnc', 'promoter', 'in', 'tumor', 'tissues', 'was', 'significantly', 'higher', 'than', 'that', 'in', 'adjacent', 'non', 'tumor', 'tissues', 'aza', 'deoxycytidine', 'treatment', 'could', 'significantly', 'increase', 'the', 'bnc', 'expression', 'in', 'the', 'methylated', 'hcc', 'cell', 'lines', 'which', 'implicated', 'that', 'epigenetic', 'modification', 'contributed', 'to', 'the', 'down', 'regulation', 'of', 'bnc', 'in', 'addition', 'bnc', 'in', 'tumor', 'tissues', 'was', 'more', 'likely', 'to', 'happen', 'in', 'female', 'patients', 'no', 'methylation', 'of', 'the', 'bnc', 'promoter', 'was', 'found', 'in', 'hcc', 'tumor', 'tissues', 'however', 'frequent', 'deletion', 'of', 'the', 'bnc', 'gene', 'was', 'observed', 'which', 'indicated', 'that', 'the', 'chromosomal', 'loss', 'of', 'the', 'bnc', 'gene', 'might', 'be', 'one', 'important', 'reason', 'for', 'its', 'lower', 'expression', 'level', 'in', 'hcc', 'our', 'results', 'suggested', 'that', 'bnc', 'and', 'bnc', 'were', 'down', 'regulated', 'in', 'hcc', 'which', 'may', 'provide', 'new', 'insight', 'into', 'the', 'tumorigenesis', 'of', 'hcc'], ['an', 'increasingly', 'recognized', 'resistance', 'mechanism', 'to', 'androgen', 'receptor', 'ar', 'directed', 'therapy', 'in', 'prostate', 'cancer', 'involves', 'epithelial', 'plasticity', 'in', 'which', 'tumor', 'cells', 'demonstrate', 'low', 'to', 'absent', 'ar', 'expression', 'and', 'often', 'have', 'neuroendocrine', 'features', 'the', 'etiology', 'and', 'molecular', 'basis', 'for', 'this', 'alternative', 'treatment', 'resistant', 'cell', 'state', 'remain', 'incompletely', 'understood', 'here', 'by', 'analyzing', 'whole', 'exome', 'sequencing', 'data', 'of', 'metastatic', 'biopsies', 'from', 'patients', 'we', 'observed', 'substantial', 'genomic', 'overlap', 'between', 'castration', 'resistant', 'tumors', 'that', 'were', 'histologically', 'characterized', 'as', 'prostate', 'adenocarcinomas', 'crpc', 'adeno', 'and', 'neuroendocrine', 'prostate', 'cancer', 'crpc', 'ne', 'analysis', 'of', 'biopsy', 'samples', 'from', 'the', 'same', 'individuals', 'over', 'time', 'points', 'to', 'model', 'most', 'consistent', 'with', 'divergent', 'clonal', 'evolution', 'genome', 'wide', 'dna', 'methylation', 'analysis', 'revealed', 'marked', 'epigenetic', 'differences', 'between', 'crpc', 'ne', 'tumors', 'and', 'crpc', 'adeno', 'and', 'also', 'designated', 'samples', 'of', 'crpc', 'adeno', 'with', 'clinical', 'features', 'of', 'ar', 'independence', 'as', 'crpc', 'ne', 'suggesting', 'that', 'epigenetic', 'modifiers', 'may', 'play', 'role', 'in', 'the', 'induction', 'and', 'or', 'maintenance', 'of', 'this', 'treatment', 'resistant', 'state', 'this', 'study', 'supports', 'the', 'emergence', 'of', 'an', 'alternative', 'ar', 'indifferent', 'cell', 'state', 'through', 'divergent', 'clonal', 'evolution', 'as', 'mechanism', 'of', 'treatment', 'resistance', 'in', 'advanced', 'prostate', 'cancer'], ['tp', 'is', 'the', 'most', 'frequently', 'mutated', 'gene', 'in', 'human', 'cancer', 'functionally', 'is', 'activated', 'by', 'host', 'of', 'stress', 'stimuli', 'and', 'in', 'turn', 'governs', 'an', 'exquisitely', 'complex', 'anti', 'proliferative', 'transcriptional', 'program', 'that', 'touches', 'upon', 'bewildering', 'array', 'of', 'biological', 'responses', 'despite', 'the', 'many', 'unveiled', 'facets', 'of', 'the', 'network', 'clear', 'appreciation', 'of', 'how', 'and', 'in', 'what', 'contexts', 'exerts', 'its', 'diverse', 'effects', 'remains', 'unclear', 'how', 'can', 'we', 'interpret', 'disparate', 'activities', 'and', 'the', 'consequences', 'of', 'its', 'dysfunction', 'to', 'understand', 'how', 'cell', 'type', 'mutation', 'profile', 'and', 'epigenetic', 'cell', 'state', 'dictate', 'outcomes', 'and', 'how', 'might', 'we', 'restore', 'its', 'tumor', 'suppressive', 'activities', 'in', 'cancer'], ['the', 'field', 'of', 'machine', 'learning', 'which', 'aims', 'to', 'develop', 'computer', 'algorithms', 'that', 'improve', 'with', 'experience', 'holds', 'promise', 'to', 'enable', 'computers', 'to', 'assist', 'humans', 'in', 'the', 'analysis', 'of', 'large', 'complex', 'data', 'sets', 'here', 'we', 'provide', 'an', 'overview', 'of', 'machine', 'learning', 'applications', 'for', 'the', 'analysis', 'of', 'genome', 'sequencing', 'data', 'sets', 'including', 'the', 'annotation', 'of', 'sequence', 'elements', 'and', 'epigenetic', 'proteomic', 'or', 'metabolomic', 'data', 'we', 'present', 'considerations', 'and', 'recurrent', 'challenges', 'in', 'the', 'application', 'of', 'supervised', 'semi', 'supervised', 'and', 'unsupervised', 'machine', 'learning', 'methods', 'as', 'well', 'as', 'of', 'generative', 'and', 'discriminative', 'modelling', 'approaches', 'we', 'provide', 'general', 'guidelines', 'to', 'assist', 'in', 'the', 'selection', 'of', 'these', 'machine', 'learning', 'methods', 'and', 'their', 'practical', 'application', 'for', 'the', 'analysis', 'of', 'genetic', 'and', 'genomic', 'data', 'sets'], ['mammalian', 'dna', 'methylation', 'is', 'critical', 'epigenetic', 'mechanism', 'orchestrating', 'gene', 'expression', 'networks', 'in', 'many', 'biological', 'processes', 'however', 'investigation', 'of', 'the', 'functions', 'of', 'specific', 'methylation', 'events', 'remains', 'challenging', 'here', 'we', 'demonstrate', 'that', 'fusion', 'of', 'tet', 'or', 'dnmt', 'with', 'catalytically', 'inactive', 'cas', 'dcas', 'enables', 'targeted', 'dna', 'methylation', 'editing', 'targeting', 'of', 'the', 'dcas', 'tet', 'or', 'dnmt', 'fusion', 'protein', 'to', 'methylated', 'or', 'unmethylated', 'promoter', 'sequences', 'caused', 'activation', 'or', 'silencing', 'respectively', 'of', 'an', 'endogenous', 'reporter', 'targeted', 'demethylation', 'of', 'the', 'bdnf', 'promoter', 'iv', 'or', 'the', 'myod', 'distal', 'enhancer', 'by', 'dcas', 'tet', 'induced', 'bdnf', 'expression', 'in', 'post', 'mitotic', 'neurons', 'or', 'activated', 'myod', 'facilitating', 'reprogramming', 'of', 'fibroblasts', 'into', 'myoblasts', 'respectively', 'targeted', 'de', 'novo', 'methylation', 'of', 'ctcf', 'loop', 'anchor', 'site', 'by', 'dcas', 'dnmt', 'blocked', 'ctcf', 'binding', 'and', 'interfered', 'with', 'dna', 'looping', 'causing', 'altered', 'gene', 'expression', 'in', 'the', 'neighboring', 'loop', 'finally', 'we', 'show', 'that', 'these', 'tools', 'can', 'edit', 'dna', 'methylation', 'in', 'mice', 'demonstrating', 'their', 'wide', 'utility', 'for', 'functional', 'studies', 'of', 'epigenetic', 'regulation'], ['diverse', 'classes', 'of', 'rna', 'ranging', 'from', 'small', 'to', 'long', 'non', 'coding', 'rnas', 'have', 'emerged', 'as', 'key', 'regulators', 'of', 'gene', 'expression', 'genome', 'stability', 'and', 'defence', 'against', 'foreign', 'genetic', 'elements', 'small', 'rnas', 'modify', 'chromatin', 'structure', 'and', 'silence', 'transcription', 'by', 'guiding', 'argonaute', 'containing', 'complexes', 'to', 'complementary', 'nascent', 'rna', 'scaffolds', 'and', 'then', 'mediating', 'the', 'recruitment', 'of', 'histone', 'and', 'dna', 'in', 'addition', 'recent', 'advances', 'suggest', 'that', 'chromatin', 'associated', 'long', 'non', 'coding', 'rna', 'scaffolds', 'also', 'recruit', 'chromatin', 'modifying', 'complexes', 'independently', 'of', 'small', 'rnas', 'these', 'co', 'transcriptional', 'silencing', 'mechanisms', 'form', 'powerful', 'rna', 'surveillance', 'systems', 'that', 'detect', 'and', 'silence', 'inappropriate', 'transcription', 'events', 'and', 'provide', 'memory', 'of', 'these', 'events', 'via', 'self', 'reinforcing', 'epigenetic', 'loops'], ['next', 'generation', 'sequencing', 'has', 'revealed', 'that', 'more', 'than', 'of', 'human', 'cancers', 'harbour', 'mutations', 'in', 'enzymes', 'that', 'are', 'involved', 'in', 'chromatin', 'organization', 'tumour', 'cells', 'not', 'only', 'are', 'activated', 'by', 'genetic', 'and', 'epigenetic', 'alterations', 'but', 'also', 'routinely', 'use', 'epigenetic', 'processes', 'to', 'ensure', 'their', 'escape', 'from', 'chemotherapy', 'and', 'host', 'immune', 'surveillance', 'hence', 'growing', 'emphasis', 'of', 'recent', 'drug', 'discovery', 'efforts', 'has', 'been', 'on', 'targeting', 'the', 'epigenome', 'including', 'dna', 'methylation', 'and', 'histone', 'modifications', 'with', 'several', 'new', 'drugs', 'being', 'tested', 'and', 'some', 'already', 'approved', 'by', 'the', 'us', 'food', 'and', 'drug', 'administration', 'fda', 'the', 'future', 'will', 'see', 'the', 'increasing', 'success', 'of', 'combining', 'epigenetic', 'drugs', 'with', 'other', 'therapies', 'as', 'epigenetic', 'drugs', 'target', 'the', 'epigenome', 'as', 'whole', 'these', 'true', 'genomic', 'medicines', 'lessen', 'the', 'need', 'for', 'precision', 'approaches', 'to', 'individualized', 'therapies'], ['one', 'carbon', 'metabolism', 'mediated', 'by', 'the', 'folate', 'cofactor', 'supports', 'multiple', 'physiological', 'processes', 'these', 'include', 'biosynthesis', 'purines', 'and', 'thymidine', 'amino', 'acid', 'homeostasis', 'glycine', 'serine', 'and', 'methionine', 'epigenetic', 'maintenance', 'and', 'redox', 'defense', 'both', 'within', 'eukaryotic', 'cells', 'and', 'across', 'organs', 'metabolic', 'reactions', 'are', 'here', 'we', 'review', 'the', 'fundamentals', 'of', 'mammalian', 'metabolism', 'including', 'the', 'pathways', 'active', 'in', 'different', 'compartments', 'cell', 'types', 'and', 'biological', 'states', 'emphasis', 'is', 'given', 'to', 'recent', 'discoveries', 'enabled', 'by', 'modern', 'genetics', 'analytical', 'chemistry', 'and', 'isotope', 'tracing', 'an', 'emerging', 'theme', 'is', 'the', 'biological', 'importance', 'of', 'mitochondrial', 'reactions', 'both', 'for', 'producing', 'units', 'that', 'are', 'exported', 'to', 'the', 'cytosol', 'and', 'for', 'making', 'additional', 'products', 'including', 'glycine', 'and', 'nadph', 'increased', 'clarity', 'regarding', 'differential', 'folate', 'pathway', 'usage', 'in', 'cancer', 'stem', 'cells', 'development', 'and', 'adult', 'physiology', 'is', 'reviewed', 'and', 'highlights', 'new', 'opportunities', 'for', 'selective', 'therapeutic', 'intervention'], ['acetyl', 'coenzyme', 'acetyl', 'coa', 'is', 'central', 'metabolic', 'intermediate', 'the', 'abundance', 'of', 'acetyl', 'coa', 'in', 'distinct', 'subcellular', 'compartments', 'reflects', 'the', 'general', 'energetic', 'state', 'of', 'the', 'cell', 'moreover', 'acetyl', 'coa', 'concentrations', 'influence', 'the', 'activity', 'or', 'specificity', 'of', 'multiple', 'enzymes', 'either', 'in', 'an', 'allosteric', 'manner', 'or', 'by', 'altering', 'substrate', 'availability', 'finally', 'by', 'influencing', 'the', 'acetylation', 'profile', 'of', 'several', 'proteins', 'including', 'histones', 'acetyl', 'coa', 'controls', 'key', 'cellular', 'processes', 'including', 'energy', 'metabolism', 'mitosis', 'and', 'autophagy', 'both', 'directly', 'and', 'via', 'the', 'epigenetic', 'regulation', 'of', 'gene', 'expression', 'thus', 'acetyl', 'coa', 'determines', 'the', 'balance', 'between', 'cellular', 'catabolism', 'and', 'anabolism', 'by', 'simultaneously', 'operating', 'as', 'metabolic', 'intermediate', 'and', 'as', 'second', 'messenger'], ['cancer', 'growth', 'and', 'progression', 'are', 'associated', 'with', 'immune', 'suppression', 'cancer', 'cells', 'have', 'the', 'ability', 'to', 'activate', 'different', 'immune', 'checkpoint', 'pathways', 'that', 'harbor', 'functions', 'monoclonal', 'antibodies', 'that', 'target', 'immune', 'checkpoints', 'provided', 'an', 'immense', 'breakthrough', 'in', 'cancer', 'therapeutics', 'among', 'the', 'immune', 'checkpoint', 'inhibitors', 'pd', 'pd', 'and', 'ctla', 'inhibitors', 'showed', 'promising', 'therapeutic', 'outcomes', 'and', 'some', 'have', 'been', 'approved', 'for', 'certain', 'cancer', 'treatments', 'while', 'others', 'are', 'under', 'clinical', 'trials', 'recent', 'reports', 'have', 'shown', 'that', 'patients', 'with', 'various', 'malignancies', 'benefit', 'from', 'immune', 'checkpoint', 'inhibitor', 'treatment', 'however', 'mainstream', 'initiation', 'of', 'immune', 'checkpoint', 'therapy', 'to', 'treat', 'cancers', 'is', 'obstructed', 'by', 'the', 'low', 'response', 'rate', 'and', 'immune', 'related', 'adverse', 'events', 'in', 'some', 'cancer', 'patients', 'this', 'has', 'given', 'rise', 'to', 'the', 'need', 'for', 'developing', 'sets', 'of', 'biomarkers', 'that', 'predict', 'the', 'response', 'to', 'immune', 'checkpoint', 'blockade', 'and', 'immune', 'related', 'adverse', 'events', 'in', 'this', 'review', 'we', 'discuss', 'different', 'predictive', 'biomarkers', 'for', 'anti', 'pd', 'pd', 'and', 'anti', 'ctla', 'inhibitors', 'including', 'immune', 'cells', 'pd', 'overexpression', 'neoantigens', 'and', 'genetic', 'and', 'epigenetic', 'signatures', 'potential', 'approaches', 'for', 'further', 'developing', 'highly', 'reliable', 'predictive', 'biomarkers', 'should', 'facilitate', 'patient', 'selection', 'for', 'and', 'decision', 'making', 'related', 'to', 'immune', 'checkpoint', 'inhibitor', 'based', 'therapies'], ['nucleosome', 'positioning', 'varies', 'between', 'cell', 'types', 'by', 'deep', 'sequencing', 'cell', 'free', 'dna', 'cfdna', 'isolated', 'from', 'circulating', 'blood', 'plasma', 'we', 'generated', 'maps', 'of', 'genome', 'wide', 'in', 'vivo', 'nucleosome', 'occupancy', 'and', 'found', 'that', 'short', 'cfdna', 'fragments', 'harbor', 'footprints', 'of', 'transcription', 'factors', 'the', 'cfdna', 'nucleosome', 'occupancies', 'correlate', 'well', 'with', 'the', 'nuclear', 'architecture', 'gene', 'structure', 'and', 'expression', 'observed', 'in', 'cells', 'suggesting', 'that', 'they', 'could', 'inform', 'the', 'cell', 'type', 'of', 'origin', 'nucleosome', 'spacing', 'inferred', 'from', 'cfdna', 'in', 'healthy', 'individuals', 'correlates', 'most', 'strongly', 'with', 'epigenetic', 'features', 'of', 'lymphoid', 'and', 'myeloid', 'cells', 'consistent', 'with', 'hematopoietic', 'cell', 'death', 'as', 'the', 'normal', 'source', 'of', 'cfdna', 'we', 'build', 'on', 'this', 'observation', 'to', 'show', 'how', 'nucleosome', 'footprints', 'can', 'be', 'used', 'to', 'infer', 'cell', 'types', 'contributing', 'to', 'cfdna', 'in', 'pathological', 'states', 'such', 'as', 'cancer', 'since', 'this', 'strategy', 'does', 'not', 'rely', 'on', 'genetic', 'differences', 'to', 'distinguish', 'between', 'contributing', 'tissues', 'it', 'may', 'enable', 'the', 'noninvasive', 'monitoring', 'of', 'much', 'broader', 'set', 'of', 'clinical', 'conditions', 'than', 'currently', 'possible'], ['the', 'cas', 'protein', 'crispr', 'associated', 'protein', 'derived', 'from', 'type', 'ii', 'crispr', 'clustered', 'regularly', 'interspaced', 'short', 'palindromic', 'repeats', 'bacterial', 'immune', 'systems', 'is', 'emerging', 'as', 'powerful', 'tool', 'for', 'engineering', 'the', 'genome', 'in', 'diverse', 'organisms', 'as', 'an', 'rna', 'guided', 'dna', 'endonuclease', 'cas', 'can', 'be', 'easily', 'programmed', 'to', 'target', 'new', 'sites', 'by', 'altering', 'its', 'guide', 'rna', 'sequence', 'and', 'its', 'development', 'as', 'tool', 'has', 'made', 'sequence', 'specific', 'gene', 'editing', 'several', 'magnitudes', 'easier', 'the', 'nuclease', 'deactivated', 'form', 'of', 'cas', 'further', 'provides', 'versatile', 'rna', 'guided', 'dna', 'targeting', 'platform', 'for', 'regulating', 'and', 'imaging', 'the', 'genome', 'as', 'well', 'as', 'for', 'rewriting', 'the', 'epigenetic', 'status', 'all', 'in', 'sequence', 'specific', 'manner', 'with', 'all', 'of', 'these', 'advances', 'we', 'have', 'just', 'begun', 'to', 'explore', 'the', 'possible', 'applications', 'of', 'cas', 'in', 'biomedical', 'research', 'and', 'therapeutics', 'in', 'this', 'review', 'we', 'describe', 'the', 'current', 'models', 'of', 'cas', 'function', 'and', 'the', 'structural', 'and', 'biochemical', 'studies', 'that', 'support', 'it', 'we', 'focus', 'on', 'the', 'applications', 'of', 'cas', 'for', 'genome', 'editing', 'regulation', 'and', 'imaging', 'discuss', 'other', 'possible', 'applications', 'and', 'some', 'technical', 'considerations', 'and', 'highlight', 'the', 'many', 'advantages', 'that', 'crispr', 'cas', 'technology', 'offers'], ['carcinogenesis', 'cannot', 'be', 'explained', 'only', 'by', 'genetic', 'alterations', 'but', 'also', 'involves', 'epigenetic', 'processes', 'modification', 'of', 'histones', 'by', 'acetylation', 'plays', 'key', 'role', 'in', 'epigenetic', 'regulation', 'of', 'gene', 'expression', 'and', 'is', 'controlled', 'by', 'the', 'balance', 'between', 'histone', 'deacetylases', 'hdac', 'and', 'histone', 'hat', 'hdac', 'inhibitors', 'induce', 'cancer', 'cell', 'cycle', 'arrest', 'differentiation', 'and', 'cell', 'death', 'reduce', 'angiogenesis', 'and', 'modulate', 'immune', 'response', 'mechanisms', 'of', 'anticancer', 'effects', 'of', 'hdac', 'inhibitors', 'are', 'not', 'uniform', 'they', 'may', 'be', 'different', 'and', 'depend', 'on', 'the', 'cancer', 'type', 'hdac', 'inhibitors', 'doses', 'etc', 'hdac', 'inhibitors', 'seem', 'to', 'be', 'promising', 'anti', 'cancer', 'drugs', 'particularly', 'in', 'the', 'combination', 'with', 'other', 'anti', 'cancer', 'drugs', 'and', 'or', 'radiotherapy', 'hdac', 'inhibitors', 'vorinostat', 'romidepsin', 'and', 'belinostat', 'have', 'been', 'approved', 'for', 'some', 'cell', 'lymphoma', 'and', 'panobinostat', 'for', 'multiple', 'myeloma', 'other', 'hdac', 'inhibitors', 'are', 'in', 'clinical', 'trials', 'for', 'the', 'treatment', 'of', 'hematological', 'and', 'solid', 'malignancies', 'the', 'results', 'of', 'such', 'studies', 'are', 'promising', 'but', 'further', 'larger', 'studies', 'are', 'needed', 'because', 'of', 'the', 'reversibility', 'of', 'epigenetic', 'changes', 'during', 'cancer', 'development', 'the', 'potency', 'of', 'epigenetic', 'therapies', 'seems', 'to', 'be', 'of', 'great', 'importance', 'here', 'we', 'summarize', 'the', 'data', 'on', 'different', 'classes', 'of', 'hdac', 'inhibitors', 'mechanisms', 'of', 'their', 'actions', 'and', 'discuss', 'novel', 'results', 'of', 'preclinical', 'and', 'clinical', 'studies', 'including', 'the', 'combination', 'with', 'other', 'therapeutic', 'modalities'], ['the', 'finding', 'that', 'small', 'non', 'coding', 'rnas', 'ncrnas', 'are', 'able', 'to', 'control', 'gene', 'expression', 'in', 'sequence', 'specific', 'manner', 'has', 'had', 'massive', 'impact', 'on', 'biology', 'recent', 'improvements', 'in', 'high', 'throughput', 'sequencing', 'and', 'computational', 'prediction', 'methods', 'have', 'allowed', 'the', 'discovery', 'and', 'classification', 'of', 'several', 'types', 'of', 'ncrnas', 'based', 'on', 'their', 'precursor', 'structures', 'biogenesis', 'pathways', 'and', 'modes', 'of', 'action', 'ncrnas', 'are', 'classified', 'as', 'small', 'interfering', 'rnas', 'sirnas', 'micrornas', 'mirnas', 'piwi', 'interacting', 'rnas', 'pirnas', 'endogenous', 'small', 'interfering', 'rnas', 'endo', 'sirnas', 'or', 'esirnas', 'promoter', 'associate', 'rnas', 'prnas', 'small', 'nucleolar', 'rnas', 'snornas', 'and', 'sno', 'derived', 'rnas', 'among', 'these', 'mirnas', 'appear', 'as', 'important', 'cytoplasmic', 'regulators', 'of', 'gene', 'expression', 'mirnas', 'act', 'as', 'post', 'transcriptional', 'regulators', 'of', 'their', 'messenger', 'rna', 'mrna', 'targets', 'via', 'mrna', 'degradation', 'and', 'or', 'translational', 'repression', 'however', 'it', 'is', 'becoming', 'evident', 'that', 'mirnas', 'also', 'have', 'specific', 'nuclear', 'functions', 'among', 'these', 'the', 'most', 'studied', 'and', 'debated', 'activity', 'is', 'the', 'mirna', 'guided', 'transcriptional', 'control', 'of', 'gene', 'expression', 'although', 'available', 'data', 'detail', 'quite', 'precisely', 'the', 'effectors', 'of', 'this', 'activity', 'the', 'mechanisms', 'by', 'which', 'mirnas', 'identify', 'their', 'gene', 'targets', 'to', 'control', 'transcription', 'are', 'still', 'matter', 'of', 'debate', 'here', 'we', 'focus', 'on', 'nuclear', 'functions', 'of', 'mirnas', 'and', 'on', 'alternative', 'mechanisms', 'of', 'target', 'recognition', 'at', 'the', 'promoter', 'lavel', 'by', 'mirnas', 'in', 'carrying', 'out', 'transcriptional', 'gene', 'silencing'], ['since', 'decades', 'it', 'has', 'been', 'known', 'that', 'non', 'protein', 'coding', 'rnas', 'have', 'important', 'cellular', 'functions', 'deep', 'sequencing', 'recently', 'facilitated', 'the', 'discovery', 'of', 'thousands', 'of', 'novel', 'transcripts', 'now', 'classified', 'as', 'long', 'noncoding', 'rnas', 'lncrnas', 'in', 'many', 'vertebrate', 'and', 'invertebrate', 'species', 'lncrnas', 'are', 'involved', 'in', 'wide', 'range', 'of', 'cellular', 'mechanisms', 'from', 'almost', 'all', 'aspects', 'of', 'gene', 'expression', 'to', 'protein', 'translation', 'and', 'stability', 'recent', 'findings', 'implicate', 'lncrnas', 'as', 'key', 'players', 'of', 'cellular', 'differentiation', 'cell', 'lineage', 'choice', 'organogenesis', 'and', 'tissue', 'homeostasis', 'moreover', 'lncrnas', 'are', 'involved', 'in', 'pathological', 'conditions', 'such', 'as', 'cancer', 'and', 'cardiovascular', 'disease', 'and', 'therefore', 'provide', 'novel', 'biomarkers', 'and', 'pharmaceutical', 'targets', 'here', 'we', 'discuss', 'examples', 'illustrating', 'the', 'versatility', 'of', 'lncrnas', 'in', 'gene', 'control', 'development', 'and', 'differentiation', 'as', 'well', 'as', 'in', 'human', 'disease'], ['identifying', 'and', 'validating', 'molecular', 'targets', 'of', 'interventions', 'that', 'extend', 'the', 'human', 'health', 'span', 'and', 'lifespan', 'has', 'been', 'difficult', 'as', 'most', 'clinical', 'biomarkers', 'are', 'not', 'sufficiently', 'representative', 'of', 'the', 'fundamental', 'mechanisms', 'of', 'ageing', 'to', 'serve', 'as', 'their', 'indicators', 'in', 'recent', 'breakthrough', 'biomarkers', 'of', 'ageing', 'based', 'on', 'dna', 'methylation', 'data', 'have', 'enabled', 'accurate', 'age', 'estimates', 'for', 'any', 'tissue', 'across', 'the', 'entire', 'life', 'course', 'these', 'epigenetic', 'clocks', 'link', 'developmental', 'and', 'maintenance', 'processes', 'to', 'biological', 'ageing', 'giving', 'rise', 'to', 'unified', 'theory', 'of', 'life', 'course', 'epigenetic', 'biomarkers', 'may', 'help', 'to', 'address', 'long', 'standing', 'questions', 'in', 'many', 'fields', 'including', 'the', 'central', 'question', 'why', 'do', 'we', 'age'], ['epithelial', 'mesenchymal', 'transition', 'emt', 'is', 'process', 'in', 'which', 'epithelial', 'cells', 'acquire', 'mesenchymal', 'features', 'in', 'cancer', 'emt', 'is', 'associated', 'with', 'tumor', 'initiation', 'invasion', 'metastasis', 'and', 'resistance', 'to', 'therapy', 'recently', 'it', 'has', 'been', 'demonstrated', 'that', 'emt', 'is', 'not', 'binary', 'process', 'but', 'occurs', 'through', 'distinct', 'cellular', 'states', 'here', 'we', 'review', 'the', 'recent', 'studies', 'that', 'demonstrate', 'the', 'existence', 'of', 'these', 'different', 'emt', 'states', 'in', 'cancer', 'and', 'the', 'mechanisms', 'regulating', 'their', 'functions', 'we', 'discuss', 'the', 'different', 'functional', 'characteristics', 'such', 'as', 'proliferation', 'propagation', 'plasticity', 'invasion', 'and', 'metastasis', 'associated', 'with', 'the', 'distinct', 'emt', 'states', 'we', 'summarize', 'the', 'role', 'of', 'the', 'transcriptional', 'and', 'epigenetic', 'landscapes', 'gene', 'regulatory', 'network', 'and', 'their', 'surrounding', 'niche', 'in', 'controlling', 'the', 'transition', 'through', 'the', 'different', 'emt', 'states'], ['methylation', 'of', 'dna', 'and', 'of', 'histone', 'at', 'lys', 'are', 'highly', 'correlated', 'with', 'gene', 'silencing', 'in', 'eukaryotes', 'from', 'fungi', 'to', 'humans', 'both', 'of', 'these', 'epigenetic', 'marks', 'need', 'to', 'be', 'established', 'at', 'specific', 'regions', 'of', 'the', 'genome', 'and', 'then', 'maintained', 'at', 'these', 'sites', 'through', 'cell', 'division', 'protein', 'structural', 'domains', 'that', 'specifically', 'recognize', 'methylated', 'dna', 'and', 'methylated', 'histones', 'are', 'key', 'for', 'targeting', 'enzymes', 'that', 'catalyse', 'these', 'marks', 'to', 'appropriate', 'genome', 'sites', 'genetic', 'genomic', 'structural', 'and', 'biochemical', 'data', 'reveal', 'connections', 'between', 'these', 'two', 'epigenetic', 'marks', 'and', 'these', 'domains', 'mediate', 'much', 'of', 'the', 'crosstalk'], ['this', 'year', 'is', 'the', 'tenth', 'anniversary', 'of', 'the', 'publication', 'in', 'this', 'journal', 'of', 'model', 'suggesting', 'the', 'existence', 'of', 'tumour', 'progenitor', 'genes', 'these', 'genes', 'are', 'epigenetically', 'disrupted', 'at', 'the', 'earliest', 'stages', 'of', 'malignancies', 'even', 'before', 'mutations', 'and', 'thus', 'cause', 'altered', 'differentiation', 'throughout', 'tumour', 'evolution', 'the', 'past', 'decade', 'of', 'discovery', 'in', 'cancer', 'epigenetics', 'has', 'revealed', 'number', 'of', 'similarities', 'between', 'cancer', 'genes', 'and', 'stem', 'cell', 'reprogramming', 'genes', 'widespread', 'mutations', 'in', 'epigenetic', 'regulators', 'and', 'the', 'part', 'played', 'by', 'chromatin', 'structure', 'in', 'cellular', 'plasticity', 'in', 'both', 'development', 'and', 'cancer', 'in', 'the', 'light', 'of', 'these', 'discoveries', 'we', 'suggest', 'here', 'framework', 'for', 'cancer', 'epigenetics', 'involving', 'three', 'types', 'of', 'genes', 'epigenetic', 'mediators', 'corresponding', 'to', 'the', 'tumour', 'progenitor', 'genes', 'suggested', 'earlier', 'epigenetic', 'modifiers', 'of', 'the', 'mediators', 'which', 'are', 'frequently', 'mutated', 'in', 'cancer', 'and', 'epigenetic', 'modulators', 'upstream', 'of', 'the', 'modifiers', 'which', 'are', 'responsive', 'to', 'changes', 'in', 'the', 'cellular', 'environment', 'and', 'often', 'linked', 'to', 'the', 'nuclear', 'architecture', 'we', 'suggest', 'that', 'this', 'classification', 'is', 'helpful', 'in', 'framing', 'new', 'diagnostic', 'and', 'therapeutic', 'approaches', 'to', 'cancer'], ['specification', 'of', 'primordial', 'germ', 'cells', 'pgcs', 'marks', 'the', 'beginning', 'of', 'the', 'totipotent', 'state', 'however', 'without', 'tractable', 'experimental', 'model', 'the', 'mechanism', 'of', 'human', 'pgc', 'hpgc', 'specification', 'remains', 'unclear', 'here', 'we', 'demonstrate', 'specification', 'of', 'hpgc', 'like', 'cells', 'hpgclcs', 'from', 'germline', 'competent', 'pluripotent', 'stem', 'cells', 'the', 'characteristics', 'of', 'hpgclcs', 'are', 'consistent', 'with', 'the', 'embryonic', 'hpgcs', 'and', 'germline', 'seminoma', 'that', 'share', 'cd', 'cell', 'surface', 'marker', 'which', 'collectively', 'defines', 'likely', 'progression', 'of', 'the', 'early', 'human', 'germline', 'remarkably', 'sox', 'is', 'the', 'key', 'regulator', 'of', 'hpgc', 'like', 'fate', 'whereas', 'blimp', 'represses', 'endodermal', 'and', 'other', 'somatic', 'genes', 'during', 'specification', 'of', 'hpgclcs', 'notable', 'mechanistic', 'differences', 'between', 'mouse', 'and', 'human', 'pgc', 'specification', 'could', 'be', 'attributed', 'to', 'their', 'divergent', 'embryonic', 'development', 'and', 'pluripotent', 'states', 'which', 'might', 'affect', 'other', 'early', 'cell', 'fate', 'decisions', 'we', 'have', 'established', 'foundation', 'for', 'future', 'studies', 'on', 'resetting', 'of', 'the', 'epigenome', 'in', 'hpgclcs', 'and', 'hpgcs', 'for', 'totipotency', 'and', 'the', 'transmission', 'of', 'genetic', 'and', 'epigenetic', 'information'], ['the', 'discovery', 'of', 'long', 'non', 'coding', 'rna', 'lncrna', 'has', 'dramatically', 'altered', 'our', 'understanding', 'of', 'cancer', 'here', 'we', 'describe', 'comprehensive', 'analysis', 'of', 'lncrna', 'alterations', 'at', 'transcriptional', 'genomic', 'and', 'epigenetic', 'levels', 'in', 'human', 'tumor', 'specimens', 'across', 'cancer', 'types', 'from', 'the', 'cancer', 'genome', 'atlas', 'our', 'results', 'suggest', 'that', 'the', 'expression', 'and', 'dysregulation', 'of', 'lncrnas', 'are', 'highly', 'cancer', 'type', 'specific', 'compared', 'with', 'protein', 'coding', 'genes', 'using', 'the', 'integrative', 'data', 'generated', 'by', 'this', 'analysis', 'we', 'present', 'clinically', 'guided', 'small', 'interfering', 'rna', 'screening', 'strategy', 'and', 'co', 'expression', 'analysis', 'approach', 'to', 'identify', 'cancer', 'driver', 'lncrnas', 'and', 'predict', 'their', 'functions', 'this', 'provides', 'resource', 'for', 'investigating', 'lncrnas', 'in', 'cancer', 'and', 'lays', 'the', 'groundwork', 'for', 'the', 'development', 'of', 'new', 'diagnostics', 'and', 'treatments'], ['plant', 'genonnes', 'encode', 'various', 'small', 'rnas', 'that', 'function', 'in', 'distinct', 'yet', 'overlapping', 'genetic', 'and', 'epigenetic', 'silencing', 'pathways', 'however', 'the', 'abundance', 'and', 'diversity', 'of', 'small', 'rna', 'classes', 'varies', 'among', 'plant', 'species', 'suggesting', 'coevolution', 'between', 'environmental', 'adaptations', 'and', 'gene', 'silencing', 'mechanisms', 'biogenesis', 'of', 'small', 'rnas', 'in', 'plants', 'is', 'well', 'understood', 'but', 'we', 'are', 'just', 'beginning', 'to', 'uncover', 'their', 'intricate', 'regulation', 'and', 'activity', 'here', 'we', 'discuss', 'the', 'biogenesis', 'of', 'plant', 'small', 'rnas', 'such', 'as', 'nnicrornas', 'secondary', 'sirnas', 'and', 'heterochromatic', 'sirnas', 'and', 'their', 'diverse', 'cellular', 'and', 'developmental', 'functions', 'including', 'in', 'reproductive', 'transitions', 'genomic', 'imprinting', 'and', 'paramutation', 'we', 'also', 'discuss', 'the', 'diversification', 'of', 'small', 'rna', 'directed', 'silencing', 'pathways', 'through', 'the', 'expansion', 'of', 'rna', 'dependent', 'rna', 'polymerases', 'dicer', 'proteins', 'and', 'argonaute', 'proteins'], ['epigenetic', 'modifications', 'including', 'dna', 'methylation', 'represent', 'potential', 'mechanism', 'for', 'environmental', 'impacts', 'on', 'human', 'disease', 'maternal', 'smoking', 'in', 'pregnancy', 'remains', 'an', 'important', 'public', 'health', 'problem', 'that', 'impacts', 'child', 'health', 'in', 'myriad', 'of', 'ways', 'and', 'has', 'potential', 'lifelong', 'consequences', 'the', 'mechanisms', 'are', 'largely', 'unknown', 'but', 'epigenetics', 'most', 'likely', 'plays', 'role', 'we', 'formed', 'the', 'pregnancy', 'and', 'childhood', 'epigenetics', 'pace', 'consortium', 'and', 'meta', 'analyzed', 'across', 'cohorts', 'the', 'association', 'between', 'maternal', 'smoking', 'in', 'pregnancy', 'and', 'newborn', 'blood', 'dna', 'methylation', 'at', 'over', 'cpg', 'sites', 'cpgs', 'by', 'using', 'the', 'illumina', 'beadchip', 'over', 'cpgs', 'were', 'differentially', 'methylated', 'in', 'relation', 'to', 'maternal', 'smoking', 'at', 'genome', 'wide', 'statistical', 'significance', 'false', 'discovery', 'rate', 'including', 'cpgs', 'corresponding', 'to', 'genes', 'not', 'previously', 'related', 'to', 'smoking', 'and', 'methylation', 'in', 'either', 'newborns', 'or', 'adults', 'several', 'genes', 'are', 'relevant', 'to', 'diseases', 'that', 'can', 'be', 'caused', 'by', 'maternal', 'smoking', 'orofacial', 'clefts', 'and', 'asthma', 'or', 'adult', 'smoking', 'certain', 'cancers', 'number', 'of', 'differentially', 'methylated', 'cpgs', 'were', 'associated', 'with', 'gene', 'expression', 'we', 'observed', 'enrichment', 'in', 'pathways', 'and', 'processes', 'critical', 'to', 'development', 'in', 'older', 'children', 'cohorts', 'of', 'cpgs', 'gave', 'at', 'least', 'nominal', 'levels', 'of', 'significance', 'far', 'more', 'than', 'expected', 'by', 'chance', 'value', 'results', 'were', 'robust', 'to', 'different', 'normalization', 'methods', 'used', 'across', 'studies', 'and', 'cell', 'type', 'adjustment', 'in', 'this', 'large', 'scale', 'meta', 'analysis', 'of', 'methylation', 'data', 'we', 'identified', 'numerous', 'loci', 'involved', 'in', 'response', 'to', 'maternal', 'smoking', 'in', 'pregnancy', 'with', 'persistence', 'into', 'later', 'childhood', 'and', 'provide', 'insights', 'into', 'mechanisms', 'underlying', 'effects', 'of', 'this', 'important', 'exposure'], ['in', 'vincent', 'allfrey', 'and', 'colleagues', 'reported', 'the', 'identification', 'of', 'histone', 'acetylation', 'and', 'with', 'deep', 'insight', 'proposed', 'regulatory', 'role', 'for', 'this', 'protein', 'modification', 'in', 'transcription', 'regulation', 'subsequently', 'histone', 'hats', 'histone', 'deacetylases', 'hdacs', 'and', 'acetyl', 'lys', 'binding', 'proteins', 'were', 'identified', 'as', 'transcription', 'regulators', 'thereby', 'providing', 'compelling', 'evidence', 'for', 'his', 'daring', 'hypothesis', 'during', 'the', 'past', 'years', 'reversible', 'protein', 'acetylation', 'and', 'its', 'modifying', 'enzymes', 'have', 'been', 'implicated', 'in', 'many', 'cellular', 'functions', 'beyond', 'transcription', 'regulation', 'here', 'we', 'review', 'the', 'progress', 'accomplished', 'during', 'the', 'past', 'years', 'and', 'discuss', 'the', 'future', 'of', 'protein', 'acetylation'], ['methyladenosine', 'is', 'the', 'most', 'prevalent', 'and', 'internal', 'modification', 'that', 'occurs', 'in', 'the', 'messenger', 'rnas', 'mrna', 'of', 'most', 'eukaryotes', 'although', 'its', 'functional', 'relevance', 'remained', 'mystery', 'for', 'decades', 'this', 'modification', 'is', 'installed', 'by', 'the', 'methylation', 'writers', 'and', 'can', 'be', 'reversed', 'by', 'demethylases', 'that', 'serve', 'as', 'erasers', 'in', 'this', 'review', 'we', 'mainly', 'summarize', 'recent', 'progress', 'in', 'the', 'study', 'of', 'the', 'mrna', 'methylation', 'machineries', 'across', 'eukaryotes', 'and', 'discuss', 'their', 'newly', 'uncovered', 'biological', 'functions', 'the', 'broad', 'roles', 'of', 'in', 'regulating', 'cell', 'fates', 'and', 'embryonic', 'development', 'highlight', 'the', 'existence', 'of', 'another', 'layer', 'of', 'epigenetic', 'regulation', 'at', 'the', 'rna', 'level', 'where', 'mrna', 'is', 'subjected', 'to', 'chemical', 'modifications', 'that', 'affect', 'protein', 'expression'], ['the', 'bromo', 'and', 'extra', 'terminal', 'bet', 'proteins', 'brd', 'brd', 'and', 'brd', 'play', 'important', 'roles', 'in', 'transcriptional', 'regulation', 'epigenetics', 'and', 'cancer', 'and', 'are', 'the', 'targets', 'of', 'pan', 'bet', 'selective', 'bromodomain', 'inhibitor', 'jq', 'however', 'the', 'lack', 'of', 'intra', 'bet', 'selectivity', 'limits', 'the', 'scope', 'of', 'current', 'inhibitors', 'as', 'probes', 'for', 'target', 'validation', 'and', 'could', 'lead', 'to', 'unwanted', 'side', 'effects', 'or', 'toxicity', 'in', 'therapeutic', 'setting', 'we', 'designed', 'proteolysis', 'targeted', 'chimeras', 'protacs', 'that', 'tether', 'jq', 'to', 'ligand', 'for', 'the', 'ubiquitin', 'ligase', 'vhl', 'aimed', 'at', 'triggering', 'the', 'intracellular', 'destruction', 'of', 'bet', 'proteins', 'compound', 'mz', 'potently', 'and', 'rapidly', 'induces', 'reversible', 'long', 'lasting', 'and', 'unexpectedly', 'selective', 'removal', 'of', 'brd', 'over', 'brd', 'and', 'brd', 'the', 'activity', 'of', 'mz', 'is', 'dependent', 'on', 'binding', 'to', 'vhl', 'but', 'is', 'achieved', 'at', 'sufficiently', 'low', 'concentration', 'not', 'to', 'induce', 'stabilization', 'of', 'hif', 'alpha', 'gene', 'expression', 'profiles', 'of', 'selected', 'cancer', 'related', 'genes', 'responsive', 'to', 'jq', 'reveal', 'distinct', 'and', 'more', 'limited', 'transcriptional', 'responses', 'induced', 'by', 'mz', 'consistent', 'with', 'selective', 'suppression', 'of', 'brd', 'our', 'discovery', 'opens', 'up', 'new', 'opportunities', 'to', 'elucidate', 'the', 'cellular', 'phenotypes', 'and', 'therapeutic', 'implications', 'associated', 'with', 'selective', 'targeting', 'of', 'brd'], ['we', 'define', 'the', 'chromatin', 'accessibility', 'and', 'transcriptional', 'landscapes', 'in', 'human', 'primary', 'blood', 'cell', 'types', 'that', 'span', 'the', 'hematopoietic', 'hierarchy', 'exploiting', 'the', 'finding', 'that', 'the', 'enhancer', 'landscape', 'better', 'reflects', 'cell', 'identity', 'than', 'mrna', 'levels', 'we', 'enable', 'enhancer', 'cytometry', 'for', 'enumeration', 'of', 'pure', 'cell', 'types', 'from', 'complex', 'populations', 'we', 'identify', 'regulators', 'governing', 'hematopoietic', 'differentiation', 'and', 'further', 'show', 'the', 'lineage', 'ontogeny', 'of', 'genetic', 'elements', 'linked', 'to', 'diverse', 'human', 'diseases', 'in', 'acute', 'myeloid', 'leukemia', 'aml', 'chromatin', 'accessibility', 'uncovers', 'unique', 'regulatory', 'evolution', 'in', 'cancer', 'cells', 'with', 'progressively', 'increasing', 'mutation', 'burden', 'single', 'aml', 'cells', 'exhibit', 'distinctive', 'mixed', 'regulome', 'profiles', 'corresponding', 'to', 'disparate', 'developmental', 'stages', 'method', 'to', 'account', 'for', 'this', 'regulatory', 'heterogeneity', 'identified', 'cancer', 'specific', 'deviations', 'and', 'implicated', 'hox', 'factors', 'as', 'key', 'regulators', 'of', 'preleukemic', 'hematopoietic', 'stem', 'cell', 'characteristics', 'thus', 'regulome', 'dynamics', 'can', 'provide', 'diverse', 'insights', 'into', 'hematopoietic', 'development', 'and', 'disease'], ['treatment', 'of', 'cancer', 'has', 'been', 'revolutionized', 'by', 'immune', 'checkpoint', 'blockade', 'therapies', 'despite', 'the', 'high', 'rate', 'of', 'response', 'in', 'advanced', 'melanoma', 'the', 'majority', 'of', 'patients', 'succumb', 'to', 'disease', 'to', 'identify', 'factors', 'associated', 'with', 'success', 'or', 'failure', 'of', 'checkpoint', 'therapy', 'we', 'profiled', 'transcriptomes', 'of', 'individual', 'immune', 'cells', 'from', 'tumor', 'samples', 'of', 'melanoma', 'patients', 'treated', 'with', 'checkpoint', 'inhibitors', 'two', 'distinct', 'states', 'of', 'cd', 'cells', 'were', 'defined', 'by', 'clustering', 'and', 'associated', 'with', 'patient', 'tumor', 'regression', 'or', 'progression', 'single', 'transcription', 'factor', 'tcf', 'was', 'visualized', 'within', 'cd', 'cells', 'in', 'fixed', 'tumor', 'samples', 'and', 'predicted', 'positive', 'clinical', 'outcome', 'in', 'an', 'independent', 'cohort', 'of', 'checkpoint', 'treated', 'patients', 'we', 'delineated', 'the', 'epigenetic', 'landscape', 'and', 'clonality', 'of', 'these', 'cell', 'states', 'and', 'demonstrated', 'enhanced', 'antitumor', 'immunity', 'by', 'targeting', 'novel', 'combinations', 'of', 'factors', 'in', 'exhausted', 'cells', 'our', 'study', 'of', 'immune', 'cell', 'transcriptomes', 'from', 'tumors', 'demonstrates', 'strategy', 'for', 'identifying', 'predictors', 'mechanisms', 'and', 'targets', 'for', 'enhancing', 'checkpoint', 'immunotherapy'], ['the', 'extent', 'of', 'tumor', 'heterogeneity', 'is', 'an', 'emerging', 'theme', 'that', 'researchers', 'are', 'only', 'beginning', 'to', 'understand', 'how', 'genetic', 'and', 'epigenetic', 'heterogeneity', 'affects', 'tumor', 'evolution', 'and', 'clinical', 'progression', 'is', 'unknown', 'the', 'precise', 'nature', 'of', 'the', 'environmental', 'factors', 'that', 'influence', 'this', 'heterogeneity', 'is', 'also', 'yet', 'to', 'be', 'characterized', 'nature', 'medicine', 'nature', 'biotechnology', 'and', 'the', 'volkswagen', 'foundation', 'organized', 'meeting', 'focused', 'on', 'identifying', 'the', 'obstacles', 'that', 'need', 'to', 'be', 'overcome', 'to', 'advance', 'translational', 'research', 'in', 'and', 'tumor', 'heterogeneity', 'once', 'these', 'key', 'questions', 'were', 'established', 'the', 'attendees', 'devised', 'potential', 'solutions', 'their', 'ideas', 'are', 'presented', 'here'], ['single', 'stranded', 'guanine', 'rich', 'dna', 'sequences', 'can', 'fold', 'into', 'four', 'stranded', 'dna', 'structures', 'called', 'quadruplexes', 'that', 'arise', 'from', 'the', 'self', 'stacking', 'of', 'two', 'or', 'more', 'guanine', 'quartets', 'there', 'has', 'been', 'considerable', 'recent', 'progress', 'in', 'the', 'detection', 'and', 'mapping', 'of', 'structures', 'in', 'the', 'human', 'genome', 'and', 'in', 'biologically', 'relevant', 'contexts', 'these', 'advancements', 'many', 'of', 'which', 'align', 'with', 'predictions', 'made', 'previously', 'in', 'computational', 'studies', 'provide', 'important', 'new', 'insights', 'into', 'the', 'functions', 'of', 'structures', 'in', 'for', 'example', 'the', 'regulation', 'of', 'transcription', 'and', 'genome', 'stability', 'and', 'uncover', 'their', 'potential', 'relevance', 'for', 'cancer', 'therapy'], ['humans', 'show', 'strong', 'sex', 'differences', 'in', 'immunity', 'to', 'infection', 'and', 'autoimmunity', 'suggesting', 'sex', 'hormones', 'modulate', 'immune', 'responses', 'indeed', 'receptors', 'for', 'estrogens', 'ers', 'regulate', 'cells', 'and', 'pathways', 'in', 'the', 'innate', 'and', 'adaptive', 'immune', 'system', 'as', 'well', 'as', 'immune', 'cell', 'development', 'ers', 'are', 'ligand', 'dependent', 'transcription', 'factors', 'that', 'mediate', 'long', 'range', 'chromatin', 'interactions', 'and', 'form', 'complexes', 'at', 'gene', 'regulatory', 'elements', 'thus', 'promoting', 'epigenetic', 'changes', 'and', 'transcription', 'ers', 'also', 'participate', 'in', 'membrane', 'initiated', 'steroid', 'signaling', 'to', 'generate', 'rapid', 'responses', 'estradiol', 'and', 'er', 'activity', 'show', 'profound', 'dose', 'and', 'context', 'dependent', 'effects', 'on', 'innate', 'immune', 'signaling', 'pathways', 'and', 'myeloid', 'cell', 'development', 'while', 'estradiol', 'most', 'often', 'promotes', 'the', 'production', 'of', 'type', 'interferon', 'innate', 'pathways', 'leading', 'to', 'pro', 'inflammatory', 'cytokine', 'production', 'may', 'be', 'enhanced', 'or', 'dampened', 'by', 'er', 'activity', 'regulation', 'of', 'innate', 'immune', 'cells', 'and', 'signaling', 'by', 'ers', 'may', 'contribute', 'to', 'the', 'reported', 'sex', 'differences', 'in', 'innate', 'immune', 'pathways', 'here', 'we', 'review', 'the', 'recent', 'literature', 'and', 'highlight', 'several', 'molecular', 'mechanisms', 'by', 'which', 'ers', 'regulate', 'the', 'development', 'or', 'functional', 'responses', 'of', 'innate', 'immune', 'cells', 'elsevier', 'inc', 'all', 'rights', 'reserved'], ['epigenetic', 'studies', 'relied', 'so', 'far', 'on', 'correlations', 'between', 'epigenetic', 'marks', 'and', 'gene', 'expression', 'pattern', 'technologies', 'developed', 'for', 'epigenome', 'editing', 'now', 'enable', 'direct', 'study', 'of', 'functional', 'relevance', 'of', 'precise', 'epigenetic', 'modifications', 'and', 'gene', 'regulation', 'the', 'reversible', 'nature', 'of', 'epigenetic', 'modifications', 'including', 'dna', 'methylation', 'has', 'been', 'already', 'exploited', 'in', 'cancer', 'therapy', 'for', 'remodeling', 'the', 'aberrant', 'epigenetic', 'landscape', 'however', 'this', 'was', 'achieved', 'non', 'selectively', 'using', 'epigenetic', 'inhibitors', 'epigenetic', 'editing', 'at', 'specific', 'loci', 'represents', 'novel', 'approach', 'that', 'might', 'selectively', 'and', 'heritably', 'alter', 'gene', 'expression', 'here', 'we', 'developed', 'crispr', 'cas', 'based', 'tool', 'for', 'specific', 'dna', 'methylation', 'consisting', 'of', 'deactivated', 'cas', 'dcas', 'nuclease', 'and', 'catalytic', 'domain', 'of', 'the', 'dna', 'dnmt', 'targeted', 'by', 'co', 'expression', 'of', 'guide', 'rna', 'to', 'any', 'bp', 'dna', 'sequence', 'followed', 'by', 'the', 'ngg', 'trinucleotide', 'we', 'demonstrated', 'targeted', 'cpg', 'methylation', 'in', 'similar', 'to', 'bp', 'wide', 'region', 'by', 'the', 'fusion', 'protein', 'we', 'also', 'showed', 'that', 'multiple', 'guide', 'rnas', 'could', 'target', 'the', 'dcas', 'dnmt', 'construct', 'to', 'multiple', 'adjacent', 'sites', 'which', 'enabled', 'methylation', 'of', 'larger', 'part', 'of', 'the', 'promoter', 'dna', 'methylation', 'activity', 'was', 'specific', 'for', 'the', 'targeted', 'region', 'and', 'heritable', 'across', 'mitotic', 'divisions', 'finally', 'we', 'demonstrated', 'that', 'directed', 'dna', 'methylation', 'of', 'wider', 'promoter', 'region', 'of', 'the', 'target', 'loci', 'il', 'st', 'and', 'bach', 'decreased', 'their', 'expression'], ['the', 'dna', 'dnmt', 'family', 'comprises', 'conserved', 'set', 'of', 'dna', 'modifying', 'enzymes', 'that', 'have', 'central', 'role', 'in', 'epigenetic', 'gene', 'regulation', 'recent', 'studies', 'have', 'shown', 'that', 'the', 'functions', 'of', 'the', 'canonical', 'dnmt', 'enzymes', 'dnmt', 'dnmt', 'and', 'dnmt', 'go', 'beyond', 'their', 'traditional', 'roles', 'of', 'establishing', 'and', 'maintaining', 'dna', 'methylation', 'patterns', 'this', 'review', 'analyses', 'how', 'molecular', 'interactions', 'and', 'changes', 'in', 'gene', 'copy', 'numbers', 'modulate', 'the', 'activity', 'of', 'dnmts', 'in', 'diverse', 'gene', 'regulatory', 'functions', 'including', 'transcriptional', 'silencing', 'transcriptional', 'activation', 'and', 'post', 'transcriptional', 'regulation', 'by', 'dnmt', 'dependent', 'trna', 'methylation', 'this', 'mechanistic', 'diversity', 'enables', 'the', 'dnmt', 'family', 'to', 'function', 'as', 'versatile', 'toolkit', 'for', 'epigenetic', 'regulation'], ['estimates', 'of', 'biological', 'age', 'based', 'on', 'dna', 'methylation', 'patterns', 'often', 'referred', 'to', 'as', 'epigenetic', 'age', 'dnam', 'age', 'have', 'been', 'shown', 'to', 'be', 'robust', 'biomarkers', 'of', 'age', 'in', 'humans', 'we', 'previously', 'demonstrated', 'that', 'independent', 'of', 'chronological', 'age', 'epigenetic', 'age', 'assessed', 'in', 'blood', 'predicted', 'all', 'cause', 'mortality', 'in', 'four', 'human', 'cohorts', 'here', 'we', 'expanded', 'our', 'original', 'observation', 'to', 'different', 'cohorts', 'for', 'total', 'sample', 'size', 'of', 'individuals', 'including', 'three', 'racial', 'ethnic', 'groups', 'in', 'addition', 'we', 'examined', 'whether', 'incorporating', 'information', 'on', 'blood', 'cell', 'composition', 'into', 'the', 'epigenetic', 'age', 'metrics', 'improves', 'their', 'predictive', 'power', 'for', 'mortality', 'all', 'considered', 'measures', 'of', 'epigenetic', 'age', 'acceleration', 'were', 'predictive', 'of', 'mortality', 'independent', 'of', 'chronological', 'age', 'even', 'after', 'adjusting', 'for', 'additional', 'risk', 'factors', 'and', 'within', 'the', 'racial', 'ethnic', 'groups', 'that', 'we', 'examined', 'non', 'hispanic', 'whites', 'hispanics', 'african', 'americans', 'epigenetic', 'age', 'estimates', 'that', 'incorporated', 'information', 'on', 'blood', 'cell', 'composition', 'led', 'to', 'the', 'smallest', 'values', 'for', 'time', 'to', 'death', 'overall', 'this', 'study', 'strengthens', 'the', 'evidence', 'that', 'epigenetic', 'age', 'predicts', 'all', 'cause', 'mortality', 'above', 'and', 'beyond', 'chronological', 'age', 'and', 'traditional', 'risk', 'factors', 'and', 'demonstrates', 'that', 'epigenetic', 'age', 'estimates', 'that', 'incorporate', 'information', 'on', 'blood', 'cell', 'counts', 'lead', 'to', 'highly', 'significant', 'associations', 'with', 'all', 'cause', 'mortality'], ['childhood', 'trauma', 'confers', 'higher', 'risk', 'of', 'adulthood', 'physical', 'and', 'mental', 'illness', 'however', 'the', 'biological', 'mechanism', 'mediating', 'this', 'association', 'remains', 'largely', 'unknown', 'recent', 'research', 'has', 'suggested', 'dysregulation', 'of', 'the', 'immune', 'system', 'as', 'possible', 'biological', 'mediator', 'the', 'present', 'paper', 'conducted', 'meta', 'analysis', 'to', 'establish', 'whether', 'early', 'life', 'adversity', 'contributes', 'to', 'potentially', 'pathogenic', 'pro', 'inflammatory', 'phenotypes', 'in', 'adult', 'individuals', 'systematic', 'search', 'of', 'pubmed', 'psycinfo', 'embase', 'scopus', 'and', 'medline', 'identified', 'articles', 'for', 'the', 'meta', 'analysis', 'including', 'studies', 'encompassing', 'sample', 'of', 'individuals', 'for', 'reactive', 'protein', 'crp', 'studies', 'including', 'individuals', 'for', 'interleukin', 'il', 'and', 'studies', 'including', 'individuals', 'for', 'tumour', 'necrosis', 'factor', 'alpha', 'tnf', 'alpha', 'random', 'effects', 'meta', 'analysis', 'showed', 'that', 'individuals', 'exposed', 'to', 'childhood', 'trauma', 'had', 'significantly', 'elevated', 'baseline', 'peripheral', 'levels', 'of', 'crp', 'fisher', 'confidence', 'interval', 'ci', 'il', 'ci', 'and', 'tnf', 'alpha', 'ci', 'subgroup', 'analyses', 'for', 'specific', 'types', 'of', 'trauma', 'sexual', 'physical', 'or', 'emotional', 'abuse', 'revealed', 'that', 'these', 'impact', 'differentially', 'the', 'single', 'inflammatory', 'markers', 'moreover', 'meta', 'regression', 'revealed', 'greater', 'effect', 'sizes', 'in', 'clinical', 'samples', 'for', 'the', 'association', 'between', 'childhood', 'trauma', 'and', 'crp', 'but', 'not', 'for', 'il', 'or', 'tnf', 'alpha', 'age', 'body', 'mass', 'index', 'bmi', 'and', 'gender', 'had', 'no', 'moderating', 'effects', 'the', 'analysis', 'demonstrates', 'that', 'childhood', 'trauma', 'contributes', 'to', 'pro', 'inflammatory', 'state', 'in', 'adulthood', 'with', 'specific', 'inflammatory', 'profiles', 'depending', 'on', 'the', 'specific', 'type', 'of', 'trauma'], ['the', 'heart', 'must', 'continuously', 'pump', 'blood', 'to', 'supply', 'the', 'body', 'with', 'oxygen', 'and', 'nutrients', 'to', 'maintain', 'the', 'high', 'energy', 'consumption', 'required', 'by', 'this', 'role', 'the', 'heart', 'is', 'equipped', 'with', 'multiple', 'complex', 'biological', 'systems', 'that', 'allow', 'adaptation', 'to', 'changes', 'of', 'systemic', 'demand', 'the', 'processes', 'of', 'growth', 'hypertrophy', 'angiogenesis', 'and', 'metabolic', 'plasticity', 'are', 'critically', 'involved', 'in', 'maintenance', 'of', 'cardiac', 'homeostasis', 'cardiac', 'hypertrophy', 'is', 'classified', 'as', 'physiological', 'when', 'it', 'is', 'associated', 'with', 'normal', 'cardiac', 'function', 'or', 'as', 'pathological', 'when', 'associated', 'with', 'cardiac', 'dysfunction', 'physiological', 'hypertrophy', 'of', 'the', 'heart', 'occurs', 'in', 'response', 'to', 'normal', 'growth', 'of', 'children', 'or', 'during', 'pregnancy', 'as', 'well', 'as', 'in', 'athletes', 'in', 'contrast', 'pathological', 'hypertrophy', 'is', 'induced', 'by', 'factors', 'such', 'as', 'prolonged', 'and', 'abnormal', 'hemodynamic', 'stress', 'due', 'to', 'hypertension', 'myocardial', 'infarction', 'etc', 'pathological', 'hypertrophy', 'is', 'associated', 'with', 'fibrosis', 'capillary', 'rarefaction', 'increased', 'production', 'of', 'pro', 'inflammatory', 'cytokines', 'and', 'cellular', 'dysfunction', 'impairment', 'of', 'signaling', 'suppression', 'of', 'autophagy', 'and', 'abnormal', 'cardiomyocyte', 'non', 'cardiomyocyte', 'interactions', 'as', 'well', 'as', 'undesirable', 'epigenetic', 'changes', 'with', 'these', 'complex', 'responses', 'leading', 'to', 'maladaptive', 'cardiac', 'remodeling', 'and', 'heart', 'failure', 'this', 'review', 'describes', 'the', 'key', 'molecules', 'and', 'cellular', 'responses', 'involved', 'in', 'physiological', 'pathological', 'cardiac', 'hypertrophy', 'elsevier', 'ltd', 'all', 'rights', 'reserved'], ['the', 'process', 'of', 'aging', 'results', 'in', 'host', 'of', 'changes', 'at', 'the', 'cellular', 'and', 'molecular', 'levels', 'which', 'include', 'senescence', 'telomere', 'shortening', 'and', 'changes', 'in', 'gene', 'expression', 'epigenetic', 'patterns', 'also', 'change', 'over', 'the', 'lifespan', 'suggesting', 'that', 'epigenetic', 'changes', 'may', 'constitute', 'an', 'important', 'component', 'of', 'the', 'aging', 'process', 'the', 'epigenetic', 'mark', 'that', 'has', 'been', 'most', 'highly', 'studied', 'is', 'dna', 'methylation', 'the', 'presence', 'of', 'methyl', 'groups', 'at', 'cpg', 'dinucleotides', 'these', 'dinucleotides', 'are', 'often', 'located', 'near', 'gene', 'promoters', 'and', 'associate', 'with', 'gene', 'expression', 'levels', 'early', 'studies', 'indicated', 'that', 'global', 'levels', 'of', 'dna', 'methylation', 'increase', 'over', 'the', 'first', 'few', 'years', 'of', 'life', 'and', 'then', 'decrease', 'beginning', 'in', 'late', 'adulthood', 'recently', 'with', 'the', 'advent', 'of', 'microarray', 'and', 'next', 'generation', 'sequencing', 'technologies', 'increases', 'in', 'variability', 'of', 'dna', 'methylation', 'with', 'age', 'have', 'been', 'observed', 'and', 'number', 'of', 'site', 'specific', 'patterns', 'have', 'been', 'identified', 'it', 'has', 'also', 'been', 'shown', 'that', 'certain', 'cpg', 'sites', 'are', 'highly', 'associated', 'with', 'age', 'to', 'the', 'extent', 'that', 'prediction', 'models', 'using', 'small', 'number', 'of', 'these', 'sites', 'can', 'accurately', 'predict', 'the', 'chronological', 'age', 'of', 'the', 'donor', 'together', 'these', 'observations', 'point', 'to', 'the', 'existence', 'of', 'two', 'phenomena', 'that', 'both', 'contribute', 'to', 'age', 'related', 'dna', 'methylation', 'changes', 'epigenetic', 'drift', 'and', 'the', 'epigenetic', 'clock', 'in', 'this', 'review', 'we', 'focus', 'on', 'healthy', 'human', 'aging', 'throughout', 'the', 'lifetime', 'and', 'discuss', 'the', 'dynamics', 'of', 'dna', 'methylation', 'as', 'well', 'as', 'how', 'interactions', 'between', 'the', 'genome', 'environment', 'and', 'the', 'epigenome', 'influence', 'aging', 'rates', 'we', 'also', 'discuss', 'the', 'impact', 'of', 'determining', 'epigenetic', 'age', 'for', 'human', 'health', 'and', 'outline', 'some', 'important', 'caveats', 'to', 'existing', 'and', 'future', 'studies'], ['the', 'ability', 'to', 'rapidly', 'restore', 'the', 'integrity', 'of', 'broken', 'skin', 'barrier', 'is', 'critical', 'and', 'is', 'the', 'ultimate', 'goal', 'of', 'therapies', 'for', 'hard', 'to', 'heal', 'ulcers', 'unfortunately', 'effective', 'treatments', 'to', 'enhance', 'healing', 'and', 'reduce', 'scarring', 'are', 'still', 'lacking', 'deeper', 'understanding', 'of', 'the', 'physiology', 'of', 'normal', 'repair', 'and', 'of', 'the', 'pathology', 'of', 'delayed', 'healing', 'is', 'prerequisite', 'for', 'the', 'development', 'of', 'more', 'effective', 'therapeutic', 'interventions', 'transition', 'from', 'the', 'inflammatory', 'to', 'the', 'proliferative', 'phase', 'is', 'key', 'step', 'during', 'healing', 'and', 'accumulating', 'evidence', 'associates', 'compromised', 'transition', 'with', 'wound', 'healing', 'disorders', 'thus', 'targeting', 'factors', 'that', 'impact', 'this', 'phase', 'transition', 'may', 'offer', 'rationale', 'for', 'therapeutic', 'development', 'this', 'review', 'summarizes', 'mechanisms', 'regulating', 'the', 'inflammation', 'proliferation', 'transition', 'at', 'cellular', 'and', 'molecular', 'levels', 'we', 'propose', 'that', 'identification', 'of', 'such', 'mechanisms', 'will', 'reveal', 'promising', 'targets', 'for', 'development', 'of', 'more', 'effective', 'therapies'], ['adult', 'somatic', 'stem', 'cells', 'in', 'various', 'organs', 'maintain', 'homeostatic', 'tissue', 'regeneration', 'and', 'enhance', 'plasticity', 'since', 'its', 'initial', 'discovery', 'five', 'decades', 'ago', 'investigations', 'of', 'adult', 'neurogenesis', 'and', 'neural', 'stem', 'cells', 'have', 'led', 'to', 'an', 'established', 'and', 'expanding', 'field', 'that', 'has', 'significantly', 'influenced', 'many', 'facets', 'of', 'neuroscience', 'developmental', 'biology', 'and', 'regenerative', 'medicine', 'here', 'we', 'review', 'recent', 'progress', 'and', 'focus', 'on', 'questions', 'related', 'to', 'adult', 'mammalian', 'neural', 'stem', 'cells', 'that', 'also', 'apply', 'to', 'other', 'somatic', 'stem', 'cells', 'we', 'further', 'discuss', 'emerging', 'topics', 'that', 'are', 'guiding', 'the', 'field', 'toward', 'better', 'understanding', 'adult', 'neural', 'stem', 'cells', 'and', 'ultimately', 'applying', 'these', 'principles', 'to', 'improve', 'human', 'health'], ['the', 'dogma', 'that', 'adaptive', 'immunity', 'is', 'the', 'only', 'arm', 'of', 'the', 'immune', 'response', 'with', 'memory', 'capacity', 'has', 'been', 'recently', 'challenged', 'by', 'several', 'studies', 'demonstrating', 'evidence', 'for', 'memory', 'like', 'innate', 'immune', 'training', 'however', 'the', 'underlying', 'mechanisms', 'and', 'location', 'for', 'generating', 'such', 'innate', 'memory', 'responses', 'in', 'vivo', 'remain', 'unknown', 'here', 'we', 'show', 'that', 'access', 'of', 'bacillus', 'calmette', 'guerin', 'bcg', 'to', 'the', 'bone', 'marrow', 'bm', 'changes', 'the', 'transcriptional', 'landscape', 'of', 'hematopoietic', 'stem', 'cells', 'hscs', 'and', 'multipotent', 'progenitors', 'mpps', 'leading', 'to', 'local', 'cell', 'expansion', 'and', 'enhanced', 'myelopoiesis', 'at', 'the', 'expense', 'of', 'lymphopoiesis', 'importantly', 'bcg', 'educated', 'hscs', 'generate', 'epigenetically', 'modified', 'macrophages', 'that', 'provide', 'significantly', 'better', 'protection', 'against', 'virulent', 'tuberculosis', 'infection', 'than', 'naive', 'macrophages', 'by', 'using', 'parabiotic', 'and', 'chimeric', 'mice', 'as', 'well', 'as', 'adoptive', 'transfer', 'approaches', 'we', 'demonstrate', 'that', 'training', 'of', 'the', 'monocyte', 'macrophage', 'lineage', 'via', 'bcg', 'induced', 'hsc', 'reprogramming', 'is', 'sustainable', 'in', 'vivo', 'our', 'results', 'indicate', 'that', 'targeting', 'the', 'hsc', 'compartment', 'provides', 'novel', 'approach', 'for', 'vaccine', 'development'], ['experimental', 'modelling', 'of', 'human', 'disorders', 'enables', 'the', 'definition', 'of', 'the', 'cellular', 'and', 'molecular', 'mechanisms', 'underlying', 'diseases', 'and', 'the', 'development', 'of', 'therapies', 'for', 'treating', 'them', 'the', 'availability', 'of', 'human', 'pluripotent', 'stem', 'cells', 'pscs', 'which', 'are', 'capable', 'of', 'self', 'renewal', 'and', 'have', 'the', 'potential', 'to', 'differentiate', 'into', 'virtually', 'any', 'cell', 'type', 'can', 'now', 'help', 'to', 'overcome', 'the', 'limitations', 'of', 'animal', 'models', 'for', 'certain', 'disorders', 'the', 'ability', 'to', 'model', 'human', 'diseases', 'using', 'cultured', 'pscs', 'has', 'revolutionized', 'the', 'ways', 'in', 'which', 'we', 'study', 'monogenic', 'complex', 'and', 'epigenetic', 'disorders', 'as', 'well', 'as', 'early', 'and', 'late', 'onset', 'diseases', 'several', 'strategies', 'are', 'used', 'to', 'generate', 'such', 'disease', 'models', 'using', 'either', 'embryonic', 'stem', 'cells', 'es', 'cells', 'or', 'patient', 'specific', 'induced', 'pscs', 'ipscs', 'creating', 'new', 'possibilities', 'for', 'the', 'establishment', 'of', 'models', 'and', 'their', 'use', 'in', 'drug', 'screening'], ['resetting', 'of', 'the', 'epigenome', 'in', 'human', 'primordial', 'germ', 'cells', 'hpgcs', 'is', 'critical', 'for', 'development', 'we', 'show', 'that', 'the', 'transcriptional', 'program', 'of', 'hpgcs', 'is', 'distinct', 'from', 'that', 'in', 'mice', 'with', 'co', 'expression', 'of', 'somatic', 'specifiers', 'and', 'naive', 'pluripotency', 'genes', 'tfcp', 'and', 'klf', 'this', 'unique', 'gene', 'regulatory', 'network', 'established', 'by', 'sox', 'and', 'blimp', 'drives', 'comprehensive', 'germline', 'dna', 'demethylation', 'by', 'repressing', 'dna', 'methylation', 'pathways', 'and', 'activating', 'tet', 'mediated', 'base', 'resolution', 'methylome', 'analysis', 'reveals', 'progressive', 'dna', 'demethylation', 'to', 'basal', 'levels', 'in', 'week', 'in', 'vivo', 'hpgcs', 'concurrently', 'hpgcs', 'undergo', 'chromatin', 'reorganization', 'reactivation', 'and', 'imprint', 'erasure', 'despite', 'global', 'hypomethylation', 'evolutionarily', 'young', 'and', 'potentially', 'hazardous', 'retroelements', 'like', 'sva', 'remain', 'methylated', 'remarkably', 'some', 'loci', 'associated', 'with', 'metabolic', 'and', 'neurological', 'disorders', 'are', 'also', 'resistant', 'to', 'dna', 'demethylation', 'revealing', 'potential', 'for', 'epigenetic', 'inheritance', 'that', 'may', 'have', 'phenotypic', 'consequences', 'we', 'provide', 'comprehensive', 'insight', 'on', 'early', 'human', 'germline', 'transcriptional', 'network', 'and', 'epigenetic', 'reprogramming', 'that', 'subsequently', 'impacts', 'human', 'development', 'and', 'disease'], ['the', 'cbioportal', 'for', 'cancer', 'genomics', 'http', 'cbioportal', 'org', 'provides', 'web', 'resource', 'for', 'exploring', 'visualizing', 'and', 'analyzing', 'cancer', 'genomics', 'data', 'the', 'portal', 'reduces', 'molecular', 'profiling', 'data', 'from', 'cancer', 'tissues', 'and', 'cell', 'lines', 'into', 'readily', 'understandable', 'genetic', 'epigenetic', 'gene', 'expression', 'and', 'proteomic', 'events', 'the', 'query', 'interface', 'combined', 'with', 'customized', 'data', 'storage', 'enables', 'researchers', 'to', 'interactively', 'explore', 'genetic', 'alterations', 'across', 'samples', 'genes', 'and', 'pathways', 'and', 'when', 'available', 'in', 'the', 'underlying', 'data', 'to', 'link', 'these', 'to', 'clinical', 'outcomes', 'the', 'portal', 'provides', 'graphical', 'summaries', 'of', 'gene', 'level', 'data', 'from', 'multiple', 'platforms', 'network', 'visualization', 'and', 'analysis', 'survival', 'analysis', 'patient', 'centric', 'queries', 'and', 'software', 'programmatic', 'access', 'the', 'intuitive', 'web', 'interface', 'of', 'the', 'portal', 'makes', 'complex', 'cancer', 'genomics', 'profiles', 'accessible', 'to', 'researchers', 'and', 'clinicians', 'without', 'requiring', 'bioinformatics', 'expertise', 'thus', 'facilitating', 'biological', 'discoveries', 'here', 'we', 'provide', 'practical', 'guide', 'to', 'the', 'analysis', 'and', 'visualization', 'features', 'of', 'the', 'cbioportal', 'for', 'cancer', 'genomics'], ['aging', 'is', 'characterized', 'by', 'progressive', 'loss', 'of', 'physiological', 'integrity', 'leading', 'to', 'impaired', 'function', 'and', 'increased', 'vulnerability', 'to', 'death', 'this', 'deterioration', 'is', 'the', 'primary', 'risk', 'factor', 'for', 'major', 'human', 'pathologies', 'including', 'cancer', 'diabetes', 'cardiovascular', 'disorders', 'and', 'diseases', 'aging', 'research', 'has', 'experienced', 'an', 'unprecedented', 'advance', 'over', 'recent', 'years', 'particularly', 'with', 'the', 'discovery', 'that', 'the', 'rate', 'of', 'aging', 'is', 'controlled', 'at', 'least', 'to', 'some', 'extent', 'by', 'genetic', 'pathways', 'and', 'biochemical', 'processes', 'conserved', 'in', 'evolution', 'this', 'review', 'enumerates', 'nine', 'tentative', 'hallmarks', 'that', 'represent', 'common', 'denominators', 'of', 'aging', 'in', 'different', 'organisms', 'with', 'special', 'emphasis', 'on', 'mammalian', 'aging', 'these', 'hallmarks', 'are', 'genomic', 'instability', 'telomere', 'attrition', 'epigenetic', 'alterations', 'loss', 'of', 'proteostasis', 'deregulated', 'nutrient', 'sensing', 'mitochondrial', 'dysfunction', 'cellular', 'senescence', 'stem', 'cell', 'exhaustion', 'and', 'altered', 'intercellular', 'communication', 'major', 'challenge', 'is', 'to', 'dissect', 'the', 'between', 'the', 'candidate', 'hallmarks', 'and', 'their', 'relative', 'contributions', 'to', 'aging', 'with', 'the', 'final', 'goal', 'of', 'identifying', 'pharmaceutical', 'targets', 'to', 'improve', 'human', 'health', 'during', 'aging', 'with', 'minimal', 'side', 'effects'], ['dna', 'methylation', 'is', 'frequently', 'described', 'as', 'silencing', 'epigenetic', 'mark', 'and', 'indeed', 'this', 'function', 'of', 'methylcytosine', 'was', 'originally', 'proposed', 'in', 'the', 'now', 'thanks', 'to', 'improved', 'genome', 'scale', 'mapping', 'of', 'methylation', 'we', 'can', 'evaluate', 'dna', 'methylation', 'in', 'different', 'genomic', 'contexts', 'transcriptional', 'start', 'sites', 'with', 'or', 'without', 'cpg', 'islands', 'in', 'gene', 'bodies', 'at', 'regulatory', 'elements', 'and', 'at', 'repeat', 'sequences', 'the', 'emerging', 'picture', 'is', 'that', 'the', 'function', 'of', 'dna', 'methylation', 'seems', 'to', 'vary', 'with', 'context', 'and', 'the', 'relationship', 'between', 'dna', 'methylation', 'and', 'transcription', 'is', 'more', 'nuanced', 'than', 'we', 'realized', 'at', 'first', 'improving', 'our', 'understanding', 'of', 'the', 'functions', 'of', 'dna', 'methylation', 'is', 'necessary', 'for', 'interpreting', 'changes', 'in', 'this', 'mark', 'that', 'are', 'observed', 'in', 'diseases', 'such', 'as', 'cancer'], ['the', 'cancer', 'genome', 'atlas', 'tcga', 'research', 'network', 'has', 'profiled', 'and', 'analyzed', 'large', 'numbers', 'of', 'human', 'tumors', 'to', 'discover', 'molecular', 'aberrations', 'at', 'the', 'dna', 'rna', 'protein', 'and', 'epigenetic', 'levels', 'the', 'resulting', 'rich', 'data', 'provide', 'major', 'opportunity', 'to', 'develop', 'an', 'integrated', 'picture', 'of', 'commonalities', 'differences', 'and', 'emergent', 'themes', 'across', 'tumor', 'lineages', 'the', 'pan', 'cancer', 'initiative', 'compares', 'the', 'first', 'tumor', 'types', 'profiled', 'by', 'tcga', 'analysis', 'of', 'the', 'molecular', 'aberrations', 'and', 'their', 'functional', 'roles', 'across', 'tumor', 'types', 'will', 'teach', 'us', 'how', 'to', 'extend', 'therapies', 'effective', 'in', 'one', 'cancer', 'type', 'to', 'others', 'with', 'similar', 'genomic', 'profile'], ['the', 'central', 'dogma', 'of', 'gene', 'expression', 'is', 'that', 'dna', 'is', 'transcribed', 'into', 'messenger', 'rnas', 'which', 'in', 'turn', 'serve', 'as', 'the', 'template', 'for', 'protein', 'synthesis', 'the', 'discovery', 'of', 'extensive', 'transcription', 'of', 'large', 'rna', 'transcripts', 'that', 'do', 'not', 'code', 'for', 'proteins', 'termed', 'long', 'noncoding', 'rnas', 'lncrnas', 'provides', 'an', 'important', 'new', 'perspective', 'on', 'the', 'centrality', 'of', 'rna', 'in', 'gene', 'regulation', 'here', 'we', 'discuss', 'genome', 'scale', 'strategies', 'to', 'discover', 'and', 'characterize', 'lncrnas', 'an', 'emerging', 'theme', 'from', 'multiple', 'model', 'systems', 'is', 'that', 'lncrnas', 'form', 'extensive', 'networks', 'of', 'rnp', 'complexes', 'with', 'numerous', 'chromatin', 'regulators', 'and', 'then', 'target', 'these', 'enzymatic', 'activities', 'to', 'appropriate', 'locations', 'in', 'the', 'genome', 'consistent', 'with', 'this', 'notion', 'lncrnas', 'can', 'function', 'as', 'modular', 'scaffolds', 'to', 'specify', 'higher', 'order', 'organization', 'in', 'rnp', 'complexes', 'and', 'in', 'chromatin', 'states', 'the', 'importance', 'of', 'these', 'modes', 'of', 'regulation', 'is', 'underscored', 'by', 'the', 'newly', 'recognized', 'roles', 'of', 'long', 'rnas', 'for', 'proper', 'gene', 'control', 'across', 'all', 'kingdoms', 'of', 'life'], ['metastases', 'represent', 'the', 'end', 'products', 'of', 'multistep', 'cell', 'biological', 'process', 'termed', 'the', 'invasion', 'metastasis', 'cascade', 'which', 'involves', 'dissemination', 'of', 'cancer', 'cells', 'to', 'anatomically', 'distant', 'organ', 'sites', 'and', 'their', 'subsequent', 'adaptation', 'to', 'foreign', 'tissue', 'each', 'of', 'these', 'events', 'is', 'driven', 'by', 'the', 'acquisition', 'of', 'genetic', 'and', 'or', 'epigenetic', 'alterations', 'within', 'tumor', 'cells', 'and', 'the', 'co', 'option', 'of', 'nonneoplastic', 'stromal', 'cells', 'which', 'together', 'endow', 'incipient', 'metastatic', 'cells', 'with', 'traits', 'needed', 'to', 'generate', 'macroscopic', 'metastases', 'recent', 'advances', 'provide', 'provocative', 'insights', 'into', 'these', 'cell', 'biological', 'and', 'molecular', 'changes', 'which', 'have', 'implications', 'regarding', 'the', 'steps', 'of', 'the', 'invasion', 'metastasis', 'cascade', 'that', 'appear', 'amenable', 'to', 'therapeutic', 'targeting'], ['background', 'it', 'is', 'not', 'yet', 'known', 'whether', 'dna', 'methylation', 'levels', 'can', 'be', 'used', 'to', 'accurately', 'predict', 'age', 'across', 'broad', 'spectrum', 'of', 'human', 'tissues', 'and', 'cell', 'types', 'nor', 'whether', 'the', 'resulting', 'age', 'prediction', 'is', 'biologically', 'meaningful', 'measure', 'results', 'developed', 'multi', 'tissue', 'predictor', 'of', 'age', 'that', 'allows', 'one', 'to', 'estimate', 'the', 'dna', 'methylation', 'age', 'of', 'most', 'tissues', 'and', 'cell', 'types', 'the', 'predictor', 'which', 'is', 'freely', 'available', 'was', 'developed', 'using', 'samples', 'from', 'illumina', 'dna', 'methylation', 'array', 'datasets', 'encompassing', 'healthy', 'tissues', 'and', 'cell', 'types', 'found', 'that', 'dna', 'methylation', 'age', 'has', 'the', 'following', 'properties', 'first', 'it', 'is', 'close', 'to', 'zero', 'for', 'embryonic', 'and', 'induced', 'pluripotent', 'stem', 'cells', 'second', 'it', 'correlates', 'with', 'cell', 'passage', 'number', 'third', 'it', 'gives', 'rise', 'to', 'highly', 'heritable', 'measure', 'of', 'age', 'acceleration', 'and', 'fourth', 'it', 'is', 'applicable', 'to', 'chimpanzee', 'tissues', 'analysis', 'of', 'cancer', 'samples', 'from', 'datasets', 'showed', 'that', 'all', 'of', 'the', 'considered', 'cancer', 'types', 'exhibit', 'significant', 'age', 'acceleration', 'with', 'an', 'average', 'of', 'years', 'low', 'age', 'acceleration', 'of', 'cancer', 'tissue', 'is', 'associated', 'with', 'high', 'number', 'of', 'somatic', 'mutations', 'and', 'tp', 'mutations', 'while', 'mutations', 'in', 'steroid', 'receptors', 'greatly', 'accelerate', 'dna', 'methylation', 'age', 'in', 'breast', 'cancer', 'finally', 'characterize', 'the', 'cpg', 'sites', 'that', 'together', 'form', 'an', 'aging', 'clock', 'in', 'terms', 'of', 'chromatin', 'states', 'and', 'tissue', 'variance', 'conclusions', 'propose', 'that', 'dna', 'methylation', 'age', 'measures', 'the', 'cumulative', 'effect', 'of', 'an', 'epigenetic', 'maintenance', 'system', 'this', 'novel', 'epigenetic', 'clock', 'can', 'be', 'used', 'to', 'address', 'host', 'of', 'questions', 'in', 'developmental', 'biology', 'cancer', 'and', 'aging', 'research'], ['cytosine', 'dna', 'methylation', 'is', 'stable', 'epigenetic', 'mark', 'that', 'is', 'crucial', 'for', 'diverse', 'biological', 'processes', 'including', 'gene', 'and', 'transposon', 'silencing', 'imprinting', 'and', 'chromosome', 'inactivation', 'recent', 'findings', 'in', 'plants', 'and', 'animals', 'have', 'greatly', 'increased', 'our', 'understanding', 'of', 'the', 'pathways', 'used', 'to', 'accurately', 'target', 'maintain', 'and', 'modify', 'patterns', 'of', 'dna', 'methylation', 'and', 'have', 'revealed', 'unanticipated', 'mechanistic', 'similarities', 'between', 'these', 'organisms', 'key', 'roles', 'have', 'emerged', 'for', 'small', 'rnas', 'proteins', 'with', 'domains', 'that', 'bind', 'methylated', 'dna', 'and', 'dna', 'glycosylases', 'in', 'these', 'processes', 'drawing', 'on', 'insights', 'from', 'both', 'plants', 'and', 'animals', 'should', 'deepen', 'our', 'understanding', 'of', 'the', 'regulation', 'and', 'biological', 'significance', 'of', 'dna', 'methylation'], ['flexible', 'statistical', 'framework', 'is', 'developed', 'for', 'the', 'analysis', 'of', 'read', 'counts', 'from', 'rna', 'seq', 'gene', 'expression', 'studies', 'it', 'provides', 'the', 'ability', 'to', 'analyse', 'complex', 'experiments', 'involving', 'multiple', 'treatment', 'conditions', 'and', 'blocking', 'variables', 'while', 'still', 'taking', 'full', 'account', 'of', 'biological', 'variation', 'biological', 'variation', 'between', 'rna', 'samples', 'is', 'estimated', 'separately', 'from', 'the', 'technical', 'variation', 'associated', 'with', 'sequencing', 'technologies', 'novel', 'empirical', 'bayes', 'methods', 'allow', 'each', 'gene', 'to', 'have', 'its', 'own', 'specific', 'variability', 'even', 'when', 'there', 'are', 'relatively', 'few', 'biological', 'replicates', 'from', 'which', 'to', 'estimate', 'such', 'variability', 'the', 'pipeline', 'is', 'implemented', 'in', 'the', 'edger', 'package', 'of', 'the', 'bioconductor', 'project', 'case', 'study', 'analysis', 'of', 'carcinoma', 'data', 'demonstrates', 'the', 'ability', 'of', 'generalized', 'linear', 'model', 'methods', 'glms', 'to', 'detect', 'differential', 'expression', 'in', 'paired', 'design', 'and', 'even', 'to', 'detect', 'tumour', 'specific', 'expression', 'changes', 'the', 'case', 'study', 'demonstrates', 'the', 'need', 'to', 'allow', 'for', 'gene', 'specific', 'variability', 'rather', 'than', 'assuming', 'common', 'dispersion', 'across', 'genes', 'or', 'fixed', 'relationship', 'between', 'abundance', 'and', 'variability', 'genewise', 'dispersions', 'de', 'prioritize', 'genes', 'with', 'inconsistent', 'results', 'and', 'allow', 'the', 'main', 'analysis', 'to', 'focus', 'on', 'changes', 'that', 'are', 'consistent', 'between', 'biological', 'replicates', 'parallel', 'computational', 'approaches', 'are', 'developed', 'to', 'make', 'non', 'linear', 'model', 'fitting', 'faster', 'and', 'more', 'reliable', 'making', 'the', 'application', 'of', 'glms', 'to', 'genomic', 'data', 'more', 'convenient', 'and', 'practical', 'simulations', 'demonstrate', 'the', 'ability', 'of', 'adjusted', 'profile', 'likelihood', 'estimators', 'to', 'return', 'accurate', 'estimators', 'of', 'biological', 'variability', 'in', 'complex', 'situations', 'when', 'variation', 'is', 'gene', 'specific', 'empirical', 'bayes', 'estimators', 'provide', 'an', 'advantageous', 'compromise', 'between', 'the', 'extremes', 'of', 'assuming', 'common', 'dispersion', 'or', 'separate', 'genewise', 'dispersion', 'the', 'methods', 'developed', 'here', 'can', 'also', 'be', 'applied', 'to', 'count', 'data', 'arising', 'from', 'dna', 'seq', 'applications', 'including', 'chip', 'seq', 'for', 'epigenetic', 'marks', 'and', 'dna', 'methylation', 'analyses'], ['cancer', 'associated', 'idh', 'mutations', 'are', 'characterized', 'by', 'neomorphic', 'enzyme', 'activity', 'and', 'resultant', 'hg', 'production', 'mutational', 'and', 'epigenetic', 'profiling', 'of', 'large', 'acute', 'myeloid', 'leukemia', 'aml', 'patient', 'cohort', 'revealed', 'that', 'idh', 'mutant', 'amls', 'display', 'global', 'dna', 'and', 'specific', 'signature', 'furthermore', 'expression', 'of', 'hg', 'producing', 'idh', 'alleles', 'in', 'cells', 'induced', 'global', 'dna', 'in', 'the', 'aml', 'cohort', 'idh', 'mutations', 'were', 'mutually', 'exclusive', 'with', 'mutations', 'in', 'the', 'alpha', 'ketoglutarate', 'dependent', 'enzyme', 'tet', 'and', 'tet', 'loss', 'of', 'function', 'mutations', 'were', 'associated', 'with', 'similar', 'epigenetic', 'defects', 'as', 'idh', 'mutants', 'consistent', 'with', 'these', 'genetic', 'and', 'epigenetic', 'data', 'expression', 'of', 'idh', 'mutants', 'impaired', 'tet', 'catalytic', 'function', 'in', 'cells', 'finally', 'either', 'expression', 'of', 'mutant', 'idh', 'or', 'tet', 'depletion', 'impaired', 'hematopoietic', 'differentiation', 'and', 'increased', 'stem', 'progenitor', 'cell', 'marker', 'expression', 'suggesting', 'shared', 'proleukemogenic', 'effect'], ['the', 'epigenetic', 'regulation', 'of', 'dna', 'templated', 'processes', 'has', 'been', 'intensely', 'studied', 'over', 'the', 'last', 'years', 'dna', 'methylation', 'histone', 'modification', 'nucleosome', 'remodeling', 'and', 'rna', 'mediated', 'targeting', 'regulate', 'many', 'biological', 'processes', 'that', 'are', 'fundamental', 'to', 'the', 'genesis', 'of', 'cancer', 'here', 'we', 'present', 'the', 'basic', 'principles', 'behind', 'these', 'epigenetic', 'pathways', 'and', 'highlight', 'the', 'evidence', 'suggesting', 'that', 'their', 'misregulation', 'can', 'culminate', 'in', 'cancer', 'this', 'information', 'along', 'with', 'the', 'promising', 'clinical', 'and', 'preclinical', 'results', 'seen', 'with', 'epigenetic', 'drugs', 'against', 'chromatin', 'regulators', 'signifies', 'that', 'it', 'is', 'time', 'to', 'embrace', 'the', 'central', 'role', 'of', 'epigenetics', 'in', 'cancer'], ['dna', 'methylation', 'is', 'among', 'the', 'best', 'studied', 'epigenetic', 'modifications', 'and', 'is', 'essential', 'to', 'mammalian', 'development', 'although', 'the', 'methylation', 'status', 'of', 'most', 'cpg', 'dinucteotides', 'in', 'the', 'genome', 'is', 'stably', 'propagated', 'through', 'mitosis', 'improvements', 'to', 'methods', 'for', 'measuring', 'methylation', 'have', 'identified', 'numerous', 'regions', 'in', 'which', 'it', 'is', 'dynamically', 'regulated', 'in', 'this', 'review', 'we', 'discuss', 'key', 'concepts', 'in', 'the', 'function', 'of', 'dna', 'methylation', 'in', 'mammals', 'stemming', 'from', 'more', 'than', 'two', 'decades', 'of', 'research', 'including', 'many', 'recent', 'studies', 'that', 'have', 'elucidated', 'when', 'and', 'where', 'dna', 'methylation', 'has', 'regulatory', 'role', 'in', 'the', 'genome', 'we', 'include', 'insights', 'from', 'early', 'development', 'embryonic', 'stem', 'cells', 'and', 'adult', 'lineages', 'particularly', 'haematopoiesis', 'to', 'highlight', 'the', 'general', 'features', 'of', 'this', 'modification', 'as', 'it', 'participates', 'in', 'both', 'global', 'and', 'localized', 'epigenetic', 'regulation'], ['methylation', 'of', 'the', 'position', 'of', 'adenosine', 'is', 'modification', 'of', 'rna', 'with', 'poorly', 'understood', 'prevalence', 'and', 'physiological', 'relevance', 'the', 'recent', 'discovery', 'that', 'fto', 'an', 'obesity', 'risk', 'gene', 'encodes', 'an', 'demethylase', 'implicates', 'as', 'an', 'important', 'regulator', 'of', 'physiological', 'processes', 'here', 'we', 'present', 'method', 'for', 'transcriptome', 'wide', 'localization', 'which', 'combines', 'specific', 'methylated', 'rna', 'with', 'next', 'generation', 'sequencing', 'merip', 'seq', 'we', 'use', 'this', 'method', 'to', 'identify', 'mrnas', 'of', 'mammalian', 'genes', 'that', 'contain', 'indicating', 'that', 'is', 'common', 'base', 'modification', 'of', 'mrna', 'the', 'modification', 'exhibits', 'tissue', 'specific', 'regulation', 'and', 'is', 'markedly', 'increased', 'throughout', 'brain', 'development', 'we', 'find', 'that', 'sites', 'are', 'enriched', 'near', 'stop', 'codons', 'and', 'in', 'utrs', 'and', 'we', 'uncover', 'an', 'association', 'between', 'residues', 'and', 'microrna', 'binding', 'sites', 'within', 'utrs', 'these', 'findings', 'provide', 'resource', 'for', 'identifying', 'transcripts', 'that', 'are', 'substrates', 'for', 'adenosine', 'methylation', 'and', 'reveal', 'insights', 'into', 'the', 'epigenetic', 'regulation', 'of', 'the', 'mammalian', 'transcriptome'], ['chromatin', 'regulators', 'have', 'become', 'attractive', 'targets', 'for', 'cancer', 'therapy', 'but', 'it', 'is', 'unclear', 'why', 'inhibition', 'of', 'these', 'ubiquitous', 'regulators', 'should', 'have', 'gene', 'specific', 'effects', 'in', 'tumor', 'cells', 'here', 'we', 'investigate', 'how', 'inhibition', 'of', 'the', 'widely', 'expressed', 'transcriptional', 'coactivator', 'brd', 'leads', 'to', 'selective', 'inhibition', 'of', 'the', 'myc', 'oncogene', 'in', 'multiple', 'myeloma', 'mm', 'brd', 'and', 'mediator', 'were', 'found', 'to', 'co', 'occupy', 'thousands', 'of', 'enhancers', 'associated', 'with', 'active', 'genes', 'they', 'also', 'co', 'occupied', 'small', 'set', 'of', 'exceptionally', 'large', 'super', 'enhancers', 'associated', 'with', 'genes', 'that', 'feature', 'prominently', 'in', 'mm', 'biology', 'including', 'the', 'myc', 'oncogene', 'treatment', 'of', 'mm', 'tumor', 'cells', 'with', 'the', 'bet', 'bromodomain', 'inhibitor', 'jq', 'led', 'to', 'preferential', 'loss', 'of', 'brd', 'at', 'super', 'enhancers', 'and', 'consequent', 'transcription', 'elongation', 'defects', 'that', 'preferentially', 'impacted', 'genes', 'with', 'super', 'enhancers', 'including', 'myc', 'super', 'enhancers', 'were', 'found', 'at', 'key', 'oncogenic', 'drivers', 'in', 'many', 'other', 'tumor', 'cells', 'these', 'observations', 'have', 'implications', 'for', 'the', 'discovery', 'of', 'cancer', 'therapeutics', 'directed', 'at', 'components', 'of', 'super', 'enhancers', 'in', 'diverse', 'tumor', 'types'], ['the', 'ability', 'to', 'measure', 'human', 'aging', 'from', 'molecular', 'profiles', 'has', 'practical', 'implications', 'in', 'many', 'fields', 'including', 'disease', 'prevention', 'and', 'treatment', 'forensics', 'and', 'extension', 'of', 'life', 'although', 'chronological', 'age', 'has', 'been', 'linked', 'to', 'changes', 'in', 'dna', 'methylation', 'the', 'methylome', 'has', 'not', 'yet', 'been', 'used', 'to', 'measure', 'and', 'compare', 'human', 'aging', 'rates', 'here', 'we', 'build', 'quantitative', 'model', 'of', 'aging', 'using', 'measurements', 'at', 'more', 'than', 'cpg', 'markers', 'from', 'the', 'whole', 'blood', 'of', 'human', 'individuals', 'aged', 'to', 'this', 'model', 'measures', 'the', 'rate', 'at', 'which', 'an', 'individual', 'methylome', 'ages', 'which', 'we', 'show', 'is', 'impacted', 'by', 'gender', 'and', 'genetic', 'variants', 'we', 'also', 'show', 'that', 'differences', 'in', 'aging', 'rates', 'help', 'explain', 'epigenetic', 'drift', 'and', 'are', 'reflected', 'in', 'the', 'transcriptome', 'moreover', 'we', 'show', 'how', 'our', 'aging', 'model', 'is', 'upheld', 'in', 'other', 'human', 'tissues', 'and', 'reveals', 'an', 'advanced', 'aging', 'rate', 'in', 'tumor', 'tissue', 'our', 'model', 'highlights', 'specific', 'components', 'of', 'the', 'aging', 'process', 'and', 'provides', 'quantitative', 'readout', 'for', 'studying', 'the', 'role', 'of', 'methylation', 'in', 'age', 'related', 'disease'], ['organisms', 'require', 'an', 'appropriate', 'balance', 'of', 'stability', 'and', 'reversibility', 'in', 'gene', 'expression', 'programmes', 'to', 'maintain', 'cell', 'identity', 'or', 'to', 'enable', 'responses', 'to', 'stimuli', 'epigenetic', 'regulation', 'is', 'integral', 'to', 'this', 'dynamic', 'control', 'post', 'translational', 'modification', 'of', 'histones', 'by', 'methylation', 'is', 'an', 'important', 'and', 'widespread', 'type', 'of', 'chromatin', 'modification', 'that', 'is', 'known', 'to', 'influence', 'biological', 'processes', 'in', 'the', 'context', 'of', 'development', 'and', 'cellular', 'responses', 'to', 'evaluate', 'how', 'histone', 'methylation', 'contributes', 'to', 'stable', 'or', 'reversible', 'control', 'we', 'provide', 'broad', 'overview', 'of', 'how', 'histone', 'methylation', 'is', 'regulated', 'and', 'leads', 'to', 'biological', 'outcomes', 'the', 'importance', 'of', 'appropriately', 'maintaining', 'or', 'reprogramming', 'histone', 'methylation', 'is', 'illustrated', 'by', 'its', 'links', 'to', 'disease', 'and', 'ageing', 'and', 'possibly', 'to', 'transmission', 'of', 'traits', 'across', 'generations'], ['chromosomes', 'are', 'the', 'physical', 'realization', 'of', 'genetic', 'information', 'and', 'thus', 'form', 'the', 'basis', 'for', 'its', 'readout', 'and', 'propagation', 'here', 'we', 'present', 'high', 'resolution', 'chromosomal', 'contact', 'map', 'derived', 'from', 'modified', 'genome', 'wide', 'chromosome', 'conformation', 'capture', 'approach', 'applied', 'to', 'drosophila', 'embryonic', 'nuclei', 'the', 'data', 'show', 'that', 'the', 'entire', 'genome', 'is', 'linearly', 'partitioned', 'into', 'well', 'demarcated', 'physical', 'domains', 'that', 'overlap', 'extensively', 'with', 'active', 'and', 'repressive', 'epigenetic', 'marks', 'chromosomal', 'contacts', 'are', 'hierarchically', 'organized', 'between', 'domains', 'global', 'modeling', 'of', 'contact', 'density', 'and', 'clustering', 'of', 'domains', 'show', 'that', 'inactive', 'domains', 'are', 'condensed', 'and', 'confined', 'to', 'their', 'chromosomal', 'territories', 'whereas', 'active', 'domains', 'reach', 'out', 'of', 'the', 'territory', 'to', 'form', 'remote', 'intra', 'and', 'contacts', 'moreover', 'we', 'systematically', 'identify', 'specific', 'long', 'range', 'contacts', 'between', 'polycomb', 'repressed', 'domains', 'together', 'these', 'observations', 'allow', 'for', 'quantitative', 'prediction', 'of', 'the', 'drosophila', 'chromosomal', 'contact', 'map', 'laying', 'the', 'foundation', 'for', 'detailed', 'studies', 'of', 'chromosome', 'structure', 'and', 'function', 'in', 'genetically', 'tractable', 'system'], ['hybridization', 'has', 'many', 'and', 'varied', 'impacts', 'on', 'the', 'process', 'of', 'speciation', 'hybridization', 'may', 'slow', 'or', 'reverse', 'differentiation', 'by', 'allowing', 'gene', 'flow', 'and', 'recombination', 'it', 'may', 'accelerate', 'speciation', 'via', 'adaptive', 'introgression', 'or', 'cause', 'near', 'instantaneous', 'speciation', 'by', 'it', 'may', 'have', 'multiple', 'effects', 'at', 'different', 'stages', 'and', 'in', 'different', 'spatial', 'contexts', 'within', 'single', 'speciation', 'event', 'we', 'offer', 'perspective', 'on', 'the', 'context', 'and', 'evolutionary', 'significance', 'of', 'hybridization', 'during', 'speciation', 'highlighting', 'issues', 'of', 'current', 'interest', 'and', 'debate', 'in', 'secondary', 'contact', 'zones', 'it', 'is', 'uncertain', 'if', 'barriers', 'to', 'gene', 'flow', 'will', 'be', 'strengthened', 'or', 'broken', 'down', 'due', 'to', 'recombination', 'and', 'gene', 'flow', 'theory', 'and', 'empirical', 'evidence', 'suggest', 'the', 'latter', 'is', 'more', 'likely', 'except', 'within', 'and', 'around', 'strongly', 'selected', 'genomic', 'regions', 'hybridization', 'may', 'contribute', 'to', 'speciation', 'through', 'the', 'formation', 'of', 'new', 'hybrid', 'taxa', 'whereas', 'introgression', 'of', 'few', 'loci', 'may', 'promote', 'adaptive', 'divergence', 'and', 'so', 'facilitate', 'speciation', 'gene', 'regulatory', 'networks', 'epigenetic', 'effects', 'and', 'the', 'evolution', 'of', 'selfish', 'genetic', 'material', 'in', 'the', 'genome', 'suggest', 'that', 'the', 'model', 'of', 'hybrid', 'requires', 'broader', 'interpretation', 'finally', 'although', 'the', 'incidence', 'of', 'reinforcement', 'remains', 'uncertain', 'this', 'and', 'other', 'interactions', 'in', 'areas', 'of', 'sympatry', 'may', 'have', 'knock', 'on', 'effects', 'on', 'speciation', 'both', 'within', 'and', 'outside', 'regions', 'of', 'hybridization'], ['glioblastoma', 'gbm', 'is', 'brain', 'tumor', 'that', 'carries', 'dismal', 'prognosis', 'and', 'displays', 'considerable', 'heterogeneity', 'we', 'have', 'recently', 'identified', 'recurrent', 'mutations', 'affecting', 'two', 'critical', 'amino', 'acids', 'and', 'of', 'histone', 'in', 'one', 'third', 'of', 'pediatric', 'gbm', 'here', 'we', 'show', 'that', 'each', 'mutation', 'defines', 'an', 'epigenetic', 'subgroup', 'of', 'gbm', 'with', 'distinct', 'global', 'methylation', 'pattern', 'and', 'that', 'they', 'are', 'mutually', 'exclusive', 'with', 'idh', 'mutations', 'which', 'characterize', 'third', 'mutation', 'defined', 'subgroup', 'three', 'further', 'epigenetic', 'subgroups', 'were', 'enriched', 'for', 'hallmark', 'genetic', 'events', 'of', 'adult', 'gbm', 'and', 'or', 'established', 'transcriptomic', 'signatures', 'we', 'also', 'demonstrate', 'that', 'the', 'two', 'mutations', 'give', 'rise', 'to', 'gbms', 'in', 'separate', 'anatomic', 'compartments', 'with', 'differential', 'regulation', 'of', 'transcription', 'factors', 'olig', 'olig', 'and', 'foxg', 'possibly', 'reflecting', 'different', 'cellular', 'origins'], ['epigenetic', 'phenomena', 'in', 'animals', 'and', 'plants', 'are', 'mediated', 'by', 'dna', 'methylation', 'and', 'stable', 'chromatin', 'modifications', 'there', 'has', 'been', 'considerable', 'interest', 'in', 'whether', 'environmental', 'factors', 'modulate', 'the', 'establishment', 'and', 'maintenance', 'of', 'epigenetic', 'modifications', 'and', 'could', 'thereby', 'influence', 'gene', 'expression', 'and', 'phenotype', 'chemical', 'pollutants', 'dietary', 'components', 'temperature', 'changes', 'and', 'other', 'external', 'stresses', 'can', 'indeed', 'have', 'long', 'lasting', 'effects', 'on', 'development', 'metabolism', 'and', 'health', 'sometimes', 'even', 'in', 'subsequent', 'generations', 'although', 'the', 'underlying', 'mechanisms', 'remain', 'largely', 'unknown', 'particularly', 'in', 'humans', 'mechanistic', 'insights', 'are', 'emerging', 'from', 'experimental', 'model', 'systems', 'these', 'have', 'implications', 'for', 'structuring', 'future', 'research', 'and', 'understanding', 'disease', 'and', 'development'], ['methylation', 'of', 'cytosine', 'bases', 'in', 'dna', 'provides', 'layer', 'of', 'epigenetic', 'control', 'in', 'many', 'eukaryotes', 'that', 'has', 'important', 'implications', 'for', 'normal', 'biology', 'and', 'disease', 'therefore', 'profiling', 'dna', 'methylation', 'across', 'the', 'genome', 'is', 'vital', 'to', 'understanding', 'the', 'influence', 'of', 'epigenetics', 'there', 'has', 'been', 'revolution', 'in', 'dna', 'methylation', 'analysis', 'technology', 'over', 'the', 'past', 'decade', 'analyses', 'that', 'previously', 'were', 'restricted', 'to', 'specific', 'loci', 'can', 'now', 'be', 'performed', 'on', 'genome', 'scale', 'and', 'entire', 'methylomes', 'can', 'be', 'characterized', 'at', 'single', 'base', 'pair', 'resolution', 'however', 'there', 'is', 'such', 'diversity', 'of', 'dna', 'methylation', 'profiling', 'techniques', 'that', 'it', 'can', 'be', 'challenging', 'to', 'select', 'one', 'this', 'review', 'discusses', 'the', 'different', 'approaches', 'and', 'their', 'relative', 'merits', 'and', 'introduces', 'considerations', 'for', 'data', 'analysis'], ['we', 'have', 'developed', 'new', 'generation', 'of', 'genome', 'wide', 'dna', 'methylation', 'bead', 'chip', 'which', 'allows', 'high', 'throughput', 'methylation', 'profiling', 'of', 'the', 'human', 'genome', 'the', 'new', 'high', 'density', 'beadchip', 'can', 'assay', 'over', 'cpg', 'sites', 'and', 'analyze', 'twelve', 'samples', 'in', 'parallel', 'the', 'innovative', 'content', 'includes', 'coverage', 'of', 'of', 'refseq', 'genes', 'with', 'multiple', 'probes', 'per', 'gene', 'of', 'cpg', 'islands', 'from', 'the', 'ucsc', 'database', 'cpg', 'island', 'shores', 'and', 'additional', 'content', 'selected', 'from', 'whole', 'genome', 'bisulfite', 'sequencing', 'data', 'and', 'input', 'from', 'dna', 'methylation', 'experts', 'the', 'well', 'characterized', 'infinium', 'assay', 'is', 'used', 'for', 'analysis', 'of', 'cpg', 'methylation', 'using', 'bisulfite', 'converted', 'genomic', 'dna', 'we', 'applied', 'this', 'technology', 'to', 'analyze', 'dna', 'methylation', 'in', 'normal', 'and', 'tumor', 'dna', 'samples', 'and', 'compared', 'results', 'with', 'whole', 'genome', 'bisulfite', 'sequencing', 'wgbs', 'data', 'obtained', 'for', 'the', 'same', 'samples', 'highly', 'comparable', 'dna', 'methylation', 'profiles', 'were', 'generated', 'by', 'the', 'array', 'and', 'sequencing', 'methods', 'average', 'of', 'the', 'ability', 'to', 'determine', 'genome', 'wide', 'methylation', 'patterns', 'will', 'rapidly', 'advance', 'methylation', 'research', 'elsevier', 'inc', 'all', 'rights', 'reserved'], ['expression', 'of', 'the', 'ink', 'arf', 'ink', 'tumor', 'suppressor', 'locus', 'in', 'normal', 'and', 'cancerous', 'cell', 'growth', 'is', 'controlled', 'by', 'methylation', 'of', 'histone', 'at', 'lysine', 'me', 'as', 'directed', 'by', 'the', 'polycomb', 'group', 'proteins', 'the', 'antisense', 'noncoding', 'rna', 'anril', 'of', 'the', 'ink', 'arf', 'ink', 'locus', 'is', 'also', 'important', 'for', 'expression', 'of', 'the', 'protein', 'coding', 'genes', 'in', 'cis', 'but', 'its', 'mechanism', 'has', 'remained', 'elusive', 'here', 'we', 'report', 'that', 'chromobox', 'cbx', 'within', 'the', 'polycomb', 'repressive', 'complex', 'binds', 'to', 'anril', 'and', 'both', 'cbx', 'and', 'anril', 'are', 'found', 'at', 'elevated', 'levels', 'in', 'prostate', 'cancer', 'tissues', 'in', 'concert', 'with', 'me', 'recognition', 'binding', 'to', 'rna', 'contributes', 'to', 'cbx', 'function', 'and', 'disruption', 'of', 'either', 'interaction', 'impacts', 'the', 'ability', 'of', 'cbx', 'to', 'repress', 'the', 'ink', 'arf', 'ink', 'locus', 'and', 'control', 'senescence', 'structure', 'guided', 'analysis', 'reveals', 'the', 'molecular', 'interplay', 'between', 'noncoding', 'rna', 'and', 'me', 'as', 'mediated', 'by', 'the', 'conserved', 'chromodomain', 'our', 'study', 'suggests', 'mechanism', 'by', 'which', 'noncoding', 'rna', 'participates', 'directly', 'in', 'epigenetic', 'transcriptional', 'repression'], ['determining', 'how', 'somatic', 'copy', 'number', 'alterations', 'scnas', 'promote', 'cancer', 'is', 'an', 'important', 'goal', 'we', 'characterized', 'scna', 'patterns', 'in', 'cancers', 'from', 'the', 'cancer', 'genome', 'atlas', 'pan', 'cancer', 'data', 'set', 'whole', 'genome', 'doubling', 'observed', 'in', 'of', 'cancers', 'was', 'associated', 'with', 'higher', 'rates', 'of', 'every', 'other', 'type', 'of', 'scna', 'tp', 'mutations', 'ccne', 'amplifications', 'and', 'alterations', 'of', 'the', 'ppp', 'complex', 'scnas', 'that', 'were', 'internal', 'to', 'chromosomes', 'tended', 'to', 'be', 'shorter', 'than', 'telomere', 'bounded', 'scnas', 'suggesting', 'different', 'mechanisms', 'underlying', 'their', 'generation', 'significantly', 'recurrent', 'focal', 'scnas', 'were', 'observed', 'in', 'regions', 'including', 'without', 'known', 'oncogene', 'or', 'tumor', 'suppressor', 'gene', 'targets', 'and', 'with', 'significantly', 'mutated', 'genes', 'amplified', 'regions', 'without', 'known', 'oncogenes', 'were', 'enriched', 'for', 'genes', 'involved', 'in', 'epigenetic', 'regulation', 'when', 'levels', 'of', 'genomic', 'disruption', 'were', 'accounted', 'for', 'of', 'region', 'pairs', 'were', 'anticorrelated', 'and', 'these', 'regions', 'tended', 'to', 'encompass', 'genes', 'whose', 'proteins', 'physically', 'interact', 'suggesting', 'related', 'functions', 'these', 'results', 'provide', 'insights', 'into', 'mechanisms', 'of', 'generation', 'and', 'functional', 'consequences', 'of', 'cancer', 'related', 'scnas'], ['high', 'throughput', 'technologies', 'are', 'widely', 'used', 'for', 'example', 'to', 'assay', 'genetic', 'variants', 'gene', 'and', 'protein', 'expression', 'and', 'epigenetic', 'modifications', 'one', 'often', 'overlooked', 'complication', 'with', 'such', 'studies', 'is', 'batch', 'effects', 'which', 'occur', 'because', 'measurements', 'are', 'affected', 'by', 'laboratory', 'conditions', 'reagent', 'lots', 'and', 'personnel', 'differences', 'this', 'becomes', 'major', 'problem', 'when', 'batch', 'effects', 'are', 'correlated', 'with', 'an', 'outcome', 'of', 'interest', 'and', 'lead', 'to', 'incorrect', 'conclusions', 'using', 'both', 'published', 'studies', 'and', 'our', 'own', 'analyses', 'we', 'argue', 'that', 'batch', 'effects', 'as', 'well', 'as', 'other', 'technical', 'and', 'biological', 'artefacts', 'are', 'widespread', 'and', 'critical', 'to', 'address', 'we', 'review', 'experimental', 'and', 'computational', 'approaches', 'for', 'doing', 'so'], ['cytosine', 'methylation', 'is', 'the', 'major', 'covalent', 'modification', 'of', 'mammalian', 'genomic', 'dna', 'and', 'plays', 'important', 'roles', 'in', 'transcriptional', 'regulation', 'the', 'molecular', 'mechanism', 'underlying', 'the', 'enzymatic', 'removal', 'of', 'this', 'epigenetic', 'mark', 'however', 'remains', 'elusive', 'here', 'we', 'show', 'that', 'methylcytosine', 'mc', 'hydroxylase', 'tet', 'by', 'converting', 'mcs', 'to', 'hmcs', 'promotes', 'dna', 'demethylation', 'in', 'mammalian', 'cells', 'through', 'process', 'that', 'requires', 'the', 'base', 'excision', 'repair', 'pathway', 'though', 'expression', 'of', 'the', 'known', 'human', 'dna', 'glycosylases', 'individually', 'did', 'not', 'enhance', 'removal', 'of', 'hmcs', 'in', 'mammalian', 'cells', 'demethylation', 'of', 'both', 'exogenously', 'introduced', 'and', 'endogenous', 'hmcs', 'is', 'promoted', 'by', 'the', 'aid', 'activation', 'induced', 'deaminase', 'apobec', 'apolipoprotein', 'mrna', 'editing', 'enzyme', 'complex', 'family', 'of', 'cytidine', 'deaminases', 'furthermore', 'tet', 'and', 'apobec', 'are', 'involved', 'in', 'neuronal', 'activity', 'induced', 'region', 'specific', 'active', 'dna', 'demethylation', 'and', 'subsequent', 'gene', 'expression', 'in', 'the', 'dentate', 'gyrus', 'of', 'the', 'adult', 'mouse', 'brain', 'in', 'vivo', 'our', 'study', 'suggests', 'tet', 'induced', 'oxidation', 'deamination', 'mechanism', 'for', 'active', 'dna', 'demethylation', 'in', 'mammals'], ['nuclear', 'factor', 'erythroid', 'related', 'factor', 'nrf', 'also', 'called', 'nfe', 'is', 'transcription', 'factor', 'that', 'regulates', 'the', 'cellular', 'redox', 'status', 'nrf', 'is', 'controlled', 'through', 'complex', 'transcriptional', 'epigenetic', 'and', 'post', 'translational', 'network', 'that', 'ensures', 'its', 'activity', 'increases', 'during', 'redox', 'perturbation', 'inflammation', 'growth', 'factor', 'stimulation', 'and', 'nutrient', 'energy', 'fluxes', 'thereby', 'enabling', 'the', 'factor', 'to', 'orchestrate', 'adaptive', 'responses', 'to', 'diverse', 'forms', 'of', 'stress', 'besides', 'mediating', 'stress', 'stimulated', 'induction', 'of', 'antioxidant', 'and', 'detoxification', 'genes', 'nr', 'contributes', 'to', 'adaptation', 'by', 'upregulating', 'the', 'repair', 'and', 'degradation', 'of', 'damaged', 'macromolecules', 'and', 'by', 'modulating', 'intermediary', 'metabolism', 'in', 'the', 'latter', 'case', 'nrf', 'inhibits', 'lipogenesis', 'supports', 'oxidation', 'of', 'fatty', 'acids', 'facilitates', 'flux', 'through', 'the', 'pentose', 'phosphate', 'pathway', 'and', 'increases', 'nadph', 'regeneration', 'and', 'purine', 'biosynthesis', 'these', 'observations', 'suggest', 'nrf', 'directs', 'metabolic', 'reprogramming', 'during', 'stress'], ['genomes', 'of', 'complex', 'organisms', 'encode', 'an', 'abundance', 'and', 'diversity', 'of', 'long', 'noncoding', 'rnas', 'incrnas', 'that', 'are', 'expressed', 'throughout', 'the', 'cell', 'and', 'fulfill', 'wide', 'variety', 'of', 'regulatory', 'roles', 'at', 'almost', 'every', 'stage', 'of', 'gene', 'expression', 'these', 'roles', 'which', 'encompass', 'sensory', 'guiding', 'scaffolding', 'and', 'allosteric', 'capacities', 'derive', 'from', 'folded', 'modular', 'domains', 'in', 'incrnas', 'in', 'this', 'diverse', 'functional', 'repertoire', 'we', 'focus', 'on', 'the', 'well', 'characterized', 'ability', 'for', 'incrnas', 'to', 'function', 'as', 'epigenetic', 'modulators', 'many', 'incrnas', 'bind', 'to', 'chromatin', 'modifying', 'proteins', 'and', 'recruit', 'their', 'catalytic', 'activity', 'to', 'specific', 'sites', 'in', 'the', 'genome', 'thereby', 'modulating', 'chromatin', 'states', 'and', 'impacting', 'gene', 'expression', 'considering', 'this', 'regulatory', 'potential', 'in', 'combination', 'with', 'the', 'abundance', 'of', 'incrnas', 'suggests', 'that', 'incrnas', 'may', 'be', 'part', 'of', 'broad', 'epigenetic', 'regulatory', 'network'], ['bromodomains', 'brds', 'are', 'protein', 'interaction', 'modules', 'that', 'specifically', 'recognize', 'epsilon', 'lysine', 'acetylation', 'motifs', 'key', 'event', 'in', 'the', 'reading', 'process', 'of', 'epigenetic', 'marks', 'the', 'brds', 'in', 'the', 'human', 'genome', 'cluster', 'into', 'eight', 'families', 'based', 'on', 'structure', 'sequence', 'similarity', 'here', 'we', 'present', 'high', 'resolution', 'crystal', 'structures', 'covering', 'all', 'brd', 'families', 'comprehensive', 'crossfamily', 'structural', 'analysis', 'identifies', 'conserved', 'and', 'family', 'specific', 'structural', 'features', 'that', 'are', 'necessary', 'for', 'specific', 'acetylation', 'dependent', 'substrate', 'recognition', 'screening', 'of', 'more', 'than', 'representative', 'brds', 'against', 'systematic', 'histone', 'peptide', 'arrays', 'identifies', 'new', 'brd', 'substrates', 'and', 'reveals', 'strong', 'influence', 'of', 'flanking', 'modifications', 'such', 'as', 'acetylation', 'and', 'phosphorylation', 'suggesting', 'that', 'brds', 'recognize', 'combinations', 'of', 'marks', 'rather', 'than', 'singly', 'acetylated', 'sequences', 'we', 'further', 'uncovered', 'structural', 'mechanism', 'for', 'the', 'simultaneous', 'binding', 'and', 'recognition', 'of', 'diverse', 'diacetyl', 'containing', 'peptides', 'by', 'brd', 'these', 'data', 'provide', 'foundation', 'for', 'structure', 'based', 'drug', 'design', 'of', 'specific', 'inhibitors', 'for', 'this', 'emerging', 'target', 'family'], ['epigenetics', 'is', 'one', 'of', 'the', 'most', 'promising', 'and', 'expanding', 'fields', 'in', 'the', 'current', 'biomedical', 'research', 'landscape', 'since', 'the', 'inception', 'of', 'epigenetics', 'in', 'the', 'the', 'discoveries', 'regarding', 'its', 'implications', 'in', 'normal', 'and', 'disease', 'biology', 'have', 'not', 'stopped', 'compiling', 'vast', 'amount', 'of', 'knowledge', 'in', 'the', 'past', 'decade', 'the', 'field', 'has', 'moved', 'from', 'just', 'one', 'recognized', 'marker', 'dna', 'methylation', 'to', 'variety', 'of', 'others', 'including', 'wide', 'spectrum', 'of', 'histone', 'modifications', 'from', 'the', 'methodological', 'standpoint', 'the', 'successful', 'initial', 'single', 'gene', 'candidate', 'approaches', 'have', 'been', 'complemented', 'by', 'the', 'current', 'comprehensive', 'epigenomic', 'approaches', 'that', 'allow', 'the', 'interrogation', 'of', 'genomes', 'to', 'search', 'for', 'translational', 'applications', 'in', 'an', 'unbiased', 'manner', 'most', 'important', 'the', 'discovery', 'of', 'mutations', 'in', 'the', 'epigenetic', 'machinery', 'and', 'the', 'approval', 'of', 'the', 'first', 'epigenetic', 'drugs', 'for', 'the', 'treatment', 'of', 'subtypes', 'of', 'leukemias', 'and', 'lymphomas', 'has', 'been', 'an', 'eye', 'opener', 'for', 'many', 'biomedical', 'scientists', 'and', 'clinicians', 'herein', 'we', 'will', 'summarize', 'the', 'progress', 'in', 'the', 'field', 'of', 'cancer', 'epigenetics', 'research', 'that', 'has', 'reached', 'mainstream', 'oncology', 'in', 'the', 'development', 'of', 'new', 'biomarkers', 'of', 'the', 'disease', 'and', 'new', 'pharmacological', 'strategies'], ['abnormalities', 'of', 'chromosome', 'are', 'common', 'in', 'myeloid', 'malignancies', 'but', 'no', 'specific', 'target', 'genes', 'have', 'yet', 'been', 'identified', 'here', 'we', 'describe', 'the', 'finding', 'of', 'homozygous', 'ezh', 'mutations', 'in', 'of', 'individuals', 'with', 'acquired', 'uniparental', 'disomy', 'screening', 'of', 'total', 'of', 'individuals', 'with', 'myeloid', 'disorders', 'revealed', 'monoallelic', 'or', 'biallelic', 'ezh', 'mutations', 'in', 'individuals', 'the', 'mutations', 'were', 'found', 'most', 'commonly', 'in', 'those', 'with', 'myelodysplastic', 'neoplasms', 'out', 'of', 'individuals', 'or', 'and', 'in', 'those', 'with', 'myelofibrosis', 'out', 'of', 'individuals', 'or', 'ezh', 'encodes', 'the', 'catalytic', 'subunit', 'of', 'the', 'polycomb', 'repressive', 'complex', 'prc', 'highly', 'conserved', 'histone', 'lysine', 'that', 'influences', 'stem', 'cell', 'renewal', 'by', 'epigenetic', 'repression', 'of', 'genes', 'involved', 'in', 'cell', 'fate', 'decisions', 'ezh', 'has', 'oncogenic', 'activity', 'and', 'its', 'overexpression', 'has', 'previously', 'been', 'causally', 'linked', 'to', 'differentiation', 'blocks', 'in', 'epithelial', 'tumors', 'notably', 'the', 'mutations', 'we', 'identified', 'resulted', 'in', 'premature', 'chain', 'termination', 'or', 'direct', 'abrogation', 'of', 'histone', 'activity', 'suggesting', 'that', 'ezh', 'acts', 'as', 'tumor', 'suppressor', 'for', 'myeloid', 'malignancies'], ['with', 'rare', 'exceptions', 'spontaneous', 'tumors', 'originate', 'from', 'single', 'cell', 'yet', 'at', 'the', 'time', 'of', 'clinical', 'diagnosis', 'the', 'majority', 'of', 'human', 'tumors', 'display', 'startling', 'heterogeneity', 'in', 'many', 'morphological', 'and', 'physiological', 'features', 'such', 'as', 'expression', 'of', 'cell', 'surface', 'receptors', 'proliferative', 'and', 'angiogenic', 'potential', 'to', 'substantial', 'extent', 'this', 'heterogeneity', 'might', 'be', 'attributed', 'to', 'morphological', 'and', 'epigenetic', 'plasticity', 'but', 'there', 'is', 'also', 'strong', 'evidence', 'for', 'the', 'co', 'existence', 'of', 'genetically', 'divergent', 'tumor', 'cell', 'clones', 'within', 'tumors', 'in', 'this', 'perspective', 'we', 'summarize', 'the', 'sources', 'of', 'intra', 'tumor', 'phenotypic', 'heterogeneity', 'with', 'emphasis', 'on', 'genetic', 'heterogeneity', 'we', 'review', 'experimental', 'evidence', 'for', 'the', 'existence', 'of', 'both', 'intra', 'tumor', 'clonal', 'heterogeneity', 'as', 'well', 'as', 'frequent', 'evolutionary', 'divergence', 'between', 'primary', 'tumors', 'and', 'metastatic', 'outgrowths', 'furthermore', 'we', 'discuss', 'potential', 'biological', 'and', 'clinical', 'implications', 'of', 'intra', 'tumor', 'clonal', 'heterogeneity', 'elsevier', 'all', 'rights', 'reserved'], ['since', 'the', 'human', 'genome', 'was', 'sequenced', 'the', 'term', 'epigenetics', 'is', 'increasingly', 'being', 'associated', 'with', 'the', 'hope', 'that', 'we', 'are', 'more', 'than', 'just', 'the', 'sum', 'of', 'our', 'genes', 'might', 'what', 'we', 'eat', 'the', 'air', 'we', 'breathe', 'or', 'even', 'the', 'emotions', 'we', 'feel', 'influence', 'not', 'only', 'our', 'genes', 'but', 'those', 'of', 'descendants', 'the', 'environment', 'can', 'certainly', 'influence', 'gene', 'expression', 'and', 'can', 'lead', 'to', 'disease', 'but', 'consequences', 'are', 'another', 'matter', 'although', 'the', 'inheritance', 'of', 'epigenetic', 'characters', 'can', 'certainly', 'occur', 'particularly', 'in', 'plants', 'how', 'much', 'is', 'due', 'to', 'the', 'environment', 'and', 'the', 'extent', 'to', 'which', 'it', 'happens', 'in', 'humans', 'remain', 'unclear'], ['the', 'incorporation', 'of', 'histone', 'variants', 'has', 'been', 'implicated', 'in', 'the', 'epigenetic', 'memory', 'of', 'cellular', 'state', 'using', 'genome', 'editing', 'with', 'zinc', 'finger', 'nucleases', 'to', 'tag', 'endogenous', 'we', 'report', 'genome', 'wide', 'profiles', 'of', 'variants', 'in', 'mammalian', 'embryonic', 'stem', 'cells', 'and', 'neuronal', 'precursor', 'cells', 'genome', 'wide', 'patterns', 'of', 'are', 'dependent', 'on', 'amino', 'acid', 'sequence', 'and', 'change', 'with', 'cellular', 'differentiation', 'at', 'developmentally', 'regulated', 'loci', 'the', 'chaperone', 'hira', 'is', 'required', 'for', 'enrichment', 'at', 'active', 'and', 'repressed', 'genes', 'strikingly', 'hira', 'is', 'not', 'essential', 'for', 'localization', 'of', 'at', 'telomeres', 'and', 'many', 'transcription', 'factor', 'binding', 'sites', 'immunoaffinity', 'purification', 'and', 'mass', 'spectrometry', 'reveal', 'that', 'the', 'proteins', 'atrx', 'and', 'daxx', 'associate', 'with', 'in', 'hira', 'independent', 'manner', 'atrx', 'is', 'required', 'for', 'hira', 'independent', 'localization', 'of', 'at', 'telomeres', 'and', 'for', 'the', 'repression', 'of', 'telomeric', 'rna', 'our', 'data', 'demonstrate', 'that', 'multiple', 'and', 'distinct', 'factors', 'are', 'responsible', 'for', 'localization', 'at', 'specific', 'genomic', 'locations', 'in', 'mammalian', 'cells'], ['cyclin', 'dependent', 'kinases', 'cdks', 'are', 'serine', 'threonine', 'kinases', 'and', 'their', 'catalytic', 'activities', 'are', 'modulated', 'by', 'interactions', 'with', 'cyclins', 'and', 'cdk', 'inhibitors', 'ckis', 'close', 'cooperation', 'between', 'this', 'trio', 'is', 'necessary', 'for', 'ensuring', 'orderly', 'progression', 'through', 'the', 'cell', 'cycle', 'in', 'addition', 'to', 'their', 'well', 'established', 'function', 'in', 'cell', 'cycle', 'control', 'it', 'is', 'becoming', 'increasingly', 'apparent', 'that', 'mammalian', 'cdks', 'cyclins', 'and', 'ckis', 'play', 'indispensable', 'roles', 'in', 'processes', 'such', 'as', 'transcription', 'epigenetic', 'regulation', 'metabolism', 'stem', 'cell', 'self', 'renewal', 'neuronal', 'functions', 'and', 'spermatogenesis', 'even', 'more', 'remarkably', 'they', 'can', 'accomplish', 'some', 'of', 'these', 'tasks', 'individually', 'without', 'the', 'need', 'for', 'cdk', 'cyclin', 'complex', 'formation', 'or', 'kinase', 'activity', 'in', 'this', 'review', 'we', 'discuss', 'the', 'latest', 'revelations', 'about', 'cdks', 'cyclins', 'and', 'ckis', 'with', 'the', 'goal', 'of', 'showcasing', 'their', 'functional', 'diversity', 'beyond', 'cell', 'cycle', 'regulation', 'and', 'their', 'impact', 'on', 'development', 'and', 'disease', 'in', 'mammals'], ['during', 'the', 'course', 'of', 'malignant', 'cancer', 'progression', 'neoplastic', 'cells', 'undergo', 'dynamic', 'and', 'reversible', 'transitions', 'between', 'multiple', 'phenotypic', 'states', 'the', 'extremes', 'of', 'which', 'are', 'defined', 'by', 'the', 'expression', 'of', 'epithelial', 'and', 'mesenchymal', 'phenotypes', 'this', 'plasticity', 'is', 'enabled', 'by', 'underlying', 'shifts', 'in', 'epigenetic', 'regulation', 'small', 'cohort', 'of', 'pleiotropically', 'acting', 'transcription', 'factors', 'is', 'widely', 'recognized', 'to', 'effect', 'these', 'shifts', 'by', 'controlling', 'the', 'expression', 'of', 'constituency', 'of', 'key', 'target', 'genes', 'these', 'master', 'regulators', 'depend', 'on', 'complex', 'epigenetic', 'regulatory', 'mechanisms', 'notably', 'the', 'induction', 'of', 'changes', 'in', 'the', 'modifications', 'of', 'chromatin', 'associated', 'histones', 'in', 'order', 'to', 'achieve', 'the', 'widespread', 'changes', 'in', 'gene', 'expression', 'observed', 'during', 'epithelial', 'mesenchymal', 'transitions', 'emts', 'these', 'associations', 'indicate', 'that', 'an', 'understanding', 'of', 'the', 'functional', 'interactions', 'between', 'such', 'emt', 'inducing', 'transcription', 'factors', 'and', 'the', 'modulators', 'of', 'chromatin', 'configuration', 'will', 'provide', 'crucial', 'insights', 'into', 'the', 'fundamental', 'mechanisms', 'underlying', 'cancer', 'progression', 'and', 'may', 'in', 'the', 'longer', 'term', 'generate', 'new', 'diagnostic', 'and', 'therapeutic', 'modalities', 'for', 'treating', 'high', 'grade', 'malignancies'], ['despite', 'the', 'success', 'of', 'genome', 'wide', 'association', 'studies', 'gwass', 'in', 'identifying', 'loci', 'associated', 'with', 'common', 'diseases', 'substantial', 'proportion', 'of', 'the', 'causality', 'remains', 'unexplained', 'recent', 'advances', 'in', 'genomic', 'technologies', 'have', 'placed', 'us', 'in', 'position', 'to', 'initiate', 'large', 'scale', 'studies', 'of', 'human', 'disease', 'associated', 'epigenetic', 'variation', 'specifically', 'variation', 'in', 'dna', 'methylation', 'such', 'epigenome', 'wide', 'association', 'studies', 'ewass', 'present', 'novel', 'opportunities', 'but', 'also', 'create', 'new', 'challenges', 'that', 'are', 'not', 'encountered', 'in', 'gwass', 'we', 'discuss', 'ewas', 'design', 'cohort', 'and', 'sample', 'selections', 'statistical', 'significance', 'and', 'power', 'confounding', 'factors', 'and', 'follow', 'up', 'studies', 'we', 'also', 'discuss', 'how', 'integration', 'of', 'ewass', 'with', 'gwass', 'can', 'help', 'to', 'dissect', 'complex', 'gwas', 'haplotypes', 'for', 'functional', 'analysis'], ['discoveries', 'over', 'the', 'past', 'decade', 'portend', 'paradigm', 'shift', 'in', 'molecular', 'biology', 'evidence', 'suggests', 'that', 'rna', 'is', 'not', 'only', 'functional', 'as', 'messenger', 'between', 'dna', 'and', 'protein', 'but', 'also', 'involved', 'in', 'the', 'regulation', 'of', 'genome', 'organization', 'and', 'gene', 'expression', 'which', 'is', 'increasingly', 'elaborate', 'in', 'complex', 'organisms', 'regulatory', 'rna', 'seems', 'to', 'operate', 'at', 'many', 'levels', 'in', 'particular', 'it', 'plays', 'an', 'important', 'part', 'in', 'the', 'epigenetic', 'processes', 'that', 'control', 'differentiation', 'and', 'development', 'these', 'discoveries', 'suggest', 'central', 'role', 'for', 'rna', 'in', 'human', 'evolution', 'and', 'ontogeny', 'here', 'we', 'review', 'the', 'emergence', 'of', 'the', 'previously', 'unsuspected', 'world', 'of', 'regulatory', 'rna', 'from', 'historical', 'perspective'], ['dna', 'methylation', 'an', 'important', 'type', 'of', 'epigenetic', 'modification', 'in', 'humans', 'participates', 'in', 'crucial', 'cellular', 'processes', 'such', 'as', 'embryonic', 'development', 'inactivation', 'genomic', 'imprinting', 'and', 'chromosome', 'stability', 'several', 'platforms', 'have', 'been', 'developed', 'to', 'study', 'genome', 'wide', 'dna', 'methylation', 'many', 'investigators', 'in', 'the', 'field', 'have', 'chosen', 'the', 'illumina', 'infinium', 'microarray', 'for', 'its', 'ability', 'to', 'reliably', 'assess', 'dna', 'methylation', 'following', 'sodium', 'bisulfite', 'conversion', 'here', 'we', 'analyzed', 'methylation', 'profiles', 'of', 'adult', 'males', 'and', 'adult', 'females', 'generated', 'by', 'the', 'infinium', 'microarray', 'among', 'the', 'autosomal', 'cpg', 'sites', 'that', 'displayed', 'significant', 'methylation', 'differences', 'between', 'the', 'two', 'sexes', 'we', 'observed', 'significant', 'enrichment', 'of', 'cross', 'reactive', 'probes', 'co', 'hybridizing', 'to', 'the', 'sex', 'chromosomes', 'with', 'more', 'than', 'sequence', 'identity', 'this', 'could', 'lead', 'investigators', 'to', 'mistakenly', 'infer', 'the', 'existence', 'of', 'significant', 'autosomal', 'sex', 'associated', 'methylation', 'using', 'sequence', 'identity', 'cutoffs', 'derived', 'from', 'the', 'sex', 'methylation', 'analysis', 'we', 'concluded', 'that', 'of', 'the', 'array', 'probes', 'can', 'potentially', 'generate', 'spurious', 'signals', 'because', 'of', 'co', 'hybridization', 'to', 'alternate', 'genomic', 'sequences', 'highly', 'homologous', 'to', 'the', 'intended', 'targets', 'additionally', 'we', 'discovered', 'probes', 'targeting', 'polymorphic', 'cpgs', 'that', 'overlapped', 'snps', 'the', 'methylation', 'levels', 'detected', 'by', 'these', 'probes', 'are', 'simply', 'the', 'reflection', 'of', 'underlying', 'genetic', 'polymorphisms', 'but', 'could', 'be', 'misinterpreted', 'as', 'true', 'signals', 'the', 'existence', 'of', 'probes', 'that', 'are', 'cross', 'reactive', 'or', 'of', 'target', 'polymorphic', 'cpgs', 'in', 'the', 'illumina', 'microarrays', 'can', 'confound', 'data', 'obtained', 'from', 'such', 'microarrays', 'therefore', 'investigators', 'should', 'exercise', 'caution', 'when', 'significant', 'biological', 'associations', 'are', 'found', 'using', 'these', 'array', 'platforms', 'list', 'of', 'all', 'cross', 'reactive', 'probes', 'and', 'polymorphic', 'cpgs', 'identified', 'by', 'us', 'are', 'annotated', 'in', 'this', 'paper'], ['cancer', 'therapy', 'is', 'challenged', 'by', 'the', 'diversity', 'of', 'molecular', 'implementations', 'of', 'oncogenic', 'processes', 'and', 'by', 'the', 'resulting', 'variation', 'in', 'therapeutic', 'responses', 'projects', 'such', 'as', 'the', 'cancer', 'genome', 'atlas', 'tcga', 'provide', 'molecular', 'tumor', 'maps', 'in', 'unprecedented', 'detail', 'the', 'interpretation', 'of', 'these', 'maps', 'remains', 'major', 'challenge', 'here', 'we', 'distilled', 'thousands', 'of', 'genetic', 'and', 'epigenetic', 'features', 'altered', 'in', 'cancers', 'to', 'similar', 'to', 'selected', 'functional', 'events', 'sfes', 'using', 'this', 'simplified', 'description', 'we', 'derived', 'hierarchical', 'classification', 'of', 'tcga', 'tumors', 'from', 'cancer', 'types', 'the', 'top', 'classes', 'are', 'dominated', 'by', 'either', 'mutations', 'class', 'or', 'copy', 'number', 'changes', 'class', 'this', 'distinction', 'is', 'clearest', 'at', 'the', 'extremes', 'of', 'genomic', 'instability', 'indicating', 'the', 'presence', 'of', 'different', 'oncogenic', 'processes', 'the', 'full', 'hierarchy', 'shows', 'functional', 'event', 'patterns', 'characteristic', 'of', 'multiple', 'cross', 'tissue', 'groups', 'of', 'tumors', 'termed', 'oncogenic', 'signature', 'classes', 'targetable', 'functional', 'events', 'in', 'tumor', 'class', 'are', 'suggestive', 'of', 'class', 'specific', 'combination', 'therapy', 'these', 'results', 'may', 'assist', 'in', 'the', 'definition', 'of', 'clinical', 'trials', 'to', 'match', 'actionable', 'oncogenic', 'signatures', 'with', 'personalized', 'therapies'], ['dna', 'methylation', 'is', 'one', 'of', 'the', 'best', 'characterized', 'epigenetic', 'modifications', 'and', 'has', 'been', 'implicated', 'in', 'numerous', 'biological', 'processes', 'including', 'transposable', 'element', 'silencing', 'genomic', 'imprinting', 'and', 'chromosome', 'inactivation', 'compared', 'with', 'other', 'epigenetic', 'modifications', 'dna', 'methylation', 'is', 'thought', 'to', 'be', 'relatively', 'stable', 'despite', 'its', 'role', 'in', 'long', 'term', 'silencing', 'dna', 'methylation', 'is', 'more', 'dynamic', 'than', 'originally', 'thought', 'as', 'active', 'dna', 'demethylation', 'has', 'been', 'observed', 'during', 'specific', 'stages', 'of', 'development', 'in', 'the', 'past', 'decade', 'many', 'enzymes', 'have', 'been', 'proposed', 'to', 'carry', 'out', 'active', 'dna', 'demethylation', 'and', 'growing', 'evidence', 'suggests', 'that', 'depending', 'on', 'the', 'context', 'this', 'process', 'may', 'be', 'achieved', 'by', 'multiple', 'mechanisms', 'insight', 'into', 'how', 'dna', 'methylation', 'is', 'dynamically', 'regulated', 'will', 'broaden', 'our', 'understanding', 'of', 'epigenetic', 'regulation', 'and', 'have', 'great', 'implications', 'in', 'somatic', 'cell', 'reprogramming', 'and', 'regenerative', 'medicine'], ['dna', 'methylation', 'is', 'one', 'of', 'the', 'most', 'intensely', 'studied', 'epigenetic', 'modifications', 'in', 'mammals', 'in', 'normal', 'cells', 'it', 'assures', 'the', 'proper', 'regulation', 'of', 'gene', 'expression', 'and', 'stable', 'gene', 'silencing', 'dna', 'methylation', 'is', 'associated', 'with', 'histone', 'modifications', 'and', 'the', 'interplay', 'of', 'these', 'epigenetic', 'modifications', 'is', 'crucial', 'to', 'regulate', 'the', 'functioning', 'of', 'the', 'genome', 'by', 'changing', 'chromatin', 'architecture', 'the', 'covalent', 'addition', 'of', 'methyl', 'group', 'occurs', 'generally', 'in', 'cytosine', 'within', 'cpg', 'dinucleotides', 'which', 'are', 'concentrated', 'in', 'large', 'clusters', 'called', 'cpg', 'islands', 'dna', 'are', 'responsible', 'for', 'establishing', 'and', 'maintenance', 'of', 'methylation', 'pattern', 'it', 'is', 'commonly', 'known', 'that', 'inactivation', 'of', 'certain', 'tumor', 'suppressor', 'genes', 'occurs', 'as', 'consequence', 'of', 'within', 'the', 'promoter', 'regions', 'and', 'numerous', 'studies', 'have', 'demonstrated', 'broad', 'range', 'of', 'genes', 'silenced', 'by', 'dna', 'methylation', 'in', 'different', 'cancer', 'types', 'on', 'the', 'other', 'hand', 'global', 'hypomethylation', 'inducing', 'genomic', 'instability', 'also', 'contributes', 'to', 'cell', 'transformation', 'apart', 'from', 'dna', 'methylation', 'alterations', 'in', 'promoter', 'regions', 'and', 'repetitive', 'dna', 'sequences', 'this', 'phenomenon', 'is', 'associated', 'also', 'with', 'regulation', 'of', 'expression', 'of', 'noncoding', 'rnas', 'such', 'as', 'micrornas', 'that', 'may', 'play', 'role', 'in', 'tumor', 'suppression', 'dna', 'methylation', 'seems', 'to', 'be', 'promising', 'in', 'putative', 'translational', 'use', 'in', 'patients', 'and', 'hypermethylated', 'promoters', 'may', 'serve', 'as', 'biomarkers', 'moreover', 'unlike', 'genetic', 'alterations', 'dna', 'methylation', 'is', 'reversible', 'what', 'makes', 'it', 'extremely', 'interesting', 'for', 'therapy', 'approaches', 'the', 'importance', 'of', 'dna', 'methylation', 'alterations', 'in', 'tumorigenesis', 'encourages', 'us', 'to', 'decode', 'the', 'human', 'epigenome', 'different', 'dna', 'methylome', 'mapping', 'techniques', 'are', 'indispensable', 'to', 'realize', 'this', 'project', 'in', 'the', 'future', 'elsevier', 'inc'], ['epigenetic', 'information', 'can', 'be', 'inherited', 'through', 'the', 'mammalian', 'germline', 'and', 'represents', 'plausible', 'carrier', 'of', 'environmental', 'information', 'to', 'test', 'whether', 'inheritance', 'of', 'environmental', 'information', 'occurs', 'in', 'mammals', 'we', 'carried', 'out', 'an', 'expression', 'profiling', 'screen', 'for', 'genes', 'in', 'mice', 'that', 'responded', 'to', 'paternal', 'diet', 'offspring', 'of', 'males', 'fed', 'low', 'protein', 'diet', 'exhibited', 'elevated', 'hepatic', 'expression', 'of', 'many', 'genes', 'involved', 'in', 'lipid', 'and', 'cholesterol', 'biosynthesis', 'and', 'decreased', 'levels', 'of', 'cholesterol', 'esters', 'relative', 'to', 'the', 'offspring', 'of', 'males', 'fed', 'control', 'diet', 'epigenomic', 'profiling', 'of', 'offspring', 'livers', 'revealed', 'numerous', 'modest', 'similar', 'to', 'changes', 'in', 'cytosine', 'methylation', 'depending', 'on', 'paternal', 'diet', 'including', 'reproducible', 'changes', 'in', 'methylation', 'over', 'likely', 'enhancer', 'for', 'the', 'key', 'lipid', 'regulator', 'ppara', 'these', 'results', 'in', 'conjunction', 'with', 'recent', 'human', 'epidemiological', 'data', 'indicate', 'that', 'parental', 'diet', 'can', 'affect', 'cholesterol', 'and', 'lipid', 'metabolism', 'in', 'offspring', 'and', 'define', 'model', 'system', 'to', 'study', 'environmental', 'reprogramming', 'of', 'the', 'heritable', 'epigenome'], ['induction', 'of', 'cell', 'death', 'and', 'inhibition', 'of', 'cell', 'survival', 'are', 'the', 'main', 'principles', 'of', 'cancer', 'therapy', 'resistance', 'to', 'agents', 'is', 'major', 'problem', 'in', 'oncology', 'which', 'limits', 'the', 'effectiveness', 'of', 'anticancer', 'drugs', 'variety', 'of', 'factors', 'contribute', 'to', 'drug', 'resistance', 'including', 'host', 'factors', 'specific', 'genetic', 'or', 'epigenetic', 'alterations', 'in', 'the', 'cancer', 'cells', 'and', 'so', 'on', 'although', 'various', 'mechanisms', 'by', 'which', 'cancer', 'cells', 'become', 'resistant', 'to', 'anticancer', 'drugs', 'in', 'the', 'have', 'been', 'well', 'elucidated', 'how', 'to', 'circumvent', 'this', 'resistance', 'to', 'improve', 'anticancer', 'efficacy', 'remains', 'to', 'be', 'defined', 'autophagy', 'an', 'important', 'homeostatic', 'cellular', 'recycling', 'mechanism', 'is', 'now', 'emerging', 'as', 'crucial', 'player', 'in', 'response', 'to', 'metabolic', 'and', 'therapeutic', 'stresses', 'which', 'attempts', 'to', 'maintain', 'restore', 'metabolic', 'homeostasis', 'through', 'the', 'catabolic', 'lysis', 'of', 'excessive', 'or', 'unnecessary', 'proteins', 'and', 'injured', 'or', 'aged', 'organelles', 'recently', 'several', 'studies', 'have', 'shown', 'that', 'autophagy', 'constitutes', 'potential', 'target', 'for', 'cancer', 'therapy', 'and', 'the', 'induction', 'of', 'autophagy', 'in', 'response', 'to', 'therapeutics', 'can', 'be', 'viewed', 'as', 'having', 'prodeath', 'or', 'prosurvival', 'role', 'which', 'contributes', 'to', 'the', 'anticancer', 'efficacy', 'of', 'these', 'drugs', 'as', 'well', 'as', 'drug', 'resistance', 'thus', 'understanding', 'the', 'novel', 'function', 'of', 'autophagy', 'may', 'allow', 'us', 'to', 'develop', 'promising', 'therapeutic', 'strategy', 'to', 'enhance', 'the', 'effects', 'of', 'chemotherapy', 'and', 'improve', 'clinical', 'outcomes', 'in', 'the', 'treatment', 'of', 'cancer', 'patients'], ['dna', 'methylation', 'is', 'critical', 'epigenetic', 'regulator', 'in', 'mammalian', 'development', 'here', 'we', 'present', 'whole', 'genome', 'comparative', 'view', 'of', 'dna', 'methylation', 'using', 'bisulfite', 'sequencing', 'of', 'three', 'cultured', 'cell', 'types', 'representing', 'progressive', 'stages', 'of', 'differentiation', 'human', 'embryonic', 'stem', 'cells', 'hescs', 'fibroblastic', 'differentiated', 'derivative', 'of', 'the', 'hescs', 'and', 'neonatal', 'fibroblasts', 'as', 'reference', 'we', 'compared', 'our', 'maps', 'with', 'methylome', 'map', 'of', 'fully', 'differentiated', 'adult', 'cell', 'type', 'mature', 'peripheral', 'blood', 'mononuclear', 'cells', 'monocytes', 'we', 'observed', 'many', 'notable', 'common', 'and', 'cell', 'type', 'specific', 'features', 'among', 'all', 'cell', 'types', 'promoter', 'hypomethylation', 'both', 'cg', 'and', 'ca', 'and', 'higher', 'levels', 'of', 'gene', 'body', 'methylation', 'were', 'positively', 'correlated', 'with', 'transcription', 'in', 'all', 'cell', 'types', 'exons', 'were', 'more', 'highly', 'methylated', 'than', 'introns', 'and', 'sharp', 'transitions', 'of', 'methylation', 'occurred', 'at', 'exon', 'intron', 'boundaries', 'suggesting', 'role', 'for', 'differential', 'methylation', 'in', 'transcript', 'splicing', 'developmental', 'stage', 'was', 'reflected', 'in', 'both', 'the', 'level', 'of', 'global', 'methylation', 'and', 'extent', 'of', 'non', 'cpg', 'methylation', 'with', 'hesc', 'highest', 'fibroblasts', 'intermediate', 'and', 'monocytes', 'lowest', 'differentiation', 'associated', 'differential', 'methylation', 'profiles', 'were', 'observed', 'for', 'developmentally', 'regulated', 'genes', 'including', 'the', 'hox', 'clusters', 'other', 'homeobox', 'transcription', 'factors', 'and', 'pluripotence', 'associated', 'genes', 'such', 'as', 'pou', 'tcf', 'and', 'klf', 'our', 'results', 'highlight', 'the', 'value', 'of', 'high', 'resolution', 'methylation', 'maps', 'in', 'conjunction', 'with', 'other', 'systems', 'level', 'analyses', 'for', 'investigation', 'of', 'previously', 'undetectable', 'developmental', 'regulatory', 'mechanisms'], ['rna', 'directed', 'dna', 'methylation', 'rddm', 'is', 'the', 'major', 'small', 'rna', 'mediated', 'epigenetic', 'pathway', 'in', 'plants', 'rddm', 'requires', 'specialized', 'transcriptional', 'machinery', 'that', 'comprises', 'two', 'plant', 'specific', 'rna', 'polymerases', 'pot', 'iv', 'and', 'pot', 'and', 'growing', 'number', 'of', 'accessory', 'proteins', 'the', 'functions', 'of', 'which', 'in', 'the', 'rddm', 'mechanism', 'are', 'only', 'partially', 'understood', 'recent', 'work', 'has', 'revealed', 'variations', 'in', 'the', 'canonical', 'rddm', 'pathway', 'and', 'identified', 'factors', 'that', 'recruit', 'pot', 'iv', 'and', 'pol', 'to', 'specific', 'target', 'sequences', 'rddm', 'which', 'represses', 'subset', 'of', 'transposons', 'and', 'genes', 'is', 'implicated', 'in', 'pathogen', 'defence', 'stress', 'responses', 'and', 'reproduction', 'as', 'well', 'as', 'in', 'interallelic', 'and', 'intercellular', 'communication'], ['tumor', 'heterogeneity', 'is', 'major', 'barrier', 'to', 'effective', 'cancer', 'diagnosis', 'and', 'treatment', 'we', 'recently', 'identified', 'cancer', 'specific', 'differentially', 'dna', 'methylated', 'regions', 'cdmrs', 'in', 'colon', 'cancer', 'which', 'also', 'distinguish', 'normal', 'tissue', 'types', 'from', 'each', 'other', 'suggesting', 'that', 'these', 'cdmrs', 'might', 'be', 'generalized', 'across', 'cancer', 'types', 'here', 'we', 'show', 'stochastic', 'methylation', 'variation', 'of', 'the', 'same', 'cdmrs', 'distinguishing', 'cancer', 'from', 'normal', 'tissue', 'in', 'colon', 'lung', 'breast', 'thyroid', 'and', 'wilms', 'tumors', 'with', 'intermediate', 'variation', 'in', 'adenomas', 'whole', 'genome', 'bisulfite', 'sequencing', 'shows', 'these', 'variable', 'cdmrs', 'are', 'related', 'to', 'loss', 'of', 'sharply', 'delimited', 'methylation', 'boundaries', 'at', 'cpg', 'islands', 'furthermore', 'we', 'find', 'hypomethylation', 'of', 'discrete', 'blocks', 'encompassing', 'half', 'the', 'genome', 'with', 'extreme', 'gene', 'expression', 'variability', 'genes', 'associated', 'with', 'the', 'cdmrs', 'and', 'large', 'blocks', 'are', 'involved', 'in', 'mitosis', 'and', 'matrix', 'remodeling', 'respectively', 'we', 'suggest', 'model', 'for', 'cancer', 'involving', 'loss', 'of', 'epigenetic', 'stability', 'of', 'well', 'defined', 'genomic', 'domains', 'that', 'underlies', 'increased', 'methylation', 'variability', 'in', 'cancer', 'that', 'may', 'contribute', 'to', 'tumor', 'heterogeneity'], ['emergence', 'of', 'form', 'and', 'function', 'during', 'embryogenesis', 'arises', 'in', 'large', 'part', 'through', 'cell', 'type', 'and', 'cell', 'state', 'specific', 'variation', 'in', 'gene', 'expression', 'patterns', 'mediated', 'by', 'specialized', 'cis', 'regulatory', 'elements', 'called', 'enhancers', 'recent', 'large', 'scale', 'epigenomic', 'mapping', 'revealed', 'unexpected', 'complexity', 'and', 'dynamics', 'of', 'enhancer', 'utilization', 'patterns', 'with', 'putative', 'human', 'enhancers', 'annotated', 'by', 'the', 'encode', 'project', 'alone', 'these', 'large', 'scale', 'efforts', 'were', 'largely', 'enabled', 'through', 'the', 'understanding', 'that', 'enhancers', 'share', 'certain', 'stereotypical', 'chromatin', 'features', 'however', 'an', 'important', 'question', 'still', 'lingers', 'what', 'is', 'the', 'functional', 'significance', 'of', 'enhancer', 'chromatin', 'modification', 'here', 'we', 'give', 'an', 'overview', 'of', 'enhancer', 'associated', 'modifications', 'of', 'histones', 'and', 'dna', 'and', 'discuss', 'enzymatic', 'activities', 'involved', 'in', 'their', 'dynamic', 'deposition', 'and', 'removal', 'we', 'describe', 'potential', 'downstream', 'effectors', 'of', 'these', 'marks', 'and', 'propose', 'models', 'for', 'exploring', 'functions', 'of', 'chromatin', 'modification', 'in', 'regulating', 'enhancer', 'activity', 'during', 'development'], ['the', 'generation', 'of', 'properly', 'functioning', 'gametes', 'in', 'vitro', 'requires', 'reconstitution', 'of', 'the', 'multistepped', 'pathway', 'of', 'germ', 'cell', 'development', 'we', 'demonstrate', 'here', 'the', 'generation', 'of', 'primordial', 'germ', 'cell', 'like', 'cells', 'pgclcs', 'in', 'mice', 'with', 'robust', 'capacity', 'for', 'spermatogenesis', 'pgclcs', 'were', 'generated', 'from', 'embryonic', 'stem', 'cells', 'escs', 'and', 'induced', 'pluripotent', 'stem', 'cells', 'ipscs', 'through', 'epiblast', 'like', 'cells', 'epilcs', 'cellular', 'state', 'highly', 'similar', 'to', 'pregastrulating', 'epiblasts', 'but', 'distinct', 'from', 'epiblast', 'stem', 'cells', 'episcs', 'reflecting', 'epiblast', 'development', 'epilc', 'induction', 'from', 'escs', 'ipscs', 'is', 'progressive', 'process', 'and', 'epilcs', 'highly', 'competent', 'for', 'the', 'pgc', 'fate', 'are', 'transient', 'entity', 'the', 'global', 'transcription', 'profiles', 'epigenetic', 'reprogramming', 'and', 'cellular', 'dynamics', 'during', 'pgclc', 'induction', 'from', 'epilcs', 'meticulously', 'capture', 'those', 'associated', 'with', 'pgc', 'specification', 'from', 'the', 'epiblasts', 'furthermore', 'we', 'identify', 'integrin', 'beta', 'and', 'ssea', 'as', 'markers', 'that', 'allow', 'the', 'isolation', 'of', 'pgclcs', 'with', 'spermatogenic', 'capacity', 'from', 'tumorigenic', 'cells', 'our', 'findings', 'provide', 'paradigm', 'for', 'the', 'first', 'step', 'of', 'in', 'vitro', 'gametogenesis'], ['observational', 'epidemiological', 'studies', 'are', 'prone', 'to', 'confounding', 'reverse', 'causation', 'and', 'various', 'biases', 'and', 'have', 'generated', 'findings', 'that', 'have', 'proved', 'to', 'be', 'unreliable', 'indicators', 'of', 'the', 'causal', 'effects', 'of', 'modifiable', 'exposures', 'on', 'disease', 'outcomes', 'mendelian', 'randomization', 'mr', 'is', 'method', 'that', 'utilizes', 'genetic', 'variants', 'that', 'are', 'robustly', 'associated', 'with', 'such', 'modifiable', 'exposures', 'to', 'generate', 'more', 'reliable', 'evidence', 'regarding', 'which', 'interventions', 'should', 'produce', 'health', 'benefits', 'the', 'approach', 'is', 'being', 'widely', 'applied', 'and', 'various', 'ways', 'to', 'strengthen', 'inference', 'given', 'the', 'known', 'potential', 'limitations', 'of', 'mr', 'are', 'now', 'available', 'developments', 'of', 'mr', 'including', 'two', 'sample', 'mr', 'bidirectional', 'mr', 'network', 'mr', 'two', 'step', 'mr', 'factorial', 'mr', 'and', 'multiphenotype', 'mr', 'are', 'outlined', 'in', 'this', 'review', 'the', 'integration', 'of', 'genetic', 'information', 'into', 'population', 'based', 'epidemiological', 'studies', 'presents', 'translational', 'opportunities', 'which', 'capitalize', 'on', 'the', 'investment', 'in', 'genomic', 'discovery', 'research'], ['successful', 'reprogramming', 'of', 'differentiated', 'human', 'somatic', 'cells', 'into', 'pluripotent', 'state', 'would', 'allow', 'creation', 'of', 'patient', 'and', 'disease', 'specific', 'stem', 'cells', 'we', 'previously', 'reported', 'generation', 'of', 'induced', 'pluripotent', 'stem', 'ips', 'cells', 'capable', 'of', 'germline', 'transmission', 'from', 'mouse', 'somatic', 'cells', 'by', 'transduction', 'of', 'four', 'defined', 'transcription', 'factors', 'here', 'we', 'demonstrate', 'the', 'generation', 'of', 'ips', 'cells', 'from', 'adult', 'human', 'dermal', 'fibroblasts', 'with', 'the', 'same', 'four', 'factors', 'oct', 'sox', 'klf', 'and', 'myc', 'human', 'ips', 'cells', 'were', 'similar', 'to', 'human', 'embryonic', 'stem', 'es', 'cells', 'in', 'morphology', 'proliferation', 'surface', 'antigens', 'gene', 'expression', 'epigenetic', 'status', 'of', 'pluripotent', 'cell', 'specific', 'genes', 'and', 'telomerase', 'activity', 'furthermore', 'these', 'cells', 'could', 'differentiate', 'into', 'cell', 'types', 'of', 'the', 'three', 'germ', 'layers', 'in', 'vitro', 'and', 'in', 'teratomas', 'these', 'findings', 'demonstrate', 'that', 'ips', 'cells', 'can', 'be', 'generated', 'from', 'adult', 'human', 'fibroblasts'], ['the', 'most', 'highly', 'conserved', 'noncoding', 'elements', 'hcnes', 'in', 'mammalian', 'genomes', 'cluster', 'within', 'regions', 'enriched', 'for', 'genes', 'encoding', 'developmentally', 'important', 'transcription', 'factors', 'tfs', 'this', 'suggests', 'that', 'hcne', 'rich', 'regions', 'may', 'contain', 'key', 'regulatory', 'controls', 'involved', 'in', 'development', 'we', 'explored', 'this', 'by', 'examining', 'histone', 'methylation', 'in', 'mouse', 'embryonic', 'stem', 'es', 'cells', 'across', 'large', 'hcne', 'rich', 'loci', 'we', 'identified', 'specific', 'modification', 'pattern', 'termed', 'bivalent', 'domains', 'consisting', 'of', 'large', 'regions', 'of', 'lysine', 'methylation', 'harboring', 'smaller', 'regions', 'of', 'lysine', 'methylation', 'bivalent', 'domains', 'tend', 'to', 'coincide', 'with', 'tf', 'genes', 'expressed', 'at', 'low', 'levels', 'we', 'propose', 'that', 'bivalent', 'domains', 'silence', 'developmental', 'genes', 'in', 'es', 'cells', 'while', 'keeping', 'them', 'poised', 'for', 'activation', 'we', 'also', 'found', 'striking', 'correspondences', 'between', 'genome', 'sequence', 'and', 'histone', 'methylation', 'in', 'es', 'cells', 'which', 'become', 'notably', 'weaker', 'in', 'differentiated', 'cells', 'these', 'results', 'highlight', 'the', 'importance', 'of', 'dna', 'sequence', 'in', 'defining', 'the', 'initial', 'epigenetic', 'landscape', 'and', 'suggest', 'novel', 'chromatin', 'based', 'mechanism', 'for', 'maintaining', 'pluripotency'], ['rna', 'is', 'not', 'only', 'messenger', 'operating', 'between', 'dna', 'and', 'protein', 'transcription', 'of', 'essentially', 'the', 'entire', 'eukaryotic', 'genome', 'generates', 'myriad', 'of', 'non', 'protein', 'coding', 'rna', 'species', 'that', 'show', 'complex', 'overlapping', 'patterns', 'of', 'expression', 'and', 'regulation', 'although', 'long', 'noncoding', 'rnas', 'lncrnas', 'are', 'among', 'the', 'least', 'well', 'understood', 'of', 'these', 'transcript', 'species', 'they', 'cannot', 'all', 'be', 'dismissed', 'as', 'merely', 'transcriptional', 'noise', 'here', 'we', 'review', 'the', 'evolution', 'of', 'lncrnas', 'and', 'their', 'roles', 'in', 'transcriptional', 'regulation', 'epigenetic', 'gene', 'regulation', 'and', 'disease'], ['noncoding', 'rnas', 'ncrna', 'participate', 'in', 'epigenetic', 'regulation', 'but', 'are', 'poorly', 'understood', 'here', 'we', 'characterize', 'the', 'transcriptional', 'landscape', 'of', 'the', 'four', 'human', 'hox', 'loci', 'at', 'five', 'base', 'pair', 'resolution', 'in', 'anatomic', 'sites', 'and', 'identify', 'hox', 'ncrnas', 'that', 'extend', 'known', 'transcribed', 'regions', 'by', 'more', 'than', 'kilobases', 'hox', 'ncrnas', 'are', 'spatially', 'expressed', 'along', 'developmental', 'axes', 'and', 'possess', 'unique', 'sequence', 'motifs', 'and', 'their', 'expression', 'demarcates', 'broad', 'chromosomal', 'domains', 'of', 'differential', 'histone', 'methylation', 'and', 'rna', 'polymerase', 'accessibility', 'we', 'identified', 'kilobase', 'ncrna', 'residing', 'in', 'the', 'hoxc', 'locus', 'termed', 'hotair', 'which', 'represses', 'transcription', 'in', 'trans', 'across', 'kilobases', 'of', 'the', 'hoxd', 'locus', 'hotair', 'interacts', 'with', 'polycomb', 'repressive', 'complex', 'prc', 'and', 'is', 'required', 'for', 'prc', 'occupancy', 'and', 'histone', 'lysine', 'trimethylation', 'of', 'hoxd', 'locus', 'thus', 'transcription', 'of', 'ncrna', 'may', 'demarcate', 'chromosomal', 'domains', 'of', 'gene', 'silencing', 'at', 'distance', 'these', 'results', 'have', 'broad', 'implications', 'for', 'gene', 'regulation', 'in', 'development', 'and', 'disease', 'states'], ['over', 'the', 'past', 'several', 'years', 'it', 'has', 'become', 'clear', 'that', 'alterations', 'in', 'the', 'expression', 'of', 'microrna', 'mirna', 'genes', 'contribute', 'to', 'the', 'pathogenesis', 'of', 'most', 'if', 'not', 'all', 'human', 'malignancies', 'these', 'alterations', 'can', 'be', 'caused', 'by', 'various', 'mechanisms', 'including', 'deletions', 'amplifications', 'or', 'mutations', 'involving', 'mirna', 'loci', 'epigenetic', 'silencing', 'or', 'the', 'dysregulation', 'of', 'transcription', 'factors', 'that', 'target', 'specific', 'mirnas', 'because', 'malignant', 'cells', 'show', 'dependence', 'on', 'the', 'dysregulated', 'expression', 'of', 'mirna', 'genes', 'which', 'in', 'turn', 'control', 'or', 'are', 'controlled', 'by', 'the', 'dysregulation', 'of', 'multiple', 'protein', 'coding', 'oncogenes', 'or', 'tumour', 'suppressor', 'genes', 'these', 'small', 'rnas', 'provide', 'important', 'opportunities', 'for', 'the', 'development', 'of', 'future', 'mirna', 'based', 'therapies'], ['the', 'genomes', 'of', 'many', 'animals', 'plants', 'and', 'fungi', 'are', 'tagged', 'by', 'methylation', 'of', 'dna', 'cytosine', 'to', 'understand', 'the', 'biological', 'significance', 'of', 'this', 'epigenetic', 'mark', 'it', 'is', 'essential', 'to', 'know', 'where', 'in', 'the', 'genome', 'it', 'is', 'located', 'new', 'techniques', 'are', 'making', 'it', 'easier', 'to', 'map', 'dna', 'methylation', 'patterns', 'on', 'large', 'scale', 'and', 'the', 'results', 'have', 'already', 'provided', 'surprises', 'in', 'particular', 'the', 'conventional', 'view', 'that', 'dna', 'methylation', 'functions', 'predominantly', 'to', 'irreversibly', 'silence', 'transcription', 'is', 'being', 'challenged', 'not', 'only', 'is', 'promoter', 'methylation', 'often', 'highly', 'dynamic', 'during', 'development', 'but', 'many', 'organisms', 'also', 'seem', 'to', 'target', 'dna', 'methylation', 'specifically', 'to', 'the', 'bodies', 'of', 'active', 'genes'], ['dna', 'methylation', 'is', 'crucial', 'epigenetic', 'modification', 'of', 'the', 'genome', 'that', 'is', 'involved', 'in', 'regulating', 'many', 'cellular', 'processes', 'these', 'include', 'embryonic', 'development', 'transcription', 'chromatin', 'structure', 'chromosome', 'inactivation', 'genomic', 'imprinting', 'and', 'chromosome', 'stability', 'consistent', 'with', 'these', 'important', 'roles', 'growing', 'number', 'of', 'human', 'diseases', 'have', 'been', 'found', 'to', 'be', 'associated', 'with', 'aberrant', 'dna', 'methylation', 'the', 'study', 'of', 'these', 'diseases', 'has', 'provided', 'new', 'and', 'fundamental', 'insights', 'into', 'the', 'roles', 'that', 'dna', 'methylation', 'and', 'other', 'epigenetic', 'modifications', 'have', 'in', 'development', 'and', 'normal', 'cellular', 'homeostasis'], ['methylation', 'of', 'dna', 'at', 'position', 'five', 'of', 'the', 'cytosine', 'ring', 'occurs', 'at', 'most', 'cpg', 'dinucleotides', 'in', 'the', 'mammalian', 'genome', 'and', 'is', 'essential', 'for', 'embryonic', 'viability', 'with', 'several', 'of', 'the', 'key', 'proteins', 'now', 'known', 'it', 'has', 'become', 'possible', 'to', 'approach', 'the', 'biological', 'significance', 'of', 'this', 'epigenetic', 'system', 'through', 'both', 'biochemistry', 'and', 'genetics', 'as', 'result', 'advances', 'have', 'been', 'made', 'in', 'our', 'understanding', 'of', 'the', 'mechanisms', 'by', 'which', 'dna', 'methylation', 'is', 'targeted', 'to', 'specific', 'regions', 'of', 'the', 'genome', 'and', 'interpreted', 'by', 'methyl', 'cpg', 'binding', 'proteins', 'recent', 'studies', 'have', 'illuminated', 'the', 'role', 'of', 'dna', 'methylation', 'in', 'controlling', 'gene', 'expression', 'and', 'have', 'strengthened', 'its', 'links', 'with', 'histone', 'modification', 'and', 'chromatin', 'remodelling'], ['epidemiological', 'evidence', 'increasingly', 'suggests', 'that', 'environmental', 'exposures', 'early', 'in', 'development', 'have', 'role', 'in', 'susceptibility', 'to', 'disease', 'in', 'later', 'life', 'in', 'addition', 'some', 'of', 'these', 'environmental', 'effects', 'seem', 'to', 'be', 'passed', 'on', 'through', 'subsequent', 'generations', 'epigenetic', 'modifications', 'provide', 'plausible', 'link', 'between', 'the', 'environment', 'and', 'alterations', 'in', 'gene', 'expression', 'that', 'might', 'lead', 'to', 'disease', 'phenotypes', 'an', 'increasing', 'body', 'of', 'evidence', 'from', 'animal', 'studies', 'supports', 'the', 'role', 'of', 'environmental', 'epigenetics', 'in', 'disease', 'susceptibility', 'furthermore', 'recent', 'studies', 'have', 'demonstrated', 'for', 'the', 'first', 'time', 'that', 'heritable', 'environmentally', 'induced', 'epigenetic', 'modifications', 'underlie', 'reversible', 'alterations', 'in', 'phenotype', 'methods', 'are', 'now', 'becoming', 'available', 'to', 'investigate', 'the', 'relevance', 'of', 'these', 'phenomena', 'to', 'human', 'disease'], ['deciphering', 'the', 'multiple', 'layers', 'of', 'epigenetic', 'regulation', 'that', 'control', 'transcription', 'is', 'critical', 'to', 'understanding', 'how', 'plants', 'develop', 'and', 'respond', 'to', 'their', 'environment', 'using', 'sequencing', 'by', 'synthesis', 'technology', 'we', 'directly', 'sequenced', 'the', 'cytosine', 'methylome', 'methylc', 'seq', 'transcriptome', 'mrna', 'seq', 'and', 'small', 'rna', 'transcriptome', 'smrna', 'seq', 'to', 'generate', 'highly', 'integrated', 'epigenome', 'maps', 'for', 'wild', 'type', 'arabidopsis', 'thaliana', 'and', 'mutants', 'defective', 'in', 'dna', 'or', 'demethylase', 'activity', 'at', 'singlebase', 'resolution', 'we', 'discovered', 'extensive', 'previously', 'undetected', 'dna', 'methylation', 'identified', 'the', 'context', 'and', 'level', 'of', 'methylation', 'at', 'each', 'site', 'and', 'observed', 'local', 'sequence', 'effects', 'upon', 'methylation', 'state', 'deep', 'sequencing', 'of', 'smrnas', 'revealed', 'direct', 'relationship', 'between', 'the', 'location', 'of', 'smrnas', 'and', 'dna', 'methylation', 'perturbation', 'of', 'smrna', 'biogenesis', 'upon', 'loss', 'of', 'cpg', 'dna', 'methylation', 'and', 'tendency', 'for', 'smrnas', 'to', 'direct', 'strand', 'specific', 'dna', 'methylation', 'in', 'regions', 'of', 'rna', 'dna', 'homology', 'finally', 'strandspecific', 'mrna', 'seq', 'revealed', 'altered', 'transcript', 'abundance', 'of', 'hundreds', 'of', 'genes', 'transposons', 'and', 'unannotated', 'intergenic', 'transcripts', 'upon', 'modification', 'of', 'the', 'dna', 'methylation', 'state'], ['plants', 'and', 'animals', 'use', 'small', 'rnas', 'micrornas', 'mirnas', 'and', 'sirnas', 'as', 'guides', 'for', 'and', 'epigenetic', 'regulation', 'in', 'plants', 'mirnas', 'and', 'trans', 'acting', 'ta', 'sirnas', 'form', 'through', 'distinct', 'biogenesis', 'pathways', 'although', 'they', 'both', 'interact', 'with', 'target', 'transcripts', 'and', 'guide', 'cleavage', 'an', 'integrated', 'approach', 'to', 'identify', 'targets', 'of', 'arabidopsis', 'thaliana', 'mirnas', 'and', 'ta', 'sirnas', 'revealed', 'several', 'new', 'classes', 'of', 'small', 'rna', 'regulated', 'genes', 'including', 'conventional', 'genes', 'such', 'as', 'argonaute', 'and', 'an', 'ubiquitin', 'conjugating', 'enzyme', 'surprisingly', 'five', 'tasirna', 'generating', 'transcripts', 'were', 'identified', 'as', 'targets', 'of', 'mir', 'or', 'mir', 'rather', 'than', 'functioning', 'as', 'negative', 'regulators', 'mir', 'and', 'mir', 'guided', 'cleavage', 'was', 'shown', 'to', 'set', 'the', 'nucleotide', 'phase', 'for', 'ta', 'sirna', 'precursor', 'processing', 'these', 'data', 'support', 'model', 'in', 'which', 'mirna', 'guided', 'formation', 'of', 'or', 'terminus', 'within', 'pre', 'ta', 'sirna', 'transcripts', 'followed', 'by', 'rdr', 'dependent', 'formation', 'of', 'dsrna', 'and', 'dicer', 'like', 'processing', 'yields', 'phased', 'ta', 'sirnas', 'that', 'negatively', 'regulate', 'other', 'genes'], ['for', 'the', 'past', 'years', 'it', 'has', 'been', 'known', 'that', 'alterations', 'in', 'dna', 'methylation', 'dnam', 'occur', 'in', 'cancer', 'including', 'hypomethylation', 'of', 'oncogenes', 'and', 'of', 'tumor', 'suppressor', 'genes', 'however', 'most', 'studies', 'of', 'cancer', 'methylation', 'have', 'assumed', 'that', 'functionally', 'important', 'dnam', 'will', 'occur', 'in', 'promoters', 'and', 'that', 'most', 'dnam', 'changes', 'in', 'cancer', 'occur', 'in', 'cpg', 'islands', 'here', 'we', 'show', 'that', 'most', 'methylation', 'alterations', 'in', 'colon', 'cancer', 'occur', 'not', 'in', 'promoters', 'and', 'also', 'not', 'in', 'cpg', 'islands', 'but', 'in', 'sequences', 'up', 'to', 'kb', 'distant', 'which', 'we', 'term', 'cpg', 'island', 'shores', 'cpg', 'island', 'shore', 'methylation', 'was', 'strongly', 'related', 'to', 'gene', 'expression', 'and', 'it', 'was', 'highly', 'conserved', 'in', 'mouse', 'discriminating', 'tissue', 'types', 'regardless', 'of', 'species', 'of', 'origin', 'there', 'was', 'notable', 'overlap', 'of', 'the', 'locations', 'of', 'colon', 'cancer', 'related', 'methylation', 'changes', 'with', 'those', 'that', 'distinguished', 'normal', 'tissues', 'with', 'enriched', 'closer', 'to', 'the', 'associated', 'cpg', 'islands', 'and', 'hypomethylation', 'enriched', 'further', 'from', 'the', 'associated', 'cpg', 'island', 'and', 'resembling', 'that', 'of', 'noncolon', 'normal', 'tissues', 'thus', 'methylation', 'changes', 'in', 'cancer', 'are', 'at', 'sites', 'that', 'vary', 'normally', 'in', 'tissue', 'differentiation', 'consistent', 'with', 'the', 'epigenetic', 'progenitor', 'model', 'of', 'cancer', 'which', 'proposes', 'that', 'epigenetic', 'alterations', 'affecting', 'tissue', 'specific', 'differentiation', 'are', 'the', 'predominant', 'mechanism', 'by', 'which', 'epigenetic', 'changes', 'cause', 'cancer'], ['an', 'altered', 'pattern', 'of', 'epigenetic', 'modifications', 'is', 'central', 'to', 'many', 'common', 'human', 'diseases', 'including', 'cancer', 'many', 'studies', 'have', 'explored', 'the', 'mosaic', 'patterns', 'of', 'dna', 'methylation', 'and', 'histone', 'modification', 'in', 'cancer', 'cells', 'on', 'gene', 'by', 'gene', 'basis', 'among', 'their', 'results', 'has', 'been', 'the', 'seminal', 'finding', 'of', 'transcriptional', 'silencing', 'of', 'tumour', 'suppressor', 'genes', 'by', 'cpg', 'island', 'promoter', 'however', 'recent', 'technological', 'advances', 'are', 'now', 'allowing', 'cancer', 'epigenetics', 'to', 'be', 'studied', 'genome', 'wide', 'an', 'approach', 'that', 'has', 'already', 'begun', 'to', 'provide', 'both', 'biological', 'insight', 'and', 'new', 'avenues', 'for', 'translational', 'research', 'it', 'is', 'time', 'to', 'upgrade', 'cancer', 'epigenetics', 'research', 'and', 'put', 'together', 'an', 'ambitious', 'plan', 'to', 'tackle', 'the', 'many', 'unanswered', 'questions', 'in', 'this', 'field', 'using', 'epigenomics', 'approaches'], ['the', 'term', 'non', 'coding', 'rna', 'ncrna', 'is', 'commonly', 'employed', 'for', 'rna', 'that', 'does', 'not', 'encode', 'protein', 'but', 'this', 'does', 'not', 'mean', 'that', 'such', 'rnas', 'do', 'not', 'contain', 'information', 'nor', 'have', 'function', 'although', 'it', 'has', 'been', 'generally', 'assumed', 'that', 'most', 'genetic', 'information', 'is', 'transacted', 'by', 'proteins', 'recent', 'evidence', 'suggests', 'that', 'the', 'majority', 'of', 'the', 'genomes', 'of', 'mammals', 'and', 'other', 'complex', 'organisms', 'is', 'in', 'fact', 'transcribed', 'into', 'ncrnas', 'many', 'of', 'which', 'are', 'alternatively', 'spliced', 'and', 'or', 'processed', 'into', 'smaller', 'products', 'these', 'ncrnas', 'include', 'micrornas', 'and', 'snornas', 'many', 'if', 'not', 'most', 'of', 'which', 'remain', 'to', 'be', 'identified', 'as', 'well', 'as', 'likely', 'other', 'classes', 'of', 'yet', 'to', 'be', 'discovered', 'small', 'regulatory', 'rnas', 'and', 'tens', 'of', 'thousands', 'of', 'longer', 'transcripts', 'including', 'complex', 'patterns', 'of', 'interlacing', 'and', 'overlapping', 'sense', 'and', 'antisense', 'transcripts', 'most', 'of', 'whose', 'functions', 'are', 'unknown', 'these', 'rnas', 'including', 'those', 'derived', 'from', 'introns', 'appear', 'to', 'comprise', 'hidden', 'layer', 'of', 'internal', 'signals', 'that', 'control', 'various', 'levels', 'of', 'gene', 'expression', 'in', 'physiology', 'and', 'development', 'including', 'chromatin', 'architecture', 'epigenetic', 'memory', 'transcription', 'rna', 'splicing', 'editing', 'translation', 'and', 'turnover', 'rna', 'regulatory', 'networks', 'may', 'determine', 'most', 'of', 'our', 'complex', 'characteristics', 'play', 'significant', 'role', 'in', 'disease', 'and', 'constitute', 'an', 'unexplored', 'world', 'of', 'genetic', 'variation', 'both', 'within', 'and', 'between', 'species'], [], ['chromosome', 'inactivation', 'is', 'the', 'mammalian', 'dosage', 'compensation', 'mechanism', 'by', 'which', 'transcription', 'of', 'linked', 'genes', 'is', 'equalized', 'between', 'females', 'and', 'males', 'in', 'an', 'ethyl', 'nitrosourea', 'enu', 'mutagenesis', 'screen', 'on', 'mice', 'for', 'modifiers', 'of', 'epigenetic', 'reprogramming', 'we', 'identified', 'the', 'mommed', 'modifier', 'of', 'murine', 'metastable', 'epialleles', 'mutation', 'as', 'semidominant', 'suppressor', 'of', 'variegation', 'mommed', 'shows', 'homozygous', 'female', 'specific', 'mid', 'gestation', 'lethality', 'and', 'hypomethylation', 'of', 'the', 'linked', 'gene', 'hprt', 'suggestive', 'of', 'defect', 'in', 'inactivation', 'here', 'we', 'report', 'that', 'the', 'causative', 'point', 'mutation', 'lies', 'in', 'previously', 'uncharacterized', 'gene', 'smchd', 'structural', 'maintenance', 'of', 'chromosomes', 'hinge', 'domain', 'containing', 'we', 'find', 'that', 'smchd', 'is', 'not', 'required', 'for', 'correct', 'xist', 'expression', 'but', 'localizes', 'to', 'the', 'inactive', 'and', 'has', 'role', 'in', 'the', 'maintenance', 'of', 'inactivation', 'and', 'the', 'of', 'cpg', 'islands', 'associated', 'with', 'the', 'inactive', 'this', 'finding', 'links', 'group', 'of', 'proteins', 'normally', 'associated', 'with', 'structural', 'aspects', 'of', 'chromosome', 'biology', 'with', 'epigenetic', 'gene', 'silencing'], [], ['the', 'packaging', 'of', 'chromosomal', 'dna', 'by', 'nucleosomes', 'condenses', 'and', 'organizes', 'the', 'genome', 'but', 'occludes', 'many', 'regulatory', 'dna', 'elements', 'however', 'this', 'constraint', 'also', 'allows', 'nucleosomes', 'and', 'other', 'chromatin', 'components', 'to', 'actively', 'participate', 'in', 'the', 'regulation', 'of', 'transcription', 'chromosome', 'segregation', 'dna', 'replication', 'and', 'dna', 'repair', 'to', 'enable', 'dynamic', 'access', 'to', 'packaged', 'dna', 'and', 'to', 'tailor', 'nucleosome', 'composition', 'in', 'chromosomal', 'regions', 'cells', 'have', 'evolved', 'set', 'of', 'specialized', 'chromatin', 'remodeling', 'complexes', 'remodelers', 'remodelers', 'use', 'the', 'energy', 'of', 'atp', 'hydrolysis', 'to', 'move', 'destabilize', 'eject', 'or', 'restructure', 'nucleosomes', 'here', 'we', 'address', 'many', 'aspects', 'of', 'remodeler', 'biology', 'their', 'targeting', 'mechanism', 'regulation', 'shared', 'and', 'unique', 'properties', 'and', 'specialization', 'for', 'particular', 'biological', 'processes', 'we', 'also', 'address', 'roles', 'for', 'remodelers', 'in', 'development', 'cancer', 'and', 'human', 'syndromes'], ['ectopic', 'expression', 'of', 'the', 'four', 'transcription', 'factors', 'oct', 'sox', 'myc', 'and', 'klf', 'is', 'sufficient', 'to', 'confer', 'pluripotent', 'state', 'upon', 'the', 'fibroblast', 'genome', 'generating', 'induced', 'pluripotent', 'stem', 'ps', 'cells', 'it', 'remains', 'unknown', 'if', 'nuclear', 'reprogramming', 'induced', 'by', 'these', 'four', 'factors', 'globally', 'resets', 'epigenetic', 'differences', 'between', 'differentiated', 'and', 'pluripotent', 'cells', 'here', 'using', 'novel', 'selection', 'approaches', 'we', 'have', 'generated', 'ps', 'cells', 'from', 'fibroblasts', 'to', 'characterize', 'their', 'epigenetic', 'state', 'female', 'ps', 'cells', 'showed', 'reactivation', 'of', 'somatically', 'silenced', 'chromosome', 'and', 'underwent', 'random', 'inactivation', 'upon', 'differentiation', 'genome', 'wide', 'analysis', 'of', 'two', 'key', 'histone', 'modifications', 'indicated', 'that', 'ps', 'cells', 'are', 'highly', 'similar', 'to', 'es', 'cells', 'consistent', 'with', 'these', 'observations', 'ps', 'cells', 'gave', 'rise', 'to', 'viable', 'high', 'degree', 'chimeras', 'with', 'contribution', 'to', 'the', 'germline', 'these', 'data', 'show', 'that', 'transcription', 'factor', 'induced', 'reprogramming', 'leads', 'to', 'the', 'global', 'reversion', 'of', 'the', 'somatic', 'epigenome', 'into', 'an', 'es', 'like', 'state', 'our', 'results', 'provide', 'paradigm', 'for', 'studying', 'the', 'epigenetic', 'modifications', 'that', 'accompany', 'nuclear', 'reprogramming', 'and', 'suggest', 'that', 'abnormal', 'epigenetic', 'reprogramming', 'does', 'not', 'pose', 'problem', 'for', 'the', 'potential', 'therapeutic', 'applications', 'of', 'ps', 'cells'], ['cancer', 'is', 'widely', 'perceived', 'as', 'heterogeneous', 'group', 'of', 'disorders', 'with', 'markedly', 'different', 'biological', 'properties', 'which', 'are', 'caused', 'by', 'series', 'of', 'clonally', 'selected', 'genetic', 'changes', 'in', 'key', 'tumour', 'suppressor', 'genes', 'and', 'oncogenes', 'however', 'recent', 'data', 'suggest', 'that', 'cancer', 'has', 'fundamentally', 'common', 'basis', 'that', 'is', 'grounded', 'in', 'polyclonal', 'epigenetic', 'disruption', 'of', 'stem', 'progenitor', 'cells', 'mediated', 'by', 'tumour', 'progenitor', 'genes', 'furthermore', 'tumour', 'cell', 'heterogeneity', 'is', 'due', 'in', 'part', 'to', 'epigenetic', 'variation', 'in', 'progenitor', 'cells', 'and', 'epigenetic', 'plasticity', 'together', 'with', 'genetic', 'lesions', 'drives', 'tumour', 'progression', 'this', 'crucial', 'early', 'role', 'for', 'epigenetic', 'alterations', 'in', 'cancer', 'is', 'in', 'addition', 'to', 'epigenetic', 'alterations', 'that', 'can', 'substitute', 'for', 'genetic', 'variation', 'later', 'in', 'tumour', 'progression', 'therefore', 'non', 'neoplastic', 'but', 'epigenetically', 'disrupted', 'stem', 'progenitor', 'cells', 'might', 'be', 'crucial', 'target', 'for', 'cancer', 'risk', 'assessment', 'and', 'chemoprevention'], ['polyploids', 'organisms', 'that', 'have', 'multiple', 'sets', 'of', 'chromosomes', 'are', 'common', 'in', 'certain', 'plant', 'and', 'animal', 'taxa', 'and', 'can', 'be', 'surprisingly', 'stable', 'the', 'evidence', 'that', 'has', 'emerged', 'from', 'genome', 'analyses', 'also', 'indicates', 'that', 'many', 'other', 'eukaryotic', 'genomes', 'have', 'polyploid', 'ancestry', 'suggesting', 'that', 'both', 'humans', 'and', 'most', 'other', 'eukaryotes', 'have', 'either', 'benefited', 'from', 'or', 'endured', 'polyploidy', 'studies', 'of', 'polyploids', 'soon', 'after', 'their', 'formation', 'have', 'revealed', 'genetic', 'and', 'epigenetic', 'interactions', 'between', 'redundant', 'genes', 'these', 'interactions', 'can', 'be', 'related', 'to', 'the', 'phenotypes', 'and', 'evolutionary', 'fates', 'of', 'polyploids', 'here', 'consider', 'the', 'advantages', 'and', 'challenges', 'of', 'polyploidy', 'and', 'its', 'evolutionary', 'potential'], ['cpg', 'island', 'and', 'global', 'genomic', 'hypomethylation', 'are', 'common', 'epigenetic', 'features', 'of', 'cancer', 'cells', 'less', 'attention', 'has', 'been', 'focused', 'on', 'histone', 'modifications', 'in', 'cancer', 'cells', 'we', 'characterized', 'post', 'translational', 'modifications', 'to', 'histone', 'in', 'comprehensive', 'panel', 'of', 'normal', 'tissues', 'cancer', 'cell', 'lines', 'and', 'primary', 'tumors', 'using', 'immunodetection', 'high', 'performance', 'capillary', 'electrophoresis', 'and', 'mass', 'spectrometry', 'we', 'found', 'that', 'cancer', 'cells', 'had', 'loss', 'of', 'monoacetylated', 'and', 'trimethylated', 'forms', 'of', 'histone', 'these', 'changes', 'appeared', 'early', 'and', 'accumulated', 'during', 'the', 'tumorigenic', 'process', 'as', 'we', 'showed', 'in', 'mouse', 'model', 'of', 'multistage', 'skin', 'carcinogenesis', 'the', 'losses', 'occurred', 'predominantly', 'at', 'the', 'acetylated', 'lys', 'and', 'trimethylated', 'lys', 'residues', 'of', 'histone', 'and', 'were', 'associated', 'with', 'the', 'hypomethylation', 'of', 'dna', 'repetitive', 'sequences', 'well', 'known', 'characteristic', 'of', 'cancer', 'cells', 'our', 'data', 'suggest', 'that', 'the', 'global', 'loss', 'of', 'monoacetylation', 'and', 'trimethylation', 'of', 'histone', 'is', 'common', 'hallmark', 'of', 'human', 'tumor', 'cells'], ['cytosine', 'methylation', 'is', 'required', 'for', 'mammalian', 'development', 'and', 'is', 'often', 'perturbed', 'in', 'human', 'cancer', 'to', 'determine', 'how', 'this', 'epigenetic', 'modification', 'is', 'distributed', 'in', 'the', 'genomes', 'of', 'primary', 'and', 'transformed', 'cells', 'we', 'used', 'an', 'immunocapturing', 'approach', 'followed', 'by', 'dna', 'microarray', 'analysis', 'to', 'generate', 'methylation', 'profiles', 'of', 'all', 'human', 'chromosomes', 'at', 'kb', 'resolution', 'and', 'for', 'large', 'set', 'of', 'cpg', 'islands', 'in', 'primary', 'cells', 'we', 'identified', 'broad', 'genomic', 'regions', 'of', 'differential', 'methylation', 'with', 'higher', 'levels', 'in', 'gene', 'rich', 'neighborhoods', 'female', 'and', 'male', 'cells', 'had', 'profiles', 'for', 'autosomes', 'but', 'differences', 'on', 'the', 'chromosome', 'the', 'inactive', 'chromosome', 'xi', 'was', 'hypermethylated', 'at', 'only', 'subset', 'of', 'gene', 'rich', 'regions', 'and', 'unexpectedly', 'overall', 'hypomethylated', 'relative', 'to', 'its', 'active', 'counterpart', 'the', 'chromosomal', 'methylation', 'profile', 'of', 'transformed', 'cells', 'was', 'similar', 'to', 'that', 'of', 'primary', 'cells', 'nevertheless', 'we', 'detected', 'large', 'genomic', 'segments', 'with', 'hypomethylation', 'in', 'the', 'transformed', 'cell', 'residing', 'in', 'gene', 'poor', 'areas', 'furthermore', 'analysis', 'of', 'cpg', 'islands', 'showed', 'that', 'only', 'small', 'set', 'of', 'promoters', 'was', 'methylated', 'differentially', 'suggesting', 'that', 'aberrant', 'methylation', 'of', 'cpg', 'island', 'promoters', 'in', 'malignancy', 'might', 'be', 'less', 'frequent', 'than', 'previously', 'hypothesized'], ['inheritance', 'requires', 'genome', 'duplication', 'reproduction', 'of', 'chromatin', 'and', 'its', 'epigenetic', 'information', 'mechanisms', 'to', 'ensure', 'genome', 'integrity', 'and', 'faithful', 'transmission', 'of', 'the', 'information', 'to', 'progeny', 'proliferating', 'cell', 'nuclear', 'antigen', 'pcna', 'cofactor', 'of', 'dna', 'polymerases', 'that', 'encircles', 'dna', 'orchestrates', 'several', 'of', 'these', 'functions', 'by', 'recruiting', 'crucial', 'players', 'to', 'the', 'replication', 'fork', 'remarkably', 'many', 'factors', 'that', 'are', 'involved', 'in', 'replication', 'linked', 'processes', 'interact', 'with', 'particular', 'face', 'of', 'pcna', 'and', 'through', 'the', 'same', 'interaction', 'domain', 'indicating', 'that', 'these', 'interactions', 'do', 'not', 'occur', 'simultaneously', 'during', 'replication', 'switching', 'of', 'pcna', 'partners', 'may', 'be', 'triggered', 'by', 'affinity', 'driven', 'competition', 'phosphorylation', 'proteolysis', 'and', 'modification', 'of', 'pcna', 'by', 'ubiquitin', 'and', 'sumo'], ['cytosine', 'methylation', 'is', 'important', 'for', 'transposon', 'silencing', 'and', 'epigenetic', 'regulation', 'of', 'endogenous', 'genes', 'although', 'the', 'extent', 'to', 'which', 'this', 'dna', 'modification', 'functions', 'to', 'regulate', 'the', 'genome', 'is', 'still', 'unknown', 'here', 'we', 'report', 'the', 'first', 'comprehensive', 'dna', 'methylation', 'map', 'of', 'an', 'entire', 'genome', 'at', 'base', 'pair', 'resolution', 'using', 'the', 'flowering', 'plant', 'arabidopsis', 'thaliana', 'as', 'model', 'we', 'find', 'that', 'pericentromeric', 'heterochromatin', 'repetitive', 'sequences', 'and', 'regions', 'producing', 'small', 'interfering', 'rnas', 'are', 'heavily', 'methylated', 'unexpectedly', 'over', 'one', 'third', 'of', 'expressed', 'genes', 'contain', 'methylation', 'within', 'transcribed', 'regions', 'whereas', 'only', 'similar', 'to', 'of', 'genes', 'show', 'methylation', 'within', 'promoter', 'regions', 'interestingly', 'genes', 'methylated', 'in', 'transcribed', 'regions', 'are', 'highly', 'expressed', 'and', 'constitutively', 'active', 'whereas', 'promoter', 'methylated', 'genes', 'show', 'greater', 'degree', 'of', 'tissue', 'specific', 'expression', 'whole', 'genome', 'tiling', 'array', 'transcriptional', 'profiling', 'of', 'dna', 'null', 'mutants', 'identified', 'hundreds', 'of', 'genes', 'and', 'intergenic', 'noncoding', 'rnas', 'with', 'altered', 'expression', 'levels', 'many', 'of', 'which', 'may', 'be', 'epigenetically', 'controlled', 'by', 'dna', 'methylation'], ['overlapping', 'epigenetic', 'mechanisms', 'have', 'evolved', 'in', 'eukaryotic', 'cells', 'to', 'silence', 'the', 'expression', 'and', 'mobility', 'of', 'transposable', 'elements', 'tes', 'owing', 'to', 'their', 'ability', 'to', 'recruit', 'the', 'silencing', 'machinery', 'tes', 'have', 'served', 'as', 'building', 'blocks', 'for', 'epigenetic', 'phenomena', 'both', 'at', 'the', 'level', 'of', 'single', 'genes', 'and', 'across', 'larger', 'chromosomal', 'regions', 'important', 'progress', 'has', 'been', 'made', 'recently', 'in', 'understanding', 'these', 'silencing', 'mechanisms', 'in', 'addition', 'new', 'insights', 'have', 'been', 'gained', 'into', 'how', 'this', 'silencing', 'has', 'been', 'co', 'opted', 'to', 'serve', 'essential', 'functions', 'in', 'host', 'cells', 'highlighting', 'the', 'importance', 'of', 'tes', 'in', 'the', 'epigenetic', 'regulation', 'of', 'the', 'genome'], ['cellular', 'senescence', 'state', 'of', 'irreversible', 'growth', 'arrest', 'can', 'be', 'triggered', 'by', 'multiple', 'mechanisms', 'including', 'telomere', 'shortening', 'the', 'epigenetic', 'derepression', 'of', 'the', 'ink', 'arf', 'locus', 'and', 'dna', 'damage', 'together', 'these', 'mechanisms', 'limit', 'excessive', 'or', 'aberrant', 'cellular', 'proliferation', 'and', 'so', 'the', 'state', 'of', 'senescence', 'protects', 'against', 'the', 'development', 'of', 'cancer', 'recent', 'evidence', 'suggests', 'that', 'cellular', 'senescence', 'also', 'may', 'be', 'involved', 'in', 'aging'], ['histones', 'comprise', 'the', 'major', 'protein', 'component', 'of', 'chromatin', 'the', 'scaffold', 'in', 'which', 'the', 'eukaryotic', 'genome', 'is', 'packaged', 'and', 'are', 'subject', 'to', 'many', 'types', 'of', 'post', 'translational', 'modifications', 'ptms', 'especially', 'on', 'their', 'flexible', 'tails', 'these', 'modifications', 'may', 'constitute', 'histone', 'code', 'and', 'could', 'be', 'used', 'to', 'manage', 'epigenetic', 'information', 'that', 'helps', 'extend', 'the', 'genetic', 'message', 'beyond', 'dna', 'sequences', 'this', 'proposed', 'code', 'read', 'in', 'part', 'by', 'histone', 'ptm', 'binding', 'effector', 'modules', 'and', 'their', 'associated', 'complexes', 'is', 'predicted', 'to', 'define', 'unique', 'functional', 'states', 'of', 'chromatin', 'and', 'or', 'regulate', 'various', 'chromatin', 'templated', 'processes', 'wealth', 'of', 'structural', 'and', 'functional', 'data', 'show', 'how', 'chromatin', 'effector', 'modules', 'target', 'their', 'cognate', 'covalent', 'histone', 'modifications', 'here', 'we', 'summarize', 'key', 'features', 'in', 'molecular', 'recognition', 'of', 'histone', 'ptms', 'by', 'diverse', 'family', 'of', 'reader', 'pockets', 'highlighting', 'specific', 'readout', 'mechanisms', 'for', 'individual', 'marks', 'common', 'themes', 'and', 'insights', 'into', 'the', 'downstream', 'functional', 'consequences', 'of', 'the', 'interactions', 'changes', 'in', 'these', 'interactions', 'may', 'have', 'far', 'reaching', 'implications', 'for', 'human', 'biology', 'and', 'disease', 'notably', 'cancer'], ['chromatin', 'followed', 'by', 'sequencing', 'chip', 'seq', 'is', 'technique', 'for', 'genome', 'wide', 'profiling', 'of', 'dna', 'binding', 'proteins', 'histone', 'modifications', 'or', 'nucleosomes', 'owing', 'to', 'the', 'tremendous', 'progress', 'in', 'next', 'generation', 'sequencing', 'technology', 'chip', 'seq', 'offers', 'higher', 'resolution', 'less', 'noise', 'and', 'greater', 'coverage', 'than', 'its', 'array', 'based', 'predecessor', 'chip', 'chip', 'with', 'the', 'decreasing', 'cost', 'of', 'sequencing', 'chip', 'seq', 'has', 'become', 'an', 'indispensable', 'tool', 'for', 'studying', 'gene', 'regulation', 'and', 'epigenetic', 'mechanisms', 'in', 'this', 'review', 'describe', 'the', 'benefits', 'and', 'challenges', 'in', 'harnessing', 'this', 'technique', 'with', 'an', 'emphasis', 'on', 'issues', 'related', 'to', 'experimental', 'design', 'and', 'data', 'analysis', 'chip', 'seq', 'experiments', 'generate', 'large', 'quantities', 'of', 'data', 'and', 'effective', 'computational', 'analysis', 'will', 'be', 'crucial', 'for', 'uncovering', 'biological', 'mechanisms'], ['inflammation', 'is', 'linked', 'clinically', 'and', 'to', 'cancer', 'and', 'nf', 'kappa', 'appears', 'to', 'play', 'causative', 'role', 'but', 'the', 'mechanisms', 'are', 'poorly', 'understood', 'we', 'show', 'that', 'transient', 'activation', 'of', 'src', 'oncoprotein', 'can', 'mediate', 'an', 'epigenetic', 'switch', 'from', 'immortalized', 'breast', 'cells', 'to', 'stably', 'transformed', 'line', 'that', 'forms', 'self', 'renewing', 'mammospheres', 'that', 'contain', 'cancer', 'stem', 'cells', 'src', 'activation', 'triggers', 'an', 'inflammatory', 'response', 'mediated', 'by', 'nf', 'kappa', 'that', 'directly', 'activates', 'lin', 'transcription', 'and', 'rapidly', 'reduces', 'let', 'microrna', 'levels', 'let', 'directly', 'inhibits', 'il', 'expression', 'resulting', 'in', 'higher', 'levels', 'of', 'il', 'than', 'achieved', 'by', 'nf', 'kappa', 'activation', 'il', 'mediated', 'activation', 'of', 'the', 'stat', 'transcription', 'factor', 'is', 'necessary', 'for', 'transformation', 'and', 'il', 'activates', 'nf', 'kappa', 'thereby', 'completing', 'positive', 'feedback', 'loop', 'this', 'regulatory', 'circuit', 'operates', 'in', 'other', 'cancer', 'cells', 'lines', 'and', 'its', 'transcriptional', 'signature', 'is', 'found', 'in', 'human', 'cancer', 'tissues', 'thus', 'inflammation', 'activates', 'positive', 'feedback', 'loop', 'that', 'maintains', 'the', 'epigenetic', 'transformed', 'state', 'for', 'many', 'generations', 'in', 'the', 'absence', 'of', 'the', 'inducing', 'signal'], ['expression', 'profiling', 'of', 'cells', 'revealed', 'that', 'out', 'of', 'human', 'mirnas', 'were', 'upregulated', 'more', 'than', 'fold', 'by', 'simultaneous', 'treatment', 'with', 'the', 'chromatin', 'modifying', 'drugs', 'aza', 'deoxycytidine', 'and', 'phenylbutyric', 'acid', 'one', 'of', 'these', 'mir', 'is', 'embedded', 'in', 'cpg', 'island', 'and', 'is', 'highly', 'induced', 'from', 'its', 'own', 'promoter', 'after', 'treatment', 'mir', 'is', 'usually', 'expressed', 'as', 'part', 'of', 'mirna', 'cluster', 'in', 'normal', 'cells', 'but', 'not', 'in', 'cancer', 'cells', 'suggesting', 'that', 'it', 'is', 'subject', 'to', 'epigenetic', 'silencing', 'in', 'addition', 'the', 'proto', 'oncogene', 'bcl', 'potential', 'target', 'of', 'mir', 'was', 'translationally', 'downregulated', 'after', 'treatment', 'these', 'results', 'suggest', 'that', 'dna', 'demethylation', 'and', 'histone', 'deacetylase', 'inhibition', 'can', 'activate', 'expression', 'of', 'mirnas', 'that', 'may', 'act', 'as', 'tumor', 'suppressors'], ['polycomb', 'proteins', 'form', 'chromatin', 'modifying', 'complexes', 'that', 'implement', 'transcriptional', 'silencing', 'in', 'higher', 'eukaryotes', 'hundreds', 'of', 'genes', 'are', 'silenced', 'by', 'polycomb', 'proteins', 'including', 'dozens', 'of', 'genes', 'that', 'encode', 'crucial', 'developmental', 'regulators', 'in', 'organisms', 'ranging', 'from', 'plants', 'to', 'humans', 'two', 'main', 'families', 'of', 'complexes', 'called', 'polycomb', 'repressive', 'complex', 'prc', 'and', 'prc', 'are', 'targeted', 'to', 'repressed', 'regions', 'recent', 'studies', 'have', 'advanced', 'our', 'understanding', 'of', 'these', 'complexes', 'including', 'their', 'potential', 'mechanisms', 'of', 'gene', 'silencing', 'the', 'roles', 'of', 'chromatin', 'modifications', 'their', 'means', 'of', 'delivery', 'to', 'target', 'genes', 'and', 'the', 'functional', 'distinctions', 'among', 'variant', 'complexes', 'emerging', 'concepts', 'include', 'the', 'existence', 'of', 'polycomb', 'barrier', 'to', 'transcription', 'elongation', 'and', 'the', 'involvement', 'of', 'non', 'coding', 'rnas', 'in', 'the', 'targeting', 'of', 'polycomb', 'complexes', 'these', 'findings', 'have', 'an', 'impact', 'on', 'the', 'epigenetic', 'programming', 'of', 'gene', 'expression', 'in', 'many', 'biological', 'systems'], ['recent', 'meeting', 'december', 'regarding', 'chromatin', 'based', 'epigenetics', 'was', 'hosted', 'by', 'the', 'banbury', 'conference', 'center', 'and', 'cold', 'spring', 'harbor', 'laboratory', 'the', 'intent', 'was', 'to', 'discuss', 'aspects', 'of', 'epigenetic', 'control', 'of', 'genomic', 'function', 'and', 'to', 'arrive', 'at', 'consensus', 'definition', 'of', 'epigenetics', 'to', 'be', 'considered', 'by', 'the', 'broader', 'community', 'it', 'was', 'evident', 'that', 'multiple', 'mechanistic', 'steps', 'lead', 'to', 'the', 'stable', 'heritance', 'of', 'the', 'epigenetic', 'phenotype', 'below', 'we', 'provide', 'our', 'view', 'and', 'interpretation', 'of', 'the', 'proceedings', 'at', 'the', 'meeting'], ['epigenetic', 'genome', 'modifications', 'are', 'thought', 'to', 'be', 'important', 'for', 'specifying', 'the', 'lineage', 'and', 'developmental', 'stage', 'of', 'cells', 'within', 'multicellular', 'organism', 'here', 'we', 'show', 'that', 'the', 'epigenetic', 'profile', 'of', 'pluripotent', 'embryonic', 'stem', 'cells', 'es', 'is', 'distinct', 'from', 'that', 'of', 'embryonic', 'carcinoma', 'cells', 'haematopoietic', 'stem', 'cells', 'hsc', 'and', 'their', 'differentiated', 'progeny', 'silent', 'lineage', 'specific', 'genes', 'replicated', 'earlier', 'in', 'pluripotent', 'cells', 'than', 'in', 'tissue', 'specific', 'stem', 'cells', 'or', 'differentiated', 'cells', 'and', 'had', 'unexpectedly', 'high', 'levels', 'of', 'acetylated', 'and', 'methylated', 'unusually', 'in', 'es', 'cells', 'these', 'markers', 'of', 'open', 'chromatin', 'were', 'also', 'combined', 'with', 'trimethylation', 'at', 'some', 'non', 'expressed', 'genes', 'thus', 'pluripotency', 'of', 'es', 'cells', 'is', 'characterized', 'by', 'specific', 'epigenetic', 'profile', 'where', 'lineage', 'specific', 'genes', 'may', 'be', 'accessible', 'but', 'if', 'so', 'carry', 'repressive', 'trimethylation', 'modifications', 'methylation', 'is', 'functionally', 'important', 'for', 'preventing', 'expression', 'of', 'these', 'genes', 'in', 'es', 'cells', 'as', 'premature', 'expression', 'occurs', 'in', 'embryonic', 'ectoderm', 'development', 'eed', 'deficient', 'es', 'cells', 'our', 'data', 'suggest', 'that', 'lineage', 'specific', 'genes', 'are', 'primed', 'for', 'expression', 'in', 'es', 'cells', 'but', 'are', 'held', 'in', 'check', 'by', 'opposing', 'chromatin', 'modifications'], ['dna', 'methylation', 'is', 'the', 'most', 'stable', 'type', 'of', 'epigenetic', 'modification', 'modulating', 'the', 'transcriptional', 'plasticity', 'of', 'mammalian', 'genomes', 'using', 'bisulfite', 'dna', 'sequencing', 'we', 'report', 'high', 'resolution', 'methylation', 'profiles', 'of', 'human', 'chromosomes', 'and', 'providing', 'resource', 'of', 'about', 'million', 'cpg', 'methylation', 'values', 'derived', 'from', 'different', 'tissues', 'analysis', 'of', 'six', 'annotation', 'categories', 'showed', 'that', 'evolutionarily', 'conserved', 'regions', 'are', 'the', 'predominant', 'sites', 'for', 'differential', 'dna', 'methylation', 'and', 'that', 'core', 'region', 'surrounding', 'the', 'transcriptional', 'start', 'site', 'is', 'an', 'informative', 'surrogate', 'for', 'promoter', 'methylation', 'we', 'find', 'that', 'of', 'the', 'analyzed', 'genes', 'are', 'differentially', 'methylated', 'in', 'their', 'utrs', 'and', 'that', 'about', 'one', 'third', 'of', 'the', 'differentially', 'methylated', 'utrs', 'are', 'inversely', 'correlated', 'with', 'transcription', 'despite', 'the', 'fact', 'that', 'our', 'study', 'controlled', 'for', 'factors', 'reported', 'to', 'affect', 'dna', 'methylation', 'such', 'as', 'sex', 'and', 'age', 'we', 'did', 'not', 'find', 'any', 'significant', 'attributable', 'effects', 'our', 'data', 'suggest', 'dna', 'methylation', 'to', 'be', 'ontogenetically', 'more', 'stable', 'than', 'previously', 'thought'], ['the', 'polycomb', 'group', 'pcg', 'proteins', 'form', 'chromatin', 'modifying', 'complexes', 'that', 'are', 'essential', 'for', 'embryonic', 'development', 'and', 'stem', 'cell', 'renewal', 'and', 'are', 'commonly', 'deregulated', 'in', 'cancer', 'here', 'we', 'identify', 'their', 'target', 'genes', 'using', 'genome', 'wide', 'location', 'analysis', 'in', 'human', 'embryonic', 'fibroblasts', 'we', 'find', 'that', 'polycomb', 'repressive', 'complex', 'prc', 'prc', 'and', 'tri', 'methylated', 'histone', 'co', 'occupy', 'silenced', 'genes', 'with', 'strong', 'functional', 'bias', 'for', 'embryonic', 'development', 'and', 'cell', 'fate', 'decisions', 'we', 'functionally', 'identify', 'genes', 'derepressed', 'in', 'human', 'embryonic', 'fibroblasts', 'depleted', 'of', 'the', 'prc', 'components', 'ezh', 'eed', 'suz', 'and', 'the', 'prc', 'component', 'bmi', 'interestingly', 'several', 'markers', 'of', 'osteogenesis', 'adipogenesis', 'and', 'chrondrogenesis', 'are', 'among', 'these', 'genes', 'consistent', 'with', 'the', 'mesenchymal', 'origin', 'of', 'fibroblasts', 'using', 'neuronal', 'model', 'of', 'differentiation', 'we', 'delineate', 'two', 'different', 'mechanisms', 'for', 'regulating', 'pcg', 'target', 'genes', 'for', 'genes', 'activated', 'during', 'differentiation', 'pcgs', 'are', 'displaced', 'however', 'for', 'genes', 'repressed', 'during', 'differentiation', 'we', 'paradoxically', 'find', 'that', 'they', 'are', 'already', 'bound', 'by', 'the', 'pcgs', 'in', 'cells', 'despite', 'being', 'actively', 'transcribed', 'our', 'results', 'are', 'consistent', 'with', 'the', 'hypothesis', 'that', 'pcgs', 'are', 'part', 'of', 'preprogrammed', 'memory', 'system', 'established', 'during', 'embryogenesis', 'marking', 'certain', 'key', 'genes', 'for', 'repressive', 'signals', 'during', 'subsequent', 'developmental', 'and', 'differentiation', 'processes'], ['epigenetic', 'marking', 'systems', 'confer', 'stability', 'of', 'gene', 'expression', 'during', 'mammalian', 'development', 'genome', 'wide', 'epigenetic', 'reprogramming', 'occurs', 'at', 'stages', 'when', 'developmental', 'potency', 'of', 'cells', 'changes', 'at', 'fertilization', 'the', 'paternal', 'genome', 'exchanges', 'protamines', 'for', 'histones', 'undergoes', 'dna', 'demethylation', 'and', 'acquires', 'histone', 'modifications', 'whereas', 'the', 'maternal', 'genome', 'appears', 'epigenetically', 'more', 'static', 'during', 'preimplantation', 'development', 'there', 'is', 'passive', 'dna', 'demethylation', 'and', 'further', 'reorganization', 'of', 'histone', 'modifications', 'in', 'blastocysts', 'embryonic', 'and', 'extraembryonic', 'lineages', 'first', 'show', 'different', 'epigenetic', 'marks', 'this', 'epigenetic', 'reprogramming', 'is', 'likely', 'to', 'be', 'needed', 'for', 'totipotency', 'correct', 'initiation', 'of', 'embryonic', 'gene', 'expression', 'and', 'early', 'lineage', 'development', 'in', 'the', 'embryo', 'comparative', 'work', 'demonstrates', 'reprogramming', 'in', 'all', 'mammalian', 'species', 'analysed', 'but', 'the', 'extent', 'and', 'timing', 'varies', 'consistent', 'with', 'notable', 'differences', 'between', 'species', 'during', 'preimplantation', 'development', 'parental', 'imprinting', 'marks', 'originate', 'in', 'sperm', 'and', 'oocytes', 'and', 'are', 'generally', 'protected', 'from', 'this', 'genome', 'wide', 'reprogramming', 'early', 'primordial', 'germ', 'cells', 'possess', 'imprinting', 'marks', 'similar', 'to', 'those', 'of', 'somatic', 'cells', 'however', 'rapid', 'dna', 'demethylation', 'after', 'midgestation', 'erases', 'these', 'parental', 'imprints', 'in', 'preparation', 'for', 'sex', 'specific', 'de', 'novo', 'methylation', 'during', 'gametogenesis', 'aberrant', 'reprogramming', 'of', 'somatic', 'epigenetic', 'marks', 'after', 'somatic', 'cell', 'nuclear', 'transfer', 'leads', 'to', 'epigenetic', 'defects', 'in', 'cloned', 'embryos', 'and', 'stem', 'cells', 'links', 'between', 'epigenetic', 'marking', 'systems', 'appear', 'to', 'be', 'developmentally', 'regulated', 'contributing', 'to', 'plasticity', 'number', 'of', 'activities', 'that', 'confer', 'epigenetic', 'marks', 'are', 'firmly', 'established', 'while', 'for', 'those', 'that', 'remove', 'marks', 'particularly', 'methylation', 'some', 'interesting', 'candidates', 'have', 'emerged', 'recently', 'which', 'need', 'thorough', 'testing', 'in', 'vivo', 'mechanistic', 'understanding', 'of', 'reprogramming', 'will', 'be', 'crucial', 'for', 'medical', 'applications', 'of', 'stem', 'cell', 'technology'], ['background', 'in', 'animal', 'models', 'variations', 'in', 'early', 'maternal', 'care', 'are', 'associated', 'with', 'differences', 'in', 'hypothalamic', 'pituitary', 'adrenal', 'hpa', 'stress', 'response', 'in', 'the', 'offspring', 'mediated', 'via', 'changes', 'in', 'the', 'epigenetic', 'regulation', 'of', 'glucocorticoid', 'receptor', 'gr', 'gene', 'nr', 'expression', 'objective', 'to', 'study', 'this', 'in', 'humans', 'relationships', 'between', 'prenatal', 'exposure', 'to', 'maternal', 'mood', 'and', 'the', 'methylation', 'status', 'of', 'cpg', 'rich', 'region', 'in', 'the', 'promoter', 'and', 'exon', 'of', 'the', 'human', 'gr', 'gene', 'nr', 'in', 'newborns', 'and', 'hpa', 'stress', 'reactivity', 'at', 'age', 'three', 'months', 'were', 'examined', 'results', 'prenatal', 'exposure', 'to', 'increased', 'third', 'trimester', 'maternal', 'depressed', 'anxious', 'mood', 'was', 'associated', 'with', 'increased', 'methylation', 'of', 'nr', 'at', 'predicted', 'ngfi', 'binding', 'site', 'increased', 'nr', 'methylation', 'at', 'this', 'site', 'was', 'also', 'associated', 'with', 'increased', 'salivary', 'cortisol', 'stress', 'responses', 'at', 'months', 'controlling', 'for', 'prenatal', 'sri', 'exposure', 'postnatal', 'age', 'and', 'pre', 'and', 'postnatal', 'maternal', 'mood', 'methods', 'the', 'methylation', 'status', 'of', 'cpg', 'rich', 'region', 'of', 'the', 'nr', 'gene', 'including', 'exon', 'in', 'genomic', 'dna', 'from', 'cord', 'blood', 'mononuclear', 'cells', 'was', 'quantified', 'by', 'bisulfite', 'pyrosequencing', 'in', 'infants', 'of', 'depressed', 'mothers', 'treated', 'with', 'serotonin', 'reuptake', 'inhibitor', 'antidepressant', 'sri', 'infants', 'of', 'depressed', 'non', 'treated', 'mothers', 'and', 'infants', 'of', 'non', 'depressed', 'non', 'treated', 'mothers', 'to', 'study', 'the', 'functional', 'implications', 'of', 'the', 'newborn', 'methylation', 'status', 'of', 'nr', 'in', 'newborns', 'hpa', 'function', 'was', 'assessed', 'at', 'three', 'months', 'using', 'salivary', 'cortisol', 'obtained', 'before', 'and', 'following', 'non', 'noxious', 'stressor', 'and', 'at', 'late', 'afternoon', 'basal', 'time', 'conclusions', 'methylation', 'status', 'of', 'the', 'human', 'nr', 'gene', 'in', 'newborns', 'is', 'sensitive', 'to', 'prenatal', 'maternal', 'mood', 'and', 'may', 'offer', 'potential', 'epigenetic', 'process', 'that', 'links', 'antenatal', 'maternal', 'mood', 'and', 'altered', 'hpa', 'stress', 'reactivity', 'during', 'infancy'], ['the', 'formation', 'of', 'heterochromatin', 'which', 'requires', 'methylation', 'of', 'histone', 'at', 'lysine', 'and', 'the', 'subsequent', 'recruitment', 'of', 'chromodomain', 'proteins', 'such', 'as', 'heterochromatin', 'protein', 'hp', 'serves', 'as', 'model', 'for', 'the', 'role', 'of', 'histone', 'modifications', 'and', 'chromatin', 'assembly', 'in', 'epigenetic', 'control', 'of', 'the', 'genome', 'recent', 'studies', 'in', 'pombe', 'indicate', 'that', 'heterochromatin', 'serves', 'as', 'dynamic', 'platform', 'to', 'recruit', 'and', 'spread', 'myriad', 'of', 'regulatory', 'proteins', 'across', 'extended', 'domains', 'to', 'control', 'various', 'chromosomal', 'processes', 'including', 'transcription', 'chromosome', 'segregation', 'and', 'long', 'range', 'chromatin', 'interactions'], ['compelling', 'evidence', 'suggests', 'that', 'the', 'transcription', 'factor', 'foxp', 'acts', 'as', 'master', 'switch', 'governing', 'the', 'development', 'and', 'function', 'of', 'cd', 'regulatory', 'cells', 'tregs', 'however', 'whether', 'transcriptional', 'control', 'of', 'foxp', 'expression', 'itself', 'contributes', 'to', 'the', 'development', 'of', 'stable', 'treg', 'lineage', 'has', 'thus', 'far', 'not', 'been', 'investigated', 'we', 'here', 'identified', 'an', 'evolutionarily', 'conserved', 'region', 'within', 'the', 'foxp', 'locus', 'upstream', 'of', 'exon', 'possessing', 'transcriptional', 'activity', 'bisulphite', 'sequencing', 'and', 'chromatin', 'revealed', 'complete', 'demethylation', 'of', 'cpg', 'motifs', 'as', 'well', 'as', 'histone', 'modifications', 'within', 'the', 'conserved', 'region', 'in', 'ex', 'vivo', 'isolated', 'foxp', 'cd', 'cd', 'tregs', 'but', 'not', 'in', 'naive', 'cd', 'cd', 'cells', 'partial', 'dna', 'demethylation', 'is', 'already', 'found', 'within', 'developing', 'foxp', 'thymocytes', 'however', 'tregs', 'induced', 'by', 'tgf', 'beta', 'in', 'vitro', 'display', 'only', 'incomplete', 'demethylation', 'despite', 'high', 'foxp', 'expression', 'in', 'contrast', 'to', 'natural', 'tregs', 'these', 'tgf', 'beta', 'induced', 'foxp', 'tregs', 'lose', 'both', 'foxp', 'expression', 'and', 'suppressive', 'activity', 'upon', 'restimulation', 'in', 'the', 'absence', 'of', 'tgf', 'beta', 'our', 'data', 'suggest', 'that', 'expression', 'of', 'foxp', 'must', 'be', 'stabilized', 'by', 'epigenetic', 'modification', 'to', 'allow', 'the', 'development', 'of', 'permanent', 'suppressor', 'cell', 'lineage', 'finding', 'of', 'significant', 'importance', 'for', 'therapeutic', 'applications', 'involving', 'induction', 'or', 'transfer', 'of', 'tregs', 'and', 'for', 'the', 'understanding', 'of', 'long', 'term', 'cell', 'lineage', 'decisions'], ['many', 'genes', 'associated', 'with', 'cpg', 'islands', 'undergo', 'de', 'novo', 'methylation', 'in', 'cancer', 'studies', 'have', 'suggested', 'that', 'the', 'pattern', 'of', 'this', 'modification', 'may', 'be', 'partially', 'determined', 'by', 'an', 'instructive', 'mechanism', 'that', 'recognizes', 'specifically', 'marked', 'regions', 'of', 'the', 'genome', 'using', 'chromatin', 'analysis', 'here', 'we', 'show', 'that', 'genes', 'methylated', 'in', 'cancer', 'cells', 'are', 'specifically', 'packaged', 'with', 'nucleosomes', 'containing', 'histone', 'trimethylated', 'on', 'lys', 'this', 'chromatin', 'mark', 'is', 'established', 'on', 'these', 'unmethylated', 'cpg', 'island', 'genes', 'early', 'in', 'development', 'and', 'then', 'maintained', 'in', 'differentiated', 'cell', 'types', 'by', 'the', 'presence', 'of', 'an', 'ezh', 'containing', 'polycomb', 'complex', 'in', 'cancer', 'cells', 'as', 'opposed', 'to', 'normal', 'cells', 'the', 'presence', 'of', 'this', 'complex', 'brings', 'about', 'the', 'recruitment', 'of', 'dna', 'methyl', 'transferases', 'leading', 'to', 'de', 'novo', 'methylation', 'these', 'results', 'suggest', 'that', 'tumor', 'specific', 'targeting', 'of', 'de', 'novo', 'methylation', 'is', 'pre', 'programmed', 'by', 'an', 'established', 'epigenetic', 'system', 'that', 'normally', 'has', 'role', 'in', 'marking', 'embryonic', 'genes', 'for', 'repression'], ['ageing', 'results', 'from', 'complex', 'genetically', 'and', 'epigenetically', 'programmed', 'processes', 'that', 'are', 'elicited', 'in', 'part', 'by', 'noxious', 'or', 'stressful', 'events', 'that', 'cause', 'programmed', 'cell', 'death', 'here', 'we', 'report', 'that', 'administration', 'of', 'spermidine', 'natural', 'polyamine', 'whose', 'intracellular', 'concentration', 'declines', 'during', 'human', 'ageing', 'markedly', 'extended', 'the', 'lifespan', 'of', 'yeast', 'flies', 'and', 'worms', 'and', 'human', 'immune', 'cells', 'in', 'addition', 'spermidine', 'administration', 'potently', 'inhibited', 'oxidative', 'stress', 'in', 'ageing', 'mice', 'in', 'ageing', 'yeast', 'spermidine', 'treatment', 'triggered', 'epigenetic', 'deacetylation', 'of', 'histone', 'through', 'inhibition', 'of', 'histone', 'hat', 'suppressing', 'oxidative', 'stress', 'and', 'necrosis', 'conversely', 'depletion', 'of', 'endogenous', 'polyamines', 'led', 'to', 'generation', 'of', 'reactive', 'oxygen', 'species', 'early', 'necrotic', 'death', 'and', 'decreased', 'lifespan', 'the', 'altered', 'acetylation', 'status', 'of', 'the', 'chromatin', 'led', 'to', 'significant', 'upregulation', 'of', 'various', 'autophagy', 'related', 'transcripts', 'triggering', 'autophagy', 'in', 'yeast', 'flies', 'worms', 'and', 'human', 'cells', 'finally', 'we', 'found', 'that', 'enhanced', 'autophagy', 'is', 'crucial', 'for', 'polyamine', 'induced', 'suppression', 'of', 'necrosis', 'and', 'enhanced', 'longevity'], ['induced', 'pluripotent', 'stem', 'ips', 'cells', 'are', 'derived', 'by', 'epigenetic', 'reprogramming', 'but', 'their', 'dna', 'methylation', 'patterns', 'have', 'not', 'yet', 'been', 'analyzed', 'on', 'genome', 'wide', 'scale', 'here', 'we', 'find', 'substantial', 'and', 'hypomethylation', 'of', 'cytosine', 'phosphate', 'guanine', 'cpg', 'island', 'shores', 'in', 'nine', 'human', 'ips', 'cell', 'lines', 'as', 'compared', 'to', 'their', 'parental', 'fibroblasts', 'the', 'differentially', 'methylated', 'regions', 'dmrs', 'in', 'the', 'reprogrammed', 'cells', 'denoted', 'dmrs', 'were', 'significantly', 'enriched', 'in', 'tissue', 'specific', 'dmrs', 'fold', 'and', 'cancer', 'specific', 'dmrs', 'dmrs', 'fold', 'notably', 'even', 'though', 'the', 'ips', 'cells', 'are', 'derived', 'from', 'fibroblasts', 'their', 'dmrs', 'can', 'distinguish', 'between', 'normal', 'brain', 'liver', 'and', 'spleen', 'cells', 'and', 'between', 'colon', 'cancer', 'and', 'normal', 'colon', 'cells', 'thus', 'many', 'dmrs', 'are', 'broadly', 'involved', 'in', 'tissue', 'differentiation', 'epigenetic', 'reprogramming', 'and', 'cancer', 'we', 'observed', 'colocalization', 'of', 'hypomethylated', 'dmrs', 'with', 'hypermethylated', 'dmrs', 'and', 'bivalent', 'chromatin', 'marks', 'and', 'colocalization', 'of', 'hypermethylated', 'dmrs', 'with', 'hypomethylated', 'dmrs', 'and', 'the', 'absence', 'of', 'bivalent', 'marks', 'suggesting', 'two', 'mechanisms', 'for', 'epigenetic', 'reprogramming', 'in', 'ips', 'cells', 'and', 'cancer'], ['histone', 'methylation', 'has', 'important', 'roles', 'in', 'regulating', 'gene', 'expression', 'and', 'forms', 'part', 'of', 'the', 'epigenetic', 'memory', 'system', 'that', 'regulates', 'cell', 'fate', 'and', 'identity', 'enzymes', 'that', 'directly', 'remove', 'methyl', 'marks', 'from', 'histones', 'have', 'recently', 'been', 'identified', 'revealing', 'new', 'level', 'of', 'plasticity', 'within', 'this', 'epigenetic', 'modification', 'system', 'here', 'we', 'analyse', 'the', 'evolutionary', 'relationship', 'between', 'jumonji', 'jmjc', 'domain', 'containing', 'proteins', 'and', 'discuss', 'their', 'cellular', 'functions', 'in', 'relation', 'to', 'their', 'potential', 'enzymatic', 'activities'], ['protein', 'lysine', 'deacetylases', 'have', 'pivotal', 'role', 'in', 'numerous', 'biological', 'processes', 'and', 'can', 'be', 'divided', 'into', 'the', 'rpd', 'hda', 'and', 'sirtuin', 'families', 'each', 'having', 'members', 'in', 'diverse', 'organisms', 'including', 'prokaryotes', 'in', 'vertebrates', 'the', 'rpd', 'hda', 'family', 'contains', 'members', 'traditionally', 'referred', 'to', 'as', 'histone', 'deacetylases', 'hdac', 'which', 'are', 'further', 'grouped', 'into', 'classes', 'ii', 'and', 'iv', 'whereas', 'most', 'class', 'hdacs', 'are', 'subunits', 'of', 'multiprotein', 'nuclear', 'complexes', 'that', 'are', 'crucial', 'for', 'transcriptional', 'repression', 'and', 'epigenetic', 'landscaping', 'class', 'ii', 'members', 'regulate', 'cytoplasmic', 'processes', 'or', 'function', 'as', 'signal', 'transducers', 'that', 'shuttle', 'between', 'the', 'cytoplasm', 'and', 'the', 'nucleus', 'little', 'is', 'known', 'about', 'class', 'iv', 'hdac', 'although', 'its', 'evolutionary', 'conservation', 'implies', 'fundamental', 'role', 'in', 'various', 'organisms'], ['recent', 'investigations', 'have', 'implicated', 'long', 'antisense', 'noncoding', 'rnas', 'in', 'the', 'epigenetic', 'regulation', 'of', 'chromosomal', 'domains', 'here', 'we', 'show', 'that', 'kcnq', 'ot', 'is', 'an', 'rna', 'polymerase', 'ii', 'encoded', 'kb', 'long', 'moderately', 'stable', 'nuclear', 'transcript', 'and', 'that', 'its', 'stability', 'is', 'important', 'for', 'bidirectional', 'silencing', 'of', 'genes', 'in', 'the', 'kcnq', 'domain', 'kcnq', 'ot', 'interacts', 'with', 'chromatin', 'and', 'with', 'the', 'and', 'specific', 'histone', 'and', 'the', 'prc', 'complex', 'in', 'lineage', 'specific', 'manner', 'this', 'interaction', 'correlates', 'with', 'the', 'presence', 'of', 'extended', 'regions', 'of', 'chromatin', 'enriched', 'with', 'me', 'and', 'me', 'in', 'the', 'kcnq', 'domain', 'in', 'placenta', 'whereas', 'fetal', 'liver', 'lacks', 'both', 'chromatin', 'interactions', 'and', 'heterochromatin', 'structures', 'in', 'addition', 'the', 'kcnq', 'domain', 'is', 'more', 'often', 'found', 'in', 'contact', 'with', 'the', 'nucleolar', 'compartment', 'in', 'placenta', 'than', 'in', 'liver', 'taken', 'together', 'our', 'data', 'describe', 'mechanism', 'whereby', 'kcnq', 'ot', 'establishes', 'lineage', 'specific', 'transcriptional', 'silencing', 'patterns', 'through', 'recruitment', 'of', 'chromatin', 'remodeling', 'complexes', 'and', 'maintenance', 'of', 'these', 'patterns', 'through', 'subsequent', 'cell', 'divisions', 'occurs', 'via', 'targeting', 'the', 'associated', 'regions', 'to', 'the', 'perinucleolar', 'compartment'], ['it', 'is', 'more', 'evident', 'now', 'than', 'ever', 'that', 'nucleosomes', 'can', 'transmit', 'epigenetic', 'information', 'from', 'one', 'cell', 'generation', 'to', 'the', 'next', 'it', 'has', 'been', 'demonstrated', 'during', 'the', 'past', 'decade', 'that', 'the', 'post', 'translational', 'modifications', 'of', 'histone', 'proteins', 'within', 'the', 'chromosome', 'impact', 'chromatin', 'structure', 'gene', 'transcription', 'and', 'epigenetic', 'information', 'multiple', 'modifications', 'decorate', 'each', 'histone', 'tail', 'within', 'the', 'nucleosome', 'including', 'some', 'amino', 'acids', 'that', 'can', 'be', 'modified', 'in', 'several', 'different', 'ways', 'covalent', 'modifications', 'of', 'histone', 'tails', 'known', 'thus', 'far', 'include', 'acetylation', 'phosphorylation', 'surnoylation', 'ubiquitination', 'and', 'methylation', 'large', 'body', 'of', 'experimental', 'evidence', 'compiled', 'during', 'the', 'past', 'several', 'years', 'has', 'demonstrated', 'the', 'impact', 'of', 'histone', 'acetylation', 'on', 'transcriptional', 'control', 'although', 'histone', 'modification', 'by', 'methylation', 'and', 'ubiquitination', 'was', 'discovered', 'long', 'ago', 'it', 'was', 'only', 'recently', 'that', 'functional', 'roles', 'for', 'these', 'modifications', 'in', 'transcriptional', 'regulation', 'began', 'to', 'surface', 'highlighted', 'in', 'this', 'review', 'are', 'the', 'recent', 'biochemical', 'molecular', 'cellular', 'and', 'physiological', 'functions', 'of', 'histone', 'methylation', 'and', 'ubiquitination', 'involved', 'in', 'the', 'regulation', 'of', 'gene', 'expression', 'as', 'determined', 'by', 'combination', 'of', 'enzymological', 'structural', 'and', 'genetic', 'methodologies'], ['cells', 'employ', 'elaborate', 'mechanisms', 'to', 'introduce', 'structural', 'and', 'chemical', 'variation', 'into', 'chromatin', 'here', 'we', 'focus', 'on', 'one', 'such', 'element', 'of', 'variation', 'methylation', 'of', 'lysine', 'in', 'histone', 'we', 'assess', 'growing', 'body', 'of', 'literature', 'including', 'treatment', 'of', 'how', 'the', 'mark', 'is', 'established', 'the', 'patterns', 'of', 'methylation', 'and', 'the', 'functional', 'consequences', 'of', 'this', 'epigenetic', 'signature', 'we', 'discuss', 'structural', 'aspects', 'of', 'the', 'methyl', 'recognition', 'by', 'the', 'downstream', 'effectors', 'and', 'propose', 'distinction', 'between', 'sequence', 'specific', 'recruitment', 'mechanisms', 'and', 'stabilization', 'on', 'chromatin', 'through', 'methyllysine', 'recognition', 'finally', 'we', 'hypothesize', 'how', 'the', 'unique', 'properties', 'of', 'the', 'polyvalent', 'chromatin', 'fiber', 'and', 'associated', 'effectors', 'may', 'amplify', 'small', 'differences', 'in', 'methyl', 'lysine', 'recognition', 'simultaneously', 'allowing', 'for', 'dynamic', 'chromatin', 'architecture'], ['embryonic', 'stem', 'cells', 'rely', 'on', 'polycomb', 'group', 'proteins', 'to', 'reversibly', 'repress', 'genes', 'required', 'for', 'differentiation', 'we', 'report', 'that', 'stem', 'cell', 'polycomb', 'group', 'targets', 'are', 'up', 'to', 'fold', 'more', 'likely', 'to', 'have', 'cancer', 'specific', 'promoter', 'dna', 'than', 'non', 'targets', 'supporting', 'stem', 'cell', 'origin', 'of', 'cancer', 'in', 'which', 'reversible', 'gene', 'repression', 'is', 'replaced', 'by', 'permanent', 'silencing', 'locking', 'the', 'cell', 'into', 'perpetual', 'state', 'of', 'self', 'renewal', 'and', 'thereby', 'predisposing', 'to', 'subsequent', 'malignant', 'transformation'], ['adult', 'cancers', 'may', 'derive', 'from', 'stem', 'or', 'early', 'progenitor', 'cells', 'epigenetic', 'modulation', 'of', 'gene', 'expression', 'is', 'essential', 'for', 'normal', 'function', 'of', 'these', 'early', 'cells', 'but', 'is', 'highly', 'abnormal', 'in', 'cancers', 'which', 'often', 'show', 'aberrant', 'promoter', 'cpg', 'island', 'and', 'transcriptional', 'silencing', 'of', 'tumor', 'suppressor', 'genes', 'and', 'pro', 'differentiation', 'factors', 'we', 'find', 'that', 'for', 'such', 'genes', 'both', 'normal', 'and', 'malignant', 'embryonic', 'cells', 'generally', 'lack', 'the', 'of', 'dna', 'found', 'in', 'adult', 'cancers', 'in', 'embryonic', 'stem', 'cells', 'these', 'genes', 'are', 'held', 'in', 'transcription', 'ready', 'state', 'mediated', 'by', 'bivalent', 'promoter', 'chromatin', 'pattern', 'consisting', 'of', 'the', 'repressive', 'mark', 'histone', 'methylated', 'at', 'lys', 'by', 'polycomb', 'group', 'proteins', 'plus', 'the', 'active', 'mark', 'methylated', 'however', 'embryonic', 'carcinoma', 'cells', 'add', 'two', 'key', 'repressive', 'marks', 'dimethylated', 'and', 'trimethylated', 'both', 'associated', 'with', 'dna', 'in', 'adult', 'cancers', 'we', 'hypothesize', 'that', 'cell', 'chromatin', 'patterns', 'and', 'transient', 'silencing', 'of', 'these', 'important', 'regulatory', 'genes', 'in', 'stem', 'or', 'progenitor', 'cells', 'may', 'leave', 'these', 'genes', 'vulnerable', 'to', 'aberrant', 'dna', 'and', 'heritable', 'gene', 'silencing', 'during', 'tumor', 'initiation', 'and', 'progression'], [], ['patterns', 'of', 'dna', 'methylation', 'and', 'chromatin', 'structure', 'are', 'profoundly', 'altered', 'in', 'neoplasia', 'and', 'include', 'genome', 'wide', 'losses', 'of', 'and', 'regional', 'gains', 'in', 'dna', 'methylation', 'the', 'recent', 'explosion', 'in', 'our', 'knowledge', 'of', 'how', 'chromatin', 'organization', 'modulates', 'gene', 'transcription', 'has', 'further', 'highlighted', 'the', 'importance', 'of', 'epigenetic', 'mechanisms', 'in', 'the', 'initiation', 'and', 'progression', 'of', 'human', 'cancer', 'these', 'epigenetic', 'changes', 'in', 'particular', 'aberrant', 'promoter', 'that', 'is', 'associated', 'with', 'inappropriate', 'gene', 'silencing', 'affect', 'virtually', 'every', 'step', 'in', 'tumour', 'progression', 'in', 'this', 'review', 'we', 'discuss', 'these', 'epigenetic', 'events', 'and', 'the', 'molecular', 'alterations', 'that', 'might', 'cause', 'them', 'and', 'or', 'underlie', 'altered', 'gene', 'expression', 'in', 'cancer'], ['cells', 'of', 'multicellular', 'organism', 'are', 'genetically', 'homogeneous', 'but', 'structurally', 'and', 'functionally', 'heterogeneous', 'owing', 'to', 'the', 'differential', 'expression', 'of', 'genes', 'many', 'of', 'these', 'differences', 'in', 'gene', 'expression', 'arise', 'during', 'development', 'and', 'are', 'subsequently', 'retained', 'through', 'mitosis', 'stable', 'alterations', 'of', 'this', 'kind', 'are', 'said', 'to', 'be', 'epigenetic', 'because', 'they', 'are', 'heritable', 'in', 'the', 'short', 'term', 'but', 'do', 'not', 'involve', 'mutations', 'of', 'the', 'dna', 'itself', 'research', 'over', 'the', 'past', 'few', 'years', 'has', 'focused', 'on', 'two', 'molecular', 'mechanisms', 'that', 'mediate', 'epigenetic', 'phenomena', 'dna', 'methylation', 'and', 'histone', 'modifications', 'here', 'we', 'review', 'advances', 'in', 'the', 'understanding', 'of', 'the', 'mechanism', 'and', 'role', 'of', 'dna', 'methylation', 'in', 'biological', 'processes', 'epigenetic', 'effects', 'by', 'means', 'of', 'dna', 'methylation', 'have', 'an', 'important', 'role', 'in', 'development', 'but', 'can', 'also', 'arise', 'stochastically', 'as', 'animals', 'age', 'identification', 'of', 'proteins', 'that', 'mediate', 'these', 'effects', 'has', 'provided', 'insight', 'into', 'this', 'complex', 'process', 'and', 'diseases', 'that', 'occur', 'when', 'it', 'is', 'perturbed', 'external', 'influences', 'on', 'epigenetic', 'processes', 'are', 'seen', 'in', 'the', 'effects', 'of', 'diet', 'on', 'long', 'term', 'diseases', 'such', 'as', 'cancer', 'thus', 'epigenetic', 'mechanisms', 'seem', 'to', 'allow', 'an', 'organism', 'to', 'respond', 'to', 'the', 'environment', 'through', 'changes', 'in', 'gene', 'expression', 'the', 'extent', 'to', 'which', 'environmental', 'effects', 'can', 'provoke', 'epigenetic', 'responses', 'represents', 'an', 'exciting', 'area', 'of', 'future', 'research'], ['epithelial', 'and', 'hematopoietic', 'cells', 'have', 'high', 'turnover', 'and', 'their', 'progenitor', 'cells', 'divide', 'continuously', 'making', 'them', 'prime', 'targets', 'for', 'genetic', 'and', 'epigenetic', 'changes', 'that', 'lead', 'to', 'cell', 'transformation', 'and', 'tumorigenesis', 'the', 'consequent', 'changes', 'in', 'cell', 'behavior', 'and', 'responsiveness', 'result', 'not', 'only', 'from', 'genetic', 'alterations', 'such', 'as', 'activation', 'of', 'oncogenes', 'or', 'inactivation', 'of', 'tumor', 'suppressor', 'genes', 'but', 'also', 'from', 'altered', 'production', 'of', 'or', 'responsiveness', 'to', 'stimulatory', 'or', 'inhibitory', 'growth', 'and', 'differentiation', 'factors', 'among', 'these', 'transforming', 'growth', 'factor', 'beta', 'tgf', 'beta', 'and', 'its', 'signaling', 'effectors', 'act', 'as', 'key', 'determinants', 'of', 'carcinoma', 'cell', 'behavior', 'the', 'autocrine', 'and', 'paracrine', 'effects', 'of', 'tgf', 'beta', 'on', 'tumor', 'cells', 'and', 'the', 'tumor', 'micro', 'environment', 'exert', 'both', 'positive', 'and', 'negative', 'influences', 'on', 'cancer', 'development', 'accordingly', 'the', 'tgf', 'beta', 'signaling', 'pathway', 'has', 'been', 'considered', 'as', 'both', 'tumor', 'suppressor', 'pathway', 'and', 'promoter', 'of', 'tumor', 'progression', 'and', 'invasion', 'here', 'we', 'evaluate', 'the', 'role', 'of', 'tgf', 'beta', 'in', 'tumor', 'development', 'and', 'attempt', 'to', 'reconcile', 'the', 'positive', 'and', 'negative', 'effects', 'of', 'tgf', 'beta', 'in', 'carcinogenesis'], ['genomic', 'imprinting', 'affects', 'several', 'dozen', 'mammalian', 'genes', 'and', 'results', 'in', 'the', 'expression', 'of', 'those', 'genes', 'from', 'only', 'one', 'of', 'the', 'two', 'parental', 'chromosomes', 'this', 'is', 'brought', 'about', 'by', 'epigenetic', 'instructions', 'imprints', 'that', 'are', 'laid', 'down', 'in', 'the', 'parental', 'germ', 'cells', 'imprinting', 'is', 'particularly', 'important', 'genetic', 'mechanism', 'in', 'mammals', 'and', 'is', 'thought', 'to', 'influence', 'the', 'transfer', 'of', 'nutrients', 'to', 'the', 'fetus', 'and', 'the', 'newborn', 'from', 'the', 'mother', 'consistent', 'with', 'this', 'view', 'is', 'the', 'fact', 'that', 'imprinted', 'genes', 'tend', 'to', 'affect', 'growth', 'in', 'the', 'womb', 'and', 'behaviour', 'after', 'birth', 'aberrant', 'imprinting', 'disturbs', 'development', 'and', 'is', 'the', 'cause', 'of', 'various', 'disease', 'syndromes', 'the', 'study', 'of', 'imprinting', 'also', 'provides', 'new', 'insights', 'into', 'epigenetic', 'gene', 'modification', 'during', 'development'], ['the', 'developmental', 'programme', 'of', 'embryogenesis', 'is', 'controlled', 'by', 'both', 'genetic', 'and', 'epigenetic', 'mechanisms', 'an', 'emerging', 'theme', 'from', 'recent', 'studies', 'is', 'that', 'the', 'regulation', 'of', 'higher', 'order', 'chromatin', 'structures', 'by', 'dna', 'methylation', 'and', 'histone', 'modification', 'is', 'crucial', 'for', 'genome', 'reprogramming', 'during', 'early', 'embryogenesis', 'and', 'gametogenesis', 'and', 'for', 'tissue', 'specific', 'gene', 'expression', 'and', 'global', 'gene', 'silencing', 'disruptions', 'to', 'chromatin', 'modification', 'can', 'lead', 'to', 'the', 'dysregulation', 'of', 'developmental', 'processes', 'such', 'as', 'chromosome', 'inactivation', 'and', 'genomic', 'imprinting', 'and', 'to', 'various', 'diseases', 'understanding', 'the', 'process', 'of', 'epigenetic', 'reprogramming', 'in', 'development', 'is', 'important', 'for', 'studies', 'of', 'cloning', 'and', 'the', 'clinical', 'application', 'of', 'stem', 'cell', 'therapy'], ['early', 'nutrition', 'affects', 'adult', 'metabolism', 'in', 'humans', 'and', 'other', 'mammals', 'potentially', 'via', 'persistent', 'alterations', 'in', 'dna', 'methylation', 'with', 'viable', 'yellow', 'agouti', 'vy', 'mice', 'which', 'harbor', 'transposable', 'element', 'in', 'the', 'agouti', 'gene', 'we', 'tested', 'the', 'hypothesis', 'that', 'the', 'metastable', 'methylation', 'status', 'of', 'specific', 'transposable', 'element', 'insertion', 'sites', 'renders', 'them', 'epigenetically', 'labile', 'to', 'early', 'methyl', 'donor', 'nutrition', 'our', 'results', 'show', 'that', 'dietary', 'methyl', 'supplementation', 'of', 'ala', 'dams', 'with', 'extra', 'folic', 'acid', 'vitamin', 'choline', 'and', 'betaine', 'alter', 'the', 'phenotype', 'of', 'their', 'vy', 'offspring', 'via', 'increased', 'cpg', 'methylation', 'at', 'the', 'vy', 'locus', 'and', 'that', 'the', 'epigenetic', 'metastability', 'which', 'confers', 'this', 'lability', 'is', 'due', 'to', 'the', 'vy', 'transposable', 'element', 'these', 'findings', 'suggest', 'that', 'dietary', 'supplementation', 'long', 'presumed', 'to', 'be', 'purely', 'beneficial', 'may', 'have', 'unintended', 'deleterious', 'influences', 'on', 'the', 'establishment', 'of', 'epigenetic', 'gene', 'regulation', 'in', 'humans'], ['recently', 'the', 'concept', 'that', 'epigenetic', 'as', 'well', 'as', 'genetic', 'events', 'might', 'be', 'central', 'to', 'the', 'evolution', 'of', 'human', 'cancer', 'is', 're', 'emerging', 'cancers', 'often', 'exhibit', 'an', 'aberrant', 'methylation', 'of', 'gene', 'promoter', 'regions', 'that', 'is', 'associated', 'with', 'loss', 'of', 'gene', 'function', 'this', 'dna', 'change', 'constitutes', 'heritable', 'state', 'not', 'mediated', 'by', 'altered', 'nucleotide', 'sequence', 'that', 'appears', 'to', 'be', 'tightly', 'linked', 'to', 'the', 'formation', 'of', 'repressive', 'chromatin', 'this', 'epigenetic', 'process', 'acts', 'as', 'an', 'alternative', 'to', 'mutations', 'to', 'disrupt', 'tumor', 'suppressor', 'gene', 'function', 'and', 'can', 'predispose', 'to', 'genetic', 'alterations', 'through', 'inactivating', 'dna', 'repair', 'genes', 'dissecting', 'the', 'molecular', 'processes', 'that', 'mediate', 'these', 'methylation', 'changes', 'will', 'enhance', 'our', 'understanding', 'of', 'chromatin', 'modeling', 'and', 'gene', 'regulation', 'and', 'might', 'present', 'novel', 'possibilities', 'for', 'cancer', 'therapy', 'methylation', 'changes', 'constitute', 'potentially', 'sensitive', 'molecular', 'markers', 'to', 'define', 'risk', 'states', 'monitor', 'prevention', 'strategies', 'achieve', 'early', 'diagnosis', 'and', 'track', 'the', 'prognosis', 'of', 'cancer'], ['polyploidy', 'is', 'prominent', 'process', 'in', 'plants', 'and', 'has', 'been', 'significant', 'in', 'the', 'evolutionary', 'history', 'of', 'vertebrates', 'and', 'other', 'eukaryotes', 'in', 'plants', 'approaches', 'combining', 'phylogenetic', 'and', 'molecular', 'genetic', 'perspectives', 'have', 'enhanced', 'our', 'awareness', 'of', 'the', 'myriad', 'genetic', 'interactions', 'made', 'possible', 'by', 'polyploidy', 'here', 'processes', 'and', 'mechanisms', 'of', 'gene', 'and', 'genome', 'evolution', 'in', 'polyploids', 'are', 'reviewed', 'genes', 'duplicated', 'by', 'polyploidy', 'may', 'retain', 'their', 'original', 'or', 'similar', 'function', 'undergo', 'diversification', 'in', 'protein', 'function', 'or', 'regulation', 'or', 'one', 'copy', 'may', 'become', 'silenced', 'through', 'mutational', 'or', 'epigenetic', 'means', 'duplicated', 'genes', 'also', 'may', 'interact', 'through', 'inter', 'locus', 'recombination', 'gene', 'conversion', 'or', 'concerted', 'evolution', 'recent', 'experiments', 'have', 'illuminated', 'important', 'processes', 'in', 'polyploids', 'that', 'operate', 'above', 'the', 'organizational', 'level', 'of', 'duplicated', 'genes', 'these', 'include', 'inter', 'genomic', 'chromosomal', 'exchanges', 'saltational', 'non', 'mendelian', 'genomic', 'evolution', 'in', 'nascent', 'polyploids', 'inter', 'genomic', 'invasion', 'and', 'cytonuclear', 'stabilization', 'notwithstanding', 'many', 'recent', 'insights', 'much', 'remains', 'to', 'be', 'learned', 'about', 'many', 'aspects', 'of', 'polyploid', 'evolution', 'including', 'the', 'role', 'of', 'transposable', 'elements', 'in', 'structural', 'and', 'regulatory', 'gene', 'evolution', 'processes', 'and', 'significance', 'of', 'epigenetic', 'silencing', 'underlying', 'controls', 'of', 'chromosome', 'pairing', 'mechanisms', 'and', 'functional', 'significance', 'of', 'rapid', 'genome', 'changes', 'cytonuclear', 'accommodation', 'and', 'coordination', 'of', 'regulatory', 'factors', 'contributed', 'by', 'two', 'sometimes', 'divergent', 'progenitor', 'genomes', 'continued', 'application', 'of', 'molecular', 'genetic', 'approaches', 'to', 'questions', 'of', 'polyploid', 'genome', 'evolution', 'holds', 'promise', 'for', 'producing', 'lasting', 'insight', 'into', 'processes', 'by', 'which', 'novel', 'genotypes', 'are', 'generated', 'and', 'ultimately', 'into', 'how', 'polyploidy', 'facilitates', 'evolution', 'and', 'adaptation'], ['histone', 'lysine', 'methylation', 'has', 'been', 'proposed', 'to', 'provide', 'major', 'switch', 'for', 'the', 'functional', 'organization', 'of', 'chromosomal', 'subdomains', 'here', 'we', 'show', 'that', 'the', 'murine', 'suv', 'histone', 'hmtases', 'govern', 'methylation', 'at', 'pericentric', 'heterochromatin', 'and', 'induce', 'specialized', 'histone', 'methylation', 'pattern', 'that', 'differs', 'from', 'the', 'broad', 'methylation', 'present', 'at', 'other', 'chromosomal', 'regions', 'suv', 'deficient', 'mice', 'display', 'severely', 'impaired', 'viability', 'and', 'chromosomal', 'instabilities', 'that', 'are', 'associated', 'with', 'an', 'increased', 'tumor', 'risk', 'and', 'perturbed', 'chromosome', 'interactions', 'during', 'male', 'meiosis', 'these', 'in', 'vivo', 'data', 'assign', 'crucial', 'role', 'for', 'pericentric', 'methylation', 'in', 'protecting', 'genome', 'stability', 'and', 'define', 'the', 'suv', 'hmtases', 'as', 'important', 'epigenetic', 'regulators', 'for', 'mammalian', 'development'], ['transcriptional', 'activation', 'of', 'gene', 'involves', 'an', 'orchestrated', 'recruitment', 'of', 'components', 'of', 'the', 'basal', 'transcription', 'machinery', 'and', 'intermediate', 'factors', 'concomitant', 'with', 'an', 'alteration', 'in', 'local', 'chromatin', 'structure', 'generated', 'by', 'modifications', 'of', 'histone', 'tails', 'and', 'nucleosome', 'remodeling', 'we', 'provide', 'here', 'comprehensive', 'picture', 'of', 'events', 'resulting', 'in', 'transcriptional', 'activation', 'of', 'gene', 'through', 'evaluating', 'the', 'estrogen', 'receptor', 'alpha', 'nr', 'target', 'ps', 'gene', 'promoter', 'in', 'mcf', 'cells', 'this', 'description', 'integrates', 'chromatin', 'remodeling', 'with', 'kinetic', 'evaluation', 'of', 'cyclical', 'networks', 'of', 'association', 'of', 'transcription', 'factors', 'with', 'the', 'promoter', 'as', 'determined', 'by', 'chromatin', 'assays', 'we', 'define', 'the', 'concept', 'of', 'transcriptional', 'clock', 'that', 'directs', 'and', 'achieves', 'the', 'sequential', 'and', 'combinatorial', 'assembly', 'of', 'productive', 'complex', 'on', 'promoter', 'furthermore', 'the', 'unanticipated', 'findings', 'of', 'key', 'roles', 'for', 'histone', 'deacetylases', 'and', 'nucleosome', 'remodeling', 'complexes', 'in', 'limiting', 'transcription', 'implies', 'that', 'transcriptional', 'activation', 'is', 'cyclical', 'process', 'that', 'requires', 'both', 'activating', 'and', 'repressive', 'epigenetic', 'processes'], ['is', 'slow', 'process', 'during', 'which', 'genomic', 'changes', 'progressively', 'alter', 'the', 'hepatocellular', 'phenotype', 'to', 'produce', 'cellular', 'intermediates', 'that', 'evolve', 'into', 'hepatocellular', 'carcinoma', 'during', 'the', 'long', 'pre', 'neoplastic', 'stage', 'in', 'which', 'the', 'liver', 'is', 'often', 'the', 'site', 'of', 'chronic', 'hepatitis', 'cirrhosis', 'or', 'both', 'hepatocyte', 'cycling', 'is', 'accelerated', 'by', 'upregulation', 'of', 'mitogenic', 'pathways', 'in', 'part', 'through', 'epigenetic', 'mechanisms', 'this', 'leads', 'to', 'the', 'production', 'of', 'monoclonal', 'populations', 'of', 'aberrant', 'and', 'dysplastic', 'hepatocytes', 'that', 'have', 'telomere', 'erosion', 'and', 'telomerase', 're', 'expression', 'sometimes', 'microsatellite', 'instability', 'and', 'occasionally', 'structural', 'aberrations', 'in', 'genes', 'and', 'chromosomes', 'development', 'of', 'dysplastic', 'hepatocytes', 'in', 'foci', 'and', 'nodules', 'and', 'emergence', 'of', 'hepatocellular', 'carcinoma', 'are', 'associated', 'with', 'the', 'accumulation', 'of', 'irreversible', 'structural', 'alterations', 'in', 'genes', 'and', 'chromosomes', 'but', 'the', 'genomic', 'basis', 'of', 'the', 'malignant', 'phenotype', 'is', 'heterogeneous', 'the', 'malignant', 'hepatocyte', 'phenotype', 'may', 'be', 'produced', 'by', 'the', 'disruption', 'of', 'number', 'of', 'genes', 'that', 'function', 'in', 'different', 'regulatory', 'pathways', 'producing', 'several', 'molecular', 'variants', 'of', 'hepatocellular', 'carcinoma', 'new', 'strategies', 'should', 'enable', 'these', 'variants', 'to', 'be', 'characterized'], ['enhancer', 'of', 'zeste', 'is', 'polycomb', 'group', 'protein', 'essential', 'for', 'the', 'establishment', 'and', 'maintenance', 'of', 'repression', 'of', 'homeotic', 'and', 'other', 'genes', 'in', 'the', 'early', 'embryo', 'it', 'is', 'found', 'in', 'complex', 'that', 'includes', 'esc', 'and', 'is', 'recruited', 'to', 'polycomb', 'response', 'elements', 'we', 'show', 'that', 'this', 'complex', 'contains', 'activity', 'that', 'methylates', 'lysine', 'and', 'lysine', 'of', 'histone', 'but', 'the', 'activity', 'is', 'lost', 'when', 'the', 'set', 'domain', 'is', 'mutated', 'the', 'lysine', 'position', 'is', 'trimethylated', 'and', 'this', 'mark', 'is', 'closely', 'associated', 'with', 'polycomb', 'binding', 'sites', 'on', 'polytene', 'chromosomes', 'but', 'is', 'also', 'found', 'in', 'centric', 'heterochromatin', 'chromosome', 'and', 'telomeric', 'sites', 'histone', 'methylated', 'in', 'vitro', 'by', 'the', 'esc', 'complex', 'binds', 'specifically', 'to', 'polycomb', 'protein'], ['cytosine', 'dna', 'methylation', 'occurs', 'in', 'the', 'context', 'of', 'cpg', 'dinucleotides', 'in', 'vertebrates', 'aberrant', 'methylation', 'of', 'cpg', 'islands', 'in', 'human', 'tumors', 'has', 'been', 'shown', 'to', 'cause', 'transcriptional', 'silencing', 'of', 'tumor', 'suppressor', 'genes', 'most', 'methods', 'used', 'to', 'analyze', 'cytosine', 'methylation', 'patterns', 'require', 'cumbersome', 'manual', 'techniques', 'that', 'employ', 'gel', 'electrophoresis', 'restriction', 'enzyme', 'digestion', 'radiolabeled', 'dntps', 'or', 'hybridization', 'probes', 'the', 'development', 'of', 'high', 'throughput', 'technology', 'for', 'the', 'analysis', 'of', 'dna', 'methylation', 'would', 'significantly', 'expand', 'our', 'ability', 'to', 'derive', 'molecular', 'information', 'from', 'clinical', 'specimens', 'this', 'study', 'describes', 'high', 'throughput', 'quantitative', 'methylation', 'assay', 'that', 'utilizes', 'fluorescence', 'based', 'real', 'time', 'pcr', 'taqman', 'technology', 'that', 'requires', 'no', 'further', 'manipulations', 'after', 'the', 'pcr', 'step', 'methy', 'light', 'is', 'highly', 'sensitive', 'assay', 'capable', 'of', 'detecting', 'methylated', 'alleles', 'in', 'the', 'presence', 'of', 'fold', 'excess', 'of', 'unmethylated', 'alleles', 'the', 'assay', 'is', 'also', 'highly', 'quantitative', 'and', 'can', 'very', 'accurately', 'determine', 'the', 'relative', 'prevalence', 'of', 'particular', 'pattern', 'of', 'dna', 'methylation', 'we', 'show', 'that', 'methylight', 'can', 'distinguish', 'between', 'mono', 'allelic', 'and', 'bi', 'allelic', 'methylation', 'of', 'the', 'mlh', 'mismatch', 'repair', 'gene', 'in', 'human', 'colorectal', 'tumor', 'specimens', 'the', 'development', 'of', 'this', 'technique', 'should', 'considerably', 'enhance', 'our', 'ability', 'to', 'rapidly', 'and', 'accurately', 'generate', 'epigenetic', 'profiles', 'of', 'tumor', 'samples'], ['cancer', 'associated', 'dna', 'hypomethylation', 'is', 'as', 'prevalent', 'as', 'cancer', 'linked', 'but', 'these', 'two', 'types', 'of', 'epigenetic', 'abnormalities', 'usually', 'seem', 'to', 'affect', 'different', 'dna', 'sequences', 'much', 'more', 'of', 'the', 'genome', 'is', 'generally', 'subject', 'to', 'rather', 'than', 'overmethylation', 'genomic', 'in', 'cancer', 'has', 'been', 'observed', 'most', 'often', 'in', 'cpg', 'islands', 'in', 'gene', 'regions', 'in', 'contrast', 'very', 'frequent', 'hypomethylation', 'is', 'seen', 'in', 'both', 'highly', 'and', 'moderately', 'repeated', 'dna', 'sequences', 'in', 'cancer', 'including', 'heterochromatic', 'dna', 'repeats', 'dispersed', 'and', 'endogenous', 'retroviral', 'elements', 'also', 'unique', 'sequences', 'including', 'transcription', 'control', 'sequences', 'are', 'often', 'subject', 'to', 'cancer', 'associated', 'the', 'high', 'frequency', 'of', 'cancer', 'linked', 'dna', 'hypomethylation', 'the', 'nature', 'of', 'the', 'affected', 'sequences', 'and', 'the', 'absence', 'of', 'associations', 'with', 'dna', 'are', 'consistent', 'with', 'an', 'independent', 'role', 'for', 'dna', 'in', 'cancer', 'formation', 'or', 'tumor', 'progression', 'increased', 'karyotypic', 'instability', 'and', 'activation', 'of', 'tumor', 'promoting', 'genes', 'by', 'cis', 'or', 'trans', 'effects', 'that', 'might', 'include', 'altered', 'heterochromatin', 'euchromatin', 'interactions', 'may', 'be', 'important', 'consequences', 'of', 'dna', 'hypomethylation', 'which', 'favor', 'oncogenesis', 'the', 'relationship', 'of', 'dna', 'hypomethylation', 'to', 'tumorigenesis', 'is', 'important', 'to', 'be', 'considered', 'in', 'the', 'light', 'of', 'cancer', 'therapies', 'involving', 'decreasing', 'dna', 'methylation', 'inducing', 'dna', 'hypomethylation', 'may', 'have', 'short', 'term', 'anticancer', 'effects', 'but', 'might', 'also', 'help', 'speed', 'tumor', 'progression', 'from', 'cancer', 'cells', 'surviving', 'the', 'dna', 'demethylation', 'chemotherapy'], ['sirtuins', 'sir', 'like', 'proteins', 'are', 'present', 'in', 'prokaryotes', 'and', 'eukaryotes', 'here', 'two', 'new', 'human', 'sirtuins', 'sirt', 'and', 'sirt', 'are', 'found', 'to', 'be', 'similar', 'to', 'particular', 'subset', 'of', 'insect', 'nematode', 'plant', 'and', 'protozoan', 'sirtuins', 'molecular', 'phylogenetic', 'analysis', 'of', 'sirtuin', 'conserved', 'core', 'domain', 'sequences', 'from', 'diverse', 'array', 'of', 'organisms', 'including', 'archaeans', 'bacteria', 'yeasts', 'plants', 'protozoans', 'and', 'metazoans', 'shows', 'that', 'eukaryotic', 'sir', 'like', 'proteins', 'group', 'into', 'four', 'main', 'branches', 'designated', 'here', 'as', 'classes', 'iv', 'prokaryotic', 'sirtuins', 'include', 'members', 'of', 'classes', 'ii', 'and', 'iii', 'fifth', 'class', 'of', 'sirtuin', 'is', 'present', 'in', 'gram', 'positive', 'bacteria', 'and', 'thermotoga', 'maritima', 'saccharomyces', 'cerevisiae', 'has', 'five', 'class', 'sirtuins', 'caenorhabditis', 'elegans', 'and', 'drosophila', 'melanogaster', 'have', 'sirtuin', 'genes', 'from', 'classes', 'ii', 'and', 'tv', 'the', 'seven', 'human', 'sirtuin', 'genes', 'include', 'all', 'four', 'classes', 'sirt', 'sirt', 'and', 'sirt', 'are', 'class', 'sirt', 'is', 'class', 'ii', 'sirt', 'is', 'class', 'iii', 'and', 'sirt', 'and', 'sirt', 'are', 'class', 'iv', 'academic', 'press'], ['chromatin', 'is', 'the', 'physiologically', 'relevant', 'substrate', 'for', 'all', 'genetic', 'processes', 'inside', 'the', 'nuclei', 'of', 'eukaryotic', 'cells', 'dynamic', 'changes', 'in', 'the', 'local', 'and', 'global', 'organization', 'of', 'chromatin', 'are', 'emerging', 'as', 'key', 'regulators', 'of', 'genomic', 'function', 'indeed', 'multitude', 'of', 'signals', 'from', 'outside', 'and', 'inside', 'the', 'cell', 'converges', 'on', 'this', 'gigantic', 'signaling', 'platform', 'numerous', 'post', 'translational', 'modifications', 'of', 'histones', 'the', 'main', 'protein', 'components', 'of', 'chromatin', 'have', 'been', 'documented', 'and', 'analyzed', 'in', 'detail', 'these', 'marks', 'appear', 'to', 'crucially', 'mediate', 'the', 'functional', 'activity', 'of', 'the', 'genome', 'in', 'response', 'to', 'upstream', 'signaling', 'pathways', 'different', 'layers', 'of', 'cross', 'talk', 'between', 'several', 'components', 'of', 'this', 'complex', 'regulatory', 'system', 'are', 'emerging', 'and', 'these', 'epigenetic', 'circuits', 'are', 'the', 'focus', 'of', 'this', 'review'], ['allelic', 'loss', 'at', 'the', 'short', 'arm', 'of', 'chromosome', 'is', 'one', 'of', 'the', 'most', 'common', 'and', 'earliest', 'events', 'in', 'the', 'pathogenesis', 'of', 'lung', 'cancer', 'and', 'is', 'observed', 'in', 'more', 'than', 'of', 'small', 'cell', 'lung', 'cancers', 'sclcs', 'and', 'in', 'of', 'non', 'small', 'cell', 'lung', 'cancers', 'nsclcs', 'frequent', 'and', 'early', 'loss', 'of', 'heterozygosity', 'and', 'the', 'presence', 'of', 'homozygous', 'deletions', 'suggested', 'critical', 'role', 'of', 'the', 'region', 'in', 'tumorigenesis', 'and', 'region', 'of', 'common', 'homozygous', 'deletion', 'in', 'was', 'narrowed', 'to', 'kb', 'ref', 'several', 'putative', 'tumour', 'suppressor', 'genes', 'located', 'at', 'have', 'been', 'characterized', 'but', 'none', 'of', 'these', 'genes', 'appear', 'to', 'be', 'altered', 'in', 'lung', 'cancer', 'here', 'we', 'describe', 'the', 'cloning', 'and', 'of', 'human', 'ras', 'effector', 'homologue', 'rassf', 'located', 'in', 'the', 'kb', 'region', 'of', 'minimal', 'homozygous', 'deletion', 'we', 'identified', 'three', 'transcripts', 'and', 'derived', 'from', 'alternative', 'splicing', 'and', 'promoter', 'usage', 'the', 'major', 'transcripts', 'and', 'were', 'expressed', 'in', 'all', 'normal', 'tissues', 'transcript', 'was', 'missing', 'in', 'all', 'sclc', 'cell', 'lines', 'analysed', 'and', 'in', 'several', 'other', 'cancer', 'cell', 'lines', 'loss', 'of', 'expression', 'was', 'correlated', 'with', 'methylation', 'of', 'the', 'cpg', 'island', 'promoter', 'sequence', 'of', 'rassf', 'the', 'promoter', 'was', 'highly', 'methylated', 'in', 'of', 'primary', 'lung', 'tumours', 'and', 'of', 'tumours', 'analysed', 'carried', 'missense', 'mutations', 're', 'expression', 'of', 'transcript', 'in', 'lung', 'carcinoma', 'cells', 'reduced', 'colony', 'formation', 'suppressed', 'anchorage', 'independent', 'growth', 'and', 'inhibited', 'tumour', 'formation', 'in', 'nude', 'mice', 'these', 'characteristics', 'indicate', 'potential', 'role', 'for', 'rassf', 'asa', 'lung', 'tumour', 'suppresor', 'gene'], ['deposition', 'of', 'the', 'major', 'histone', 'is', 'coupled', 'to', 'dna', 'synthesis', 'during', 'dna', 'replication', 'and', 'possibly', 'dna', 'repair', 'whereas', 'histone', 'variant', 'serves', 'as', 'the', 'replacement', 'variant', 'for', 'the', 'dna', 'synthesis', 'independent', 'deposition', 'pathway', 'to', 'address', 'how', 'histories', 'and', 'are', 'deposited', 'into', 'chromatin', 'through', 'distinct', 'pathways', 'we', 'have', 'purified', 'deposition', 'machineries', 'for', 'these', 'histories', 'the', 'and', 'complexes', 'contain', 'distinct', 'histone', 'chaperones', 'caf', 'and', 'hira', 'that', 'we', 'show', 'are', 'necessary', 'to', 'mediate', 'dna', 'synthesis', 'dependent', 'and', 'independent', 'nucleosome', 'assembly', 'respectively', 'notably', 'these', 'complexes', 'possess', 'one', 'molecule', 'each', 'of', 'and', 'suggesting', 'that', 'histories', 'and', 'exist', 'as', 'dimeric', 'units', 'that', 'are', 'important', 'intermediates', 'in', 'nucleosome', 'formation', 'this', 'finding', 'provides', 'new', 'insights', 'into', 'possible', 'mechanisms', 'for', 'maintenance', 'of', 'epigenetic', 'information', 'after', 'chromatin', 'duplication'], ['dynamic', 'epigenetic', 'modification', 'of', 'the', 'genome', 'occurs', 'during', 'early', 'development', 'of', 'the', 'mouse', 'active', 'demethylation', 'of', 'the', 'paternal', 'genome', 'occurs', 'in', 'the', 'zygote', 'followed', 'by', 'passive', 'demethylation', 'during', 'cleavage', 'stages', 'and', 'de', 'novo', 'methylation', 'which', 'is', 'thought', 'to', 'happen', 'after', 'implantation', 'we', 'have', 'investigated', 'these', 'processes', 'by', 'using', 'indirect', 'with', 'an', 'antibody', 'to', 'methyl', 'cytosine', 'in', 'contrast', 'to', 'previous', 'work', 'we', 'show', 'that', 'demethylation', 'of', 'the', 'male', 'pronucleus', 'is', 'completed', 'within', 'of', 'fertilisation', 'this', 'activity', 'is', 'intricately', 'linked', 'with', 'and', 'not', 'separable', 'from', 'pronucleus', 'formation', 'in', 'conditions', 'permissive', 'for', 'polyspermy', 'up', 'to', 'five', 'male', 'pronuclei', 'underwent', 'demethylation', 'in', 'the', 'same', 'oocyte', 'paternal', 'demethylation', 'in', 'fertilised', 'oocytes', 'deficient', 'for', 'mbd', 'the', 'only', 'candidate', 'demethylase', 'occurred', 'normally', 'passive', 'loss', 'of', 'methylation', 'occurred', 'in', 'stepwise', 'fashion', 'up', 'to', 'the', 'morulae', 'stage', 'without', 'any', 'evidence', 'of', 'spatial', 'de', 'novo', 'methylation', 'was', 'observed', 'specifically', 'in', 'the', 'inner', 'cell', 'mass', 'icm', 'but', 'not', 'in', 'the', 'trophectoderm', 'of', 'the', 'blastocyst', 'and', 'hence', 'may', 'have', 'an', 'important', 'role', 'in', 'early', 'lineage', 'specification', 'this', 'is', 'the', 'first', 'complete', 'and', 'detailed', 'analysis', 'of', 'the', 'epigenetic', 'reprogramming', 'cycle', 'during', 'preimplantation', 'development', 'the', 'three', 'phases', 'of', 'methylation', 'reprogramming', 'may', 'have', 'roles', 'in', 'imprinting', 'the', 'control', 'of', 'gene', 'expression', 'and', 'the', 'establishment', 'of', 'nuclear', 'totipotency', 'elsevier', 'science'], ['azacytidine', 'was', 'first', 'synthesized', 'almost', 'years', 'ago', 'it', 'was', 'demonstrated', 'to', 'have', 'wide', 'range', 'of', 'anti', 'metabolic', 'activities', 'when', 'tested', 'against', 'cultured', 'cancer', 'cells', 'and', 'to', 'be', 'an', 'effective', 'agent', 'for', 'acute', 'myelogenous', 'leukemia', 'however', 'because', 'of', 'azacytidine', 'general', 'toxicity', 'other', 'nucleoside', 'analogs', 'were', 'favored', 'as', 'therapeutics', 'the', 'finding', 'that', 'azacytidine', 'was', 'incorporated', 'into', 'dna', 'and', 'that', 'when', 'present', 'in', 'dna', 'it', 'inhibited', 'dna', 'methylation', 'led', 'to', 'widespread', 'use', 'of', 'azacytidine', 'and', 'aza', 'deoxycytidine', 'decitabine', 'to', 'demonstrate', 'the', 'correlation', 'between', 'loss', 'of', 'methylation', 'in', 'specific', 'gene', 'regions', 'and', 'activation', 'of', 'the', 'associated', 'genes', 'there', 'is', 'now', 'revived', 'interest', 'in', 'the', 'use', 'of', 'decitabine', 'as', 'therapeutic', 'agent', 'for', 'cancers', 'in', 'which', 'epigenetic', 'silencing', 'of', 'critical', 'regulatory', 'genes', 'has', 'occurred', 'here', 'the', 'current', 'status', 'of', 'our', 'understanding', 'of', 'the', 'mechanism', 'by', 'which', 'azacytosine', 'residues', 'in', 'dna', 'inhibit', 'dna', 'methylation', 'is', 'reviewed', 'with', 'an', 'emphasis', 'on', 'the', 'interactions', 'of', 'these', 'residues', 'with', 'bacterial', 'and', 'mammalian', 'dna', 'cytosine', 'the', 'implications', 'of', 'these', 'mechanistic', 'studies', 'for', 'development', 'of', 'less', 'toxic', 'inhibitors', 'of', 'dna', 'methylation', 'are', 'discussed'], ['the', 'enzyme', 'catalyzed', 'acetylation', 'of', 'the', 'terminal', 'tail', 'domains', 'of', 'core', 'histones', 'provides', 'rich', 'potential', 'source', 'of', 'epigenetic', 'information', 'this', 'may', 'be', 'used', 'both', 'to', 'mediate', 'transient', 'changes', 'in', 'transcription', 'through', 'modification', 'of', 'promoter', 'proximal', 'nucleosomes', 'and', 'for', 'the', 'longer', 'term', 'maintenance', 'and', 'modulation', 'of', 'patterns', 'of', 'gene', 'expression', 'the', 'latter', 'may', 'be', 'achieved', 'by', 'setting', 'specific', 'patterns', 'of', 'histone', 'acetylation', 'perhaps', 'involving', 'acetylation', 'of', 'particular', 'lysine', 'residues', 'across', 'relatively', 'large', 'chromatin', 'domains', 'the', 'histone', 'acetylating', 'and', 'deacetylating', 'enzymes', 'hats', 'and', 'hdacs', 'respectively', 'can', 'be', 'targeted', 'to', 'specific', 'regions', 'of', 'the', 'genome', 'and', 'show', 'varying', 'degrees', 'of', 'substrate', 'specificity', 'properties', 'that', 'are', 'consistent', 'with', 'role', 'in', 'maintaining', 'dynamic', 'acetylation', 'based', 'epigenetic', 'code', 'the', 'code', 'may', 'be', 'read', 'ie', 'exert', 'functional', 'effect', 'either', 'through', 'non', 'histone', 'proteins', 'that', 'bind', 'in', 'an', 'acetylation', 'dependent', 'manner', 'or', 'through', 'direct', 'effects', 'on', 'chromatin', 'structure', 'recent', 'evidence', 'raises', 'the', 'interesting', 'possibility', 'that', 'an', 'acetylation', 'based', 'code', 'may', 'operate', 'through', 'both', 'mitosis', 'and', 'meiosis', 'providing', 'possible', 'mechanism', 'for', 'germ', 'line', 'transmission', 'of', 'epigenetic', 'changes', 'bioessays', 'john', 'wiley', 'sons', 'inc'], ['we', 'have', 'come', 'long', 'way', 'since', 'the', 'first', 'reports', 'of', 'the', 'existence', 'of', 'aberrant', 'dna', 'methylation', 'in', 'human', 'cancer', 'of', 'cpg', 'islands', 'located', 'in', 'the', 'promoter', 'regions', 'of', 'tumor', 'suppressor', 'genes', 'is', 'now', 'firmly', 'established', 'as', 'an', 'important', 'mechanism', 'for', 'gene', 'inactivation', 'cpg', 'island', 'has', 'been', 'described', 'in', 'almost', 'every', 'tumor', 'type', 'many', 'cellular', 'pathways', 'are', 'inactivated', 'by', 'this', 'type', 'of', 'epigenetic', 'lesion', 'dna', 'repair', 'hmlh', 'mgmt', 'cell', 'cycle', 'ink', 'ink', 'arf', 'apoptosis', 'dapk', 'cell', 'adherence', 'cdh', 'cdh', 'detoxification', 'gstp', 'etc', 'however', 'we', 'still', 'know', 'little', 'of', 'the', 'mechanisms', 'of', 'aberrant', 'methylation', 'and', 'why', 'certain', 'genes', 'are', 'selected', 'over', 'others', 'is', 'not', 'an', 'isolated', 'layer', 'of', 'epigenetic', 'control', 'but', 'is', 'linked', 'to', 'the', 'other', 'pieces', 'of', 'the', 'puzzle', 'such', 'as', 'methyl', 'binding', 'proteins', 'dna', 'and', 'histone', 'deacetylase', 'but', 'our', 'understanding', 'of', 'the', 'degree', 'of', 'specificity', 'of', 'these', 'epigenetic', 'layers', 'in', 'the', 'silencing', 'of', 'specific', 'tumor', 'suppressor', 'genes', 'remains', 'incomplete', 'the', 'explosion', 'of', 'user', 'friendly', 'technologies', 'has', 'given', 'rise', 'to', 'rapidly', 'increasing', 'list', 'of', 'hypermethylated', 'genes', 'careful', 'functional', 'and', 'genetic', 'studies', 'are', 'necessary', 'to', 'determine', 'which', 'events', 'are', 'truly', 'relevant', 'for', 'human', 'tumorigenesis', 'the', 'development', 'of', 'cpg', 'island', 'profiles', 'for', 'every', 'form', 'of', 'human', 'tumors', 'has', 'yielded', 'valuable', 'pilot', 'clinical', 'data', 'in', 'monitoring', 'and', 'treating', 'cancer', 'patients', 'based', 'in', 'our', 'knowledge', 'of', 'dna', 'methylation', 'basic', 'and', 'translational', 'will', 'both', 'be', 'needed', 'in', 'the', 'near', 'future', 'to', 'fully', 'understand', 'the', 'mechanisms', 'roles', 'and', 'uses', 'of', 'cpg', 'island', 'in', 'human', 'cancer', 'the', 'expectations', 'are', 'high'], ['genome', 'wide', 'epigenetic', 'reprogramming', 'in', 'mammalian', 'germ', 'cells', 'zygote', 'and', 'early', 'embryos', 'plays', 'crucial', 'role', 'in', 'regulating', 'genome', 'functions', 'at', 'critical', 'stages', 'of', 'development', 'we', 'show', 'here', 'that', 'mouse', 'primordial', 'germ', 'cells', 'pgcs', 'exhibit', 'dynamic', 'changes', 'in', 'epigenetic', 'modifications', 'between', 'days', 'and', 'post', 'coitum', 'dpc', 'first', 'contrary', 'to', 'previous', 'suggestions', 'we', 'show', 'that', 'pgcs', 'do', 'indeed', 'acquire', 'genome', 'wide', 'de', 'novo', 'methylation', 'during', 'early', 'development', 'and', 'migration', 'into', 'the', 'genital', 'ridge', 'however', 'following', 'their', 'entry', 'into', 'the', 'genital', 'ridge', 'there', 'is', 'rapid', 'erasure', 'of', 'dna', 'methylation', 'of', 'regions', 'within', 'imprinted', 'and', 'non', 'imprinted', 'loci', 'for', 'most', 'genes', 'the', 'erasure', 'commences', 'simultaneously', 'in', 'pgcs', 'in', 'both', 'male', 'and', 'female', 'embryos', 'which', 'is', 'completed', 'within', 'day', 'of', 'development', 'based', 'on', 'the', 'kinetics', 'of', 'this', 'process', 'we', 'suggest', 'that', 'this', 'is', 'an', 'active', 'demethylation', 'process', 'initiated', 'upon', 'the', 'entry', 'of', 'pgcs', 'into', 'the', 'gonadal', 'anlagen', 'the', 'timing', 'of', 'reprogramming', 'in', 'pgcs', 'is', 'crucial', 'since', 'it', 'ensures', 'that', 'germ', 'cells', 'of', 'both', 'sexes', 'acquire', 'an', 'equivalent', 'epigenetic', 'state', 'prior', 'to', 'the', 'differentiation', 'of', 'the', 'definitive', 'male', 'and', 'female', 'germ', 'cells', 'in', 'which', 'new', 'parental', 'imprints', 'are', 'established', 'subsequently', 'some', 'repetitive', 'elements', 'however', 'show', 'incomplete', 'erasure', 'which', 'may', 'be', 'essential', 'for', 'chromosome', 'stability', 'and', 'for', 'preventing', 'activation', 'of', 'transposons', 'to', 'reduce', 'the', 'risk', 'of', 'germline', 'mutations', 'aberrant', 'epigenetic', 'reprogramming', 'in', 'the', 'germ', 'line', 'would', 'cause', 'the', 'inheritance', 'of', 'epimutations', 'that', 'may', 'have', 'consequences', 'for', 'human', 'diseases', 'as', 'suggested', 'by', 'studies', 'on', 'mouse', 'models', 'elsevier', 'science', 'ireland', 'ltd', 'all', 'rights', 'reserved'], ['aberrant', 'wnt', 'pathway', 'signaling', 'is', 'an', 'early', 'progression', 'event', 'in', 'of', 'colorectal', 'cancers', 'it', 'occurs', 'through', 'mutations', 'mainly', 'of', 'apc', 'and', 'less', 'often', 'of', 'ctnnb', 'encoding', 'beta', 'catenin', 'or', 'axin', 'encoding', 'axin', 'also', 'known', 'as', 'conductin', 'these', 'mutations', 'allow', 'ligand', 'independent', 'wnt', 'signaling', 'that', 'culminates', 'in', 'abnormal', 'accumulation', 'of', 'free', 'beta', 'catenin', 'in', 'the', 'nucleus', 'we', 'previously', 'identified', 'frequent', 'promoter', 'and', 'gene', 'silencing', 'of', 'the', 'genes', 'encoding', 'secreted', 'frizzled', 'related', 'proteins', 'sfrps', 'in', 'colorectal', 'cancer', 'sfrps', 'possess', 'domain', 'similar', 'to', 'one', 'in', 'the', 'wnt', 'receptor', 'frizzled', 'proteins', 'and', 'can', 'inhibit', 'wnt', 'receptor', 'binding', 'to', 'downregulate', 'pathway', 'signaling', 'during', 'development', 'here', 'we', 'show', 'that', 'restoration', 'of', 'sfrp', 'function', 'in', 'colorectal', 'cancer', 'cells', 'attenuates', 'wnt', 'signaling', 'even', 'in', 'the', 'presence', 'of', 'downstream', 'mutations', 'we', 'also', 'show', 'that', 'the', 'epigenetic', 'loss', 'of', 'sfrp', 'function', 'occurs', 'early', 'in', 'colorectal', 'cancer', 'progression', 'and', 'may', 'thus', 'provide', 'constitutive', 'wnt', 'signaling', 'that', 'is', 'required', 'to', 'complement', 'downstream', 'mutations', 'in', 'the', 'evolution', 'of', 'colorectal', 'cancer'], ['methylation', 'of', 'position', 'specific', 'lysine', 'residues', 'in', 'histone', 'termini', 'is', 'central', 'modification', 'for', 'regulating', 'epigenetic', 'transitions', 'in', 'chromatin', 'each', 'methylatable', 'lysine', 'residue', 'can', 'exist', 'in', 'mono', 'di', 'or', 'trimethylated', 'state', 'thereby', 'extending', 'the', 'indexing', 'potential', 'of', 'this', 'particular', 'modification', 'here', 'we', 'examine', 'all', 'possible', 'methylation', 'states', 'for', 'histone', 'lysine', 'and', 'lysine', 'in', 'mammalian', 'chromatin', 'using', 'highly', 'specific', 'antibodies', 'together', 'with', 'quantitative', 'mass', 'spectrometry', 'we', 'demonstrate', 'that', 'pericentric', 'heterochromatin', 'is', 'selectively', 'enriched', 'for', 'monomethylation', 'and', 'trimethylation', 'this', 'heterochromatic', 'methylation', 'profile', 'is', 'dependent', 'on', 'the', 'suv', 'histone', 'hmtases', 'but', 'independent', 'of', 'the', 'euchromatic', 'hmtase', 'in', 'suv', 'double', 'null', 'cells', 'pericentric', 'heterochromatin', 'is', 'converted', 'to', 'alternative', 'methylation', 'imprints', 'and', 'accumulates', 'trimethylation', 'and', 'monomethylation', 'our', 'data', 'underscore', 'the', 'selective', 'presence', 'of', 'distinct', 'histone', 'lysine', 'methylation', 'states', 'in', 'partitioning', 'chromosomal', 'subdomains', 'but', 'also', 'reveal', 'surprising', 'plasticity', 'in', 'propagating', 'methylation', 'patterns', 'in', 'eukaryotic', 'chromatin'], ['during', 'the', 'development', 'of', 'multicellular', 'organisms', 'cells', 'become', 'different', 'from', 'one', 'another', 'by', 'changing', 'their', 'genetic', 'program', 'in', 'response', 'to', 'transient', 'stimuli', 'long', 'after', 'the', 'stimulus', 'is', 'gone', 'cellular', 'memory', 'mechanisms', 'enable', 'cells', 'to', 'remember', 'their', 'chosen', 'fate', 'over', 'many', 'cell', 'divisions', 'the', 'polycomb', 'and', 'trithorax', 'groups', 'of', 'proteins', 'respectively', 'work', 'to', 'maintain', 'repressed', 'or', 'active', 'transcription', 'states', 'of', 'developmentally', 'important', 'genes', 'through', 'many', 'rounds', 'of', 'cell', 'division', 'here', 'we', 'review', 'current', 'ideas', 'on', 'the', 'protein', 'and', 'dna', 'components', 'of', 'this', 'transcriptional', 'memory', 'system', 'and', 'how', 'they', 'interact', 'dynamically', 'with', 'each', 'other', 'to', 'orchestrate', 'cellular', 'memory', 'for', 'several', 'hundred', 'genes'], ['dna', 'methylation', 'has', 'recently', 'moved', 'to', 'centre', 'stage', 'in', 'the', 'aetiology', 'of', 'human', 'syndromes', 'such', 'as', 'the', 'fragile', 'icf', 'and', 'rett', 'syndromes', 'these', 'diseases', 'result', 'from', 'the', 'misregulation', 'of', 'genes', 'that', 'occurs', 'with', 'the', 'toss', 'of', 'appropriate', 'epigenetic', 'controls', 'during', 'neuronal', 'development', 'recent', 'advances', 'have', 'connected', 'dna', 'methylation', 'to', 'chromatin', 'remodeling', 'enzymes', 'and', 'understanding', 'this', 'link', 'will', 'be', 'central', 'to', 'the', 'design', 'of', 'new', 'therapeutic', 'tools'], ['the', 'centromere', 'is', 'chromosomal', 'locus', 'that', 'ensures', 'delivery', 'of', 'one', 'copy', 'of', 'each', 'chromosome', 'to', 'each', 'daughter', 'at', 'cell', 'division', 'efforts', 'to', 'understand', 'the', 'nature', 'and', 'specification', 'of', 'the', 'centromere', 'have', 'demonstrated', 'that', 'this', 'central', 'element', 'for', 'ensuring', 'inheritance', 'is', 'itself', 'epigenetically', 'determined', 'the', 'kinetochore', 'the', 'protein', 'complex', 'assembled', 'at', 'each', 'centromere', 'serves', 'as', 'the', 'attachment', 'site', 'for', 'spindle', 'microtubules', 'and', 'the', 'site', 'at', 'which', 'motors', 'generate', 'forces', 'to', 'power', 'chromosome', 'movement', 'unattached', 'kinetochores', 'are', 'also', 'the', 'signal', 'generators', 'for', 'the', 'mitotic', 'checkpoint', 'which', 'arrests', 'mitosis', 'until', 'all', 'kinetochores', 'have', 'correctly', 'attached', 'to', 'spindle', 'microtubules', 'thereby', 'representing', 'the', 'major', 'cell', 'cycle', 'control', 'mechanism', 'protecting', 'against', 'loss', 'of', 'chromosome', 'aneuploidy'], ['on', 'the', 'histone', 'tail', 'lys', 'and', 'lys', 'are', 'both', 'methylation', 'sites', 'associated', 'with', 'epigenetic', 'repression', 'and', 'reside', 'within', 'highly', 'related', 'sequence', 'motif', 'arks', 'here', 'we', 'show', 'that', 'the', 'chromodomain', 'proteins', 'polycomb', 'pc', 'and', 'hp', 'heterochromatin', 'protein', 'are', 'highly', 'discriminatory', 'for', 'binding', 'to', 'these', 'sites', 'in', 'vivo', 'and', 'in', 'vitro', 'in', 'drosophila', 'cells', 'and', 'on', 'polytene', 'chromosomes', 'methyl', 'lys', 'and', 'pc', 'are', 'both', 'excluded', 'from', 'areas', 'that', 'are', 'enriched', 'in', 'methyl', 'lys', 'and', 'hp', 'swapping', 'of', 'the', 'chromodomain', 'regions', 'of', 'pc', 'and', 'hp', 'is', 'sufficient', 'for', 'switching', 'the', 'nuclear', 'localization', 'patterns', 'of', 'these', 'factors', 'indicating', 'role', 'for', 'their', 'chromodomains', 'in', 'both', 'target', 'site', 'binding', 'and', 'discrimination', 'to', 'better', 'understand', 'the', 'molecular', 'basis', 'for', 'the', 'selection', 'of', 'methyl', 'lysine', 'binding', 'sites', 'we', 'solved', 'the', 'angstrom', 'structure', 'of', 'the', 'pc', 'chromodomain', 'in', 'complex', 'with', 'peptide', 'bearing', 'trimethyl', 'lys', 'and', 'compared', 'it', 'with', 'our', 'previously', 'determined', 'structure', 'of', 'the', 'hp', 'chromodomain', 'in', 'complex', 'with', 'peptide', 'bearing', 'trimethyl', 'lys', 'the', 'pc', 'chromodomain', 'distinguishes', 'its', 'methylation', 'target', 'on', 'the', 'tail', 'via', 'an', 'extended', 'recognition', 'groove', 'that', 'binds', 'five', 'additional', 'residues', 'preceding', 'the', 'arks', 'motif'], ['dna', 'methylation', 'is', 'essential', 'for', 'the', 'control', 'of', 'number', 'of', 'biological', 'mechanisms', 'in', 'mammals', 'mammalian', 'development', 'is', 'accompanied', 'by', 'two', 'major', 'waves', 'of', 'genome', 'wide', 'demethylation', 'and', 'remethylation', 'one', 'during', 'germ', 'cell', 'development', 'and', 'the', 'other', 'after', 'fertilisation', 'most', 'previous', 'studies', 'have', 'suggested', 'that', 'the', 'genome', 'wide', 'demethylation', 'observed', 'after', 'fertilisation', 'occurs', 'passively', 'that', 'is', 'by', 'the', 'lack', 'of', 'maintenance', 'methylation', 'following', 'dna', 'replication', 'and', 'cell', 'division', 'although', 'one', 'other', 'study', 'has', 'reported', 'that', 'replication', 'independent', 'demethylation', 'may', 'also', 'occur', 'during', 'early', 'embryogenesis', 'here', 'we', 'report', 'that', 'genes', 'that', 'are', 'highly', 'methylated', 'in', 'sperm', 'are', 'rapidly', 'demethylated', 'in', 'the', 'zygote', 'only', 'hours', 'after', 'fertilisation', 'before', 'the', 'first', 'round', 'of', 'dna', 'replication', 'commences', 'by', 'contrast', 'the', 'oocyte', 'derived', 'maternal', 'alleles', 'are', 'unaffected', 'by', 'this', 'reprogramming', 'they', 'either', 'remain', 'methylated', 'after', 'fertilisation', 'or', 'become', 'further', 'methylated', 'de', 'novo', 'these', 'results', 'provide', 'the', 'first', 'direct', 'evidence', 'for', 'active', 'demethylation', 'of', 'single', 'copy', 'genes', 'in', 'the', 'mammalian', 'zygote', 'and', 'moreover', 'reveal', 'striking', 'asymmetry', 'in', 'epigenetic', 'methylation', 'reprogramming', 'whereas', 'paternally', 'sperm', 'derived', 'sequences', 'are', 'exposed', 'to', 'putative', 'active', 'demethylases', 'in', 'the', 'oocyte', 'cytoplasm', 'maternally', 'oocyte', 'derived', 'sequences', 'are', 'protected', 'from', 'this', 'reaction', 'these', 'results', 'whose', 'generality', 'is', 'supported', 'by', 'findings', 'of', 'mayer', 'et', 'al', 'have', 'important', 'implications', 'for', 'the', 'establishment', 'of', 'biparental', 'genetic', 'totipotency', 'after', 'fertilisation', 'the', 'establishment', 'and', 'maintenance', 'of', 'genomic', 'imprinting', 'and', 'the', 'reprogramming', 'of', 'somatic', 'cells', 'during', 'cloning'], ['aberrant', 'of', 'gene', 'promoters', 'is', 'major', 'mechanism', 'associated', 'with', 'inactivation', 'of', 'tumor', 'suppressor', 'genes', 'in', 'cancer', 'we', 'previously', 'showed', 'this', 'transcriptional', 'silencing', 'to', 'be', 'mediated', 'by', 'both', 'methylation', 'and', 'histone', 'deacetylase', 'activity', 'with', 'methylation', 'being', 'dominant', 'here', 'we', 'have', 'used', 'cdna', 'microarray', 'analysis', 'to', 'screen', 'for', 'genes', 'that', 'are', 'epigenetically', 'silenced', 'in', 'human', 'colorectal', 'cancer', 'by', 'screening', 'over', 'genes', 'we', 'show', 'that', 'our', 'approach', 'can', 'identify', 'substantial', 'number', 'of', 'genes', 'with', 'promoter', 'in', 'given', 'cancer', 'these', 'are', 'distinct', 'from', 'genes', 'with', 'unmethylated', 'promoters', 'for', 'which', 'increased', 'expression', 'is', 'produced', 'by', 'histone', 'deacetylase', 'inhibition', 'alone', 'many', 'of', 'the', 'hypermethylated', 'genes', 'we', 'identified', 'have', 'high', 'potential', 'for', 'roles', 'in', 'tumorigenesis', 'by', 'virtue', 'of', 'their', 'predicted', 'function', 'and', 'chromosome', 'position', 'we', 'also', 'identified', 'group', 'of', 'genes', 'that', 'are', 'preferentially', 'hypermethylated', 'in', 'colorectal', 'cancer', 'and', 'gastric', 'cancer', 'one', 'of', 'these', 'genes', 'sfrp', 'belongs', 'to', 'gene', 'family', 'we', 'show', 'that', 'of', 'four', 'genes', 'in', 'this', 'family', 'occurs', 'very', 'frequently', 'in', 'colorectal', 'cancer', 'providing', 'for', 'unique', 'potential', 'mechanism', 'for', 'loss', 'of', 'tumor', 'suppressor', 'gene', 'function', 'and', 'ii', 'construction', 'of', 'molecular', 'marker', 'panel', 'that', 'could', 'detect', 'virtually', 'all', 'colorectal', 'cancer'], ['gene', 'function', 'in', 'cancer', 'can', 'be', 'disrupted', 'either', 'through', 'genetic', 'alterations', 'which', 'directly', 'mutate', 'or', 'delete', 'genes', 'or', 'epigenetic', 'alterations', 'which', 'alter', 'the', 'heritable', 'state', 'of', 'gene', 'expression', 'the', 'latter', 'events', 'are', 'mediated', 'by', 'formation', 'of', 'repressive', 'chromatin', 'states', 'around', 'gene', 'transcription', 'start', 'sites', 'and', 'an', 'associated', 'gain', 'of', 'methylation', 'in', 'normally', 'unmethylated', 'cpg', 'islands', 'in', 'these', 'regions', 'the', 'genes', 'affected', 'include', 'over', 'half', 'of', 'the', 'tumor', 'suppressor', 'genes', 'that', 'cause', 'familial', 'cancers', 'when', 'mutated', 'in', 'the', 'germline', 'the', 'selective', 'advantage', 'for', 'genetic', 'and', 'epigenetic', 'dysfunction', 'in', 'these', 'genes', 'is', 'very', 'similar', 'the', 'aberrant', 'methylation', 'can', 'begin', 'very', 'early', 'in', 'tumor', 'progression', 'and', 'mediate', 'most', 'of', 'the', 'important', 'pathway', 'abnormalities', 'in', 'cancer', 'including', 'loss', 'of', 'cell', 'cycle', 'control', 'altered', 'function', 'of', 'transcription', 'factors', 'altered', 'receptor', 'function', 'disruption', 'of', 'normal', 'cell', 'cell', 'and', 'cell', 'substratum', 'interaction', 'inactivation', 'of', 'signal', 'transduction', 'pathways', 'loss', 'of', 'apoptotic', 'signals', 'and', 'genetic', 'instability', 'the', 'active', 'role', 'of', 'the', 'aberrant', 'methylation', 'in', 'transcriptional', 'silencing', 'of', 'genes', 'is', 'becoming', 'increasingly', 'understood', 'and', 'involves', 'synergy', 'between', 'the', 'methylation', 'and', 'histone', 'deacetylase', 'hdac', 'activity', 'this', 'synergy', 'can', 'be', 'mediated', 'directly', 'by', 'hdac', 'interaction', 'with', 'dna', 'methylating', 'enzymes', 'and', 'by', 'recruitment', 'through', 'complexes', 'involving', 'methyl', 'cytosine', 'binding', 'proteins', 'in', 'the', 'translational', 'arena', 'the', 'promoter', 'changes', 'hold', 'great', 'promise', 'as', 'dna', 'tumor', 'markers', 'and', 'their', 'potentially', 'reversible', 'state', 'creates', 'target', 'for', 'cancer', 'therapeutic', 'strategies', 'involving', 'gene', 'reactivation'], ['dna', 'methylation', 'plays', 'an', 'important', 'role', 'in', 'mammalian', 'development', 'and', 'correlates', 'with', 'chromatin', 'associated', 'gene', 'silencing', 'the', 'recruitment', 'of', 'mecp', 'to', 'methylated', 'cpg', 'dinucleotides', 'represents', 'major', 'mechanism', 'by', 'which', 'dna', 'methylation', 'can', 'repress', 'transcription', 'mecp', 'silences', 'gene', 'expression', 'partly', 'by', 'recruiting', 'histone', 'deacetylase', 'hdac', 'activity', 'resulting', 'in', 'chromatin', 'remodeling', 'here', 'we', 'show', 'that', 'mecp', 'associates', 'with', 'histone', 'activity', 'in', 'vivo', 'and', 'that', 'this', 'activity', 'is', 'directed', 'against', 'lys', 'of', 'histone', 'two', 'characterized', 'repression', 'domains', 'of', 'mecp', 'are', 'involved', 'in', 'tethering', 'the', 'histone', 'to', 'mecp', 'we', 'asked', 'if', 'mecp', 'can', 'deliver', 'lys', 'methylation', 'to', 'the', 'gene', 'whose', 'activity', 'it', 'represses', 'we', 'show', 'that', 'the', 'presence', 'of', 'mecp', 'on', 'nucleosomes', 'within', 'the', 'repressor', 'region', 'of', 'the', 'gene', 'the', 'differentially', 'methylated', 'domain', 'coincides', 'with', 'an', 'increase', 'in', 'lys', 'methylation', 'our', 'data', 'provide', 'evidence', 'that', 'mecp', 'reinforces', 'repressive', 'chromatin', 'state', 'by', 'acting', 'as', 'bridge', 'between', 'two', 'global', 'epigenetic', 'modifications', 'dna', 'methylation', 'and', 'histone', 'methylation'], ['large', 'set', 'of', 'endogenous', 'small', 'rnas', 'of', 'predominantly', 'to', 'nucleotides', 'was', 'identified', 'in', 'arabidopsis', 'these', 'small', 'rnas', 'resembled', 'micro', 'rnas', 'from', 'animals', 'and', 'were', 'similar', 'in', 'size', 'to', 'small', 'interfering', 'rnas', 'that', 'accumulated', 'during', 'rna', 'silencing', 'triggered', 'by', 'multiple', 'types', 'of', 'inducers', 'among', 'the', 'sequences', 'identified', 'the', 'vast', 'majority', 'arose', 'from', 'intergenic', 'regions', 'although', 'small', 'rnas', 'corresponding', 'to', 'predicted', 'protein', 'coding', 'genes', 'transposon', 'like', 'sequences', 'and', 'structural', 'rna', 'gene', 'also', 'were', 'identified', 'evidence', 'consistent', 'with', 'the', 'derivation', 'of', 'small', 'rnas', 'of', 'both', 'polarities', 'and', 'from', 'highly', 'base', 'paired', 'precursors', 'was', 'obtained', 'through', 'the', 'identification', 'and', 'analysis', 'of', 'clusters', 'of', 'small', 'rna', 'loci', 'the', 'accumulation', 'of', 'specific', 'small', 'rnas', 'was', 'regulated', 'developmentally', 'we', 'propose', 'that', 'arabidopsis', 'small', 'rnas', 'participate', 'in', 'wide', 'range', 'of', 'post', 'transcriptional', 'and', 'epigenetic', 'events'], ['recent', 'results', 'from', 'diverse', 'organisms', 'point', 'to', 'self', 'reinforcing', 'network', 'of', 'interactions', 'among', 'the', 'three', 'best', 'characterized', 'covalent', 'modifications', 'that', 'mark', 'heterochromatin', 'histone', 'hypoacetylation', 'histone', 'lys', 'methylation', 'and', 'cytosine', 'methylation', 'these', 'modification', 'systems', 'suggest', 'mechanistic', 'basis', 'for', 'spreading', 'of', 'heterochromatin', 'over', 'large', 'domains', 'and', 'for', 'stable', 'epigenetic', 'inheritance', 'of', 'the', 'silent', 'state', 'all', 'three', 'modifications', 'used', 'in', 'packaging', 'heterochromatin', 'are', 'also', 'used', 'in', 'stable', 'silencing', 'of', 'euchromatic', 'genes'], ['recent', 'data', 'in', 'humans', 'and', 'animals', 'suggest', 'that', 'assisted', 'reproductive', 'technology', 'art', 'might', 'affect', 'the', 'epigenetics', 'of', 'early', 'embryogenesis', 'and', 'might', 'cause', 'birth', 'defects', 'we', 'report', 'the', 'first', 'evidence', 'to', 'our', 'knowledge', 'that', 'art', 'is', 'associated', 'with', 'human', 'overgrowth', 'syndrome', 'namely', 'beckwith', 'wiedemann', 'syndrome', 'bws', 'in', 'prospective', 'study', 'the', 'prevalence', 'of', 'art', 'was', 'of', 'versus', 'the', 'background', 'rate', 'of', 'in', 'the', 'united', 'states', 'total', 'of', 'seven', 'children', 'with', 'bws', 'were', 'born', 'after', 'art', 'five', 'of', 'whom', 'were', 'conceived', 'after', 'sperm', 'injection', 'molecular', 'studies', 'of', 'six', 'of', 'the', 'children', 'indicate', 'that', 'five', 'of', 'the', 'six', 'have', 'specific', 'epigenetic', 'alterations', 'associated', 'with', 'bws', 'four', 'at', 'lit', 'and', 'one', 'at', 'both', 'lit', 'and', 'we', 'discuss', 'the', 'implications', 'of', 'our', 'finding', 'that', 'art', 'is', 'associated', 'with', 'human', 'overgrowth', 'similar', 'to', 'the', 'large', 'offspring', 'syndrome', 'reported', 'in', 'ruminants'], ['cadmium', 'is', 'heavy', 'metal', 'of', 'considerable', 'environmental', 'and', 'occupational', 'concern', 'cadmium', 'compounds', 'are', 'classified', 'as', 'human', 'carcinogens', 'by', 'several', 'regulatory', 'agencies', 'the', 'most', 'convincing', 'data', 'that', 'cadmium', 'is', 'carcinogenic', 'in', 'humans', 'comes', 'from', 'studies', 'indicating', 'occupational', 'cadmium', 'exposure', 'is', 'associated', 'with', 'lung', 'cancer', 'cadmium', 'exposure', 'has', 'also', 'been', 'linked', 'to', 'human', 'prostate', 'and', 'renal', 'cancer', 'although', 'this', 'linkage', 'is', 'weaker', 'than', 'for', 'lung', 'cancer', 'other', 'target', 'sites', 'of', 'cadmium', 'carcinogenesis', 'in', 'humans', 'such', 'as', 'liver', 'pancreas', 'and', 'stomach', 'are', 'considered', 'equivocal', 'in', 'animals', 'cadmium', 'effectively', 'induces', 'cancers', 'at', 'multiple', 'sites', 'and', 'by', 'various', 'routes', 'cadmium', 'inhalation', 'in', 'rats', 'induces', 'pulmonary', 'adenocarcinomas', 'in', 'accord', 'with', 'its', 'role', 'in', 'human', 'lung', 'cancer', 'cadmium', 'can', 'induce', 'tumors', 'and', 'or', 'preneoplastic', 'lesions', 'within', 'the', 'rat', 'prostate', 'after', 'ingestion', 'or', 'injection', 'at', 'relatively', 'high', 'doses', 'cadmium', 'induces', 'benign', 'testicular', 'tumors', 'in', 'rats', 'but', 'these', 'appear', 'to', 'be', 'due', 'to', 'early', 'toxic', 'lesions', 'and', 'loss', 'of', 'testicular', 'function', 'rather', 'than', 'from', 'specific', 'carcinogenic', 'effect', 'of', 'cadmium', 'like', 'many', 'other', 'metals', 'cadmium', 'salts', 'will', 'induce', 'mesenchymal', 'tumors', 'at', 'the', 'site', 'of', 'subcutaneous', 'or', 'intramuscular', 'injections', 'but', 'the', 'human', 'relevance', 'of', 'these', 'is', 'dubious', 'other', 'targets', 'of', 'cadmium', 'in', 'rodents', 'include', 'the', 'liver', 'adrenal', 'pancreas', 'pituitary', 'and', 'hematopoietic', 'system', 'with', 'the', 'exception', 'of', 'testicular', 'tumors', 'in', 'rodents', 'the', 'mechanisms', 'of', 'cadmium', 'carcinogenesis', 'are', 'poorly', 'defined', 'cadmium', 'can', 'cause', 'any', 'number', 'of', 'molecular', 'lesions', 'that', 'would', 'be', 'relevant', 'to', 'oncogenesis', 'in', 'various', 'cellular', 'model', 'systems', 'most', 'studies', 'indicate', 'cadmium', 'is', 'poorly', 'mutagenic', 'and', 'probably', 'acts', 'through', 'indirect', 'or', 'epigenetic', 'mechanisms', 'potentially', 'including', 'aberrant', 'activation', 'of', 'oncogenes', 'and', 'suppression', 'of', 'apoptosis', 'elsevier', 'all', 'rights', 'reserved'], ['diverse', 'post', 'translational', 'modifications', 'of', 'histone', 'amino', 'termini', 'represent', 'an', 'important', 'epigenetic', 'mechanism', 'for', 'the', 'organisation', 'of', 'chromatin', 'structure', 'and', 'the', 'regulation', 'of', 'gene', 'activity', 'within', 'the', 'past', 'two', 'years', 'great', 'progress', 'has', 'been', 'made', 'in', 'understanding', 'the', 'functional', 'implications', 'of', 'histone', 'methylation', 'in', 'particular', 'through', 'the', 'of', 'histone', 'that', 'direct', 'the', 'site', 'specific', 'methylation', 'of', 'for', 'example', 'lysine', 'and', 'lysine', 'positions', 'in', 'the', 'histone', 'amino', 'terminus', 'all', 'known', 'histone', 'of', 'this', 'type', 'contain', 'the', 'evolutionarily', 'conserved', 'set', 'domain', 'and', 'appear', 'to', 'be', 'able', 'to', 'stimulate', 'either', 'gene', 'repression', 'or', 'gene', 'activation', 'methylation', 'of', 'lys', 'and', 'lys', 'has', 'been', 'visualised', 'in', 'native', 'chromatin', 'indicating', 'opposite', 'roles', 'in', 'structuring', 'repressive', 'or', 'accessible', 'chromatin', 'domains', 'for', 'example', 'at', 'the', 'mating', 'type', 'loci', 'in', 'pombe', 'at', 'pericentric', 'heterochromatin', 'and', 'at', 'the', 'inactive', 'chromosome', 'in', 'mammals', 'striking', 'differences', 'between', 'these', 'distinct', 'marks', 'have', 'been', 'observed', 'lys', 'methylation', 'is', 'also', 'important', 'to', 'direct', 'additional', 'epigenetic', 'signals', 'such', 'as', 'dna', 'methylation', 'for', 'example', 'in', 'neurospora', 'crassa', 'and', 'in', 'arabidopsis', 'thaliana', 'together', 'the', 'available', 'data', 'strongly', 'establish', 'histone', 'lysine', 'methylation', 'as', 'central', 'modification', 'for', 'the', 'epigenetic', 'organisation', 'of', 'eukaryotic', 'genomes'], ['polyploidy', 'has', 'long', 'been', 'recognized', 'as', 'prominent', 'force', 'shaping', 'the', 'evolution', 'of', 'eukaryotes', 'especially', 'flowering', 'plants', 'new', 'phenotypes', 'often', 'arise', 'with', 'polyploid', 'formation', 'and', 'can', 'contribute', 'to', 'the', 'success', 'of', 'polyploids', 'in', 'nature', 'or', 'their', 'selection', 'for', 'use', 'in', 'agriculture', 'although', 'the', 'causes', 'of', 'novel', 'variation', 'in', 'polyploids', 'are', 'not', 'well', 'understood', 'they', 'could', 'involve', 'changes', 'in', 'gene', 'expression', 'through', 'increased', 'variation', 'in', 'dosage', 'regulated', 'gene', 'expression', 'altered', 'regulatory', 'interactions', 'and', 'rapid', 'genetic', 'and', 'epigenetic', 'changes', 'new', 'research', 'approaches', 'are', 'being', 'used', 'to', 'study', 'these', 'mechanisms', 'and', 'the', 'results', 'should', 'provide', 'more', 'complete', 'understanding', 'of', 'polyploidy'], ['adult', 'germline', 'stem', 'cells', 'are', 'capable', 'of', 'self', 'renewal', 'tissue', 'regeneration', 'and', 'production', 'of', 'large', 'numbers', 'of', 'differentiated', 'progeny', 'we', 'show', 'here', 'that', 'the', 'classical', 'mouse', 'mutant', 'luxoid', 'affects', 'adult', 'germline', 'stem', 'cell', 'self', 'renewal', 'young', 'homozygous', 'luxoid', 'mutant', 'mice', 'produce', 'limited', 'numbers', 'of', 'normal', 'spermatozoa', 'and', 'then', 'progressively', 'lose', 'their', 'germ', 'line', 'after', 'birth', 'transplantation', 'studies', 'showed', 'that', 'germ', 'cells', 'from', 'mutant', 'mice', 'did', 'not', 'colonize', 'recipient', 'testes', 'suggesting', 'that', 'the', 'defect', 'is', 'intrinsic', 'to', 'the', 'stem', 'cells', 'we', 'determined', 'that', 'the', 'luxoid', 'mutant', 'contains', 'nonsense', 'mutation', 'in', 'the', 'gene', 'encoding', 'plzf', 'transcriptional', 'repressor', 'that', 'regulates', 'the', 'epigenetic', 'state', 'of', 'cells', 'and', 'showed', 'that', 'plzf', 'is', 'coexpressed', 'with', 'oct', 'in', 'spermatogonia', 'this', 'is', 'the', 'first', 'gene', 'shown', 'to', 'be', 'required', 'in', 'germ', 'cells', 'for', 'stem', 'cell', 'self', 'renewal', 'in', 'mammals'], ['the', 'covalent', 'modification', 'of', 'nucleosomal', 'histones', 'has', 'emerged', 'as', 'major', 'determinant', 'of', 'chromatin', 'structure', 'and', 'gene', 'activity', 'to', 'understand', 'the', 'interplay', 'between', 'various', 'histone', 'modifications', 'including', 'acetylation', 'and', 'methylation', 'we', 'performed', 'genome', 'wide', 'chromatin', 'structure', 'analysis', 'in', 'higher', 'eukaryote', 'we', 'found', 'binary', 'pattern', 'of', 'histone', 'modifications', 'among', 'euchromatic', 'genes', 'with', 'active', 'genes', 'being', 'hyperacetylated', 'for', 'and', 'and', 'hypermethylated', 'at', 'lys', 'and', 'lys', 'of', 'and', 'inactive', 'genes', 'being', 'hypomethylated', 'and', 'deacetylated', 'at', 'the', 'same', 'residues', 'furthermore', 'the', 'degree', 'of', 'modification', 'correlates', 'with', 'the', 'level', 'of', 'transcription', 'and', 'modifications', 'are', 'largely', 'restricted', 'to', 'transcribed', 'regions', 'suggesting', 'that', 'their', 'regulation', 'is', 'tightly', 'linked', 'to', 'polymerase', 'activity'], ['manipulation', 'or', 'non', 'physiological', 'embryo', 'culture', 'environments', 'can', 'lead', 'to', 'defective', 'fetal', 'programming', 'in', 'livestock', 'our', 'demonstration', 'of', 'reduced', 'fetal', 'methylation', 'and', 'expression', 'of', 'ovine', 'igf', 'suggests', 'pre', 'implantation', 'embryo', 'procedures', 'may', 'be', 'vulnerable', 'to', 'epigenetic', 'alterations', 'in', 'imprinted', 'genes', 'this', 'highlights', 'the', 'potential', 'benefits', 'of', 'epigenetic', 'diagnostic', 'screening', 'in', 'developing', 'embryo', 'procedures'], ['proteins', 'from', 'the', 'polycomb', 'group', 'pcg', 'are', 'epigenetic', 'chromatin', 'modifiers', 'involved', 'in', 'cancer', 'development', 'and', 'also', 'in', 'the', 'maintenance', 'of', 'embryonic', 'and', 'adult', 'stem', 'cells', 'the', 'therapeutic', 'potential', 'of', 'stem', 'cells', 'and', 'the', 'growing', 'conviction', 'that', 'tumors', 'contain', 'stem', 'cells', 'highlights', 'the', 'importance', 'of', 'understanding', 'the', 'extrinsic', 'and', 'intrinsic', 'circuitry', 'controlling', 'stem', 'cell', 'fate', 'and', 'their', 'connections', 'to', 'cancer'], ['genomes', 'are', 'organized', 'into', 'active', 'regions', 'known', 'as', 'euchromatin', 'and', 'inactive', 'regions', 'known', 'as', 'heterochromatin', 'or', 'silenced', 'chromatin', 'this', 'review', 'describes', 'contemporary', 'knowledge', 'and', 'models', 'for', 'how', 'silenced', 'chromatin', 'in', 'saccharomyces', 'cerevisiae', 'forms', 'functions', 'and', 'is', 'inherited', 'in', 'cerevisiae', 'sir', 'proteins', 'are', 'the', 'key', 'structural', 'components', 'of', 'silenced', 'chromatin', 'sir', 'proteins', 'interact', 'first', 'with', 'silencers', 'which', 'dictate', 'which', 'regions', 'are', 'silenced', 'and', 'then', 'with', 'histone', 'tails', 'in', 'nucleosomes', 'as', 'the', 'sir', 'proteins', 'spread', 'from', 'silencers', 'along', 'chromosomes', 'importantly', 'the', 'spreading', 'of', 'silenced', 'chromatin', 'requires', 'the', 'histone', 'deacetylase', 'activity', 'of', 'sir', 'this', 'requirement', 'leads', 'to', 'general', 'model', 'for', 'the', 'spreading', 'and', 'inheritance', 'of', 'silenced', 'chromatin', 'or', 'other', 'special', 'chromatin', 'states', 'such', 'chromatin', 'domains', 'are', 'marked', 'by', 'modifications', 'of', 'the', 'nucleosomes', 'or', 'dna', 'and', 'this', 'mark', 'is', 'able', 'to', 'recruit', 'an', 'enzyme', 'that', 'makes', 'further', 'marks', 'thus', 'among', 'different', 'organisms', 'multiple', 'forms', 'of', 'repressive', 'chromatin', 'can', 'be', 'formed', 'using', 'similar', 'strategies', 'but', 'completely', 'different', 'proteins', 'we', 'also', 'describe', 'emerging', 'evidence', 'that', 'mutations', 'that', 'cause', 'global', 'changes', 'in', 'the', 'modification', 'of', 'histones', 'can', 'alter', 'the', 'balance', 'between', 'euchromatin', 'and', 'silenced', 'chromatin', 'within', 'cell'], ['lymphocytes', 'differentiate', 'in', 'discrete', 'stages', 'within', 'the', 'thymus', 'immature', 'thymocytes', 'lacking', 'cd', 'and', 'cd', 'coreceptors', 'differentiate', 'into', 'double', 'positive', 'cells', 'cd', 'cd', 'which', 'are', 'selected', 'to', 'become', 'either', 'cd', 'cd', 'helper', 'cells', 'or', 'cd', 'cd', 'cytotoxic', 'cells', 'stage', 'specific', 'transcriptional', 'silencer', 'regulates', 'expression', 'of', 'cd', 'in', 'both', 'immature', 'and', 'cd', 'cd', 'thymocytes', 'we', 'show', 'here', 'that', 'binding', 'sites', 'for', 'runt', 'domain', 'transcription', 'factors', 'are', 'essential', 'for', 'cd', 'silencer', 'function', 'at', 'both', 'stages', 'and', 'that', 'different', 'runx', 'family', 'members', 'are', 'required', 'to', 'fulfill', 'unique', 'functions', 'at', 'each', 'stage', 'runxi', 'is', 'required', 'for', 'active', 'repression', 'in', 'cd', 'cd', 'thymocytes', 'whereas', 'runx', 'is', 'required', 'for', 'establishing', 'epigenetic', 'silencing', 'in', 'cytotoxic', 'lineage', 'thymocytes', 'runx', 'deficient', 'cytotoxic', 'cells', 'but', 'not', 'helper', 'cells', 'have', 'defective', 'responses', 'to', 'antigen', 'suggesting', 'that', 'runx', 'proteins', 'have', 'critical', 'functions', 'in', 'lineage', 'specification', 'and', 'homeostasis', 'of', 'cd', 'lineage', 'lymphocytes'], ['the', 'cyclin', 'cdk', 'lnk', 'rb', 'pathway', 'plays', 'key', 'role', 'in', 'controlling', 'cell', 'growth', 'by', 'integrating', 'multiple', 'mitogenic', 'and', 'antimitogenic', 'stimuli', 'the', 'components', 'of', 'this', 'pathway', 'are', 'gene', 'families', 'with', 'high', 'level', 'of', 'structural', 'and', 'functional', 'redundancy', 'and', 'are', 'expressed', 'in', 'an', 'overlapping', 'fashion', 'in', 'most', 'tissues', 'and', 'cell', 'types', 'using', 'classical', 'transgenic', 'technology', 'as', 'well', 'as', 'gene', 'targeting', 'in', 'es', 'cells', 'series', 'of', 'mouse', 'models', 'have', 'been', 'developed', 'to', 'study', 'the', 'in', 'vivo', 'function', 'of', 'individual', 'components', 'of', 'this', 'pathway', 'in', 'both', 'normal', 'homeostasis', 'and', 'tumor', 'development', 'these', 'models', 'have', 'proven', 'to', 'be', 'useful', 'to', 'define', 'specific', 'as', 'well', 'as', 'redundant', 'roles', 'among', 'members', 'of', 'these', 'cell', 'cycle', 'regulatory', 'gene', 'families', 'this', 'pathway', 'is', 'deregulated', 'in', 'the', 'vast', 'majority', 'of', 'human', 'tumors', 'by', 'genetic', 'and', 'epigenetic', 'alterations', 'that', 'target', 'at', 'least', 'some', 'of', 'its', 'key', 'members', 'such', 'as', 'cyclin', 'cdk', 'ink', 'and', 'ink', 'prb', 'etc', 'as', 'consequence', 'some', 'of', 'these', 'molecules', 'are', 'currently', 'being', 'considered', 'as', 'targets', 'for', 'cancer', 'therapy', 'and', 'several', 'novel', 'molecules', 'such', 'as', 'cdk', 'inhibitors', 'are', 'under', 'development', 'as', 'potential', 'anti', 'cancer', 'drugs', 'elsevier', 'science', 'all', 'rights', 'reserved'], ['twin', 'studies', 'have', 'been', 'vital', 'for', 'establishing', 'an', 'important', 'genetic', 'contribution', 'to', 'the', 'etiology', 'of', 'schizophrenia', 'the', 'five', 'newest', 'studies', 'since', 'from', 'europe', 'and', 'japan', 'have', 'confirmed', 'earlier', 'findings', 'they', 'yielded', 'probandwise', 'concordance', 'rates', 'of', 'in', 'monozygotic', 'mz', 'pairs', 'and', 'in', 'dizygotic', 'dz', 'pairs', 'and', 'heritability', 'estimates', 'of', 'approximately', 'twin', 'studies', 'are', 'also', 'valuable', 'for', 'investigating', 'the', 'etiological', 'relationships', 'between', 'schizophrenia', 'and', 'other', 'disorders', 'and', 'the', 'genetic', 'basis', 'of', 'clinical', 'heterogeneity', 'within', 'schizophrenia', 'studies', 'of', 'discordant', 'mz', 'pairs', 'provide', 'further', 'insights', 'into', 'noninherited', 'factors', 'that', 'contribute', 'to', 'the', 'multifactorial', 'etiology', 'of', 'this', 'disorder', 'more', 'recently', 'twin', 'studies', 'have', 'begun', 'to', 'be', 'used', 'to', 'directly', 'investigate', 'molecular', 'genetic', 'and', 'epigenetic', 'processes', 'underlying', 'schizophrenia', 'wiley', 'liss', 'inc'], ['interspecific', 'or', 'intergeneric', 'hybridization', 'followed', 'by', 'chromosome', 'doubling', 'can', 'lead', 'to', 'the', 'formation', 'of', 'new', 'allopolyploid', 'species', 'recent', 'studies', 'indicate', 'that', 'allopolyploid', 'formation', 'is', 'associated', 'with', 'genetic', 'and', 'epigenetic', 'changes', 'although', 'little', 'is', 'known', 'about', 'the', 'type', 'of', 'changes', 'that', 'occur', 'how', 'rapidly', 'they', 'occur', 'and', 'the', 'type', 'of', 'sequences', 'involved', 'to', 'address', 'these', 'matters', 'we', 'have', 'surveyed', 'hybrids', 'between', 'diploid', 'species', 'from', 'the', 'wheat', 'aegilops', 'and', 'triticum', 'group', 'and', 'their', 'derived', 'allotetraploids', 'by', 'screening', 'large', 'number', 'of', 'loci', 'using', 'amplified', 'fragment', 'length', 'polymorphism', 'and', 'dna', 'gel', 'blot', 'analysis', 'and', 'by', 'assaying', 'the', 'extent', 'of', 'cytosine', 'methylation', 'we', 'found', 'that', 'sequence', 'elimination', 'is', 'one', 'of', 'the', 'major', 'and', 'immediate', 'responses', 'of', 'the', 'wheat', 'genome', 'to', 'wide', 'hybridization', 'or', 'allopolyploidy', 'that', 'it', 'affects', 'large', 'fraction', 'of', 'the', 'genome', 'and', 'that', 'it', 'is', 'reproducible', 'in', 'one', 'cross', 'between', 'ae', 'sharonensis', 'ae', 'umbellulata', 'of', 'the', 'loci', 'from', 'ae', 'sharonensis', 'were', 'eliminated', 'compared', 'with', 'only', 'from', 'ae', 'umbellulata', 'with', 'most', 'changes', 'occurring', 'in', 'the', 'hybrid', 'in', 'contrast', 'crosses', 'between', 'ae', 'longissima', 'urartu', 'showed', 'that', 'sequence', 'elimination', 'was', 'more', 'frequent', 'after', 'chromosome', 'doubling', 'alterations', 'in', 'cytosine', 'methylation', 'occurred', 'in', 'similar', 'to', 'of', 'the', 'loci', 'either', 'in', 'the', 'hybrid', 'or', 'in', 'the', 'allopolyploid', 'for', 'eight', 'of', 'nine', 'bands', 'that', 'were', 'isolated', 'the', 'sequences', 'that', 'underwent', 'elimination', 'corresponded', 'to', 'low', 'copy', 'dna', 'whereas', 'alterations', 'in', 'methylation', 'patterns', 'affected', 'both', 'repetitive', 'dna', 'sequences', 'such', 'as', 'and', 'low', 'copy', 'dna', 'in', 'approximately', 'equal', 'proportions'], ['cytosine', 'methylation', 'in', 'cpg', 'dinucleotides', 'is', 'believed', 'to', 'be', 'important', 'in', 'gene', 'regulation', 'and', 'is', 'generally', 'associated', 'with', 'reduced', 'levels', 'of', 'transcription', 'methylation', 'mediated', 'gene', 'silencing', 'involves', 'series', 'of', 'dna', 'protein', 'and', 'protein', 'protein', 'interactions', 'that', 'begins', 'with', 'the', 'binding', 'of', 'methyl', 'cpg', 'binding', 'proteins', 'mbps', 'followed', 'by', 'the', 'recruitment', 'of', 'histone', 'modifying', 'enzymes', 'that', 'together', 'promote', 'chromatin', 'condensation', 'and', 'inactivation', 'it', 'is', 'widely', 'known', 'that', 'alterations', 'in', 'methylation', 'patterns', 'and', 'associated', 'gene', 'activities', 'are', 'often', 'found', 'in', 'human', 'tumors', 'however', 'the', 'mechanisms', 'by', 'which', 'methylation', 'patterns', 'are', 'altered', 'are', 'not', 'currently', 'understood', 'in', 'this', 'paper', 'we', 'investigate', 'the', 'impact', 'of', 'oxidative', 'damage', 'to', 'methyl', 'cpg', 'site', 'on', 'mbp', 'binding', 'by', 'the', 'selective', 'placement', 'of', 'oxoguanine', 'oxog', 'and', 'hmc', 'in', 'mbp', 'recognition', 'sequence', 'duplexes', 'containing', 'these', 'specific', 'modifications', 'were', 'assayed', 'for', 'binding', 'to', 'the', 'methyl', 'cpg', 'binding', 'domain', 'mbd', 'of', 'one', 'member', 'of', 'the', 'mbp', 'family', 'methyl', 'cpg', 'binding', 'protein', 'mecp', 'our', 'results', 'reveal', 'that', 'oxidation', 'of', 'either', 'single', 'guanine', 'to', 'oxog', 'or', 'of', 'single', 'mc', 'to', 'hmc', 'significantly', 'inhibits', 'binding', 'of', 'the', 'mbd', 'to', 'the', 'oligonucleotide', 'duplex', 'reducing', 'the', 'binding', 'affinity', 'by', 'at', 'least', 'an', 'order', 'of', 'magnitude', 'oxidative', 'damage', 'to', 'dna', 'could', 'therefore', 'result', 'in', 'heritable', 'epigenetic', 'changes', 'in', 'chromatin', 'organization'], ['rett', 'syndrome', 'rtt', 'mim', 'is', 'progressive', 'disorder', 'and', 'one', 'of', 'the', 'most', 'common', 'causes', 'of', 'mental', 'retardation', 'in', 'females', 'with', 'an', 'incidence', 'of', 'in', 'ref', 'patients', 'with', 'classic', 'rtt', 'appear', 'to', 'develop', 'normally', 'until', 'months', 'of', 'age', 'then', 'gradually', 'lose', 'speech', 'and', 'purposeful', 'hand', 'use', 'and', 'develop', 'microcephaly', 'seizures', 'autism', 'ataxia', 'intermittent', 'and', 'stereotypic', 'hand', 'movements', 'after', 'initial', 'regression', 'the', 'condition', 'stabilizes', 'and', 'patients', 'usually', 'survive', 'into', 'adulthood', 'as', 'rtt', 'occurs', 'almost', 'exclusively', 'in', 'females', 'it', 'has', 'been', 'proposed', 'that', 'rtt', 'is', 'caused', 'by', 'an', 'linked', 'dominant', 'mutation', 'with', 'lethality', 'in', 'hemizygous', 'males', 'previous', 'exclusion', 'mapping', 'studies', 'using', 'rtt', 'families', 'mapped', 'the', 'locus', 'to', 'xq', 'refs', 'using', 'systematic', 'gene', 'screening', 'approach', 'we', 'have', 'identified', 'mutations', 'in', 'the', 'gene', 'mecp', 'encoding', 'linked', 'methyl', 'cpg', 'binding', 'protein', 'mecp', 'as', 'the', 'cause', 'of', 'some', 'eases', 'of', 'rtt', 'mecp', 'selectively', 'binds', 'cpg', 'dinucleotides', 'in', 'the', 'mammalian', 'genome', 'and', 'mediates', 'transcriptional', 'repression', 'through', 'interaction', 'with', 'histone', 'deacetylase', 'and', 'the', 'corepressor', 'sin', 'refs', 'in', 'of', 'sporadic', 'patients', 'we', 'found', 'de', 'novo', 'missense', 'mutations', 'in', 'the', 'region', 'encoding', 'the', 'highly', 'conserved', 'methyl', 'binding', 'domain', 'mbd', 'as', 'well', 'as', 'de', 'novo', 'frameshift', 'and', 'de', 'novo', 'nonsense', 'mutation', 'both', 'of', 'which', 'disrupt', 'the', 'transcription', 'repression', 'domain', 'trd', 'in', 'two', 'affected', 'half', 'sisters', 'of', 'rtt', 'family', 'we', 'found', 'segregation', 'of', 'an', 'additional', 'missense', 'mutation', 'not', 'detected', 'in', 'their', 'obligate', 'carrier', 'mother', 'this', 'suggests', 'that', 'the', 'mother', 'is', 'germline', 'mosaic', 'for', 'this', 'mutation', 'our', 'study', 'reports', 'the', 'first', 'disease', 'causing', 'mutations', 'in', 'rtt', 'and', 'points', 'to', 'abnormal', 'epigenetic', 'regulation', 'as', 'the', 'mechanism', 'underlying', 'the', 'pathogenesis', 'of', 'rtt'], ['the', 'discovery', 'of', 'numerous', 'hypermethylated', 'promoters', 'of', 'tumour', 'suppressor', 'genes', 'along', 'with', 'better', 'understanding', 'of', 'gene', 'silencing', 'mechanisms', 'has', 'moved', 'dna', 'methylation', 'from', 'obscurity', 'to', 'recognition', 'as', 'an', 'alternative', 'mechanism', 'of', 'tumour', 'suppressor', 'inactivation', 'in', 'cancer', 'epigenetic', 'events', 'can', 'also', 'facilitate', 'genetic', 'damage', 'as', 'illustrated', 'by', 'the', 'increased', 'mutagenicity', 'of', 'methylcytosine', 'and', 'the', 'silencing', 'of', 'the', 'mlh', 'mismatch', 'repair', 'gene', 'by', 'dna', 'methylation', 'in', 'colorectal', 'tumours', 'we', 'review', 'here', 'current', 'mechanistic', 'understanding', 'of', 'the', 'role', 'of', 'dna', 'methylation', 'in', 'malignant', 'transformation', 'and', 'suggest', 'knudson', 'two', 'hit', 'hypothesis', 'should', 'now', 'be', 'expanded', 'to', 'include', 'epigenetic', 'mechanisms', 'of', 'gene', 'inactivation'], ['densely', 'methylated', 'dna', 'associates', 'with', 'repressive', 'chromatin', 'characterized', 'by', 'the', 'presence', 'of', 'underacetylated', 'histones', 'recently', 'these', 'two', 'epigenetic', 'processes', 'have', 'been', 'dynamically', 'linked', 'the', 'methyl', 'cpg', 'binding', 'protein', 'mecp', 'appears', 'to', 'reside', 'in', 'complex', 'with', 'histone', 'deacetylase', 'activity', 'mecp', 'can', 'mediate', 'formation', 'of', 'repressive', 'chromatin', 'on', 'methylated', 'promoter', 'templates', 'in', 'vitro', 'and', 'this', 'process', 'can', 'be', 'reversed', 'by', 'trichostatin', 'tsa', 'specific', 'inhibitor', 'of', 'histone', 'deacetylase', 'little', 'is', 'known', 'however', 'about', 'the', 'relative', 'roles', 'of', 'methylation', 'and', 'histone', 'deacetylase', 'activity', 'in', 'the', 'stable', 'inhibition', 'of', 'transcription', 'on', 'densely', 'methylated', 'endogenous', 'promoters', 'such', 'as', 'those', 'for', 'silenced', 'alleles', 'of', 'imprinted', 'genes', 'genes', 'on', 'the', 'female', 'inactive', 'chromosome', 'and', 'tumour', 'suppressor', 'genes', 'inactivated', 'in', 'cancer', 'cells', 'we', 'show', 'here', 'that', 'the', 'hypermethylated', 'genes', 'mlh', 'timp', 'timp', 'cdkn', 'ink', 'and', 'cdkn', 'ink', 'cannot', 'be', 'reactivated', 'with', 'tsa', 'alone', 'in', 'tumour', 'cells', 'in', 'which', 'we', 'have', 'shown', 'that', 'tsa', 'alone', 'can', 'upregulate', 'the', 'expression', 'of', 'non', 'methylated', 'genes', 'following', 'minimal', 'demethylation', 'and', 'slight', 'gene', 'reactivation', 'in', 'the', 'presence', 'of', 'low', 'dose', 'aza', 'deoxycytidine', 'aza', 'dc', 'however', 'tsa', 'treatment', 'results', 'in', 'robust', 're', 'expression', 'of', 'each', 'gene', 'tsa', 'does', 'not', 'contribute', 'to', 'demethylation', 'of', 'the', 'genes', 'and', 'none', 'of', 'the', 'treatments', 'alter', 'the', 'chromatin', 'structure', 'associated', 'with', 'the', 'hypermethylated', 'promoters', 'thus', 'although', 'dna', 'methylation', 'and', 'histone', 'deacetylation', 'appear', 'to', 'act', 'as', 'synergistic', 'layers', 'for', 'the', 'silencing', 'of', 'genes', 'in', 'cancer', 'dense', 'cpg', 'island', 'methylation', 'is', 'dominant', 'for', 'the', 'stable', 'maintenance', 'of', 'silent', 'state', 'at', 'these', 'loci'], [], ['viable', 'yellow', 'vy', 'alpha', 'mice', 'are', 'larger', 'obese', 'more', 'susceptible', 'to', 'cancer', 'and', 'on', 'average', 'shorter', 'lived', 'than', 'their', 'non', 'yellow', 'siblings', 'they', 'are', 'epigenetic', 'mosaics', 'ranging', 'from', 'yellow', 'phenotype', 'with', 'maximum', 'ectopic', 'agouti', 'overexpression', 'through', 'continuum', 'of', 'mottled', 'agouti', 'yellow', 'phenotypes', 'with', 'partial', 'agouti', 'overexpression', 'to', 'pseudoagouti', 'phenotype', 'with', 'minimal', 'ectopic', 'expression', 'pseudoagouti', 'vy', 'mice', 'are', 'lean', 'healthy', 'and', 'longer', 'lived', 'than', 'their', 'yellow', 'siblings', 'here', 'we', 'report', 'that', 'feeding', 'pregnant', 'black', 'dams', 'methyl', 'supplemented', 'diets', 'alters', 'epigenetic', 'regulation', 'of', 'agouti', 'expression', 'in', 'their', 'offspring', 'as', 'indicated', 'by', 'increased', 'agouti', 'black', 'mottling', 'in', 'the', 'direction', 'of', 'the', 'pseudoagouti', 'phenotype', 'we', 'also', 'present', 'confirmatory', 'evidence', 'that', 'epigenetic', 'phenotypes', 'are', 'maternally', 'heritable', 'thus', 'vy', 'expression', 'already', 'known', 'to', 'be', 'modulated', 'by', 'imprinting', 'strain', 'specific', 'modification', 'and', 'maternal', 'epigenetic', 'inheritance', 'is', 'also', 'modulated', 'by', 'maternal', 'diet', 'these', 'observations', 'suggest', 'at', 'least', 'in', 'this', 'special', 'case', 'that', 'maternal', 'dietary', 'supplementation', 'may', 'positively', 'affect', 'health', 'and', 'longevity', 'of', 'the', 'offspring', 'therefore', 'this', 'experimental', 'system', 'should', 'be', 'useful', 'for', 'identifying', 'maternal', 'factors', 'that', 'modulate', 'epigenetic', 'mechanisms', 'especially', 'dna', 'methylation', 'in', 'developing', 'embryos'], ['the', 'yeast', 'sir', 'protein', 'regulates', 'epigenetic', 'gene', 'silencing', 'and', 'as', 'possible', 'antiaging', 'effect', 'it', 'suppresses', 'recombination', 'of', 'rdna', 'studies', 'involving', 'cobb', 'bacterial', 'sir', 'like', 'gene', 'have', 'suggested', 'it', 'could', 'encode', 'pyridine', 'nucleotide', 'transferase', 'here', 'five', 'human', 'sirtuin', 'cdnas', 'are', 'characterized', 'the', 'sirt', 'sequence', 'has', 'the', 'closest', 'homology', 'to', 'the', 'cerevisiae', 'sir', 'the', 'sirt', 'and', 'sirt', 'sirtuins', 'more', 'closely', 'resemble', 'prokaryotic', 'sirtuin', 'sequences', 'the', 'five', 'human', 'sirtuins', 'are', 'widely', 'expressed', 'in', 'fetal', 'and', 'adult', 'tissues', 'recombinant', 'coli', 'cobt', 'and', 'cobb', 'proteins', 'each', 'showed', 'weak', 'nad', 'dependent', 'mono', 'adp', 'activity', 'using', 'as', 'substrate', 'recombinant', 'coli', 'cobb', 'and', 'human', 'sirt', 'sirtuin', 'proteins', 'were', 'able', 'to', 'cause', 'radioactivity', 'to', 'be', 'transfered', 'from', 'nad', 'to', 'bovine', 'serum', 'albumin', 'bsa', 'when', 'conserved', 'histidine', 'within', 'the', 'human', 'sirt', 'sirtuin', 'was', 'converted', 'to', 'tyrosine', 'the', 'mutant', 'recombinant', 'protein', 'was', 'unable', 'to', 'transfer', 'radioactivity', 'from', 'nad', 'to', 'bsa', 'these', 'results', 'suggest', 'that', 'the', 'sirtuins', 'may', 'function', 'via', 'mono', 'adp', 'ribosylation', 'of', 'proteins', 'academic', 'press'], ['gene', 'activation', 'in', 'eukaryotes', 'requires', 'chromatin', 'remodeling', 'complexes', 'like', 'swi', 'snf', 'and', 'histone', 'acetylases', 'like', 'saga', 'how', 'these', 'factors', 'are', 'recruited', 'to', 'promoters', 'is', 'not', 'yet', 'understood', 'using', 'chip', 'we', 'measured', 'recruitment', 'of', 'swi', 'snf', 'saga', 'the', 'repressor', 'ash', 'and', 'transcription', 'factors', 'swi', 'and', 'sbf', 'to', 'the', 'ho', 'endonuclease', 'promoter', 'as', 'cells', 'progress', 'through', 'the', 'yeast', 'cell', 'cycle', 'swi', 'entry', 'into', 'nuclei', 'at', 'the', 'end', 'of', 'anaphase', 'recruits', 'swi', 'snf', 'which', 'then', 'recruits', 'saga', 'these', 'two', 'factors', 'then', 'facilitate', 'sbf', 'binding', 'ash', 'which', 'only', 'accumulates', 'in', 'daughter', 'cell', 'nuclei', 'binds', 'to', 'ho', 'soon', 'after', 'swi', 'and', 'aborts', 'recruitment', 'of', 'swi', 'snf', 'saga', 'and', 'sbf', 'swi', 'remains', 'at', 'ho', 'for', 'only', 'min', 'swi', 'snf', 'and', 'saga', 'subsequent', 'persistence', 'at', 'ho', 'is', 'self', 'sustaining', 'and', 'constitutes', 'an', 'epigenetic', 'memory', 'of', 'ho', 'transient', 'interaction', 'with', 'swi'], ['altering', 'cytosine', 'methylation', 'by', 'genetic', 'means', 'leads', 'to', 'variety', 'of', 'developmental', 'defects', 'in', 'mice', 'plants', 'and', 'fungi', 'deregulation', 'of', 'cytosine', 'methylation', 'also', 'has', 'role', 'in', 'human', 'carcinogenesis', 'in', 'some', 'cases', 'these', 'defects', 'have', 'been', 'tied', 'to', 'the', 'inheritance', 'of', 'epigenetic', 'alterations', 'such', 'as', 'chromatin', 'imprints', 'and', 'dna', 'methylation', 'patterns', 'that', 'do', 'not', 'involve', 'changes', 'in', 'dna', 'sequence', 'using', 'forward', 'genetic', 'screen', 'we', 'identified', 'gene', 'ddm', 'decrease', 'in', 'dna', 'methylation', 'from', 'the', 'flowering', 'plant', 'arabidopsis', 'thaliana', 'required', 'to', 'maintain', 'normal', 'cytosine', 'methylation', 'patterns', 'additional', 'ddm', 'alleles', 'som', 'were', 'isolated', 'in', 'selection', 'for', 'mutations', 'that', 'relieved', 'transgene', 'silencing', 'unpublished', 'data', 'loss', 'of', 'ddm', 'function', 'causes', 'reduction', 'of', 'genomic', 'cytosine', 'methylation', 'with', 'most', 'of', 'the', 'immediate', 'hypomethylation', 'occurring', 'in', 'repeated', 'sequences', 'in', 'contrast', 'many', 'low', 'copy', 'sequences', 'initially', 'retain', 'their', 'methylation', 'in', 'ddm', 'homozygotes', 'but', 'lose', 'methylation', 'over', 'time', 'as', 'the', 'mutants', 'are', 'propagated', 'through', 'multiple', 'generations', 'by', 'self', 'pollination', 'the', 'progressive', 'effect', 'of', 'ddm', 'mutations', 'on', 'low', 'copy', 'sequence', 'methylation', 'suggests', 'that', 'ddm', 'mutations', 'compromise', 'the', 'efficiency', 'of', 'methylation', 'of', 'newly', 'incorporated', 'cytosines', 'after', 'dna', 'replication', 'in', 'parallel', 'with', 'the', 'slow', 'decay', 'of', 'methylation', 'during', 'inbreeding', 'ddm', 'mutants', 'accumulate', 'heritable', 'alterations', 'mutations', 'or', 'stable', 'epialleles', 'at', 'dispersed', 'sites', 'in', 'the', 'genome', 'that', 'lead', 'to', 'morphological', 'abnormalities', 'here', 'we', 'report', 'that', 'ddm', 'encodes', 'swi', 'snf', 'like', 'protein', 'implicating', 'chromatin', 'remodelling', 'as', 'an', 'important', 'process', 'for', 'maintenance', 'of', 'dna', 'methylation', 'and', 'genome', 'integrity'], ['chromatin', 'assembly', 'factor', 'caf', 'is', 'required', 'for', 'inheritance', 'of', 'epigenetically', 'determined', 'chromosomal', 'states', 'in', 'vivo', 'and', 'promotes', 'assembly', 'of', 'chromatin', 'during', 'dna', 'replication', 'in', 'vitro', 'herein', 'we', 'demonstrate', 'that', 'after', 'dna', 'replication', 'replicated', 'but', 'not', 'unreplicated', 'dna', 'is', 'also', 'competent', 'for', 'caf', 'dependent', 'chromatin', 'assembly', 'the', 'proliferating', 'cell', 'nuclear', 'antigen', 'pcna', 'dna', 'polymerase', 'clamp', 'is', 'component', 'of', 'the', 'replication', 'dependent', 'marking', 'of', 'dna', 'for', 'chromatin', 'assembly', 'the', 'clamp', 'loader', 'replication', 'factor', 'rfc', 'can', 'reverse', 'this', 'mark', 'by', 'unloading', 'pcna', 'from', 'the', 'replicated', 'dna', 'pcna', 'binds', 'directly', 'to', 'the', 'largest', 'subunit', 'of', 'caf', 'and', 'the', 'two', 'proteins', 'colocalize', 'at', 'sites', 'of', 'dna', 'replication', 'in', 'cells', 'we', 'suggest', 'that', 'pcna', 'and', 'caf', 'connect', 'dna', 'replication', 'to', 'chromatin', 'assembly', 'and', 'the', 'inheritance', 'of', 'epigenetic', 'chromosome', 'states'], ['handful', 'of', 'autosomal', 'genes', 'in', 'the', 'mammalian', 'genome', 'are', 'inherited', 'in', 'silent', 'state', 'from', 'one', 'of', 'the', 'two', 'parents', 'and', 'in', 'fully', 'active', 'form', 'from', 'the', 'other', 'thereby', 'rendering', 'the', 'organism', 'functionally', 'hemizygous', 'for', 'imprinted', 'genes', 'to', 'date', 'imprinted', 'genes', 'have', 'been', 'identified', 'are', 'expressed', 'from', 'the', 'maternal', 'chromosome', 'while', 'the', 'rest', 'are', 'expressed', 'from', 'the', 'paternal', 'chromosome', 'allele', 'specific', 'methylation', 'of', 'cpg', 'residues', 'established', 'in', 'one', 'of', 'the', 'germlines', 'and', 'maintained', 'throughout', 'embryogenesis', 'has', 'been', 'clearly', 'implicated', 'in', 'the', 'maintenance', 'of', 'imprinting', 'in', 'somatic', 'cells', 'although', 'the', 'function', 'of', 'imprinting', 'remains', 'subject', 'of', 'some', 'debate', 'the', 'process', 'is', 'thought', 'to', 'have', 'an', 'important', 'role', 'in', 'regulating', 'the', 'rate', 'of', 'fetal', 'growth'], ['members', 'of', 'the', 'plasmodium', 'falciparum', 'var', 'gene', 'family', 'encode', 'clonally', 'variant', 'adhesins', 'which', 'play', 'an', 'important', 'role', 'in', 'the', 'pathogenicity', 'of', 'tropical', 'malaria', 'here', 'we', 'employ', 'selective', 'panning', 'protocol', 'to', 'generate', 'isogenic', 'falciparum', 'populations', 'with', 'defined', 'adhesive', 'phenotypes', 'for', 'cd', 'icam', 'and', 'csa', 'expressing', 'single', 'and', 'distinct', 'var', 'gene', 'variants', 'this', 'technique', 'has', 'established', 'the', 'framework', 'for', 'examining', 'var', 'gene', 'expression', 'its', 'regulation', 'and', 'switching', 'it', 'was', 'found', 'that', 'var', 'gene', 'switching', 'occurs', 'irt', 'situ', 'ubiquitous', 'transcription', 'of', 'all', 'var', 'gene', 'variants', 'appears', 'to', 'occur', 'in', 'early', 'ring', 'stages', 'however', 'var', 'gene', 'expression', 'is', 'tightly', 'regulated', 'in', 'trophozoites', 'and', 'is', 'exerted', 'through', 'silencing', 'mechanism', 'transcriptional', 'control', 'is', 'mutually', 'exclusive', 'in', 'parasites', 'that', 'express', 'defined', 'adhesive', 'phenotypes', 'lit', 'situ', 'var', 'gene', 'switching', 'is', 'apparently', 'mediated', 'at', 'the', 'level', 'of', 'transcriptional', 'initiation', 'as', 'demonstrated', 'by', 'nuclear', 'run', 'on', 'analyses', 'our', 'results', 'suggest', 'that', 'an', 'epigenetic', 'mechanism', 'is', 'involved', 'in', 'var', 'gene', 'regulation'], ['nuclear', 'receptors', 'nrs', 'are', 'ligand', 'inducible', 'transcription', 'factors', 'that', 'mediate', 'complex', 'effects', 'on', 'development', 'differentiation', 'and', 'homeostasis', 'they', 'regulate', 'the', 'transcription', 'of', 'their', 'target', 'genes', 'through', 'binding', 'to', 'cognate', 'dna', 'sequences', 'as', 'homodimers', 'or', 'heterodimers', 'the', 'molecular', 'mechanisms', 'underlying', 'transcriptional', 'activation', 'by', 'nrs', 'are', 'still', 'poorly', 'understood', 'although', 'intermediary', 'factors', 'mediators', 'appear', 'to', 'be', 'involved', 'in', 'mediating', 'the', 'transactivation', 'functions', 'of', 'nrs', 'tif', 'has', 'been', 'identified', 'previously', 'as', 'protein', 'that', 'interacts', 'specifically', 'with', 'the', 'ligand', 'binding', 'domain', 'of', 'several', 'nuclear', 'receptors', 'both', 'in', 'yeast', 'and', 'in', 'vitro', 'the', 'characteristics', 'of', 'these', 'interactions', 'have', 'led', 'us', 'to', 'suggest', 'that', 'tif', 'might', 'be', 'mediator', 'of', 'the', 'nr', 'ligand', 'inducible', 'activation', 'function', 'af', 'using', 'two', 'hybrid', 'screening', 'in', 'yeast', 'we', 'have', 'now', 'identified', 'two', 'tlfp', 'binding', 'proteins', 'mhp', 'alpha', 'and', 'mmod', 'that', 'are', 'mouse', 'homologues', 'of', 'the', 'drosophila', 'protein', 'using', 'mhp', 'alpha', 'as', 'bait', 'in', 'second', 'two', 'hybrid', 'screening', 'we', 'have', 'isolated', 'cdnas', 'encoding', 'proteins', 'that', 'are', 'also', 'very', 'likely', 'to', 'be', 'involved', 'in', 'chromatin', 'structure', 'and', 'function', 'as', 'well', 'as', 'protein', 'structurally', 'and', 'functionally', 'related', 'to', 'tif', 'renamed', 'tlf', 'alpha', 'which', 'was', 'named', 'tif', 'beta', 'here', 'we', 'discuss', 'how', 'the', 'function', 'of', 'members', 'of', 'the', 'tif', 'family', 'in', 'the', 'control', 'of', 'transcription', 'could', 'be', 'exerted', 'at', 'the', 'level', 'of', 'the', 'structure', 'of', 'the', 'chromatin', 'template'], ['the', 'past', 'few', 'years', 'have', 'seen', 'wider', 'acceptance', 'of', 'role', 'for', 'dna', 'methylation', 'in', 'cancer', 'this', 'can', 'be', 'attributed', 'to', 'three', 'developments', 'first', 'the', 'documentation', 'of', 'the', 'over', 'representation', 'of', 'mutations', 'at', 'cpg', 'dinucleotides', 'has', 'convincingly', 'implicated', 'dna', 'methylation', 'in', 'the', 'generation', 'of', 'oncogenic', 'point', 'mutations', 'the', 'second', 'important', 'advance', 'has', 'been', 'the', 'demonstration', 'of', 'epigenetic', 'silencing', 'of', 'tumor', 'suppressor', 'genes', 'by', 'dna', 'methylation', 'the', 'third', 'development', 'has', 'been', 'the', 'utilization', 'of', 'experimental', 'methods', 'to', 'manipulate', 'dna', 'methylation', 'levels', 'these', 'studies', 'demonstrate', 'that', 'dna', 'methylation', 'changes', 'in', 'cancer', 'cells', 'are', 'not', 'mere', 'by', 'products', 'of', 'malignant', 'transformation', 'but', 'can', 'play', 'an', 'instrumental', 'role', 'in', 'the', 'cancer', 'process', 'it', 'seems', 'clear', 'that', 'dna', 'methylation', 'plays', 'variety', 'of', 'roles', 'in', 'different', 'cancer', 'types', 'and', 'probably', 'at', 'different', 'stages', 'of', 'oncogenesis', 'dna', 'methylation', 'is', 'intricately', 'involved', 'in', 'wide', 'diversity', 'of', 'cellular', 'processes', 'likewise', 'it', 'appears', 'to', 'exert', 'its', 'influence', 'on', 'the', 'cancer', 'process', 'through', 'diverse', 'array', 'of', 'mechanisms', 'it', 'is', 'our', 'task', 'not', 'only', 'to', 'identify', 'these', 'mechanisms', 'but', 'to', 'determine', 'their', 'relative', 'importance', 'for', 'each', 'stage', 'and', 'type', 'of', 'cancer', 'our', 'hope', 'then', 'will', 'be', 'to', 'translate', 'that', 'knowledge', 'into', 'clinical', 'applications'], ['the', 'inactive', 'chromosome', 'differs', 'from', 'the', 'active', 'in', 'number', 'of', 'ways', 'some', 'of', 'these', 'such', 'as', 'allocyclic', 'replication', 'and', 'altered', 'histone', 'acetylation', 'are', 'associated', 'with', 'all', 'types', 'of', 'epigenetic', 'silencing', 'whereas', 'others', 'such', 'as', 'dna', 'methylation', 'are', 'of', 'more', 'restricted', 'use', 'these', 'features', 'are', 'acquired', 'progressively', 'by', 'the', 'inactive', 'after', 'onset', 'of', 'initiation', 'initiation', 'of', 'inactivation', 'is', 'controlled', 'by', 'the', 'inactivation', 'center', 'xic', 'and', 'influenced', 'by', 'the', 'chromosome', 'controlling', 'element', 'xce', 'which', 'causes', 'primary', 'nonrandom', 'inactivation', 'other', 'examples', 'of', 'nonrandom', 'inactivation', 'are', 'also', 'presented', 'in', 'this', 'review', 'the', 'definition', 'of', 'major', 'role', 'for', 'xist', 'noncoding', 'rna', 'in', 'inactivation', 'has', 'enabled', 'investigation', 'of', 'the', 'mechanism', 'leading', 'to', 'establishment', 'of', 'the', 'chromosome', 'and', 'also', 'of', 'the', 'interactions', 'between', 'inactivation', 'and', 'imprinting', 'as', 'well', 'as', 'between', 'inactivation', 'and', 'developmental', 'processes', 'in', 'the', 'early', 'embryo'], ['the', 'centromere', 'is', 'required', 'to', 'ensure', 'the', 'equal', 'distribution', 'of', 'replicated', 'chromosomes', 'to', 'daughter', 'nuclei', 'centromeres', 'are', 'frequently', 'associated', 'with', 'heterochromatin', 'an', 'enigmatic', 'nuclear', 'component', 'that', 'causes', 'the', 'epigenetic', 'transcriptional', 'repression', 'of', 'nearby', 'marker', 'genes', 'position', 'effect', 'variegation', 'of', 'silencing', 'the', 'process', 'of', 'chromosome', 'segregation', 'by', 'movement', 'along', 'microtubules', 'to', 'spindle', 'poles', 'is', 'highly', 'conserved', 'yet', 'the', 'putative', 'cis', 'acting', 'centromeric', 'dna', 'sequences', 'bear', 'little', 'or', 'no', 'similarity', 'across', 'species', 'recently', 'studies', 'in', 'several', 'systems', 'have', 'revealed', 'that', 'the', 'centromere', 'itself', 'might', 'be', 'epigenetically', 'regulated', 'and', 'that', 'the', 'higher', 'order', 'structure', 'of', 'the', 'underlying', 'heterochromatin', 'contributes', 'to', 'centromere', 'function', 'and', 'kinetochore', 'assembly'], ['the', 'chrome', 'and', 'set', 'domains', 'are', 'conserved', 'sequence', 'moths', 'present', 'in', 'chromosomal', 'proteins', 'that', 'function', 'in', 'epigenetic', 'control', 'of', 'gene', 'expression', 'presumably', 'by', 'modulating', 'higher', 'order', 'chromatin', 'based', 'on', 'sequence', 'information', 'from', 'the', 'set', 'domain', 'we', 'have', 'isolated', 'human', 'suv', 'and', 'mouse', 'suv', 'homologues', 'of', 'the', 'dominant', 'drosophila', 'modifier', 'of', 'position', 'effect', 'variegation', 'pev', 'su', 'var', 'mammalian', 'homologues', 'contain', 'in', 'addition', 'to', 'the', 'set', 'domain', 'the', 'characteristic', 'chrome', 'domain', 'combination', 'that', 'is', 'also', 'preserved', 'in', 'the', 'pombe', 'silencing', 'factor', 'clr', 'chromatin', 'dependent', 'gene', 'regulation', 'is', 'demonstrated', 'by', 'the', 'potential', 'of', 'human', 'suv', 'to', 'increase', 'repression', 'of', 'the', 'pericentromeric', 'white', 'marker', 'gene', 'in', 'transgenic', 'flies', 'immunodetection', 'of', 'endogenous', 'suv', 'suv', 'proteins', 'in', 'variety', 'of', 'mammalian', 'cell', 'lines', 'reveals', 'enriched', 'distribution', 'at', 'heterochromatic', 'foci', 'during', 'interphase', 'and', 'centromere', 'specific', 'localization', 'during', 'metaphase', 'in', 'addition', 'suv', 'suv', 'proteins', 'associate', 'with', 'currently', 'the', 'only', 'other', 'characterized', 'mammalian', 'su', 'var', 'homologue', 'these', 'data', 'indicate', 'the', 'existence', 'of', 'mammalian', 'su', 'var', 'complex', 'and', 'define', 'suv', 'suv', 'as', 'novel', 'components', 'of', 'mammalian', 'higher', 'order', 'chromatin'], ['post', 'transcriptional', 'gene', 'silencing', 'ptgs', 'is', 'homology', 'dependent', 'process', 'that', 'reduces', 'cytoplasmic', 'rna', 'levels', 'in', 'several', 'experimental', 'systems', 'there', 'is', 'also', 'an', 'association', 'of', 'ptgs', 'with', 'methylation', 'of', 'dna', 'to', 'investigate', 'this', 'association', 'we', 'used', 'plants', 'carrying', 'transgene', 'encoding', 'the', 'green', 'fluorescent', 'protein', 'gfp', 'gene', 'silencing', 'was', 'induced', 'using', 'potato', 'virus', 'rna', 'vectors', 'carrying', 'parts', 'of', 'the', 'coding', 'sequence', 'or', 'the', 'promoter', 'of', 'the', 'gfp', 'transgene', 'in', 'each', 'instance', 'homology', 'based', 'rna', 'directed', 'methylation', 'was', 'associated', 'with', 'silencing', 'when', 'the', 'gfp', 'transcribed', 'region', 'was', 'targeted', 'ptgs', 'affected', 'both', 'transgene', 'and', 'viral', 'rna', 'revels', 'when', 'methylation', 'was', 'targeted', 'to', 'promoter', 'region', 'transgene', 'rna', 'levels', 'were', 'reduced', 'however', 'viral', 'rna', 'revels', 'were', 'unaffected', 'for', 'comparison', 'we', 'induced', 'ptgs', 'of', 'the', 'gene', 'encoding', 'the', 'endogenous', 'ribulose', 'bisphosphate', 'carboxylase', 'oxygenase', 'rubisco', 'small', 'subunit', 'rbcs', 'by', 'inoculation', 'with', 'potato', 'virus', 'rbcs', 'in', 'this', 'example', 'no', 'methylation', 'of', 'the', 'rbcs', 'dna', 'was', 'associated', 'with', 'the', 'reduction', 'in', 'rbcs', 'transcript', 'levels', 'and', 'viral', 'rna', 'levels', 'were', 'unaffected', 'finally', 'we', 'investigated', 'dna', 'methylation', 'by', 'using', 'gfp', 'transformed', 'plants', 'in', 'which', 'ptgs', 'was', 'induced', 'by', 'localized', 'introduction', 'of', 'dna', 'carrying', 'gfp', 'sequences', 'in', 'these', 'plants', 'there', 'was', 'methylation', 'of', 'gfp', 'transgene', 'associated', 'with', 'systemic', 'spread', 'of', 'gene', 'silencing', 'signal', 'from', 'the', 'infiltrated', 'part', 'of', 'the', 'plant', 'this', 'transgene', 'methylation', 'was', 'not', 'affected', 'when', 'systemic', 'ptgs', 'was', 'blocked', 'by', 'suppressors', 'of', 'silencing', 'encoded', 'by', 'potato', 'virus', 'and', 'cucumber', 'mosaic', 'virus', 'combined', 'these', 'data', 'support', 'an', 'epigenetic', 'model', 'of', 'ptgs', 'in', 'which', 'transgene', 'methylation', 'is', 'associated', 'with', 'an', 'rna', 'dna', 'interaction', 'that', 'ensures', 'that', 'ptgs', 'is', 'maintained'], ['although', 'the', 'link', 'between', 'the', 'brca', 'tumour', 'suppressor', 'gene', 'and', 'hereditary', 'breast', 'and', 'ovarian', 'cancer', 'is', 'established', 'the', 'role', 'if', 'any', 'of', 'brca', 'in', 'non', 'familial', 'cancers', 'is', 'unclear', 'brca', 'mutations', 'are', 'rare', 'in', 'sporadic', 'cancers', 'but', 'loss', 'of', 'brca', 'resulting', 'from', 'reduced', 'expression', 'or', 'incorrect', 'subcellular', 'localization', 'is', 'postulated', 'to', 'be', 'important', 'in', 'non', 'familial', 'breast', 'and', 'ovarian', 'cancers', 'epigenetic', 'loss', 'however', 'has', 'not', 'received', 'general', 'acceptance', 'due', 'to', 'controversy', 'regarding', 'the', 'subcellular', 'localization', 'of', 'brca', 'proteins', 'reports', 'of', 'which', 'have', 'ranged', 'from', 'exclusively', 'nuclear', 'to', 'conditionally', 'nuclear', 'to', 'the', 'er', 'golgi', 'to', 'cytoplasmic', 'invaginations', 'into', 'the', 'nucleus', 'in', 'an', 'attempt', 'to', 'resolve', 'this', 'issue', 'we', 'have', 'comprehensively', 'characterized', 'anti', 'brca', 'antibodies', 'these', 'reagents', 'detect', 'kd', 'protein', 'localized', 'in', 'discrete', 'nuclear', 'foci', 'in', 'all', 'epithelial', 'cell', 'lines', 'including', 'those', 'derived', 'from', 'breast', 'malignancies', 'staining', 'of', 'human', 'breast', 'specimens', 'also', 'revealed', 'brca', 'nuclear', 'foci', 'in', 'benign', 'breast', 'invasive', 'lobular', 'cancers', 'and', 'low', 'grade', 'ductal', 'carcinomas', 'conversely', 'brca', 'expression', 'was', 'reduced', 'or', 'undetectable', 'in', 'the', 'majority', 'of', 'high', 'grade', 'ductal', 'carcinomas', 'suggesting', 'that', 'absence', 'of', 'brca', 'may', 'contribute', 'to', 'the', 'pathogenesis', 'of', 'significant', 'percentage', 'of', 'sporadic', 'breast', 'cancers'], ['cpg', 'island', 'is', 'known', 'to', 'be', 'associated', 'with', 'gene', 'silencing', 'in', 'cancer', 'this', 'epigenetic', 'event', 'is', 'generally', 'accepted', 'as', 'stochastic', 'process', 'in', 'tumor', 'cells', 'resulting', 'from', 'aberrant', 'dna', 'dna', 'mtase', 'activities', 'specific', 'patterns', 'of', 'cpg', 'island', 'methylation', 'could', 'result', 'from', 'clonal', 'selection', 'of', 'cells', 'having', 'growth', 'advantages', 'due', 'to', 'silencing', 'of', 'associated', 'tumor', 'suppressor', 'genes', 'alternatively', 'methylation', 'patterns', 'may', 'be', 'determined', 'by', 'other', 'as', 'yet', 'unidentified', 'factors', 'to', 'explore', 'further', 'the', 'underlying', 'mechanisms', 'we', 'developed', 'novel', 'array', 'based', 'method', 'called', 'differential', 'methylation', 'hybridization', 'dmh', 'which', 'allows', 'genome', 'wide', 'screening', 'of', 'hypermethylated', 'cpg', 'islands', 'in', 'tumor', 'cells', 'dmh', 'was', 'used', 'to', 'determine', 'the', 'methylation', 'status', 'of', 'cpg', 'island', 'loci', 'in', 'group', 'of', 'breast', 'cancer', 'cell', 'lines', 'between', 'and', 'of', 'these', 'loci', 'were', 'hypermethylated', 'extensively', 'in', 'these', 'cells', 'relative', 'to', 'normal', 'control', 'pattern', 'analysis', 'of', 'positive', 'loci', 'by', 'southern', 'hybridization', 'indicated', 'that', 'cpg', 'islands', 'might', 'differ', 'in', 'their', 'susceptibility', 'to', 'loci', 'exhibiting', 'preexisting', 'methylation', 'in', 'normal', 'controls', 'were', 'more', 'susceptible', 'to', 'de', 'novo', 'methylation', 'in', 'these', 'cancer', 'cells', 'than', 'loci', 'without', 'this', 'condition', 'in', 'addition', 'these', 'cell', 'lines', 'exhibited', 'different', 'intrinsic', 'abilities', 'to', 'methylate', 'cpg', 'islands', 'not', 'directly', 'associated', 'with', 'activities', 'our', 'study', 'provides', 'evidence', 'that', 'aside', 'from', 'random', 'dna', 'mtase', 'action', 'additional', 'cellular', 'factors', 'exist', 'that', 'govern', 'aberrant', 'methylation', 'in', 'breast', 'cancer', 'cells'], ['genomic', 'reprogramming', 'of', 'primordial', 'germ', 'cells', 'pgcs', 'which', 'includes', 'genome', 'wide', 'demethylation', 'prevents', 'aberrant', 'epigenetic', 'modifications', 'from', 'being', 'transmitted', 'to', 'subsequent', 'generations', 'this', 'process', 'also', 'ensures', 'that', 'homologous', 'chromosomes', 'first', 'acquire', 'an', 'identical', 'epigenetic', 'status', 'before', 'an', 'appropriate', 'switch', 'in', 'the', 'imprintable', 'loci', 'in', 'the', 'female', 'and', 'male', 'germ', 'lines', 'embryonic', 'germ', 'eg', 'cells', 'have', 'similar', 'epigenotype', 'to', 'pgcs', 'from', 'which', 'they', 'are', 'derived', 'we', 'used', 'eg', 'cells', 'to', 'investigate', 'the', 'mechanism', 'of', 'epigenetic', 'modifications', 'in', 'the', 'germ', 'line', 'by', 'analysing', 'the', 'effects', 'on', 'somatic', 'nucleus', 'in', 'the', 'eg', 'thymic', 'lymphocyte', 'hybrid', 'cells', 'there', 'were', 'striking', 'changes', 'in', 'methylation', 'of', 'the', 'somatic', 'nucleus', 'resulting', 'in', 'demethylation', 'of', 'several', 'imprinted', 'and', 'non', 'imprinted', 'genes', 'these', 'epigenetic', 'modifications', 'were', 'heritable', 'and', 'affected', 'gene', 'expression', 'as', 'judged', 'by', 're', 'activation', 'of', 'the', 'silent', 'maternal', 'allele', 'of', 'peg', 'mest', 'imprinted', 'gene', 'in', 'the', 'somatic', 'nucleus', 'this', 'remarkable', 'change', 'in', 'the', 'epigenotype', 'of', 'the', 'somatic', 'nucleus', 'is', 'consistent', 'with', 'the', 'observed', 'pluripotency', 'of', 'the', 'eg', 'somatic', 'hybrid', 'cells', 'as', 'they', 'differentiated', 'into', 'variety', 'of', 'tissues', 'in', 'chimeric', 'embryos', 'the', 'epigenetic', 'modifications', 'observed', 'in', 'eg', 'somatic', 'cell', 'hybrids', 'in', 'vitro', 'are', 'comparable', 'to', 'the', 'reprogramming', 'events', 'that', 'occur', 'during', 'germ', 'cell', 'development'], ['to', 'determine', 'which', 'cd', 'components', 'are', 'required', 'for', 'early', 'cell', 'development', 'we', 'generated', 'mice', 'with', 'targeted', 'mutation', 'of', 'the', 'cd', 'epsilon', 'gene', 'and', 'characterized', 'their', 'cell', 'populations', 'relative', 'to', 'those', 'found', 'in', 'cd', 'zeta', 'eta', 'and', 'recombinase', 'activating', 'gene', 'rag', 'deficient', 'mice', 'in', 'the', 'absence', 'of', 'intact', 'cd', 'epsilon', 'subunit', 'thymocytes', 'do', 'not', 'progress', 'beyond', 'the', 'cd', 'low', 'cd', 'triple', 'negative', 'stage', 'and', 'appear', 'to', 'be', 'arrested', 'at', 'the', 'very', 'same', 'developmental', 'control', 'point', 'as', 'rag', 'deficient', 'thymocytes', 'in', 'contrast', 'the', 'disruption', 'of', 'the', 'cd', 'zeta', 'eta', 'gene', 'does', 'not', 'totally', 'abrogate', 'the', 'progression', 'through', 'this', 'control', 'point', 'cd', 'epsilon', 'deficient', 'thymocytes', 'do', 'rearrange', 'their', 'cell', 'receptor', 'tcr', 'beta', 'gene', 'segments', 'and', 'produce', 'low', 'levels', 'of', 'full', 'length', 'tcr', 'beta', 'transcripts', 'taken', 'together', 'these', 'results', 'establish', 'an', 'essential', 'role', 'for', 'the', 'cd', 'epsilon', 'gene', 'products', 'during', 'cell', 'development', 'and', 'further', 'suggest', 'that', 'the', 'cd', 'epsilon', 'polypeptides', 'start', 'to', 'exert', 'their', 'function', 'as', 'part', 'of', 'pre', 'tcr', 'through', 'which', 'cd', 'low', 'cd', 'triple', 'negative', 'cells', 'monitor', 'the', 'occurrence', 'of', 'productive', 'tcr', 'beta', 'gene', 'rearrangements', 'finally', 'the', 'absence', 'of', 'intact', 'cd', 'epsilon', 'polypeptides', 'had', 'no', 'discernible', 'effect', 'on', 'the', 'completion', 'of', 'tcr', 'gamma', 'and', 'tcr', 'delta', 'gene', 'rearrangements', 'emphasizing', 'that', 'they', 'are', 'probably', 'not', 'subjected', 'to', 'the', 'same', 'epigenetic', 'controls', 'as', 'those', 'operating', 'on', 'the', 'expression', 'of', 'tcr', 'alpha', 'and', 'beta', 'genes'], ['the', 'constitutive', 'dnase', 'hypersensitive', 'site', 'at', 'the', 'end', 'of', 'the', 'chicken', 'beta', 'globin', 'locus', 'marks', 'the', 'boundary', 'of', 'the', 'active', 'chromatin', 'domain', 'in', 'erythroid', 'cells', 'the', 'dna', 'sequence', 'containing', 'this', 'site', 'has', 'the', 'properties', 'of', 'an', 'insulator', 'as', 'shown', 'by', 'its', 'ability', 'in', 'stable', 'transformation', 'experiments', 'to', 'block', 'enhancer', 'promoter', 'interaction', 'when', 'it', 'lies', 'between', 'the', 'two', 'but', 'not', 'when', 'it', 'lies', 'outside', 'and', 'to', 'protect', 'against', 'position', 'effects', 'in', 'drosophila', 'we', 'now', 'show', 'that', 'the', 'chicken', 'insulator', 'can', 'protect', 'stably', 'integrated', 'gene', 'which', 'is', 'otherwise', 'subject', 'to', 'great', 'variability', 'of', 'expression', 'from', 'chromatin', 'mediated', 'repression', 'in', 'cell', 'culture', 'when', 'the', 'integrated', 'reporter', 'gene', 'is', 'surrounded', 'by', 'insulator', 'elements', 'stably', 'transformed', 'cell', 'lines', 'display', 'consistent', 'enhancer', 'dependent', 'expression', 'levels', 'in', 'accord', 'with', 'the', 'strength', 'of', 'the', 'enhancer', 'in', 'the', 'absence', 'of', 'insulators', 'long', 'term', 'nonselective', 'propagation', 'of', 'cells', 'carrying', 'the', 'integrated', 'reporter', 'gene', 'results', 'in', 'gradual', 'extinction', 'of', 'the', 'reporter', 'expression', 'with', 'expression', 'patterns', 'from', 'tandemly', 'repeated', 'inserted', 'genes', 'suggesting', 'that', 'the', 'extinction', 'of', 'adjacent', 'genes', 'is', 'coupled', 'we', 'show', 'that', 'the', 'uninsulated', 'reporter', 'genes', 'in', 'addition', 'to', 'becoming', 'inactive', 'lose', 'several', 'epigenetic', 'hallmarks', 'of', 'active', 'chromatin', 'including', 'nuclease', 'accessibility', 'dna', 'hypomethylation', 'and', 'histone', 'during', 'time', 'in', 'culture', 'treatment', 'with', 'inhibitors', 'of', 'histone', 'deacetylase', 'or', 'dna', 'methylation', 'reverses', 'the', 'extinction', 'of', 'the', 'uninsulated', 'genes', 'extinction', 'is', 'completely', 'prevented', 'by', 'flanking', 'the', 'reporter', 'construct', 'with', 'insulators', 'furthermore', 'in', 'contrast', 'to', 'the', 'uninsulated', 'reporter', 'genes', 'chromatin', 'over', 'the', 'insulated', 'genes', 'retains', 'nuclease', 'accessibility', 'and', 'histone', 'however', 'there', 'is', 'no', 'clear', 'correlation', 'between', 'the', 'presence', 'of', 'the', 'insulators', 'and', 'the', 'level', 'of', 'dna', 'methylation', 'this', 'leads', 'us', 'to', 'propose', 'model', 'for', 'the', 'insulator', 'ability', 'to', 'protect', 'against', 'extinction', 'in', 'the', 'transformed', 'cell', 'lines', 'and', 'to', 'function', 'as', 'chromatin', 'boundary', 'for', 'the', 'chicken', 'beta', 'globin', 'locus', 'in', 'normal', 'erythroid', 'cells'], ['kruppel', 'associated', 'box', 'krab', 'domains', 'are', 'present', 'in', 'approximately', 'one', 'third', 'of', 'all', 'human', 'zinc', 'finger', 'proteins', 'zfps', 'and', 'are', 'potent', 'transcriptional', 'repression', 'modules', 'we', 'have', 'previously', 'cloned', 'corepressor', 'for', 'the', 'krab', 'domain', 'kap', 'which', 'is', 'required', 'for', 'krab', 'mediated', 'repression', 'in', 'vivo', 'to', 'characterize', 'the', 'repression', 'mechanism', 'utilized', 'by', 'kap', 'we', 'have', 'analyzed', 'the', 'ability', 'of', 'kap', 'to', 'interact', 'with', 'murine', 'and', 'and', 'human', 'hp', 'alpha', 'and', 'hp', 'gamma', 'homologues', 'of', 'the', 'hpi', 'protein', 'family', 'class', 'of', 'nonhistone', 'heterochromatin', 'associated', 'proteins', 'with', 'well', 'established', 'epigenetic', 'gene', 'silencing', 'function', 'in', 'drosophila', 'in', 'vitro', 'studies', 'confirmed', 'that', 'kap', 'is', 'capable', 'of', 'directly', 'interacting', 'with', 'and', 'hhp', 'alpha', 'which', 'are', 'normally', 'found', 'in', 'centromeric', 'heterochromatin', 'as', 'well', 'as', 'and', 'hhp', 'gamma', 'both', 'of', 'which', 'are', 'found', 'in', 'euchromatin', 'mapping', 'of', 'the', 'region', 'in', 'kap', 'required', 'for', 'hpi', 'interaction', 'showed', 'that', 'amino', 'acid', 'substitutions', 'which', 'abolish', 'hp', 'binding', 'in', 'vitro', 'reduce', 'kap', 'mediated', 'repression', 'in', 'vivo', 'we', 'observed', 'colocalization', 'of', 'kap', 'with', 'and', 'in', 'interphase', 'nuclei', 'lending', 'support', 'to', 'the', 'biochemical', 'evidence', 'that', 'and', 'directly', 'interact', 'with', 'kap', 'the', 'colocalization', 'of', 'kap', 'with', 'is', 'sometimes', 'found', 'in', 'subnuclear', 'territories', 'of', 'potential', 'pericentromeric', 'heterochromatin', 'whereas', 'colocalization', 'of', 'kap', 'and', 'occurs', 'in', 'punctate', 'euchromatic', 'domains', 'throughout', 'the', 'nucleus', 'this', 'work', 'suggests', 'mechanism', 'for', 'the', 'recruitment', 'of', 'hp', 'like', 'gene', 'products', 'by', 'the', 'krab', 'zfp', 'kap', 'complex', 'to', 'specific', 'loci', 'within', 'the', 'genome', 'through', 'formation', 'of', 'heterochromatin', 'like', 'complexes', 'that', 'silence', 'gene', 'activity', 'we', 'speculate', 'that', 'gene', 'specific', 'repression', 'may', 'be', 'consequence', 'of', 'the', 'formation', 'of', 'such', 'complexes', 'ultimately', 'leading', 'to', 'silenced', 'genes', 'in', 'newly', 'formed', 'heterochromatic', 'chromosomal', 'environments'], ['flower', 'pigmentation', 'patterns', 'were', 'scored', 'in', 'sense', 'chalcone', 'synthase', 'chs', 'transgenotes', 'and', 'antisense', 'chs', 'transgenotes', 'upon', 'first', 'flowering', 'of', 'sense', 'transgenotes', 'were', 'found', 'to', 'be', 'phenotypically', 'altered', 'as', 'were', 'of', 'the', 'antisense', 'transgenotes', 'the', 'observed', 'patterns', 'document', 'the', 'range', 'of', 'phenotypic', 'variations', 'that', 'occur', 'as', 'well', 'as', 'confirm', 'and', 'extend', 'the', 'finding', 'that', 'sense', 'chs', 'constructs', 'produce', 'several', 'types', 'of', 'morphology', 'based', 'flower', 'pigmentation', 'patterns', 'that', 'antisense', 'chs', 'constructs', 'do', 'not', 'long', 'term', 'monitoring', 'for', 'epigenetic', 'variations', 'in', 'one', 'population', 'of', 'sense', 'chs', 'transgenotes', 'showed', 'that', 'were', 'capable', 'of', 'producing', 'cosuppression', 'phenotype', 'the', 'primary', 'determinant', 'of', 'sense', 'specific', 'patterns', 'of', 'cosuppression', 'of', 'chs', 'was', 'found', 'to', 'be', 'the', 'repetitiveness', 'and', 'organization', 'pattern', 'of', 'the', 'transgene', 'not', 'position', 'effects', 'by', 'or', 'readthrough', 'from', 'flanking', 'plant', 'dna', 'sequences', 'the', 'degree', 'of', 'cosuppression', 'observed', 'in', 'progeny', 'of', 'transgenotes', 'carrying', 'multiple', 'dispersed', 'copies', 'as', 'compared', 'to', 'that', 'observed', 'with', 'single', 'copy', 'of', 'the', 'transgene', 'suggests', 'that', 'sense', 'cosuppression', 'of', 'chs', 'is', 'subject', 'to', 'transgene', 'dosage', 'effect'], ['transgenic', 'gpt', 'chinese', 'hamster', 'cell', 'line', 'was', 'found', 'to', 'be', 'susceptible', 'to', 'carcinogenic', 'nickel', 'induced', 'inactivation', 'of', 'gpt', 'expression', 'without', 'mutagenesis', 'or', 'deletion', 'of', 'the', 'transgene', 'many', 'nickel', 'induced', 'thioguanine', 'resistant', 'variants', 'spontaneously', 'reverted', 'to', 'actively', 'express', 'gpt', 'as', 'indicated', 'by', 'both', 'reversion', 'assays', 'and', 'direct', 'enzyme', 'measurements', 'since', 'reversion', 'was', 'enhanced', 'in', 'many', 'of', 'the', 'nickel', 'induced', 'variant', 'cell', 'lines', 'following', 'treatment', 'with', 'the', 'demethylating', 'agent', 'azacytidine', 'the', 'involvement', 'of', 'dna', 'methylation', 'in', 'silencing', 'gpt', 'expression', 'was', 'suspected', 'this', 'was', 'confirmed', 'by', 'demonstrations', 'of', 'increased', 'dna', 'methylation', 'as', 'well', 'as', 'by', 'evidence', 'indicating', 'condensed', 'chromatin', 'and', 'of', 'the', 'gpt', 'integration', 'site', 'in', 'thioguanine', 'resistant', 'cells', 'upon', 'reversion', 'to', 'active', 'gpt', 'expression', 'dna', 'methylation', 'and', 'condensation', 'are', 'lost', 'we', 'propose', 'that', 'dna', 'condensation', 'and', 'methylation', 'result', 'in', 'of', 'the', 'gpt', 'sequence', 'with', 'subsequent', 'inheritance', 'of', 'the', 'now', 'silenced', 'gene', 'this', 'mechanism', 'is', 'supported', 'by', 'direct', 'evidence', 'showing', 'that', 'acute', 'nickel', 'treatment', 'of', 'cultured', 'cells', 'and', 'of', 'isolated', 'nuclei', 'in', 'vitro', 'can', 'indeed', 'facilitate', 'gpt', 'sequence', 'specific', 'chromatin', 'condensation', 'epigenetic', 'mechanisms', 'have', 'been', 'implicated', 'in', 'the', 'actions', 'of', 'some', 'nonmutagenic', 'carcinogens', 'and', 'dna', 'methylation', 'changes', 'are', 'now', 'known', 'to', 'be', 'important', 'in', 'carcinogenesis', 'this', 'paper', 'further', 'supports', 'the', 'emerging', 'theory', 'that', 'nickel', 'is', 'human', 'carcinogen', 'that', 'can', 'alter', 'gene', 'expression', 'by', 'enhanced', 'dna', 'methylation', 'and', 'compaction', 'rather', 'than', 'by', 'mutagenic', 'mechanisms'], ['in', 'vitro', 'manipulation', 'of', 'preimplantation', 'mammalian', 'embryos', 'can', 'influence', 'differentiation', 'and', 'growth', 'at', 'later', 'stages', 'of', 'development', 'in', 'the', 'mouse', 'culture', 'of', 'embryonic', 'stem', 'es', 'cells', 'affects', 'their', 'totipotency', 'and', 'may', 'give', 'rise', 'to', 'fetal', 'abnormalities', 'to', 'investigate', 'whether', 'this', 'is', 'associated', 'with', 'epigenetic', 'alterations', 'in', 'imprinted', 'genes', 'we', 'analysed', 'two', 'maternally', 'expressed', 'genes', 'igf', 'and', 'two', 'paternally', 'expressed', 'genes', 'igf', 'af', 'rs', 'in', 'es', 'cells', 'and', 'in', 'completely', 'es', 'cell', 'derived', 'fetuses', 'altered', 'allelic', 'methylation', 'patterns', 'were', 'detected', 'in', 'all', 'four', 'genes', 'and', 'these', 'were', 'consistently', 'associated', 'with', 'allelic', 'changes', 'in', 'gene', 'expression', 'all', 'the', 'methylation', 'changes', 'that', 'had', 'arisen', 'in', 'the', 'es', 'cells', 'persisted', 'on', 'in', 'vivo', 'differentiation', 'to', 'fetal', 'stages', 'alterations', 'included', 'loss', 'of', 'methylation', 'with', 'biallelic', 'expression', 'of', 'af', 'rs', 'maternal', 'methylation', 'and', 'predominantly', 'maternal', 'expression', 'of', 'igf', 'and', 'biallelic', 'methylation', 'and', 'expression', 'of', 'igf', 'in', 'many', 'of', 'the', 'es', 'fetuses', 'the', 'levels', 'of', 'expression', 'were', 'strongly', 'reduced', 'and', 'this', 'biallelic', 'repression', 'was', 'associated', 'with', 'biallellic', 'methylation', 'of', 'the', 'upstream', 'region', 'surprisingly', 'biallelic', 'repression', 'was', 'not', 'associated', 'with', 'equal', 'levels', 'of', 'igf', 'expression', 'from', 'both', 'parental', 'chromosomes', 'but', 'rather', 'with', 'strong', 'activation', 'of', 'the', 'maternal', 'igf', 'allele', 'es', 'fetuses', 'derived', 'from', 'two', 'of', 'the', 'four', 'es', 'lines', 'appeared', 'developmentally', 'compromised', 'with', 'polyhydramnios', 'poor', 'mandible', 'development', 'and', 'interstitial', 'bleeding', 'and', 'in', 'chimeric', 'fetuses', 'the', 'degree', 'of', 'chimerism', 'correlated', 'with', 'increased', 'fetal', 'mass', 'our', 'study', 'establishes', 'model', 'for', 'how', 'early', 'embryonic', 'epigenetic', 'alterations', 'in', 'imprinted', 'genes', 'persist', 'to', 'later', 'developmental', 'stages', 'and', 'are', 'associated', 'with', 'aberrant', 'phenotypes'], ['polycomb', 'group', 'pcg', 'and', 'trithorax', 'group', 'trxg', 'gene', 'products', 'are', 'responsible', 'for', 'the', 'maintenance', 'of', 'repressed', 'and', 'active', 'expression', 'patterns', 'of', 'many', 'developmentally', 'important', 'regulatory', 'genes', 'including', 'the', 'homeotic', 'genes', 'in', 'drosophila', 'embryos', 'polycomb', 'protein', 'and', 'the', 'trxg', 'protein', 'gaga', 'factor', 'colocalize', 'at', 'the', 'fab', 'dna', 'element', 'of', 'the', 'bithorax', 'complex', 'in', 'transgenic', 'lines', 'the', 'fab', 'element', 'induces', 'extensive', 'silencing', 'on', 'flanking', 'gal', 'driven', 'lacz', 'reporter', 'and', 'mini', 'white', 'genes', 'however', 'short', 'single', 'pulse', 'of', 'gal', 'during', 'embryogenesis', 'is', 'sufficient', 'to', 'release', 'pcg', 'dependent', 'silencing', 'from', 'the', 'transgene', 'such', 'an', 'activated', 'state', 'of', 'fab', 'is', 'mitotically', 'inheritable', 'through', 'development', 'and', 'can', 'be', 'transmitted', 'in', 'gal', 'independent', 'manner', 'to', 'the', 'subsequent', 'generations', 'through', 'female', 'meiosis', 'thus', 'fab', 'is', 'switchable', 'chromosomal', 'element', 'which', 'can', 'convey', 'memory', 'of', 'epigenetically', 'determined', 'active', 'and', 'repressed', 'chromatin', 'states'], ['methylation', 'of', 'cytosine', 'residues', 'in', 'dna', 'provides', 'mechanism', 'of', 'gene', 'control', 'there', 'are', 'two', 'classes', 'of', 'in', 'arabidopsis', 'one', 'has', 'carboxy', 'terminal', 'domain', 'fused', 'to', 'an', 'amino', 'terminal', 'regulatory', 'domain', 'and', 'is', 'similar', 'to', 'mammalian', 'the', 'second', 'class', 'apparently', 'lacks', 'an', 'amino', 'terminal', 'domain', 'and', 'is', 'less', 'well', 'conserved', 'methylcytosine', 'can', 'occur', 'at', 'any', 'cytosine', 'residue', 'but', 'it', 'is', 'likely', 'that', 'clonal', 'transmission', 'of', 'methylation', 'patterns', 'only', 'occurs', 'for', 'cytosines', 'in', 'strand', 'symmetrical', 'sequences', 'cpg', 'and', 'cpnpg', 'in', 'plants', 'as', 'in', 'mammals', 'dna', 'methylation', 'has', 'dual', 'roles', 'in', 'defense', 'against', 'invading', 'dna', 'and', 'transposable', 'elements', 'and', 'in', 'gene', 'regulation', 'although', 'originally', 'reported', 'as', 'having', 'no', 'phenotypic', 'consequence', 'reduced', 'dna', 'methylation', 'disrupts', 'normal', 'plant', 'development'], ['the', 'trilaminar', 'kinetochore', 'directs', 'the', 'segregation', 'of', 'chromosomes', 'in', 'mitosis', 'and', 'meiosis', 'despite', 'its', 'importance', 'the', 'molecular', 'architecture', 'of', 'this', 'structure', 'remains', 'poorly', 'understood', 'the', 'best', 'known', 'component', 'of', 'the', 'kinetochore', 'plates', 'is', 'cenp', 'protein', 'that', 'is', 'required', 'for', 'kinetochore', 'assembly', 'but', 'whose', 'molecular', 'role', 'in', 'kinetochore', 'structure', 'and', 'function', 'is', 'unknown', 'here', 'we', 'have', 'raised', 'for', 'the', 'first', 'time', 'monospecific', 'antisera', 'to', 'cenp', 'kd', 'centromere', 'specific', 'histone', 'variant', 'that', 'is', 'identical', 'to', 'the', 'carboxy', 'terminal', 'domain', 'of', 'histone', 'and', 'that', 'resembles', 'the', 'yeast', 'centromeric', 'component', 'cse', 'we', 'have', 'found', 'by', 'simultaneous', 'with', 'centromere', 'antigens', 'of', 'known', 'ultrastructural', 'location', 'that', 'cenp', 'is', 'concentrated', 'in', 'the', 'region', 'of', 'the', 'inner', 'kinetochore', 'plate', 'at', 'active', 'centromeres', 'because', 'cenp', 'was', 'previously', 'shown', 'to', 'co', 'purify', 'with', 'nucleosomes', 'our', 'data', 'suggest', 'specific', 'nucleosomal', 'substructure', 'for', 'the', 'kinetochore', 'in', 'human', 'cells', 'these', 'kinetochore', 'specific', 'nucleosomes', 'are', 'enriched', 'in', 'alpha', 'satellite', 'dna', 'however', 'the', 'association', 'of', 'cenp', 'with', 'neocentromeres', 'lacking', 'detectable', 'alpha', 'satellite', 'dna', 'and', 'the', 'lack', 'of', 'cenp', 'association', 'with', 'alpha', 'satellite', 'rich', 'inactive', 'centromeres', 'of', 'dicentric', 'chromosomes', 'together', 'suggest', 'that', 'cenp', 'association', 'with', 'kinetochores', 'is', 'unlikely', 'to', 'be', 'determined', 'solely', 'by', 'dna', 'sequence', 'recognition', 'we', 'speculate', 'that', 'cenp', 'binding', 'could', 'be', 'consequence', 'of', 'epigenetic', 'tagging', 'of', 'mammalian', 'centromeres'], ['nucleolar', 'dominance', 'is', 'an', 'epigenetic', 'phenomenon', 'that', 'describes', 'nucleolus', 'formation', 'around', 'rrna', 'genes', 'inherited', 'from', 'only', 'one', 'progenitor', 'of', 'an', 'interspecific', 'hy', 'brid', 'or', 'allopolyploid', 'the', 'phenomenon', 'is', 'widespread', 'occurring', 'in', 'plants', 'insects', 'amphibians', 'and', 'mammals', 'yet', 'its', 'molecular', 'basis', 'remains', 'unclear', 'we', 'have', 'demonstrated', 'nucleolar', 'dominance', 'in', 'three', 'allotetraploids', 'of', 'the', 'plant', 'genus', 'brassica', 'in', 'brassica', 'napus', 'accurately', 'initiated', 'pre', 'rrna', 'transcripts', 'from', 'one', 'progenitor', 'brassica', 'rapa', 'are', 'detected', 'readily', 'whereas', 'transcripts', 'from', 'the', 'similar', 'to', 'rrna', 'genes', 'inherited', 'from', 'the', 'other', 'progenitor', 'brassica', 'oleracea', 'are', 'undetectable', 'nuclear', 'run', 'on', 'confirmed', 'that', 'dominance', 'is', 'controlled', 'at', 'the', 'level', 'of', 'transcription', 'growth', 'of', 'napus', 'seedlings', 'on', 'aza', 'deoxycytidine', 'to', 'inhibit', 'cytosine', 'methylation', 'caused', 'the', 'normally', 'silent', 'under', 'dominant', 'oleracea', 'rrna', 'genes', 'to', 'become', 'expressed', 'to', 'high', 'levels', 'the', 'histone', 'deacetylase', 'inhibitors', 'sodium', 'butyrate', 'and', 'trichostatin', 'also', 'de', 'epressed', 'silent', 'rrna', 'genes', 'these', 'results', 'reveal', 'an', 'enforcement', 'mechanism', 'for', 'nucleolar', 'dominance', 'in', 'which', 'dna', 'methylation', 'and', 'histone', 'modifications', 'combine', 'to', 'regulate', 'rrna', 'gene', 'loci', 'spanning', 'tens', 'of', 'megabase', 'pairs', 'of', 'dna'], ['genomic', 'imprinting', 'is', 'an', 'epigenetic', 'chromosomal', 'modification', 'in', 'the', 'gamete', 'or', 'zygote', 'causing', 'preferential', 'expression', 'of', 'specific', 'parental', 'allele', 'in', 'somatic', 'cells', 'of', 'the', 'offspring', 'we', 'and', 'others', 'have', 'identified', 'three', 'imprinted', 'human', 'genes', 'on', 'igf', 'refs', 'refs', 'and', 'kip', 'ref', 'although', 'the', 'latter', 'gene', 'is', 'separated', 'by', 'kb', 'from', 'the', 'other', 'two', 'and', 'it', 'is', 'unclear', 'whether', 'there', 'are', 'other', 'imprinted', 'genes', 'within', 'this', 'large', 'interval', 'we', 'previously', 'mapped', 'an', 'embryonal', 'tumour', 'suppressor', 'gene', 'to', 'this', 'region', 'as', 'well', 'as', 'five', 'balanced', 'germline', 'chromosomal', 'rearrangement', 'breakpoints', 'from', 'patients', 'with', 'beckwith', 'wiedemann', 'syndrome', 'bws', 'condition', 'characterized', 'by', 'prenatal', 'overgrowth', 'and', 'cancer', 'we', 'isolated', 'the', 'upstream', 'exons', 'of', 'the', 'previously', 'identified', 'gene', 'kvlqt', 'which', 'causes', 'the', 'familial', 'cardiac', 'defect', 'long', 'qt', 'lqt', 'syndrome', 'we', 'found', 'that', 'kvlqt', 'spans', 'much', 'of', 'the', 'interval', 'between', 'kip', 'and', 'igf', 'and', 'that', 'it', 'is', 'also', 'imprinted', 'we', 'demonstrated', 'that', 'the', 'gene', 'is', 'disrupted', 'by', 'chromosomal', 'chromosomal', 'translocation', 'in', 'an', 'embryonal', 'rhabdoid', 'tumour', 'furthermore', 'the', 'lack', 'of', 'parent', 'of', 'origin', 'effect', 'in', 'lqt', 'syndrome', 'appears', 'to', 'be', 'due', 'to', 'relative', 'lack', 'of', 'imprinting', 'in', 'the', 'affected', 'tissue', 'cardiac', 'muscle', 'representing', 'novel', 'mechanism', 'for', 'variable', 'penetrance', 'of', 'human', 'disease', 'gene'], ['multisubunit', 'cohesin', 'complex', 'holds', 'sister', 'chromatids', 'together', 'after', 'dna', 'replication', 'using', 'chromatin', 'we', 'detected', 'cohesin', 'association', 'with', 'centromeres', 'and', 'with', 'discrete', 'sites', 'along', 'chromosome', 'arms', 'from', 'phase', 'until', 'metaphase', 'in', 'cerevisiae', 'short', 'dna', 'sequences', 'bp', 'are', 'sufficient', 'to', 'confer', 'cohesin', 'association', 'cohesin', 'association', 'with', 'centromere', 'depends', 'on', 'mif', 'the', 'centromere', 'binding', 'factor', 'cbf', 'and', 'centromere', 'specific', 'histone', 'variant', 'cse', 'because', 'only', 'active', 'centromeres', 'confer', 'cohesin', 'association', 'with', 'centromeric', 'dna', 'we', 'suggest', 'that', 'cohesin', 'is', 'recruited', 'by', 'the', 'same', 'chromatin', 'structure', 'that', 'confers', 'the', 'attachment', 'of', 'microtubules', 'propagation', 'of', 'this', 'structure', 'might', 'be', 'partly', 'epigenetic', 'finally', 'cohesion', 'associated', 'with', 'minimal', 'centromeres', 'is', 'insufficient', 'to', 'resist', 'the', 'splitting', 'force', 'exerted', 'by', 'microtubules', 'and', 'appears', 'to', 'be', 'reinforced', 'by', 'cohesion', 'provided', 'by', 'their', 'flanking', 'dna', 'sequences'], ['loss', 'of', 'imprinting', 'loi', 'is', 'an', 'epigenetic', 'alteration', 'of', 'some', 'cancers', 'involving', 'loss', 'of', 'parental', 'origin', 'specific', 'expression', 'of', 'imprinted', 'genes', 'we', 'observed', 'loi', 'of', 'the', 'insulin', 'like', 'growth', 'factor', 'ii', 'gene', 'in', 'twelve', 'of', 'twenty', 'seven', 'informative', 'colorectal', 'cancer', 'patients', 'as', 'well', 'as', 'in', 'the', 'matched', 'normal', 'colonic', 'mucosa', 'of', 'the', 'patients', 'with', 'loi', 'in', 'their', 'cancers', 'and', 'in', 'peripheral', 'blood', 'samples', 'of', 'four', 'patients', 'ten', 'of', 'eleven', 'cancers', 'with', 'microsatellite', 'instability', 'showed', 'loi', 'compared', 'with', 'only', 'two', 'of', 'sixteen', 'tumors', 'without', 'microsatellite', 'instability', 'control', 'patients', 'without', 'cancer', 'showed', 'loi', 'in', 'colonic', 'mucosa', 'of', 'only', 'two', 'of', 'sixteen', 'cases', 'and', 'two', 'of', 'fifteen', 'blood', 'samples', 'these', 'data', 'suggest', 'that', 'loi', 'in', 'tumor', 'and', 'normal', 'tissue', 'identifies', 'most', 'colorectal', 'cancer', 'patients', 'with', 'microsatellite', 'instability', 'in', 'their', 'tumors', 'and', 'that', 'loi', 'may', 'identify', 'an', 'important', 'subset', 'of', 'the', 'population', 'with', 'cancer', 'or', 'at', 'risk', 'of', 'developing', 'cancer'], ['silencing', 'is', 'an', 'epigenetic', 'form', 'of', 'transcriptional', 'regulation', 'whereby', 'genes', 'are', 'heritably', 'but', 'not', 'necessarily', 'permanently', 'inactivated', 'we', 'have', 'identified', 'the', 'saccharomyces', 'cerevisiae', 'genes', 'sas', 'and', 'sas', 'through', 'screen', 'for', 'enhancers', 'of', 'sir', 'epigenetic', 'silencing', 'defects', 'sas', 'sas', 'and', 'pombe', 'homologue', 'are', 'closely', 'related', 'to', 'several', 'human', 'genes', 'including', 'one', 'associated', 'with', 'acute', 'myeloid', 'leukaemia', 'arising', 'from', 'the', 'recurrent', 'translocation', 'and', 'one', 'implicated', 'in', 'hiv', 'tat', 'interactions', 'all', 'of', 'these', 'genes', 'encode', 'proteins', 'with', 'an', 'atypical', 'zinc', 'finger', 'and', 'well', 'conserved', 'similarities', 'to', 'sequence', 'similarities', 'and', 'yeast', 'mutant', 'phenotypes', 'suggest', 'that', 'sas', 'like', 'genes', 'function', 'in', 'transcriptional', 'regulation', 'and', 'cell', 'cycle', 'exit', 'and', 'reveal', 'novel', 'connections', 'between', 'transcriptional', 'silencing', 'and', 'human', 'disease'], ['the', 'multisubunit', 'msl', 'dosage', 'compensation', 'complex', 'binds', 'to', 'hundreds', 'of', 'sites', 'along', 'the', 'drosophila', 'single', 'male', 'chromosome', 'mediating', 'its', 'the', 'male', 'chromosome', 'is', 'also', 'coated', 'with', 'noncoding', 'rox', 'rnas', 'when', 'either', 'msl', 'mle', 'or', 'mof', 'is', 'mutant', 'partial', 'msl', 'complex', 'is', 'bound', 'at', 'only', 'similar', 'to', 'unusual', 'sites', 'distributed', 'along', 'the', 'we', 'show', 'that', 'two', 'of', 'these', 'sites', 'are', 'the', 'rox', 'and', 'rox', 'genes', 'and', 'postulate', 'that', 'one', 'of', 'their', 'functions', 'is', 'to', 'provide', 'entry', 'sites', 'for', 'the', 'msl', 'complex', 'to', 'recognize', 'the', 'chromosome', 'the', 'rox', 'gene', 'provides', 'nucleation', 'site', 'for', 'extensive', 'spreading', 'of', 'the', 'msl', 'complex', 'into', 'flanking', 'chromatin', 'even', 'when', 'moved', 'to', 'an', 'autosome', 'the', 'spreading', 'can', 'occur', 'in', 'cis', 'or', 'in', 'trans', 'between', 'paired', 'homologs', 'we', 'present', 'model', 'for', 'how', 'the', 'dosage', 'compensation', 'complex', 'recognizes', 'chromatin'], ['defective', 'dna', 'mismatch', 'repair', 'in', 'human', 'turners', 'leads', 'to', 'genome', 'wide', 'instability', 'of', 'microsatellite', 'repeats', 'and', 'molecular', 'phenotype', 'referred', 'to', 'as', 'microsatellite', 'instability', 'msi', 'msi', 'has', 'been', 'reported', 'in', 'variety', 'of', 'cancers', 'and', 'is', 'consistent', 'feature', 'of', 'tumors', 'from', 'patients', 'with', 'hereditary', 'non', 'polyposis', 'colorectal', 'cancer', 'approximately', 'of', 'cancers', 'of', 'the', 'uterine', 'endometrium', 'the', 'fifth', 'most', 'common', 'cancer', 'of', 'women', 'world', 'wide', 'exhibit', 'msi', 'although', 'the', 'frequency', 'of', 'msi', 'is', 'higher', 'in', 'endometrial', 'cancers', 'than', 'in', 'any', 'other', 'common', 'malignancy', 'the', 'genetic', 'basis', 'of', 'msi', 'in', 'these', 'tumors', 'has', 'remained', 'elusive', 'we', 'investigated', 'the', 'role', 'that', 'methylation', 'of', 'the', 'mlh', 'dna', 'mismatch', 'repair', 'gene', 'plays', 'in', 'the', 'genesis', 'of', 'msi', 'in', 'large', 'series', 'of', 'sporadic', 'endometrial', 'cancers', 'the', 'mlh', 'promoter', 'was', 'methylated', 'in', 'of', 'msi', 'positive', 'cancers', 'investigated', 'in', 'msi', 'negative', 'tumors', 'on', 'the', 'other', 'hand', 'there', 'was', 'evidence', 'for', 'limited', 'methylation', 'in', 'only', 'one', 'of', 'tumors', 'studied', 'investigation', 'of', 'subset', 'of', 'the', 'tumors', 'revealed', 'that', 'methylation', 'of', 'the', 'mlh', 'promoter', 'in', 'msi', 'positive', 'tumors', 'was', 'associated', 'with', 'loss', 'of', 'mlh', 'expression', 'proved', 'that', 'two', 'msi', 'positive', 'tumors', 'lacking', 'mlh', 'methylation', 'failed', 'to', 'express', 'the', 'msh', 'mismatch', 'repair', 'gene', 'both', 'of', 'these', 'cancers', 'came', 'from', 'women', 'who', 'had', 'family', 'and', 'medical', 'histories', 'suggestive', 'of', 'inherited', 'cancer', 'susceptibility', 'these', 'observations', 'suggest', 'that', 'epigenetic', 'changes', 'in', 'the', 'mlh', 'locus', 'account', 'for', 'msi', 'in', 'most', 'cases', 'of', 'sporadic', 'endometrial', 'cancers', 'and', 'provide', 'additional', 'evidence', 'that', 'the', 'msh', 'gene', 'may', 'contribute', 'substantially', 'to', 'inherited', 'forms', 'of', 'endometrial', 'cancer'], ['several', 'lines', 'of', 'evidence', 'strongly', 'suggest', 'that', 'dna', 'methylation', 'is', 'involved', 'in', 'embryo', 'development', 'perhaps', 'the', 'most', 'direct', 'evidence', 'comes', 'from', 'experiments', 'with', 'deficient', 'mice', 'embryos', 'that', 'express', 'low', 'levels', 'of', 'the', 'maintenance', 'do', 'not', 'develop', 'to', 'term', 'and', 'die', 'at', 'the', 'to', 'somite', 'stage', 'corresponding', 'to', 'the', 'level', 'of', 'the', 'enzyme', 'in', 'the', 'mouse', 'dramatic', 'methylation', 'changes', 'have', 'been', 'observed', 'during', 'the', 'early', 'steps', 'of', 'embryo', 'development', 'most', 'genes', 'are', 'subject', 'to', 'process', 'of', 'active', 'demethylation', 'starting', 'promptly', 'after', 'fertilization', 'except', 'for', 'small', 'number', 'of', 'methylated', 'cpg', 'sites', 'in', 'imprinted', 'genes', 'the', 'vast', 'majority', 'of', 'the', 'sites', 'are', 'unmethylated', 'by', 'the', 'stage', 'of', 'cavitation', 'cell', 'such', 'genome', 'wide', 'demethylation', 'may', 'signify', 'an', 'erasure', 'of', 'epigenetic', 'information', 'originating', 'in', 'the', 'highly', 'differentiated', 'gametes', 'this', 'erasure', 'may', 'be', 'essential', 'for', 'the', 'establishment', 'of', 'pluripotent', 'state', 'before', 'specific', 'cell', 'lineages', 'can', 'be', 'determined', 'the', 'process', 'of', 'laying', 'down', 'new', 'developmental', 'program', 'involves', 'initially', 'global', 'de', 'novo', 'methylation', 'at', 'the', 'stage', 'of', 'pregastrulation', 'followed', 'by', 'gene', 'specific', 'demethylations', 'associated', 'with', 'the', 'onset', 'of', 'activity', 'of', 'these', 'genes', 'cpg', 'islands', 'characteristic', 'of', 'housekeeping', 'genes', 'appear', 'to', 'be', 'protected', 'from', 'the', 'global', 'de', 'novo', 'methylation', 'an', 'exception', 'to', 'this', 'rule', 'is', 'the', 'of', 'cpg', 'islands', 'in', 'linked', 'housekeeping', 'genes', 'on', 'the', 'inactive', 'chromosome', 'and', 'of', 'specific', 'differentially', 'methylated', 'cpg', 'sites', 'in', 'imprinted', 'genes', 'primordial', 'germ', 'cells', 'escape', 'the', 'global', 'de', 'novo', 'methylation', 'which', 'takes', 'place', 'at', 'the', 'pregastrula', 'stage', 'and', 'undergo', 'very', 'similar', 'de', 'novo', 'methylation', 'process', 'in', 'the', 'differentiated', 'gonads', 'days', 'post', 'coitum', 'forming', 'the', 'methylation', 'patterns', 'which', 'are', 'specific', 'to', 'the', 'gametes', 'specific', 'demethylations', 'then', 'form', 'terminal', 'methylation', 'pattern', 'which', 'is', 'then', 'clonaly', 'inherited', 'in', 'the', 'soma', 'and', 'erased', 'after', 'fertilization'], ['during', 'the', 'development', 'of', 'multicellular', 'organism', 'from', 'zygote', 'large', 'number', 'of', 'epigenetic', 'interactions', 'take', 'place', 'on', 'every', 'level', 'of', 'suborganismal', 'organization', 'this', 'raises', 'the', 'possibility', 'that', 'the', 'system', 'of', 'epigenetic', 'interactions', 'may', 'compensate', 'or', 'buffer', 'some', 'of', 'the', 'changes', 'that', 'occur', 'as', 'mutations', 'on', 'its', 'lowest', 'levels', 'and', 'thus', 'stabilize', 'the', 'phenotype', 'with', 'respect', 'to', 'mutations', 'this', 'hypothetical', 'phenomenon', 'will', 'be', 'called', 'epigenetic', 'stability', 'its', 'potential', 'importance', 'stems', 'from', 'the', 'fact', 'that', 'phenotypic', 'variation', 'with', 'genetic', 'basis', 'is', 'an', 'essential', 'prerequisite', 'for', 'evolution', 'thus', 'variation', 'in', 'epigenetic', 'stability', 'might', 'profoundly', 'affect', 'attainable', 'rates', 'of', 'evolution', 'while', 'representing', 'systemic', 'property', 'of', 'developmental', 'system', 'epigenetic', 'stability', 'might', 'itself', 'be', 'genetically', 'determined', 'and', 'thus', 'be', 'subject', 'to', 'evolutionary', 'change', 'whether', 'or', 'not', 'this', 'is', 'the', 'case', 'should', 'ideally', 'be', 'answered', 'directly', 'that', 'is', 'by', 'experimentation', 'the', 'time', 'seals', 'involved', 'and', 'our', 'insufficient', 'quantitative', 'understanding', 'of', 'developmental', 'pathways', 'will', 'probably', 'preclude', 'such', 'an', 'approach', 'in', 'the', 'foreseeable', 'future', 'preliminary', 'answers', 'are', 'sought', 'here', 'by', 'using', 'biochemically', 'motivated', 'model', 'of', 'small', 'bur', 'central', 'part', 'of', 'developmental', 'pathway', 'modeled', 'are', 'sets', 'of', 'transcriptional', 'regulators', 'that', 'mutually', 'regulate', 'each', 'other', 'expression', 'and', 'thereby', 'form', 'stable', 'gene', 'expression', 'patterns', 'such', 'gene', 'expression', 'patterns', 'crucially', 'involved', 'in', 'determining', 'developmental', 'pattern', 'formation', 'events', 'are', 'most', 'likely', 'subject', 'to', 'strong', 'stabilizing', 'natural', 'selection', 'after', 'long', 'periods', 'of', 'stabilizing', 'selection', 'the', 'fraction', 'of', 'mutations', 'causing', 'changes', 'in', 'gene', 'expression', 'pattern', 'is', 'substantially', 'reduced', 'in', 'the', 'model', 'epigenetic', 'stability', 'has', 'increased', 'this', 'phenomenon', 'is', 'found', 'for', 'widely', 'varying', 'regulatory', 'scenarios', 'among', 'transcription', 'factor', 'genes', 'its', 'discussed', 'that', 'only', 'epistatic', 'nonlinear', 'gene', 'interactions', 'can', 'cause', 'such', 'change', 'in', 'epigenetic', 'stability', 'evidence', 'from', 'paleontology', 'molecular', 'evolution', 'development', 'and', 'genetics', 'consistent', 'with', 'the', 'existence', 'of', 'variation', 'in', 'epigenetic', 'stability', 'is', 'discussed', 'the', 'relation', 'of', 'epigenetic', 'stability', 'to', 'developmental', 'canalization', 'is', 'outlined', 'experimental', 'scenarios', 'at', 'suggested', 'thar', 'may', 'provide', 'further', 'evidence'], ['chromosome', 'inactivation', 'is', 'the', 'process', 'by', 'which', 'cell', 'recognizes', 'the', 'presence', 'of', 'two', 'copies', 'of', 'an', 'chromosome', 'early', 'in', 'the', 'development', 'of', 'xx', 'embryos', 'and', 'chooses', 'one', 'to', 'be', 'active', 'and', 'one', 'to', 'be', 'inactive', 'although', 'it', 'is', 'commonly', 'believed', 'that', 'the', 'initiation', 'of', 'inactivation', 'is', 'random', 'with', 'an', 'equal', 'probability', 'that', 'either', 'chromosome', 'will', 'be', 'the', 'inactive', 'in', 'given', 'cell', 'significant', 'variation', 'in', 'the', 'proportion', 'of', 'cells', 'with', 'either', 'inactive', 'is', 'observed', 'both', 'in', 'mice', 'heterozygous', 'for', 'alleles', 'at', 'the', 'xce', 'locus', 'and', 'among', 'normal', 'human', 'females', 'in', 'the', 'population', 'families', 'in', 'which', 'multiple', 'females', 'demonstrate', 'extremely', 'skewed', 'inactivation', 'patterns', 'that', 'are', 'otherwise', 'quite', 'rare', 'in', 'the', 'general', 'population', 'are', 'thought', 'to', 'reflect', 'possible', 'genetic', 'influences', 'on', 'the', 'inactivation', 'process', 'here', 'we', 'report', 'rare', 'cytosine', 'to', 'guanine', 'mutation', 'in', 'the', 'xist', 'minimal', 'promoter', 'that', 'underlies', 'both', 'epigenetic', 'and', 'functional', 'differences', 'between', 'the', 'two', 'chromosomes', 'in', 'nine', 'females', 'from', 'two', 'unrelated', 'families', 'all', 'females', 'demonstrate', 'preferential', 'inactivation', 'of', 'the', 'chromosome', 'carrying', 'the', 'mutation', 'suggesting', 'that', 'there', 'is', 'an', 'association', 'between', 'alterations', 'in', 'the', 'regulation', 'of', 'xist', 'expression', 'and', 'chromosome', 'inactivation'], ['genomic', 'imprinting', 'is', 'an', 'epigenetic', 'process', 'that', 'results', 'in', 'the', 'preferential', 'silencing', 'of', 'one', 'of', 'the', 'two', 'parental', 'copies', 'of', 'gene', 'although', 'the', 'precise', 'mechanisms', 'by', 'which', 'genomic', 'imprinting', 'occurs', 'are', 'unknown', 'the', 'tendency', 'of', 'imprinted', 'genes', 'to', 'exist', 'in', 'chromosomal', 'clusters', 'suggests', 'long', 'range', 'regulation', 'through', 'shared', 'regulatory', 'elements', 'we', 'characterize', 'kb', 'region', 'on', 'the', 'distal', 'end', 'of', 'mouse', 'chromosome', 'that', 'contains', 'cluster', 'of', 'four', 'maternally', 'expressed', 'genes', 'mash', 'kvlqt', 'and', 'kip', 'as', 'well', 'as', 'two', 'paternally', 'expressed', 'genes', 'igf', 'and', 'ins', 'and', 'assess', 'the', 'expression', 'and', 'imprinting', 'of', 'mash', 'kvlqt', 'and', 'kip', 'during', 'development', 'in', 'embryonic', 'and', 'extraembryonic', 'tissues', 'unlike', 'igf', 'and', 'ins', 'which', 'depend', 'on', 'for', 'their', 'imprinting', 'mash', 'kip', 'and', 'kvlqt', 'are', 'unaffected', 'by', 'deletion', 'of', 'the', 'gene', 'region', 'suggesting', 'that', 'these', 'more', 'telomeric', 'genes', 'are', 'not', 'regulated', 'by', 'the', 'mechanism', 'that', 'controls', 'igf', 'and', 'ins', 'mutations', 'in', 'human', 'kip', 'have', 'been', 'implicated', 'in', 'beckwith', 'wiedemann', 'syndrome', 'disease', 'that', 'has', 'also', 'been', 'associated', 'with', 'loss', 'of', 'imprinting', 'of', 'igf', 'we', 'find', 'however', 'that', 'deletion', 'of', 'the', 'gene', 'has', 'no', 'effect', 'on', 'imprinting', 'within', 'the', 'cluster', 'surprisingly', 'the', 'three', 'maternally', 'expressed', 'genes', 'are', 'regulated', 'very', 'differently', 'by', 'dna', 'methylation', 'kip', 'activated', 'kvlqt', 'is', 'silenced', 'and', 'mash', 'is', 'unaffected', 'in', 'mice', 'lacking', 'dna', 'we', 'conclude', 'that', 'is', 'not', 'global', 'regulator', 'of', 'imprinting', 'on', 'distal', 'chromosome', 'and', 'that', 'the', 'telomeric', 'genes', 'are', 'imprinted', 'by', 'separate', 'mechanism'], ['skeletal', 'myoblasts', 'are', 'derived', 'from', 'paraxial', 'mesoderm', 'but', 'how', 'myoblasts', 'acquire', 'their', 'identity', 'is', 'still', 'matter', 'of', 'speculation', 'the', 'of', 'molecular', 'markers', 'and', 'in', 'some', 'cases', 'the', 'analysis', 'of', 'mutations', 'in', 'the', 'corresponding', 'genes', 'has', 'now', 'made', 'it', 'possible', 'to', 'ask', 'specific', 'questions', 'about', 'this', 'process', 'specification', 'of', 'somite', 'cell', 'fate', 'depends', 'on', 'epigenetic', 'factors', 'adjacent', 'tissues', 'such', 'as', 'the', 'neural', 'tube', 'notochord', 'dorsal', 'ectoderm', 'and', 'lateral', 'mesoderm', 'act', 'either', 'positively', 'or', 'negatively', 'on', 'the', 'different', 'myogenic', 'precursor', 'populations', 'in', 'the', 'somite', 'candidate', 'molecules', 'for', 'this', 'complex', 'signalling', 'activity', 'include', 'sonic', 'hedgehog', 'and', 'the', 'wnt', 'proteins', 'as', 'positive', 'signals', 'and', 'bmp', 'as', 'possible', 'inhibitor', 'although', 'it', 'is', 'generally', 'assumed', 'that', 'induction', 'is', 'required', 'some', 'observations', 'suggest', 'that', 'embryonic', 'cells', 'might', 'have', 'tendency', 'to', 'undergo', 'myogenesis', 'as', 'default', 'pathway', 'by', 'analogy', 'with', 'drosophila', 'where', 'the', 'neurogenic', 'genes', 'affect', 'myogenesis', 'the', 'vertebrate', 'homologues', 'of', 'notch', 'and', 'its', 'ligands', 'could', 'be', 'candidate', 'molecules', 'for', 'repression', 'or', 'derepression', 'mechanism', 'similar', 'studies', 'with', 'cultured', 'muscle', 'cells', 'also', 'implicate', 'other', 'hlh', 'factors', 'as', 'potential', 'inhibitors', 'of', 'the', 'myod', 'family', 'and', 'hence', 'of', 'inappropriate', 'myogenesis'], ['overexpression', 'of', 'chimeric', 'transgenes', 'in', 'plants', 'can', 'trigger', 'post', 'transcriptional', 'gene', 'silencing', 'that', 'is', 'dependent', 'on', 'epigenetic', 'information', 'and', 'physiological', 'conditions', 'the', 'current', 'view', 'is', 'that', 'unproductive', 'rna', 'serves', 'as', 'crucial', 'signal', 'for', 'gene', 'silencing', 'although', 'direct', 'evidence', 'is', 'lacking', 'for', 'this', 'theory', 'signalling', 'cascade', 'then', 'leads', 'to', 'strongly', 'enhanced', 'turnover', 'of', 'all', 'rnas', 'that', 'share', 'critical', 'degree', 'of', 'sequence', 'similarity', 'the', 'molecular', 'details', 'of', 'the', 'mechanism', 'are', 'however', 'insufficiently', 'understood', 'to', 'explain', 'the', 'phenomenon', 'completely', 'and', 'to', 'comprehend', 'its', 'biological', 'significance'], [], ['represents', 'the', 'sum', 'of', 'complex', 'of', 'simultaneous', 'and', 'sequential', 'biochemical', 'events', 'that', 'ultimately', 'lead', 'to', 'the', 'occurrence', 'of', 'skin', 'cancer', 'these', 'events', 'initiated', 'by', 'uv', 'radiation', 'of', 'appropriate', 'wavelength', 'include', 'the', 'formation', 'of', 'dna', 'photoproducts', 'dna', 'repair', 'mutation', 'of', 'proto', 'oncogenes', 'and', 'tumor', 'suppressor', 'genes', 'uv', 'production', 'of', 'radical', 'species', 'with', 'subsequent', 'effects', 'on', 'mutation', 'and', 'extra', 'nuclear', 'function', 'and', 'other', 'epigenetic', 'events', 'that', 'influence', 'the', 'course', 'of', 'carcinogenesis', 'the', 'epigenetic', 'influences', 'may', 'include', 'immunological', 'responses', 'antioxidant', 'defenses', 'and', 'dietary', 'factors', 'this', 'review', 'represents', 'an', 'effort', 'to', 'provide', 'current', 'research', 'results', 'in', 'the', 'aforementioned', 'areas', 'and', 'an', 'attempt', 'to', 'meld', 'these', 'events', 'into', 'comprehensive', 'overview', 'of', 'if', 'effective', 'prevention', 'and', 'intervention', 'strategies', 'for', 'skin', 'cancer', 'are', 'to', 'be', 'developed', 'more', 'thorough', 'understanding', 'of', 'the', 'disease', 'process', 'is', 'imperative', 'elsevier', 'science'], [], ['several', 'proteins', 'that', 'contribute', 'to', 'epigenetic', 'mechanisms', 'of', 'gene', 'regulation', 'contain', 'characteristic', 'motif', 'of', 'unknown', 'function', 'called', 'the', 'set', 'suvar', 'enhancer', 'of', 'zeste', 'trithorax', 'domain', 'we', 'have', 'demonstrated', 'that', 'set', 'domains', 'mediate', 'highly', 'conserved', 'interactions', 'with', 'specific', 'family', 'of', 'proteins', 'that', 'display', 'similarity', 'with', 'dual', 'specificity', 'phosphatases', 'dsptpases', 'these', 'include', 'myotubularin', 'the', 'gene', 'of', 'which', 'is', 'mutated', 'in', 'subset', 'of', 'patients', 'with', 'linked', 'myotubular', 'myopathy', 'and', 'sbf', 'newly', 'isolated', 'homologue', 'of', 'myotubularin', 'in', 'contrast', 'with', 'myotubularin', 'sbf', 'lacks', 'functional', 'catalytic', 'domain', 'which', 'phospho', 'tyrosine', 'and', 'serine', 'containing', 'peptides', 'in', 'vitro', 'competitive', 'interference', 'of', 'endogenous', 'set', 'domain', 'dsptpase', 'interactions', 'by', 'forced', 'expression', 'of', 'sbf', 'induced', 'oncogenic', 'transformation', 'of', 'nih', 'fibroblasts', 'and', 'impaired', 'the', 'in', 'vitro', 'differentiation', 'of', 'myoblast', 'cells', 'we', 'conclude', 'that', 'myotubularin', 'type', 'phosphatases', 'link', 'set', 'domain', 'containing', 'components', 'of', 'the', 'epigenetic', 'regulatory', 'machinery', 'with', 'signalling', 'pathways', 'involved', 'in', 'growth', 'and', 'differentiation'], ['methylguanine', 'dna', 'mgmt', 'repairs', 'the', 'mutagenic', 'and', 'cytotoxic', 'alkylguanine', 'lesions', 'produced', 'bg', 'environmental', 'carcinogens', 'and', 'the', 'nitrosoureas', 'as', 'such', 'mgmt', 'mediated', 'repair', 'of', 'alkylguanine', 'lesions', 'constitutes', 'major', 'form', 'of', 'resistance', 'to', 'nitrosourea', 'chemotherapy', 'and', 'makes', 'control', 'of', 'mgmt', 'expression', 'of', 'clinical', 'interest', 'the', 'variability', 'of', 'expression', 'in', 'cell', 'lines', 'and', 'tissues', 'along', 'with', 'the', 'ease', 'with', 'which', 'the', 'mgmt', 'phenotype', 'reverts', 'under', 'various', 'conditions', 'suggests', 'that', 'mgmt', 'is', 'under', 'epigenetic', 'control', 'one', 'such', 'epigenetic', 'mechanism', 'methylation', 'of', 'cytosines', 'has', 'been', 'linked', 'to', 'mgmt', 'expression', 'we', 'have', 'used', 'an', 'isogenic', 'human', 'multiple', 'myeloma', 'tumor', 'cell', 'line', 'model', 'composed', 'of', 'an', 'mgmt', 'positive', 'parent', 'cell', 'line', 'rpmi', 'and', 'its', 'mgmt', 'negative', 'variant', 'termed', 'to', 'study', 'the', 'control', 'of', 'mgmt', 'expression', 'the', 'loss', 'of', 'mgmt', 'activity', 'in', 'was', 'found', 'to', 'he', 'due', 'to', 'the', 'loss', 'of', 'detectable', 'mgmt', 'gene', 'expression', 'bisulfite', 'sequencing', 'of', 'the', 'mgmt', 'cpg', 'island', 'promoter', 'revealed', 'large', 'increases', 'in', 'the', 'levels', 'of', 'cpg', 'methylation', 'within', 'discrete', 'regions', 'of', 'the', 'mgmt', 'cpg', 'island', 'compared', 'to', 'those', 'in', 'these', 'changes', 'in', 'cpg', 'methylation', 'are', 'associated', 'with', 'local', 'of', 'the', 'mgmt', 'transcription', 'start', 'site', 'and', 'provide', 'likely', 'mechanism', 'for', 'the', 'loss', 'of', 'mgmt', 'transcription', 'in'], ['inheritance', 'of', 'the', 'active', 'and', 'inactive', 'states', 'of', 'gene', 'expression', 'by', 'individual', 'cells', 'is', 'crucial', 'for', 'development', 'in', 'fission', 'yeast', 'mating', 'type', 'region', 'consists', 'of', 'three', 'loci', 'called', 'mat', 'mat', 'and', 'mat', 'silent', 'mat', 'and', 'mat', 'loci', 'are', 'separated', 'by', 'kb', 'interval', 'designated', 'the', 'region', 'and', 'serve', 'as', 'donors', 'of', 'information', 'for', 'active', 'mat', 'interconversion', 'in', 'strain', 'carrying', 'replacement', 'of', 'kb', 'of', 'the', 'region', 'with', 'the', 'ura', 'gene', 'we', 'discovered', 'that', 'ura', 'silencing', 'and', 'efficiency', 'of', 'mating', 'type', 'switching', 'were', 'covariegated', 'and', 'were', 'regulated', 'by', 'an', 'epigenetic', 'mechanism', 'genetic', 'analyses', 'demonstrated', 'that', 'epigenetic', 'states', 'were', 'remarkably', 'stable', 'not', 'only', 'in', 'mitosis', 'but', 'also', 'in', 'meiosis', 'and', 'were', 'linked', 'to', 'the', 'mating', 'type', 'region', 'this', 'study', 'indicates', 'that', 'different', 'epigenetic', 'states', 'are', 'heritable', 'forms', 'of', 'chromatin', 'organization', 'at', 'the', 'mat', 'region'], ['microdeletions', 'of', 'region', 'termed', 'the', 'imprinting', 'center', 'ic', 'in', 'chromosome', 'have', 'been', 'identified', 'in', 'several', 'families', 'with', 'prader', 'willi', 'syndrome', 'pws', 'or', 'angelman', 'syndrome', 'who', 'show', 'epigenetic', 'inheritance', 'for', 'this', 'region', 'that', 'is', 'consistent', 'with', 'mutation', 'in', 'the', 'imprinting', 'process', 'the', 'ic', 'controls', 'resetting', 'of', 'parental', 'imprints', 'in', 'during', 'gametogenesis', 'we', 'have', 'identified', 'larger', 'series', 'of', 'cases', 'of', 'familial', 'pws', 'including', 'one', 'case', 'with', 'deletion', 'of', 'only', 'kb', 'that', 'narrows', 'the', 'pws', 'critical', 'region', 'to', 'kb', 'spanning', 'the', 'snrpn', 'gene', 'cpg', 'island', 'and', 'exon', 'identification', 'of', 'strong', 'dnase', 'hypersensitive', 'site', 'specific', 'for', 'the', 'paternal', 'allele', 'and', 'six', 'evolutionarily', 'conserved', 'human', 'mouse', 'sequences', 'that', 'are', 'potential', 'transcription', 'factor', 'binding', 'sites', 'is', 'consistent', 'with', 'this', 'region', 'defining', 'the', 'snrpn', 'gene', 'promoter', 'these', 'findings', 'suggest', 'that', 'promoter', 'elements', 'at', 'snrpn', 'play', 'key', 'role', 'in', 'the', 'initiation', 'of', 'imprint', 'switching', 'during', 'spermatogenesis', 'we', 'also', 'identified', 'three', 'patients', 'with', 'sporadic', 'pws', 'who', 'have', 'an', 'imprinting', 'mutation', 'im', 'and', 'no', 'detectable', 'mutation', 'in', 'the', 'ic', 'an', 'inherited', 'mutation', 'or', 'trans', 'factor', 'gene', 'mutation', 'are', 'unlikely', 'thus', 'the', 'disease', 'in', 'these', 'patients', 'may', 'arise', 'from', 'developmental', 'or', 'stochastic', 'failure', 'to', 'switch', 'the', 'maternal', 'to', 'paternal', 'imprint', 'during', 'parental', 'spermatogenesis', 'these', 'studies', 'allow', 'better', 'understanding', 'of', 'novel', 'mechanism', 'of', 'human', 'disease', 'since', 'the', 'epigenetic', 'effect', 'of', 'an', 'im', 'in', 'the', 'parental', 'germ', 'line', 'determines', 'the', 'phenotypic', 'effect', 'in', 'the', 'patient'], ['the', 'wassilewskija', 'strain', 'of', 'arabidopsis', 'has', 'four', 'genes', 'encoding', 'the', 'tryptophan', 'enzyme', 'isomerase', 'pai', 'located', 'at', 'three', 'unlinked', 'sites', 'these', 'four', 'pai', 'genes', 'are', 'methylated', 'over', 'their', 'regions', 'of', 'dna', 'homology', 'when', 'pai', 'copy', 'number', 'is', 'reduced', 'by', 'deletion', 'of', 'two', 'tandemly', 'arrayed', 'genes', 'mepai', 'pai', 'mutant', 'with', 'fluorescent', 'tryptophan', 'deficient', 'phenotypes', 'results', 'because', 'the', 'two', 'remaining', 'methylated', 'pai', 'genes', 'mepai', 'and', 'mepai', 'supply', 'insufficient', 'pai', 'activity', 'these', 'two', 'methylated', 'genes', 'can', 'be', 'inherited', 'through', 'meiosis', 'even', 'when', 'they', 'are', 'segregated', 'away', 'from', 'each', 'other', 'in', 'crosses', 'to', 'strain', 'with', 'unmethylated', 'pai', 'genes', 'however', 'the', 'mutant', 'phenotypes', 'conferred', 'by', 'the', 'methylated', 'pai', 'genes', 'are', 'unstable', 'and', 'mutant', 'plants', 'yield', 'occasional', 'revertant', 'somatic', 'sectors', 'and', 'progeny', 'revertant', 'lines', 'display', 'coordinately', 'reduced', 'methylation', 'of', 'both', 'pai', 'and', 'pai', 'implying', 'that', 'this', 'hypomethylation', 'acts', 'in', 'concerted', 'manner', 'across', 'the', 'genome', 'rather', 'than', 'at', 'individual', 'sites'], ['we', 'report', 'that', 'cpg', 'island', 'methylation', 'an', 'epigenetic', 'modification', 'of', 'dna', 'known', 'to', 'correlate', 'closely', 'with', 'silencing', 'of', 'gene', 'transcription', 'appears', 'in', 'the', 'oestrogen', 'receptor', 'er', 'gene', 'in', 'subpopulation', 'of', 'cells', 'which', 'increases', 'as', 'direct', 'function', 'of', 'age', 'in', 'human', 'colonic', 'mucosa', 'this', 'same', 'methylation', 'change', 'characterizes', 'virtually', 'all', 'cells', 'in', 'art', 'colorectal', 'tumours', 'examined', 'including', 'the', 'earliest', 'stages', 'of', 'tumour', 'formation', 'er', 'gene', 'expression', 'is', 'diminished', 'or', 'absent', 'in', 'colorectal', 'tumours', 'and', 'introduction', 'of', 'an', 'exogenous', 'er', 'gene', 'in', 'cultured', 'colon', 'carcinoma', 'cells', 'resulted', 'in', 'marked', 'growth', 'suppression', 'our', 'data', 'suggest', 'that', 'methylation', 'associated', 'inactivation', 'of', 'the', 'er', 'gene', 'in', 'ageing', 'cotorectal', 'mucosa', 'could', 'be', 'one', 'of', 'the', 'earliest', 'events', 'that', 'predispose', 'to', 'sporadic', 'cotorectal', 'tumorigenesis'], ['very', 'large', 'oxidative', 'damage', 'rate', 'to', 'dna', 'occurs', 'as', 'part', 'of', 'normal', 'metabolism', 'in', 'each', 'rat', 'cell', 'the', 'steady', 'state', 'level', 'is', 'estimated', 'to', 'be', 'about', 'oxidative', 'adducts', 'and', 'about', 'new', 'adducts', 'are', 'formed', 'daily', 'it', 'is', 'argued', 'that', 'this', 'endogenous', 'dna', 'damage', 'is', 'major', 'contributor', 'to', 'aging', 'and', 'the', 'degenerative', 'diseases', 'of', 'aging', 'such', 'as', 'cancer', 'the', 'oxidative', 'damage', 'rate', 'in', 'mammalian', 'species', 'with', 'high', 'metabolic', 'rate', 'short', 'life', 'span', 'and', 'high', 'age', 'specific', 'cancer', 'rate', 'is', 'much', 'higher', 'than', 'the', 'rate', 'in', 'humans', 'long', 'lived', 'creature', 'with', 'lower', 'metabolic', 'rate', 'and', 'lower', 'age', 'specific', 'cancer', 'rate', 'it', 'is', 'argued', 'that', 'deficiency', 'of', 'micronutrients', 'such', 'as', 'dietary', 'antioxidants', 'or', 'folate', 'is', 'major', 'contributor', 'to', 'human', 'cancer', 'and', 'degenerative', 'diseases', 'understanding', 'the', 'role', 'of', 'mitogenesis', 'in', 'mutagenesis', 'is', 'critical', 'for', 'clarifying', 'the', 'mechanisms', 'of', 'carcinogenesis', 'and', 'interpreting', 'high', 'dose', 'animal', 'cancer', 'tests', 'high', 'dose', 'animal', 'cancer', 'tests', 'have', 'been', 'done', 'mainly', 'on', 'synthetic', 'industrial', 'chemicals', 'yet', 'almost', 'all', 'of', 'the', 'chemicals', 'humans', 'are', 'exposed', 'to', 'are', 'natural', 'about', 'half', 'of', 'natural', 'chemicals', 'tested', 'in', 'high', 'dose', 'animal', 'cancer', 'tests', 'are', 'rodent', 'carcinogens', 'finding', 'that', 'is', 'consistent', 'with', 'the', 'view', 'that', 'high', 'dose', 'tests', 'frequently', 'increase', 'mitogenesis', 'rates', 'animals', 'have', 'numerous', 'defenses', 'against', 'toxins', 'that', 'make', 'them', 'very', 'well', 'buffered', 'against', 'low', 'doses', 'of', 'almost', 'all', 'toxins', 'whether', 'synthetic', 'or', 'natural'], ['the', 'expression', 'of', 'the', 'gene', 'is', 'governed', 'by', 'parental', 'imprinting', 'in', 'mammals', 'an', 'unusual', 'gene', 'encoding', 'an', 'rna', 'with', 'no', 'known', 'function', 'is', 'exclusively', 'expressed', 'from', 'the', 'maternal', 'chromosome', 'in', 'mouse', 'it', 'lies', 'kb', 'downstream', 'from', 'the', 'igf', 'gene', 'which', 'encodes', 'fetal', 'specific', 'growth', 'factor', 'insulin', 'like', 'growth', 'factor', 'ii', 'and', 'is', 'expressed', 'primarily', 'from', 'the', 'paternally', 'inherited', 'chromosome', 'in', 'this', 'report', 'we', 'have', 'utilized', 'interspecific', 'hybrid', 'mice', 'to', 'identify', 'male', 'specific', 'dna', 'methylation', 'of', 'to', 'kb', 'domain', 'surrounding', 'the', 'gene', 'and', 'its', 'promoter', 'this', 'allele', 'specific', 'methylation', 'could', 'function', 'as', 'mark', 'to', 'suppress', 'transcription', 'of', 'the', 'paternal', 'allele', 'consistent', 'with', 'this', 'proposal', 'the', 'promoter', 'displayed', 'an', 'open', 'chromatin', 'conformation', 'only', 'on', 'the', 'relatively', 'unmethylated', 'active', 'maternal', 'allele', 'in', 'contrast', 'cell', 'type', 'specific', 'enhancer', 'that', 'lies', 'outside', 'the', 'methylation', 'domain', 'is', 'hypersensitive', 'to', 'restriction', 'enzyme', 'digestion', 'in', 'nuclei', 'on', 'both', 'maternal', 'and', 'paternal', 'chromosomes', 'that', 'the', 'allele', 'specific', 'methylation', 'domain', 'coupled', 'to', 'the', 'two', 'enhancers', 'contains', 'all', 'the', 'information', 'necessary', 'for', 'its', 'imprinting', 'was', 'tested', 'by', 'examining', 'two', 'transgenic', 'lines', 'containing', 'an', 'internally', 'deleted', 'transgene', 'both', 'displayed', 'paternal', 'specific', 'methylation', 'of', 'the', 'transgene', 'and', 'maternal', 'specific', 'expression', 'although', 'neither', 'line', 'has', 'been', 'tested', 'in', 'an', 'inbred', 'genetic', 'background', 'and', 'therefore', 'the', 'action', 'of', 'complex', 'modifiers', 'cannot', 'be', 'formally', 'excluded', 'the', 'result', 'suggests', 'that', 'the', 'sequences', 'necessary', 'for', 'the', 'imprinting', 'of', 'have', 'been', 'identified'], ['the', 'insulin', 'like', 'growth', 'factor', 'ii', 'igf', 'and', 'genes', 'are', 'imprinted', 'in', 'mouse', 'and', 'human', 'with', 'expression', 'of', 'the', 'paternal', 'igf', 'and', 'maternal', 'alleles', 'igf', 'undergoes', 'loss', 'of', 'imprinting', 'loi', 'in', 'most', 'wilms', 'tumours', 'wt', 'we', 'now', 'show', 'that', 'loi', 'of', 'igf', 'is', 'associated', 'with', 'fold', 'down', 'regulation', 'of', 'expression', 'ii', 'these', 'changes', 'are', 'associated', 'with', 'alterations', 'in', 'parental', 'origin', 'specific', 'tissue', 'independent', 'sites', 'of', 'dna', 'methylation', 'in', 'the', 'promoter', 'and', 'iii', 'loss', 'of', 'heterozygosity', 'is', 'also', 'associated', 'with', 'loss', 'of', 'expression', 'thus', 'imprinting', 'of', 'large', 'domain', 'of', 'the', 'maternal', 'chromosome', 'results', 'in', 'reversal', 'to', 'paternal', 'epigenotype', 'these', 'data', 'also', 'suggest', 'an', 'epigenetic', 'mechanism', 'for', 'inactivation', 'of', 'as', 'tumour', 'suppressor', 'gene'], ['the', 'recent', 'discovery', 'of', 'widely', 'distributed', 'targeting', 'determinants', 'which', 'govern', 'the', 'polarized', 'cell', 'surface', 'distribution', 'of', 'plasma', 'membrane', 'proteins', 'in', 'epithelial', 'cells', 'has', 'significantly', 'changed', 'our', 'view', 'of', 'how', 'polarized', 'cells', 'generate', 'functionally', 'distinct', 'membrane', 'domains', 'together', 'with', 'the', 'surprising', 'finding', 'that', 'the', 'same', 'determinants', 'are', 'recognized', 'on', 'both', 'the', 'biosynthetic', 'and', 'the', 'endocytic', 'pathways', 'it', 'now', 'appears', 'likely', 'that', 'common', 'epigenetic', 'code', 'may', 'exist', 'that', 'controls', 'molecular', 'sorting', 'of', 'membrane', 'proteins', 'in', 'wide', 'variety', 'of', 'polarized', 'and', 'perhaps', 'even', 'non', 'polarized', 'cell', 'types'], ['inactivation', 'of', 'the', 'retinoblastoma', 'gene', 'appears', 'to', 'have', 'fundamental', 'role', 'in', 'the', 'genesis', 'of', 'retinoblastoma', 'osteosarcoma', 'and', 'other', 'malignant', 'tumors', 'the', 'gene', 'is', 'generally', 'inactivated', 'because', 'of', 'loss', 'of', 'function', 'mutations', 'although', 'epigenetic', 'phenomena', 'such', 'as', 'of', 'the', 'promoter', 'region', 'could', 'possibly', 'have', 'the', 'same', 'effect', 'we', 'investigated', 'the', 'methylation', 'pattern', 'at', 'the', 'end', 'of', 'the', 'retinoblastoma', 'gene', 'including', 'its', 'promoter', 'region', 'and', 'exon', 'in', 'dna', 'purified', 'from', 'primary', 'retinoblastomas', 'we', 'found', 'five', 'tumors', 'with', 'evidence', 'for', 'all', 'from', 'unilateral', 'simplex', 'patients', 'no', 'methylation', 'abnormalities', 'were', 'detected', 'in', 'dna', 'purified', 'from', 'the', 'leukocytes', 'from', 'these', 'patients', 'it', 'is', 'interesting', 'that', 'in', 'one', 'of', 'these', 'tumors', 'the', 'was', 'confined', 'to', 'one', 'allele', 'there', 'were', 'no', 'mutations', 'in', 'bp', 'sequence', 'including', 'the', 'hypermethylated', 'region', 'that', 'might', 'account', 'for', 'the', 'allele', 'specific', 'we', 'believe', 'that', 'the', 'of', 'the', 'retinoblastoma', 'gene', 'that', 'we', 'found', 'in', 'these', 'tumors', 'corresponds', 'to', 'the', 'allelic', 'inactivation', 'of', 'the', 'gene', 'and', 'we', 'speculate', 'that', 'erroneous', 'without', 'alteration', 'of', 'nucleotide', 'sequence', 'occasionally', 'plays', 'role', 'in', 'the', 'genesis', 'of', 'this', 'cancer', 'if', 'this', 'is', 'true', 'then', 'retinoblastomas', 'with', 'might', 'be', 'treatable', 'with', 'agents', 'that', 'interfere', 'with', 'methylation', 'of', 'dna'], ['becker', 'muscular', 'dystrophy', 'bmd', 'often', 'results', 'from', 'in', 'frame', 'mutations', 'of', 'the', 'dystrophin', 'gene', 'that', 'allow', 'production', 'of', 'an', 'altered', 'but', 'partially', 'functional', 'protein', 'to', 'address', 'potential', 'structure', 'function', 'relationships', 'for', 'the', 'various', 'domains', 'of', 'dystrophin', 'we', 'examined', 'both', 'the', 'dystrophin', 'gene', 'and', 'protein', 'in', 'patients', 'with', 'abnormal', 'dystrophin', 'eighty', 'six', 'percent', 'of', 'bmd', 'patients', 'with', 'dystrophin', 'of', 'altered', 'size', 'have', 'deletions', 'or', 'duplications', 'and', 'the', 'observed', 'sizes', 'of', 'dystrophin', 'fit', 'well', 'with', 'predictions', 'based', 'on', 'dna', 'data', 'deletions', 'within', 'the', 'amino', 'terminal', 'domain', 'tended', 'to', 'result', 'in', 'low', 'levels', 'of', 'dystrophin', 'and', 'more', 'severe', 'phenotype', 'the', 'phenotypes', 'of', 'patients', 'with', 'deletions', 'or', 'duplications', 'in', 'the', 'central', 'rod', 'domain', 'were', 'more', 'variable', 'this', 'region', 'can', 'be', 'divided', 'into', 'three', 'portions', 'based', 'on', 'differences', 'in', 'clinical', 'presentations', 'of', 'patients', 'deletions', 'around', 'exons', 'were', 'most', 'common', 'and', 'generally', 'caused', 'typical', 'bmd', 'however', 'phenotypic', 'variability', 'among', 'patients', 'with', 'similar', 'mutations', 'suggests', 'that', 'epigenetic', 'and', 'or', 'environmental', 'factors', 'play', 'an', 'important', 'role', 'in', 'determining', 'the', 'clinical', 'progression', 'in', 'contrast', 'deletions', 'or', 'duplications', 'in', 'the', 'proximal', 'portion', 'of', 'this', 'domain', 'tended', 'to', 'cause', 'severe', 'cramps', 'and', 'myalgia', 'finally', 'loss', 'of', 'the', 'middle', 'of', 'this', 'region', 'probably', 'causes', 'very', 'mild', 'phenotype', 'as', 'only', 'one', 'such', 'patient', 'was', 'found', 'and', 'his', 'only', 'symptom', 'was', 'elevated', 'serum', 'creatine', 'phosphokinase', 'levels'], ['massive', 'cell', 'migration', 'proliferation', 'phenotypic', 'differentiation', 'and', 'enhanced', 'biosynthetic', 'activities', 'characterize', 'the', 'sites', 'of', 'wound', 'healing', 'and', 'fibrosis', 'regulation', 'of', 'cellular', 'functions', 'by', 'extracellular', 'matrix', 'which', 'consists', 'of', 'dynamic', 'assemblage', 'of', 'variety', 'of', 'interacting', 'molecules', 'capable', 'of', 'reorganization', 'in', 'response', 'to', 'endogenous', 'and', 'exogenous', 'stimuli', 'represents', 'fundamental', 'epigenetic', 'mechanism', 'regulating', 'cellular', 'behavior', 'and', 'phenotype', 'interactions', 'of', 'the', 'individual', 'components', 'of', 'extracellular', 'matrix', 'with', 'specific', 'cell', 'surface', 'molecules', 'integrin', 'receptors', 'and', 'proteoglycans', 'initiate', 'cascade', 'of', 'signal', 'transduction', 'leading', 'to', 'varied', 'short', 'term', 'or', 'persistent', 'cellular', 'responses', 'extracellular', 'matrix', 'also', 'serves', 'as', 'an', 'important', 'reservoir', 'of', 'cytokines', 'and', 'growth', 'factors', 'thus', 'modulating', 'the', 'action', 'of', 'host', 'of', 'potent', 'biological', 'response', 'modifiers', 'by', 'their', 'selective', 'local', 'accumulation', 'and', 'release', 'currently', 'known', 'mechanisms', 'by', 'which', 'extracellular', 'matrix', 'modulates', 'different', 'facets', 'of', 'the', 'process', 'of', 'tissue', 'remodeling', 'after', 'injury', 'which', 'culminate', 'either', 'in', 'normal', 'wound', 'repair', 'or', 'fibrosis', 'are', 'discussed'], ['to', 'test', 'the', 'potential', 'role', 'of', 'as', 'tumour', 'suppressor', 'gene', 'we', 'have', 'examined', 'its', 'expression', 'and', 'dna', 'methylation', 'in', 'wilms', 'tumours', 'wts', 'in', 'most', 'wts', 'rna', 'was', 'reduced', 'at', 'least', 'fold', 'from', 'fetal', 'kidney', 'levels', 'of', 'the', 'expression', 'negative', 'tumours', 'ten', 'retained', 'heterozygosity', 'in', 'nine', 'of', 'these', 'dna', 'was', 'biallelically', 'hypermethylated', 'and', 'in', 'two', 'cases', 'locally', 'restricted', 'to', 'sequences', 'was', 'also', 'present', 'in', 'the', 'non', 'neoplastic', 'kidney', 'parenchyma', 'igf', 'mrna', 'was', 'expressed', 'in', 'most', 'but', 'not', 'all', 'wts', 'and', 'expression', 'patterns', 'were', 'consistent', 'with', 'igf', 'enhancer', 'competition', 'without', 'obligate', 'inverse', 'coupling', 'these', 'observations', 'implicate', 'genetic', 'and', 'epigenetic', 'inactivation', 'of', 'in', 'wilms', 'tumorigenesis'], ['segmentation', 'genes', 'provide', 'the', 'signals', 'for', 'the', 'activation', 'and', 'regulation', 'of', 'homeotic', 'genes', 'in', 'drosophila', 'but', 'cannot', 'maintain', 'the', 'resulting', 'pattern', 'of', 'expression', 'because', 'their', 'activity', 'ceases', 'halfway', 'through', 'embryogenesis', 'maintenance', 'of', 'the', 'pattern', 'is', 'due', 'to', 'the', 'polycomb', 'group', 'of', 'genes', 'pc', 'and', 'the', 'trithorax', 'group', 'of', 'genes', 'trx', 'responsible', 'for', 'the', 'persistence', 'of', 'the', 'active', 'or', 'repressed', 'state', 'of', 'homeotic', 'genes', 'we', 'have', 'identified', 'regulatory', 'element', 'in', 'the', 'ubx', 'gene', 'that', 'responds', 'to', 'pc', 'and', 'trx', 'genes', 'transposons', 'carrying', 'this', 'element', 'create', 'new', 'binding', 'sites', 'for', 'pc', 'products', 'in', 'the', 'polytene', 'chromosomes', 'this', 'pc', 'maintenance', 'element', 'pre', 'establishes', 'repressive', 'complex', 'that', 'keeps', 'enhancers', 'repressed', 'in', 'cells', 'in', 'which', 'they', 'were', 'originally', 'repressed', 'and', 'maintains', 'this', 'state', 'through', 'many', 'cell', 'divisions', 'the', 'trx', 'products', 'stimulate', 'the', 'expression', 'of', 'enhancers', 'in', 'cells', 'in', 'which', 'they', 'were', 'originally', 'active', 'this', 'mechanism', 'is', 'responsible', 'for', 'the', 'correct', 'regulation', 'of', 'imaginal', 'disc', 'enhancers', 'which', 'lack', 'themselves', 'antero', 'posterior', 'positional', 'information', 'the', 'pre', 'also', 'causes', 'severe', 'variegation', 'of', 'the', 'mini', 'white', 'gene', 'present', 'in', 'the', 'transposon', 'phenomenon', 'very', 'similar', 'to', 'heterochromatic', 'position', 'effect', 'variegation', 'the', 'significance', 'of', 'this', 'mechanism', 'for', 'homeotic', 'gene', 'regulation', 'is', 'discussed'], ['chromatin', 'structure', 'at', 'pombe', 'centromeres', 'is', 'unusual', 'the', 'insertion', 'of', 'the', 'ura', 'gene', 'within', 'these', 'centromeres', 'resulted', 'in', 'genetically', 'identical', 'cells', 'mosaic', 'for', 'its', 'expression', 'placement', 'of', 'the', 'ade', 'gene', 'within', 'cen', 'or', 'cen', 'resulted', 'in', 'red', 'white', 'sectored', 'colonies', 'demonstrating', 'the', 'instability', 'of', 'gene', 'expression', 'the', 'occurrence', 'of', 'pink', 'colonies', 'implied', 'that', 'intermediate', 'levels', 'of', 'repression', 'were', 'established', 'repression', 'of', 'both', 'genes', 'within', 'centromeres', 'was', 'temperature', 'sensitive', 'the', 'chromatin', 'structure', 'of', 'the', 'ura', 'gene', 'at', 'centromeres', 'was', 'altered', 'suggesting', 'that', 'the', 'unusual', 'chromatin', 'encroaches', 'into', 'the', 'gene', 'and', 'inhibits', 'normal', 'expression', 'these', 'repressive', 'effects', 'at', 'pombe', 'centromeres', 'resemble', 'the', 'classical', 'phenomenon', 'of', 'position', 'effect', 'variegation', 'imposed', 'by', 'drosophila', 'heterochromatin', 'on', 'nearby', 'genes', 'however', 'since', 'the', 'epigenetic', 'states', 'can', 'be', 'set', 'at', 'intermediate', 'levels', 'of', 'expression', 'purely', 'euchromatin', 'heterochromatin', 'dichotomy', 'does', 'not', 'apply', 'model', 'for', 'the', 'epigenetic', 'regulation', 'of', 'genes', 'placed', 'within', 'pombe', 'centromeres', 'is', 'presented'], ['the', 'mouse', 'insulin', 'like', 'growth', 'factor', 'ii', 'igf', 'gene', 'which', 'is', 'located', 'on', 'distal', 'chromosome', 'chr', 'has', 'been', 'shown', 'previously', 'to', 'undergo', 'tissue', 'specific', 'parental', 'imprinting', 'this', 'imprinting', 'results', 'in', 'expression', 'of', 'igf', 'from', 'the', 'paternally', 'inherited', 'chromosome', 'and', 'repression', 'of', 'the', 'maternally', 'inherited', 'allele', 'in', 'most', 'tissues', 'of', 'the', 'developing', 'embryo', 'we', 'are', 'using', 'embryos', 'with', 'the', 'maternal', 'duplication', 'and', 'paternal', 'deficiency', 'of', 'distal', 'chr', 'to', 'characterize', 'the', 'mechanism', 'that', 'underlies', 'repression', 'of', 'the', 'maternal', 'allele', 'we', 'show', 'that', 'the', 'chromatin', 'of', 'the', 'region', 'of', 'the', 'repressed', 'igf', 'allele', 'is', 'potentially', 'active', 'for', 'transcription', 'rather', 'than', 'heterochromatic', 'in', 'particular', 'cpg', 'island', 'that', 'comprises', 'one', 'of', 'the', 'two', 'strong', 'promoters', 'is', 'unmethylated', 'at', 'both', 'parental', 'alleles', 'and', 'dnase', 'hypersensitive', 'sites', 'in', 'and', 'around', 'the', 'strong', 'promoters', 'are', 'consistently', 'present', 'on', 'both', 'parental', 'chromosomes', 'in', 'agreement', 'with', 'the', 'chromatin', 'state', 'primary', 'transcripts', 'from', 'the', 'maternal', 'igf', 'allele', 'have', 'been', 'detected', 'at', 'low', 'but', 'significant', 'levels', 'these', 'findings', 'differ', 'from', 'observations', 'in', 'other', 'instances', 'of', 'imprinting', 'namely', 'chromosome', 'inactivation', 'and', 'transgene', 'imprinting', 'in', 'mice', 'although', 'no', 'parent', 'specific', 'differences', 'were', 'detected', 'in', 'either', 'dna', 'methylation', 'or', 'sensitivity', 'to', 'nucleases', 'at', 'these', 'promoters', 'we', 'have', 'observed', 'parental', 'methylation', 'differences', 'in', 'region', 'several', 'kilobases', 'upstream', 'of', 'the', 'first', 'exon', 'the', 'differential', 'activity', 'of', 'the', 'parental', 'igf', 'alleles', 'could', 'be', 'achieved', 'through', 'epigenetic', 'modifications', 'situated', 'outside', 'the', 'promoters', 'or', 'by', 'subtle', 'and', 'yet', 'unidentified', 'modifications', 'at', 'the', 'promoters'], ['the', 'insulin', 'like', 'growth', 'factor', 'igf', 'and', 'genes', 'are', 'reciprocally', 'imprinted', 'and', 'closely', 'linked', 'igf', 'encodes', 'fetal', 'growth', 'factor', 'and', 'is', 'predominantly', 'expressed', 'from', 'the', 'paternal', 'allele', 'while', 'is', 'expressed', 'from', 'the', 'maternal', 'allele', 'and', 'encodes', 'transcript', 'which', 'may', 'downregulate', 'cellular', 'proliferation', 'one', 'of', 'the', 'epigenetic', 'modifications', 'thought', 'to', 'be', 'involved', 'in', 'parental', 'imprinting', 'is', 'dna', 'methylation', 'here', 'we', 'analyse', 'methylation', 'in', 'two', 'regions', 'of', 'the', 'igf', 'gene', 'one', 'approx', 'kb', 'upstream', 'of', 'the', 'gene', 'and', 'one', 'in', 'the', 'part', 'of', 'the', 'gene', 'both', 'regions', 'are', 'more', 'methylated', 'on', 'the', 'expressed', 'paternal', 'chromosome', 'genomic', 'sequencing', 'of', 'individual', 'chromosomes', 'in', 'the', 'first', 'region', 'shows', 'this', 'parent', 'specific', 'methylation', 'to', 'be', 'highly', 'mosaic', 'interestingly', 'individual', 'sperm', 'chromosomes', 'carry', 'different', 'methylation', 'patterns', 'into', 'the', 'egg', 'in', 'the', 'more', 'region', 'which', 'is', 'fully', 'methylated', 'in', 'sperm', 'the', 'level', 'of', 'methylation', 'on', 'the', 'paternal', 'allele', 'is', 'highly', 'tissue', 'specific', 'and', 'is', 'correlated', 'with', 'expression', 'of', 'the', 'gene', 'in', 'fetal', 'tissues', 'hence', 'the', 'paternal', 'allele', 'is', 'highly', 'methylated', 'in', 'fetal', 'liver', 'high', 'expression', 'but', 'is', 'undermethylated', 'in', 'fetal', 'brain', 'virtually', 'no', 'expression', 'adult', 'choroid', 'plexus', 'brain', 'tissue', 'in', 'which', 'igf', 'is', 'expressed', 'from', 'both', 'alleles', 'and', 'is', 'not', 'expressed', 'represents', 'an', 'apparent', 'loss', 'of', 'imprinting', 'here', 'both', 'igf', 'and', 'adopt', 'paternal', 'type', 'methylation', 'pattern', 'on', 'both', 'parental', 'chromosomes', 'analysis', 'of', 'early', 'passage', 'androgenetic', 'and', 'parthenogenetic', 'embryonic', 'stem', 'es', 'cells', 'shows', 'that', 'the', 'methylation', 'patterns', 'of', 'igf', 'and', 'on', 'maternal', 'and', 'paternal', 'chromosomes', 'are', 'very', 'similar', 'androgenetic', 'and', 'parthenogenetic', 'teratomas', 'derived', 'from', 'these', 'es', 'cells', 'show', 'the', 'appropriate', 'paternal', 'and', 'maternal', 'patterns', 'respectively', 'of', 'allelic', 'methylation', 'in', 'both', 'genes', 'our', 'results', 'suggest', 'that', 'allelic', 'methylation', 'patterns', 'in', 'igf', 'and', 'arise', 'early', 'in', 'embryogenesis', 'and', 'change', 'progressively', 'during', 'development', 'some', 'of', 'these', 'developmental', 'changes', 'are', 'apparently', 'under', 'tissue', 'specific', 'control'], ['in', 'several', 'plant', 'systems', 'expression', 'of', 'structurally', 'intact', 'genes', 'may', 'be', 'silenced', 'epigenetically', 'when', 'transgenic', 'construct', 'increases', 'the', 'copy', 'number', 'of', 'dna', 'sequences', 'here', 'we', 'report', 'epigenetic', 'silencing', 'in', 'arabidopsis', 'lines', 'containing', 'transgenic', 'inserts', 'of', 'defined', 'genetic', 'structure', 'all', 'at', 'the', 'same', 'genomic', 'locus', 'these', 'comprise', 'an', 'allelic', 'series', 'that', 'includes', 'single', 'copy', 'of', 'the', 'primary', 'insert', 'which', 'carries', 'repeated', 'drug', 'resistance', 'transgenes', 'and', 'set', 'of', 'its', 'derivatives', 'which', 'as', 'result', 'of', 'recombination', 'within', 'the', 'insert', 'carry', 'different', 'numbers', 'and', 'alleles', 'of', 'resistance', 'genes', 'although', 'the', 'drug', 'resistance', 'genes', 'remained', 'intact', 'both', 'the', 'primary', 'and', 'some', 'recombinant', 'lines', 'nevertheless', 'segregated', 'many', 'progeny', 'that', 'were', 'partly', 'or', 'fully', 'drug', 'sensitive', 'because', 'of', 'silencing', 'as', 'in', 'other', 'systems', 'silencing', 'was', 'reversible', 'and', 'correlated', 'with', 'decreased', 'steady', 'state', 'mrna', 'and', 'increased', 'dna', 'methylation', 'each', 'different', 'number', 'and', 'combination', 'of', 'genes', 'on', 'the', 'same', 'or', 'different', 'homologous', 'chromosomes', 'conditioned', 'its', 'own', 'idiosyncratic', 'segregation', 'pattern', 'strikingly', 'lines', 'with', 'single', 'gene', 'segregated', 'only', 'few', 'slightly', 'drug', 'sensitive', 'progeny', 'whereas', 'multi', 'gene', 'lines', 'segregated', 'many', 'highly', 'sensitive', 'progeny', 'indicating', 'dependence', 'of', 'silencing', 'at', 'this', 'locus', 'on', 'repeated', 'sequences', 'this', 'argues', 'strongly', 'against', 'explanations', 'based', 'on', 'antisense', 'rna', 'but', 'is', 'consistent', 'with', 'explanations', 'based', 'on', 'ectopic', 'dna', 'pairing', 'one', 'possibility', 'is', 'that', 'silencing', 'reflects', 'the', 'interaction', 'of', 'paired', 'homologous', 'dna', 'with', 'flanking', 'heterologous', 'dna', 'which', 'induces', 'condensation', 'of', 'chromatin', 'into', 'non', 'transcribable', 'state'], ['modifiers', 'of', 'position', 'effect', 'variegation', 'in', 'drosophila', 'encode', 'proteins', 'that', 'are', 'thought', 'to', 'modify', 'chromatin', 'rendering', 'it', 'heritably', 'changed', 'in', 'its', 'expressibility', 'in', 'an', 'attempt', 'to', 'identify', 'similar', 'modifier', 'genes', 'in', 'other', 'species', 'we', 'have', 'utilized', 'known', 'sequence', 'homology', 'termed', 'chromo', 'box', 'between', 'suppressor', 'of', 'position', 'effect', 'variegation', 'heterochromatin', 'protein', 'hp', 'and', 'repressor', 'of', 'homeotic', 'genes', 'polycomb', 'pc', 'pcr', 'generated', 'probe', 'encompassing', 'the', 'hp', 'chromo', 'box', 'was', 'used', 'to', 'clone', 'full', 'length', 'murine', 'cdnas', 'that', 'contain', 'conserved', 'chromo', 'box', 'motifs', 'sequence', 'comparisons', 'in', 'situ', 'hybridization', 'experiments', 'and', 'rna', 'northern', 'blot', 'analysis', 'suggest', 'that', 'the', 'murine', 'and', 'human', 'sequences', 'presented', 'in', 'this', 'report', 'are', 'homologues', 'of', 'the', 'drosophila', 'hp', 'gene', 'chromo', 'box', 'sequences', 'can', 'also', 'be', 'detected', 'in', 'other', 'animal', 'species', 'and', 'in', 'plants', 'predicting', 'strongly', 'conserved', 'structural', 'role', 'for', 'the', 'peptide', 'encoded', 'by', 'this', 'sequence', 'we', 'propose', 'that', 'epigenetic', 'yet', 'heritable', 'changes', 'in', 'gene', 'expressibility', 'characteristic', 'of', 'chromosomal', 'imprinting', 'phenomena', 'can', 'largely', 'be', 'explained', 'by', 'the', 'action', 'of', 'such', 'modifier', 'genes', 'the', 'evolutionary', 'conservation', 'of', 'the', 'chromo', 'box', 'motif', 'now', 'enables', 'the', 'isolation', 'and', 'study', 'of', 'putative', 'modifier', 'genes', 'in', 'those', 'animal', 'and', 'plant', 'species', 'where', 'chromosomal', 'imprinting', 'has', 'been', 'described'], [], [], ['the', 'molecular', 'mechanisms', 'introducing', 'epigenetic', 'modifications', 'that', 'lead', 'to', 'differential', 'silencing', 'of', 'some', 'autosomal', 'alleles', 'depending', 'on', 'their', 'parental', 'legacy', 'are', 'still', 'largely', 'unknown', 'but', 'recent', 'results', 'from', 'studies', 'of', 'endogenously', 'imprinted', 'genes', 'and', 'particular', 'transgenes', 'make', 'dna', 'methylation', 'strong', 'candidate', 'at', 'the', 'same', 'time', 'these', 'results', 'have', 'raised', 'new', 'questions', 'about', 'the', 'details', 'of', 'the', 'imprinting', 'process'], ['previous', 'work', 'has', 'shown', 'that', 'two', 'homologous', 'unlinked', 'transgene', 'loci', 'can', 'interact', 'in', 'plant', 'nuclei', 'leading', 'to', 'non', 'reciprocal', 'trans', 'inactivation', 'and', 'methylation', 'of', 'genes', 'at', 'one', 'locus', 'here', 'we', 'report', 'the', 'structure', 'and', 'methylation', 'of', 'different', 'transgene', 'loci', 'that', 'contain', 'the', 'same', 'construct', 'but', 'are', 'variably', 'able', 'to', 'inactivate', 'and', 'methylate', 'partially', 'homologous', 'unlinked', 'target', 'locus', 'silencing', 'loci', 'comprised', 'multiple', 'methylated', 'copies', 'of', 'the', 'transgene', 'construct', 'whereas', 'non', 'silencing', 'locus', 'contained', 'single', 'unmethylated', 'copy', 'the', 'correspondence', 'between', 'strength', 'of', 'silencing', 'activity', 'and', 'copy', 'number', 'degree', 'of', 'methylation', 'was', 'further', 'demonstrated', 'by', 'producing', 'novel', 'alleles', 'of', 'strong', 'silencing', 'locus', 'reducing', 'the', 'transgene', 'copy', 'number', 'and', 'methylation', 'within', 'this', 'silencing', 'locus', 'decreased', 'its', 'ability', 'to', 'inactivate', 'the', 'target', 'locus', 'the', 'strong', 'silencing', 'locus', 'which', 'was', 'located', 'close', 'to', 'telomere', 'trans', 'inactivated', 'various', 'structural', 'variants', 'of', 'the', 'original', 'target', 'construct', 'regardless', 'of', 'their', 'location', 'in', 'the', 'genome', 'this', 'suggests', 'that', 'the', 'silencing', 'locus', 'can', 'scan', 'the', 'entire', 'genome', 'for', 'homologous', 'regions', 'process', 'possibly', 'aided', 'by', 'its', 'telomeric', 'location', 'our', 'data', 'support', 'the', 'idea', 'that', 'epistatic', 'trans', 'inactivation', 'of', 'unlinked', 'homologous', 'transgenes', 'in', 'plants', 'results', 'from', 'pre', 'existing', 'epigenetic', 'difference', 'between', 'transgene', 'loci', 'which', 'is', 'subsequently', 'equalized', 'by', 'epigene', 'conversion', 'involving', 'dna', 'dna', 'pairing'], [], ['major', 'epigenetic', 'modifications', 'apparently', 'occur', 'during', 'early', 'development', 'in', 'the', 'mouse', 'the', 'factors', 'that', 'induce', 'such', 'modifications', 'are', 'complex', 'and', 'may', 'involve', 'the', 'various', 'components', 'of', 'zygote', 'we', 'have', 'started', 'to', 'explore', 'whether', 'changes', 'in', 'the', 'composition', 'brought', 'about', 'by', 'can', 'induce', 'phenotypic', 'effects', 'through', 'epigenetic', 'modifications', 'hybrids', 'were', 'therefore', 'prepared', 'by', 'transplanting', 'female', 'pronucleus', 'into', 'recipient', 'egg', 'from', 'different', 'genotype', 'as', 'result', 'the', 'maternal', 'genome', 'was', 'of', 'different', 'genetic', 'background', 'as', 'compared', 'with', 'the', 'egg', 'cytoplasm', 'specifically', 'experimental', 'zygotes', 'had', 'cytoplasm', 'from', 'the', 'inbred', 'strain', 'bl', 'maternal', 'genome', 'from', 'dba', 'and', 'paternal', 'genome', 'from', 'bl', 'termed', 'bdb', 'hybrids', 'the', 'mirror', 'image', 'combination', 'termed', 'dbd', 'was', 'also', 'made', 'the', 'reconstituted', 'zygotes', 'were', 'transferred', 'to', 'recipients', 'and', 'allowed', 'to', 'develop', 'to', 'term', 'mice', 'born', 'from', 'manipulated', 'zygotes', 'showed', 'transcriptional', 'repression', 'and', 'dna', 'methylation', 'of', 'major', 'urinary', 'protein', 'genes', 'in', 'their', 'liver', 'as', 'well', 'as', 'growth', 'deficiency', 'resulting', 'in', 'reduced', 'adult', 'body', 'weight', 'no', 'altered', 'phenotype', 'was', 'observed', 'in', 'controls', 'in', 'which', 'the', 'maternal', 'pronucleus', 'was', 'simply', 'transplanted', 'back', 'into', 'another', 'zygote', 'of', 'the', 'same', 'genetic', 'background', 'these', 'results', 'clearly', 'demonstrate', 'phenotypic', 'as', 'well', 'as', 'molecular', 'effects', 'on', 'dna', 'methylation', 'and', 'expression', 'of', 'at', 'least', 'one', 'gene', 'phenotype', 'was', 'therefore', 'no', 'longer', 'predicted', 'by', 'genotype', 'as', 'result', 'of', 'epigenetic', 'modifications', 'in', 'experimental', 'embryos', 'what', 'precisely', 'triggers', 'the', 'phenotypic', 'and', 'epigenetic', 'changes', 'is', 'unknown', 'but', 'presumably', 'interactions', 'in', 'hybrid', 'zygotes', 'may', 'be', 'partly', 'responsible'], ['the', 'dna', 'of', 'eukaryotic', 'cells', 'is', 'organized', 'in', 'complex', 'with', 'proteins', 'either', 'as', 'interphase', 'chromatin', 'or', 'mitotic', 'chromosomes', 'nucleosomes', 'the', 'structural', 'subunits', 'of', 'chromatin', 'have', 'long', 'been', 'considered', 'as', 'static', 'structures', 'incompatible', 'with', 'processes', 'occurring', 'in', 'chromatin', 'during', 'the', 'past', 'few', 'years', 'it', 'has', 'become', 'evident', 'that', 'the', 'histone', 'part', 'of', 'the', 'nucleosome', 'has', 'important', 'regulatory', 'functions', 'some', 'of', 'these', 'functions', 'are', 'mediated', 'by', 'the', 'terminal', 'core', 'histone', 'domains', 'which', 'contain', 'sites', 'for', 'modifications', 'among', 'them', 'lysine', 'residues', 'for', 'reversible', 'acetylation', 'recent', 'results', 'indicate', 'that', 'acetylation', 'and', 'deacetylation', 'of', 'terminal', 'lysines', 'of', 'nucleosomal', 'core', 'histones', 'represent', 'means', 'of', 'molecular', 'communication', 'between', 'chromatin', 'and', 'the', 'cellular', 'signal', 'transduction', 'network', 'resulting', 'in', 'heritable', 'epigenetic', 'information', 'data', 'on', 'enzymes', 'involved', 'in', 'acetylation', 'and', 'the', 'pattern', 'of', 'acetylated', 'lysine', 'sites', 'on', 'chromosomes', 'as', 'well', 'as', 'genetic', 'data', 'on', 'yeast', 'transcriptional', 'repression', 'suggest', 'that', 'acetylation', 'may', 'lead', 'to', 'structural', 'transitions', 'as', 'well', 'as', 'specific', 'signalling', 'within', 'distinct', 'chromatin', 'domains'], [], ['messenger', 'rna', 'and', 'methylation', 'levels', 'of', 'four', 'imprinted', 'genes', 'igf', 'igf', 'and', 'snrpn', 'were', 'examined', 'by', 'northern', 'and', 'southern', 'blotting', 'in', 'mouse', 'parthenogenetic', 'androgenetic', 'and', 'normal', 'or', 'wild', 'type', 'embryonic', 'stem', 'cell', 'lines', 'during', 'their', 'differentiation', 'in', 'vitro', 'as', 'embryoid', 'bodies', 'zn', 'most', 'instances', 'mrna', 'levels', 'in', 'parthenogenetic', 'and', 'androgenetic', 'embryoid', 'bodies', 'differed', 'from', 'wild', 'type', 'as', 'expected', 'from', 'previously', 'determined', 'patterns', 'of', 'monoallelic', 'expression', 'in', 'midgestation', 'embryos', 'and', 'at', 'later', 'stages', 'of', 'development', 'these', 'findings', 'implicate', 'aberrant', 'mrna', 'levels', 'of', 'these', 'genes', 'in', 'the', 'abnormal', 'development', 'of', 'parthenogenetic', 'and', 'androgenetic', 'embryos', 'and', 'chimeras', 'whereas', 'complete', 'silence', 'of', 'one', 'of', 'the', 'parental', 'alleles', 'has', 'previously', 'been', 'observed', 'in', 'vivo', 'we', 'detected', 'some', 'mrna', 'in', 'the', 'corresponding', 'embryonic', 'stem', 'cell', 'line', 'this', 'leakage', 'phenomenon', 'could', 'be', 'explained', 'by', 'partial', 'erasure', 'bypass', 'or', 'override', 'of', 'imprints', 'or', 'could', 'represent', 'the', 'actual', 'activity', 'status', 'at', 'very', 'early', 'stages', 'of', 'development', 'the', 'mrna', 'levels', 'of', 'igf', 'and', 'igf', 'and', 'the', 'degree', 'of', 'methylation', 'at', 'specific', 'associated', 'sequences', 'were', 'correlated', 'according', 'to', 'previous', 'studies', 'in', 'embryos', 'and', 'thereby', 'are', 'consistent', 'with', 'suggestions', 'that', 'the', 'methylation', 'might', 'play', 'role', 'in', 'controlling', 'transcription', 'of', 'these', 'genes', 'paternal', 'specific', 'methylation', 'of', 'the', 'promoter', 'region', 'is', 'absent', 'in', 'sperm', 'yet', 'we', 'observed', 'its', 'presence', 'in', 'androgenetic', 'embryonic', 'stem', 'cells', 'or', 'before', 'the', 'potential', 'expression', 'phase', 'of', 'this', 'gene', 'in', 'embryoid', 'bodies', 'as', 'such', 'methylation', 'is', 'likely', 'to', 'invoke', 'repressive', 'effect', 'this', 'finding', 'raises', 'the', 'possibility', 'that', 'it', 'is', 'part', 'of', 'the', 'imprinting', 'mechanism', 'of', 'taking', 'the', 'form', 'of', 'secondary', 'imprint', 'or', 'epigenetic', 'modification', 'necessary', 'for', 'repression', 'of', 'the', 'paternal', 'allele'], ['the', 'snrpn', 'gene', 'encodes', 'small', 'nuclear', 'subunit', 'smn', 'thought', 'to', 'be', 'involved', 'in', 'splicing', 'of', 'pre', 'mrna', 'closely', 'related', 'protein', 'smb', 'is', 'constitutively', 'expressed', 'in', 'all', 'tissues', 'except', 'the', 'brain', 'where', 'smn', 'is', 'predominantly', 'expressed', 'the', 'mouse', 'homolog', 'of', 'the', 'snrpn', 'gene', 'has', 'been', 'shown', 'to', 'be', 'functionally', 'imprinted', 'in', 'mouse', 'brain', 'being', 'expressed', 'only', 'from', 'the', 'paternally', 'derived', 'chromosome', 'snrpn', 'has', 'been', 'mapped', 'to', 'human', 'chromosome', 'within', 'the', 'shortest', 'region', 'of', 'deletion', 'overlap', 'for', 'the', 'prader', 'willi', 'syndrome', 'we', 'have', 'now', 'demonstrated', 'functional', 'imprinting', 'of', 'the', 'human', 'snrpn', 'gene', 'using', 'reverse', 'transcription', 'followed', 'by', 'the', 'polymerase', 'chain', 'reaction', 'rt', 'pcr', 'no', 'expression', 'was', 'observed', 'in', 'cultured', 'skin', 'fibroblasts', 'of', 'prader', 'willi', 'patients', 'but', 'was', 'found', 'in', 'all', 'angelman', 'patients', 'and', 'normal', 'controls', 'examined', 'we', 'have', 'also', 'demonstrated', 'parent', 'specific', 'dna', 'methylation', 'imprint', 'within', 'intron', 'of', 'the', 'snrpn', 'gene', 'which', 'suggests', 'an', 'epigenetic', 'mechanism', 'by', 'which', 'parent', 'specific', 'expression', 'of', 'this', 'gene', 'might', 'be', 'inherited', 'our', 'findings', 'indicate', 'that', 'snrpn', 'is', 'expressed', 'only', 'from', 'the', 'paternally', 'derived', 'chromosome', 'in', 'humans', 'and', 'therefore', 'may', 'fullfill', 'one', 'major', 'criterion', 'for', 'being', 'involved', 'in', 'the', 'pathogenesis', 'of', 'the', 'prader', 'willi', 'syndrome'], ['development', 'in', 'mammals', 'is', 'influenced', 'by', 'genome', 'imprinting', 'which', 'results', 'in', 'differences', 'in', 'the', 'expression', 'of', 'some', 'homologous', 'maternal', 'and', 'paternal', 'alleles', 'this', 'process', 'initiated', 'in', 'the', 'germline', 'can', 'continue', 'following', 'fertilization', 'with', 'interactions', 'between', 'oocyte', 'cytoplasmic', 'factors', 'and', 'the', 'parental', 'genomes', 'involving', 'modifier', 'genes', 'further', 'epigenetic', 'modifications', 'may', 'follow', 'to', 'render', 'the', 'imprints', 'heritable', 'through', 'subsequent', 'cell', 'divisions', 'during', 'development', 'imprinting', 'of', 'genes', 'can', 'be', 'critical', 'for', 'their', 'dosage', 'affecting', 'embryonic', 'growth', 'cell', 'proliferation', 'and', 'differentiation', 'the', 'cumulative', 'effects', 'of', 'all', 'the', 'imprinted', 'genes', 'are', 'observed', 'in', 'androgenones', 'ag', 'and', 'parthenogenones', 'pg', 'which', 'reveal', 'complementary', 'phenotypes', 'with', 'respect', 'to', 'embryonic', 'and', 'extraembryonic', 'tissues', 'the', 'presence', 'of', 'pg', 'cells', 'in', 'chimeras', 'causes', 'growth', 'retardation', 'while', 'that', 'of', 'ag', 'cells', 'enhanced', 'growth', 'ag', 'cells', 'apparently', 'have', 'higher', 'cell', 'proliferation', 'rate', 'and', 'unlike', 'pg', 'cells', 'are', 'less', 'prone', 'to', 'selective', 'elimination', 'however', 'the', 'pg', 'germ', 'cells', 'are', 'exempt', 'from', 'cell', 'selection', 'in', 'chimeras', 'pg', 'cells', 'are', 'more', 'likely', 'to', 'be', 'found', 'in', 'ectodermal', 'derivatives', 'such', 'as', 'epidermis', 'and', 'brain', 'in', 'contrast', 'to', 'ag', 'cells', 'which', 'make', 'pronounced', 'contributions', 'to', 'many', 'mesodermal', 'derivatives', 'such', 'as', 'muscle', 'kidney', 'dermis', 'and', 'skeleton', 'the', 'presence', 'of', 'androgenetic', 'cells', 'in', 'chimeras', 'also', 'results', 'in', 'the', 'elongation', 'of', 'the', 'anterior', 'posterior', 'axis', 'and', 'sometimes', 'in', 'the', 'abnormal', 'development', 'of', 'skeletal', 'elements', 'along', 'the', 'axis', 'genetic', 'studies', 'highlight', 'the', 'influence', 'of', 'subsets', 'of', 'imprinted', 'genes', 'and', 'identify', 'those', 'that', 'are', 'critical', 'for', 'development'], ['zea', 'mays', 'transformants', 'produced', 'by', 'particle', 'bombardment', 'of', 'embryogenic', 'suspension', 'culture', 'cells', 'of', 'the', 'genotype', 'and', 'selected', 'on', 'kanamycin', 'or', 'bialaphos', 'were', 'characterized', 'with', 'respect', 'to', 'transgene', 'integration', 'expression', 'and', 'inheritance', 'selection', 'on', 'bialaphos', 'mediated', 'by', 'the', 'bar', 'or', 'par', 'genes', 'was', 'more', 'efficient', 'than', 'selection', 'on', 'kanamycin', 'mediated', 'by', 'the', 'nptii', 'gene', 'most', 'transformants', 'contained', 'multicopy', 'single', 'locus', 'transgene', 'insertion', 'events', 'transgene', 'expression', 'cassette', 'was', 'more', 'likely', 'to', 'be', 'rearranged', 'if', 'expression', 'of', 'that', 'gene', 'was', 'not', 'selected', 'for', 'during', 'callus', 'growth', 'not', 'all', 'plants', 'regenerated', 'from', 'calli', 'representing', 'single', 'transformation', 'events', 'expressed', 'the', 'transgenes', 'and', 'nonselectable', 'gene', 'uida', 'was', 'expressed', 'in', 'fewer', 'plants', 'than', 'was', 'the', 'selectable', 'transgene', 'mendelian', 'inheritance', 'of', 'transgenes', 'consistent', 'with', 'transgene', 'insertion', 'at', 'single', 'locus', 'was', 'observed', 'for', 'approximately', 'two', 'thirds', 'of', 'the', 'transformants', 'assessed', 'transgene', 'expression', 'was', 'typically', 'but', 'not', 'always', 'predictable', 'in', 'progeny', 'plants', 'transgene', 'silencing', 'as', 'well', 'as', 'poor', 'transgene', 'transmission', 'to', 'progeny', 'was', 'observed', 'in', 'some', 'plant', 'lines', 'in', 'which', 'the', 'parent', 'plants', 'had', 'expressed', 'the', 'transgene'], ['although', 'angelman', 'as', 'and', 'prader', 'willi', 'pws', 'syndromes', 'are', 'human', 'genetic', 'disorders', 'with', 'distinctly', 'different', 'developmental', 'and', 'phenotypes', 'they', 'both', 'have', 'abnormalities', 'in', 'inheritance', 'of', 'chromosome', 'whether', 'as', 'or', 'pws', 'arises', 'depends', 'on', 'the', 'parental', 'origin', 'of', 'deletion', 'or', 'uniparental', 'disomy', 'the', 'inheritance', 'of', 'copies', 'of', 'genetic', 'locus', 'from', 'only', 'one', 'parent', 'for', 'normal', 'development', 'requires', 'genetic', 'contribution', 'for', 'this', 'genetic', 'region', 'from', 'both', 'male', 'and', 'female', 'parent', 'the', 'dependence', 'on', 'parental', 'origin', 'implies', 'that', 'genes', 'in', 'human', 'have', 'distinct', 'functions', 'depending', 'upon', 'epigenetic', 'parent', 'of', 'origin', 'differences', 'known', 'as', 'genomic', 'imprinting', 'here', 'review', 'the', 'role', 'of', 'uniparental', 'disomy', 'and', 'genomic', 'imprinting', 'in', 'the', 'pathogenesis', 'of', 'as', 'and', 'pws', 'and', 'briefly', 'discuss', 'phenotype', 'genotype', 'correlations', 'using', 'candidate', 'genes', 'and', 'mouse', 'models', 'in', 'particular', 'for'], ['because', 'development', 'is', 'epigenetic', 'diverse', 'aspects', 'of', 'morphology', 'are', 'integrated', 'during', 'ontogeny', 'using', 'the', 'method', 'of', 'thin', 'plate', 'splines', 'and', 'the', 'decomposition', 'of', 'these', 'splines', 'by', 'their', 'principal', 'warps', 'we', 'examine', 'the', 'ontogeny', 'of', 'integrated', 'features', 'of', 'skull', 'growth', 'of', 'the', 'cotton', 'rat', 'sigmodon', 'fulviventer', 'as', 'observed', 'in', 'landmark', 'locations', 'in', 'the', 'ventral', 'view', 'postnatal', 'growth', 'of', 'the', 'skull', 'in', 'sigmodon', 'is', 'not', 'adequately', 'described', 'by', 'the', 'familiar', 'contrast', 'between', 'relatively', 'rapid', 'facial', 'elongation', 'and', 'slow', 'precocial', 'growth', 'of', 'the', 'cranial', 'base', 'no', 'developmental', 'units', 'corresponding', 'to', 'facial', 'skull', 'and', 'cranial', 'base', 'emerge', 'from', 'analysis', 'of', 'geometric', 'shape', 'change', 'rather', 'skull', 'growth', 'is', 'both', 'more', 'integrated', 'and', 'more', 'complex', 'exhibiting', 'both', 'skull', 'wide', 'integration', 'and', 'locally', 'individualized', 'regions', 'like', 'skull', 'shape', 'integration', 'has', 'an', 'ontogeny', 'different', 'regions', 'of', 'the', 'skull', 'can', 'be', 'partitioned', 'into', 'developmentally', 'individualized', 'parts', 'in', 'different', 'ways', 'at', 'different', 'ages', 'the', 'effective', 'count', 'of', 'individualized', 'parts', 'decreases', 'substantially', 'before', 'weaning', 'occurs', 'suggesting', 'that', 'the', 'integration', 'required', 'by', 'the', 'functionally', 'demanding', 'activity', 'of', 'chewing', 'gradually', 'develops', 'before', 'the', 'functional', 'transition', 'occurs', 'our', 'description', 'of', 'skull', 'growth', 'and', 'integration', 'does', 'not', 'depend', 'upon', 'arbitrary', 'priori', 'choices', 'about', 'what', 'to', 'measure', 'rather', 'we', 'base', 'our', 'decomposition', 'of', 'the', 'whole', 'into', 'parts', 'upon', 'results', 'of', 'the', 'data', 'analysis', 'our', 'approach', 'complicates', 'the', 'study', 'of', 'heterochrony', 'but', 'because', 'it', 'expresses', 'the', 'spatiotemporal', 'organization', 'of', 'ontogeny', 'it', 'enables', 'the', 'study', 'of', 'heterotopy'], ['novel', 'epigenetic', 'mechanism', 'that', 'can', 'affect', 'minichromosome', 'centromere', 'function', 'in', 'vivo', 'has', 'been', 'identified', 'in', 'pombe', 'this', 'epigenetic', 'system', 'can', 'result', 'in', 'the', 'conversion', 'of', 'nonfunctional', 'centromere', 'to', 'functional', 'one', 'without', 'changes', 'in', 'the', 'content', 'structural', 'arrangement', 'or', 'chemical', 'modification', 'state', 'of', 'the', 'minichromosomal', 'dna', 'the', 'conversion', 'from', 'centromere', 'inactive', 'to', 'an', 'active', 'state', 'which', 'is', 'evident', 'with', 'minichromosomes', 'carrying', 'abbreviated', 'centromeric', 'dna', 'constructions', 'occurs', 'with', 'relatively', 'high', 'frequency', 'during', 'mitotic', 'cell', 'divisions', 'and', 'readily', 'provides', 'an', 'in', 'vivo', 'essay', 'for', 'proper', 'centromere', 'formation', 'the', 'centromere', 'targeted', 'epigenetic', 'system', 'supports', 'model', 'for', 'centromere', 'function', 'that', 'involves', 'specific', 'de', 'novo', 'folding', 'of', 'centromeric', 'components', 'into', 'higher', 'order', 'chromatin', 'structure'], ['during', 'development', 'patterns', 'of', 'differential', 'gene', 'expression', 'defining', 'determined', 'states', 'of', 'cells', 'need', 'to', 'be', 'maintained', 'over', 'many', 'cell', 'generations', 'in', 'drosophila', 'genetic', 'and', 'molecular', 'analyses', 'led', 'to', 'the', 'discovery', 'of', 'set', 'of', 'proteins', 'which', 'seem', 'to', 'exert', 'such', 'memory', 'function', 'by', 'using', 'epigenetic', 'mechanisms', 'recent', 'experiments', 'demonstrate', 'that', 'in', 'particular', 'the', 'heritable', 'inactivation', 'of', 'regulatory', 'genes', 'relies', 'on', 'stable', 'changes', 'in', 'the', 'higher', 'order', 'constitution', 'of', 'chromatin'], ['the', 'ability', 'of', 'chemicals', 'with', 'tumor', 'promoting', 'or', 'tumor', 'inhibiting', 'activity', 'to', 'modulate', 'gap', 'junctional', 'intercellular', 'communication', 'is', 'reviewed', 'the', 'two', 'most', 'extensively', 'used', 'types', 'of', 'assays', 'for', 'screening', 'tests', 'are', 'metabolic', 'cooperation', 'assays', 'involving', 'exchange', 'between', 'cells', 'of', 'precursors', 'of', 'nucleic', 'acid', 'synthesis', 'and', 'dye', 'transfer', 'assays', 'that', 'measure', 'exchange', 'of', 'fluorescent', 'dye', 'from', 'loaded', 'cells', 'to', 'adjacent', 'cells', 'about', 'substances', 'of', 'different', 'biological', 'activities', 'have', 'been', 'studied', 'using', 'various', 'assays', 'for', 'tumor', 'promoters', 'epigenetic', 'carcinogens', 'metabolic', 'cooperation', 'assays', 'have', 'sensitivity', 'of', 'and', 'dye', 'transfer', 'assays', 'thirty', 'percent', 'of', 'dna', 'reactive', 'carcinogens', 'also', 'possess', 'the', 'ability', 'to', 'uncouple', 'cells', 'the', 'complete', 'estimation', 'of', 'the', 'predictive', 'power', 'of', 'these', 'assays', 'could', 'not', 'be', 'made', 'because', 'the', 'majority', 'of', 'the', 'substances', 'studied', 'for', 'intercellular', 'communication', 'effects', 'in', 'vitro', 'have', 'not', 'yet', 'been', 'studied', 'for', 'promoting', 'activity', 'in', 'vivo', 'both', 'metabolic', 'cooperation', 'assays', 'and', 'dye', 'transfer', 'assays', 'respond', 'well', 'to', 'the', 'following', 'classes', 'of', 'substances', 'phorbol', 'esters', 'organochlorine', 'pesticides', 'polybrominated', 'biphenyls', 'promoters', 'for', 'urinary', 'bladder', 'some', 'biological', 'toxins', 'peroxisome', 'proliferators', 'and', 'some', 'complex', 'mixtures', 'results', 'of', 'in', 'vitro', 'assays', 'for', 'such', 'tumor', 'promoters', 'nongenotoxic', 'carcinogens', 'such', 'as', 'some', 'bile', 'acids', 'some', 'peroxides', 'alkanes', 'some', 'hormones', 'mineral', 'dusts', 'ascorbic', 'acid', 'okadaic', 'acid', 'and', 'benz', 'pyrene', 'do', 'not', 'correlate', 'with', 'the', 'data', 'of', 'in', 'vivo', 'two', 'stage', 'or', 'complete', 'carcinogenesis', 'enhancement', 'of', 'intercellular', 'communication', 'was', 'found', 'for', 'chemicals', 'among', 'these', 'camp', 'retinoids', 'and', 'carotenoids', 'have', 'demonstrated', 'inhibition', 'of', 'carcinogenesis', 'we', 'examine', 'number', 'of', 'factors', 'that', 'are', 'important', 'for', 'routine', 'screening', 'including', 'the', 'requirement', 'for', 'for', 'some', 'agents', 'to', 'exert', 'effects', 'on', 'gap', 'junctions', 'we', 'also', 'discuss', 'the', 'mechanisms', 'of', 'tumor', 'promoter', 'and', 'tumor', 'inhibitor', 'effects', 'on', 'gap', 'junctional', 'permeability', 'including', 'influences', 'of', 'protein', 'kinase', 'activation', 'changes', 'in', 'proton', 'and', 'ca', 'intracellular', 'concentrations', 'and', 'effects', 'of', 'oxy', 'radical', 'production'], ['ascobolus', 'immersus', 'artificial', 'gene', 'repeats', 'were', 'shown', 'previously', 'to', 'be', 'subject', 'premeiotically', 'to', 'both', 'cytosine', 'methylation', 'and', 'inactivation', 'we', 'studied', 'sexual', 'progenies', 'of', 'strains', 'harbouring', 'two', 'wild', 'type', 'copies', 'of', 'the', 'endogenous', 'met', 'gene', 'lying', 'either', 'in', 'tandem', 'array', 'or', 'at', 'ectopic', 'unlinked', 'positions', 'by', 'investigating', 'the', 'methylation', 'status', 'ii', 'searching', 'for', 'mutations', 'and', 'iii', 'analysing', 'the', 'inheritance', 'of', 'inactivation', 'both', 'in', 'mitotic', 'and', 'sexual', 'offspring', 'of', 'the', 'tandem', 'progeny', 'and', 'of', 'the', 'ectopic', 'progeny', 'had', 'methylated', 'repeats', 'and', 'displayed', 'gene', 'inactivation', 'similar', 'methylation', 'patterns', 'involving', 'all', 'or', 'most', 'of', 'the', 'cytosine', 'residues', 'within', 'the', 'repeats', 'were', 'observed', 'in', 'both', 'arrangements', 'the', 'inactivated', 'met', 'copies', 'were', 'totally', 'devoid', 'of', 'mutation', 'as', 'deduced', 'from', 'extensive', 'restriction', 'site', 'analysis', 'and', 'dna', 'sequencing', 'ii', 'the', 'finding', 'that', 'all', 'the', 'met', 'derivatives', 'tested', 'reverted', 'to', 'prototrophy', 'in', 'selective', 'conditions', 'and', 'iii', 'the', 'finding', 'that', 'an', 'inactivated', 'copy', 'of', 'met', 'stripped', 'of', 'its', 'methylation', 'through', 'amplification', 'in', 'escherichia', 'coli', 'regained', 'activity', 'when', 'reintroduced', 'in', 'immersus', 'in', 'the', 'absence', 'of', 'selection', 'gene', 'silencing', 'and', 'methylation', 'were', 'faithfully', 'maintained', 'through', 'mitotic', 'divisions', 'and', 'through', 'five', 'successive', 'sexual', 'cycles', 'altogether', 'these', 'data', 'show', 'the', 'epimutational', 'nature', 'of', 'this', 'methylation', 'induced', 'premeiotically', 'mip', 'process'], ['transposable', 'elements', 'from', 'the', 'robertson', 'mutator', 'family', 'are', 'highly', 'active', 'insertional', 'mutagens', 'in', 'maize', 'however', 'mutations', 'caused', 'by', 'the', 'insertion', 'of', 'responder', 'non', 'autonomous', 'elements', 'frequently', 'depend', 'on', 'the', 'presence', 'of', 'active', 'regulator', 'autonomous', 'elements', 'for', 'their', 'phenotypic', 'effects', 'the', 'hcf', 'mu', 'mutation', 'has', 'been', 'previously', 'shown', 'to', 'depend', 'on', 'mu', 'activity', 'in', 'this', 'way', 'the', 'dominant', 'lesion', 'mimic', 'mutation', 'also', 'requires', 'mu', 'activity', 'for', 'its', 'phenotypic', 'effects', 'we', 'have', 'used', 'double', 'mutants', 'to', 'show', 'that', 'the', 'loss', 'of', 'mu', 'activity', 'results', 'in', 'the', 'coordinate', 'suppression', 'of', 'both', 'mutant', 'phenotypes', 'this', 'loss', 'can', 'occur', 'somatically', 'resulting', 'in', 'large', 'clones', 'of', 'cells', 'that', 'have', 'wild', 'type', 'phenotype', 'autonomous', 'and', 'non', 'autonomous', 'mutator', 'elements', 'within', 'these', 'clones', 'are', 'insensitive', 'to', 'digestion', 'with', 'methylation', 'sensitive', 'enzymes', 'suggesting', 'extensive', 'methylation', 'of', 'cg', 'and', 'non', 'cc', 'cytosine', 'residues', 'our', 'data', 'are', 'consistent', 'with', 'the', 'sectors', 'being', 'caused', 'by', 'the', 'cycling', 'of', 'mudr', 'regulatory', 'elements', 'between', 'active', 'and', 'inactive', 'phases', 'the', 'pattern', 'of', 'sectors', 'suggests', 'that', 'they', 'are', 'clonal', 'and', 'that', 'they', 'are', 'derived', 'from', 'the', 'apical', 'cells', 'of', 'the', 'vegetative', 'shoot', 'meristem', 'we', 'propose', 'that', 'these', 'cells', 'are', 'more', 'likely', 'to', 'undergo', 'epigenetic', 'loss', 'of', 'mu', 'activity', 'because', 'of', 'their', 'longer', 'cell', 'division', 'cycle', 'during', 'shoot', 'growth', 'coordinate', 'suppression', 'of', 'unlinked', 'mutations', 'can', 'be', 'used', 'to', 'perform', 'mosaic', 'analysis', 'in', 'maize'], ['the', 'neural', 'crest', 'is', 'fascinating', 'structure', 'of', 'the', 'vertebrate', 'embryo', 'its', 'ontogeny', 'includes', 'transient', 'period', 'during', 'which', 'its', 'component', 'cells', 'undergo', 'an', 'epithelio', 'mesenchymal', 'transition', 'and', 'become', 'migratory', 'this', 'phase', 'was', 'shown', 'recently', 'to', 'be', 'controlled', 'by', 'the', 'slug', 'gene', 'which', 'belongs', 'to', 'the', 'snail', 'family', 'of', 'drosophila', 'transcription', 'factors', 'after', 'homing', 'to', 'specific', 'sites', 'in', 'the', 'embryo', 'the', 'crest', 'derived', 'cells', 'produce', 'large', 'variety', 'of', 'phenotypes', 'recent', 'advances', 'have', 'shown', 'that', 'during', 'migration', 'most', 'crest', 'cells', 'exhibit', 'various', 'degrees', 'of', 'some', 'being', 'already', 'committed', 'to', 'single', 'and', 'definite', 'fate', 'moreover', 'several', 'lines', 'of', 'evidence', 'point', 'to', 'the', 'existence', 'of', 'totipotent', 'stem', 'cells', 'in', 'the', 'neural', 'crest', 'the', 'progeny', 'of', 'which', 'become', 'progressively', 'diversified', 'through', 'combination', 'of', 'intrinsic', 'and', 'extrinsic', 'influences', 'the', 'latter', 'have', 'been', 'documented', 'by', 'the', 'disruption', 'of', 'several', 'neurotrophin', 'genes', 'which', 'results', 'in', 'severe', 'deficiencies', 'of', 'selected', 'subsets', 'of', 'neural', 'crest', 'derivatives', 'the', 'neural', 'crest', 'has', 'also', 'been', 'shown', 'to', 'play', 'an', 'important', 'role', 'in', 'the', 'development', 'of', 'the', 'vertebrate', 'head', 'and', 'hypobranchial', 'region', 'the', 'genetic', 'control', 'of', 'this', 'process', 'depends', 'on', 'the', 'activity', 'of', 'developmental', 'genes', 'among', 'which', 'the', 'vertebrate', 'hox', 'genes', 'are', 'essential', 'particularly', 'at', 'the', 'rhombencephalic', 'level'], ['the', 'ability', 'of', 'simian', 'virus', 'encoded', 'large', 'antigen', 'to', 'disrupt', 'the', 'growth', 'control', 'of', 'variety', 'of', 'cell', 'types', 'is', 'related', 'to', 'its', 'ability', 'to', 'interfere', 'with', 'certain', 'cellular', 'proteins', 'such', 'as', 'and', 'the', 'retinoblastoma', 'susceptibility', 'gene', 'product', 'prb', 'we', 'have', 'used', 'wild', 'type', 'and', 'mutant', 'forms', 'of', 'antigen', 'in', 'transgenic', 'mice', 'to', 'dissect', 'the', 'roles', 'of', 'prb', 'and', 'other', 'cellular', 'proteins', 'in', 'tumorigenesis', 'of', 'different', 'cell', 'types', 'in', 'this', 'study', 'using', 'cell', 'specific', 'promoter', 'to', 'target', 'expression', 'specifically', 'to', 'brain', 'epithelium', 'the', 'choroid', 'plexus', 'and', 'to', 'and', 'lymphoid', 'cells', 'we', 'characterize', 'the', 'tumorigenic', 'capacity', 'of', 'antigen', 'fragment', 'that', 'comprises', 'only', 'the', 'amino', 'terminal', 'residues', 'this', 'fragment', 'dl', 'retains', 'the', 'ability', 'to', 'interact', 'with', 'prb', 'and', 'but', 'lacks', 'the', 'binding', 'domain', 'while', 'loss', 'of', 'the', 'binding', 'region', 'results', 'in', 'loss', 'of', 'the', 'capacity', 'to', 'induce', 'lymphoid', 'abnormalities', 'dl', 'retains', 'the', 'ability', 'to', 'induce', 'choroid', 'plexus', 'tumors', 'that', 'are', 'histologically', 'from', 'those', 'induced', 'by', 'wild', 'type', 'antigen', 'tumors', 'induced', 'by', 'dl', 'develop', 'much', 'more', 'slowly', 'however', 'reaching', 'an', 'end', 'point', 'at', 'around', 'months', 'of', 'age', 'rather', 'than', 'at', 'to', 'months', 'analysis', 'of', 'tumor', 'progression', 'indicates', 'that', 'tumor', 'induction', 'by', 'dl', 'does', 'not', 'require', 'secondary', 'genetic', 'or', 'epigenetic', 'events', 'rather', 'the', 'tumor', 'growth', 'rate', 'is', 'significantly', 'slowed', 'indicating', 'that', 'the', 'antigen', 'terminal', 'region', 'contributes', 'to', 'tumor', 'progression', 'in', 'this', 'cell', 'type', 'in', 'contrast', 'the', 'prb', 'binding', 'region', 'appears', 'essential', 'for', 'tumorigenesis', 'as', 'mutation', 'of', 'residue', 'known', 'to', 'disrupt', 'prb', 'and', 'binding', 'to', 'wild', 'type', 'antigen', 'abolishes', 'the', 'ability', 'of', 'the', 'dl', 'protein', 'to', 'induce', 'growth', 'abnormalities', 'in', 'the', 'brain'], ['in', 'this', 'study', 'we', 'used', 'the', 'ade', 'gene', 'in', 'colony', 'color', 'assay', 'to', 'monitor', 'transcription', 'from', 'the', 'normally', 'silent', 'hmr', 'mating', 'type', 'locus', 'in', 'saccharomyces', 'cerevisiae', 'this', 'sensitive', 'assay', 'reveals', 'that', 'some', 'previously', 'identified', 'cis', 'and', 'trans', 'acting', 'mutations', 'destabilize', 'silencing', 'causing', 'genetically', 'identical', 'cells', 'to', 'switch', 'between', 'repressed', 'and', 'derepressed', 'transcriptional', 'states', 'deletion', 'of', 'the', 'autonomously', 'replicating', 'sequence', 'ars', 'consensus', 'element', 'at', 'the', 'hmr', 'silencer', 'or', 'mutation', 'of', 'the', 'silencer', 'binding', 'protein', 'rap', 'rap', 'results', 'in', 'the', 'presence', 'of', 'large', 'sectors', 'within', 'individual', 'colonies', 'of', 'both', 'repressed', 'ade', 'pink', 'and', 'derepressed', 'ade', 'white', 'cells', 'these', 'results', 'suggest', 'that', 'both', 'the', 'ars', 'consensus', 'element', 'and', 'the', 'ra', 'protein', 'play', 'role', 'in', 'the', 'establishment', 'of', 'repression', 'at', 'hmr', 'in', 'diploid', 'cells', 'the', 'two', 'copies', 'of', 'hmr', 'appear', 'to', 'behave', 'identically', 'suggesting', 'that', 'the', 'switching', 'event', 'though', 'apparently', 'stochastic', 'reflects', 'some', 'property', 'of', 'the', 'cell', 'rather', 'than', 'specific', 'event', 'at', 'each', 'hmr', 'locus', 'in', 'the', 'ade', 'assay', 'system', 'silencing', 'depends', 'completely', 'upon', 'the', 'function', 'of', 'the', 'sir', 'genes', 'known', 'trans', 'acting', 'regulators', 'of', 'the', 'silent', 'loci', 'and', 'is', 'sensitive', 'to', 'the', 'gene', 'dosage', 'of', 'two', 'sir', 'genes', 'sir', 'and', 'sir', 'using', 'the', 'ade', 'colony', 'color', 'assay', 'in', 'genetic', 'screen', 'for', 'suppressors', 'of', 'rap', 'silencer', 'ars', 'element', 'deletion', 'double', 'mutants', 'we', 'have', 'identified', 'large', 'number', 'of', 'genes', 'that', 'may', 'affect', 'the', 'establishment', 'of', 'repression', 'at', 'the', 'hmr', 'silent', 'mating', 'type', 'locus'], ['we', 'analysed', 'de', 'novo', 'dna', 'methylation', 'occurring', 'in', 'plants', 'obtained', 'from', 'the', 'transgenic', 'petunia', 'line', 'this', 'line', 'contains', 'one', 'copy', 'of', 'the', 'maize', 'al', 'gene', 'that', 'leads', 'to', 'the', 'production', 'of', 'brick', 'red', 'pelargonidin', 'pigment', 'in', 'the', 'flowers', 'due', 'to', 'its', 'integration', 'into', 'an', 'unmethylated', 'genomic', 'region', 'the', 'transgene', 'is', 'hypomethylated', 'and', 'active', 'several', 'epigenetic', 'variants', 'of', 'line', 'were', 'selected', 'that', 'exhibit', 'characteristic', 'and', 'somatically', 'stable', 'pigmentation', 'patterns', 'displaying', 'fully', 'coloured', 'marbled', 'or', 'colourless', 'flowers', 'analysis', 'of', 'the', 'dna', 'methylation', 'patterns', 'revealed', 'that', 'the', 'decrease', 'in', 'pigmentation', 'among', 'the', 'epigenetic', 'variants', 'was', 'correlated', 'with', 'an', 'increase', 'in', 'methylation', 'specifically', 'of', 'the', 'transgene', 'dna', 'no', 'change', 'in', 'methylation', 'of', 'the', 'hypomethylated', 'integration', 'region', 'could', 'be', 'detected', 'similar', 'increase', 'in', 'methylation', 'specifically', 'in', 'the', 'transgene', 'region', 'was', 'also', 'observed', 'among', 'progeny', 'of', 'del', 'deletion', 'derivative', 'of', 'that', 'no', 'longer', 'produces', 'pelargonidin', 'pigments', 'due', 'to', 'deletion', 'in', 'the', 'al', 'coding', 'region', 'again', 'de', 'novo', 'methylation', 'is', 'specifically', 'directed', 'to', 'the', 'transgene', 'while', 'the', 'hypomethylated', 'character', 'of', 'neighbouring', 'regions', 'is', 'not', 'affected', 'possible', 'mechanisms', 'for', 'transgene', 'specific', 'methylation', 'and', 'its', 'consequences', 'for', 'long', 'term', 'use', 'of', 'transgenic', 'material', 'are', 'discussed'], ['plant', 'cells', 'growing', 'in', 'an', 'artificial', 'culture', 'environment', 'make', 'numerous', 'genetic', 'mistakes', 'these', 'alterations', 'are', 'manifested', 'as', 'increased', 'frequencies', 'of', 'single', 'gene', 'mutations', 'chromosome', 'breakages', 'transposable', 'element', 'activations', 'quantitative', 'trait', 'variations', 'and', 'modifications', 'of', 'normal', 'dna', 'methylation', 'patterns', 'evidence', 'is', 'presented', 'that', 'indicates', 'high', 'frequency', 'of', 'dna', 'hypomethylation', 'as', 'the', 'result', 'of', 'the', 'tissue', 'culture', 'process', 'fifteen', 'percent', 'of', 'the', 'methylation', 'changes', 'appear', 'to', 'have', 'been', 'homozygous', 'in', 'the', 'original', 'regenerated', 'plants', 'hypothesis', 'is', 'advanced', 'that', 'relates', 'dna', 'methylation', 'to', 'the', 'variety', 'of', 'genetic', 'alterations', 'found', 'among', 'maize', 'tissue', 'culture', 'regenerants', 'and', 'their', 'progenies', 'the', 'epigenetic', 'nature', 'of', 'dna', 'methylation', 'raises', 'questions', 'concerning', 'the', 'stability', 'of', 'tissue', 'culture', 'induced', 'changes', 'in', 'self', 'pollinations', 'and', 'crosses'], ['in', 'this', 'essay', 'review', 'describe', 'the', 'relationship', 'between', 'plant', 'organization', 'variation', 'and', 'evolution', 'the', 'hypothetical', 'optimal', 'design', 'of', 'an', 'organism', 'living', 'in', 'given', 'environment', 'must', 'be', 'specified', 'to', 'test', 'whether', 'evolution', 'is', 'adaptive', 'in', 'variable', 'environment', 'this', 'design', 'should', 'include', 'system', 'for', 'plasticity', 'plasticity', 'in', 'plants', 'is', 'made', 'possible', 'by', 'modular', 'growth', 'in', 'modular', 'growth', 'new', 'parts', 'can', 'constantly', 'be', 'added', 'to', 'body', 'resulting', 'in', 'population', 'like', 'structure', 'of', 'the', 'organism', 'the', 'developmental', 'basis', 'for', 'modular', 'growth', 'is', 'an', 'open', 'ended', 'ontogeny', 'in', 'which', 'the', 'formation', 'of', 'organs', 'is', 'not', 'controlled', 'by', 'patterns', 'of', 'cell', 'divisions', 'and', 'migrations', 'and', 'where', 'there', 'is', 'no', 'separation', 'into', 'germ', 'line', 'and', 'soma', 'epigenetic', 'pattern', 'formation', 'late', 'differentiation', 'and', 'loose', 'integration', 'of', 'the', 'individual', 'arising', 'from', 'zygote', 'genet', 'are', 'characteristic', 'for', 'plants', 'because', 'the', 'phenotype', 'is', 'the', 'result', 'of', 'an', 'interaction', 'between', 'genotype', 'and', 'environment', 'the', 'contributions', 'of', 'the', 'two', 'cannot', 'be', 'separated', 'nevertheless', 'with', 'appropriate', 'experimental', 'procedures', 'it', 'is', 'possible', 'to', 'separate', 'the', 'variation', 'among', 'phenotypes', 'into', 'variation', 'among', 'genotypes', 'variation', 'among', 'environments', 'and', 'covariation', 'of', 'genotypes', 'and', 'environments', 'several', 'genotypes', 'must', 'be', 'grown', 'in', 'several', 'environments', 'if', 'the', 'environments', 'can', 'be', 'put', 'on', 'scale', 'as', 'in', 'gradient', 'experiments', 'reaction', 'norms', 'of', 'genotypes', 'may', 'be', 'quantified', 'local', 'specialization', 'of', 'genotypes', 'in', 'heterogeneous', 'environment', 'may', 'be', 'recognized', 'by', 'transplant', 'experiments', 'new', 'results', 'from', 'both', 'types', 'of', 'experiments', 'indicate', 'that', 'the', 'general', 'purpose', 'genotype', 'is', 'more', 'common', 'than', 'originally', 'thought', 'and', 'suggest', 'that', 'phenotypic', 'plasticity', 'in', 'plants', 'has', 'relatively', 'low', 'costs', 'however', 'the', 'critical', 'experiments', 'to', 'measure', 'these', 'costs', 'have', 'not', 'yet', 'been', 'published', 'the', 'genetic', 'analysis', 'of', 'phenotypic', 'variation', 'and', 'the', 'use', 'of', 'phenotypic', 'characters', 'for', 'reconstructing', 'phylogenies', 'is', 'hampered', 'by', 'the', 'particular', 'plant', 'features', 'mentioned', 'above', 'environment', 'induced', 'phenotypic', 'effects', 'can', 'be', 'persistent', 'and', 'can', 'be', 'transmitted', 'to', 'clonal', 'or', 'even', 'sexual', 'progeny', 'gene', 'flow', 'among', 'taxa', 'via', 'sexual', 'and', 'possibly', 'other', 'mechanisms', 'together', 'with', 'the', 'loose', 'integration', 'of', 'the', 'individual', 'make', 'it', 'difficult', 'to', 'recognize', 'phylogenies', 'for', 'systematic', 'groups', 'in', 'plants', 'some', 'of', 'these', 'problems', 'are', 'illustrated', 'with', 'species', 'from', 'the', 'genera', 'solidago', 'and', 'carex'], ['mating', 'type', 'genes', 'resident', 'in', 'the', 'silent', 'cassette', 'hml', 'at', 'the', 'left', 'arm', 'of', 'chromosome', 'iii', 'are', 'repressed', 'by', 'the', 'action', 'of', 'four', 'sir', 'gene', 'products', 'mediated', 'independently', 'through', 'two', 'cis', 'acting', 'sites', 'termed', 'the', 'and', 'silencers', 'we', 'have', 'found', 'that', 'in', 'the', 'absence', 'of', 'the', 'silencer', 'deletion', 'of', 'any', 'one', 'of', 'three', 'distinct', 'elements', 'within', 'yields', 'partial', 'derepression', 'of', 'the', 'mating', 'type', 'genes', 'resident', 'at', 'hml', 'whereas', 'deletion', 'of', 'any', 'two', 'yields', 'full', 'derepression', 'these', 'elements', 'correspond', 'to', 'binding', 'site', 'for', 'the', 'abundant', 'dna', 'binding', 'protein', 'rap', 'an', 'autonomous', 'replicating', 'sequence', 'ars', 'and', 'an', 'as', 'yet', 'undistinguished', 'region', 'from', 'detailed', 'deletion', 'analysis', 'of', 'the', 'site', 'we', 'conclude', 'that', 'the', 'ars', 'element', 'contributes', 'to', 'silencer', 'function', 'in', 'capacity', 'distinct', 'from', 'its', 'role', 'as', 'an', 'initiator', 'of', 'dna', 'replication', 'in', 'addition', 'we', 'find', 'that', 'strains', 'deleted', 'for', 'any', 'one', 'of', 'these', 'elements', 'comprise', 'two', 'genetically', 'identical', 'but', 'phenotypically', 'distinct', 'types', 'of', 'cells', 'those', 'with', 'hml', 'apparently', 'fully', 'derepressed', 'and', 'those', 'with', 'hml', 'apparently', 'completely', 'repressed', 'these', 'results', 'reinforce', 'the', 'notion', 'that', 'epigenetic', 'inheritance', 'is', 'an', 'intrinsic', 'characteristic', 'of', 'silencer', 'action'], ['two', 'instances', 'of', 'genetic', 'transmission', 'of', 'spontaneous', 'epimutation', 'of', 'the', 'maize', 'rr', 'gene', 'were', 'identified', 'transmission', 'gave', 'rise', 'to', 'two', 'similar', 'moderately', 'stable', 'alleles', 'designated', 'pr', 'and', 'pr', 'that', 'exhibited', 'mendelian', 'behavior', 'both', 'isolates', 'of', 'pr', 'conditioned', 'variable', 'and', 'variegated', 'phenotype', 'unlike', 'the', 'uniform', 'pigmentation', 'conditioned', 'by', 'rr', 'extensive', 'genomic', 'analysis', 'failed', 'to', 'reveal', 'insertions', 'deletions', 'or', 'restriction', 'site', 'polymorphisms', 'between', 'the', 'new', 'allele', 'and', 'its', 'progenitor', 'however', 'methylation', 'of', 'the', 'gene', 'was', 'increased', 'in', 'pr', 'relative', 'to', 'rr', 'and', 'was', 'greatly', 'reduced', 'though', 'not', 'lost', 'in', 'revertant', 'to', 'uniform', 'pigmentation', 'variability', 'in', 'pigmentation', 'conditioned', 'by', 'pr', 'correlated', 'with', 'variability', 'in', 'transcript', 'levels', 'of', 'the', 'gene', 'and', 'both', 'correlated', 'inversely', 'with', 'variability', 'in', 'its', 'methylation', 'part', 'of', 'the', 'variability', 'in', 'methylation', 'could', 'be', 'accounted', 'for', 'by', 'developmental', 'decrease', 'in', 'methylation', 'in', 'all', 'tissues', 'of', 'plants', 'carrying', 'pr', 'we', 'hypothesize', 'that', 'the', 'variegated', 'phenotype', 'results', 'from', 'general', 'epigenetic', 'pathway', 'which', 'causes', 'progressive', 'decrease', 'in', 'methylation', 'and', 'increase', 'in', 'expression', 'potential', 'of', 'the', 'gene', 'as', 'function', 'of', 'cell', 'divisions', 'in', 'each', 'meristem', 'of', 'the', 'plant', 'this', 'renders', 'all', 'tissues', 'chimeric', 'for', 'functional', 'gene', 'chimerism', 'is', 'visualized', 'as', 'variegation', 'only', 'in', 'pericarp', 'due', 'to', 'the', 'tissue', 'specificity', 'of', 'gene', 'expression', 'therefore', 'this', 'allele', 'that', 'originates', 'from', 'epimutation', 'may', 'exemplify', 'an', 'epigenetic', 'mechanism', 'for', 'variegation', 'in', 'maize'], ['previous', 'work', 'has', 'shown', 'that', 'two', 'unlinked', 'partially', 'homologous', 'transgene', 'loci', 'can', 'interact', 'in', 'plant', 'nuclei', 'leading', 'to', 'reversible', 'methylation', 'and', 'inactivation', 'of', 'one', 'transgene', 'locus', 'in', 'the', 'presence', 'of', 'the', 'second', 'to', 'study', 'whether', 'the', 'chromosomal', 'location', 'of', 'transgene', 'influences', 'its', 'susceptibility', 'to', 'trans', 'inactivation', 'we', 'retransformed', 'four', 'transgenic', 'lines', 'which', 'contained', 'the', 'same', 'construct', 'integrated', 'in', 'different', 'chromosomal', 'locations', 'with', 'second', 'partially', 'homologous', 'construct', 'at', 'least', 'double', 'transformants', 'dts', 'were', 'regenerated', 'from', 'each', 'single', 'transformant', 'st', 'and', 'screened', 'for', 'inactivation', 'of', 'markers', 'chloramphenicol', 'cat', 'hygromycin', 'resistance', 'hyg', 'at', 'the', 'resident', 'locus', 'for', 'two', 'sts', 'locus', 'markers', 'were', 'inactivated', 'in', 'less', 'than', 'of', 'the', 'dts', 'suggesting', 'that', 'at', 'these', 'integration', 'sites', 'was', 'relatively', 'resistant', 'to', 'trans', 'inactivation', 'in', 'contrast', 'the', 'other', 'two', 'sts', 'appeared', 'to', 'be', 'more', 'sensitive', 'to', 'trans', 'inactivation', 'of', 'the', 'dts', 'were', 'cat', 'and', 'or', 'hyg', 'inactivation', 'of', 'locus', 'markers', 'could', 'be', 'attributed', 'to', 'two', 'distinct', 'phenomena', 'regeneration', 'from', 'cells', 'containing', 'different', 'epigenetic', 'states', 'of', 'in', 'which', 'either', 'both', 'one', 'or', 'none', 'of', 'the', 'alleles', 'was', 'active', 'this', 'instability', 'in', 'the', 'expression', 'of', 'the', 'locus', 'which', 'was', 'independent', 'of', 'was', 'more', 'pronounced', 'in', 'the', 'homozygous', 'state', 'and', 'was', 'associated', 'with', 'cellular', 'mosaicism', 'of', 'expression', 'and', 'methylation', 'the', 'presence', 'of', 'an', 'unlinked', 'locus', 'could', 'weaken', 'the', 'hyg', 'phenotype', 'by', 'transcriptional', 'inactivation', 'and', 'increased', 'methylation', 'of', 'the', 'hph', 'gene', 'at', 'the', 'locus', 'these', 'results', 'indicated', 'that', 'susceptible', 'transgene', 'locus', 'is', 'inherently', 'unstable', 'and', 'partially', 'methylated', 'and', 'that', 'these', 'characteristics', 'are', 'exacerbated', 'when', 'the', 'locus', 'is', 'homozygous', 'for', 'the', 'transgene', 'and', 'or', 'when', 'an', 'unlinked', 'homologous', 'transgene', 'is', 'present'], ['an', 'illustrative', 'list', 'is', 'presented', 'of', 'human', 'and', 'animal', 'studies', 'which', 'each', 'point', 'to', 'the', 'existence', 'of', 'third', 'source', 'in', 'addition', 'to', 'genetic', 'and', 'environmental', 'factors', 'underlying', 'phenotypic', 'differences', 'in', 'development', 'it', 'is', 'argued', 'that', 'this', 'third', 'source', 'may', 'consist', 'of', 'nonlinear', 'epigenetic', 'processes', 'that', 'can', 'create', 'variability', 'at', 'all', 'phenotypical', 'somatic', 'and', 'behavioral', 'levels', 'in', 'quantitative', 'genetic', 'analysis', 'with', 'human', 'subjects', 'these', 'processes', 'are', 'confounded', 'with', 'within', 'family', 'environmental', 'influences', 'preliminary', 'model', 'to', 'quantify', 'these', 'influences', 'is', 'introduced'], ['analysis', 'of', 'the', 'developmental', 'capacities', 'of', 'androgenetic', 'and', 'gynogenetic', 'mouse', 'embryos', 'bearing', 'two', 'paternal', 'or', 'two', 'maternal', 'pronuclei', 'respectively', 'revealed', 'defect', 'in', 'blastocyst', 'formation', 'of', 'androgenetic', 'but', 'not', 'gynogenetic', 'embryos', 'that', 'was', 'function', 'of', 'the', 'maternal', 'genotype', 'androgenetic', 'embryos', 'constructed', 'using', 'fertilized', 'eggs', 'from', 'bl', 'or', 'mice', 'developed', 'to', 'the', 'blastocyst', 'stage', 'at', 'frequencies', 'similar', 'to', 'those', 'previously', 'reported', 'whereas', 'androgenetic', 'embryos', 'constructed', 'with', 'fertilized', 'eggs', 'from', 'dba', 'mice', 'developed', 'poorly', 'the', 'majority', 'failing', 'to', 'progress', 'beyond', 'the', 'cell', 'stage', 'and', 'unable', 'to', 'form', 'blastocoel', 'like', 'cavity', 'regardless', 'of', 'whether', 'the', 'male', 'pronuclei', 'were', 'of', 'bl', 'or', 'dba', 'origin', 'this', 'impaired', 'development', 'was', 'observed', 'even', 'in', 'androgenetic', 'embryos', 'constructed', 'by', 'transplanting', 'two', 'male', 'pronuclei', 'from', 'fertilized', 'dba', 'eggs', 'to', 'enucleated', 'bl', 'eggs', 'indicating', 'that', 'the', 'defect', 'cannot', 'be', 'explained', 'as', 'the', 'lack', 'of', 'some', 'essential', 'component', 'in', 'the', 'dba', 'cytoplasm', 'that', 'might', 'otherwise', 'compensate', 'for', 'androgeny', 'rather', 'the', 'dba', 'egg', 'cytoplasm', 'apparently', 'modifies', 'the', 'incoming', 'male', 'pronuclei', 'differently', 'than', 'does', 'bl', 'egg', 'cytoplasm', 'several', 'specific', 'alterations', 'in', 'the', 'protein', 'synthesis', 'pattern', 'of', 'dba', 'androgenones', 'were', 'observed', 'that', 'reflect', 'defect', 'in', 'the', 'regulatory', 'mechanisms', 'that', 'normally', 'modulate', 'the', 'synthesis', 'of', 'these', 'proteins', 'between', 'the', 'cell', 'and', 'blastocyst', 'stages', 'these', 'results', 'are', 'consistent', 'with', 'model', 'in', 'which', 'cytoplasmic', 'factors', 'present', 'in', 'the', 'egg', 'direct', 'strain', 'dependent', 'modification', 'of', 'paternal', 'genome', 'function', 'in', 'response', 'to', 'epigenetic', 'modifications', 'genomic', 'imprinting', 'established', 'during', 'gametogenesis', 'and', 'indicate', 'that', 'preimplantation', 'development', 'can', 'be', 'affected', 'by', 'these', 'modifications', 'at', 'both', 'the', 'morphological', 'and', 'biochemical', 'levels'], ['functional', 'consequences', 'of', 'either', 'suppressing', 'or', 'intensifying', 'spontaneous', 'neuronal', 'firing', 'have', 'been', 'studied', 'in', 'developing', 'rat', 'cerebral', 'cortex', 'cultures', 'using', 'respectively', 'tetrodotoxin', 'ttx', 'and', 'picrotoxin', 'ptx', 'added', 'chronically', 'to', 'the', 'growth', 'medium', 'simple', 'measures', 'derived', 'from', 'the', 'interspike', 'interval', 'histogram', 'were', 'able', 'to', 'powerfully', 'discriminate', 'between', 'age', 'and', 'treatment', 'groups', 'after', 'return', 'to', 'control', 'medium', 'most', 'ttx', 'treated', 'neurons', 'spontaneously', 'displayed', 'stereotyped', 'clustering', 'of', 'action', 'potentials', 'phasic', 'firing', 'which', 'closely', 'resembled', 'the', 'characteristic', 'firing', 'patterns', 'seen', 'acutely', 'in', 'the', 'presence', 'of', 'ptx', 'the', 'ttx', 'syndrome', 'thus', 'suggests', 'that', 'gabaergic', 'synaptic', 'inhibition', 'is', 'ineffective', 'in', 'cortical', 'networks', 'grown', 'under', 'conditions', 'which', 'prevent', 'the', 'expression', 'of', 'bioelectric', 'activity', 'in', 'contrast', 'after', 'return', 'to', 'control', 'medium', 'neurons', 'which', 'had', 'been', 'partially', 'disinhibited', 'throughout', 'development', 'by', 'continuous', 'exposure', 'to', 'ptx', 'had', 'even', 'less', 'phasic', 'firing', 'than', 'was', 'measured', 'in', 'age', 'matched', 'controls', 'based', 'upon', 'these', 'and', 'previous', 'findings', 'two', 'main', 'factor', 'model', 'is', 'put', 'forth', 'which', 'can', 'economically', 'account', 'for', 'the', 'major', 'effects', 'the', 'working', 'hypothesis', 'embodied', 'in', 'this', 'model', 'is', 'that', 'phasic', 'neuronal', 'discharges', 'not', 'only', 'accelerate', 'the', 'maturation', 'of', 'excitatory', 'connections', 'within', 'the', 'neocortex', 'but', 'even', 'more', 'important', 'are', 'crucial', 'for', 'the', 'development', 'of', 'adequate', 'inhibitory', 'synaptic', 'transmission'], ['differences', 'in', 'the', 'activity', 'of', 'maternally', 'and', 'paternally', 'derived', 'genomes', 'in', 'maize', 'endosperm', 'have', 'been', 'observed', 'at', 'three', 'levels', 'of', 'genetic', 'manipulation', 'when', 'the', 'balance', 'of', 'entire', 'chromosome', 'sets', 'departs', 'from', 'the', 'standard', 'ratio', 'of', 'two', 'of', 'maternal', 'origin', 'to', 'one', 'of', 'paternal', 'origin', 'development', 'is', 'impaired', 'often', 'leading', 'to', 'seed', 'failure', 'at', 'the', 'level', 'of', 'individual', 'chromosomes', 'absence', 'of', 'paternal', 'representative', 'for', 'of', 'the', 'chromosome', 'arms', 'tested', 'causes', 'marked', 'reduction', 'in', 'kernel', 'size', 'replacement', 'of', 'the', 'missing', 'arms', 'by', 'ones', 'of', 'maternal', 'origin', 'does', 'not', 'complement', 'this', 'defect', 'at', 'the', 'gene', 'level', 'some', 'alleles', 'of', 'confer', 'solid', 'coloration', 'on', 'the', 'aleurone', 'layer', 'when', 'transmitted', 'maternally', 'but', 'patchy', 'coloration', 'mottled', 'when', 'transmitted', 'via', 'pollen', 'in', 'contrast', 'with', 'the', 'endosperm', 'no', 'effect', 'of', 'parentage', 'on', 'phenotype', 'has', 'been', 'detected', 'in', 'embryonic', 'and', 'seedling', 'tissues', 'furthermore', 'gynogenetic', 'and', 'androgenetic', 'haploid', 'plants', 'are', 'viable', 'in', 'maize', 'and', 'are', 'similar', 'in', 'appearance', 'the', 'detection', 'of', 'parental', 'effects', 'in', 'the', 'endosperm', 'but', 'not', 'the', 'embryo', 'points', 'to', 'the', 'few', 'cell', 'divisions', 'of', 'the', 'gametophytes', 'as', 'critical', 'stage', 'in', 'imprinting', 'chromosomally', 'based', 'epigenetic', 'variation', 'originating', 'at', 'this', 'stage', 'would', 'be', 'reflected', 'as', 'imprinting', 'effects', 'separate', 'fertilization', 'establishes', 'line', 'of', 'genetic', 'descent', 'in', 'the', 'embryo', 'that', 'appears', 'to', 'be', 'relatively', 'free', 'of', 'imprinted', 'genes'], [], ['herein', 'we', 'describe', 'general', 'multivariate', 'quantitative', 'genetic', 'model', 'that', 'incorporates', 'two', 'potentially', 'important', 'developmental', 'phenomena', 'maternal', 'effects', 'and', 'epigenetic', 'effects', 'maternal', 'and', 'epigenetic', 'effects', 'are', 'defined', 'as', 'partial', 'regression', 'coefficients', 'and', 'phenotypic', 'variances', 'are', 'derived', 'in', 'terms', 'of', 'age', 'specific', 'genetic', 'and', 'environmental', 'variances', 'as', 'starting', 'point', 'the', 'traditional', 'quantitative', 'genetic', 'model', 'of', 'additive', 'gene', 'effects', 'and', 'random', 'environmental', 'effects', 'is', 'cast', 'in', 'developmental', 'time', 'framework', 'from', 'this', 'framework', 'we', 'first', 'extend', 'maternal', 'effects', 'model', 'to', 'include', 'multiple', 'developmental', 'ages', 'for', 'the', 'occurrence', 'of', 'maternal', 'effects', 'an', 'example', 'of', 'maternal', 'effects', 'occurring', 'at', 'multiple', 'developmental', 'ages', 'is', 'prenatal', 'and', 'postnatal', 'maternal', 'effects', 'in', 'mammals', 'subsequently', 'model', 'of', 'intrinsic', 'and', 'epigenetic', 'effects', 'in', 'the', 'absence', 'of', 'maternal', 'effects', 'is', 'described', 'it', 'is', 'shown', 'that', 'genetic', 'correlations', 'can', 'arise', 'through', 'epigenetic', 'effects', 'and', 'in', 'the', 'absence', 'of', 'other', 'developmental', 'effects', 'epigenetic', 'effects', 'are', 'in', 'general', 'confounded', 'with', 'age', 'specific', 'intrinsic', 'genetic', 'effects', 'finally', 'the', 'two', 'effects', 'are', 'incorporated', 'into', 'the', 'basic', 'quantitative', 'genetic', 'model', 'for', 'this', 'more', 'biologically', 'realistic', 'model', 'combining', 'maternal', 'and', 'epigenetic', 'effects', 'it', 'is', 'shown', 'that', 'the', 'phenotypic', 'regressions', 'of', 'offspring', 'on', 'mother', 'and', 'offspring', 'on', 'father', 'can', 'be', 'used', 'in', 'some', 'cases', 'to', 'estimate', 'simultaneously', 'maternal', 'effects', 'and', 'epigenetic', 'effects'], ['butylated', 'hydroxyanisole', 'bha', 'is', 'synthetic', 'food', 'antioxidant', 'used', 'to', 'prevent', 'oils', 'fats', 'and', 'shortenings', 'from', 'oxidative', 'deterioration', 'and', 'rancidity', 'this', 'review', 'depicts', 'the', 'current', 'knowledge', 'on', 'bha', 'the', 'physical', 'and', 'chemical', 'characteristics', 'of', 'bha', 'are', 'summarized', 'and', 'its', 'function', 'as', 'food', 'antioxidant', 'is', 'made', 'clear', 'the', 'toxicological', 'characteristics', 'of', 'bha', 'and', 'its', 'metabolic', 'fate', 'in', 'man', 'and', 'animal', 'are', 'briefly', 'reviewed', 'special', 'emphasis', 'is', 'laid', 'on', 'the', 'carcinogenicity', 'of', 'bha', 'in', 'the', 'forestomach', 'of', 'rodents', 'and', 'to', 'related', 'events', 'in', 'the', 'forestomach', 'and', 'other', 'tissues', 'in', 'experimental', 'animals', 'at', 'present', 'there', 'is', 'sufficient', 'evidence', 'for', 'carcinogenicity', 'of', 'bha', 'but', 'there', 'is', 'hardly', 'any', 'indication', 'that', 'bha', 'is', 'genotoxic', 'therefore', 'risk', 'assessment', 'for', 'this', 'epigenetic', 'carcinogen', 'is', 'based', 'on', 'non', 'stochastic', 'principles', 'however', 'the', 'mechanism', 'underlying', 'the', 'tumorigenicity', 'of', 'bha', 'is', 'not', 'known', 'in', 'the', 'last', 'part', 'of', 'this', 'review', 'an', 'attempt', 'is', 'made', 'to', 'unravel', 'the', 'unknown', 'mechanism', 'of', 'carcinogenicity', 'it', 'is', 'hypothesized', 'that', 'bha', 'gives', 'rise', 'to', 'tumor', 'formation', 'in', 'rodent', 'forestomach', 'by', 'inducing', 'heritable', 'changes', 'in', 'dna', 'evidence', 'is', 'being', 'provided', 'that', 'reactive', 'oxygen', 'species', 'in', 'particular', 'may', 'play', 'crucial', 'role', 'the', 'key', 'question', 'with', 'respect', 'to', 'risk', 'assessment', 'for', 'bha', 'is', 'whether', 'or', 'not', 'the', 'underlying', 'mechanism', 'is', 'thresholded', 'which', 'is', 'important', 'for', 'the', 'choice', 'of', 'the', 'appropriate', 'model', 'to', 'assess', 'the', 'risk', 'if', 'any', 'for', 'man', 'and', 'to', 'manage', 'any', 'potential', 'risk']]\n"
     ]
    }
   ],
   "source": [
    "print(data)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 13,
   "metadata": {},
   "outputs": [],
   "source": [
    "###MAY NEED CLEANING STEP, MAY NOT BECAUSE ABSTRACTS ARE SO CLEAN###"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 14,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "[['the', 'american', 'college', 'of', 'medical', 'genetics', 'and', 'genomics', 'acmg', 'previously', 'developed', 'guidance', 'for', 'the', 'interpretation', 'of', 'sequence', 'variants', 'in', 'the', 'past', 'decade', 'sequencing', 'technology', 'has', 'evolved', 'rapidly', 'with', 'the', 'advent', 'of', 'high', 'throughput', 'next', 'generation', 'sequencing', 'by', 'adopting', 'and', 'leveraging', 'next', 'generation', 'sequencing', 'clinical', 'laboratories', 'are', 'now', 'performing', 'an', 'ever', 'increasing', 'catalogue', 'of', 'genetic', 'testing', 'spanning', 'genotyping', 'single', 'genes', 'gene', 'panels', 'exomes', 'genomes', 'transcriptomes', 'and', 'epigenetic', 'assays', 'for', 'genetic', 'disorders', 'by', 'virtue', 'of', 'increased', 'complexity', 'this', 'shift', 'in', 'genetic', 'testing', 'has', 'been', 'accompanied', 'by', 'new', 'challenges', 'in', 'sequence', 'interpretation', 'in', 'this', 'context', 'the', 'acmg', 'convened', 'workgroup', 'in', 'comprising', 'representatives', 'from', 'the', 'acmg', 'the', 'association', 'for', 'molecular', 'pathology', 'amp', 'and', 'the', 'college', 'of', 'american', 'pathologists', 'to', 'revisit', 'and', 'revise', 'the', 'standards', 'and', 'guidelines', 'for', 'the', 'interpretation', 'of', 'sequence', 'variants', 'the', 'group', 'consisted', 'of', 'clinical', 'laboratory', 'directors', 'and', 'clinicians', 'this', 'report', 'represents', 'expert', 'opinion', 'of', 'the', 'workgroup', 'with', 'input', 'from', 'acmg', 'amp', 'and', 'college', 'of', 'american', 'pathologists', 'stakeholders', 'these', 'recommendations', 'primarily', 'apply', 'to', 'the', 'breadth', 'of', 'genetic', 'tests', 'used', 'in', 'clinical', 'laboratories', 'including', 'genotyping', 'single', 'genes', 'panels', 'exomes', 'and', 'genomes', 'this', 'report', 'recommends', 'the', 'use', 'of', 'specific', 'standard', 'terminology', 'pathogenic', 'likely', 'pathogenic', 'uncertain', 'significance', 'likely', 'benign', 'and', 'benign', 'to', 'describe', 'variants', 'identified', 'in', 'genes', 'that', 'cause', 'mendelian', 'disorders', 'moreover', 'this', 'recommendation', 'describes', 'process', 'for', 'classifying', 'variants', 'into', 'these', 'five', 'categories', 'based', 'on', 'criteria', 'using', 'typical', 'types', 'of', 'variant', 'evidence', 'population', 'data', 'computational', 'data', 'functional', 'data', 'segregation', 'data', 'because', 'of', 'the', 'increased', 'complexity', 'of', 'analysis', 'and', 'interpretation', 'of', 'clinical', 'genetic', 'testing', 'described', 'in', 'this', 'report', 'the', 'acmg', 'strongly', 'recommends', 'that', 'clinical', 'molecular', 'genetic', 'testing', 'should', 'be', 'performed', 'in', 'clinical', 'laboratory', 'improvement', 'amendments', 'approved', 'laboratory', 'with', 'results', 'interpreted', 'by', 'board', 'certified', 'clinical', 'molecular', 'geneticist', 'or', 'molecular', 'genetic', 'pathologist', 'or', 'the', 'equivalent']]\n"
     ]
    }
   ],
   "source": [
    "def sent_to_words(sentences):\n",
    "    for sentence in sentences:\n",
    "        yield(gensim.utils.simple_preprocess(str(sentence), deacc=True))  # deacc=True removes punctuations\n",
    "\n",
    "data_words = list(sent_to_words(data))\n",
    "\n",
    "print(data_words[:1])"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 15,
   "metadata": {},
   "outputs": [],
   "source": [
    "###THIS STEP NEEDS TO BE CAREFULLY MONITORED TO MAKE SURE IT'S WORKING AS INTENDED###"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 16,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "['the', 'american', 'college', 'of', 'medical', 'genetics', 'and', 'genomics', 'acmg', 'previously', 'developed', 'guidance', 'for', 'the', 'interpretation', 'of', 'sequence', 'variants', 'in', 'the', 'past_decade', 'sequencing', 'technology', 'has', 'evolved', 'rapidly', 'with', 'the', 'advent', 'of', 'high_throughput', 'next_generation_sequencing', 'by', 'adopting', 'and', 'leveraging', 'next_generation_sequencing', 'clinical', 'laboratories', 'are', 'now', 'performing', 'an', 'ever', 'increasing', 'catalogue', 'of', 'genetic', 'testing', 'spanning', 'genotyping', 'single', 'genes', 'gene', 'panels', 'exomes', 'genomes', 'transcriptomes', 'and', 'epigenetic', 'assays', 'for', 'genetic', 'disorders', 'by', 'virtue', 'of', 'increased', 'complexity', 'this', 'shift', 'in', 'genetic', 'testing', 'has_been', 'accompanied', 'by', 'new', 'challenges', 'in', 'sequence', 'interpretation', 'in', 'this', 'context', 'the', 'acmg', 'convened', 'workgroup', 'in', 'comprising', 'representatives', 'from', 'the', 'acmg', 'the', 'association', 'for', 'molecular', 'pathology', 'amp', 'and', 'the', 'college', 'of', 'american', 'pathologists', 'to', 'revisit', 'and', 'revise', 'the', 'standards', 'and', 'guidelines', 'for', 'the', 'interpretation', 'of', 'sequence', 'variants', 'the', 'group', 'consisted', 'of', 'clinical', 'laboratory', 'directors', 'and', 'clinicians', 'this', 'report', 'represents', 'expert', 'opinion', 'of', 'the', 'workgroup', 'with', 'input', 'from', 'acmg', 'amp', 'and', 'college', 'of', 'american', 'pathologists', 'stakeholders', 'these', 'recommendations', 'primarily', 'apply', 'to', 'the', 'breadth', 'of', 'genetic', 'tests', 'used', 'in', 'clinical', 'laboratories', 'including', 'genotyping', 'single', 'genes', 'panels', 'exomes', 'and', 'genomes', 'this', 'report', 'recommends', 'the', 'use', 'of', 'specific', 'standard', 'terminology', 'pathogenic', 'likely', 'pathogenic', 'uncertain', 'significance', 'likely', 'benign', 'and', 'benign', 'to', 'describe', 'variants', 'identified', 'in', 'genes', 'that', 'cause', 'mendelian', 'disorders', 'moreover', 'this', 'recommendation', 'describes', 'process', 'for', 'classifying', 'variants', 'into', 'these', 'five', 'categories', 'based', 'on', 'criteria', 'using', 'typical', 'types', 'of', 'variant', 'evidence', 'population', 'data', 'computational', 'data', 'functional', 'data', 'segregation', 'data', 'because', 'of', 'the', 'increased', 'complexity', 'of', 'analysis', 'and', 'interpretation', 'of', 'clinical', 'genetic', 'testing', 'described', 'in', 'this', 'report', 'the', 'acmg', 'strongly', 'recommends', 'that', 'clinical', 'molecular', 'genetic', 'testing', 'should', 'be', 'performed', 'in', 'clinical', 'laboratory', 'improvement', 'amendments', 'approved', 'laboratory', 'with', 'results', 'interpreted', 'by', 'board', 'certified', 'clinical', 'molecular', 'geneticist', 'or', 'molecular', 'genetic', 'pathologist', 'or', 'the', 'equivalent']\n"
     ]
    }
   ],
   "source": [
    "# Build the bigram and trigram models\n",
    "bigram = gensim.models.Phrases(data_words, min_count=5, threshold=100) # higher threshold fewer phrases.\n",
    "trigram = gensim.models.Phrases(bigram[data_words], threshold=100)  \n",
    "\n",
    "# Faster way to get a sentence clubbed as a trigram/bigram\n",
    "bigram_mod = gensim.models.phrases.Phraser(bigram)\n",
    "trigram_mod = gensim.models.phrases.Phraser(trigram)\n",
    "\n",
    "# See trigram example\n",
    "print(trigram_mod[bigram_mod[data_words[0]]])"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 17,
   "metadata": {},
   "outputs": [],
   "source": [
    "# Define functions for stopwords, bigrams, trigrams and lemmatization\n",
    "def remove_stopwords(texts):\n",
    "    return [[word for word in simple_preprocess(str(doc)) if word not in stop_words] for doc in texts]\n",
    "\n",
    "def make_bigrams(texts):\n",
    "    return [bigram_mod[doc] for doc in texts]\n",
    "\n",
    "def make_trigrams(texts):\n",
    "    return [trigram_mod[bigram_mod[doc]] for doc in texts]\n",
    "\n",
    "def lemmatization(texts, allowed_postags=['NOUN', 'ADJ', 'VERB', 'ADV']):\n",
    "    \"\"\"https://spacy.io/api/annotation\"\"\"\n",
    "    texts_out = []\n",
    "    for sent in texts:\n",
    "        doc = nlp(\" \".join(sent)) \n",
    "        texts_out.append([token.lemma_ for token in doc if token.pos_ in allowed_postags])\n",
    "    return texts_out"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 18,
   "metadata": {},
   "outputs": [],
   "source": [
    "###en_core_web_sm needs to be explicitly loaded and I believe you need to use \"python -m spacy download en\" in cmd.exe to properly load everything.###"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 19,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "[['genomic', 'previously', 'develop', 'guidance', 'interpretation', 'sequence', 'variant', 'past_decade', 'sequence', 'technology', 'evolve', 'rapidly', 'advent', 'high_throughput', 'next_generation', 'sequence', 'adopt', 'leverage', 'next_generation', 'sequence', 'clinical', 'laboratory', 'perform', 'ever', 'increase', 'catalogue', 'genetic', 'testing', 'span', 'genotype', 'single', 'gene', 'gene', 'panel', 'exome', 'genome', 'transcriptome', 'epigenetic', 'assay', 'genetic', 'disorder', 'virtue', 'increase', 'complexity', 'shift', 'genetic', 'testing', 'accompany', 'new', 'challenge', 'sequence', 'interpretation', 'context', 'convene', 'comprise', 'representative', 'standard', 'guideline', 'interpretation', 'group', 'consist', 'clinical', 'laboratory', 'director', 'clinician', 'report', 'represent', 'expert', 'opinion', 'workgroup', 'input', 'pathologist', 'stakeholder', 'recommendation', 'primarily', 'apply', 'breadth', 'genetic', 'test', 'use', 'clinical', 'laboratory', 'include', 'genotype', 'single', 'gene', 'panel', 'exome', 'genome', 'report', 'recommend', 'use', 'specific', 'standard', 'terminology', 'pathogenic', 'likely', 'pathogenic', 'uncertain', 'significance', 'likely', 'benign', 'benign', 'describe', 'variant', 'identify', 'gene', 'cause', 'mendelian', 'disorder', 'recommendation', 'describe', 'process', 'classify', 'variant', 'category', 'base', 'criterion', 'use', 'typical', 'type', 'variant', 'evidence', 'population', 'datum', 'computational', 'datum', 'functional', 'datum', 'segregation', 'datum', 'increase', 'complexity', 'analysis', 'interpretation', 'clinical', 'genetic', 'testing', 'describe', 'report', 'strongly', 'recommend', 'clinical', 'molecular', 'genetic', 'testing', 'perform', 'clinical', 'laboratory', 'improvement', 'amendment', 'approve', 'laboratory', 'result', 'interpret', 'board', 'certify', 'clinical', 'molecular', 'geneticist', 'molecular', 'genetic', 'pathologist', 'equivalent']]\n"
     ]
    }
   ],
   "source": [
    "# Remove Stop Words\n",
    "data_words_nostops = remove_stopwords(data_words)\n",
    "\n",
    "# Form Bigrams\n",
    "data_words_bigrams = make_bigrams(data_words_nostops)\n",
    "\n",
    "# Initialize spacy 'en' model, keeping only tagger component (for efficiency)\n",
    "# python3 -m spacy download en\n",
    "nlp = spacy.load('en_core_web_sm', disable=['parser', 'ner'])\n",
    "\n",
    "# Do lemmatization keeping only noun, adj, vb, adv\n",
    "data_lemmatized = lemmatization(data_words_bigrams, allowed_postags=['NOUN', 'ADJ', 'VERB', 'ADV'])\n",
    "\n",
    "print(data_lemmatized[:1])"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 20,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "[[(0, 1), (1, 1), (2, 1), (3, 1), (4, 1), (5, 1), (6, 1), (7, 1), (8, 1), (9, 2), (10, 1), (11, 1), (12, 1), (13, 1), (14, 1), (15, 1), (16, 1), (17, 1), (18, 7), (19, 1), (20, 2), (21, 1), (22, 1), (23, 1), (24, 1), (25, 1), (26, 1), (27, 4), (28, 3), (29, 1), (30, 1), (31, 2), (32, 1), (33, 1), (34, 1), (35, 1), (36, 1), (37, 2), (38, 1), (39, 1), (40, 4), (41, 7), (42, 1), (43, 2), (44, 1), (45, 2), (46, 1), (47, 1), (48, 1), (49, 1), (50, 1), (51, 1), (52, 1), (53, 3), (54, 1), (55, 1), (56, 4), (57, 5), (58, 1), (59, 2), (60, 1), (61, 3), (62, 1), (63, 2), (64, 1), (65, 2), (66, 1), (67, 2), (68, 2), (69, 2), (70, 1), (71, 1), (72, 1), (73, 1), (74, 1), (75, 2), (76, 2), (77, 3), (78, 1), (79, 1), (80, 1), (81, 1), (82, 5), (83, 1), (84, 1), (85, 2), (86, 1), (87, 1), (88, 1), (89, 2), (90, 1), (91, 1), (92, 1), (93, 1), (94, 4), (95, 1), (96, 1), (97, 1), (98, 1), (99, 3), (100, 4), (101, 1), (102, 1)]]\n"
     ]
    }
   ],
   "source": [
    "# Create Dictionary\n",
    "id2word = corpora.Dictionary(data_lemmatized)\n",
    "\n",
    "# Create Corpus\n",
    "texts = data_lemmatized\n",
    "\n",
    "# Term Document Frequency\n",
    "corpus = [id2word.doc2bow(text) for text in texts]\n",
    "\n",
    "# View\n",
    "print(corpus[:1])"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 21,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "'accompany'"
      ]
     },
     "execution_count": 21,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "#to call an individual dictionary term switch value as appropriate\n",
    "id2word[0]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 22,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "[[('accompany', 1),\n",
       "  ('adopt', 1),\n",
       "  ('advent', 1),\n",
       "  ('amendment', 1),\n",
       "  ('analysis', 1),\n",
       "  ('apply', 1),\n",
       "  ('approve', 1),\n",
       "  ('assay', 1),\n",
       "  ('base', 1),\n",
       "  ('benign', 2),\n",
       "  ('board', 1),\n",
       "  ('breadth', 1),\n",
       "  ('catalogue', 1),\n",
       "  ('category', 1),\n",
       "  ('cause', 1),\n",
       "  ('certify', 1),\n",
       "  ('challenge', 1),\n",
       "  ('classify', 1),\n",
       "  ('clinical', 7),\n",
       "  ('clinician', 1),\n",
       "  ('complexity', 2),\n",
       "  ('comprise', 1),\n",
       "  ('computational', 1),\n",
       "  ('consist', 1),\n",
       "  ('context', 1),\n",
       "  ('convene', 1),\n",
       "  ('criterion', 1),\n",
       "  ('datum', 4),\n",
       "  ('describe', 3),\n",
       "  ('develop', 1),\n",
       "  ('director', 1),\n",
       "  ('disorder', 2),\n",
       "  ('epigenetic', 1),\n",
       "  ('equivalent', 1),\n",
       "  ('ever', 1),\n",
       "  ('evidence', 1),\n",
       "  ('evolve', 1),\n",
       "  ('exome', 2),\n",
       "  ('expert', 1),\n",
       "  ('functional', 1),\n",
       "  ('gene', 4),\n",
       "  ('genetic', 7),\n",
       "  ('geneticist', 1),\n",
       "  ('genome', 2),\n",
       "  ('genomic', 1),\n",
       "  ('genotype', 2),\n",
       "  ('group', 1),\n",
       "  ('guidance', 1),\n",
       "  ('guideline', 1),\n",
       "  ('high_throughput', 1),\n",
       "  ('identify', 1),\n",
       "  ('improvement', 1),\n",
       "  ('include', 1),\n",
       "  ('increase', 3),\n",
       "  ('input', 1),\n",
       "  ('interpret', 1),\n",
       "  ('interpretation', 4),\n",
       "  ('laboratory', 5),\n",
       "  ('leverage', 1),\n",
       "  ('likely', 2),\n",
       "  ('mendelian', 1),\n",
       "  ('molecular', 3),\n",
       "  ('new', 1),\n",
       "  ('next_generation', 2),\n",
       "  ('opinion', 1),\n",
       "  ('panel', 2),\n",
       "  ('past_decade', 1),\n",
       "  ('pathogenic', 2),\n",
       "  ('pathologist', 2),\n",
       "  ('perform', 2),\n",
       "  ('population', 1),\n",
       "  ('previously', 1),\n",
       "  ('primarily', 1),\n",
       "  ('process', 1),\n",
       "  ('rapidly', 1),\n",
       "  ('recommend', 2),\n",
       "  ('recommendation', 2),\n",
       "  ('report', 3),\n",
       "  ('represent', 1),\n",
       "  ('representative', 1),\n",
       "  ('result', 1),\n",
       "  ('segregation', 1),\n",
       "  ('sequence', 5),\n",
       "  ('shift', 1),\n",
       "  ('significance', 1),\n",
       "  ('single', 2),\n",
       "  ('span', 1),\n",
       "  ('specific', 1),\n",
       "  ('stakeholder', 1),\n",
       "  ('standard', 2),\n",
       "  ('strongly', 1),\n",
       "  ('technology', 1),\n",
       "  ('terminology', 1),\n",
       "  ('test', 1),\n",
       "  ('testing', 4),\n",
       "  ('transcriptome', 1),\n",
       "  ('type', 1),\n",
       "  ('typical', 1),\n",
       "  ('uncertain', 1),\n",
       "  ('use', 3),\n",
       "  ('variant', 4),\n",
       "  ('virtue', 1),\n",
       "  ('workgroup', 1)]]"
      ]
     },
     "execution_count": 22,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "# Human readable format of corpus (term-frequency)\n",
    "[[(id2word[id], freq) for id, freq in cp] for cp in corpus[:1]]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 23,
   "metadata": {},
   "outputs": [],
   "source": [
    "###Loads of room for customization at this step. Read carefully and adjust accordingly.###"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 24,
   "metadata": {},
   "outputs": [],
   "source": [
    "# Build LDA model\n",
    "lda_model = gensim.models.ldamodel.LdaModel(corpus=corpus,\n",
    "                                           id2word=id2word,\n",
    "                                           num_topics=14, \n",
    "                                           random_state=100,\n",
    "                                           update_every=1,\n",
    "                                           chunksize=100,\n",
    "                                           passes=10,\n",
    "                                           alpha='auto',\n",
    "                                           per_word_topics=True)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 25,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "[(0,\n",
      "  '0.019*\"locus\" + 0.014*\"study\" + 0.013*\"complex\" + 0.012*\"mechanism\" + '\n",
      "  '0.011*\"gene\" + 0.011*\"growth\" + 0.011*\"cell\" + 0.009*\"induce\" + '\n",
      "  '0.009*\"appear\" + 0.009*\"bind\"'),\n",
      " (1,\n",
      "  '0.018*\"age\" + 0.011*\"mortality\" + 0.010*\"cyclin\" + 0.007*\"blood\" + '\n",
      "  '0.007*\"cki\" + 0.006*\"chronological\" + 0.006*\"apparent\" + 0.006*\"improve\" + '\n",
      "  '0.006*\"estimate\" + 0.004*\"racial\"'),\n",
      " (2,\n",
      "  '0.019*\"cardiac\" + 0.013*\"localization\" + 0.012*\"hypertrophy\" + '\n",
      "  '0.010*\"fibrosis\" + 0.009*\"partition\" + 0.008*\"heart\" + 0.008*\"pathological\" '\n",
      "  '+ 0.007*\"spectrometry\" + 0.006*\"supply\" + 0.006*\"physiological\"'),\n",
      " (3,\n",
      "  '0.042*\"methylation\" + 0.032*\"dna\" + 0.031*\"gene\" + 0.023*\"imprint\" + '\n",
      "  '0.021*\"epigenetic\" + 0.018*\"chromosome\" + 0.015*\"modification\" + '\n",
      "  '0.015*\"silence\" + 0.011*\"development\" + 0.011*\"maternal\"'),\n",
      " (4,\n",
      "  '0.123*\"cell\" + 0.030*\"stem\" + 0.016*\"somatic\" + 0.015*\"embryonic\" + '\n",
      "  '0.014*\"early\" + 0.012*\"development\" + 0.012*\"pluripotent\" + 0.012*\"adult\" + '\n",
      "  '0.011*\"differentiate\" + 0.011*\"epigenetic\"'),\n",
      " (5,\n",
      "  '0.026*\"extensive\" + 0.025*\"synthesis\" + 0.022*\"assembly\" + 0.019*\"complexe\" '\n",
      "  '+ 0.016*\"modular\" + 0.010*\"cooperation\" + 0.008*\"incrna\" + 0.008*\"kinase\" + '\n",
      "  '0.007*\"lncrna\" + 0.007*\"abundance\"'),\n",
      " (6,\n",
      "  '0.035*\"gene\" + 0.022*\"epigenetic\" + 0.019*\"cancer\" + 0.017*\"methylation\" + '\n",
      "  '0.014*\"dna\" + 0.013*\"cell\" + 0.012*\"expression\" + 0.008*\"sequence\" + '\n",
      "  '0.008*\"genetic\" + 0.008*\"tumor\"'),\n",
      " (7,\n",
      "  '0.044*\"age\" + 0.031*\"effect\" + 0.024*\"replication\" + 0.017*\"model\" + '\n",
      "  '0.015*\"rate\" + 0.013*\"measure\" + 0.013*\"review\" + 0.012*\"assay\" + '\n",
      "  '0.011*\"appropriate\" + 0.010*\"disease\"'),\n",
      " (8,\n",
      "  '0.030*\"hybrid\" + 0.018*\"transform\" + 0.010*\"type\" + 0.009*\"speciation\" + '\n",
      "  '0.009*\"hybridization\" + 0.009*\"functionally\" + 0.008*\"alternative\" + '\n",
      "  '0.008*\"gap\" + 0.008*\"evolution\" + 0.007*\"length\"'),\n",
      " (9,\n",
      "  '0.026*\"cell\" + 0.020*\"class\" + 0.015*\"response\" + 0.015*\"yeast\" + '\n",
      "  '0.015*\"sirt\" + 0.014*\"beta\" + 0.013*\"cd\" + 0.012*\"cancer\" + 0.012*\"tumor\" + '\n",
      "  '0.011*\"therapy\"'),\n",
      " (10,\n",
      "  '0.047*\"chromatin\" + 0.041*\"histone\" + 0.033*\"domain\" + 0.025*\"protein\" + '\n",
      "  '0.015*\"structure\" + 0.013*\"cell\" + 0.012*\"repression\" + '\n",
      "  '0.012*\"transcription\" + 0.011*\"chromosomal\" + 0.011*\"nucleosome\"'),\n",
      " (11,\n",
      "  '0.014*\"special\" + 0.007*\"subgroup\" + 0.007*\"gbm\" + 0.007*\"deterioration\" + '\n",
      "  '0.006*\"anatomic\" + 0.002*\"pediatric\" + 0.002*\"glioblastoma\" + 0.002*\"gbms\" '\n",
      "  '+ 0.002*\"olig\" + 0.002*\"transcriptomic\"'),\n",
      " (12,\n",
      "  '0.010*\"inappropriate\" + 0.009*\"learning\" + 0.009*\"machine\" + '\n",
      "  '0.007*\"independently\" + 0.007*\"complementary\" + 0.006*\"application\" + '\n",
      "  '0.006*\"nascent\" + 0.005*\"computer\" + 0.005*\"supervise\" + 0.005*\"assist\"'),\n",
      " (13,\n",
      "  '0.038*\"mirna\" + 0.019*\"clone\" + 0.016*\"subunit\" + 0.015*\"individual\" + '\n",
      "  '0.014*\"nuclear\" + 0.014*\"sexual\" + 0.012*\"suppressor\" + 0.012*\"interval\" + '\n",
      "  '0.011*\"originate\" + 0.011*\"cytoplasmic\"')]\n"
     ]
    }
   ],
   "source": [
    "# Print the Keyword in the topics\n",
    "pprint(lda_model.print_topics())\n",
    "doc_lda = lda_model[corpus]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 26,
   "metadata": {},
   "outputs": [],
   "source": [
    "###Model perplexity and topic coherence provide a convenient measure to judge how good a given topic model is. \n",
    "###In my experience, topic coherence score, in particular, has been more helpful.\n",
    "\n",
    "###Need to learn more about these values."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 27,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "Perplexity:  -8.95312963779006\n",
      "\n",
      "Coherence Score:  0.3903205723456517\n"
     ]
    }
   ],
   "source": [
    "# Compute Perplexity\n",
    "print('\\nPerplexity: ', lda_model.log_perplexity(corpus))  # a measure of how good the model is. lower the better.\n",
    "\n",
    "# Compute Coherence Score\n",
    "coherence_model_lda = CoherenceModel(model=lda_model, texts=data_lemmatized, dictionary=id2word, coherence='c_v')\n",
    "coherence_lda = coherence_model_lda.get_coherence()\n",
    "print('\\nCoherence Score: ', coherence_lda)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 28,
   "metadata": {},
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "C:\\Users\\brady\\Anaconda3\\lib\\site-packages\\pyLDAvis\\_prepare.py:232: FutureWarning: In a future version of pandas all arguments of DataFrame.drop except for the argument 'labels' will be keyword-only\n",
      "  head(R).drop('saliency', 1)\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "\n",
       "<link rel=\"stylesheet\" type=\"text/css\" href=\"https://cdn.rawgit.com/bmabey/pyLDAvis/files/ldavis.v1.0.0.css\">\n",
       "\n",
       "\n",
       "<div id=\"ldavis_el247521862265818952181013123\"></div>\n",
       "<script type=\"text/javascript\">\n",
       "\n",
       "var ldavis_el247521862265818952181013123_data = {\"mdsDat\": {\"x\": [0.2560907308745579, 0.3056149621804882, 0.17408219267860636, 0.15147320914688106, 0.024776445827556573, -0.03693657085615907, -0.05250089966282471, -0.11698005559843776, -0.12671273353867943, -0.12008522629034896, -0.12210055688533179, -0.11398833856142886, -0.11172859241142742, -0.11100456690345242], \"y\": [-0.08814071841684419, -0.003207277559931221, -0.09358300534545111, 0.28321630077774174, -0.055475408768534774, -0.042534318889342206, -0.04910390978077363, 0.014149918374849555, 0.01129368762378135, 0.007116960096062812, 0.007234470626274431, 0.00023044805191026565, 0.004433303304328961, 0.004369549905927113], \"topics\": [1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14], \"cluster\": [1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1], \"Freq\": [48.36761194666697, 19.18403478721898, 11.080312106543104, 7.554944108562848, 3.8438597651858064, 2.983950280013745, 2.940217427020777, 1.5249885570999286, 1.1810644829301407, 0.416468232255639, 0.3250192279936091, 0.24875765727860297, 0.18353689007593502, 0.1652345311539177]}, \"tinfo\": {\"Term\": [\"cell\", \"methylation\", \"chromatin\", \"cancer\", \"dna\", \"histone\", \"tumor\", \"age\", \"effect\", \"imprint\", \"domain\", \"protein\", \"model\", \"modification\", \"genetic\", \"locus\", \"development\", \"chromosome\", \"type\", \"stem\", \"use\", \"human\", \"sequence\", \"silence\", \"gene\", \"include\", \"study\", \"complex\", \"early\", \"epigenetic\", \"mutation\", \"promoter\", \"line\", \"loss\", \"normal\", \"allele\", \"deletion\", \"formation\", \"transcript\", \"non\", \"characterize\", \"mutant\", \"cpg_island\", \"reduce\", \"hypomethylation\", \"progeny\", \"produce\", \"sirtuin\", \"contrast\", \"investigate\", \"genotype\", \"addition\", \"repair\", \"primary\", \"colorectal\", \"particular\", \"defect\", \"possible\", \"examine\", \"origin\", \"small\", \"alteration\", \"cancer\", \"tissue\", \"event\", \"link\", \"transgene\", \"tumor\", \"base\", \"genetic\", \"cause\", \"part\", \"sequence\", \"gene\", \"epigenetic\", \"associate\", \"express\", \"expression\", \"suggest\", \"molecular\", \"use\", \"human\", \"level\", \"pattern\", \"methylation\", \"dna\", \"show\", \"region\", \"change\", \"cell\", \"identify\", \"type\", \"result\", \"specific\", \"mechanism\", \"include\", \"also\", \"imprint\", \"igf\", \"paternal\", \"parental\", \"centromere\", \"cytosine\", \"demethylation\", \"embryo\", \"stable\", \"male\", \"methyl\", \"inherit\", \"fetal\", \"zygote\", \"mecp\", \"recruit\", \"androgenetic\", \"think\", \"chemical\", \"kinetochore\", \"reprogramme\", \"arabidopsis\", \"embryogenesis\", \"silent\", \"maternally\", \"paternally\", \"protect\", \"numerous\", \"layer\", \"partially\", \"maternal\", \"active\", \"division\", \"chromosome\", \"inactivation\", \"element\", \"modification\", \"mammalian\", \"methylation\", \"dna\", \"silence\", \"inheritance\", \"inactive\", \"development\", \"gene\", \"activity\", \"epigenetic\", \"state\", \"methylate\", \"mouse\", \"region\", \"mechanism\", \"expression\", \"specific\", \"genome\", \"process\", \"cell\", \"chromatin\", \"site\", \"role\", \"locus\", \"interact\", \"cadmium\", \"experiment\", \"ink\", \"action\", \"dose\", \"pc\", \"position\", \"trigger\", \"transfer\", \"poorly\", \"antisense\", \"replicate\", \"double\", \"chromodomain\", \"selection\", \"take\", \"arm\", \"screening\", \"assess\", \"ncrna\", \"reactive\", \"newborn\", \"tend\", \"directly\", \"chimera\", \"conversion\", \"selective\", \"innate\", \"homologous\", \"growth\", \"end\", \"appear\", \"alpha\", \"complex\", \"environment\", \"signal\", \"variation\", \"induce\", \"study\", \"underlie\", \"bind\", \"response\", \"pair\", \"mechanism\", \"factor\", \"target\", \"phenotypic\", \"cell\", \"protein\", \"site\", \"gene\", \"result\", \"interaction\", \"lead\", \"effect\", \"epigenetic\", \"development\", \"include\", \"human\", \"nucleosome\", \"es\", \"suv\", \"subject\", \"lys\", \"silencer\", \"isolate\", \"recruitment\", \"member\", \"kap\", \"motif\", \"lysine\", \"tail\", \"self\", \"acetylation\", \"monitor\", \"hypothesis\", \"possibility\", \"pombe\", \"peptide\", \"newly\", \"reinforce\", \"currently\", \"modifying\", \"acetylate\", \"euchromatin\", \"preferential\", \"deacetylase\", \"genetically\", \"reactivation\", \"domain\", \"histone\", \"repressive\", \"unique\", \"chromatin\", \"structure\", \"heterochromatin\", \"chromosomal\", \"repression\", \"family\", \"component\", \"protein\", \"drosophila\", \"structural\", \"contain\", \"set\", \"transcription\", \"bind\", \"complex\", \"modification\", \"cell\", \"function\", \"specific\", \"transcriptional\", \"find\", \"regulation\", \"pluripotent\", \"differentiate\", \"deficient\", \"germ\", \"neural\", \"intrinsic\", \"specification\", \"pgcs\", \"primordial_germ\", \"totipotency\", \"reprogramming\", \"ultimately\", \"reset\", \"notable\", \"klf\", \"hpgcs\", \"ip\", \"tumorigenic\", \"hpgclcs\", \"stem\", \"sox\", \"hpgc\", \"investigator\", \"polymorphic\", \"infinium\", \"totipotent\", \"sodium\", \"comparable\", \"similaritie\", \"attribute\", \"somatic\", \"germline\", \"embryonic\", \"fate\", \"adult\", \"fibroblast\", \"progenitor\", \"microarray\", \"cell\", \"probe\", \"early\", \"appear\", \"stage\", \"differentiation\", \"embryonic_stem\", \"development\", \"mouse\", \"tumour\", \"human\", \"state\", \"epigenetic\", \"generate\", \"gene\", \"yeast\", \"sirt\", \"beta\", \"exert\", \"thymocyte\", \"anti\", \"communication\", \"clear\", \"stimulus\", \"intercellular\", \"specificity\", \"autophagy\", \"immune_checkpoint\", \"gamete\", \"checkpoint\", \"emt\", \"predictive\", \"spermatogenesis\", \"cd\", \"pd\", \"epiblast\", \"subcellular\", \"release\", \"iv\", \"refer\", \"blockade\", \"coa\", \"pgclcs\", \"epilcs\", \"trial\", \"designate\", \"class\", \"acetyl\", \"transient\", \"response\", \"terminal\", \"therapy\", \"cell\", \"oncogenic\", \"inhibitor\", \"immune\", \"tumor\", \"cellular\", \"different\", \"cancer\", \"therapeutic\", \"patient\", \"state\", \"process\", \"positive\", \"factor\", \"replication\", \"appropriate\", \"vertebrate\", \"neuronal\", \"argue\", \"hold\", \"virtually\", \"reaction\", \"simultaneously\", \"defense\", \"vitro\", \"measure\", \"definition\", \"variance\", \"choice\", \"correlated\", \"noncoding_rna\", \"unit\", \"reorganization\", \"fungi\", \"folate\", \"traditional\", \"measurement\", \"batch\", \"psc\", \"nucleic\", \"publish\", \"serine\", \"incorrect\", \"reagent\", \"age\", \"eukaryotic\", \"rate\", \"moreover\", \"compartment\", \"acquire\", \"functionally\", \"quantitative\", \"general\", \"effect\", \"transmission\", \"assay\", \"metabolic\", \"model\", \"review\", \"disease\", \"lncrna\", \"acid\", \"use\", \"include\", \"genetic\", \"human\", \"major\", \"biological\", \"vivo\", \"hybrid\", \"transform\", \"speciation\", \"gap\", \"perturb\", \"ensembl\", \"pacbio\", \"debate\", \"comparison\", \"absent\", \"continue\", \"idea\", \"ensembl_org\", \"etiology\", \"xlink\", \"http\", \"rest\", \"date\", \"histologically\", \"outside\", \"incidence\", \"alternative\", \"track\", \"upgrade\", \"importantly\", \"fragment\", \"ext\", \"com\", \"grch\", \"update\", \"simple\", \"length\", \"clonal\", \"hybridization\", \"functionally\", \"past\", \"divergent\", \"evolution\", \"type\", \"transcriptome\", \"read\", \"assembly\", \"sequence\", \"full\", \"regulatory\", \"research\", \"heterogeneity\", \"mirna\", \"clone\", \"subunit\", \"sexual\", \"interval\", \"originate\", \"rare\", \"dysregulation\", \"trauma\", \"morphological\", \"spontaneous\", \"dysregulate\", \"exception\", \"necrosis\", \"crp\", \"ci\", \"quite\", \"mental\", \"elsevier\", \"cytoplasmic\", \"adulthood\", \"amplification\", \"suppressor\", \"intra\", \"meta\", \"startling\", \"angiogenic\", \"outgrowth\", \"conduct\", \"interleukin\", \"clonal\", \"evolutionary\", \"search\", \"childhood\", \"lncrna\", \"inflammatory\", \"individual\", \"nuclear\", \"heterogeneity\", \"tumor\", \"analysis\", \"oncogene\", \"cardiac\", \"hypertrophy\", \"fibrosis\", \"partition\", \"heart\", \"spectrometry\", \"supply\", \"capillary\", \"programme\", \"continuously\", \"cardiomyocyte\", \"localization\", \"underscore\", \"rarefaction\", \"pump\", \"hemodynamic\", \"impairment\", \"equip\", \"athlete\", \"prolong\", \"maladaptive\", \"critically\", \"myocardial\", \"ltd\", \"infarction\", \"consumption\", \"hypertension\", \"undesirable\", \"demand\", \"immunoaffinity\", \"pathological\", \"monocyte\", \"undetectable\", \"cytokine\", \"mass\", \"physiological\", \"dysfunction\", \"response\", \"cellular\", \"body\", \"maintain\", \"extensive\", \"synthesis\", \"complexe\", \"modular\", \"cooperation\", \"incrna\", \"kinase\", \"abundance\", \"notion\", \"specify\", \"rnp\", \"kingdom\", \"higher_ord\", \"centrality\", \"repertoire\", \"sensory\", \"threonine\", \"assembly\", \"guiding\", \"underscore\", \"mode\", \"allosteric\", \"lncrna\", \"modulator\", \"turn\", \"capacity\", \"scaffold\", \"catalytic\", \"diversity\", \"history\", \"discover\", \"mortality\", \"cyclin\", \"cki\", \"apparent\", \"racial\", \"hispanic\", \"ethnic\", \"cdks\", \"african\", \"metric\", \"orderly\", \"trio\", \"revelation\", \"showcase\", \"thank\", \"interpreting\", \"nuance\", \"chronological\", \"accomplish\", \"individually\", \"adjust\", \"improve\", \"picture\", \"indispensable\", \"cohort\", \"value\", \"blood\", \"estimate\", \"evaluate\", \"robust\", \"age\", \"inappropriate\", \"learning\", \"machine\", \"independently\", \"complementary\", \"nascent\", \"computer\", \"supervise\", \"discriminative\", \"semi\", \"unsupervised\", \"algorithm\", \"generative\", \"non_coding\", \"metabolomic\", \"experience\", \"guiding\", \"foreign\", \"assist\", \"aim\", \"proteomic\", \"guideline\", \"defence\", \"powerful\", \"surveillance\", \"consideration\", \"application\", \"recurrent\", \"regional\", \"neoplasia\", \"myogenesis\", \"hml\", \"method\", \"scaffold\", \"special\", \"gbm\", \"deterioration\", \"anatomic\", \"subgroup\", \"pediatric\", \"glioblastoma\", \"gbms\", \"olig\", \"transcriptomic\", \"dismal\", \"tentative\", \"denominator\", \"final\", \"enumerate\", \"attrition\", \"exhaustion\", \"proteostasis\", \"vulnerability\", \"diabetes\", \"singly\", \"diacetyl\", \"epsilon\", \"cover\", \"experience\", \"side\", \"nutrient\", \"bha\", \"forestomach\", \"carcinogenicity\", \"adult\", \"hallmark\"], \"Freq\": [427.0, 437.0, 151.0, 258.0, 341.0, 102.0, 112.0, 59.0, 97.0, 116.0, 75.0, 145.0, 59.0, 119.0, 118.0, 57.0, 125.0, 110.0, 70.0, 37.0, 115.0, 142.0, 126.0, 133.0, 669.0, 97.0, 117.0, 71.0, 51.0, 434.0, 91.43051625088015, 90.98088775390205, 63.089294876846594, 58.12493385140401, 47.96409051902932, 37.995932687004085, 36.80869580378546, 32.86946111497797, 30.969611537634147, 36.29119528886743, 32.150747228273566, 28.282483147344216, 30.00076975240334, 29.403437802233032, 23.67665567421992, 23.23462667153849, 22.44455072875948, 21.84039768128085, 21.74263761867069, 21.34209012640121, 21.205111753871233, 20.628360080144372, 21.011335897817474, 20.317872669486615, 20.306618158888188, 19.993065153925638, 19.929952006205433, 20.262285525788656, 18.79305296136834, 21.039467674908988, 30.043583086564148, 51.22657276803042, 241.0864470557299, 72.29014525295246, 41.56403249113082, 40.19172909927915, 66.2214743698787, 99.30484617630275, 44.22064292011254, 103.0116116296683, 51.84603878901784, 31.124948980557964, 106.171573446038, 454.4336690703206, 286.28549749516134, 81.63021920446222, 47.8431105293023, 150.55659161302773, 92.8716215376135, 49.49040852886505, 84.81859326080748, 98.18909583268479, 69.42109093841023, 75.00723367838555, 219.37169674650315, 179.33217950378224, 80.6815827452951, 83.32428701459284, 68.95456267791626, 170.64464519635646, 61.735550028990794, 55.44652230754239, 72.79655623543916, 77.47030971935806, 71.54206825210946, 61.15051945087916, 61.57474131012574, 115.66698799310389, 43.85137967967941, 37.58820285395787, 37.08253350006057, 36.16105286034891, 34.28437933888029, 35.857635543549954, 24.934995232447054, 24.90085230746765, 21.533992039406993, 22.03914980860357, 18.331896547200824, 18.198761982632913, 16.445850318122343, 14.875525612760965, 13.641674923987825, 13.427243601816864, 12.33898397020813, 12.135016339764599, 11.835769033142405, 29.351698690288277, 11.470752159449816, 11.235874105496292, 10.91582552699662, 10.666994297764038, 10.55200909275292, 10.39055772013446, 10.425403776644378, 10.169651877169787, 10.042914500824551, 56.07961032111002, 46.10222387178785, 16.796065737718177, 92.28395990338555, 50.23227809369497, 37.7164797433327, 78.00354649618494, 20.000646579784334, 215.8176453259895, 161.00117808183015, 74.31457822377126, 20.753665946745862, 19.525029852724433, 57.211383205844285, 160.87742693245883, 38.80840374907679, 108.3254953146773, 36.924834541735265, 29.046189824875693, 29.28888560689527, 43.798393781467595, 47.367789242809785, 54.91112231111042, 45.44437353594181, 36.766666829056284, 36.16499087281256, 51.316909043124205, 35.91268681627442, 31.271383398036615, 31.47301915403037, 56.98390762695331, 18.389064705762234, 16.123868730699485, 14.010763020747495, 12.23033183158803, 11.343996658673143, 10.95037196241957, 10.715665891482805, 10.7502948926123, 9.636575393552386, 10.72349613849377, 9.029868339652182, 8.424864768186394, 8.138953210938428, 8.02354176063279, 7.832198560023361, 15.771990692314294, 7.413703295093361, 7.41361671228306, 7.0188575063484775, 7.50070390181248, 6.82634406346966, 6.767856687946067, 6.540877102080938, 6.329748354987372, 14.640259907216254, 6.014975860972354, 6.01271646192259, 12.254122863796336, 5.831546741523546, 14.327966705102558, 32.4268484349753, 8.454588784395813, 26.5068436094259, 15.049068204869942, 39.324194525882874, 16.77829184622272, 21.675367495294907, 20.88357319241211, 27.544149655404077, 40.21970266653356, 14.43944963218207, 26.004171299423824, 17.32914680299223, 11.402013326061368, 35.43230270016714, 24.161977705582867, 23.27845097668148, 13.84590247828248, 31.778535592064745, 22.34511164525048, 19.127683419816893, 33.06062361173719, 20.42785900086035, 16.23690327671281, 15.982852819325347, 17.362981441195473, 19.816276510142, 15.245303334261386, 15.114972841142666, 14.79537369903594, 22.042623560334896, 18.521765032632977, 17.057583705710943, 16.776360692300155, 13.048966158744278, 11.859938465812617, 11.75713759051279, 11.57764344837883, 10.48797384859621, 10.161744470482123, 9.370580200715322, 18.67369911329806, 7.680052814575528, 7.434681963682139, 20.78637165708343, 6.742117231770134, 6.740961828832759, 6.740079112233403, 6.14432505831614, 6.087726244514495, 5.501474512970669, 5.395719067142424, 5.392971588300264, 7.11383228407857, 5.011563052061843, 4.992138601543725, 4.802608009157735, 4.351667736457348, 7.968174042685005, 4.042015063908678, 65.41964522598573, 83.0256695633017, 18.257467313508368, 7.662890442476569, 94.56190217095997, 30.79514096607044, 17.639241468965654, 22.263685033287185, 24.529785341664788, 19.586233167210604, 18.26882830719266, 50.489436636718224, 12.01392565907769, 14.278461066477663, 19.235506060798233, 16.23519919290723, 23.386157581346964, 20.449140181736805, 18.948051555470542, 20.7849341971435, 26.43700044308437, 18.342843438102207, 19.323014948640626, 15.685110261410195, 15.781623420805898, 13.777642152048122, 12.208571826290262, 11.656140336259767, 9.582285493780503, 8.919098046029195, 8.002467654311834, 7.756040821310715, 7.6162230220583425, 6.591003104274464, 6.096660970044957, 4.505271982633985, 4.161698521043201, 3.9704837518022047, 3.3646559671478977, 3.0429356220417985, 3.0452477061763754, 3.1376631321865025, 2.8725400988976477, 2.7652526249543454, 2.551720129271113, 30.50841524247139, 2.176471901703044, 1.9298448263395125, 1.9295963112647319, 1.9295722345044894, 1.9284040348651972, 1.9001065708753988, 1.8992615004292623, 1.8990886721997986, 1.8968663395526602, 3.105955230865914, 16.793320002566016, 10.05665678973201, 15.35257715853867, 6.80335965284605, 12.12132241484663, 8.525350999930625, 9.36492431645118, 3.786844341838309, 126.35021501483084, 4.696631689203966, 14.620143994794786, 10.230023766504113, 10.469366787551103, 8.872883062319305, 5.632918252629822, 12.266375121573516, 8.422894611364304, 6.167422806781517, 9.332139874048856, 7.587527337989483, 11.111193142908228, 5.418245090632129, 7.324611023211173, 11.73228734395724, 11.586175352498978, 11.516411854555988, 8.293813026586262, 7.861104203328888, 6.58631760489103, 6.001140892639265, 5.679738184400825, 5.008009048332092, 4.832384122300464, 4.732858605812653, 4.430006546634122, 4.313487433490435, 4.218703765208113, 3.9465108710035413, 3.7054545835607113, 3.6763957738506097, 3.6448709217901194, 10.335411108074975, 3.0989697183421536, 3.0989599104294587, 3.0397523173154397, 3.0365454999741606, 2.9737476556489835, 2.700308971477218, 2.491684305381754, 1.8844701385796092, 1.8844453412154374, 1.8844438607757854, 1.8839962128359993, 3.280264173076231, 15.769917054884475, 3.7062266328392512, 4.194672158446433, 12.011245107151817, 6.611770803828584, 8.625697987780985, 20.67753304909266, 3.8813226718051865, 5.705593692813822, 5.074032955714023, 9.445522534331802, 6.617634825290321, 6.937752511910789, 9.667663244340496, 5.121239734898679, 5.36650565249209, 5.486755843668834, 5.466909069693562, 4.361001774012803, 4.388759277268759, 18.54632256393697, 8.315438618919968, 7.639117554800229, 5.714223954891138, 5.191225457899643, 5.0313432913790335, 4.543720024290146, 4.316556651815412, 4.2230465301167115, 4.057409260576648, 4.050807722343446, 10.543324662391495, 3.1765575752964326, 2.9202505376390016, 2.9189314699233075, 2.9159499830254214, 2.8540730525700155, 2.595221056644968, 2.592033887084198, 2.5900744315032127, 2.3648644972975172, 2.3640583598395297, 2.3586427790880635, 1.8092567281279595, 1.8091409404586942, 1.7814811808704565, 1.7803023347573848, 1.779481245190925, 1.7785821129119528, 1.7785821129119528, 34.47455573303489, 4.713266723167295, 11.43019183618992, 4.1098999137673236, 3.1112292605829883, 4.800756250749769, 5.685324082013651, 6.648652691682475, 6.4252966365845285, 24.211416807873498, 5.537734736770511, 9.388439850174647, 4.578121543714207, 13.626187585089546, 9.973325488817578, 7.724292058368541, 3.7033404144736495, 3.9772054213613646, 7.5119137683736685, 6.762425837845399, 6.8233531270700425, 5.668774650734086, 4.711981571209843, 4.797456575689886, 4.233701548488131, 12.213453133626112, 7.243774807358004, 3.8262443909242103, 3.284191585859632, 2.497874605751489, 2.4097185380241903, 2.4096665218491395, 2.3862495964402215, 2.3624744214475, 2.3565595203201886, 2.0980408326657285, 2.0961915157803803, 1.9375332918385375, 1.8395384118878297, 1.4653558953665742, 1.4653536255698447, 1.4425663796036317, 1.4425663796036317, 1.4408289447820744, 1.4402996092697855, 1.4340212569411654, 3.3929855912000484, 1.1791510783796255, 1.1790285093562334, 1.1712186116843928, 2.7828445587427337, 0.9931678119350097, 0.9931669607612361, 0.993163177766687, 0.9931629886169595, 1.937524401801347, 2.8947167867477877, 2.8332552309067838, 3.7086329818886528, 3.4697504948906976, 2.596575959934249, 1.6306898752144248, 3.091099978082314, 3.958452484427633, 1.8635031149074632, 1.7430989101460017, 1.7972384800371408, 2.5382228907142754, 1.7176554345571082, 1.966976527564239, 1.8191401269793999, 1.761931224460814, 12.01035116631655, 6.0718409838715734, 5.132339326286464, 4.55061353558297, 3.7314717265353865, 3.4977976898117156, 2.914833752042149, 2.636147825067154, 2.556093970383584, 2.512204791811672, 2.096042204427951, 1.8847642441243628, 1.8630115588441518, 1.6309099630347301, 1.2996883368503493, 1.299686871934275, 1.2788526890327059, 1.2778036626318359, 1.045752972655999, 3.421132771974637, 1.697603490136657, 1.7941398480312158, 3.9040048540454952, 1.299701960569841, 1.6488279375070127, 0.4620550753102613, 0.4620550753102613, 0.4620550386873595, 0.46203441999361244, 0.46203434674780874, 2.5157662957717264, 3.0370071666797687, 1.0764519736483078, 1.2247886428829096, 2.1359356779153593, 1.6417753920500815, 4.633894307395205, 4.5555470799382975, 2.4315528118949286, 3.0371917461051416, 1.4487781194387945, 1.3411543977436142, 2.131071735611574, 1.2995573069679887, 1.06615057454695, 0.9465921796318858, 0.873712238463424, 0.8296900926790761, 0.6494795650779999, 0.5322386145771205, 0.5314921333068224, 0.5310440172581175, 0.44786314078862327, 1.492195551882464, 0.3187062034223557, 0.23492531635185487, 0.23492526469582622, 0.23492526469582622, 0.23492516138376887, 0.2349251097277402, 0.23492500641568287, 0.2349249547596542, 0.23492500641568287, 0.2349250322436972, 0.2349249547596542, 0.2349249547596542, 0.23492482561958253, 0.23492469647951086, 0.2349245931674535, 0.2349245156834105, 0.2349238441550378, 0.23468620059514533, 0.8737114119669653, 0.438235438509236, 0.4987139036268625, 0.45683419163156813, 0.6316168070521461, 0.6411295746677445, 0.4389187444564717, 0.6182652216659988, 0.5309654484385117, 0.3578159900203379, 0.25077369588372606, 2.2704274434292153, 2.1491349276754987, 1.651006078556225, 1.3630784378142906, 0.857340214262991, 0.7025929932752351, 0.6943199815447924, 0.5993361978046859, 0.5222132150039351, 0.5229365965907894, 0.1888731827310858, 0.18887302147794563, 0.18887298116466059, 0.18887298116466059, 0.18885429595704206, 0.18885294546199302, 0.1884867395806408, 1.8966897504053861, 0.18885383235426403, 0.18887312226115824, 0.18887316257444328, 0.18885403392068925, 0.6243237022142188, 0.1836564219231459, 0.18887306179123067, 0.41548435723228255, 0.17122025724606676, 0.1857111699052664, 0.15721661126321063, 0.018862941333557703, 0.170463576885765, 0.718833681305195, 0.633650348219822, 0.43710319773401946, 0.42837164892224033, 0.29626217656808396, 0.296261898879451, 0.2962617137536957, 0.2962400231860326, 0.1553891749279197, 0.15538885095784793, 0.15536308762356815, 0.1553629950606905, 0.1553559911362815, 0.15535349193858491, 0.15528636842514104, 0.15528630671655594, 0.155286029027923, 0.4304749243368314, 0.15536060385301784, 0.14569455565561162, 0.15300483233533876, 0.42602755575429163, 0.15514986903489708, 0.14086065992314398, 0.26044015779105256, 0.14604961142713796, 0.47425719632046137, 0.3694740749981068, 0.1553070099468574, 0.1235898462507612, 1.1998244567233196, 0.47974114245650196, 0.4425126591538379, 0.4425058297348417, 0.32881822182446135, 0.3284582203844413, 0.26897780540211547, 0.22682872571268414, 0.22682098570448844, 0.11897945151378135, 0.11897945151378135, 0.11897926939594144, 0.118976480716518, 0.11897567256860345, 0.11891583547583168, 0.1187735559134107, 0.11896289017271554, 0.11891574441691173, 0.1189157899463717, 0.22309187252052953, 0.11750577671804979, 0.11531258848397284, 0.11530918515683973, 0.11086974410308531, 0.10884383110495545, 0.10878417613002359, 0.11136513877470518, 0.28984983020886357, 0.08195974354983131, 0.011194184621624864, 0.011194184621624864, 0.011241332511622116, 0.011241506092688269, 0.011483067622368798, 0.011274623083637378, 0.6294516628041675, 0.3051858505274056, 0.3003235420578766, 0.24553868728111344, 0.3051909946779394, 0.10847505071562463, 0.10847498923175371, 0.10847493799519461, 0.10847426167261447, 0.10847390301670075, 0.10847325743605607, 0.10844962713499824, 0.10844919674790178, 0.10844919674790178, 0.10844912501671904, 0.10844783385542967, 0.10844755717801052, 0.10844717802747317, 0.10843629538231991, 0.10842536150060755, 0.10819121042551158, 0.10816400381262843, 0.10813303643630721, 0.1080473074256178, 0.10840860714578121, 0.10843716640382464, 0.10828741218888142, 0.010733282257533878, 0.010370315427828694, 0.010370315427828694, 0.10447511501965258, 0.012386349779835118], \"Total\": [427.0, 437.0, 151.0, 258.0, 341.0, 102.0, 112.0, 59.0, 97.0, 116.0, 75.0, 145.0, 59.0, 119.0, 118.0, 57.0, 125.0, 110.0, 70.0, 37.0, 115.0, 142.0, 126.0, 133.0, 669.0, 97.0, 117.0, 71.0, 51.0, 434.0, 92.01128782119832, 91.56545821708904, 63.66446257622228, 58.70011514988352, 48.56754950914685, 38.5710993112994, 37.38440086385764, 33.44549257799053, 31.544925576115897, 36.99823064260974, 32.78803706195233, 28.857653133937536, 30.626908184327426, 30.039242998797942, 24.256262981977752, 23.809976037192268, 23.019754598085708, 22.415585975513036, 22.31786192750439, 21.918379640878026, 21.780282722638372, 21.204664977864496, 21.60438355278209, 20.893173131771857, 20.881782282603837, 20.568231396581904, 20.505343894278166, 20.851204694851475, 19.368260311784837, 21.68395287913036, 30.97645519271732, 53.267290578143395, 258.3435738193326, 75.64534031900028, 43.46485861880777, 42.38200742155303, 72.37648931557538, 112.3234250205831, 47.518284036955485, 118.2554119470669, 56.98206979362874, 32.92983568399309, 126.05117325747966, 669.5459823209105, 434.2908640042876, 101.86905399820358, 54.74723597240715, 219.53715184883, 125.12936016674026, 57.777936932029185, 115.20213586310261, 142.6346945456044, 91.04595968838657, 101.59851309142614, 437.5157513142017, 341.3148672317128, 118.0771624320037, 129.36732412821226, 97.54134633010042, 427.8585300122807, 82.81857551315127, 70.46592877550202, 120.17527999262988, 144.76048196521725, 155.87761068502505, 97.3170626310262, 102.67045632734165, 116.25827328944358, 44.43490845406325, 38.1717005786621, 37.66603923699958, 36.74529896940394, 34.86924677940231, 36.54907822907844, 25.518490397383236, 25.508440986632134, 22.117806065145245, 22.6968129402909, 18.915391267782926, 18.782279721761753, 17.029345124735844, 15.45905479137028, 14.225291039373973, 14.011733358394505, 12.922479515406632, 12.718519107257894, 12.422038719455974, 30.819170375271334, 12.054297997901452, 11.82369994318464, 11.500274846451674, 11.250514431269831, 11.135594265694401, 10.974211679288365, 11.013850835396848, 10.755318195065207, 10.62641432067403, 62.5794801514904, 51.37777665787181, 18.12743217524095, 110.82463089922324, 62.30730262110759, 47.935966129282804, 119.67324104114331, 23.82075194755901, 437.5157513142017, 341.3148672317128, 133.52165892776986, 25.74115684264453, 23.776260241402664, 125.3533419988846, 669.5459823209105, 75.78135011890936, 434.2908640042876, 72.86121229588016, 49.72817730635449, 53.89846443698597, 129.36732412821226, 155.87761068502505, 219.53715184883, 144.76048196521725, 93.27902786610264, 103.87186155029929, 427.8585300122807, 151.6331840975547, 85.84643135721134, 101.53616209933487, 57.56611474595456, 18.971341493586216, 16.706057638831894, 14.633044209607736, 12.812518783953152, 11.92624235102249, 11.532547429164628, 11.297868445645136, 11.336202011029545, 10.218945344865244, 11.415736782700396, 9.612808015884202, 9.00703801658013, 8.721215411625312, 8.605719581627364, 8.414449345628539, 16.959050345966606, 7.995883616569595, 7.995791714228208, 7.601036061028252, 8.129438173511602, 7.408550875136567, 7.350782643379224, 7.123047196931649, 6.911930048757637, 16.01395583474899, 6.597147934407971, 6.594896373813437, 13.464449162124968, 6.4137202752975915, 16.573491187827617, 43.84631095552615, 9.696711058636648, 37.25468835953653, 19.884085744272408, 71.15289711773924, 24.649670289254132, 35.17000695553096, 37.71053653472656, 62.37409778022579, 117.40716130437343, 23.007658168954332, 61.56675866534375, 31.45112091266634, 15.806532785395023, 155.87761068502505, 81.42110918864896, 79.6097584474443, 27.890261037675362, 427.8585300122807, 145.62634369383258, 85.84643135721134, 669.5459823209105, 120.17527999262988, 52.5153016784887, 50.082053719735775, 97.12166776347037, 434.2908640042876, 125.3533419988846, 97.3170626310262, 142.6346945456044, 22.62772116298646, 19.109357778515335, 17.642671492567185, 17.361456554158824, 13.634260830419038, 12.445357842294229, 12.342233435114023, 12.162740709502557, 11.07319555491401, 10.747405947520528, 9.95579254178705, 19.953495246253702, 8.265221879711552, 8.019773696206284, 22.453940165515622, 7.32721420281962, 7.326055090915665, 7.325175318066621, 6.729614782353013, 6.67286108269903, 6.086569161034891, 5.980806882300693, 5.978063278909753, 7.891041012742974, 5.596826816316517, 5.577687697378879, 5.38770720136089, 4.937727271908594, 9.048619937116342, 4.627457428488019, 75.26221989740198, 102.37623894544217, 22.182394238347435, 8.878423627368724, 151.6331840975547, 45.688017171686234, 23.75775476597863, 35.94572073387091, 41.33940387623969, 33.026623476219065, 30.61901161231476, 145.62634369383258, 18.359449546486513, 24.77267706189498, 42.98020944488214, 34.049881161736714, 74.77804110243771, 61.56675866534375, 71.15289711773924, 119.67324104114331, 427.8585300122807, 115.43645208299793, 144.76048196521725, 67.5636027811531, 78.4054832749929, 72.77375697813387, 12.801137126902047, 12.248717977298833, 10.174647001481295, 9.511401908807501, 8.594768135403761, 8.348577312016632, 8.208567535387553, 7.183334758449682, 6.689023516345939, 5.098068166102128, 4.7540010743457675, 4.562801807584585, 3.9796205475846205, 3.635241734428223, 3.638076335168336, 3.769439237025128, 3.4651828183768156, 3.3575590653621306, 3.1440364416607776, 37.994756296975545, 2.7691383355335932, 2.5221544820259547, 2.5219131139071047, 2.521916524461377, 2.5207313911862257, 2.492414100090637, 2.4915684309493, 2.4914689426629395, 2.489171289759392, 4.116947765006274, 22.688358412048863, 13.599017134543947, 24.864264773172636, 10.214084522068223, 19.97382791706115, 13.740838573223355, 15.676160958323619, 5.672421549782264, 427.8585300122807, 8.092414316235246, 51.4592686791523, 37.25468835953653, 43.69194084236135, 34.66756171205637, 15.270522255988457, 125.3533419988846, 53.89846443698597, 23.319883705817396, 142.6346945456044, 72.86121229588016, 434.2908640042876, 26.710283091652343, 669.5459823209105, 12.326154128874665, 12.180031892784038, 12.110243463721085, 8.887631310464245, 8.454932261208157, 7.182884773810981, 6.619913754516341, 6.273569549556669, 5.601871669180195, 5.429199945799647, 5.32668093262812, 5.02383449317107, 4.907313494779068, 4.812536185083767, 4.540363106144764, 4.299294246167063, 4.270351745321189, 4.2386931389376254, 12.178984289539732, 3.692789321268956, 3.692783078497357, 3.633575879598242, 3.630373129896091, 3.5677242825472026, 3.294128140959093, 3.085510778610496, 2.4782888592314176, 2.478264109236455, 2.478264531030577, 2.478031843377378, 4.345896797104391, 22.434110746967786, 4.991042162901471, 7.08173148260587, 31.45112091266634, 13.814097972124996, 20.832595186154332, 427.8585300122807, 6.975890608779626, 20.25612866395324, 15.26851779275463, 112.3234250205831, 43.95672033380955, 58.748815559507214, 258.3435738193326, 20.852033136531436, 49.99873526368934, 72.86121229588016, 103.87186155029929, 15.994462399108526, 81.42110918864896, 19.14265738317211, 8.911802095156748, 8.235536920651947, 6.310575513576864, 5.787555935183366, 5.627694531136693, 5.1401762310520605, 4.912886380842974, 4.819379490897882, 4.653739691095397, 4.647144091442031, 12.505839827902548, 3.772900424127754, 3.516779596397447, 3.515264553306424, 3.5122835034351363, 3.450411814673303, 3.1915487861611487, 3.188462761876673, 3.1864769459948565, 2.9611927500315685, 2.960397133735347, 2.95497466079076, 2.4055813908348056, 2.405492336643694, 2.3778076556683603, 2.3766292304171914, 2.375815375969795, 2.3749111829875873, 2.3749111829875873, 59.899930043315734, 7.340954136427042, 23.864532727147534, 6.748174278822193, 4.828261674742405, 8.477248242814913, 10.58446369356304, 13.233704967165522, 12.68559217452136, 97.12166776347037, 10.66330371189289, 29.10195558302372, 8.583292598851585, 59.97695782004787, 39.1166980944293, 48.92649676210334, 7.00311007171274, 10.660685614726262, 115.20213586310261, 97.3170626310262, 118.2554119470669, 142.6346945456044, 28.132560354641154, 36.285300840283035, 22.627499986055252, 12.82500260694728, 7.851584479624721, 4.4340637754202925, 3.892109754936102, 3.1056841320500523, 3.0175409414499303, 3.01748588682703, 2.9940679075808845, 2.970324313145908, 2.964377350884069, 2.705866908043646, 2.704010096574126, 2.545358490002875, 2.4473467818401766, 2.073171442247413, 2.073171662230728, 2.0503749216840643, 2.0503749475346673, 2.0486389018183577, 2.0481455697306874, 2.0418440241821156, 4.859976547940469, 1.7869611006340151, 1.7868403431485018, 1.779140294005969, 4.434357198596348, 1.600978675403555, 1.6009786502719745, 1.600979658289377, 1.6009797129069263, 3.359385068648326, 5.286252954894553, 5.913455265789833, 8.481886872396448, 10.58446369356304, 7.141667378155354, 3.30767109598131, 24.713347198961014, 70.46592877550202, 5.204620573721052, 6.446614201198176, 8.836187648988254, 126.05117325747966, 6.60280111484263, 46.10696591854724, 18.15184988797887, 12.994677077825093, 12.624651847389028, 6.685182675711331, 5.74585736041833, 5.164021688195939, 4.344815795101074, 4.111139251401814, 3.528171086834138, 3.2494892346370476, 3.1694415388274186, 3.125546167630392, 2.7093812868855296, 2.4981493570594755, 2.4763511249613175, 2.24425729909934, 1.9130290346882914, 1.9130299844898244, 1.892194316112982, 1.8911455655250713, 1.6590910969029893, 5.570670618635484, 2.9302150366378483, 3.2479003272664375, 7.271225706797261, 2.6039609827288497, 3.8206690129531093, 1.0753904710983888, 1.0753904826643237, 1.0753904749562588, 1.0753713603229142, 1.0753712785319371, 5.913455265789833, 7.413158508848802, 2.857208282857688, 3.405627632515735, 7.00311007171274, 5.258148911195922, 25.236192905970594, 25.025738858191588, 12.994677077825093, 112.3234250205831, 68.47280851110796, 11.52020677755121, 2.765754810474322, 1.9342250530113845, 1.7008186381913066, 1.5812681057794475, 1.5083788525530644, 1.4643580547580588, 1.2841454143706177, 1.166902536162496, 1.166159482275262, 1.1659533983929975, 1.0825270459006602, 4.9144149465988045, 1.1245086817510257, 0.8695877600348926, 0.8695878429066258, 0.8695878785707084, 0.869587747720073, 0.8695877242319616, 0.8695877107991611, 0.869587649489189, 0.8695878477239465, 0.8695879919330003, 0.8695878340118005, 0.8695879460342494, 0.8695876098192162, 0.8695876304926058, 0.869587579673188, 0.8695875454784174, 0.8695873535895821, 0.8693519687511676, 3.2608280249776063, 1.8185245512489032, 2.7874276107948273, 2.4654799971072126, 4.710050147352924, 8.330243944082202, 3.5558338474858653, 31.45112091266634, 43.95672033380955, 13.256458163939334, 20.240126214662727, 2.9094163318774715, 2.788118958258856, 2.2900015338014077, 2.0020669823227952, 1.496329148824256, 1.341584641216595, 1.3333093405655616, 1.2383247632501884, 1.1612001109617243, 2.029318303193128, 0.8278547084323157, 0.8278546877774876, 0.8278546158209213, 0.8278546758455011, 0.8278373003669007, 0.8278402223518705, 0.8275335160400047, 8.836187648988254, 0.9355847105719935, 1.1245086817510257, 1.2461215884816477, 1.4347932217128823, 7.00311007171274, 2.1805525328071433, 2.2821399629239476, 7.6341257617776765, 3.9212872509687564, 5.523157948889269, 4.935412454966351, 0.6632230781999041, 7.214555536399302, 1.3609802845736745, 1.2757885215306282, 1.079238128987155, 1.0705088884264853, 0.9383932896685437, 0.9383943707740163, 0.9383948617386592, 0.9383757188299945, 0.7975189202565519, 0.7975195794845255, 0.7974931236934716, 0.7974928919681372, 0.7974940878513828, 0.7974946139171155, 0.7974155946178005, 0.7974159325847628, 0.7974169031987757, 2.273164254319738, 1.488482286410053, 1.4103149687946401, 1.5257247431583458, 4.734535494185362, 2.5209694735706374, 2.7485633501540327, 5.52203858976615, 3.1802942241106544, 10.496567582508536, 8.41584146113672, 3.8637169391611086, 3.644863333921582, 59.899930043315734, 1.1252920597795477, 1.0880919125231525, 1.0881222659050556, 0.9743724288404197, 0.9740172415036034, 0.9145299761192098, 0.8723930967480582, 0.872426986519067, 0.7645319659102313, 0.764532068004669, 0.7645328793007655, 0.7645445678076452, 0.7645482503641464, 0.7644640311216485, 0.7654245725792647, 0.8630690851780252, 0.9355847105719935, 1.3859584733120833, 2.6150961454701096, 1.4917796686246596, 1.5718695506108011, 1.5718997727630057, 1.5143556968608656, 1.612389151302849, 1.61274668070288, 2.3666113798742003, 12.88839568785876, 4.094770672387315, 0.6573059225939782, 0.6573059225939782, 0.6613264084676491, 0.661950164233514, 19.077116843540562, 3.9212872509687564, 1.2759922748181327, 0.951766748913822, 0.9469248813286217, 0.8920747907663211, 1.3703867590607148, 0.7550086987785241, 0.7550096697198126, 0.755010054080008, 0.7550128175115779, 0.7550144379369783, 0.7550168645472335, 0.7549867462102134, 0.7549891829977577, 0.7549895909666204, 0.7549893388430655, 0.7549977688748832, 0.754998672109457, 0.7550018738940029, 0.7550716661487467, 0.7551455004156786, 0.7548450273044267, 0.7550128667532783, 0.7552046809818621, 0.7557407233543361, 0.8630690851780252, 1.4739422532539577, 2.6628560824325924, 0.6691372349338158, 0.6609585805209215, 0.6609585805209215, 19.97382791706115, 7.213320159009243], \"Category\": [\"Default\", \"Default\", \"Default\", \"Default\", \"Default\", \"Default\", \"Default\", \"Default\", \"Default\", \"Default\", \"Default\", \"Default\", \"Default\", \"Default\", \"Default\", \"Default\", \"Default\", \"Default\", \"Default\", \"Default\", \"Default\", \"Default\", \"Default\", \"Default\", \"Default\", \"Default\", \"Default\", \"Default\", \"Default\", \"Default\", \"Topic1\", \"Topic1\", \"Topic1\", \"Topic1\", \"Topic1\", \"Topic1\", \"Topic1\", \"Topic1\", \"Topic1\", \"Topic1\", \"Topic1\", \"Topic1\", \"Topic1\", \"Topic1\", \"Topic1\", \"Topic1\", \"Topic1\", \"Topic1\", \"Topic1\", \"Topic1\", \"Topic1\", \"Topic1\", \"Topic1\", \"Topic1\", \"Topic1\", \"Topic1\", \"Topic1\", \"Topic1\", \"Topic1\", \"Topic1\", \"Topic1\", \"Topic1\", \"Topic1\", \"Topic1\", \"Topic1\", \"Topic1\", \"Topic1\", \"Topic1\", \"Topic1\", \"Topic1\", \"Topic1\", \"Topic1\", \"Topic1\", \"Topic1\", \"Topic1\", \"Topic1\", \"Topic1\", \"Topic1\", \"Topic1\", \"Topic1\", \"Topic1\", \"Topic1\", \"Topic1\", \"Topic1\", \"Topic1\", \"Topic1\", \"Topic1\", \"Topic1\", \"Topic1\", \"Topic1\", \"Topic1\", \"Topic1\", \"Topic1\", \"Topic1\", \"Topic1\", \"Topic1\", \"Topic1\", \"Topic2\", \"Topic2\", \"Topic2\", \"Topic2\", \"Topic2\", \"Topic2\", \"Topic2\", \"Topic2\", \"Topic2\", \"Topic2\", \"Topic2\", \"Topic2\", \"Topic2\", \"Topic2\", \"Topic2\", \"Topic2\", \"Topic2\", \"Topic2\", \"Topic2\", \"Topic2\", \"Topic2\", \"Topic2\", \"Topic2\", \"Topic2\", \"Topic2\", \"Topic2\", \"Topic2\", \"Topic2\", \"Topic2\", \"Topic2\", \"Topic2\", \"Topic2\", \"Topic2\", \"Topic2\", \"Topic2\", \"Topic2\", \"Topic2\", \"Topic2\", \"Topic2\", \"Topic2\", \"Topic2\", \"Topic2\", \"Topic2\", \"Topic2\", \"Topic2\", \"Topic2\", \"Topic2\", \"Topic2\", \"Topic2\", \"Topic2\", \"Topic2\", \"Topic2\", \"Topic2\", \"Topic2\", \"Topic2\", \"Topic2\", \"Topic2\", \"Topic2\", \"Topic2\", \"Topic2\", \"Topic3\", \"Topic3\", \"Topic3\", \"Topic3\", \"Topic3\", \"Topic3\", \"Topic3\", \"Topic3\", \"Topic3\", \"Topic3\", \"Topic3\", \"Topic3\", \"Topic3\", \"Topic3\", \"Topic3\", \"Topic3\", \"Topic3\", \"Topic3\", \"Topic3\", \"Topic3\", \"Topic3\", \"Topic3\", \"Topic3\", \"Topic3\", \"Topic3\", \"Topic3\", \"Topic3\", \"Topic3\", \"Topic3\", \"Topic3\", \"Topic3\", \"Topic3\", \"Topic3\", \"Topic3\", \"Topic3\", \"Topic3\", \"Topic3\", \"Topic3\", \"Topic3\", \"Topic3\", \"Topic3\", \"Topic3\", \"Topic3\", \"Topic3\", \"Topic3\", \"Topic3\", \"Topic3\", \"Topic3\", \"Topic3\", \"Topic3\", \"Topic3\", \"Topic3\", \"Topic3\", \"Topic3\", \"Topic3\", \"Topic3\", \"Topic3\", \"Topic3\", \"Topic3\", \"Topic3\", \"Topic3\", \"Topic4\", \"Topic4\", \"Topic4\", \"Topic4\", \"Topic4\", \"Topic4\", \"Topic4\", \"Topic4\", \"Topic4\", \"Topic4\", \"Topic4\", \"Topic4\", \"Topic4\", \"Topic4\", \"Topic4\", \"Topic4\", \"Topic4\", \"Topic4\", \"Topic4\", \"Topic4\", \"Topic4\", \"Topic4\", \"Topic4\", \"Topic4\", \"Topic4\", \"Topic4\", \"Topic4\", \"Topic4\", \"Topic4\", \"Topic4\", \"Topic4\", \"Topic4\", \"Topic4\", \"Topic4\", \"Topic4\", \"Topic4\", \"Topic4\", \"Topic4\", \"Topic4\", \"Topic4\", \"Topic4\", \"Topic4\", \"Topic4\", \"Topic4\", \"Topic4\", \"Topic4\", \"Topic4\", \"Topic4\", \"Topic4\", \"Topic4\", \"Topic4\", \"Topic4\", \"Topic4\", \"Topic4\", \"Topic4\", \"Topic4\", \"Topic5\", \"Topic5\", \"Topic5\", \"Topic5\", \"Topic5\", \"Topic5\", \"Topic5\", \"Topic5\", \"Topic5\", \"Topic5\", \"Topic5\", \"Topic5\", \"Topic5\", \"Topic5\", \"Topic5\", \"Topic5\", \"Topic5\", \"Topic5\", \"Topic5\", \"Topic5\", \"Topic5\", \"Topic5\", \"Topic5\", \"Topic5\", \"Topic5\", \"Topic5\", \"Topic5\", \"Topic5\", \"Topic5\", \"Topic5\", \"Topic5\", \"Topic5\", \"Topic5\", \"Topic5\", \"Topic5\", \"Topic5\", \"Topic5\", \"Topic5\", \"Topic5\", \"Topic5\", \"Topic5\", \"Topic5\", \"Topic5\", \"Topic5\", \"Topic5\", \"Topic5\", \"Topic5\", \"Topic5\", \"Topic5\", \"Topic5\", \"Topic5\", \"Topic5\", \"Topic5\", \"Topic6\", \"Topic6\", \"Topic6\", \"Topic6\", \"Topic6\", \"Topic6\", \"Topic6\", \"Topic6\", \"Topic6\", \"Topic6\", \"Topic6\", \"Topic6\", \"Topic6\", \"Topic6\", \"Topic6\", \"Topic6\", \"Topic6\", \"Topic6\", \"Topic6\", \"Topic6\", \"Topic6\", \"Topic6\", \"Topic6\", \"Topic6\", \"Topic6\", \"Topic6\", \"Topic6\", \"Topic6\", \"Topic6\", \"Topic6\", \"Topic6\", \"Topic6\", \"Topic6\", \"Topic6\", \"Topic6\", \"Topic6\", \"Topic6\", \"Topic6\", \"Topic6\", \"Topic6\", \"Topic6\", \"Topic6\", \"Topic6\", \"Topic6\", \"Topic6\", \"Topic6\", \"Topic6\", \"Topic6\", \"Topic6\", \"Topic6\", \"Topic6\", \"Topic7\", \"Topic7\", \"Topic7\", \"Topic7\", \"Topic7\", \"Topic7\", \"Topic7\", \"Topic7\", \"Topic7\", \"Topic7\", \"Topic7\", \"Topic7\", \"Topic7\", \"Topic7\", \"Topic7\", \"Topic7\", \"Topic7\", \"Topic7\", \"Topic7\", \"Topic7\", \"Topic7\", \"Topic7\", \"Topic7\", \"Topic7\", \"Topic7\", \"Topic7\", \"Topic7\", \"Topic7\", \"Topic7\", \"Topic7\", \"Topic7\", \"Topic7\", \"Topic7\", \"Topic7\", \"Topic7\", \"Topic7\", \"Topic7\", \"Topic7\", \"Topic7\", \"Topic7\", \"Topic7\", \"Topic7\", \"Topic7\", \"Topic7\", \"Topic7\", \"Topic7\", \"Topic7\", \"Topic7\", \"Topic7\", \"Topic7\", \"Topic7\", \"Topic7\", \"Topic7\", \"Topic7\", \"Topic7\", \"Topic8\", \"Topic8\", \"Topic8\", \"Topic8\", \"Topic8\", \"Topic8\", \"Topic8\", \"Topic8\", \"Topic8\", \"Topic8\", \"Topic8\", \"Topic8\", \"Topic8\", \"Topic8\", \"Topic8\", \"Topic8\", \"Topic8\", \"Topic8\", \"Topic8\", \"Topic8\", \"Topic8\", \"Topic8\", \"Topic8\", \"Topic8\", \"Topic8\", \"Topic8\", \"Topic8\", \"Topic8\", \"Topic8\", \"Topic8\", \"Topic8\", \"Topic8\", \"Topic8\", \"Topic8\", \"Topic8\", \"Topic8\", \"Topic8\", \"Topic8\", \"Topic8\", \"Topic8\", \"Topic8\", \"Topic8\", \"Topic8\", \"Topic8\", \"Topic8\", \"Topic8\", \"Topic8\", \"Topic9\", \"Topic9\", \"Topic9\", \"Topic9\", \"Topic9\", \"Topic9\", \"Topic9\", \"Topic9\", \"Topic9\", \"Topic9\", \"Topic9\", \"Topic9\", \"Topic9\", \"Topic9\", \"Topic9\", \"Topic9\", \"Topic9\", \"Topic9\", \"Topic9\", \"Topic9\", \"Topic9\", \"Topic9\", \"Topic9\", \"Topic9\", \"Topic9\", \"Topic9\", \"Topic9\", \"Topic9\", \"Topic9\", \"Topic9\", \"Topic9\", \"Topic9\", \"Topic9\", \"Topic9\", \"Topic9\", \"Topic9\", \"Topic9\", \"Topic9\", \"Topic9\", \"Topic9\", \"Topic9\", \"Topic9\", \"Topic10\", \"Topic10\", \"Topic10\", \"Topic10\", \"Topic10\", \"Topic10\", \"Topic10\", \"Topic10\", \"Topic10\", \"Topic10\", \"Topic10\", \"Topic10\", \"Topic10\", \"Topic10\", \"Topic10\", \"Topic10\", \"Topic10\", \"Topic10\", \"Topic10\", \"Topic10\", \"Topic10\", \"Topic10\", \"Topic10\", \"Topic10\", \"Topic10\", \"Topic10\", \"Topic10\", \"Topic10\", \"Topic10\", \"Topic10\", \"Topic10\", \"Topic10\", \"Topic10\", \"Topic10\", \"Topic10\", \"Topic10\", \"Topic10\", \"Topic10\", \"Topic10\", \"Topic10\", \"Topic10\", \"Topic11\", \"Topic11\", \"Topic11\", \"Topic11\", \"Topic11\", \"Topic11\", \"Topic11\", \"Topic11\", \"Topic11\", \"Topic11\", \"Topic11\", \"Topic11\", \"Topic11\", \"Topic11\", \"Topic11\", \"Topic11\", \"Topic11\", \"Topic11\", \"Topic11\", \"Topic11\", \"Topic11\", \"Topic11\", \"Topic11\", \"Topic11\", \"Topic11\", \"Topic11\", \"Topic11\", \"Topic11\", \"Topic11\", \"Topic11\", \"Topic11\", \"Topic12\", \"Topic12\", \"Topic12\", \"Topic12\", \"Topic12\", \"Topic12\", \"Topic12\", \"Topic12\", \"Topic12\", \"Topic12\", \"Topic12\", \"Topic12\", \"Topic12\", \"Topic12\", \"Topic12\", \"Topic12\", \"Topic12\", \"Topic12\", \"Topic12\", \"Topic12\", \"Topic12\", \"Topic12\", \"Topic12\", \"Topic12\", \"Topic12\", \"Topic12\", \"Topic12\", \"Topic12\", \"Topic12\", \"Topic12\", \"Topic12\", \"Topic13\", \"Topic13\", \"Topic13\", \"Topic13\", \"Topic13\", \"Topic13\", \"Topic13\", \"Topic13\", \"Topic13\", \"Topic13\", \"Topic13\", \"Topic13\", \"Topic13\", \"Topic13\", \"Topic13\", \"Topic13\", \"Topic13\", \"Topic13\", \"Topic13\", \"Topic13\", \"Topic13\", \"Topic13\", \"Topic13\", \"Topic13\", \"Topic13\", \"Topic13\", \"Topic13\", \"Topic13\", \"Topic13\", \"Topic13\", \"Topic13\", \"Topic13\", \"Topic13\", \"Topic13\", \"Topic14\", \"Topic14\", \"Topic14\", \"Topic14\", \"Topic14\", \"Topic14\", \"Topic14\", \"Topic14\", \"Topic14\", \"Topic14\", \"Topic14\", \"Topic14\", \"Topic14\", \"Topic14\", \"Topic14\", \"Topic14\", \"Topic14\", \"Topic14\", \"Topic14\", \"Topic14\", \"Topic14\", \"Topic14\", \"Topic14\", \"Topic14\", \"Topic14\", \"Topic14\", \"Topic14\", \"Topic14\", \"Topic14\", \"Topic14\", \"Topic14\", \"Topic14\"], \"logprob\": [30.0, 29.0, 28.0, 27.0, 26.0, 25.0, 24.0, 23.0, 22.0, 21.0, 20.0, 19.0, 18.0, 17.0, 16.0, 15.0, 14.0, 13.0, 12.0, 11.0, 10.0, 9.0, 8.0, 7.0, 6.0, 5.0, 4.0, 3.0, 2.0, 1.0, -4.9481000900268555, -4.953000068664551, -5.3190999031066895, -5.401100158691406, -5.593200206756592, -5.826200008392334, -5.857900142669678, -5.971099853515625, -6.030700206756592, -5.872099876403809, -5.993199825286865, -6.121399879455566, -6.0625, -6.082600116729736, -6.299200057983398, -6.317999839782715, -6.35260009765625, -6.379899978637695, -6.384399890899658, -6.4029998779296875, -6.40939998626709, -6.436999797821045, -6.418600082397461, -6.452199935913086, -6.452700138092041, -6.468299865722656, -6.471499919891357, -6.454899787902832, -6.530200004577637, -6.417300224304199, -6.060999870300293, -5.527400016784668, -3.9784998893737793, -5.183000087738037, -5.736400127410889, -5.769999980926514, -5.270699977874756, -4.865499973297119, -5.674499988555908, -4.828800201416016, -5.515399932861328, -6.025700092315674, -4.798600196838379, -3.344599962234497, -3.8066999912261963, -5.061500072479248, -5.595799922943115, -4.4492998123168945, -4.932499885559082, -5.5619001388549805, -5.023200035095215, -4.876800060272217, -5.223499774932861, -5.146100044250488, -4.07289981842041, -4.274400234222412, -5.073200225830078, -5.040900230407715, -5.230199813842773, -4.324100017547607, -5.340799808502197, -5.448299884796143, -5.176000118255615, -5.113800048828125, -5.193399906158447, -5.350299835205078, -5.343400001525879, -3.7881999015808105, -4.7581000328063965, -4.912199974060059, -4.92579984664917, -4.950900077819824, -5.004199981689453, -4.959400177001953, -5.322700023651123, -5.323999881744385, -5.469299793243408, -5.446100234985352, -5.630300045013428, -5.637599945068359, -5.738900184631348, -5.839200019836426, -5.92579984664917, -5.9415998458862305, -6.026199817657471, -6.042799949645996, -6.067800045013428, -5.159599781036377, -6.099100112915039, -6.119800090789795, -6.14870023727417, -6.171800136566162, -6.182600021362305, -6.197999954223633, -6.194699764251709, -6.2195000648498535, -6.232100009918213, -4.512199878692627, -4.708099842071533, -5.717800140380859, -4.014100074768066, -4.622300148010254, -4.90880012512207, -4.182199954986572, -5.5432000160217285, -3.1644999980926514, -3.4574999809265137, -4.230599880218506, -5.506199836730957, -5.567200183868408, -4.492199897766113, -3.4583001136779785, -4.880300045013428, -3.853800058364868, -4.929999828338623, -5.170000076293945, -5.1616997718811035, -4.759300231933594, -4.681000232696533, -4.533199787139893, -4.722400188446045, -4.934299945831299, -4.950799942016602, -4.600900173187256, -4.957799911499023, -5.096199989318848, -5.089799880981445, -3.947200059890747, -5.0782999992370605, -5.209700107574463, -5.350200176239014, -5.486100196838379, -5.561299800872803, -5.59660005569458, -5.618299961090088, -5.615099906921387, -5.7245001792907715, -5.617599964141846, -5.7895002365112305, -5.858799934387207, -5.893400192260742, -5.907599925994873, -5.93179988861084, -5.231800079345703, -5.986700057983398, -5.986700057983398, -6.041399955749512, -5.974999904632568, -6.069200038909912, -6.0777997970581055, -6.1118998527526855, -6.144800186157227, -5.30620002746582, -6.195799827575684, -6.196100234985352, -5.4842000007629395, -6.2266998291015625, -5.3277997970581055, -4.511000156402588, -5.855299949645996, -4.712600231170654, -5.27869987487793, -4.31820011138916, -5.169899940490723, -4.91379976272583, -4.951099872589111, -4.674200057983398, -4.2957000732421875, -5.320099830627441, -4.731800079345703, -5.137599945068359, -5.55620002746582, -4.422399997711182, -4.805200099945068, -4.84250020980835, -5.361999988555908, -4.531199932098389, -4.883399963378906, -5.038899898529053, -4.491700172424316, -4.973100185394287, -5.202700138092041, -5.218500137329102, -5.135700225830078, -5.003499984741211, -5.265699863433838, -5.2743000984191895, -5.2957000732421875, -4.514100074768066, -4.6880998611450195, -4.770500183105469, -4.787099838256836, -5.038300037384033, -5.133900165557861, -5.142600059509277, -5.1579999923706055, -5.256800174713135, -5.288400173187256, -5.369500160217285, -4.679900169372559, -5.568399906158447, -5.600900173187256, -4.572800159454346, -5.698699951171875, -5.698800086975098, -5.698999881744385, -5.791500091552734, -5.80079984664917, -5.9019999504089355, -5.92140007019043, -5.921999931335449, -5.644999980926514, -5.995299816131592, -5.999199867248535, -6.037899971008301, -6.136499881744385, -5.531599998474121, -6.210299968719482, -3.4261999130249023, -3.1879000663757324, -4.702499866485596, -5.570700168609619, -3.057800054550171, -4.179699897766113, -4.7368998527526855, -4.5040998458862305, -4.407199859619141, -4.632199764251709, -4.701900005340576, -3.68530011177063, -5.120999813079834, -4.948299884796143, -4.650300025939941, -4.819900035858154, -4.454899787902832, -4.589099884033203, -4.66540002822876, -4.572800159454346, -4.332300186157227, -4.697800159454346, -4.6458001136779785, -4.854300022125244, -4.848199844360352, -4.984000205993652, -4.429200172424316, -4.475500106811523, -4.67140007019043, -4.743100166320801, -4.851600170135498, -4.882900238037109, -4.901000022888184, -5.045599937438965, -5.123600006103516, -5.42609977722168, -5.50540018081665, -5.5524001121521, -5.7179999351501465, -5.81850004196167, -5.817699909210205, -5.787799835205078, -5.876100063323975, -5.9141998291015625, -5.99459981918335, -3.5132999420166016, -6.153600215911865, -6.273900032043457, -6.27400016784668, -6.27400016784668, -6.274600028991699, -6.289400100708008, -6.289899826049805, -6.289999961853027, -6.291100025177002, -5.797999858856201, -4.110300064086914, -4.6230998039245605, -4.199999809265137, -5.013899803161621, -4.436399936676025, -4.788300037384033, -4.694399833679199, -5.599800109863281, -2.0922999382019043, -5.384500026702881, -4.248899936676025, -4.605999946594238, -4.582900047302246, -4.748300075531006, -5.202700138092041, -4.424499988555908, -4.8003997802734375, -5.111999988555908, -4.69789981842041, -4.904799938201904, -4.523399829864502, -5.241600036621094, -4.940100193023682, -4.215799808502197, -4.228300094604492, -4.234300136566162, -4.562600135803223, -4.616199970245361, -4.793099880218506, -4.886099815368652, -4.941199779510498, -5.0671000480651855, -5.102799892425537, -5.123600006103516, -5.189700126647949, -5.216400146484375, -5.23859977722168, -5.305300235748291, -5.368299961090088, -5.376200199127197, -5.384799957275391, -4.34250020980835, -5.546999931335449, -5.546999931335449, -5.566299915313721, -5.567399978637695, -5.5883002281188965, -5.684700012207031, -5.765100002288818, -6.04449987411499, -6.04449987411499, -6.04449987411499, -6.0447001457214355, -5.490200042724609, -3.9200000762939453, -5.368100166320801, -5.24429988861084, -4.192299842834473, -4.7891998291015625, -4.523399829864502, -3.6489999294281006, -5.321899890899658, -4.936600208282471, -5.053999900817871, -4.432600021362305, -4.788400173187256, -4.741099834442139, -4.409299850463867, -5.0447001457214355, -4.997900009155273, -4.975800037384033, -4.979400157928467, -5.205399990081787, -5.199100017547607, -3.7430999279022217, -4.545199871063232, -4.630000114440918, -4.920400142669678, -5.01639986038208, -5.047599792480469, -5.149600028991699, -5.200900077819824, -5.222799777984619, -5.262800216674805, -5.264400005340576, -4.307799816131592, -5.507500171661377, -5.591700077056885, -5.592100143432617, -5.593100070953369, -5.61460018157959, -5.709700107574463, -5.710899829864502, -5.711599826812744, -5.802599906921387, -5.802999973297119, -5.805200099945068, -6.070400238037109, -6.070499897003174, -6.085899829864502, -6.08650016784668, -6.086999893188477, -6.087500095367432, -6.087500095367432, -3.1231000423431396, -5.11299991607666, -4.227099895477295, -5.249899864196777, -5.528299808502197, -5.094600200653076, -4.9253997802734375, -4.768899917602539, -4.803100109100342, -3.4765000343322754, -4.951700210571289, -4.423900127410889, -5.142000198364258, -4.051300048828125, -4.363399982452393, -4.61899995803833, -5.354100227355957, -5.282800197601318, -4.6468000411987305, -4.751999855041504, -4.743000030517578, -4.928400039672852, -5.1132001876831055, -5.095200061798096, -5.220300197601318, -3.504300117492676, -4.026700019836426, -4.664999961853027, -4.817699909210205, -5.091400146484375, -5.127299785614014, -5.127299785614014, -5.1371002197265625, -5.14709997177124, -5.149600028991699, -5.2657999992370605, -5.26669979095459, -5.345399856567383, -5.397299766540527, -5.62470006942749, -5.62470006942749, -5.640399932861328, -5.640399932861328, -5.641600131988525, -5.642000198364258, -5.646399974822998, -4.785099983215332, -5.8420000076293945, -5.842100143432617, -5.848800182342529, -4.983399868011475, -6.013700008392334, -6.013700008392334, -6.013700008392334, -6.013700008392334, -5.345399856567383, -4.943900108337402, -4.965400218963623, -4.696199893951416, -4.762800216674805, -5.052599906921387, -5.5177998542785645, -4.878300189971924, -4.63100004196167, -5.384399890899658, -5.451200008392334, -5.420599937438965, -5.075399875640869, -5.46589994430542, -5.3302998542785645, -5.4085001945495605, -5.440400123596191, -3.265500068664551, -3.9475998878479004, -4.115699768066406, -4.236000061035156, -4.434500217437744, -4.499100208282471, -4.68149995803833, -4.781899929046631, -4.81279993057251, -4.830100059509277, -5.011199951171875, -5.117499828338623, -5.1290998458862305, -5.2621002197265625, -5.489099979400635, -5.489099979400635, -5.505300045013428, -5.506100177764893, -5.706500053405762, -4.521299839019775, -5.2220001220703125, -5.1666998863220215, -4.3892998695373535, -5.489099979400635, -5.251200199127197, -6.5233001708984375, -6.5233001708984375, -6.5233001708984375, -6.523399829864502, -6.523399829864502, -4.828700065612793, -4.640399932861328, -5.677599906921387, -5.548500061035156, -4.992400169372559, -5.255499839782715, -4.217899799346924, -4.234899997711182, -4.86269998550415, -4.6402997970581055, -5.380499839782715, -5.457699775695801, -3.9523000717163086, -4.446899890899658, -4.644800186157227, -4.763800144195557, -4.843900203704834, -4.895599842071533, -5.140500068664551, -5.339600086212158, -5.341000080108643, -5.341800212860107, -5.512199878692627, -4.308700084686279, -5.852399826049805, -6.157400131225586, -6.157400131225586, -6.157400131225586, -6.157400131225586, -6.157400131225586, -6.157400131225586, -6.157400131225586, -6.157400131225586, -6.157400131225586, -6.157400131225586, -6.157400131225586, -6.157400131225586, -6.157400131225586, -6.157400131225586, -6.157400131225586, -6.157400131225586, -6.158400058746338, -4.843900203704834, -5.533899784088135, -5.404600143432617, -5.492300033569336, -5.168399810791016, -5.15339994430542, -5.532299995422363, -5.189700126647949, -5.3420000076293945, -5.736599922180176, -6.092100143432617, -3.6410000324249268, -3.6958999633789062, -3.9595999717712402, -4.151199817657471, -4.6149001121521, -4.814000129699707, -4.825799942016602, -4.972899913787842, -5.1107001304626465, -5.109300136566162, -6.127699851989746, -6.127699851989746, -6.127699851989746, -6.127699851989746, -6.127799987792969, -6.127799987792969, -6.129700183868408, -3.8208999633789062, -6.127799987792969, -6.127699851989746, -6.127699851989746, -6.127799987792969, -4.93209981918335, -6.155700206756592, -6.127699851989746, -5.339300155639648, -6.225800037384033, -6.144499778747559, -6.311100006103516, -8.431500434875488, -6.230199813842773, -4.52370023727417, -4.649799823760986, -5.021200180053711, -5.041299819946289, -5.410099983215332, -5.410099983215332, -5.410099983215332, -5.410200119018555, -6.0553998947143555, -6.0553998947143555, -6.055600166320801, -6.055600166320801, -6.055600166320801, -6.055600166320801, -6.056099891662598, -6.056099891662598, -6.056099891662598, -5.036399841308594, -6.055600166320801, -6.119800090789795, -6.070899963378906, -5.046800136566162, -6.0569000244140625, -6.153600215911865, -5.539000034332275, -6.117400169372559, -4.939599990844727, -5.189199924468994, -6.0559000968933105, -6.28439998626709, -4.01140022277832, -4.624000072479248, -4.704800128936768, -4.704800128936768, -5.001800060272217, -5.002900123596191, -5.202600002288818, -5.3730998039245605, -5.3730998039245605, -6.0183000564575195, -6.0183000564575195, -6.0183000564575195, -6.0183000564575195, -6.0183000564575195, -6.018799781799316, -6.019999980926514, -6.018499851226807, -6.018799781799316, -6.018799781799316, -5.389699935913086, -6.030799865722656, -6.049600124359131, -6.049600124359131, -6.088900089263916, -6.107399940490723, -6.107900142669678, -6.084499835968018, -5.127900123596191, -6.390999794006348, -8.3818998336792, -8.3818998336792, -8.377699851989746, -8.377699851989746, -8.356399536132812, -8.374699592590332, -4.247399806976318, -4.97130012512207, -4.987400054931641, -5.188799858093262, -4.97130012512207, -6.00570011138916, -6.00570011138916, -6.00570011138916, -6.00570011138916, -6.00570011138916, -6.00570011138916, -6.005899906158447, -6.005899906158447, -6.005899906158447, -6.005899906158447, -6.005899906158447, -6.005899906158447, -6.00600004196167, -6.006100177764893, -6.006199836730957, -6.008299827575684, -6.008600234985352, -6.008900165557861, -6.0096001625061035, -6.00629997253418, -6.00600004196167, -6.007400035858154, -8.318900108337402, -8.353300094604492, -8.353300094604492, -6.043300151824951, -8.175600051879883], \"loglift\": [30.0, 29.0, 28.0, 27.0, 26.0, 25.0, 24.0, 23.0, 22.0, 21.0, 20.0, 19.0, 18.0, 17.0, 16.0, 15.0, 14.0, 13.0, 12.0, 11.0, 10.0, 9.0, 8.0, 7.0, 6.0, 5.0, 4.0, 3.0, 2.0, 1.0, 0.72, 0.7199, 0.7173, 0.7165, 0.7138, 0.7113, 0.7108, 0.709, 0.7079, 0.707, 0.7067, 0.7062, 0.7057, 0.7049, 0.7022, 0.7019, 0.701, 0.7003, 0.7002, 0.6997, 0.6996, 0.6988, 0.6985, 0.6984, 0.6984, 0.698, 0.6979, 0.6977, 0.6962, 0.6962, 0.6958, 0.6873, 0.6572, 0.681, 0.6816, 0.6733, 0.6375, 0.6032, 0.6544, 0.5883, 0.6319, 0.67, 0.5547, 0.3388, 0.3096, 0.5049, 0.5915, 0.3492, 0.4282, 0.5715, 0.4202, 0.3529, 0.4552, 0.4229, 0.036, 0.0828, 0.3455, 0.2864, 0.3795, -0.1929, 0.4325, 0.4866, 0.2251, 0.1012, -0.0524, 0.2617, 0.2151, 1.646, 1.6379, 1.6357, 1.6355, 1.6351, 1.6342, 1.632, 1.628, 1.627, 1.6243, 1.6217, 1.6198, 1.6195, 1.6162, 1.6126, 1.6092, 1.6085, 1.6049, 1.6041, 1.6027, 1.6023, 1.6015, 1.6001, 1.5989, 1.5978, 1.5973, 1.5964, 1.5962, 1.5951, 1.5946, 1.5414, 1.5427, 1.5748, 1.468, 1.4357, 1.4113, 1.2231, 1.4763, 0.9444, 0.8997, 1.0651, 1.4357, 1.4541, 0.8667, 0.2251, 0.9819, 0.2625, 0.9714, 1.1134, 1.0412, 0.568, 0.46, 0.2653, 0.4925, 0.7201, 0.596, -0.4697, 0.2107, 0.6412, 0.4798, 2.1898, 2.1688, 2.1645, 2.1565, 2.1535, 2.1499, 2.1482, 2.1471, 2.1469, 2.1413, 2.1374, 2.1374, 2.1332, 2.1309, 2.13, 2.1283, 2.1274, 2.1244, 2.1244, 2.1203, 2.1195, 2.1182, 2.1174, 2.1147, 2.112, 2.1103, 2.1076, 2.1076, 2.1058, 2.1048, 2.0544, 1.8983, 2.0629, 1.8596, 1.9214, 1.607, 1.8153, 1.716, 1.609, 1.3826, 1.1287, 1.7341, 1.3381, 1.604, 1.8734, 0.7186, 0.9851, 0.9704, 1.4997, -0.4, 0.3256, 0.6986, -0.8083, 0.4279, 1.0262, 1.0579, 0.4784, -0.8872, 0.0931, 0.3377, -0.066, 2.5568, 2.5517, 2.5492, 2.5487, 2.5391, 2.5348, 2.5344, 2.5337, 2.5287, 2.5269, 2.5224, 2.5167, 2.5095, 2.5072, 2.5058, 2.4997, 2.4997, 2.4997, 2.492, 2.4912, 2.4819, 2.48, 2.48, 2.4793, 2.4725, 2.4721, 2.468, 2.4566, 2.4558, 2.4477, 2.4428, 2.3735, 2.3882, 2.4357, 2.1108, 2.1885, 2.2852, 2.1039, 2.061, 2.0605, 2.0665, 1.5237, 2.1589, 2.032, 1.779, 1.8423, 1.4206, 1.4808, 1.2598, 0.8324, -0.2011, 0.7435, 0.5692, 1.1226, 0.9799, 0.9187, 3.2113, 3.2091, 3.1987, 3.1944, 3.1873, 3.1851, 3.1838, 3.1726, 3.166, 3.1351, 3.1256, 3.1196, 3.0908, 3.0808, 3.0808, 3.0752, 3.0711, 3.0646, 3.05, 3.0392, 3.0179, 2.991, 2.991, 2.991, 2.9908, 2.9874, 2.9872, 2.9872, 2.9869, 2.9769, 2.9578, 2.9569, 2.7765, 2.8523, 2.7592, 2.7814, 2.7435, 2.8546, 2.039, 2.7146, 2.0003, 1.9662, 1.83, 1.8959, 2.2614, 0.9344, 1.4025, 1.9287, 0.5319, 0.9966, -0.4071, 1.6634, -1.2567, 3.4625, 3.4619, 3.4616, 3.4428, 3.4391, 3.4252, 3.4138, 3.4125, 3.3999, 3.3955, 3.3937, 3.3861, 3.3829, 3.3802, 3.3717, 3.3633, 3.3622, 3.361, 3.3478, 3.3366, 3.3366, 3.3335, 3.3333, 3.3298, 3.3131, 3.2982, 3.238, 3.238, 3.238, 3.2379, 3.2306, 3.1594, 3.2143, 2.9882, 2.5493, 2.7751, 2.6301, 0.4822, 2.9256, 2.2449, 2.4103, 1.0361, 1.6185, 1.3756, 0.2264, 2.1079, 1.2801, 0.9257, 0.5675, 2.2124, 0.5913, 3.495, 3.4574, 3.4515, 3.4274, 3.4179, 3.4147, 3.4033, 3.3973, 3.3946, 3.3896, 3.3894, 3.356, 3.3546, 3.3408, 3.3408, 3.3406, 3.3369, 3.3199, 3.3196, 3.3195, 3.3018, 3.3017, 3.3013, 3.2418, 3.2418, 3.238, 3.2378, 3.2377, 3.2375, 3.2375, 2.9742, 3.0836, 2.7906, 3.0308, 3.0872, 2.9581, 2.9052, 2.8383, 2.8465, 2.1375, 2.8715, 2.3954, 2.8982, 2.0447, 2.1601, 1.6807, 2.8896, 2.5407, 0.7965, 0.8601, 0.6742, 0.3014, 1.7399, 1.5034, 1.8506, 4.1343, 4.1026, 4.0358, 4.0134, 3.9654, 3.9583, 3.9582, 3.9563, 3.9542, 3.9537, 3.9288, 3.9286, 3.9103, 3.8977, 3.8362, 3.8362, 3.8316, 3.8316, 3.8312, 3.8311, 3.8298, 3.8239, 3.7675, 3.7674, 3.7651, 3.7173, 3.7057, 3.7057, 3.7057, 3.7057, 3.6328, 3.581, 3.4474, 3.3559, 3.0679, 3.1714, 3.4759, 2.1044, 1.3039, 3.1561, 2.8753, 2.5906, 0.278, 2.8366, 1.0287, 1.8828, 2.1851, 4.3889, 4.3425, 4.3258, 4.3123, 4.2866, 4.2772, 4.2478, 4.2296, 4.2237, 4.2203, 4.1821, 4.157, 4.1542, 4.1195, 4.0522, 4.0522, 4.047, 4.0467, 3.9772, 3.9512, 3.8929, 3.8453, 3.8168, 3.7439, 3.5984, 3.594, 3.594, 3.594, 3.594, 3.594, 3.5841, 3.5464, 3.4626, 3.4161, 3.2513, 3.2748, 2.7439, 2.7352, 2.7627, 0.8283, 0.583, 2.2882, 5.2204, 5.0834, 5.0141, 4.968, 4.9351, 4.913, 4.7994, 4.6961, 4.6953, 4.6947, 4.5985, 4.2892, 4.2203, 4.1724, 4.1724, 4.1724, 4.1724, 4.1724, 4.1724, 4.1724, 4.1724, 4.1724, 4.1724, 4.1724, 4.1724, 4.1724, 4.1724, 4.1724, 4.1724, 4.1716, 4.1641, 4.0581, 3.7603, 3.7953, 3.4719, 2.9167, 3.3891, 1.5518, 1.0649, 1.8689, 1.0902, 5.4811, 5.4687, 5.4019, 5.3446, 5.1721, 5.0822, 5.0766, 5.0033, 4.9299, 4.373, 4.2513, 4.2513, 4.2513, 4.2513, 4.2512, 4.2512, 4.2496, 4.1903, 4.1288, 3.945, 3.8423, 3.7012, 3.3116, 3.2548, 3.2372, 2.8181, 2.5978, 2.3365, 2.2825, 2.1691, 1.9837, 5.3581, 5.2966, 5.0926, 5.0805, 4.8435, 4.8435, 4.8435, 4.8435, 4.3609, 4.3609, 4.3607, 4.3607, 4.3607, 4.3607, 4.3603, 4.3603, 4.3603, 4.3324, 3.7367, 3.7264, 3.6967, 3.5883, 3.2084, 3.0254, 2.9423, 2.9157, 2.8994, 2.8707, 2.7825, 2.6123, 2.0859, 5.448, 5.4008, 5.4008, 5.2142, 5.2135, 5.0767, 4.9535, 4.9534, 4.4402, 4.4402, 4.4402, 4.4402, 4.4401, 4.4398, 4.4373, 4.3188, 4.2378, 3.8448, 3.839, 3.7593, 3.6881, 3.6881, 3.6861, 3.605, 3.6042, 3.2441, 2.5058, 2.3893, 2.2278, 2.2278, 2.2259, 2.2249, -1.1149, 0.4489, 5.6989, 5.2682, 5.2572, 5.1155, 4.9036, 4.4654, 4.4653, 4.4653, 4.4653, 4.4653, 4.4653, 4.4651, 4.4651, 4.4651, 4.4651, 4.4651, 4.4651, 4.4651, 4.4649, 4.4647, 4.4629, 4.4625, 4.4619, 4.4604, 4.331, 3.796, 3.2032, 2.2729, 2.2508, 2.2508, 1.1523, 0.0385]}, \"token.table\": {\"Topic\": [8, 11, 4, 4, 6, 4, 4, 6, 1, 2, 7, 1, 7, 3, 2, 4, 7, 1, 2, 3, 6, 1, 3, 2, 3, 5, 5, 9, 1, 7, 12, 3, 1, 6, 1, 3, 9, 1, 2, 3, 5, 1, 2, 3, 8, 9, 3, 9, 1, 2, 3, 9, 2, 6, 3, 3, 5, 1, 3, 7, 2, 7, 3, 1, 7, 1, 8, 11, 3, 3, 1, 2, 3, 4, 6, 5, 6, 1, 4, 7, 6, 1, 2, 3, 4, 1, 2, 3, 5, 6, 7, 9, 6, 1, 3, 1, 2, 9, 3, 1, 3, 6, 3, 5, 10, 10, 1, 1, 3, 5, 6, 1, 2, 3, 4, 5, 6, 1, 2, 3, 4, 5, 6, 7, 10, 2, 1, 2, 3, 5, 1, 6, 2, 3, 9, 3, 7, 1, 2, 3, 4, 3, 1, 2, 4, 1, 2, 3, 4, 7, 9, 1, 6, 6, 8, 9, 9, 6, 1, 3, 1, 8, 6, 5, 8, 3, 7, 1, 2, 3, 4, 6, 11, 1, 2, 4, 7, 1, 1, 4, 8, 10, 1, 3, 11, 7, 1, 9, 4, 12, 3, 3, 9, 2, 8, 4, 8, 1, 2, 7, 5, 7, 1, 2, 6, 8, 1, 2, 3, 5, 1, 2, 3, 6, 5, 1, 3, 5, 3, 6, 1, 3, 1, 2, 3, 7, 5, 8, 2, 6, 1, 2, 1, 2, 1, 2, 4, 3, 3, 3, 4, 1, 6, 9, 9, 1, 2, 3, 5, 1, 2, 3, 7, 8, 1, 2, 3, 9, 2, 2, 2, 4, 5, 2, 5, 6, 3, 5, 8, 8, 1, 3, 6, 1, 2, 3, 4, 5, 7, 6, 4, 1, 3, 8, 4, 6, 7, 4, 6, 1, 6, 1, 8, 1, 8, 9, 1, 9, 6, 3, 1, 4, 1, 2, 3, 4, 8, 11, 1, 2, 3, 4, 6, 1, 4, 7, 4, 5, 9, 2, 1, 5, 10, 1, 3, 4, 7, 5, 1, 4, 8, 3, 6, 8, 1, 2, 3, 4, 6, 6, 7, 8, 7, 6, 8, 1, 2, 3, 4, 5, 1, 2, 7, 1, 3, 5, 1, 2, 3, 5, 7, 4, 9, 1, 2, 4, 1, 5, 2, 5, 8, 1, 3, 1, 1, 10, 2, 4, 1, 8, 9, 8, 1, 2, 4, 7, 3, 5, 5, 5, 5, 8, 1, 2, 3, 4, 5, 7, 8, 1, 5, 8, 10, 1, 4, 8, 1, 2, 3, 4, 6, 2, 1, 3, 6, 6, 8, 2, 5, 6, 7, 1, 2, 3, 2, 4, 8, 1, 2, 3, 6, 7, 7, 11, 1, 1, 9, 7, 1, 3, 6, 7, 5, 1, 3, 9, 2, 2, 3, 1, 3, 6, 3, 3, 3, 1, 2, 3, 6, 9, 4, 9, 5, 1, 5, 5, 4, 6, 4, 11, 2, 5, 2, 1, 2, 3, 4, 5, 7, 1, 8, 1, 2, 3, 4, 1, 1, 8, 7, 9, 11, 4, 8, 10, 3, 1, 4, 3, 4, 2, 4, 5, 1, 2, 7, 2, 2, 5, 7, 2, 10, 2, 3, 2, 1, 7, 7, 1, 2, 3, 7, 2, 4, 9, 3, 9, 2, 6, 7, 1, 3, 2, 1, 2, 1, 2, 3, 3, 5, 9, 1, 2, 3, 7, 8, 1, 2, 4, 4, 11, 3, 5, 1, 3, 7, 4, 3, 1, 7, 9, 12, 4, 1, 2, 3, 5, 1, 1, 3, 9, 5, 7, 3, 4, 1, 7, 1, 5, 11, 2, 4, 5, 9, 7, 4, 2, 2, 1, 3, 4, 9, 4, 6, 1, 9, 8, 8, 2, 3, 2, 1, 2, 2, 1, 10, 4, 8, 2, 2, 4, 7, 8, 10, 1, 6, 1, 2, 3, 6, 4, 8, 6, 5, 1, 3, 7, 9, 1, 7, 8, 10, 3, 5, 5, 4, 3, 3, 1, 6, 4, 1, 1, 6, 4, 1, 5, 3, 5, 1, 2, 3, 5, 6, 7, 1, 1, 3, 5, 1, 10, 1, 2, 1, 2, 3, 4, 6, 1, 7, 7, 1, 7, 9, 9, 3, 6, 7, 7, 4, 3, 1, 8, 7, 2, 4, 1, 1, 6, 1, 2, 3, 1, 2, 3, 4, 7, 1, 2, 3, 4, 8, 4, 6, 7, 1, 3, 7, 2, 3, 4, 3, 4, 2, 3, 5, 1, 2, 8, 5, 3, 6, 8, 10, 8, 1, 2, 3, 1, 2, 3, 4, 7, 3, 6, 1, 2, 3, 4, 4, 3, 1, 9, 3, 6, 3, 7, 4, 1, 2, 3, 4, 8, 7, 1, 4, 8, 9, 1, 2, 3, 4, 7, 3, 1, 3, 1, 2, 4, 4, 2, 5, 8, 9, 7, 6, 1, 1, 2, 3, 1, 5, 2, 3, 5, 5, 14, 8, 1, 2, 3, 4, 9, 5, 6, 5, 11, 10, 6, 9, 2, 1, 2, 5, 1, 2, 3, 4, 5, 6, 1, 3, 5, 6, 1, 4, 1, 2, 3, 4, 1, 2, 3, 7, 6, 4, 9, 1, 2, 3, 4, 5, 10, 3, 9, 1, 4, 11, 4, 3, 1, 2, 3, 5, 3, 3, 6, 1, 3, 5, 6, 1, 4, 6, 2, 6, 1, 5, 5, 5, 8, 7, 1, 1, 2, 3, 4, 6, 1, 2, 3, 4, 6, 7, 1, 6, 7, 8, 3, 8, 1, 3, 5, 6, 4, 7, 9, 6, 3, 1, 6, 9, 5, 1, 5, 6, 1, 3, 4, 5, 8, 5, 1, 3, 5, 1, 4, 7, 8, 8, 1, 2, 3, 7, 8, 3, 5, 7, 1, 3, 7, 7, 7, 2, 3, 4, 7, 8, 6, 2], \"Freq\": [0.6746779384896926, 0.8075426008402953, 0.6718252606228973, 0.20035895657885291, 0.8014358263154117, 0.8933633582199509, 0.9352478827859104, 0.04453561346599574, 0.18760519466377254, 0.3752103893275451, 0.3752103893275451, 0.35388842158099115, 0.5898140359683185, 0.922335776537102, 0.8953287392390916, 0.07785467297731231, 0.01946366824432808, 0.2903088948069618, 0.5146384953396141, 0.1451544474034809, 0.039587576564585704, 0.9903481154699617, 0.6554262192339736, 0.10013103188356046, 0.2503275797089011, 0.6007861913013627, 0.34127188192556945, 0.6825437638511389, 0.4006682475696509, 0.5676133507236721, 0.01669451031540212, 0.6703402794877518, 0.9851935951658993, 0.6969645415568536, 0.20116589977752417, 0.7543721241657155, 0.05029147494438104, 0.6038738135372356, 0.26297730589524776, 0.1168788026201101, 0.009739900218342508, 0.9574355940853183, 0.01877324694284938, 0.01877324694284938, 0.6172869293518138, 0.20576230978393795, 0.3078912217856266, 0.6157824435712532, 0.7594255461504013, 0.029208674851938506, 0.1898563865376003, 0.014604337425969253, 0.9277938473052394, 0.9745388128071072, 0.8881943192949361, 0.7247409974129736, 0.26842259163443466, 0.853480934819709, 0.07758917589270081, 0.8976860027387412, 0.9125375863376701, 0.8639225358677395, 0.8754605235081057, 0.652877087445059, 0.3092575677371332, 0.5658548911161366, 0.22634195644645466, 0.22634195644645466, 0.9840778451414571, 0.7647902366666005, 0.8049549571889225, 0.06871566707710314, 0.08834871481341833, 0.019633047736315183, 0.019633047736315183, 0.7286951817799971, 0.7962045735059993, 0.9259593626272515, 0.04208906193760234, 0.8313998468810664, 0.990896676515931, 0.016242531224287194, 0.22739543714002072, 0.42230581183146704, 0.32485062448574387, 0.3031530604753337, 0.22047495307296996, 0.22047495307296996, 0.05511873826824249, 0.05511873826824249, 0.13779684567060624, 0.027559369134121246, 0.6481908972298791, 0.7621539076575077, 0.09526923845718846, 0.0754349304794122, 0.8297842352735343, 0.0754349304794122, 0.9577364298569928, 0.9328662464371515, 0.027095700103983655, 0.038708143005690934, 0.6549538422636234, 0.26198153690544934, 0.8569696002963746, 0.7231299001725332, 0.9052791982900873, 0.9125677636549139, 0.08774690035143402, 0.08210865341692562, 0.8210865341692563, 0.39966481443081625, 0.11919827798813819, 0.07479107638471415, 0.06076774956258025, 0.29448986326481197, 0.04908164387746866, 0.3867440489395283, 0.13649789962571587, 0.13649789962571587, 0.045499299875238625, 0.045499299875238625, 0.1592475495633352, 0.06824894981285794, 0.022749649937619312, 0.9797171613700976, 0.7073923274186671, 0.1845371288918262, 0.10252062716212566, 0.010252062716212566, 0.9759657139442856, 0.8809868079904323, 0.9435060716425808, 0.5872632641644975, 0.29363163208224874, 0.9094839254257895, 0.8534208320617659, 0.1253023875550629, 0.2374150501043297, 0.013189725005796094, 0.6265119377753144, 0.9507455177867519, 0.13909861585522762, 0.2225577853683642, 0.6120339097630015, 0.05413958928909957, 0.8301403690995268, 0.01804652976303319, 0.09023264881516596, 0.43991541662670464, 0.522730959842579, 0.2674498698733118, 0.7131996529954981, 0.9563933184456366, 0.5073176113050893, 0.5073176113050893, 0.8975072621125608, 0.8070084294452473, 0.36218508210111894, 0.36218508210111894, 0.9577726522252639, 0.6246179484218105, 0.9063562189018831, 0.8027392859500604, 0.6733271485367787, 0.20711387811294454, 0.6213416343388336, 0.14054241506783122, 0.028108483013566242, 0.5481154187645417, 0.2670305886288793, 0.014054241506783121, 0.8733618604525542, 0.09797834227912897, 0.26127557941101054, 0.5878700536747737, 0.03265944742637632, 0.8450901643624785, 0.5351300120930127, 0.44206392303335834, 0.7391346536870171, 0.8576672115526001, 0.9857575099022968, 0.909794431922295, 0.6683021585095447, 0.8541451728101945, 0.9795308040709042, 0.5227312193737508, 0.8363912803733107, 0.7838289678294403, 0.4056005326237958, 0.35902320149919276, 0.5385348022487891, 0.9750712487453046, 0.4877156742489374, 0.8100892940678492, 0.6679875212369312, 0.975355502600511, 0.6603468406220002, 0.8595237949500524, 0.9828350800321751, 0.7951442293082943, 0.9897176133634584, 0.9849769609609035, 0.6903063142223849, 0.23010210474079498, 0.31909799421507185, 0.45471464175647736, 0.11966174783065193, 0.09572939826452155, 0.5617134521899254, 0.17021619763331072, 0.1361729581066486, 0.11915133834331751, 0.9796943665647463, 0.6922898183420122, 0.02884540909758384, 0.2596086818782546, 0.9366829879380091, 0.06244553252920061, 0.8316520636272665, 0.1386086772712111, 0.36789881130304297, 0.3270211656027049, 0.12263293710101432, 0.16351058280135244, 0.30232752017422787, 0.6046550403484557, 0.6078519328169206, 0.20261731093897356, 0.05516501125657682, 0.9378051913618061, 0.5244424347869967, 0.4717052067078574, 0.06643439439889014, 0.06643439439889014, 0.8636471271855718, 0.9538222207681651, 0.9296143017580384, 0.3268071836689806, 0.6536143673379612, 0.5624559767926418, 0.2812279883963209, 0.8005926444502751, 0.9232220153316143, 0.2137612966982648, 0.4080897482421419, 0.07773138061755083, 0.29149267731581563, 0.4324472691540532, 0.11325999906415678, 0.17503818037187865, 0.24711272523088754, 0.01029636355128698, 0.12516697762632054, 0.7927241916333634, 0.06258348881316027, 0.602739657796182, 0.9796817762606991, 0.9303348404355075, 0.24131017163530674, 0.12065508581765337, 0.6032754290882668, 0.5893708053416823, 0.3929138702277882, 0.9303852611544575, 0.8250220050513494, 0.10312775063141867, 0.6627913386451018, 0.7857439366027145, 0.2839794576502553, 0.68966439715062, 0.8123954037453888, 0.6585448226172597, 0.24868126168763652, 0.04605208549771046, 0.01381562564931314, 0.025328647023740755, 0.004605208549771047, 0.8070163515467444, 0.9942772656316956, 0.47529412459484754, 0.35647059344613563, 0.8172115267196366, 0.8964288198404597, 0.2724441486530566, 0.6811103716326415, 0.25881813180059726, 0.5176362636011945, 0.9662978630241327, 0.023007091976765065, 0.8497432513262659, 0.12139189304660943, 0.4046858024712429, 0.1348952674904143, 0.4046858024712429, 0.9809864021932437, 0.807639910124313, 0.9001273478323574, 0.9567387209018277, 0.8767565914047646, 0.10959457392559557, 0.6878106904838427, 0.2505270726927904, 0.05010541453855808, 0.013665113055970386, 0.6246179386167853, 0.6874231020451387, 0.42986395479956696, 0.19650923647980204, 0.29476385471970307, 0.012281827279987627, 0.04912730911995051, 0.36334322849081563, 0.6055720474846927, 0.030278602374234638, 0.1958080526628563, 0.6853281843199971, 0.09790402633142815, 0.9583501186570351, 0.3638788108422621, 0.6549818595160718, 0.5879521646490334, 0.7269899708427652, 0.06377105007392676, 0.20406736023656566, 0.6754035177138261, 0.721522339417759, 0.9866800413552979, 0.2255118284825003, 0.6765354854475009, 0.45435262214035377, 0.3029017480935692, 0.3029017480935692, 0.46778984476389146, 0.24255769728498078, 0.0952905239333853, 0.1559299482546305, 0.03465109961214011, 0.0944780981778181, 0.5668685890669086, 0.2834342945334543, 0.9414786458037165, 0.8311625816752952, 0.7707901855016552, 0.6780714274862152, 0.240461453359649, 0.0492871301917293, 0.02090969159649122, 0.01045484579824561, 0.2364887628994894, 0.2364887628994894, 0.4729775257989788, 0.4867039392803305, 0.26207135192017794, 0.1871938228001271, 0.8709960779309155, 0.025368817803813073, 0.0338250904050841, 0.008456272601271025, 0.05919390820889717, 0.884112721674271, 0.11051409020928388, 0.4931440759227325, 0.3966593654161109, 0.09648471050662158, 0.9641748120272402, 0.9462327516268715, 0.22060417825193124, 0.7353472608397708, 0.6246175551465065, 0.25087629404347006, 0.7298219463082765, 0.6361728764946958, 0.9704269110053556, 0.6629634181806592, 0.252549117502975, 0.7576473525089249, 0.6156366912458091, 0.15390917281145228, 0.15390917281145228, 0.48812897144167616, 0.08791102401013418, 0.09767891556681575, 0.8107349992045707, 0.8884632903111908, 0.8447224450984887, 0.1206746350140698, 0.7929728389965522, 0.9541874134306494, 0.7958743493017875, 0.482352724676934, 0.6870698627160912, 0.07010916966490725, 0.10516375449736089, 0.028043667865962903, 0.06309825269841653, 0.04206550179894435, 0.9356723244250744, 0.4715931796989237, 0.11789829492473093, 0.4715931796989237, 0.51700291982213, 0.9894351828982002, 0.9554937702666234, 0.7396422086344726, 0.7486243226938217, 0.06037292924950175, 0.0965966867992028, 0.0724475150994021, 0.01207458584990035, 0.9902124597710636, 0.1309884840916949, 0.5239539363667796, 0.3274712102292372, 0.8151099383105713, 0.5620692215049371, 0.9977784523876373, 0.21121396200918394, 0.4224279240183679, 0.21121396200918394, 0.16049483093194186, 0.8024741546597093, 0.03209896618638837, 0.8411751804925616, 0.16823503609851231, 0.48975337398779106, 0.6268171104925258, 0.10275690335943045, 0.1541353550391457, 0.04110276134377218, 0.07192983235160132, 0.8421367562403083, 0.7453871856293507, 0.7276528663193874, 0.7925125661592256, 0.1981281415398064, 0.7090614664996956, 0.5290657688753273, 0.44890428874270194, 0.01603229602652507, 0.01603229602652507, 0.7934205155666444, 0.19018099656149876, 0.3803619931229975, 0.3803619931229975, 0.951606009369627, 0.8158141504040716, 0.15539317150553744, 0.3455744242213883, 0.394942199110158, 0.29620664933261853, 0.9365839927609879, 0.9354944934391616, 0.9487995356620087, 0.47605172589612094, 0.2094627593942932, 0.3046731045735174, 0.9209460049207246, 0.9206374190846328, 0.3840303317264143, 0.3840303317264143, 0.958247100195754, 0.9581000212641066, 0.7930487331109816, 0.86575518731369, 0.9722713529189653, 0.8408721533431188, 0.9304570841401075, 0.7500134961747296, 0.9660250037061181, 0.8246116143852678, 0.9297725849327484, 0.45924634258631486, 0.09983616143180758, 0.31947571658178425, 0.07986892914544606, 0.019967232286361516, 0.019967232286361516, 0.3783398216213236, 0.5675097324319854, 0.7578590004011055, 0.17573542038286505, 0.04393385509571626, 0.010983463773929066, 0.9895630537142013, 0.9437967296390573, 0.047189836481952865, 0.5711748007727266, 0.2855874003863633, 0.14279370019318166, 0.4069660420889309, 0.20348302104446545, 0.20348302104446545, 0.990165833694824, 0.9880730191398115, 0.9534803655065807, 0.050116532850942565, 0.9522141241679087, 0.7905089044557926, 0.14822041958546112, 0.049406806528487036, 0.4265520041093703, 0.39100600376692274, 0.1777300017122376, 0.994673700239605, 0.8396040580091538, 0.041980202900457696, 0.12594060870137308, 0.6369358937050845, 0.21231196456836152, 0.8948620196977827, 0.09587807353904813, 0.9777330687587451, 0.07996264255430799, 0.8795890680973879, 0.6768247547221925, 0.4619008444098312, 0.30151860676752873, 0.2245351326992235, 0.0064152895056921, 0.9703051190667532, 0.9030816759632, 0.5287800253082859, 0.523468532139124, 0.523468532139124, 0.11650540727620033, 0.34951622182860104, 0.5825270363810017, 0.7338635138013712, 0.20967528965753462, 0.9692990843197226, 0.4021864681825833, 0.5831703788647458, 0.5005534071451642, 0.4936965111568743, 0.0045712639921932805, 0.17629155224515797, 0.7051662089806319, 0.950521261501701, 0.5168651616677689, 0.18340376704340186, 0.033346139462436704, 0.23342297623705693, 0.016673069731218352, 0.16712173770846617, 0.651774777063018, 0.17547782459388947, 0.887081943775978, 0.49948378791992337, 0.4585993618381878, 0.4585993618381878, 0.8480745869767611, 0.12115351242525157, 0.017307644632178796, 0.9553426181134839, 0.5498963427869218, 0.1481882296872951, 0.5927529187491803, 0.9598322466228122, 0.734764501245695, 0.9039963380338291, 0.2597476597198376, 0.5380487237053779, 0.018553404265702688, 0.1484272341256215, 0.9702798723805816, 0.9890090895894913, 0.9448541446198767, 0.8911634155328959, 0.9307988155080322, 0.9507849144806718, 0.9827254834160507, 0.9857770184245845, 0.9730195032229431, 0.8694614327606139, 0.9883142239029382, 0.825254610054663, 0.8611780093370679, 0.3596297416431348, 0.31967088146056427, 0.1198765805477116, 0.19979430091285266, 0.8411109263746713, 0.9722587547166145, 0.9079476515027346, 0.7510732604718708, 0.08680399747239301, 0.6944319797791441, 0.08680399747239301, 0.08680399747239301, 0.43005261524948496, 0.5734034869993133, 0.9684581089553729, 0.7297247348109315, 0.4882465459383783, 0.6628034314033049, 0.2530599249252141, 0.6959147935443388, 0.9823172478314278, 0.9413955264607905, 0.030367597627767435, 0.9410512048777054, 0.9723733467586165, 0.6324038259831177, 0.5600932930921761, 0.42006996981913214, 0.9955018881511902, 0.9878233471461754, 0.30667057334520653, 0.30667057334520653, 0.30667057334520653, 0.30667057334520653, 0.8800222599221262, 0.10000252953660525, 0.7381997798777749, 0.2559092570242953, 0.9736349872475323, 0.8123940303664839, 0.8991645301228312, 0.6439804934959069, 0.8070164888988339, 0.9744777649080008, 0.35854809628678525, 0.5019673348014994, 0.10756442888603557, 0.03585480962867852, 0.48017801481571215, 0.24008900740785608, 0.12004450370392804, 0.12004450370392804, 0.7933455842950968, 0.9374167217365066, 0.793047660618804, 0.8915814937481603, 0.9362508837301652, 0.9703426235080816, 0.6877380261691759, 0.25008655497060944, 0.9556085275850509, 0.9591771934855327, 0.6201976732428248, 0.9366909890696006, 0.9280385539023058, 0.9572504795638904, 0.896991912995645, 0.37071754890024716, 0.617862581500412, 0.5198713029115286, 0.3465808686076857, 0.04813623175106746, 0.009627246350213493, 0.04813623175106746, 0.019254492700426985, 0.9557008918691727, 0.2551645144901442, 0.1275822572450721, 0.5741201576028244, 0.9659816525675184, 0.8575156444716526, 0.9938245466347319, 0.9112271835318254, 0.37081202913073585, 0.1236040097102453, 0.15107156742363312, 0.34334447141734803, 0.0068668894283469605, 0.6361851081162666, 0.8314306262935494, 0.8415279819010396, 0.4533877712165075, 0.528952399752592, 0.5284869484515936, 0.8502989016589694, 0.46093506735569767, 0.04190318794142706, 0.46093506735569767, 0.8141853260839431, 0.8644055751598702, 0.9522795516617308, 0.6204807477476402, 0.3102403738738201, 0.8421367562403083, 0.9841626411192296, 0.9866197337105599, 0.7326417619013944, 0.9654038219658355, 0.9107113845081154, 0.6415839591591233, 0.3401167976265232, 0.007729927218784618, 0.3847540811780967, 0.32978921243836856, 0.09618852029452417, 0.19237704058904834, 0.013741217184932025, 0.4554626308983048, 0.23857566380387393, 0.08675478683777234, 0.17350957367554468, 0.04337739341888617, 0.8360075987065817, 0.8263613388097841, 0.9408922807159437, 0.972024957282141, 0.917303337025257, 0.9925476708736626, 0.2660899521660103, 0.12094997825727741, 0.604749891286387, 0.1352423894267374, 0.8114543365604244, 0.9409727661997352, 0.032447336765508115, 0.8413965284074886, 0.7712712525940154, 0.11018160751343077, 0.11018160751343077, 0.7538407152462844, 0.5405212757664727, 0.3815444299528043, 0.03179536916273369, 0.03179536916273369, 0.487715680397932, 0.6074460571631449, 0.22467182936171115, 0.1664235773049712, 0.5112905990101513, 0.07669358985152269, 0.07669358985152269, 0.05112905990101512, 0.25564529950507564, 0.5487174186715417, 0.27435870933577083, 0.5416789335231362, 0.30530994434940406, 0.039394831528955364, 0.10833578670462725, 0.7650548934559304, 0.9209270872809219, 0.3499919855334565, 0.3499919855334565, 0.9434490536674007, 0.058965565854212545, 0.8912358653152769, 0.07426965544293974, 0.872842584487305, 0.8409283091992968, 0.07139957342258181, 0.03173314374336969, 0.03966642967921212, 0.02379985780752727, 0.8418162540023173, 0.4698988499842365, 0.4698988499842365, 0.02936867812401478, 0.9682376066369232, 0.685992094759602, 0.177849802345082, 0.050814229241452, 0.06775230565526934, 0.016938076413817334, 0.6784526312291705, 0.36963313702033757, 0.6255330011113405, 0.43438645434577616, 0.5542172003721971, 0.007489421626651313, 0.9642149427973274, 0.956498879102352, 0.8034802619763961, 0.595347052847596, 0.297673526423798, 0.8299823675547023, 0.9852190951248087, 0.9814599548739424, 0.4077047752208036, 0.3611099437669975, 0.2213254494055791, 0.9684775037478501, 0.8027072325836101, 0.1763018693267717, 0.08815093466338585, 0.7492829446387796, 0.7222463299633654, 0.7837038042746224, 0.9021070067087277, 0.5319131226607922, 0.3108583184381253, 0.01381592526391668, 0.13125129000720845, 0.00690796263195834, 0.9745914820718221, 0.9386708277142968, 0.492776317262059, 0.492776317262059, 0.6828930921305445, 0.943687091489371, 0.7381759111132812, 0.980067735738982, 0.22887516112135123, 0.5264128705791078, 0.22887516112135123, 0.19214613041501108, 0.5078147732396722, 0.054898894404288884, 0.0686236180053611, 0.10979778880857777, 0.0686236180053611, 0.15791652808884082, 0.026319421348140134, 0.8159020617923441, 0.8925588259203596, 0.4036705429540303, 0.5651387601356425, 0.08755030854083287, 0.15321303994645752, 0.08755030854083287, 0.6785148911914547, 0.45142050460277794, 0.17034736022746338, 0.34069472045492677, 0.034069472045492676, 0.8256329575623738, 0.979180516736527, 0.8701921552114501, 0.7432308442724673, 0.1758180491827342, 0.0479503770498366, 0.0319669180332244, 0.0079917295083061, 0.7787280075988261, 0.41258518453024373, 0.5501135793736583, 0.6200601817789338, 0.9635728924138308, 0.7173295077943783, 0.9679110998383981, 0.8754504612215892, 0.6783088035074116, 0.01256127413902614, 0.2889093051976012, 0.01256127413902614, 0.8680643405930375, 0.5067287067259162, 0.5067287067259162, 0.43161258861768126, 0.19182781716341388, 0.09591390858170694, 0.23978477145426735, 0.5280187082649583, 0.04800170075135985, 0.43201530676223865, 0.9286143565322105, 0.9461932695433387, 0.9518101140978718, 0.03965875475407799, 0.9807636612718914, 0.8024348762620425, 0.5596092716540944, 0.6755850345917797, 0.9827254125294724, 0.3476956552577093, 0.20059364726406306, 0.12035618835843784, 0.3075769258048967, 0.013372909817604204, 0.34041997544890934, 0.177610421973344, 0.20721215896890136, 0.23681389596445868, 0.029601736995557335, 0.014800868497778667, 0.1921369648056877, 0.1921369648056877, 0.1921369648056877, 0.3842739296113754, 0.9635821331015261, 0.8915397927851851, 0.9118983336180805, 0.08289984851073459, 0.28241680793918755, 0.5648336158783751, 0.4688978327067107, 0.5626773992480528, 0.9465389922004663, 0.8070921305329737, 0.9785745654295668, 0.8813833800194251, 0.08012576181994774, 0.026708587273315914, 0.8935062471273112, 0.7289916285371169, 0.25729116301310007, 0.8763704384885924, 0.7805190530479623, 0.09933878856974067, 0.042573766529888854, 0.014191255509962952, 0.05676502203985181, 0.8766543384266527, 0.34771031198624547, 0.6084930459759296, 0.7175074223469091, 0.11263260709000067, 0.9010608567200054, 0.9399824978418874, 0.6246175338376292, 0.5596470909303235, 0.737833542435424, 0.08680394616887341, 0.10416473540264809, 0.06944315693509873, 0.00868039461688734, 0.6288726322355552, 0.3144363161177776, 0.8530531748629255, 0.42428460240193233, 0.5568735406525361, 0.9713999314287208, 0.9727292947262686, 0.860743700064351, 0.2209700586932437, 0.13258203521594622, 0.4419401173864874, 0.17677604695459495, 0.48235277585917075, 0.9735396681345537, 0.9395546266050667], \"Term\": [\"absent\", \"abundance\", \"accomplish\", \"acetyl\", \"acetyl\", \"acetylate\", \"acetylation\", \"acetylation\", \"acid\", \"acid\", \"acid\", \"acquire\", \"acquire\", \"action\", \"active\", \"active\", \"active\", \"activity\", \"activity\", \"activity\", \"activity\", \"addition\", \"adjust\", \"adult\", \"adult\", \"adult\", \"adulthood\", \"adulthood\", \"age\", \"age\", \"age\", \"aim\", \"allele\", \"allosteric\", \"alpha\", \"alpha\", \"alpha\", \"also\", \"also\", \"also\", \"also\", \"alteration\", \"alteration\", \"alteration\", \"alternative\", \"alternative\", \"amplification\", \"amplification\", \"analysis\", \"analysis\", \"analysis\", \"analysis\", \"androgenetic\", \"anti\", \"antisense\", \"appear\", \"appear\", \"application\", \"application\", \"appropriate\", \"arabidopsis\", \"argue\", \"arm\", \"assay\", \"assay\", \"assembly\", \"assembly\", \"assembly\", \"assess\", \"assist\", \"associate\", \"associate\", \"associate\", \"associate\", \"associate\", \"attribute\", \"autophagy\", \"base\", \"base\", \"batch\", \"beta\", \"bind\", \"bind\", \"bind\", \"bind\", \"biological\", \"biological\", \"biological\", \"biological\", \"biological\", \"biological\", \"biological\", \"blockade\", \"blood\", \"blood\", \"body\", \"body\", \"body\", \"cadmium\", \"cancer\", \"cancer\", \"cancer\", \"capacity\", \"capacity\", \"capillary\", \"cardiac\", \"catalytic\", \"cause\", \"cause\", \"cd\", \"cd\", \"cell\", \"cell\", \"cell\", \"cell\", \"cell\", \"cell\", \"cellular\", \"cellular\", \"cellular\", \"cellular\", \"cellular\", \"cellular\", \"cellular\", \"cellular\", \"centromere\", \"change\", \"change\", \"change\", \"change\", \"characterize\", \"checkpoint\", \"chemical\", \"childhood\", \"childhood\", \"chimera\", \"choice\", \"chromatin\", \"chromatin\", \"chromatin\", \"chromatin\", \"chromodomain\", \"chromosomal\", \"chromosomal\", \"chromosomal\", \"chromosome\", \"chromosome\", \"chromosome\", \"chromosome\", \"chronological\", \"ci\", \"class\", \"class\", \"clear\", \"clonal\", \"clonal\", \"clone\", \"coa\", \"cohort\", \"cohort\", \"colorectal\", \"com\", \"communication\", \"comparable\", \"comparison\", \"compartment\", \"compartment\", \"complex\", \"complex\", \"complex\", \"complex\", \"complex\", \"complexe\", \"component\", \"component\", \"component\", \"component\", \"consideration\", \"contain\", \"contain\", \"continue\", \"continuously\", \"contrast\", \"conversion\", \"cooperation\", \"correlated\", \"cpg_island\", \"crp\", \"currently\", \"cyclin\", \"cytokine\", \"cytoplasmic\", \"cytoplasmic\", \"cytosine\", \"date\", \"deacetylase\", \"debate\", \"defect\", \"defence\", \"defense\", \"deficient\", \"definition\", \"deletion\", \"demethylation\", \"designate\", \"designate\", \"development\", \"development\", \"development\", \"development\", \"different\", \"different\", \"different\", \"different\", \"differentiate\", \"differentiation\", \"differentiation\", \"differentiation\", \"directly\", \"directly\", \"discover\", \"discover\", \"disease\", \"disease\", \"disease\", \"disease\", \"divergent\", \"divergent\", \"diversity\", \"diversity\", \"division\", \"division\", \"dna\", \"dna\", \"domain\", \"domain\", \"domain\", \"dose\", \"double\", \"drosophila\", \"drosophila\", \"dysfunction\", \"dysfunction\", \"dysregulate\", \"dysregulation\", \"early\", \"early\", \"early\", \"early\", \"effect\", \"effect\", \"effect\", \"effect\", \"effect\", \"element\", \"element\", \"element\", \"elsevier\", \"embryo\", \"embryogenesis\", \"embryonic\", \"embryonic\", \"embryonic\", \"embryonic_stem\", \"embryonic_stem\", \"emt\", \"end\", \"end\", \"ensembl\", \"ensembl_org\", \"environment\", \"environment\", \"epiblast\", \"epigenetic\", \"epigenetic\", \"epigenetic\", \"epigenetic\", \"epigenetic\", \"epigenetic\", \"epilcs\", \"es\", \"estimate\", \"estimate\", \"etiology\", \"euchromatin\", \"eukaryotic\", \"eukaryotic\", \"evaluate\", \"evaluate\", \"event\", \"event\", \"evolution\", \"evolution\", \"evolutionary\", \"evolutionary\", \"evolutionary\", \"examine\", \"exception\", \"exert\", \"experiment\", \"express\", \"express\", \"expression\", \"expression\", \"expression\", \"expression\", \"ext\", \"extensive\", \"factor\", \"factor\", \"factor\", \"factor\", \"factor\", \"family\", \"family\", \"family\", \"fate\", \"fate\", \"fate\", \"fetal\", \"fibroblast\", \"fibroblast\", \"fibrosis\", \"find\", \"find\", \"find\", \"folate\", \"foreign\", \"formation\", \"fragment\", \"fragment\", \"full\", \"full\", \"full\", \"function\", \"function\", \"function\", \"function\", \"function\", \"functionally\", \"functionally\", \"functionally\", \"fungi\", \"gamete\", \"gap\", \"gene\", \"gene\", \"gene\", \"gene\", \"gene\", \"general\", \"general\", \"general\", \"generate\", \"generate\", \"generate\", \"genetic\", \"genetic\", \"genetic\", \"genetic\", \"genetic\", \"genetically\", \"genetically\", \"genome\", \"genome\", \"genome\", \"genotype\", \"germ\", \"germline\", \"germline\", \"grch\", \"growth\", \"growth\", \"guideline\", \"hallmark\", \"heart\", \"heterochromatin\", \"heterochromatin\", \"heterogeneity\", \"heterogeneity\", \"heterogeneity\", \"histologically\", \"histone\", \"histone\", \"histone\", \"hold\", \"homologous\", \"homologous\", \"hpgc\", \"hpgclcs\", \"hpgcs\", \"http\", \"human\", \"human\", \"human\", \"human\", \"human\", \"human\", \"hybrid\", \"hybridization\", \"hybridization\", \"hybridization\", \"hypertrophy\", \"hypomethylation\", \"hypothesis\", \"idea\", \"identify\", \"identify\", \"identify\", \"identify\", \"identify\", \"igf\", \"immune\", \"immune\", \"immune\", \"immune_checkpoint\", \"importantly\", \"imprint\", \"improve\", \"improve\", \"improve\", \"inactivation\", \"inactivation\", \"inactivation\", \"inactive\", \"inactive\", \"incidence\", \"include\", \"include\", \"include\", \"include\", \"include\", \"incorrect\", \"incrna\", \"indispensable\", \"individual\", \"individual\", \"individually\", \"induce\", \"induce\", \"induce\", \"induce\", \"infinium\", \"inflammatory\", \"inflammatory\", \"inflammatory\", \"inherit\", \"inheritance\", \"inheritance\", \"inhibitor\", \"inhibitor\", \"inhibitor\", \"ink\", \"innate\", \"interact\", \"interaction\", \"interaction\", \"interaction\", \"intercellular\", \"interval\", \"intra\", \"intra\", \"intrinsic\", \"investigate\", \"investigator\", \"ip\", \"isolate\", \"iv\", \"kap\", \"kinase\", \"kinetochore\", \"klf\", \"layer\", \"lead\", \"lead\", \"lead\", \"lead\", \"lead\", \"lead\", \"length\", \"length\", \"level\", \"level\", \"level\", \"level\", \"line\", \"link\", \"link\", \"lncrna\", \"lncrna\", \"lncrna\", \"localization\", \"localization\", \"localization\", \"locus\", \"loss\", \"lys\", \"lysine\", \"lysine\", \"maintain\", \"maintain\", \"maintain\", \"major\", \"major\", \"major\", \"male\", \"mammalian\", \"mammalian\", \"mammalian\", \"mass\", \"mass\", \"maternal\", \"maternal\", \"maternally\", \"measure\", \"measure\", \"measurement\", \"mechanism\", \"mechanism\", \"mechanism\", \"mechanism\", \"mecp\", \"member\", \"mental\", \"meta\", \"meta\", \"metabolic\", \"metabolic\", \"metabolic\", \"method\", \"method\", \"methyl\", \"methylate\", \"methylate\", \"methylation\", \"methylation\", \"methylation\", \"microarray\", \"microarray\", \"mirna\", \"model\", \"model\", \"model\", \"model\", \"model\", \"modification\", \"modification\", \"modification\", \"modifying\", \"modular\", \"modulator\", \"modulator\", \"molecular\", \"molecular\", \"molecular\", \"monitor\", \"monocyte\", \"moreover\", \"moreover\", \"morphological\", \"mortality\", \"motif\", \"mouse\", \"mouse\", \"mouse\", \"mouse\", \"mutant\", \"mutation\", \"ncrna\", \"necrosis\", \"neural\", \"neuronal\", \"newborn\", \"newly\", \"non\", \"noncoding_rna\", \"normal\", \"notable\", \"notion\", \"nuclear\", \"nuclear\", \"nuclear\", \"nuclear\", \"nucleic\", \"nucleosome\", \"numerous\", \"nutrient\", \"oncogene\", \"oncogene\", \"oncogene\", \"oncogene\", \"oncogenic\", \"oncogenic\", \"origin\", \"originate\", \"outside\", \"pacbio\", \"pair\", \"pair\", \"parental\", \"part\", \"part\", \"partially\", \"particular\", \"partition\", \"past\", \"past\", \"paternal\", \"paternally\", \"pathological\", \"pathological\", \"pathological\", \"pathological\", \"patient\", \"patient\", \"pattern\", \"pattern\", \"pc\", \"pd\", \"peptide\", \"perturb\", \"pgclcs\", \"pgcs\", \"phenotypic\", \"phenotypic\", \"phenotypic\", \"phenotypic\", \"physiological\", \"physiological\", \"physiological\", \"physiological\", \"picture\", \"pluripotent\", \"polymorphic\", \"pombe\", \"poorly\", \"position\", \"positive\", \"positive\", \"possibility\", \"possible\", \"powerful\", \"predictive\", \"preferential\", \"primary\", \"primordial_germ\", \"probe\", \"probe\", \"process\", \"process\", \"process\", \"process\", \"process\", \"process\", \"produce\", \"progenitor\", \"progenitor\", \"progenitor\", \"progeny\", \"programme\", \"promoter\", \"protect\", \"protein\", \"protein\", \"protein\", \"protein\", \"protein\", \"proteomic\", \"psc\", \"publish\", \"quantitative\", \"quantitative\", \"quite\", \"rare\", \"rate\", \"rate\", \"rate\", \"reaction\", \"reactivation\", \"reactive\", \"read\", \"read\", \"reagent\", \"recruit\", \"recruitment\", \"recurrent\", \"reduce\", \"refer\", \"region\", \"region\", \"region\", \"regulation\", \"regulation\", \"regulation\", \"regulation\", \"regulation\", \"regulatory\", \"regulatory\", \"regulatory\", \"regulatory\", \"regulatory\", \"reinforce\", \"release\", \"reorganization\", \"repair\", \"replicate\", \"replication\", \"repression\", \"repression\", \"repression\", \"repressive\", \"repressive\", \"reprogramme\", \"reprogramme\", \"reprogramming\", \"research\", \"research\", \"research\", \"reset\", \"response\", \"response\", \"response\", \"response\", \"rest\", \"result\", \"result\", \"result\", \"review\", \"review\", \"review\", \"review\", \"review\", \"robust\", \"robust\", \"role\", \"role\", \"role\", \"role\", \"scaffold\", \"screening\", \"search\", \"search\", \"selection\", \"selection\", \"selective\", \"selective\", \"self\", \"sequence\", \"sequence\", \"sequence\", \"sequence\", \"sequence\", \"serine\", \"set\", \"set\", \"set\", \"sexual\", \"show\", \"show\", \"show\", \"show\", \"show\", \"side\", \"signal\", \"signal\", \"silence\", \"silence\", \"silence\", \"silencer\", \"silent\", \"similaritie\", \"simple\", \"simple\", \"simultaneously\", \"sirt\", \"sirtuin\", \"site\", \"site\", \"site\", \"small\", \"sodium\", \"somatic\", \"somatic\", \"somatic\", \"sox\", \"special\", \"speciation\", \"specific\", \"specific\", \"specific\", \"specific\", \"specific\", \"specification\", \"specificity\", \"specify\", \"specify\", \"spectrometry\", \"spermatogenesis\", \"spontaneous\", \"stable\", \"stage\", \"stage\", \"stage\", \"state\", \"state\", \"state\", \"state\", \"state\", \"state\", \"stem\", \"stem\", \"stem\", \"stimulus\", \"structural\", \"structural\", \"structure\", \"structure\", \"structure\", \"structure\", \"study\", \"study\", \"study\", \"study\", \"subcellular\", \"subject\", \"subunit\", \"suggest\", \"suggest\", \"suggest\", \"suggest\", \"suggest\", \"supply\", \"suppressor\", \"suppressor\", \"surveillance\", \"suv\", \"synthesis\", \"tail\", \"take\", \"target\", \"target\", \"target\", \"target\", \"tend\", \"terminal\", \"terminal\", \"therapeutic\", \"therapeutic\", \"therapeutic\", \"therapeutic\", \"therapy\", \"therapy\", \"therapy\", \"think\", \"thymocyte\", \"tissue\", \"tissue\", \"totipotency\", \"totipotent\", \"track\", \"traditional\", \"transcript\", \"transcription\", \"transcription\", \"transcription\", \"transcription\", \"transcription\", \"transcriptional\", \"transcriptional\", \"transcriptional\", \"transcriptional\", \"transcriptional\", \"transcriptional\", \"transcriptome\", \"transcriptome\", \"transcriptome\", \"transcriptome\", \"transfer\", \"transform\", \"transgene\", \"transgene\", \"transient\", \"transient\", \"transmission\", \"transmission\", \"trauma\", \"trial\", \"trigger\", \"tumor\", \"tumor\", \"tumor\", \"tumorigenic\", \"tumour\", \"tumour\", \"turn\", \"type\", \"type\", \"type\", \"type\", \"type\", \"ultimately\", \"underlie\", \"underlie\", \"undetectable\", \"unique\", \"unique\", \"unit\", \"update\", \"upgrade\", \"use\", \"use\", \"use\", \"use\", \"use\", \"value\", \"value\", \"variance\", \"variation\", \"variation\", \"vertebrate\", \"virtually\", \"vitro\", \"vivo\", \"vivo\", \"vivo\", \"vivo\", \"xlink\", \"yeast\", \"zygote\"]}, \"R\": 30, \"lambda.step\": 0.01, \"plot.opts\": {\"xlab\": \"PC1\", \"ylab\": \"PC2\"}, \"topic.order\": [7, 4, 1, 11, 5, 10, 8, 9, 14, 3, 6, 2, 13, 12]};\n",
       "\n",
       "function LDAvis_load_lib(url, callback){\n",
       "  var s = document.createElement('script');\n",
       "  s.src = url;\n",
       "  s.async = true;\n",
       "  s.onreadystatechange = s.onload = callback;\n",
       "  s.onerror = function(){console.warn(\"failed to load library \" + url);};\n",
       "  document.getElementsByTagName(\"head\")[0].appendChild(s);\n",
       "}\n",
       "\n",
       "if(typeof(LDAvis) !== \"undefined\"){\n",
       "   // already loaded: just create the visualization\n",
       "   !function(LDAvis){\n",
       "       new LDAvis(\"#\" + \"ldavis_el247521862265818952181013123\", ldavis_el247521862265818952181013123_data);\n",
       "   }(LDAvis);\n",
       "}else if(typeof define === \"function\" && define.amd){\n",
       "   // require.js is available: use it to load d3/LDAvis\n",
       "   require.config({paths: {d3: \"https://cdnjs.cloudflare.com/ajax/libs/d3/3.5.5/d3.min\"}});\n",
       "   require([\"d3\"], function(d3){\n",
       "      window.d3 = d3;\n",
       "      LDAvis_load_lib(\"https://cdn.rawgit.com/bmabey/pyLDAvis/files/ldavis.v1.0.0.js\", function(){\n",
       "        new LDAvis(\"#\" + \"ldavis_el247521862265818952181013123\", ldavis_el247521862265818952181013123_data);\n",
       "      });\n",
       "    });\n",
       "}else{\n",
       "    // require.js not available: dynamically load d3 & LDAvis\n",
       "    LDAvis_load_lib(\"https://cdnjs.cloudflare.com/ajax/libs/d3/3.5.5/d3.min.js\", function(){\n",
       "         LDAvis_load_lib(\"https://cdn.rawgit.com/bmabey/pyLDAvis/files/ldavis.v1.0.0.js\", function(){\n",
       "                 new LDAvis(\"#\" + \"ldavis_el247521862265818952181013123\", ldavis_el247521862265818952181013123_data);\n",
       "            })\n",
       "         });\n",
       "}\n",
       "</script>"
      ],
      "text/plain": [
       "PreparedData(topic_coordinates=              x         y  topics  cluster       Freq\n",
       "topic                                                \n",
       "6      0.256091 -0.088141       1        1  48.367612\n",
       "3      0.305615 -0.003207       2        1  19.184035\n",
       "0      0.174082 -0.093583       3        1  11.080312\n",
       "10     0.151473  0.283216       4        1   7.554944\n",
       "4      0.024776 -0.055475       5        1   3.843860\n",
       "9     -0.036937 -0.042534       6        1   2.983950\n",
       "7     -0.052501 -0.049104       7        1   2.940217\n",
       "8     -0.116980  0.014150       8        1   1.524989\n",
       "13    -0.126713  0.011294       9        1   1.181064\n",
       "2     -0.120085  0.007117      10        1   0.416468\n",
       "5     -0.122101  0.007234      11        1   0.325019\n",
       "1     -0.113988  0.000230      12        1   0.248758\n",
       "12    -0.111729  0.004433      13        1   0.183537\n",
       "11    -0.111005  0.004370      14        1   0.165235, topic_info=                 Term        Freq       Total Category  logprob  loglift\n",
       "165              cell  427.000000  427.000000  Default  30.0000  30.0000\n",
       "354       methylation  437.000000  437.000000  Default  29.0000  29.0000\n",
       "292         chromatin  151.000000  151.000000  Default  28.0000  28.0000\n",
       "108            cancer  258.000000  258.000000  Default  27.0000  27.0000\n",
       "114               dna  341.000000  341.000000  Default  26.0000  26.0000\n",
       "...               ...         ...         ...      ...      ...      ...\n",
       "3911              bha    0.010733    0.669137  Topic14  -8.3189   2.2729\n",
       "3916      forestomach    0.010370    0.660959  Topic14  -8.3533   2.2508\n",
       "3913  carcinogenicity    0.010370    0.660959  Topic14  -8.3533   2.2508\n",
       "687             adult    0.104475   19.973828  Topic14  -6.0433   1.1523\n",
       "405          hallmark    0.012386    7.213320  Topic14  -8.1756   0.0385\n",
       "\n",
       "[691 rows x 6 columns], token_table=      Topic      Freq        Term\n",
       "term                             \n",
       "503       8  0.674678      absent\n",
       "1855     11  0.807543   abundance\n",
       "1089      4  0.671825  accomplish\n",
       "727       4  0.200359      acetyl\n",
       "727       6  0.801436      acetyl\n",
       "...     ...       ...         ...\n",
       "467       4  0.441940        vivo\n",
       "467       7  0.176776        vivo\n",
       "285       8  0.482353       xlink\n",
       "2532      6  0.973540       yeast\n",
       "2769      2  0.939555      zygote\n",
       "\n",
       "[929 rows x 3 columns], R=30, lambda_step=0.01, plot_opts={'xlab': 'PC1', 'ylab': 'PC2'}, topic_order=[7, 4, 1, 11, 5, 10, 8, 9, 14, 3, 6, 2, 13, 12])"
      ]
     },
     "execution_count": 28,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "pyLDAvis.enable_notebook()\n",
    "vis = pyLDAvis.gensim.prepare(lda_model, corpus, id2word)\n",
    "vis"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "###rename file.html before running command. Search computer and it should be in the user directory.\n",
    "p = pyLDAvis.gensim.prepare(lda_model, corpus, id2word)\n",
    "pyLDAvis.save_html(p, 'file.html')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python 3",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.7.3"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 2
}
